--- 
title: "Genome folding in evolution and disease"
author: "Jonas Ibn-Salem"
date: "`r Sys.Date()`"
site: bookdown::bookdown_site
output: bookdown::gitbook
documentclass: scrreprt
classoption: [
  twoside=true, 
  openright, 
  parskip=full, 
  chapterprefix=true, 
  11pt, 
  headings=normal, 
  bibliography=totoc, 
  listof=totoc, 
  titlepage=on, 
  captions=tableabove, 
  draft=false]
papersize: a4
bibliography: [bib/PhD_toy.bib]
biblio-style: unsrtnat
csl: cell.csl
link-citations: yes
github-repo: ibn-salem/thesis
description: "This is a draft PhD thesis."
---

# Abstract {-}
<!-- biblio-style: unsrtnat -->
```{r, echo = FALSE}
# Set global options to not show code chunks
knitr::opts_chunk$set(echo = FALSE)
```

<!-- - different cell types with same genome -->
<!-- - enhancer-promoter interactions -->
<!-- - Hierarchy of foling and TADs -->
<!-- - example of paralog genes for co-regulation -->
<!-- - Stability in evolution and conserved gene expression -->
<!-- - Dirsuption of TADs by rearragements in diease lead to altered gene expression  -->
<!-- - Computational appraoch for using interaction and TADs to interpret variants -->
<!-- - To improve analysis of genome folding in diverse conditions, we develped 7C. -->

<!-- Chromosome conformation capture experiments coupled with high-throughput sequencing (Hi-C) allow to measure DNA-DNA contacts genome-wide and reviled a highly organized hierarchy of three-dimensional genome organization. -->

The DNA of the human genome has a total length of 2 meters and fits into a 10 nm nucleus. Hi-C experiments probe the three-dimensional genome architecture by genome-wide DNA-DNA contact maps. In these maps, interphase chromosomes fold locally into regions of several hundred kb, called topologically associating domains (TADs). TADs can restrict the interactions of genes with distal regulatory sequences, such as enhancers, and might, therefore, represent important functional units of genomes.

In this thesis, I integrate genome-wide chromatin interaction maps with diverse genomic datasets to analyze the function of TADs for gene regulation during evolution and in disease genomes. More specifically, I show that duplicated gene pairs are enriched for colocalization in the same TAD, share more often common enhancer elements than expected and have increased contact frequencies over large genomic distances. The clustering of functionally related genes within TADs enables concerted gene expression and indicates evolutionary constraints in functional genome organization. Indeed, by analyzing evolutionary rearrangements between human and diverse mammal species, I show that TADs are highly conserved building blocks of genomes across millions of years of evolution. While TADs seems to conserve gene expression profiles between mouse and human, genes in rearranged TADs show divergent expression. Next, I analyze disruption of TADs in whole-genome sequenced subjects with diverse neurodevelopmental phenotypes. While the phenotypes cannot be explained by variants in protein-coding sequences, balanced chromosomal rearrangements disrupt TADs and promoter-enhancer interactions of phenotypically matching genes. This effect mechanism leads to altered gene expression in patient-derived cell lines. Finally, I present a computational tool, 7C, to predict long-range chromatin interactions by largely available ChIP-seq data. This enables the analysis of three-dimensional genome folding in many diverse tissues and conditions.

Together, these results show that TAD disruption is associated with altered gene regulation during evolution and in diseases. TADs are not only structural units of genomes but also important functional building blocks and represent regulatory environments for genes. Therefore, it will be increasingly important to take the three-dimensional genome structure into account, not only in genomic research but also in clinical practice.

