<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>Genome folding in evolution and disease</title>
  <meta name="description" content="This is a draft PhD thesis.">
  <meta name="generator" content="bookdown 0.5.10 and GitBook 2.6.7">

  <meta property="og:title" content="Genome folding in evolution and disease" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="This is a draft PhD thesis." />
  <meta name="github-repo" content="ibn-salem/thesis" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Genome folding in evolution and disease" />
  
  <meta name="twitter:description" content="This is a draft PhD thesis." />
  

<meta name="author" content="Jonas Ibn-Salem">


<meta name="date" content="2018-01-26">

  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  
<link rel="prev" href="1-4-methods-to-probe-the-3d-chromatin-archtiecture.html">
<link rel="next" href="1-6-dynamics-of-chromatin-structure.html">
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />










<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Thesis</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Preface</a></li>
<li class="chapter" data-level="1" data-path="1-introduction.html"><a href="1-introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a><ul>
<li class="chapter" data-level="1.1" data-path="1-1-structure-of-the-intorduction.html"><a href="1-1-structure-of-the-intorduction.html"><i class="fa fa-check"></i><b>1.1</b> Structure of the Intorduction</a></li>
<li class="chapter" data-level="1.2" data-path="1-2-cell-diversity-and-regulation-of-gene-expression.html"><a href="1-2-cell-diversity-and-regulation-of-gene-expression.html"><i class="fa fa-check"></i><b>1.2</b> Cell diversity and regulation of gene expression</a></li>
<li class="chapter" data-level="1.3" data-path="1-3-enhancers.html"><a href="1-3-enhancers.html"><i class="fa fa-check"></i><b>1.3</b> Enhancers</a></li>
<li class="chapter" data-level="1.4" data-path="1-4-methods-to-probe-the-3d-chromatin-archtiecture.html"><a href="1-4-methods-to-probe-the-3d-chromatin-archtiecture.html"><i class="fa fa-check"></i><b>1.4</b> Methods to probe the 3D chromatin archtiecture</a><ul>
<li class="chapter" data-level="1.4.1" data-path="1-4-methods-to-probe-the-3d-chromatin-archtiecture.html"><a href="1-4-methods-to-probe-the-3d-chromatin-archtiecture.html#microscopy-based-techniques-to-visuallize-the-genome-in-3d"><i class="fa fa-check"></i><b>1.4.1</b> Microscopy-based techniques to visuallize the genome in 3D</a></li>
<li class="chapter" data-level="1.4.2" data-path="1-4-methods-to-probe-the-3d-chromatin-archtiecture.html"><a href="1-4-methods-to-probe-the-3d-chromatin-archtiecture.html#proximity-ligation-based-method-to-quantify-chromatin-interactions"><i class="fa fa-check"></i><b>1.4.2</b> Proximity-ligation based method to quantify chromatin interactions</a></li>
</ul></li>
<li class="chapter" data-level="1.5" data-path="1-5-hierarchy-of-chromatin-3d-structure.html"><a href="1-5-hierarchy-of-chromatin-3d-structure.html"><i class="fa fa-check"></i><b>1.5</b> Hierarchy of chromatin 3D structure</a><ul>
<li class="chapter" data-level="1.5.1" data-path="1-5-hierarchy-of-chromatin-3d-structure.html"><a href="1-5-hierarchy-of-chromatin-3d-structure.html#chromosomal-territories-and-inter-chromosomal-contacts"><i class="fa fa-check"></i><b>1.5.1</b> Chromosomal territories and inter-chromosomal contacts</a></li>
<li class="chapter" data-level="1.5.2" data-path="1-5-hierarchy-of-chromatin-3d-structure.html"><a href="1-5-hierarchy-of-chromatin-3d-structure.html#ab-compartments"><i class="fa fa-check"></i><b>1.5.2</b> A/B compartments</a></li>
<li class="chapter" data-level="1.5.3" data-path="1-5-hierarchy-of-chromatin-3d-structure.html"><a href="1-5-hierarchy-of-chromatin-3d-structure.html#topologically-associating-domains-tads"><i class="fa fa-check"></i><b>1.5.3</b> Topologically associating domains (TADs)</a></li>
<li class="chapter" data-level="1.5.4" data-path="1-5-hierarchy-of-chromatin-3d-structure.html"><a href="1-5-hierarchy-of-chromatin-3d-structure.html#chromatin-loops"><i class="fa fa-check"></i><b>1.5.4</b> Chromatin loops</a></li>
</ul></li>
<li class="chapter" data-level="1.6" data-path="1-6-dynamics-of-chromatin-structure.html"><a href="1-6-dynamics-of-chromatin-structure.html"><i class="fa fa-check"></i><b>1.6</b> Dynamics of chromatin structure</a></li>
<li class="chapter" data-level="1.7" data-path="1-7-changes-of-genome-folding-in-evoltuion-and-disease-genomes.html"><a href="1-7-changes-of-genome-folding-in-evoltuion-and-disease-genomes.html"><i class="fa fa-check"></i><b>1.7</b> Changes of genome folding in evoltuion and disease genomes</a></li>
<li class="chapter" data-level="1.8" data-path="1-8-aims-of-this-thesis.html"><a href="1-8-aims-of-this-thesis.html"><i class="fa fa-check"></i><b>1.8</b> Aims of this thesis</a><ul>
<li class="chapter" data-level="" data-path="1-8-aims-of-this-thesis.html"><a href="1-8-aims-of-this-thesis.html#is-the-three-dimensonal-folding-of-genomes-associated-with-co-regulation-of-functionally-related-genes"><i class="fa fa-check"></i>Is the three-dimensonal folding of genomes associated with co-regulation of functionally related genes?</a></li>
<li class="chapter" data-level="" data-path="1-8-aims-of-this-thesis.html"><a href="1-8-aims-of-this-thesis.html#are-tads-functional-buidling-blocks-of-genomes-and-subjected-to-selective-preasure-during-evolution"><i class="fa fa-check"></i>Are TADs functional buidling blocks of genomes and subjected to selective preasure during evolution?</a></li>
<li class="chapter" data-level="" data-path="1-8-aims-of-this-thesis.html"><a href="1-8-aims-of-this-thesis.html#is-the-disruption-of-tads-by-rearrangements-also-associated-to-genetic-diseases"><i class="fa fa-check"></i>Is the disruption of TADs by rearrangements also associated to genetic diseases?</a></li>
<li class="chapter" data-level="" data-path="1-8-aims-of-this-thesis.html"><a href="1-8-aims-of-this-thesis.html#can-chromatin-looping-interactions-be-predicted-by-chip-seq-and-sequence-features"><i class="fa fa-check"></i>Can chromatin looping interactions be predicted by ChIP-seq and sequence features?</a></li>
</ul></li>
<li class="chapter" data-level="1.9" data-path="1-9-structure-of-this-thesis.html"><a href="1-9-structure-of-this-thesis.html"><i class="fa fa-check"></i><b>1.9</b> Structure of this thesis</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="2-ch-paralog-regulaion.html"><a href="2-ch-paralog-regulaion.html"><i class="fa fa-check"></i><b>2</b> Paralog genes in the 3D genome architecture</a><ul>
<li class="chapter" data-level="2.1" data-path="1-introduction.html"><a href="1-introduction.html#introduction"><i class="fa fa-check"></i><b>2.1</b> Introduction</a></li>
<li class="chapter" data-level="2.2" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html"><i class="fa fa-check"></i><b>2.2</b> MATERIALS AND METHODS</a><ul>
<li class="chapter" data-level="2.2.1" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html#selection-of-pairs-of-paralog-genes"><i class="fa fa-check"></i><b>2.2.1</b> Selection of pairs of paralog genes</a></li>
<li class="chapter" data-level="2.2.2" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html#enhancers-to-gene-association"><i class="fa fa-check"></i><b>2.2.2</b> Enhancers to gene association</a></li>
<li class="chapter" data-level="2.2.3" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html#topological-associating-domains"><i class="fa fa-check"></i><b>2.2.3</b> Topological associating domains</a></li>
<li class="chapter" data-level="2.2.4" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html#hi-c-interaction-maps"><i class="fa fa-check"></i><b>2.2.4</b> Hi-C interaction maps</a></li>
<li class="chapter" data-level="2.2.5" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html#randomization"><i class="fa fa-check"></i><b>2.2.5</b> Randomization</a></li>
<li class="chapter" data-level="2.2.6" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html#statistical-tests"><i class="fa fa-check"></i><b>2.2.6</b> Statistical tests</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="2-3-results.html"><a href="2-3-results.html"><i class="fa fa-check"></i><b>2.3</b> RESULTS</a><ul>
<li class="chapter" data-level="2.3.1" data-path="2-3-results.html"><a href="2-3-results.html#distribution-of-paralog-genes-in-the-human-genome"><i class="fa fa-check"></i><b>2.3.1</b> Distribution of paralog genes in the human genome</a></li>
<li class="chapter" data-level="2.3.2" data-path="2-3-results.html"><a href="2-3-results.html#co-expression-of-paralog-gene-pairs-across-tissues"><i class="fa fa-check"></i><b>2.3.2</b> Co-expression of paralog gene pairs across tissues</a></li>
<li class="chapter" data-level="2.3.3" data-path="2-3-results.html"><a href="2-3-results.html#paralog-genes-share-enhancers"><i class="fa fa-check"></i><b>2.3.3</b> Paralog genes share enhancers</a></li>
<li class="chapter" data-level="2.3.4" data-path="2-3-results.html"><a href="2-3-results.html#co-localization-of-paralogs-in-tads"><i class="fa fa-check"></i><b>2.3.4</b> Co-localization of paralogs in TADs</a></li>
<li class="chapter" data-level="2.3.5" data-path="2-3-results.html"><a href="2-3-results.html#distal-paralog-pairs-are-enriched-for-long-range-chromatin-contacts"><i class="fa fa-check"></i><b>2.3.5</b> Distal paralog pairs are enriched for long-range chromatin contacts</a></li>
<li class="chapter" data-level="2.3.6" data-path="2-3-results.html"><a href="2-3-results.html#close-paralogs-have-fewer-contacts-than-expected"><i class="fa fa-check"></i><b>2.3.6</b> Close paralogs have fewer contacts than expected</a></li>
<li class="chapter" data-level="2.3.7" data-path="2-3-results.html"><a href="2-3-results.html#paralogs-in-mouse-and-dog-genome"><i class="fa fa-check"></i><b>2.3.7</b> Paralogs in mouse and dog genome</a></li>
<li class="chapter" data-level="2.3.8" data-path="2-3-results.html"><a href="2-3-results.html#orthologs-of-human-paralogs-show-conserved-co-localization"><i class="fa fa-check"></i><b>2.3.8</b> Orthologs of human paralogs show conserved co-localization</a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="2-4-discussion.html"><a href="2-4-discussion.html"><i class="fa fa-check"></i><b>2.4</b> DISCUSSION</a></li>
<li class="chapter" data-level="2.5" data-path="2-5-conclusion.html"><a href="2-5-conclusion.html"><i class="fa fa-check"></i><b>2.5</b> CONCLUSION</a></li>
<li class="chapter" data-level="2.6" data-path="2-6-acknowledgements.html"><a href="2-6-acknowledgements.html"><i class="fa fa-check"></i><b>2.6</b> ACKNOWLEDGEMENTS</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="3-ch-TAD-evolution.html"><a href="3-ch-TAD-evolution.html"><i class="fa fa-check"></i><b>3</b> Stability of TADs in evolution</a><ul>
<li class="chapter" data-level="" data-path="3-ch-TAD-evolution.html"><a href="3-ch-TAD-evolution.html#abstract"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="" data-path="3-ch-TAD-evolution.html"><a href="3-ch-TAD-evolution.html#keywords"><i class="fa fa-check"></i>Keywords</a></li>
<li class="chapter" data-level="3.1" data-path="3-1-introduction-1.html"><a href="3-1-introduction-1.html"><i class="fa fa-check"></i><b>3.1</b> Introduction</a></li>
<li class="chapter" data-level="3.2" data-path="3-2-results-1.html"><a href="3-2-results-1.html"><i class="fa fa-check"></i><b>3.2</b> Results</a><ul>
<li class="chapter" data-level="3.2.1" data-path="3-2-results-1.html"><a href="3-2-results-1.html#identification-of-evolutionary-rearrangement-breakpoints-from-whole-genome-alignments"><i class="fa fa-check"></i><b>3.2.1</b> Identification of evolutionary rearrangement breakpoints from whole-genome alignments</a></li>
<li class="chapter" data-level="3.2.2" data-path="3-2-results-1.html"><a href="3-2-results-1.html#rearrangement-breakpoints-are-enriched-at-tad-boundaries"><i class="fa fa-check"></i><b>3.2.2</b> Rearrangement breakpoints are enriched at TAD boundaries</a></li>
<li class="chapter" data-level="3.2.3" data-path="3-2-results-1.html"><a href="3-2-results-1.html#clusters-of-conserved-non-coding-elements-are-depleted-for-rearrangement-breakpoints"><i class="fa fa-check"></i><b>3.2.3</b> Clusters of conserved non-coding elements are depleted for rearrangement breakpoints</a></li>
<li class="chapter" data-level="3.2.4" data-path="3-2-results-1.html"><a href="3-2-results-1.html#rearranged-tads-are-associated-with-divergent-gene-expression-between-species"><i class="fa fa-check"></i><b>3.2.4</b> Rearranged TADs are associated with divergent gene expression between species</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="3-3-discussion-1.html"><a href="3-3-discussion-1.html"><i class="fa fa-check"></i><b>3.3</b> Discussion</a></li>
<li class="chapter" data-level="3.4" data-path="3-4-conclusion-1.html"><a href="3-4-conclusion-1.html"><i class="fa fa-check"></i><b>3.4</b> Conclusion</a></li>
<li class="chapter" data-level="3.5" data-path="3-5-methods.html"><a href="3-5-methods.html"><i class="fa fa-check"></i><b>3.5</b> Methods</a><ul>
<li class="chapter" data-level="3.5.1" data-path="3-5-methods.html"><a href="3-5-methods.html#rearrangement-breakpoints-from-whole-genome-alignments"><i class="fa fa-check"></i><b>3.5.1</b> Rearrangement breakpoints from whole-genome alignments</a></li>
<li class="chapter" data-level="3.5.2" data-path="3-5-methods.html"><a href="3-5-methods.html#topologically-associating-domains-and-contact-domains"><i class="fa fa-check"></i><b>3.5.2</b> Topologically associating domains and contact domains</a></li>
<li class="chapter" data-level="3.5.3" data-path="3-5-methods.html"><a href="3-5-methods.html#breakpoint-distributions-at-tads"><i class="fa fa-check"></i><b>3.5.3</b> Breakpoint distributions at TADs</a></li>
<li class="chapter" data-level="3.5.4" data-path="3-5-methods.html"><a href="3-5-methods.html#quantification-of-breakpoint-enrichment"><i class="fa fa-check"></i><b>3.5.4</b> Quantification of breakpoint enrichment</a></li>
<li class="chapter" data-level="3.5.5" data-path="3-5-methods.html"><a href="3-5-methods.html#expression-data-for-mouse-and-human-orthologs"><i class="fa fa-check"></i><b>3.5.5</b> Expression data for mouse and human orthologs</a></li>
<li class="chapter" data-level="3.5.6" data-path="3-5-methods.html"><a href="3-5-methods.html#classification-of-tads-and-genes-according-to-rearrangements-and-grbs"><i class="fa fa-check"></i><b>3.5.6</b> Classification of TADs and genes according to rearrangements and GRBs</a></li>
<li class="chapter" data-level="3.5.7" data-path="3-5-methods.html"><a href="3-5-methods.html#source-code-and-implementation-details"><i class="fa fa-check"></i><b>3.5.7</b> Source code and implementation details</a></li>
</ul></li>
<li class="chapter" data-level="3.6" data-path="3-6-declarations.html"><a href="3-6-declarations.html"><i class="fa fa-check"></i><b>3.6</b> Declarations</a><ul>
<li class="chapter" data-level="3.6.1" data-path="3-6-declarations.html"><a href="3-6-declarations.html#ethics-approval-and-consent-to-participate"><i class="fa fa-check"></i><b>3.6.1</b> Ethics approval and consent to participate</a></li>
<li class="chapter" data-level="3.6.2" data-path="3-6-declarations.html"><a href="3-6-declarations.html#consent-for-publication"><i class="fa fa-check"></i><b>3.6.2</b> Consent for publication</a></li>
<li class="chapter" data-level="3.6.3" data-path="3-6-declarations.html"><a href="3-6-declarations.html#availability-of-data-and-material"><i class="fa fa-check"></i><b>3.6.3</b> Availability of data and material</a></li>
<li class="chapter" data-level="3.6.4" data-path="3-6-declarations.html"><a href="3-6-declarations.html#competing-interests"><i class="fa fa-check"></i><b>3.6.4</b> Competing interests</a></li>
<li class="chapter" data-level="3.6.5" data-path="3-6-declarations.html"><a href="3-6-declarations.html#funding"><i class="fa fa-check"></i><b>3.6.5</b> Funding</a></li>
<li class="chapter" data-level="3.6.6" data-path="3-6-declarations.html"><a href="3-6-declarations.html#authors-contributions"><i class="fa fa-check"></i><b>3.6.6</b> Authors‘ contributions</a></li>
<li class="chapter" data-level="3.6.7" data-path="3-6-declarations.html"><a href="3-6-declarations.html#acknowledgments"><i class="fa fa-check"></i><b>3.6.7</b> Acknowledgments</a></li>
</ul></li>
<li class="chapter" data-level="3.7" data-path="3-7-tables.html"><a href="3-7-tables.html"><i class="fa fa-check"></i><b>3.7</b> Tables</a><ul>
<li class="chapter" data-level="3.7.1" data-path="3-7-tables.html"><a href="3-7-tables.html#table-1"><i class="fa fa-check"></i><b>3.7.1</b> Table 1</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="4-ch-position-effect.html"><a href="4-ch-position-effect.html"><i class="fa fa-check"></i><b>4</b> Position effects of rearrangements in disease genomes</a></li>
<li class="chapter" data-level="5" data-path="5-ch-loop-prediction.html"><a href="5-ch-loop-prediction.html"><i class="fa fa-check"></i><b>5</b> Prediction of chromatin looping interactions</a></li>
<li class="chapter" data-level="6" data-path="6-discussion-2.html"><a href="6-discussion-2.html"><i class="fa fa-check"></i><b>6</b> Discussion</a><ul>
<li class="chapter" data-level="6.1" data-path="6-1-further-directions.html"><a href="6-1-further-directions.html"><i class="fa fa-check"></i><b>6.1</b> Further directions</a></li>
<li class="chapter" data-level="6.2" data-path="6-2-conclusions.html"><a href="6-2-conclusions.html"><i class="fa fa-check"></i><b>6.2</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="appendix"><span><b>Appendix</b></span></li>
<li class="chapter" data-level="A" data-path="A-supporting-information-co-regulation-of-paralog-genes-in-the-three-dimensional-chromatin-architecture.html"><a href="A-supporting-information-co-regulation-of-paralog-genes-in-the-three-dimensional-chromatin-architecture.html"><i class="fa fa-check"></i><b>A</b> Supporting Information: Co-regulation of paralog genes in the three-dimensional chromatin architecture</a></li>
<li class="chapter" data-level="B" data-path="B-supplementary-data-evolutionary-stability-of-topologically-associating-domains-is-associated-with-conserved-gene-regulation.html"><a href="B-supplementary-data-evolutionary-stability-of-topologically-associating-domains-is-associated-with-conserved-gene-regulation.html"><i class="fa fa-check"></i><b>B</b> Supplementary Data: Evolutionary stability of topologically associating domains is associated with conserved gene regulation</a><ul>
<li class="chapter" data-level="B.1" data-path="B-1-TadEvoSupTab.html"><a href="B-1-TadEvoSupTab.html"><i class="fa fa-check"></i><b>B.1</b> Supplementary Tables</a></li>
<li class="chapter" data-level="B.2" data-path="B-2-supplementary-figures.html"><a href="B-2-supplementary-figures.html"><i class="fa fa-check"></i><b>B.2</b> Supplementary Figures</a></li>
</ul></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Build with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Genome folding in evolution and disease</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="hierarchy-of-chromatin-3d-structure" class="section level2">
<h2><span class="header-section-number">1.5</span> Hierarchy of chromatin 3D structure</h2>
<ul>
<li>Chromosomal Territories</li>
<li>A/B-Compartments</li>
<li>TADs</li>
<li>Chromatin Loops
<ul>
<li>Enhancer-Promoter loops</li>
<li>Gene Loops</li>
<li>Architectural loops</li>
</ul></li>
<li>Architectural Proteins</li>
<li>(Loop extrusion model)</li>
<li>Nucleosomes / 10nm / 30nm fibers</li>
</ul>
<!-- ---------------------------------------------------------------- -->
<div class="figure" style="text-align: center"><span id="fig:GenomeHierarchy"></span>
<img src="figures/Bonev2016Fig2.png" alt="(ref:GenomeHierarchy)" width="80%" />
<p class="caption">
Figure .: (ref:GenomeHierarchy)
</p>
</div>
<p>(ref:GenomeHierarchy) <strong>Hierarchical organization of chromatin three-dimensional chromatin architecture</strong> <strong>(A)</strong><br />
Figure source <span class="citation">(Bonev and Cavalli, <a href="#ref-Bonev2016">2016</a>)</span> <a href="https://www.nature.com/articles/nrg.2016.112/figures/2" class="uri">https://www.nature.com/articles/nrg.2016.112/figures/2</a>. <!-- ---------------------------------------------------------------- --></p>
<div id="chromosomal-territories-and-inter-chromosomal-contacts" class="section level3">
<h3><span class="header-section-number">1.5.1</span> Chromosomal territories and inter-chromosomal contacts</h3>
<p>The eukaryotic genome is highly organized in the interpahse nucelus. Chromosomes occupy distinct spatial regions, called chromosome territories, and intermingle less than one would expect by chance <span class="citation">(Cremer and Cremer, <a href="#ref-Cremer2001">2001</a>)</span>. This was first observed using imaging based approaches, and is reflected in Hi-C interaction maps, where inter-chromosomal contacts occur order of magnitudes less frequent than intra-chromosomal contacts <span class="citation">(Lieberman-Aiden et al., <a href="#ref-Lieberman-Aiden2009">2009</a>)</span>. However, despite this spatial segregation of chromosome, intermingling of chromosome occurs and is associated with chromosomal translocations <span class="citation">(Branco and Pombo, <a href="#ref-Branco2006">2006</a>, <span class="citation">Roukos et al. (<a href="#ref-Roukos2013">2013</a>)</span>, <span class="citation">Roukos and Misteli (<a href="#ref-Roukos2014">2014</a>)</span>)</span>. There are also specific gene regulatory interactions between different chromosomes <span class="citation">(Laat and Grosveld, <a href="#ref-DeLaat2007">2007</a>)</span>, for example olfactory receptor genes cluster densely in the nucleus of olfactory neurons to facilitate monoallelic expression of a single receptor gene per cell <span class="citation">(Monahan and Lomvardas, <a href="#ref-Monahan2015">2015</a>)</span>. While there are specific inter-chromosomal contacts, the genome is non-randomly organized in chromosomes which occupy distinct territories in the spacial nucleus.</p>
</div>
<div id="ab-compartments" class="section level3">
<h3><span class="header-section-number">1.5.2</span> A/B compartments</h3>
<p>The ability to measure genome-wide chromatin contacts using Hi-C, revealed that individual regions on chromosomes segregate by preferential interactions into two compartments mayor clusters, referred to as A/B-compartments <span class="citation">(Lieberman-Aiden et al., <a href="#ref-Lieberman-Aiden2009">2009</a>)</span>. Interestingly, regions A-compartments are associated with active histone-modifications and active transcription, whereas B-compartment is associated with heterochromatin, lamina association, and repressed genes.</p>
<!--  further organized in mega-base scale -->
<!-- A/B-compartments that show specific preferential interaction patterns -->
<!-- and transcriptional activity [@Lieberman-Aiden2009; @Rao2014]. -->
</div>
<div id="topologically-associating-domains-tads" class="section level3">
<h3><span class="header-section-number">1.5.3</span> Topologically associating domains (TADs)</h3>
<p>Compartments could be identified by clustering of long-range interactions in Hi-C maps with bin resolution of 1 Mb. In 2012 higher resolution Hi-C maps of up to 40 kb lead to the identification of genomic regions with preferential interactions wiht them. These genomic regions were termed topologically associating domains (TADs) <span class="citation">(Dixon et al., <a href="#ref-Dixon2012">2012</a>; Nora et al., <a href="#ref-Nora2012">2012</a>; Sexton et al., <a href="#ref-Sexton2012">2012</a>)</span>. They are opperationlly defined as genomic regions with frequent interactions of loci within the domain and decreased interactions across domain boundares <span class="citation">(Merkenschlager and Nora, <a href="#ref-Merkenschlager2016">2016</a>)</span>.</p>
<p>TADs can be identified from Hi-C interaction maps computationally by different algorithms <span class="citation">(Ay and Noble, <a href="#ref-Ay2015">2015</a>)</span>. The directionality index is a score for each bin in the Hi-C matrix that quantifies the number of upstream versus downstream interactions of this bin. Using hidden Markov models TAD boundaries where than identified in regions where DI is changing drastically <span class="citation">(Dixon et al., <a href="#ref-Dixon2012">2012</a>)</span>. Other algorithms compute an insulation score as the extent to which interactions to cross potential TAD boundaries <span class="citation">(Crane et al., <a href="#ref-Crane2015">2015</a>)</span>. Later the Arrowhead algorithm was introduced to find “contact domains” as smaller nested structures along the diagonal of high resolution Hi-C matrices <span class="citation">(Rao et al., <a href="#ref-Rao2014">2014</a>)</span>. Furthermore, when analyzing Hi-C interactions at differnt length scales, hierarchies of TADs and sub-TADs could be identifed that overlap each other <span class="citation">(Filippova et al., <a href="#ref-Filippova2014">2014</a>, <span class="citation">Fraser et al. (<a href="#ref-Fraser2015">2015</a>)</span>)</span>.<br />
The different algorithms and parameters used in each is only one source of variation in reliably identifying TADs. Also the resolution of Hi-C maps, which is mainly defined by sequencing depths but also the Hi-C protocol itself <span class="citation">(Rao et al., <a href="#ref-Rao2014">2014</a>)</span>, as well as different normalization strategies for Hi-C contacts introduce variability <span class="citation">(Forcato et al., <a href="#ref-Forcato2017">2017</a>)</span>. Therefore the number and size of TADs varies between different studies and cannot be directly compared.</p>
<p>The first studies on TADs identifed around 3000 TADs in human and mouse genomes with a median size of ~800 kb <span class="citation">(Dixon et al., <a href="#ref-Dixon2012">2012</a>)</span> and ~100 kb in <em>Drosophila</em> gneomes <span class="citation">(Sexton et al., <a href="#ref-Sexton2012">2012</a>)</span>. Analysis of 1kb or 5kb resolution Hi-C matrixes resulted in nested “contact domains” with median size of 185 kb (range 40 kb - 3 Mb) in human and mouse cells <span class="citation">(Rao et al., <a href="#ref-Rao2014">2014</a>)</span>.</p>
<p>Importantly, TADs might be equivalent to “chromatin domains” of 10 kb - 1 Mb in size detected by microscopy approaches <span class="citation">(Cremer and Cremer, <a href="#ref-Cremer2010">2010</a>; Gibcus and Dekker, <a href="#ref-Gibcus2013">2013</a>)</span>. Annother connection of Hi-C derived interaction maps with previous microscopy observations, is that TADs in <em>Drosophila</em> correspond to bands of polytene chromosomes <span class="citation">(Eagen et al., <a href="#ref-Eagen2015">2015</a>)</span>.</p>
<p>The spatial positioning of TADs correlate with many genomic features measured along the linear genome <span class="citation">(Merkenschlager and Nora, <a href="#ref-Merkenschlager2016">2016</a>)</span>. TAD boudnaries are enriched for binding of “insulator protins”, such as CTCF in mammals and CP190 in Drosophila <span class="citation">(Dixon et al., <a href="#ref-Dixon2012">2012</a>; Sexton et al., <a href="#ref-Sexton2012">2012</a>)</span>. Furthermore, TAD boundares are associated with active chromatin, such as H3K4me3 and H3K27me3, DNase I hypersensivity, actie transcription, and house-keeping genes <span class="citation">(Dixon et al., <a href="#ref-Dixon2012">2012</a>)</span>. Furthermore, TADs correspond to regions of early and late replication timing <span class="citation">(Dileep et al., <a href="#ref-Dileep2015">2015</a>; Pope et al., <a href="#ref-Pope2014">2014</a>)</span> and lamina associated domains (LADs) <span class="citation">(Dixon et al., <a href="#ref-Dixon2012">2012</a>)</span>. Importantly, enhancer-promoter interactions seems to be mostly constrained within TADs <span class="citation">(Shen et al., <a href="#ref-Shen2012">2012</a>)</span></p>
<p>There is acumulating evidence, that TADs are fundamental units of chromosome organization <span class="citation">(Dixon et al., <a href="#ref-Dixon2016">2016</a>)</span></p>
<ul>
<li>[X] Definition, number, and size</li>
<li>[X] Algorithms to identify TADs</li>
<li>Functional features</li>
<li>[X] CTCF binding at boundaries</li>
<li>[X] housekeeping genes at boundaries</li>
<li>enhancer-gene associations within TADs</li>
</ul>
</div>
<div id="chromatin-loops" class="section level3">
<h3><span class="header-section-number">1.5.4</span> Chromatin loops</h3>
<p>In summary, these findings suggest a hierarchical organization of chromosome architecture. First, dynamic nucleosome contacts from clutches and fibers. These engage in dynamic long-range chromatin loops, some of which are stabilized by architectural proteins, such as CTCF and cohesin, and lead to the formation of TADs. TADs in tern cluster by their epigenomic type into A/B compartments and coalescence of compartments in the same chromosome forms chromosome territories <span class="citation">(Bonev and Cavalli, <a href="#ref-Bonev2016">2016</a>)</span>.</p>
</div>
</div>
<h3>References</h3>
<div id="refs" class="references">
<div id="ref-Bonev2016">
<p>Bonev, B., and Cavalli, G. (2016). Organization and function of the 3D genome. Nature Reviews Genetics <em>17</em>, 661–678.</p>
</div>
<div id="ref-Cremer2001">
<p>Cremer, T., and Cremer, C. (2001). Chromosome territories, nuclear architecture and gene regulation in mammalian cells. Nature Reviews Genetics <em>2</em>, 292–301.</p>
</div>
<div id="ref-Lieberman-Aiden2009">
<p>Lieberman-Aiden, E., Berkum, N.L. van, Williams, L., Imakaev, M., Ragoczy, T., Telling, A., Amit, I., Lajoie, B.R., Sabo, P.J., Dorschner, M.O., et al. (2009). Comprehensive mapping of long-range interactions reveals folding principles of the human genome. Science (New York, N.Y.) <em>326</em>, 289–293.</p>
</div>
<div id="ref-Branco2006">
<p>Branco, M.R., and Pombo, A. (2006). Intermingling of chromosome territories in interphase suggests role in translocations and transcription-dependent associations. PLoS Biology <em>4</em>, 780–788.</p>
</div>
<div id="ref-Roukos2013">
<p>Roukos, V., Voss, T.C., Schmidt, C.K., Lee, S., Wangsa, D., and Misteli, T. (2013). Spatial dynamics of chromosome translocations in living cells. Science <em>341</em>, 660–664.</p>
</div>
<div id="ref-Roukos2014">
<p>Roukos, V., and Misteli, T. (2014). The biogenesis of chromosome translocations. Nature Cell Biology <em>16</em>, 293–300.</p>
</div>
<div id="ref-DeLaat2007">
<p>Laat, W. de, and Grosveld, F. (2007). Inter-chromosomal gene regulation in the mammalian cell nucleus. Current Opinion in Genetics &amp; Development <em>17</em>, 456–464.</p>
</div>
<div id="ref-Monahan2015">
<p>Monahan, K., and Lomvardas, S. (2015). Monoallelic Expression of Olfactory Receptors. Annual Review of Cell and Developmental Biology <em>31</em>, annurev–cellbio–100814–125308.</p>
</div>
<div id="ref-Dixon2012">
<p>Dixon, J.R., Selvaraj, S., Yue, F., Kim, A., Li, Y., Shen, Y., Hu, M., Liu, J.S., and Ren, B. (2012). Topological domains in mammalian genomes identified by analysis of chromatin interactions. Nature <em>485</em>, 376–380.</p>
</div>
<div id="ref-Nora2012">
<p>Nora, E.P., Lajoie, B.R., Schulz, E.G., Giorgetti, L., Okamoto, I., Servant, N., Piolot, T., Berkum, N.L. van, Meisig, J., Sedat, J., et al. (2012). Spatial partitioning of the regulatory landscape of the X-inactivation centre. Nature <em>485</em>, 381–385.</p>
</div>
<div id="ref-Sexton2012">
<p>Sexton, T., Yaffe, E., Kenigsberg, E., Bantignies, F., Leblanc, B., Hoichman, M., Parrinello, H., Tanay, A., and Cavalli, G. (2012). Three-dimensional folding and functional organization principles of the Drosophila genome. Cell <em>148</em>, 458–472.</p>
</div>
<div id="ref-Merkenschlager2016">
<p>Merkenschlager, M., and Nora, E.P. (2016). CTCF and Cohesin in Genome Folding and Transcriptional Gene Regulation. Annual Review of Genomics and Human Genetics <em>17</em>, 17–43.</p>
</div>
<div id="ref-Ay2015">
<p>Ay, F., and Noble, W.S. (2015). Analysis methods for studying the 3D architecture of the genome. Genome Biology <em>16</em>, 183.</p>
</div>
<div id="ref-Crane2015">
<p>Crane, E., Bian, Q., McCord, R.P., Lajoie, B.R., Wheeler, B.S., Ralston, E.J., Uzawa, S., Dekker, J., and Meyer, B.J. (2015). Condensin-driven remodelling of X chromosome topology during dosage compensation. Nature.</p>
</div>
<div id="ref-Rao2014">
<p>Rao, S.S.P., Huntley, M.H., Durand, N.C., Stamenova, E.K., Bochkov, I.D., Robinson, J.T., Sanborn, A.L., Machol, I., Omer, A.D., Lander, E.S., et al. (2014). A 3D map of the human genome at kilobase resolution reveals principles of chromatin looping. Cell <em>159</em>, 1665–1680.</p>
</div>
<div id="ref-Filippova2014">
<p>Filippova, D., Patro, R., Duggal, G., and Kingsford, C. (2014). Identification of alternative topological domains in chromatin. Algorithms for Molecular Biology : AMB <em>9</em>, 14.</p>
</div>
<div id="ref-Fraser2015">
<p>Fraser, J., Ferrai, C., Chiariello, A.M., Schueler, M., Rito, T., Laudanno, G., Barbieri, M., Moore, B.L., Kraemer, D.C., Aitken, S., et al. (2015). Hierarchical folding and reorganization of chromosomes are linked to transcriptional changes in cellular differentiation. Mol Syst Biol <em>11</em>, 1–14.</p>
</div>
<div id="ref-Forcato2017">
<p>Forcato, M., Nicoletti, C., Pal, K., Livi, C.M., Ferrari, F., and Bicciato, S. (2017). Comparison of computational methods for Hi-C data analysis. Nature Methods 14–19.</p>
</div>
<div id="ref-Cremer2010">
<p>Cremer, T., and Cremer, M. (2010). Chromosome territories. Cold Spring Harbor Perspectives in Biology <em>2</em>, a003889.</p>
</div>
<div id="ref-Gibcus2013">
<p>Gibcus, J.H., and Dekker, J. (2013). The hierarchy of the 3D genome. Molecular Cell <em>49</em>, 773–782.</p>
</div>
<div id="ref-Eagen2015">
<p>Eagen, K.P., Hartl, T.A., and Kornberg, R.D. (2015). Stable Chromosome Condensation Revealed by Chromosome Conformation Capture. Cell <em>163</em>, 934–946.</p>
</div>
<div id="ref-Dileep2015">
<p>Dileep, V., Ay, F., Sima, J., Vera, D.L., Noble, W.S., and Gilbert, D.M. (2015). Topologically associating domains and their long-range contacts are established during early G1 coincident with the establishment of the replication-timing program. Genome Research <em>25</em>, 1104–1113.</p>
</div>
<div id="ref-Pope2014">
<p>Pope, B.D., Ryba, T., Dileep, V., Yue, F., Wu, W., Denas, O., Vera, D.L., Wang, Y., Hansen, R.S., Canfield, T.K., et al. (2014). Topologically associating domains are stable units of replication-timing regulation. Nature <em>515</em>, 402–405.</p>
</div>
<div id="ref-Shen2012">
<p>Shen, Y., Yue, F., McCleary, D.F., Ye, Z., Edsall, L., Kuan, S., Wagner, U., Dixon, J., Lee, L., Lobanenkov, V.V., et al. (2012). A map of the cis-regulatory sequences in the mouse genome. Nature <em>488</em>, 116–120.</p>
</div>
<div id="ref-Dixon2016">
<p>Dixon, J.R., Gorkin, D.U., and Ren, B. (2016). Chromatin Domains: The Unit of Chromosome Organization. Molecular Cell <em>62</em>, 668–680.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="1-4-methods-to-probe-the-3d-chromatin-archtiecture.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="1-6-dynamics-of-chromatin-structure.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"google": false,
"weibo": false,
"instapper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/ibn-salem/thesis/edit/master/01-intro.Rmd",
"text": "Edit"
},
"download": ["thesis.pdf", "thesis.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "";
    if (src === "" || src === "true") src = "https://cdn.bootcss.com/mathjax/2.7.1/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:" && /^https?:/.test(src))
      src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

<!-- </html> -->
