<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>Genome folding in evolution and disease</title>
  <meta name="description" content="A draft PhD thesis on genome folding in evoultion and disease.">
  <meta name="generator" content="bookdown 0.6.1 and GitBook 2.6.7">

  <meta property="og:title" content="Genome folding in evolution and disease" />
  <meta property="og:type" content="book" />
  <meta property="og:url" content="https://ibn-salem.github.io/thesis/" />
  
  <meta property="og:description" content="A draft PhD thesis on genome folding in evoultion and disease." />
  <meta name="github-repo" content="ibn-salem/thesis" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Genome folding in evolution and disease" />
  
  <meta name="twitter:description" content="A draft PhD thesis on genome folding in evoultion and disease." />
  

<meta name="author" content="Jonas Ibn-Salem">


<meta name="date" content="2018-03-02">

  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  
<link rel="prev" href="2-2-sec-paralog-methods.html">
<link rel="next" href="2-4-discussions.html">
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />










<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Thesis</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="1" data-path="1-intro.html"><a href="1-intro.html"><i class="fa fa-check"></i><b>1</b> Introduction</a><ul>
<li class="chapter" data-level="1.1" data-path="1-1-regulation-of-expression.html"><a href="1-1-regulation-of-expression.html"><i class="fa fa-check"></i><b>1.1</b> Regulation of gene expression</a></li>
<li class="chapter" data-level="1.2" data-path="1-2-distal-regulation-by-enhancers.html"><a href="1-2-distal-regulation-by-enhancers.html"><i class="fa fa-check"></i><b>1.2</b> Distal regulation by enhancers</a></li>
<li class="chapter" data-level="1.3" data-path="1-3-methods-to-probe-the-3d-chromatin-architecture.html"><a href="1-3-methods-to-probe-the-3d-chromatin-architecture.html"><i class="fa fa-check"></i><b>1.3</b> Methods to probe the 3D chromatin architecture</a><ul>
<li class="chapter" data-level="1.3.1" data-path="1-3-methods-to-probe-the-3d-chromatin-architecture.html"><a href="1-3-methods-to-probe-the-3d-chromatin-architecture.html#microscopy-based-techniques-to-visualize-the-genome-in-3d"><i class="fa fa-check"></i><b>1.3.1</b> Microscopy-based techniques to visualize the genome in 3D</a></li>
<li class="chapter" data-level="1.3.2" data-path="1-3-methods-to-probe-the-3d-chromatin-architecture.html"><a href="1-3-methods-to-probe-the-3d-chromatin-architecture.html#proximity-ligation"><i class="fa fa-check"></i><b>1.3.2</b> Proximity-ligation based method to quantify chromatin interactions</a></li>
</ul></li>
<li class="chapter" data-level="1.4" data-path="1-4-hierarchy-of-chromatin-3d-structure.html"><a href="1-4-hierarchy-of-chromatin-3d-structure.html"><i class="fa fa-check"></i><b>1.4</b> Hierarchy of chromatin 3D structure</a><ul>
<li class="chapter" data-level="1.4.1" data-path="1-4-hierarchy-of-chromatin-3d-structure.html"><a href="1-4-hierarchy-of-chromatin-3d-structure.html#chromosomal-territories-and-inter-chromosomal-contacts"><i class="fa fa-check"></i><b>1.4.1</b> Chromosomal territories and inter-chromosomal contacts</a></li>
<li class="chapter" data-level="1.4.2" data-path="1-4-hierarchy-of-chromatin-3d-structure.html"><a href="1-4-hierarchy-of-chromatin-3d-structure.html#ab-compartments"><i class="fa fa-check"></i><b>1.4.2</b> A/B compartments</a></li>
<li class="chapter" data-level="1.4.3" data-path="1-4-hierarchy-of-chromatin-3d-structure.html"><a href="1-4-hierarchy-of-chromatin-3d-structure.html#topologically-associating-domains-tads"><i class="fa fa-check"></i><b>1.4.3</b> Topologically associating domains (TADs)</a></li>
<li class="chapter" data-level="1.4.4" data-path="1-4-hierarchy-of-chromatin-3d-structure.html"><a href="1-4-hierarchy-of-chromatin-3d-structure.html#hierarchy-of-domain-structures-across-genomic-length-scales"><i class="fa fa-check"></i><b>1.4.4</b> Hierarchy of domain structures across genomic length scales</a></li>
<li class="chapter" data-level="1.4.5" data-path="1-4-hierarchy-of-chromatin-3d-structure.html"><a href="1-4-hierarchy-of-chromatin-3d-structure.html#chromatin-looping-interactions"><i class="fa fa-check"></i><b>1.4.5</b> Chromatin looping interactions</a></li>
<li class="chapter" data-level="1.4.6" data-path="1-4-hierarchy-of-chromatin-3d-structure.html"><a href="1-4-hierarchy-of-chromatin-3d-structure.html#loop-mechanism"><i class="fa fa-check"></i><b>1.4.6</b> TAD and loop formation by architectural proteins</a></li>
</ul></li>
<li class="chapter" data-level="1.5" data-path="1-5-dynamics-of-chromatin-structure.html"><a href="1-5-dynamics-of-chromatin-structure.html"><i class="fa fa-check"></i><b>1.5</b> Dynamics of chromatin structure</a><ul>
<li class="chapter" data-level="1.5.1" data-path="1-5-dynamics-of-chromatin-structure.html"><a href="1-5-dynamics-of-chromatin-structure.html#dynamics-across-the-cell-cycle"><i class="fa fa-check"></i><b>1.5.1</b> Dynamics across the cell cycle</a></li>
<li class="chapter" data-level="1.5.2" data-path="1-5-dynamics-of-chromatin-structure.html"><a href="1-5-dynamics-of-chromatin-structure.html#dynamics-across-cell-types-and-differntiation"><i class="fa fa-check"></i><b>1.5.2</b> Dynamics across cell types and differntiation</a></li>
</ul></li>
<li class="chapter" data-level="1.6" data-path="1-6-evolution-of-chromatin-organization.html"><a href="1-6-evolution-of-chromatin-organization.html"><i class="fa fa-check"></i><b>1.6</b> Evolution of chromatin organization</a></li>
<li class="chapter" data-level="1.7" data-path="1-7-disruption-of-chromatin-architecture-in-disease.html"><a href="1-7-disruption-of-chromatin-architecture-in-disease.html"><i class="fa fa-check"></i><b>1.7</b> Disruption of chromatin architecture in disease</a></li>
<li class="chapter" data-level="1.8" data-path="1-8-aims-of-this-thesis.html"><a href="1-8-aims-of-this-thesis.html"><i class="fa fa-check"></i><b>1.8</b> Aims of this thesis</a></li>
<li class="chapter" data-level="1.9" data-path="1-9-structure-of-this-thesis.html"><a href="1-9-structure-of-this-thesis.html"><i class="fa fa-check"></i><b>1.9</b> Structure of this thesis</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="2-paralog-regulation.html"><a href="2-paralog-regulation.html"><i class="fa fa-check"></i><b>2</b> Paralog genes in the 3D genome architecture</a><ul>
<li class="chapter" data-level="" data-path="2-paralog-regulation.html"><a href="2-paralog-regulation.html#preamble"><i class="fa fa-check"></i>Preamble</a></li>
<li class="chapter" data-level="" data-path="2-paralog-regulation.html"><a href="2-paralog-regulation.html#abstract-1"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="2.1" data-path="2-1-introduction.html"><a href="2-1-introduction.html"><i class="fa fa-check"></i><b>2.1</b> Introduction</a></li>
<li class="chapter" data-level="2.2" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html"><i class="fa fa-check"></i><b>2.2</b> Materials and methods</a><ul>
<li class="chapter" data-level="2.2.1" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html#selection-of-pairs-of-paralog-genes"><i class="fa fa-check"></i><b>2.2.1</b> Selection of pairs of paralog genes</a></li>
<li class="chapter" data-level="2.2.2" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html#enhancers-to-gene-association"><i class="fa fa-check"></i><b>2.2.2</b> Enhancers to gene association</a></li>
<li class="chapter" data-level="2.2.3" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html#topological-associating-domains"><i class="fa fa-check"></i><b>2.2.3</b> Topological associating domains</a></li>
<li class="chapter" data-level="2.2.4" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html#hi-c-interaction-maps"><i class="fa fa-check"></i><b>2.2.4</b> Hi-C interaction maps</a></li>
<li class="chapter" data-level="2.2.5" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html#randomization"><i class="fa fa-check"></i><b>2.2.5</b> Randomization</a></li>
<li class="chapter" data-level="2.2.6" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html#statistical-tests"><i class="fa fa-check"></i><b>2.2.6</b> Statistical tests</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="2-3-results.html"><a href="2-3-results.html"><i class="fa fa-check"></i><b>2.3</b> Results</a><ul>
<li class="chapter" data-level="2.3.1" data-path="2-3-results.html"><a href="2-3-results.html#distribution-of-paralog-genes-in-the-human-genome"><i class="fa fa-check"></i><b>2.3.1</b> Distribution of paralog genes in the human genome</a></li>
<li class="chapter" data-level="2.3.2" data-path="2-3-results.html"><a href="2-3-results.html#co-expression-of-paralog-gene-pairs-across-tissues"><i class="fa fa-check"></i><b>2.3.2</b> Co-expression of paralog gene pairs across tissues</a></li>
<li class="chapter" data-level="2.3.3" data-path="2-3-results.html"><a href="2-3-results.html#paralog-genes-share-enhancers"><i class="fa fa-check"></i><b>2.3.3</b> Paralog genes share enhancers</a></li>
<li class="chapter" data-level="2.3.4" data-path="2-3-results.html"><a href="2-3-results.html#co-localization-of-paralogs-in-tads"><i class="fa fa-check"></i><b>2.3.4</b> Co-localization of paralogs in TADs</a></li>
<li class="chapter" data-level="2.3.5" data-path="2-3-results.html"><a href="2-3-results.html#distal-paralog-pairs-are-enriched-for-long-range-chromatin-contacts"><i class="fa fa-check"></i><b>2.3.5</b> Distal paralog pairs are enriched for long-range chromatin contacts</a></li>
<li class="chapter" data-level="2.3.6" data-path="2-3-results.html"><a href="2-3-results.html#close-paralogs-have-fewer-contacts-than-expected"><i class="fa fa-check"></i><b>2.3.6</b> Close paralogs have fewer contacts than expected</a></li>
<li class="chapter" data-level="2.3.7" data-path="2-3-results.html"><a href="2-3-results.html#paralogs-in-mouse-and-dog-genome"><i class="fa fa-check"></i><b>2.3.7</b> Paralogs in mouse and dog genome</a></li>
<li class="chapter" data-level="2.3.8" data-path="2-3-results.html"><a href="2-3-results.html#orthologs-of-human-paralogs-show-conserved-co-localization"><i class="fa fa-check"></i><b>2.3.8</b> Orthologs of human paralogs show conserved co-localization</a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="2-4-discussions.html"><a href="2-4-discussions.html"><i class="fa fa-check"></i><b>2.4</b> Discussions</a></li>
<li class="chapter" data-level="2.5" data-path="2-5-conclusion.html"><a href="2-5-conclusion.html"><i class="fa fa-check"></i><b>2.5</b> Conclusion</a></li>
<li class="chapter" data-level="" data-path="acknowledgements.html"><a href="acknowledgements.html"><i class="fa fa-check"></i>Acknowledgements</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="3-TAD-evolution.html"><a href="3-TAD-evolution.html"><i class="fa fa-check"></i><b>3</b> Stability of TADs in evolution</a><ul>
<li class="chapter" data-level="" data-path="3-TAD-evolution.html"><a href="3-TAD-evolution.html#preamble-1"><i class="fa fa-check"></i>Preamble</a></li>
<li class="chapter" data-level="" data-path="3-TAD-evolution.html"><a href="3-TAD-evolution.html#abstract-2"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="3.1" data-path="3-1-introduction-1.html"><a href="3-1-introduction-1.html"><i class="fa fa-check"></i><b>3.1</b> Introduction</a></li>
<li class="chapter" data-level="3.2" data-path="3-2-results-1.html"><a href="3-2-results-1.html"><i class="fa fa-check"></i><b>3.2</b> Results</a><ul>
<li class="chapter" data-level="3.2.1" data-path="3-2-results-1.html"><a href="3-2-results-1.html#identification-of-evolutionary-rearrangement-breakpoints-from-whole-genome-alignments"><i class="fa fa-check"></i><b>3.2.1</b> Identification of evolutionary rearrangement breakpoints from whole-genome alignments</a></li>
<li class="chapter" data-level="3.2.2" data-path="3-2-results-1.html"><a href="3-2-results-1.html#rearrangement-breakpoints-are-enriched-at-tad-boundaries"><i class="fa fa-check"></i><b>3.2.2</b> Rearrangement breakpoints are enriched at TAD boundaries</a></li>
<li class="chapter" data-level="3.2.3" data-path="3-2-results-1.html"><a href="3-2-results-1.html#clusters-of-conserved-non-coding-elements-are-depleted-for-rearrangement-breakpoints"><i class="fa fa-check"></i><b>3.2.3</b> Clusters of conserved non-coding elements are depleted for rearrangement breakpoints</a></li>
<li class="chapter" data-level="3.2.4" data-path="3-2-results-1.html"><a href="3-2-results-1.html#rearranged-tads-are-associated-with-divergent-gene-expression-between-species"><i class="fa fa-check"></i><b>3.2.4</b> Rearranged TADs are associated with divergent gene expression between species</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="3-3-discussion.html"><a href="3-3-discussion.html"><i class="fa fa-check"></i><b>3.3</b> Discussion</a></li>
<li class="chapter" data-level="3.4" data-path="3-4-conclusion-1.html"><a href="3-4-conclusion-1.html"><i class="fa fa-check"></i><b>3.4</b> Conclusion</a></li>
<li class="chapter" data-level="3.5" data-path="3-5-methods.html"><a href="3-5-methods.html"><i class="fa fa-check"></i><b>3.5</b> Methods</a><ul>
<li class="chapter" data-level="3.5.1" data-path="3-5-methods.html"><a href="3-5-methods.html#rearrangement-breakpoints-from-whole-genome-alignments"><i class="fa fa-check"></i><b>3.5.1</b> Rearrangement breakpoints from whole-genome alignments</a></li>
<li class="chapter" data-level="3.5.2" data-path="3-5-methods.html"><a href="3-5-methods.html#topologically-associating-domains-and-contact-domains"><i class="fa fa-check"></i><b>3.5.2</b> Topologically associating domains and contact domains</a></li>
<li class="chapter" data-level="3.5.3" data-path="3-5-methods.html"><a href="3-5-methods.html#breakpoint-distributions-at-tads"><i class="fa fa-check"></i><b>3.5.3</b> Breakpoint distributions at TADs</a></li>
<li class="chapter" data-level="3.5.4" data-path="3-5-methods.html"><a href="3-5-methods.html#quantification-of-breakpoint-enrichment"><i class="fa fa-check"></i><b>3.5.4</b> Quantification of breakpoint enrichment</a></li>
<li class="chapter" data-level="3.5.5" data-path="3-5-methods.html"><a href="3-5-methods.html#expression-data-for-mouse-and-human-orthologs"><i class="fa fa-check"></i><b>3.5.5</b> Expression data for mouse and human orthologs</a></li>
<li class="chapter" data-level="3.5.6" data-path="3-5-methods.html"><a href="3-5-methods.html#classification-of-tads-and-genes-according-to-rearrangements-and-grbs"><i class="fa fa-check"></i><b>3.5.6</b> Classification of TADs and genes according to rearrangements and GRBs</a></li>
<li class="chapter" data-level="3.5.7" data-path="3-5-methods.html"><a href="3-5-methods.html#source-code-and-implementation-details"><i class="fa fa-check"></i><b>3.5.7</b> Source code and implementation details</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="declarations.html"><a href="declarations.html"><i class="fa fa-check"></i>Declarations</a><ul>
<li class="chapter" data-level="" data-path="declarations.html"><a href="declarations.html#availability-of-data-and-material"><i class="fa fa-check"></i>Availability of data and material</a></li>
<li class="chapter" data-level="" data-path="declarations.html"><a href="declarations.html#competing-interests"><i class="fa fa-check"></i>Competing interests</a></li>
<li class="chapter" data-level="" data-path="declarations.html"><a href="declarations.html#authors-contributions"><i class="fa fa-check"></i>Authors‘ contributions</a></li>
<li class="chapter" data-level="" data-path="declarations.html"><a href="declarations.html#acknowledgments"><i class="fa fa-check"></i>Acknowledgments</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="4-position-effect.html"><a href="4-position-effect.html"><i class="fa fa-check"></i><b>4</b> Position effects of rearrangements in disease genomes</a><ul>
<li class="chapter" data-level="" data-path="4-position-effect.html"><a href="4-position-effect.html#preamble-2"><i class="fa fa-check"></i>Preamble</a></li>
<li class="chapter" data-level="" data-path="4-position-effect.html"><a href="4-position-effect.html#abstract-3"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="4.1" data-path="4-1-introduction-2.html"><a href="4-1-introduction-2.html"><i class="fa fa-check"></i><b>4.1</b> Introduction</a></li>
<li class="chapter" data-level="4.2" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html"><i class="fa fa-check"></i><b>4.2</b> Materials and Methods</a><ul>
<li class="chapter" data-level="4.2.1" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html#selection-of-subjects-with-apparently-balanced-chromosome-abnormalities"><i class="fa fa-check"></i><b>4.2.1</b> Selection of subjects with apparently balanced chromosome abnormalities</a></li>
<li class="chapter" data-level="4.2.2" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html#clinical-descriptions-of-dgap-cases"><i class="fa fa-check"></i><b>4.2.2</b> Clinical descriptions of DGAP cases</a></li>
<li class="chapter" data-level="4.2.3" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html#analysis-of-genes-bordering-the-rearrangement-breakpoints"><i class="fa fa-check"></i><b>4.2.3</b> Analysis of genes bordering the rearrangement breakpoints</a></li>
<li class="chapter" data-level="4.2.4" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html#assessment-of-disrupted-functional-elements-and-chromatin-interactions-bordering-rearrangement-breakpoints"><i class="fa fa-check"></i><b>4.2.4</b> Assessment of disrupted functional elements and chromatin interactions bordering rearrangement breakpoints</a></li>
<li class="chapter" data-level="4.2.5" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html#ontological-analysis-of-genes-neighboring-breakpoints"><i class="fa fa-check"></i><b>4.2.5</b> Ontological analysis of genes neighboring breakpoints</a></li>
<li class="chapter" data-level="4.2.6" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html#quantitative-real-time-pcr"><i class="fa fa-check"></i><b>4.2.6</b> Quantitative real-time PCR</a></li>
<li class="chapter" data-level="4.2.7" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html#assessment-of-dgap-breakpoints-overlapping-with-non-coding-structural-variants-in-public-databases"><i class="fa fa-check"></i><b>4.2.7</b> Assessment of DGAP breakpoints overlapping with non-coding structural variants in public databases</a></li>
</ul></li>
<li class="chapter" data-level="4.3" data-path="4-3-results-2.html"><a href="4-3-results-2.html"><i class="fa fa-check"></i><b>4.3</b> Results</a><ul>
<li class="chapter" data-level="4.3.1" data-path="4-3-results-2.html"><a href="4-3-results-2.html#genomic-characterization-of-non-coding-breakpoints"><i class="fa fa-check"></i><b>4.3.1</b> Genomic characterization of non-coding breakpoints</a></li>
<li class="chapter" data-level="4.3.2" data-path="4-3-results-2.html"><a href="4-3-results-2.html#identification-of-genes-with-potential-position-effects"><i class="fa fa-check"></i><b>4.3.2</b> Identification of genes with potential position effects</a></li>
<li class="chapter" data-level="4.3.3" data-path="4-3-results-2.html"><a href="4-3-results-2.html#identification-of-subjects-with-shared-non-coding-chromosome-alterations-and-phenotypes"><i class="fa fa-check"></i><b>4.3.3</b> Identification of subjects with shared non-coding chromosome alterations and phenotypes</a></li>
</ul></li>
<li class="chapter" data-level="4.4" data-path="4-4-discussion-1.html"><a href="4-4-discussion-1.html"><i class="fa fa-check"></i><b>4.4</b> Discussion</a></li>
<li class="chapter" data-level="" data-path="declarations-1.html"><a href="declarations-1.html"><i class="fa fa-check"></i>Declarations</a><ul>
<li class="chapter" data-level="" data-path="declarations-1.html"><a href="declarations-1.html#acknowledgements-1"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="" data-path="declarations-1.html"><a href="declarations-1.html#web-resources"><i class="fa fa-check"></i>Web Resources</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="5-loop.html"><a href="5-loop.html"><i class="fa fa-check"></i><b>5</b> Prediction of chromatin looping interactions</a><ul>
<li class="chapter" data-level="" data-path="5-loop.html"><a href="5-loop.html#preamble-3"><i class="fa fa-check"></i>Preamble</a></li>
<li class="chapter" data-level="" data-path="5-loop.html"><a href="5-loop.html#abstract-4"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="5.1" data-path="5-1-introduction-3.html"><a href="5-1-introduction-3.html"><i class="fa fa-check"></i><b>5.1</b> Introduction</a></li>
<li class="chapter" data-level="5.2" data-path="5-2-results-3.html"><a href="5-2-results-3.html"><i class="fa fa-check"></i><b>5.2</b> Results</a><ul>
<li class="chapter" data-level="5.2.1" data-path="5-2-results-3.html"><a href="5-2-results-3.html#ctcf-motif-pairs-as-candidate-chromatin-loop-anchors"><i class="fa fa-check"></i><b>5.2.1</b> CTCF motif pairs as candidate chromatin loop anchors</a></li>
<li class="chapter" data-level="5.2.2" data-path="5-2-results-3.html"><a href="5-2-results-3.html#similarity-of-chip-seq-signals-at-looping-ctcf-motifs"><i class="fa fa-check"></i><b>5.2.2</b> Similarity of ChIP-seq signals at looping CTCF motifs</a></li>
<li class="chapter" data-level="5.2.3" data-path="5-2-results-3.html"><a href="5-2-results-3.html#genomic-sequence-features-of-ctcf-motif-pairs-are-associated-with-looping"><i class="fa fa-check"></i><b>5.2.3</b> Genomic sequence features of CTCF motif pairs are associated with looping</a></li>
<li class="chapter" data-level="5.2.4" data-path="5-2-results-3.html"><a href="5-2-results-3.html#chromatin-loop-prediction-using-7c"><i class="fa fa-check"></i><b>5.2.4</b> Chromatin loop prediction using 7C</a></li>
<li class="chapter" data-level="5.2.5" data-path="5-2-results-3.html"><a href="5-2-results-3.html#prediction-performance-evaluation"><i class="fa fa-check"></i><b>5.2.5</b> Prediction performance evaluation</a></li>
<li class="chapter" data-level="5.2.6" data-path="5-2-results-3.html"><a href="5-2-results-3.html#prediction-performance-of-7c-with-sequence-features-and-single-tf-chip-seq-data-sets"><i class="fa fa-check"></i><b>5.2.6</b> Prediction performance of 7C with sequence features and single TF ChIP-seq data sets</a></li>
<li class="chapter" data-level="5.2.7" data-path="5-2-results-3.html"><a href="5-2-results-3.html#comparison-of-transcription-factors-by-prediction-performance"><i class="fa fa-check"></i><b>5.2.7</b> Comparison of transcription factors by prediction performance</a></li>
<li class="chapter" data-level="5.2.8" data-path="5-2-results-3.html"><a href="5-2-results-3.html#prediction-performance-in-other-cell-types-and-for-different-tfs"><i class="fa fa-check"></i><b>5.2.8</b> Prediction performance in other cell types and for different TFs</a></li>
<li class="chapter" data-level="5.2.9" data-path="5-2-results-3.html"><a href="5-2-results-3.html#the-high-resolution-of-chip-nexus-improves-prediction-performance"><i class="fa fa-check"></i><b>5.2.9</b> The high resolution of ChIP-nexus improves prediction performance</a></li>
</ul></li>
<li class="chapter" data-level="5.3" data-path="5-3-discussion-2.html"><a href="5-3-discussion-2.html"><i class="fa fa-check"></i><b>5.3</b> Discussion</a></li>
<li class="chapter" data-level="5.4" data-path="5-4-conclusion-2.html"><a href="5-4-conclusion-2.html"><i class="fa fa-check"></i><b>5.4</b> Conclusion</a></li>
<li class="chapter" data-level="5.5" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html"><i class="fa fa-check"></i><b>5.5</b> Methods</a><ul>
<li class="chapter" data-level="5.5.1" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#ctcf-motifs-in-the-human-genome"><i class="fa fa-check"></i><b>5.5.1</b> CTCF motifs in the human genome</a></li>
<li class="chapter" data-level="5.5.2" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#loop-interaction-data-for-training-and-validation"><i class="fa fa-check"></i><b>5.5.2</b> Loop interaction data for training and validation</a></li>
<li class="chapter" data-level="5.5.3" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#chip-seq-datasets-in-gm12878-cells"><i class="fa fa-check"></i><b>5.5.3</b> ChIP-seq datasets in GM12878 cells</a></li>
<li class="chapter" data-level="5.5.4" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#chip-seq-data-types"><i class="fa fa-check"></i><b>5.5.4</b> ChIP-seq data types</a></li>
<li class="chapter" data-level="5.5.5" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#chip-nexus-data-processing-for-rad21-and-scm3"><i class="fa fa-check"></i><b>5.5.5</b> ChIP-nexus data processing for RAD21 and SCM3</a></li>
<li class="chapter" data-level="5.5.6" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#similarity-of-chip-seq-profiles-as-correlation-of-coverage-around-motifs"><i class="fa fa-check"></i><b>5.5.6</b> Similarity of ChIP-seq profiles as correlation of coverage around motifs</a></li>
<li class="chapter" data-level="5.5.7" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#genomic-sequence-features-of-chromatin-loops"><i class="fa fa-check"></i><b>5.5.7</b> Genomic sequence features of chromatin loops</a></li>
<li class="chapter" data-level="5.5.8" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#chromatin-loop-prediction-model"><i class="fa fa-check"></i><b>5.5.8</b> Chromatin Loop prediction model</a></li>
<li class="chapter" data-level="5.5.9" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#training-and-validation-of-prediction-model"><i class="fa fa-check"></i><b>5.5.9</b> Training and validation of prediction model</a></li>
<li class="chapter" data-level="5.5.10" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#analysis-of-prediction-performance"><i class="fa fa-check"></i><b>5.5.10</b> Analysis of prediction performance</a></li>
<li class="chapter" data-level="5.5.11" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#implementation-of-7c-and-compatibility-to-other-tools"><i class="fa fa-check"></i><b>5.5.11</b> Implementation of 7C and compatibility to other tools</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="declarations-2.html"><a href="declarations-2.html"><i class="fa fa-check"></i>Declarations</a><ul>
<li class="chapter" data-level="" data-path="declarations-2.html"><a href="declarations-2.html#availability-of-data-and-material-1"><i class="fa fa-check"></i>Availability of data and material</a></li>
<li class="chapter" data-level="" data-path="declarations-2.html"><a href="declarations-2.html#competing-interests-1"><i class="fa fa-check"></i>Competing interests</a></li>
<li class="chapter" data-level="" data-path="declarations-2.html"><a href="declarations-2.html#authors-contributions-1"><i class="fa fa-check"></i>Authors‘ contributions</a></li>
<li class="chapter" data-level="" data-path="declarations-2.html"><a href="declarations-2.html#acknowledgments-1"><i class="fa fa-check"></i>Acknowledgments</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="6-thesis-discussion.html"><a href="6-thesis-discussion.html"><i class="fa fa-check"></i><b>6</b> Discussion</a><ul>
<li class="chapter" data-level="6.1" data-path="6-1-co-regulation-of-functionally-related-genes-in-tads.html"><a href="6-1-co-regulation-of-functionally-related-genes-in-tads.html"><i class="fa fa-check"></i><b>6.1</b> Co-regulation of functionally related genes in TADs</a></li>
<li class="chapter" data-level="6.2" data-path="6-2-evolution-by-gene-duplications-and-altered-regulatory-environments.html"><a href="6-2-evolution-by-gene-duplications-and-altered-regulatory-environments.html"><i class="fa fa-check"></i><b>6.2</b> Evolution by gene duplications and altered regulatory environments</a></li>
<li class="chapter" data-level="6.3" data-path="6-3-tads-are-stable-across-large-evolutionary-time-scales.html"><a href="6-3-tads-are-stable-across-large-evolutionary-time-scales.html"><i class="fa fa-check"></i><b>6.3</b> TADs are stable across large evolutionary time-scales</a></li>
<li class="chapter" data-level="6.4" data-path="6-4-TAD-disease.html"><a href="6-4-TAD-disease.html"><i class="fa fa-check"></i><b>6.4</b> Gene expression changes by altered TADs in disease</a></li>
<li class="chapter" data-level="6.5" data-path="6-5-towards-predicting-regulatory-pathomechanisms-of-structural-variants.html"><a href="6-5-towards-predicting-regulatory-pathomechanisms-of-structural-variants.html"><i class="fa fa-check"></i><b>6.5</b> Towards predicting regulatory pathomechanisms of structural variants</a></li>
<li class="chapter" data-level="6.6" data-path="6-6-towards-studying-genome-folding-in-specific-conditions.html"><a href="6-6-towards-studying-genome-folding-in-specific-conditions.html"><i class="fa fa-check"></i><b>6.6</b> Towards studying genome folding in specific conditions</a><ul>
<li class="chapter" data-level="6.6.1" data-path="6-6-towards-studying-genome-folding-in-specific-conditions.html"><a href="6-6-towards-studying-genome-folding-in-specific-conditions.html#interpretation-of-non-coding-variants-by-interacting-genes"><i class="fa fa-check"></i><b>6.6.1</b> Interpretation of non-coding variants by interacting genes</a></li>
<li class="chapter" data-level="6.6.2" data-path="6-6-towards-studying-genome-folding-in-specific-conditions.html"><a href="6-6-towards-studying-genome-folding-in-specific-conditions.html#cell-type-specific-regulatory-interactions-between-enhancers-and-genes"><i class="fa fa-check"></i><b>6.6.2</b> Cell type-specific regulatory interactions between enhancers and genes</a></li>
<li class="chapter" data-level="6.6.3" data-path="6-6-towards-studying-genome-folding-in-specific-conditions.html"><a href="6-6-towards-studying-genome-folding-in-specific-conditions.html#variability-of-long-range-interactions-across-individuals"><i class="fa fa-check"></i><b>6.6.3</b> Variability of long-range interactions across individuals</a></li>
<li class="chapter" data-level="6.6.4" data-path="6-6-towards-studying-genome-folding-in-specific-conditions.html"><a href="6-6-towards-studying-genome-folding-in-specific-conditions.html#single-cell"><i class="fa fa-check"></i><b>6.6.4</b> Genome folding in single cells</a></li>
<li class="chapter" data-level="6.6.5" data-path="6-6-towards-studying-genome-folding-in-specific-conditions.html"><a href="6-6-towards-studying-genome-folding-in-specific-conditions.html#constantly-improving-targeted-experimental-methods."><i class="fa fa-check"></i><b>6.6.5</b> Constantly improving targeted experimental methods.</a></li>
</ul></li>
<li class="chapter" data-level="6.7" data-path="6-7-molecular-mechanisms-driving-genome-folding.html"><a href="6-7-molecular-mechanisms-driving-genome-folding.html"><i class="fa fa-check"></i><b>6.7</b> Molecular mechanisms driving genome folding</a></li>
<li class="chapter" data-level="6.8" data-path="6-8-conclusions.html"><a href="6-8-conclusions.html"><i class="fa fa-check"></i><b>6.8</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="appendix"><span><b>Appendix</b></span></li>
<li class="chapter" data-level="A" data-path="A-sup-paralog-regulation.html"><a href="A-sup-paralog-regulation.html"><i class="fa fa-check"></i><b>A</b> Supporting Information: Co-regulation of paralog genes</a></li>
<li class="chapter" data-level="B" data-path="B-sup-tad-evolution.html"><a href="B-sup-tad-evolution.html"><i class="fa fa-check"></i><b>B</b> Supplementary Data: Stability of TADs in evolution</a><ul>
<li class="chapter" data-level="B.1" data-path="B-1-TadEvoSupTab.html"><a href="B-1-TadEvoSupTab.html"><i class="fa fa-check"></i><b>B.1</b> Supplementary Tables</a></li>
<li class="chapter" data-level="B.2" data-path="B-2-supplementary-figures.html"><a href="B-2-supplementary-figures.html"><i class="fa fa-check"></i><b>B.2</b> Supplementary Figures</a></li>
</ul></li>
<li class="chapter" data-level="C" data-path="C-sup-position-effect.html"><a href="C-sup-position-effect.html"><i class="fa fa-check"></i><b>C</b> Supplemental Data: Position effects of rearrangements in disease genomes</a><ul>
<li class="chapter" data-level="C.1" data-path="C-1-supplemental-note.html"><a href="C-1-supplemental-note.html"><i class="fa fa-check"></i><b>C.1</b> Supplemental Note</a><ul>
<li class="chapter" data-level="C.1.1" data-path="C-1-supplemental-note.html"><a href="C-1-supplemental-note.html#case-reports"><i class="fa fa-check"></i><b>C.1.1</b> Case Reports</a></li>
<li class="chapter" data-level="C.1.2" data-path="C-1-supplemental-note.html"><a href="C-1-supplemental-note.html#nucleotide-level-nomenclature-for-dgap-karyotypes"><i class="fa fa-check"></i><b>C.1.2</b> Nucleotide Level Nomenclature for DGAP karyotypes</a></li>
</ul></li>
<li class="chapter" data-level="C.2" data-path="C-2-supplemental-figure.html"><a href="C-2-supplemental-figure.html"><i class="fa fa-check"></i><b>C.2</b> Supplemental Figure</a></li>
<li class="chapter" data-level="C.3" data-path="C-3-position-effect-sub-tab.html"><a href="C-3-position-effect-sub-tab.html"><i class="fa fa-check"></i><b>C.3</b> Supplemental Table Legends</a></li>
</ul></li>
<li class="chapter" data-level="D" data-path="D-loop-support.html"><a href="D-loop-support.html"><i class="fa fa-check"></i><b>D</b> Supplemental Information: Prediction of chromatin looping interactions</a><ul>
<li class="chapter" data-level="D.1" data-path="D-1-LoopPredSupTab.html"><a href="D-1-LoopPredSupTab.html"><i class="fa fa-check"></i><b>D.1</b> Supplementary Tables</a></li>
<li class="chapter" data-level="D.2" data-path="D-2-supplementary-figures-1.html"><a href="D-2-supplementary-figures-1.html"><i class="fa fa-check"></i><b>D.2</b> Supplementary Figures</a></li>
</ul></li>
<li class="chapter" data-level="E" data-path="E-contribution-to-individual-publications.html"><a href="E-contribution-to-individual-publications.html"><i class="fa fa-check"></i><b>E</b> Contribution to individual publications</a></li>
<li class="chapter" data-level="" data-path="zusammenfassung.html"><a href="zusammenfassung.html"><i class="fa fa-check"></i>Zusammenfassung</a></li>
<li class="chapter" data-level="" data-path="curriculum-vitae.html"><a href="curriculum-vitae.html"><i class="fa fa-check"></i>Curriculum vitae</a></li>
<li class="chapter" data-level="" data-path="acknowledgements-2.html"><a href="acknowledgements-2.html"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Build with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Genome folding in evolution and disease</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="results" class="section level2">
<h2><span class="header-section-number">2.3</span> Results</h2>
<div id="distribution-of-paralog-genes-in-the-human-genome" class="section level3">
<h3><span class="header-section-number">2.3.1</span> Distribution of paralog genes in the human genome</h3>
<p>Paralogs are homologous genes that arise from gene duplication events.
Their common ancestry and replicated sequence often leads to similar
structure and function in related pathways and protein complexes. We
therefore hypothesised that the transcription of paralogs should have a
tendency for co-regulation, which could correspond to their position in
the genome and within TADs. To test this hypothesis, we first focused on
the positions of paralogs in the linear genome.</p>
<p>From all <span class="math inline">\(19,430\)</span> protein coding genes in the human genome,
<span class="math inline">\(13,690~(70.5\%)\)</span> have at least one paralog (Fig. <a href="A-sup-paralog-regulation.html#fig:paraVSnonPara">A.1</a>A). However, many
human genes have several paralogs (Fig. <a href="A-sup-paralog-regulation.html#fig:paraVSnonPara">A.1</a>B). From all <span class="math inline">\(46,546\)</span> paralog
gene pairs we filtered for only one pair per gene <span class="math inline">\((n=6,256)\)</span> and
further for non-overlapping pairs on the same chromosome <span class="math inline">\((n=1,560)\)</span>
(see ). We will refer to close paralogs if their transcription start
sites (TSSs) are within 1 Mb of each other <span class="math inline">\((n=1,114)\)</span> and to distal
pairs for paralogs with TSSs separated by more than 1 Mb <span class="math inline">\((n=446)\)</span>
(Table <a href="2-2-sec-paralog-methods.html#tab:filter">2.1</a>).</p>
<p>We first compared basic properties between genes that have at least one
paralog copy and genes without human paralogs. Paralogs have
significantly larger gene length than non-paralog genes
(<span class="math inline">\(p = 1.7 \times 10^{-53}\)</span>, Wilcoxon rank-sum test, Fig. <a href="A-sup-paralog-regulation.html#fig:paraVSnonPara">A.1</a>C), which
fits the observation from <span class="citation">(He and Zhang <a href="references.html#ref-He2005">2005</a>)</span> in yeast. Furthermore, paralogs tend
to be associated to more enhancers compared to non-paralog genes (on
average <span class="math inline">\(3.8\)</span> vs. <span class="math inline">\(2.5\)</span> enhancers per gene, <span class="math inline">\(p=2.89\times10^{-70}\)</span>,
Fig. <a href="A-sup-paralog-regulation.html#fig:paraVSnonPara">A.1</a>D) and the distance to the nearest associated enhancer is
significantly shorter (<span class="math inline">\(p=2.71\times10^{-22}\)</span>,Fig. <a href="A-sup-paralog-regulation.html#fig:paraVSnonPara">A.1</a>E).</p>
<p>Since most genome duplication events in humans emerge through tandem
duplications <span class="citation">(Newman et al. <a href="references.html#ref-Newman2015">2015</a>)</span>, we expected some co-localization among pairs
of paralog genes. Indeed <span class="math inline">\(24.9\%\)</span> of paralog pairs are located on the
same chromosome. We compared this to random expectation by sampling
random gene pairs from all protein coding human genes and found only
<span class="math inline">\(5.3\%\)</span> of randomly sampled gene pairs on the same chromosome
(<span class="math inline">\(p&lt;10^{-16}\)</span>, Fig. <a href="2-3-results.html#fig:paraData">2.1</a>A).</p>
<!-- ---------------------------------------------------------------- -->
<div class="figure" style="text-align: center"><span id="fig:paraData"></span>
<img src="figures/paralog/fig1_09.png" alt="(A) Percent of paralog (red) and random (dark grey) gene pairs that are located on the same chromosome. The error bar indicates the standard deviation observed in 10 times replicated random sampling of gene pairs. (B) Genomic distance distribution of paralogs gene pairs (top), random gene pairs (center) and gene pairs sampled according to distance distribution of paralogs (bottom). Distances are measured in kilo base pairs (kb) between TSS of genes in pairs. P-values are calculated using Wilcoxon rank-sum test. (C) Percent of paralog (red) and sampled (grey) gene pairs that are transcribed from the same strand. Only pairs on the the same chromosome within 1 Mb are considered here. Error bars indicate the standard deviation observed in 10 times replicated sampling of gene pairs. (D) Boxplot of the genomic distance between paralogs and sampled gene pairs with the same or opposite strands. (E) Distribution of Pearson correlation coefficients of gene expression values in four independent data sets between paralog gene pairs (red) and sampled control gene pairs (grey). White boxes show 25th, 50th and 75th percent quantile of the data and the filled areas indicate the density distribution.." width="50%" />
<p class="caption">
Figure 2.1: <strong>(A)</strong> Percent of paralog (red) and random (dark grey) gene pairs that are located on the same chromosome. The error bar indicates the standard deviation observed in 10 times replicated random sampling of gene pairs. <strong>(B)</strong> Genomic distance distribution of paralogs gene pairs (top), random gene pairs (center) and gene pairs sampled according to distance distribution of paralogs (bottom). Distances are measured in kilo base pairs (kb) between TSS of genes in pairs. P-values are calculated using Wilcoxon rank-sum test. <strong>(C)</strong> Percent of paralog (red) and sampled (grey) gene pairs that are transcribed from the same strand. Only pairs on the the same chromosome within 1 Mb are considered here. Error bars indicate the standard deviation observed in 10 times replicated sampling of gene pairs. <strong>(D)</strong> Boxplot of the genomic distance between paralogs and sampled gene pairs with the same or opposite strands. <strong>(E)</strong> Distribution of Pearson correlation coefficients of gene expression values in four independent data sets between paralog gene pairs (red) and sampled control gene pairs (grey). White boxes show 25th, 50th and 75th percent quantile of the data and the filled areas indicate the density distribution..
</p>
</div>

<!-- ---------------------------------------------------------------- -->
<p>We further analysed whether paralog pairs tend to be located in close
genomic distance on the same chromosomes. We compared the distance
between paralog gene pairs to the distance of completely random genes on
the same chromosome. As expected there is a strong bias of genomic
co-localization among paralog gene pairs that is not observed for random
gene pairs (<span class="math inline">\(p=4.3\times10^{-32}\)</span>, Fig <a href="2-3-results.html#fig:paraData">2.1</a>B).</p>
<p>We also observed that close paralog genes show more often than expected
the same transcription orientation. From all paralog pairs within 1 Mb
on the same chromosome <span class="math inline">\(66.1\%\)</span> have the same sense. This is
significantly more than for randomly sampled genes with the same
distance (52.6%, <span class="math inline">\(p=3.2\times10^{-18}\)</span>, Fig. <a href="2-3-results.html#fig:paraData">2.1</a>C).</p>
<p>Furthermore, we observed that paralogs in the same strand are closer to
each other on the chromosome than pairs in opposite strands
(<span class="math inline">\(p=3.48\times10^{-8}\)</span>, Fig. <a href="2-3-results.html#fig:paraData">2.1</a>D).</p>
<p>Together, this shows that paralogs tend to be located within short
linear distance on the same chromosome and same transcription sense,
which might enable coordinated regulation by shared regulatory
mechanisms.</p>
</div>
<div id="co-expression-of-paralog-gene-pairs-across-tissues" class="section level3">
<h3><span class="header-section-number">2.3.2</span> Co-expression of paralog gene pairs across tissues</h3>
<p>To assess whether paralog genes tend to be indeed co-regulated we
compared gene expression of paralog gene pairs over several human
tissues and cell lines.</p>
<p>We compared the Pearson correlation coefficient (PCC) of gene expression
values over <span class="math inline">\(n = 18\)</span> cell-lines analysed by the ENCODE consortium by
RNA-seq <span class="citation">(Djebali et al. <a href="references.html#ref-Djebali2012">2012</a>)</span>. The distribution of PCC among paralog genes is
bimodal with one peak around <span class="math inline">\(-0.1\)</span> and another at nearly <span class="math inline">\(1.0\)</span>, which
indicates that there exists a group of paralog pairs without expression
correlation and that the expression of other paralogs is highly
positively correlated. Notably, we did not find the latter signal for
positive correlation in our control set of carefully sampled gene pairs
(Fig. <a href="2-3-results.html#fig:paraData">2.1</a>E).</p>
<p>We repeated the analysis with three other independent gene expression
data sets form FANTOM5 (<span class="math inline">\(n=56\)</span> tissues) <span class="citation">(Forrest et al. <a href="references.html#ref-Forrest2014">2014</a>)</span>, GTEx (<span class="math inline">\(n=53\)</span>
tissues) <span class="citation">(Ardlie et al. <a href="references.html#ref-GTExConsortium2015">2015</a>)</span> and the Illumina Body Map (<span class="math inline">\(n=16\)</span>
tissues), which we retrieved from the EBI Expression
Atlas <span class="citation">(Petryszak et al. <a href="references.html#ref-Petryszak2015">2015</a>)</span>. In all data sets we found more positively
correlated paralog pairs compared to the sampled gene pairs
(Fig. <a href="2-3-results.html#fig:paraData">2.1</a>E). This shows that many paralogs are expressed
with high coordination in a tissue specific manner.</p>
</div>
<div id="paralog-genes-share-enhancers" class="section level3">
<h3><span class="header-section-number">2.3.3</span> Paralog genes share enhancers</h3>
<p>We hypothesised that common gene regulation of close paralog genes is
likely to be facilitated by shared enhancer elements. Indeed we found
that paralog gene pairs within 1 Mb on the same chromosome are
associated to the same enhancer elements more often than expected by
chance (Fig. <a href="2-3-results.html#fig:sharedEnhancer">2.2</a>). We estimated the expected
background distribution of shared enhancers by carefully sampling gene
pairs with the same distributions as paralogs in distances and
associated enhancers to single genes (Fig. <a href="A-sup-paralog-regulation.html#fig:samplingDistEh">A.4</a>,
section <a href="2-2-sec-paralog-methods.html#sec:paralog-methods">2.2</a>).</p>
<!-- ---------------------------------------------------------------- -->
<div class="figure" style="text-align: center"><span id="fig:sharedEnhancer"></span>
<img src="figures/paralog/fig2_09.png" alt="Shared enhancers among paralog gene pairs. (A) Percent of close paralog (red) and sampled control (grey) gene pairs with at least one shared enhancer. (B) Percent of gene pairs versus number of shared enhancers for paralog and sampled control gene pairs." width="80%" />
<p class="caption">
Figure 2.2: <strong>Shared enhancers among paralog gene pairs. </strong> <strong>(A)</strong> Percent of close paralog (red) and sampled control (grey) gene pairs with at least one shared enhancer. <strong>(B)</strong> Percent of gene pairs versus number of shared enhancers for paralog and sampled control gene pairs.
</p>
</div>

<!-- ---------------------------------------------------------------- -->
<p>While <span class="math inline">\(27.2\)</span>% of the paralog gene pairs have at least one enhancer in
common, we observed this for only <span class="math inline">\(11.7\)</span>% of the sampled gene pairs
(<span class="math inline">\(p=4.2\times10^{-40}\)</span>, Fig. <a href="2-3-results.html#fig:sharedEnhancer">2.2</a>A). This could be
replicated when comparing against sampled gene pairs where in addition
to distance and number of enhancers linked to single genes, also the
transcription sense and gene length were taken into account during
sampling of control gene pairs (<span class="math inline">\(p=3.4\times10^{-41}\)</span> and
<span class="math inline">\(p=5\times10^{-30}\)</span>, respectively; Fig. <a href="A-sup-paralog-regulation.html#fig:ehBySampType">A.7</a>). Next, we compared the
percent of gene pairs with shared enhancers as a function of the number
of shared enhancers between paralogs and sampled gene pairs. We observed
that paralog pairs are enriched for higher number of shared enhancers
compared to the sampled gene pairs (Fig. <a href="2-3-results.html#fig:sharedEnhancer">2.2</a>B).
Together, these results indicate that paralog genes are more often
co-regulated by common enhancer elements than other genes.</p>
</div>
<div id="co-localization-of-paralogs-in-tads" class="section level3">
<h3><span class="header-section-number">2.3.4</span> Co-localization of paralogs in TADs</h3>
<p>To facilitate their function in gene regulation, distal enhancer
elements need to interact physically via chromatin looping with promoter
elements at the TSS of their target genes. These looping interactions
occur frequently within so called topological associating domains
(TADs). These are regions of hundreds of kb that show high rates of
self-interactions and few interactions across domain boundaries in
genome-wide Hi-C experiments <span class="citation">(Dixon et al. <a href="references.html#ref-Dixon2012">2012</a>; Rao et al. <a href="references.html#ref-Rao2014">2014</a>)</span>. Genes within the
same TAD are therefore likely to have common gene regulatory programs
<span class="citation">(Le Dily et al. <a href="references.html#ref-LeDily2014">2014</a>; Nora et al. <a href="references.html#ref-Nora2012">2012</a>)</span>.</p>
<p>We used TADs from Hi-C experiments in eight different human cell-types
(HeLa, HUVEC, K562, KBM7, NHEK, IMR90, GM12878, and hESC) from two
recent studies  <span class="citation">(Dixon et al. <a href="references.html#ref-Dixon2012">2012</a>; Rao et al. <a href="references.html#ref-Rao2014">2014</a>)</span>. Notably, the called TADs differ
in size between the two publications due to different resolution of Hi-C
experiments and different algorithms used to call them from Hi-C contact
matrices (Fig. <a href="A-sup-paralog-regulation.html#fig:TADsize">A.8</a>). TADs from <span class="citation">(Rao et al. <a href="references.html#ref-Rao2014">2014</a>)</span> have a median size of around
240 kb and are nested, so that several small domains can occur within
one ore more larger domains. In contrast TADs from <span class="citation">(Dixon et al. <a href="references.html#ref-Dixon2012">2012</a>)</span> are of 1
Mb on average and are defined as non-overlapping genomic intervals.</p>
<p>We hypothesised that paralog gene pairs might be located more often in
the same TAD than expected by chance. Indeed, we found that, depending
on cell-type and study, between <span class="math inline">\(35\%\)</span> and <span class="math inline">\(73\%\)</span> of close paralog pairs
are located in the same TAD (Fig. <a href="2-3-results.html#fig:TAD">2.3</a>A). In seven out of nine
data sets this difference was significant (<span class="math inline">\(p&lt;0.05\)</span>) with respect to the
sampled control gene pairs with the same linear distance. We also
calculated a set of <span class="math inline">\(n=2,624\)</span> stable TADs that are found in more than
50% of cell types analysed in <span class="citation">(Rao et al. <a href="references.html#ref-Rao2014">2014</a>)</span>. Notably, we found for paralog
pairs a <span class="math inline">\(1.25\)</span> fold enrichment to be located in the same stable TADs
compared to sampled gene pairs (<span class="math inline">\(p=0.00013\)</span>, stable_TADs in
Fig. <a href="2-3-results.html#fig:TAD">2.3</a>A).</p>
<!-- ---------------------------------------------------------------- -->
<div class="figure" style="text-align: center"><span id="fig:TAD"></span>
<img src="figures/paralog/fig3_09.png" alt="(A) Co-localization of close paralog genes within the same TAD compared against sampled gene pairs for TAD data sets from different cell types and studies. The first seven bars show values for TADs called in HeLa, HUVEC, K562, KBM7, NHEK, IMR90, and GM12878 cells by (Rao et al. 2014). The eighth bar shows the value for stable TADs across cell types form this study (at least 90% reciprocal overlap in 50% of cells). The last two bars show data for TADs called in hESC and IMR90 cells by (Dixon et al. 2012). Error bars indicate standard deviation in 10 times replicated sampling of gene pairs. P-values are computed using Fisher’s exact test. (B) Percent of gene pairs annotated to same A/B compartment according to Hi-C data in GM12878 cells from (Rao et al. 2014). Pairs located in the very same compartment interval were excluded. (C) Percent of gene pairs annotated to same sub compartment (A1, A2, B1, B2, B3, B4) according to (Rao et al. 2014). Pairs located in the same subcompartment interval were excluded. (D) Normalized Hi-C contact frequencies between TSSs of distal paralog gene pairs and sampled background gene pairs. (E) Promoter capture-C contact frequencies between distal paralog gene pairs and sampled background gene pairs." width="80%" />
<p class="caption">
Figure 2.3: <strong>(A)</strong> Co-localization of close paralog genes within the same TAD compared against sampled gene pairs for TAD data sets from different cell types and studies. The first seven bars show values for TADs called in HeLa, HUVEC, K562, KBM7, NHEK, IMR90, and GM12878 cells by <span class="citation">(Rao et al. <a href="references.html#ref-Rao2014">2014</a>)</span>. The eighth bar shows the value for stable TADs across cell types form this study (at least 90% reciprocal overlap in 50% of cells). The last two bars show data for TADs called in hESC and IMR90 cells by <span class="citation">(Dixon et al. <a href="references.html#ref-Dixon2012">2012</a>)</span>. Error bars indicate standard deviation in 10 times replicated sampling of gene pairs. P-values are computed using Fisher’s exact test. <strong>(B)</strong> Percent of gene pairs annotated to same A/B compartment according to Hi-C data in GM12878 cells from <span class="citation">(Rao et al. <a href="references.html#ref-Rao2014">2014</a>)</span>. Pairs located in the very same compartment interval were excluded. <strong>(C)</strong> Percent of gene pairs annotated to same sub compartment (A1, A2, B1, B2, B3, B4) according to <span class="citation">(Rao et al. <a href="references.html#ref-Rao2014">2014</a>)</span>. Pairs located in the same subcompartment interval were excluded. <strong>(D)</strong> Normalized Hi-C contact frequencies between TSSs of distal paralog gene pairs and sampled background gene pairs. <strong>(E)</strong> Promoter capture-C contact frequencies between distal paralog gene pairs and sampled background gene pairs.
</p>
</div>

<!-- ---------------------------------------------------------------- -->
<p>Beside TADs, Hi-C interaction maps have revealed interaction patterns of
two compartments (A and B) that alternate along chromosomes in intervals
of several Mb. Thereby loci in A compartment preferentially associate
with other loci in A and loci in B with others in
B <span class="citation">(Lieberman-Aiden et al. <a href="references.html#ref-Lieberman-Aiden2009">2009</a>; Rao et al. <a href="references.html#ref-Rao2014">2014</a>; Dekker et al. <a href="references.html#ref-Dekker2013">2013</a>)</span>. We therefore asked
whether pairs of paralogs from the same chromosome are preferentially
located within the same compartment (both A or both B) whereby we
excluded pairs that are in the same compartment interval. We found that
<span class="math inline">\(56.4\%\)</span> of paralogs on the same chromosome but not in the same
compartment interval are in compartments of the same type. This was only
observed for <span class="math inline">\(49.2\%\)</span> of sampled pairs (<span class="math inline">\(p=0.0046\)</span>, Fig. <a href="2-3-results.html#fig:TAD">2.3</a>B).
Next we tested the same for recently distinguished sub-compartment types
from high-resolution Hi-C interaction maps <span class="citation">(Rao et al. <a href="references.html#ref-Rao2014">2014</a>)</span>. Again, paralogs
are enriched to be located within the same sub-compartment type
(<span class="math inline">\(38.9\%\)</span> vs. <span class="math inline">\(31.6\%\)</span>, <span class="math inline">\(p=0.0046\)</span>, Fig. <a href="2-3-results.html#fig:TAD">2.3</a>C).</p>
<p>These results show that close paralogs are enriched to be located in the
same regulatory unit of the genome as defined both by TADs and
compartments.</p>
</div>
<div id="distal-paralog-pairs-are-enriched-for-long-range-chromatin-contacts" class="section level3">
<h3><span class="header-section-number">2.3.5</span> Distal paralog pairs are enriched for long-range chromatin contacts</h3>
<p>Since it was shown that actively transcribed genes are localized in the
same active spatial compartments and tend to contact each other
frequently in the nucleus (at their
promoters <span class="citation">(Cremer et al. <a href="references.html#ref-Cremer2015">2015</a>; Mifsud et al. <a href="references.html#ref-Mifsud2015">2015</a>)</span>) we hypothesised that this might be
the case for distal paralogs on the same chromosome too. As spatial
proximity can be approximated by Hi-C contact
frequencies <span class="citation">(Lieberman-Aiden et al. <a href="references.html#ref-Lieberman-Aiden2009">2009</a>)</span> we compared the number of normalized
Hi-C contacts between TSS of distal paralog genes (that have promoters
separated by more than 1 Mb) to the sampled gene pairs with the same
linear distances distribution. We used recently published in situ Hi-C
data from IMR90 cells at 5kb bin-size resolution <span class="citation">(Rao et al. <a href="references.html#ref-Rao2014">2014</a>)</span> and observed
significantly more normalized chromatin interactions between paralog
genes compared to sampled control gene pairs (<span class="math inline">\(p=0.0022\)</span>,
Fig. <a href="2-3-results.html#fig:TAD">2.3</a>D). We furthermore used an independent data set of high
resolution promoter-promoter interactions measured by capture
Hi-C <span class="citation">(Mifsud et al. <a href="references.html#ref-Mifsud2015">2015</a>)</span>. Again, we observed a strong enrichment of
promoter-promoter interactions between distal paralogs compared to
control genes pairs (<span class="math inline">\(p=0.027\)</span>, Fig. <a href="2-3-results.html#fig:TAD">2.3</a>E). This shows that also
distal paralogs are enriched for long-range interactions, indicating
that they tend to be in closer spatial proximity than other genes.</p>
</div>
<div id="close-paralogs-have-fewer-contacts-than-expected" class="section level3">
<h3><span class="header-section-number">2.3.6</span> Close paralogs have fewer contacts than expected</h3>
<p>The observed enrichment of Hi-C contacts of paralogs is distance
dependent. We observe for close paralogs fewer Hi-C contacts than for
equally distant sampled gene pairs (Fig. <a href="2-3-results.html#fig:closePairs">2.4</a>A). To
analyse this in more detail we focused on only those pairs on the same
chromosome that have a TSS distance of at least 10kb but less than 1Mb.
This is the distance range of most paralog pairs and allows to separate
genes in Hi-C interaction maps and TADs (Fig. <a href="A-sup-paralog-regulation.html#fig:closePairsInTAD">A.9</a>A). Consequently, we
observe paralogs more often in the same TAD in eight out of nine data
sets for this distance range (Fig. <a href="A-sup-paralog-regulation.html#fig:closePairsInTAD">A.9</a>B). For these pairs we observe
significant lower Hi-C contact frequencies if pairs are within the same
IMR90 TAD <span class="citation">(Rao et al. <a href="references.html#ref-Rao2014">2014</a>)</span> as compared to sampled genes (<span class="math inline">\(p=0.00094\)</span>) but not
if pairs are in different TADs (<span class="math inline">\(p=0.81\)</span>, Fig. <a href="2-3-results.html#fig:closePairs">2.4</a>B). We
got comparable results when analysing the Capture Hi-C data the same way
(Fig. <a href="A-sup-paralog-regulation.html#fig:closePairsInTAD">A.9</a>C). Next, we tested whether this can be explained by the nested
sub-TAD structure of TADs called from high-resolution Hi-C in
IMR90 <span class="citation">(Rao et al. <a href="references.html#ref-Rao2014">2014</a>)</span>. We divided pairs into four groups, namely, ’no TAD’,
if both pairs are not in any TAD, ’different TAD’, if pairs do not have
at least one TAD in common, ’different sub-TADs’, if they have at least
one TAD in common but are in different sub-TADs, and ’same sub-TAD’, if
they overlap exactly the same set of TADs. While we saw that paralogs
are more often in the no TAD group (<span class="math inline">\(p=1.4\times10^{-20}\)</span>), we found
that they were highly depleted from the different TAD group
(<span class="math inline">\(p=1.6\times10^{-40}\)</span>) and highly enriched to be located within the
same sub-TAD (<span class="math inline">\(p=4.2\times10^{-9}\)</span>, Fig. <a href="2-3-results.html#fig:closePairs">2.4</a>C). However,
although not always significant, paralogs have fewer Hi-C contacts than
sampled gene pairs in all of these groups (Fig. <a href="2-3-results.html#fig:closePairs">2.4</a>D). In
addition, close paralogs within the same TAD share more enhancers than
close paralogs not being in the same TAD (Fig. <a href="2-3-results.html#fig:closePairs">2.4</a>E).
However, the positive correlation of gene expression over different
tissues is not significantly higher for paralogs whether they are in the
same TAD or not (Fig. <a href="A-sup-paralog-regulation.html#fig:ExpByTAD">A.10</a>).</p>
<!-- ---------------------------------------------------------------- -->
<div class="figure" style="text-align: center"><span id="fig:closePairs"></span>
<img src="figures/paralog/fig4_09.png" alt="(A) Normalized Hi-C contacts by genomic distance between paralog (red) and sampled (grey) gene pairs. Lines show linear regression fit separately for paralogs (red) and sampled (grey) pairs with 95% confidence intervals in shaded areas. (B) Normalized Hi-C contacts between pairs of paralogs (red) and sampled gene pairs (grey) for the groups: $&lt;$10kb genomic distance, located in the same TAD, not in the same TAD, and with genomic distance $&gt;$1000kb. (C) Number of gene pairs located either in no TAD, in different TADs (or only one pair member in a TAD), both in a TAD but in different sub-TADs, or within the same sub-TAD, for paralogs (red) and sampled (grey) pairs. TADs from IMR90 cells from (Rao et al. 2014) were used, which nested in contrast to TAD calls from (Dixon et al. 2012). (D) Normalized Hi-C contacts between pairs of paralogs (red) and sampled gene pairs (grey) for the four groups of pairs in sub-TAD structures shown in (C). (E) Percent of gene pairs with at least one shared enhancer for paralog genes (red) and sampled control genes (grey) separated for pairs in the same IMR90 TAD (left) or not (right)." width="80%" />
<p class="caption">
Figure 2.4: <strong>(A)</strong> Normalized Hi-C contacts by genomic distance between paralog (red) and sampled (grey) gene pairs. Lines show linear regression fit separately for paralogs (red) and sampled (grey) pairs with 95% confidence intervals in shaded areas. <strong>(B)</strong> Normalized Hi-C contacts between pairs of paralogs (red) and sampled gene pairs (grey) for the groups: $&lt;$10kb genomic distance, located in the same TAD, not in the same TAD, and with genomic distance $&gt;$1000kb. <strong>(C)</strong> Number of gene pairs located either in no TAD, in different TADs (or only one pair member in a TAD), both in a TAD but in different sub-TADs, or within the same sub-TAD, for paralogs (red) and sampled (grey) pairs. TADs from IMR90 cells from <span class="citation">(Rao et al. <a href="references.html#ref-Rao2014">2014</a>)</span> were used, which nested in contrast to TAD calls from <span class="citation">(Dixon et al. <a href="references.html#ref-Dixon2012">2012</a>)</span>. <strong>(D)</strong> Normalized Hi-C contacts between pairs of paralogs (red) and sampled gene pairs (grey) for the four groups of pairs in sub-TAD structures shown in (C). <strong>(E)</strong> Percent of gene pairs with at least one shared enhancer for paralog genes (red) and sampled control genes (grey) separated for pairs in the same IMR90 TAD (left) or not (right).
</p>
</div>

<!-- ---------------------------------------------------------------- -->
<p>In summary, we observed that while close paralogs (situated at less than
1Mb) have more shared enhancers if they are in the same TAD than not,
these within TAD paralog pairs have fewer contacts compared to other
within TAD pairs of genes.</p>
</div>
<div id="paralogs-in-mouse-and-dog-genome" class="section level3">
<h3><span class="header-section-number">2.3.7</span> Paralogs in mouse and dog genome</h3>
<p>Next, we asked whether the co-localization and co-regulation of paralogs
is conserved in other species. For this, we conducted an analogous
analysis with paralog gene pairs from mouse (<em>M. musculus</em>) and dog (<em>C.
familaris</em>) genomes. Similar as for human data, we found that more than
two third of the genes had at least one paralog copy (Fig. <a href="A-sup-paralog-regulation.html#fig:paralogsSpecies">A.11</a>A,D),
paralog pairs clustered on the same chromosome (Fig. <a href="A-sup-paralog-regulation.html#fig:paralogsSpecies">A.11</a>B,E), and had
close linear distances (Fig. <a href="A-sup-paralog-regulation.html#fig:paralogsSpecies">A.11</a>C,F).</p>
<p>We sampled control gene pairs with the same distance distribution as
paralogs for both species separately (Fig. <a href="A-sup-paralog-regulation.html#fig:paralogsSpecies">A.11</a>C,F). We used TADs from
recently published Hi-C data in liver cells of mouse and
dog <span class="citation">(Vietri Rudan et al. <a href="references.html#ref-VietriRudan2015">2015</a>)</span>, which have a size distribution comparable to
TADs from human cells (Fig. <a href="A-sup-paralog-regulation.html#fig:TADsize">A.8</a>). We computed the fraction of paralog
pairs that are located in the same TAD for both species. Consistent with
the observation in human, we found that paralogs tend to colocalize more
frequently within the same TAD in mouse (<span class="math inline">\(p=7.2 \times 10^{-22}\)</span>) and
dog (<span class="math inline">\(p=0.00064\)</span>) than expected by chance
(Fig. <a href="2-3-results.html#fig:paralogsMouseDog">2.5</a>A). We also quantified directly the
contact frequencies between promoters of distal paralogs on the same
chromosome and found them significantly more frequently in contact than
sampled gene pairs with the same genomic distance for paralogs in mouse
(<span class="math inline">\(p=7\times 10^{-7}\)</span>) and dog (<span class="math inline">\(p=0.008\)</span>)
(Fig. <a href="2-3-results.html#fig:paralogsMouseDog">2.5</a>B). Together, these results indicate that
enriched long-range interactions between paralogs are not human specific
but rather a general evolutionary conserved feature of genome
organization.</p>
<!-- ---------------------------------------------------------------- -->
<div class="figure" style="text-align: center"><span id="fig:paralogsMouseDog"></span>
<img src="figures/paralog/fig5_09.png" alt="(A) Co-occurrence of close paralog genes with the same TAD in mouse (left panel) and dog (right panel). (B) Hi-C contacts between promoter of distal gene pairs in Hi-C experiments in liver cells from mouse (left panel) and dog (right panel). Hi-C data and TAD calls were taken from (Vietri Rudan et al. 2015)." width="50%" />
<p class="caption">
Figure 2.5: <strong>(A)</strong> Co-occurrence of close paralog genes with the same TAD in mouse (left panel) and dog (right panel). <strong>(B)</strong> Hi-C contacts between promoter of distal gene pairs in Hi-C experiments in liver cells from mouse (left panel) and dog (right panel). Hi-C data and TAD calls were taken from <span class="citation">(Vietri Rudan et al. <a href="references.html#ref-VietriRudan2015">2015</a>)</span>.
</p>
</div>

<!-- ---------------------------------------------------------------- -->
</div>
<div id="orthologs-of-human-paralogs-show-conserved-co-localization" class="section level3">
<h3><span class="header-section-number">2.3.8</span> Orthologs of human paralogs show conserved co-localization</h3>
<p>Next, we wanted to test more directly whether the spatial
co-localization of human paralogs is indeed conserved during evolution.
In cases where the gene duplication event occurred before the separation
of human and mouse (or human and dog) we can eventually assign each
human gene of a pair of paralogs to one ortholog in mouse (or dog
genomes) (Fig. <a href="A-sup-paralog-regulation.html#fig:orthologModel">A.12</a>).</p>
<p>We could map <span class="math inline">\(37.1\%\)</span> (<span class="math inline">\(n=579\)</span>) and <span class="math inline">\(34.6\%\)</span> (<span class="math inline">\(n=540\)</span>) of the close
human paralogs to one-to-one orthologs in mouse and dog, respectively
(Fig. <a href="A-sup-paralog-regulation.html#fig:orthologsSpecies">A.13</a>A,D). We hypothesised that the two one-to-one orthologs of
human paralog pairs would also be close in the mouse and dog genomes.
Indeed, we found that the orthologs of human paralogs tend to cluster on
the same chromosome (Fig. <a href="A-sup-paralog-regulation.html#fig:orthologsSpecies">A.13</a>B,E) and are biased for close linear
distances (Fig. <a href="A-sup-paralog-regulation.html#fig:orthologsSpecies">A.13</a>C,F).</p>
<p>We further investigated how many one-to-one orthologs of the human
paralog pairs were located in the same TAD in mouse and dog genomes.
Although not significant, we found that mouse orthologs of close human
paralogs share more often the same TAD in mouse than orthologs of
sampled human gene pairs (<span class="math inline">\(80\%\)</span> vs. <span class="math inline">\(76\%\)</span>, <span class="math inline">\(p=0.11\)</span>;
Fig. <a href="2-3-results.html#fig:orthologsMouseDog">2.6</a>A). Significant enrichment was observed
with orthologs in the dog genome (<span class="math inline">\(85\%\)</span> vs. <span class="math inline">\(77\%\)</span>, <span class="math inline">\(p=0.0016\)</span>;
Fig. <a href="2-3-results.html#fig:orthologsMouseDog">2.6</a>A).</p>
<!-- ---------------------------------------------------------------- -->
<div class="figure" style="text-align: center"><span id="fig:orthologsMouseDog"></span>
<img src="figures/paralog/fig6_09.png" alt="** One-to-one orthologs of human paralog genes in mouse and dog genome.** (A) Percent of mouse (left) and dog (right) orthologs of human paralog pairs that are in the same TAD in the mouse and dog genome, respectively. (B) Normalized Hi-C contacts between promoters of one-to-one orthologs of human distal paralogs in the mouse (left) and dog (right) genome. (C) Percent of gene pairs with conserved co-localization. Orthologs in the same TAD in mouse (left) and dog (right) as percent of all orthologs of human paralog pairs that are in the same TAD in human. For human TADs from IMR90 cells from (Rao et al. 2014) were used." width="75%" />
<p class="caption">
Figure 2.6: ** One-to-one orthologs of human paralog genes in mouse and dog genome.** <strong>(A)</strong> Percent of mouse (left) and dog (right) orthologs of human paralog pairs that are in the same TAD in the mouse and dog genome, respectively. <strong>(B)</strong> Normalized Hi-C contacts between promoters of one-to-one orthologs of human distal paralogs in the mouse (left) and dog (right) genome. <strong>(C)</strong> Percent of gene pairs with conserved co-localization. Orthologs in the same TAD in mouse (left) and dog (right) as percent of all orthologs of human paralog pairs that are in the same TAD in human. For human TADs from IMR90 cells from <span class="citation">(Rao et al. <a href="references.html#ref-Rao2014">2014</a>)</span> were used.
</p>
</div>

<!-- ---------------------------------------------------------------- -->
<p>For distal human paralogs we quantified the promoter contacts of their
orthologs in mouse and dog and found enriched Hi-C contacts in mouse
(<span class="math inline">\(p=0.011\)</span>) and dog (<span class="math inline">\(p=2.4\times 10^{-5}\)</span>;
Fig. <a href="2-3-results.html#fig:orthologsMouseDog">2.6</a>B).</p>
<p>These results show that both the co-localization of paralogs in TADs and
the contacts between distal paralogs are only weekly conserved at the
evolutionary distances examined here. For example, we see that given a
pair of human genes in the same TAD the likelihood of their orthologs
being in the same TAD in mouse or dog is the same whether they are
paralogs or not (Fig. <a href="2-3-results.html#fig:orthologsMouseDog">2.6</a>C).</p>
<p>All together, our results support the notion that tandem duplications
generate paralog gene pairs that are selected if they accommodate in
TADs but following evolutionary events allow their reorganization
outside TADs. While within oganisms distal paralog genes are
coordinated, such coordination can be eventually erased by evolution.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="2-2-sec-paralog-methods.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="2-4-discussions.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": true,
"facebook": false,
"twitter": true,
"google": false,
"weibo": false,
"instapper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/ibn-salem/thesis/edit/master/02-paralog_regulation.Rmd",
"text": "Edit"
},
"download": ["thesis.pdf", "thesis.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://cdn.bootcss.com/mathjax/2.7.1/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:" && /^https?:/.test(src))
      src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
