<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>Genome folding in evolution and disease</title>
  <meta name="description" content="A PhD thesis on genome folding in evoultion and disease.">
  <meta name="generator" content="bookdown 0.6.1 and GitBook 2.6.7">

  <meta property="og:title" content="Genome folding in evolution and disease" />
  <meta property="og:type" content="book" />
  <meta property="og:url" content="https://ibn-salem.github.io/thesis/" />
  
  <meta property="og:description" content="A PhD thesis on genome folding in evoultion and disease." />
  <meta name="github-repo" content="ibn-salem/thesis" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Genome folding in evolution and disease" />
  
  <meta name="twitter:description" content="A PhD thesis on genome folding in evoultion and disease." />
  

<meta name="author" content="Jonas Ibn-Salem">


<meta name="date" content="2018-08-27">

  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  
<link rel="prev" href="3-1-introduction-1.html">
<link rel="next" href="3-3-discussion.html">
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />










<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Thesis</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="1" data-path="1-intro.html"><a href="1-intro.html"><i class="fa fa-check"></i><b>1</b> Introduction</a><ul>
<li class="chapter" data-level="1.1" data-path="1-1-regulation-of-expression.html"><a href="1-1-regulation-of-expression.html"><i class="fa fa-check"></i><b>1.1</b> Regulation of gene expression</a></li>
<li class="chapter" data-level="1.2" data-path="1-2-distal-regulation-by-enhancers.html"><a href="1-2-distal-regulation-by-enhancers.html"><i class="fa fa-check"></i><b>1.2</b> Distal regulation by enhancers</a></li>
<li class="chapter" data-level="1.3" data-path="1-3-methods-to-probe-the-3d-chromatin-architecture.html"><a href="1-3-methods-to-probe-the-3d-chromatin-architecture.html"><i class="fa fa-check"></i><b>1.3</b> Methods to probe the 3D chromatin architecture</a><ul>
<li class="chapter" data-level="1.3.1" data-path="1-3-methods-to-probe-the-3d-chromatin-architecture.html"><a href="1-3-methods-to-probe-the-3d-chromatin-architecture.html#microscopy-based-techniques-to-visualize-the-genome-in-3d"><i class="fa fa-check"></i><b>1.3.1</b> Microscopy-based techniques to visualize the genome in 3D</a></li>
<li class="chapter" data-level="1.3.2" data-path="1-3-methods-to-probe-the-3d-chromatin-architecture.html"><a href="1-3-methods-to-probe-the-3d-chromatin-architecture.html#proximity-ligation"><i class="fa fa-check"></i><b>1.3.2</b> Proximity-ligation based method to quantify chromatin interactions</a></li>
</ul></li>
<li class="chapter" data-level="1.4" data-path="1-4-hierarchy-of-chromatin-3d-structure.html"><a href="1-4-hierarchy-of-chromatin-3d-structure.html"><i class="fa fa-check"></i><b>1.4</b> Hierarchy of chromatin 3D structure</a><ul>
<li class="chapter" data-level="1.4.1" data-path="1-4-hierarchy-of-chromatin-3d-structure.html"><a href="1-4-hierarchy-of-chromatin-3d-structure.html#chromosomal-territories-and-inter-chromosomal-contacts"><i class="fa fa-check"></i><b>1.4.1</b> Chromosomal territories and inter-chromosomal contacts</a></li>
<li class="chapter" data-level="1.4.2" data-path="1-4-hierarchy-of-chromatin-3d-structure.html"><a href="1-4-hierarchy-of-chromatin-3d-structure.html#ab-compartments"><i class="fa fa-check"></i><b>1.4.2</b> A/B compartments</a></li>
<li class="chapter" data-level="1.4.3" data-path="1-4-hierarchy-of-chromatin-3d-structure.html"><a href="1-4-hierarchy-of-chromatin-3d-structure.html#topologically-associating-domains-tads"><i class="fa fa-check"></i><b>1.4.3</b> Topologically associating domains (TADs)</a></li>
<li class="chapter" data-level="1.4.4" data-path="1-4-hierarchy-of-chromatin-3d-structure.html"><a href="1-4-hierarchy-of-chromatin-3d-structure.html#hierarchy-of-domain-structures-across-genomic-length-scales"><i class="fa fa-check"></i><b>1.4.4</b> Hierarchy of domain structures across genomic length scales</a></li>
<li class="chapter" data-level="1.4.5" data-path="1-4-hierarchy-of-chromatin-3d-structure.html"><a href="1-4-hierarchy-of-chromatin-3d-structure.html#chromatin-looping-interactions"><i class="fa fa-check"></i><b>1.4.5</b> Chromatin looping interactions</a></li>
<li class="chapter" data-level="1.4.6" data-path="1-4-hierarchy-of-chromatin-3d-structure.html"><a href="1-4-hierarchy-of-chromatin-3d-structure.html#loop-mechanism"><i class="fa fa-check"></i><b>1.4.6</b> TAD and loop formation by architectural proteins</a></li>
</ul></li>
<li class="chapter" data-level="1.5" data-path="1-5-dynamics-of-chromatin-structure.html"><a href="1-5-dynamics-of-chromatin-structure.html"><i class="fa fa-check"></i><b>1.5</b> Dynamics of chromatin structure</a><ul>
<li class="chapter" data-level="1.5.1" data-path="1-5-dynamics-of-chromatin-structure.html"><a href="1-5-dynamics-of-chromatin-structure.html#dynamics-across-the-cell-cycle"><i class="fa fa-check"></i><b>1.5.1</b> Dynamics across the cell cycle</a></li>
<li class="chapter" data-level="1.5.2" data-path="1-5-dynamics-of-chromatin-structure.html"><a href="1-5-dynamics-of-chromatin-structure.html#dynamics-across-cell-types-and-differntiation"><i class="fa fa-check"></i><b>1.5.2</b> Dynamics across cell types and differntiation</a></li>
</ul></li>
<li class="chapter" data-level="1.6" data-path="1-6-evolution-of-chromatin-organization.html"><a href="1-6-evolution-of-chromatin-organization.html"><i class="fa fa-check"></i><b>1.6</b> Evolution of chromatin organization</a></li>
<li class="chapter" data-level="1.7" data-path="1-7-disruption-of-chromatin-architecture-in-disease.html"><a href="1-7-disruption-of-chromatin-architecture-in-disease.html"><i class="fa fa-check"></i><b>1.7</b> Disruption of chromatin architecture in disease</a></li>
<li class="chapter" data-level="1.8" data-path="1-8-aims-of-this-thesis.html"><a href="1-8-aims-of-this-thesis.html"><i class="fa fa-check"></i><b>1.8</b> Aims of this thesis</a></li>
<li class="chapter" data-level="1.9" data-path="1-9-structure-of-this-thesis.html"><a href="1-9-structure-of-this-thesis.html"><i class="fa fa-check"></i><b>1.9</b> Structure of this thesis</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="2-paralog-regulation.html"><a href="2-paralog-regulation.html"><i class="fa fa-check"></i><b>2</b> Paralog genes in the 3D genome architecture</a><ul>
<li class="chapter" data-level="" data-path="2-paralog-regulation.html"><a href="2-paralog-regulation.html#preamble"><i class="fa fa-check"></i>Preamble</a></li>
<li class="chapter" data-level="" data-path="2-paralog-regulation.html"><a href="2-paralog-regulation.html#abstract-1"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="2.1" data-path="2-1-introduction.html"><a href="2-1-introduction.html"><i class="fa fa-check"></i><b>2.1</b> Introduction</a></li>
<li class="chapter" data-level="2.2" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html"><i class="fa fa-check"></i><b>2.2</b> Materials and methods</a><ul>
<li class="chapter" data-level="2.2.1" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html#selection-of-pairs-of-paralog-genes"><i class="fa fa-check"></i><b>2.2.1</b> Selection of pairs of paralog genes</a></li>
<li class="chapter" data-level="2.2.2" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html#enhancers-to-gene-association"><i class="fa fa-check"></i><b>2.2.2</b> Enhancers to gene association</a></li>
<li class="chapter" data-level="2.2.3" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html#topological-associating-domains"><i class="fa fa-check"></i><b>2.2.3</b> Topological associating domains</a></li>
<li class="chapter" data-level="2.2.4" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html#hi-c-interaction-maps"><i class="fa fa-check"></i><b>2.2.4</b> Hi-C interaction maps</a></li>
<li class="chapter" data-level="2.2.5" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html#randomization"><i class="fa fa-check"></i><b>2.2.5</b> Randomization</a></li>
<li class="chapter" data-level="2.2.6" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html#statistical-tests"><i class="fa fa-check"></i><b>2.2.6</b> Statistical tests</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="2-3-results.html"><a href="2-3-results.html"><i class="fa fa-check"></i><b>2.3</b> Results</a><ul>
<li class="chapter" data-level="2.3.1" data-path="2-3-results.html"><a href="2-3-results.html#distribution-of-paralog-genes-in-the-human-genome"><i class="fa fa-check"></i><b>2.3.1</b> Distribution of paralog genes in the human genome</a></li>
<li class="chapter" data-level="2.3.2" data-path="2-3-results.html"><a href="2-3-results.html#co-expression-of-paralog-gene-pairs-across-tissues"><i class="fa fa-check"></i><b>2.3.2</b> Co-expression of paralog gene pairs across tissues</a></li>
<li class="chapter" data-level="2.3.3" data-path="2-3-results.html"><a href="2-3-results.html#paralog-genes-share-enhancers"><i class="fa fa-check"></i><b>2.3.3</b> Paralog genes share enhancers</a></li>
<li class="chapter" data-level="2.3.4" data-path="2-3-results.html"><a href="2-3-results.html#co-localization-of-paralogs-in-tads"><i class="fa fa-check"></i><b>2.3.4</b> Co-localization of paralogs in TADs</a></li>
<li class="chapter" data-level="2.3.5" data-path="2-3-results.html"><a href="2-3-results.html#distal-paralog-pairs-are-enriched-for-long-range-chromatin-contacts"><i class="fa fa-check"></i><b>2.3.5</b> Distal paralog pairs are enriched for long-range chromatin contacts</a></li>
<li class="chapter" data-level="2.3.6" data-path="2-3-results.html"><a href="2-3-results.html#close-paralogs-have-fewer-contacts-than-expected"><i class="fa fa-check"></i><b>2.3.6</b> Close paralogs have fewer contacts than expected</a></li>
<li class="chapter" data-level="2.3.7" data-path="2-3-results.html"><a href="2-3-results.html#paralogs-in-mouse-and-dog-genome"><i class="fa fa-check"></i><b>2.3.7</b> Paralogs in mouse and dog genome</a></li>
<li class="chapter" data-level="2.3.8" data-path="2-3-results.html"><a href="2-3-results.html#orthologs-of-human-paralogs-show-conserved-co-localization"><i class="fa fa-check"></i><b>2.3.8</b> Orthologs of human paralogs show conserved co-localization</a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="2-4-discussions.html"><a href="2-4-discussions.html"><i class="fa fa-check"></i><b>2.4</b> Discussions</a></li>
<li class="chapter" data-level="2.5" data-path="2-5-conclusion.html"><a href="2-5-conclusion.html"><i class="fa fa-check"></i><b>2.5</b> Conclusion</a></li>
<li class="chapter" data-level="" data-path="acknowledgements.html"><a href="acknowledgements.html"><i class="fa fa-check"></i>Acknowledgements</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="3-TAD-evolution.html"><a href="3-TAD-evolution.html"><i class="fa fa-check"></i><b>3</b> Stability of TADs in evolution</a><ul>
<li class="chapter" data-level="" data-path="3-TAD-evolution.html"><a href="3-TAD-evolution.html#preamble-1"><i class="fa fa-check"></i>Preamble</a></li>
<li class="chapter" data-level="" data-path="3-TAD-evolution.html"><a href="3-TAD-evolution.html#abstract-2"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="3.1" data-path="3-1-introduction-1.html"><a href="3-1-introduction-1.html"><i class="fa fa-check"></i><b>3.1</b> Introduction</a></li>
<li class="chapter" data-level="3.2" data-path="3-2-results-1.html"><a href="3-2-results-1.html"><i class="fa fa-check"></i><b>3.2</b> Results</a><ul>
<li class="chapter" data-level="3.2.1" data-path="3-2-results-1.html"><a href="3-2-results-1.html#identification-of-evolutionary-rearrangement-breakpoints-from-whole-genome-alignments"><i class="fa fa-check"></i><b>3.2.1</b> Identification of evolutionary rearrangement breakpoints from whole-genome alignments</a></li>
<li class="chapter" data-level="3.2.2" data-path="3-2-results-1.html"><a href="3-2-results-1.html#rearrangement-breakpoints-are-enriched-at-tad-boundaries"><i class="fa fa-check"></i><b>3.2.2</b> Rearrangement breakpoints are enriched at TAD boundaries</a></li>
<li class="chapter" data-level="3.2.3" data-path="3-2-results-1.html"><a href="3-2-results-1.html#clusters-of-conserved-non-coding-elements-are-depleted-for-rearrangement-breakpoints"><i class="fa fa-check"></i><b>3.2.3</b> Clusters of conserved non-coding elements are depleted for rearrangement breakpoints</a></li>
<li class="chapter" data-level="3.2.4" data-path="3-2-results-1.html"><a href="3-2-results-1.html#rearranged-tads-are-associated-with-divergent-gene-expression-between-species"><i class="fa fa-check"></i><b>3.2.4</b> Rearranged TADs are associated with divergent gene expression between species</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="3-3-discussion.html"><a href="3-3-discussion.html"><i class="fa fa-check"></i><b>3.3</b> Discussion</a></li>
<li class="chapter" data-level="3.4" data-path="3-4-conclusion-1.html"><a href="3-4-conclusion-1.html"><i class="fa fa-check"></i><b>3.4</b> Conclusion</a></li>
<li class="chapter" data-level="3.5" data-path="3-5-methods.html"><a href="3-5-methods.html"><i class="fa fa-check"></i><b>3.5</b> Methods</a><ul>
<li class="chapter" data-level="3.5.1" data-path="3-5-methods.html"><a href="3-5-methods.html#rearrangement-breakpoints-from-whole-genome-alignments"><i class="fa fa-check"></i><b>3.5.1</b> Rearrangement breakpoints from whole-genome alignments</a></li>
<li class="chapter" data-level="3.5.2" data-path="3-5-methods.html"><a href="3-5-methods.html#topologically-associating-domains-and-contact-domains"><i class="fa fa-check"></i><b>3.5.2</b> Topologically associating domains and contact domains</a></li>
<li class="chapter" data-level="3.5.3" data-path="3-5-methods.html"><a href="3-5-methods.html#breakpoint-distributions-at-tads"><i class="fa fa-check"></i><b>3.5.3</b> Breakpoint distributions at TADs</a></li>
<li class="chapter" data-level="3.5.4" data-path="3-5-methods.html"><a href="3-5-methods.html#quantification-of-breakpoint-enrichment"><i class="fa fa-check"></i><b>3.5.4</b> Quantification of breakpoint enrichment</a></li>
<li class="chapter" data-level="3.5.5" data-path="3-5-methods.html"><a href="3-5-methods.html#expression-data-for-mouse-and-human-orthologs"><i class="fa fa-check"></i><b>3.5.5</b> Expression data for mouse and human orthologs</a></li>
<li class="chapter" data-level="3.5.6" data-path="3-5-methods.html"><a href="3-5-methods.html#classification-of-tads-and-genes-according-to-rearrangements-and-grbs"><i class="fa fa-check"></i><b>3.5.6</b> Classification of TADs and genes according to rearrangements and GRBs</a></li>
<li class="chapter" data-level="3.5.7" data-path="3-5-methods.html"><a href="3-5-methods.html#source-code-and-implementation-details"><i class="fa fa-check"></i><b>3.5.7</b> Source code and implementation details</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="declarations.html"><a href="declarations.html"><i class="fa fa-check"></i>Declarations</a><ul>
<li class="chapter" data-level="" data-path="declarations.html"><a href="declarations.html#availability-of-data-and-material"><i class="fa fa-check"></i>Availability of data and material</a></li>
<li class="chapter" data-level="" data-path="declarations.html"><a href="declarations.html#competing-interests"><i class="fa fa-check"></i>Competing interests</a></li>
<li class="chapter" data-level="" data-path="declarations.html"><a href="declarations.html#authors-contributions"><i class="fa fa-check"></i>Authors‘ contributions</a></li>
<li class="chapter" data-level="" data-path="declarations.html"><a href="declarations.html#acknowledgments"><i class="fa fa-check"></i>Acknowledgments</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="4-position-effect.html"><a href="4-position-effect.html"><i class="fa fa-check"></i><b>4</b> Position effects of rearrangements in disease genomes</a><ul>
<li class="chapter" data-level="" data-path="4-position-effect.html"><a href="4-position-effect.html#preamble-2"><i class="fa fa-check"></i>Preamble</a></li>
<li class="chapter" data-level="" data-path="4-position-effect.html"><a href="4-position-effect.html#abstract-3"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="4.1" data-path="4-1-introduction-2.html"><a href="4-1-introduction-2.html"><i class="fa fa-check"></i><b>4.1</b> Introduction</a></li>
<li class="chapter" data-level="4.2" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html"><i class="fa fa-check"></i><b>4.2</b> Materials and Methods</a><ul>
<li class="chapter" data-level="4.2.1" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html#selection-of-subjects-with-apparently-balanced-chromosome-abnormalities"><i class="fa fa-check"></i><b>4.2.1</b> Selection of subjects with apparently balanced chromosome abnormalities</a></li>
<li class="chapter" data-level="4.2.2" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html#clinical-descriptions-of-dgap-cases"><i class="fa fa-check"></i><b>4.2.2</b> Clinical descriptions of DGAP cases</a></li>
<li class="chapter" data-level="4.2.3" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html#analysis-of-genes-bordering-the-rearrangement-breakpoints"><i class="fa fa-check"></i><b>4.2.3</b> Analysis of genes bordering the rearrangement breakpoints</a></li>
<li class="chapter" data-level="4.2.4" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html#assessment-of-disrupted-functional-elements-and-chromatin-interactions-bordering-rearrangement-breakpoints"><i class="fa fa-check"></i><b>4.2.4</b> Assessment of disrupted functional elements and chromatin interactions bordering rearrangement breakpoints</a></li>
<li class="chapter" data-level="4.2.5" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html#ontological-analysis-of-genes-neighboring-breakpoints"><i class="fa fa-check"></i><b>4.2.5</b> Ontological analysis of genes neighboring breakpoints</a></li>
<li class="chapter" data-level="4.2.6" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html#quantitative-real-time-pcr"><i class="fa fa-check"></i><b>4.2.6</b> Quantitative real-time PCR</a></li>
<li class="chapter" data-level="4.2.7" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html#assessment-of-dgap-breakpoints-overlapping-with-non-coding-structural-variants-in-public-databases"><i class="fa fa-check"></i><b>4.2.7</b> Assessment of DGAP breakpoints overlapping with non-coding structural variants in public databases</a></li>
</ul></li>
<li class="chapter" data-level="4.3" data-path="4-3-results-2.html"><a href="4-3-results-2.html"><i class="fa fa-check"></i><b>4.3</b> Results</a><ul>
<li class="chapter" data-level="4.3.1" data-path="4-3-results-2.html"><a href="4-3-results-2.html#genomic-characterization-of-non-coding-breakpoints"><i class="fa fa-check"></i><b>4.3.1</b> Genomic characterization of non-coding breakpoints</a></li>
<li class="chapter" data-level="4.3.2" data-path="4-3-results-2.html"><a href="4-3-results-2.html#identification-of-genes-with-potential-position-effects"><i class="fa fa-check"></i><b>4.3.2</b> Identification of genes with potential position effects</a></li>
<li class="chapter" data-level="4.3.3" data-path="4-3-results-2.html"><a href="4-3-results-2.html#identification-of-subjects-with-shared-non-coding-chromosome-alterations-and-phenotypes"><i class="fa fa-check"></i><b>4.3.3</b> Identification of subjects with shared non-coding chromosome alterations and phenotypes</a></li>
</ul></li>
<li class="chapter" data-level="4.4" data-path="4-4-discussion-1.html"><a href="4-4-discussion-1.html"><i class="fa fa-check"></i><b>4.4</b> Discussion</a></li>
<li class="chapter" data-level="" data-path="declarations-1.html"><a href="declarations-1.html"><i class="fa fa-check"></i>Declarations</a><ul>
<li class="chapter" data-level="" data-path="declarations-1.html"><a href="declarations-1.html#acknowledgements-1"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="" data-path="declarations-1.html"><a href="declarations-1.html#web-resources"><i class="fa fa-check"></i>Web Resources</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="5-loop.html"><a href="5-loop.html"><i class="fa fa-check"></i><b>5</b> Prediction of chromatin looping interactions</a><ul>
<li class="chapter" data-level="" data-path="5-loop.html"><a href="5-loop.html#preamble-3"><i class="fa fa-check"></i>Preamble</a></li>
<li class="chapter" data-level="" data-path="5-loop.html"><a href="5-loop.html#abstract-4"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="5.1" data-path="5-1-introduction-3.html"><a href="5-1-introduction-3.html"><i class="fa fa-check"></i><b>5.1</b> Introduction</a></li>
<li class="chapter" data-level="5.2" data-path="5-2-results-3.html"><a href="5-2-results-3.html"><i class="fa fa-check"></i><b>5.2</b> Results</a><ul>
<li class="chapter" data-level="5.2.1" data-path="5-2-results-3.html"><a href="5-2-results-3.html#ctcf-motif-pairs-as-candidate-chromatin-loop-anchors"><i class="fa fa-check"></i><b>5.2.1</b> CTCF motif pairs as candidate chromatin loop anchors</a></li>
<li class="chapter" data-level="5.2.2" data-path="5-2-results-3.html"><a href="5-2-results-3.html#similarity-of-chip-seq-signals-at-looping-ctcf-motifs"><i class="fa fa-check"></i><b>5.2.2</b> Similarity of ChIP-seq signals at looping CTCF motifs</a></li>
<li class="chapter" data-level="5.2.3" data-path="5-2-results-3.html"><a href="5-2-results-3.html#genomic-sequence-features-of-ctcf-motif-pairs-are-associated-with-looping"><i class="fa fa-check"></i><b>5.2.3</b> Genomic sequence features of CTCF motif pairs are associated with looping</a></li>
<li class="chapter" data-level="5.2.4" data-path="5-2-results-3.html"><a href="5-2-results-3.html#chromatin-loop-prediction-using-7c"><i class="fa fa-check"></i><b>5.2.4</b> Chromatin loop prediction using 7C</a></li>
<li class="chapter" data-level="5.2.5" data-path="5-2-results-3.html"><a href="5-2-results-3.html#prediction-performance-evaluation"><i class="fa fa-check"></i><b>5.2.5</b> Prediction performance evaluation</a></li>
<li class="chapter" data-level="5.2.6" data-path="5-2-results-3.html"><a href="5-2-results-3.html#prediction-performance-of-7c-with-sequence-features-and-single-tf-chip-seq-data-sets"><i class="fa fa-check"></i><b>5.2.6</b> Prediction performance of 7C with sequence features and single TF ChIP-seq data sets</a></li>
<li class="chapter" data-level="5.2.7" data-path="5-2-results-3.html"><a href="5-2-results-3.html#comparison-of-transcription-factors-by-prediction-performance"><i class="fa fa-check"></i><b>5.2.7</b> Comparison of transcription factors by prediction performance</a></li>
<li class="chapter" data-level="5.2.8" data-path="5-2-results-3.html"><a href="5-2-results-3.html#prediction-performance-in-other-cell-types-and-for-different-tfs"><i class="fa fa-check"></i><b>5.2.8</b> Prediction performance in other cell types and for different TFs</a></li>
<li class="chapter" data-level="5.2.9" data-path="5-2-results-3.html"><a href="5-2-results-3.html#the-high-resolution-of-chip-nexus-improves-prediction-performance"><i class="fa fa-check"></i><b>5.2.9</b> The high resolution of ChIP-nexus improves prediction performance</a></li>
</ul></li>
<li class="chapter" data-level="5.3" data-path="5-3-discussion-2.html"><a href="5-3-discussion-2.html"><i class="fa fa-check"></i><b>5.3</b> Discussion</a></li>
<li class="chapter" data-level="5.4" data-path="5-4-conclusion-2.html"><a href="5-4-conclusion-2.html"><i class="fa fa-check"></i><b>5.4</b> Conclusion</a></li>
<li class="chapter" data-level="5.5" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html"><i class="fa fa-check"></i><b>5.5</b> Methods</a><ul>
<li class="chapter" data-level="5.5.1" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#ctcf-motifs-in-the-human-genome"><i class="fa fa-check"></i><b>5.5.1</b> CTCF motifs in the human genome</a></li>
<li class="chapter" data-level="5.5.2" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#loop-interaction-data-for-training-and-validation"><i class="fa fa-check"></i><b>5.5.2</b> Loop interaction data for training and validation</a></li>
<li class="chapter" data-level="5.5.3" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#chip-seq-datasets-in-gm12878-cells"><i class="fa fa-check"></i><b>5.5.3</b> ChIP-seq datasets in GM12878 cells</a></li>
<li class="chapter" data-level="5.5.4" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#chip-seq-data-types"><i class="fa fa-check"></i><b>5.5.4</b> ChIP-seq data types</a></li>
<li class="chapter" data-level="5.5.5" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#chip-nexus-data-processing-for-rad21-and-scm3"><i class="fa fa-check"></i><b>5.5.5</b> ChIP-nexus data processing for RAD21 and SCM3</a></li>
<li class="chapter" data-level="5.5.6" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#similarity-of-chip-seq-profiles-as-correlation-of-coverage-around-motifs"><i class="fa fa-check"></i><b>5.5.6</b> Similarity of ChIP-seq profiles as correlation of coverage around motifs</a></li>
<li class="chapter" data-level="5.5.7" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#genomic-sequence-features-of-chromatin-loops"><i class="fa fa-check"></i><b>5.5.7</b> Genomic sequence features of chromatin loops</a></li>
<li class="chapter" data-level="5.5.8" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#chromatin-loop-prediction-model"><i class="fa fa-check"></i><b>5.5.8</b> Chromatin Loop prediction model</a></li>
<li class="chapter" data-level="5.5.9" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#training-and-validation-of-prediction-model"><i class="fa fa-check"></i><b>5.5.9</b> Training and validation of prediction model</a></li>
<li class="chapter" data-level="5.5.10" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#analysis-of-prediction-performance"><i class="fa fa-check"></i><b>5.5.10</b> Analysis of prediction performance</a></li>
<li class="chapter" data-level="5.5.11" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#implementation-of-7c-and-compatibility-to-other-tools"><i class="fa fa-check"></i><b>5.5.11</b> Implementation of 7C and compatibility to other tools</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="declarations-2.html"><a href="declarations-2.html"><i class="fa fa-check"></i>Declarations</a><ul>
<li class="chapter" data-level="" data-path="declarations-2.html"><a href="declarations-2.html#availability-of-data-and-material-1"><i class="fa fa-check"></i>Availability of data and material</a></li>
<li class="chapter" data-level="" data-path="declarations-2.html"><a href="declarations-2.html#competing-interests-1"><i class="fa fa-check"></i>Competing interests</a></li>
<li class="chapter" data-level="" data-path="declarations-2.html"><a href="declarations-2.html#authors-contributions-1"><i class="fa fa-check"></i>Authors‘ contributions</a></li>
<li class="chapter" data-level="" data-path="declarations-2.html"><a href="declarations-2.html#acknowledgments-1"><i class="fa fa-check"></i>Acknowledgments</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="6-thesis-discussion.html"><a href="6-thesis-discussion.html"><i class="fa fa-check"></i><b>6</b> Discussion</a><ul>
<li class="chapter" data-level="6.1" data-path="6-1-co-regulation-of-functionally-related-genes-in-tads.html"><a href="6-1-co-regulation-of-functionally-related-genes-in-tads.html"><i class="fa fa-check"></i><b>6.1</b> Co-regulation of functionally related genes in TADs</a></li>
<li class="chapter" data-level="6.2" data-path="6-2-evolution-by-gene-duplications-and-altered-regulatory-environments.html"><a href="6-2-evolution-by-gene-duplications-and-altered-regulatory-environments.html"><i class="fa fa-check"></i><b>6.2</b> Evolution by gene duplications and altered regulatory environments</a></li>
<li class="chapter" data-level="6.3" data-path="6-3-tads-are-stable-across-large-evolutionary-time-scales.html"><a href="6-3-tads-are-stable-across-large-evolutionary-time-scales.html"><i class="fa fa-check"></i><b>6.3</b> TADs are stable across large evolutionary time-scales</a></li>
<li class="chapter" data-level="6.4" data-path="6-4-TAD-disease.html"><a href="6-4-TAD-disease.html"><i class="fa fa-check"></i><b>6.4</b> Gene expression changes by altered TADs in disease</a></li>
<li class="chapter" data-level="6.5" data-path="6-5-towards-predicting-regulatory-pathomechanisms-of-structural-variants.html"><a href="6-5-towards-predicting-regulatory-pathomechanisms-of-structural-variants.html"><i class="fa fa-check"></i><b>6.5</b> Towards predicting regulatory pathomechanisms of structural variants</a></li>
<li class="chapter" data-level="6.6" data-path="6-6-towards-studying-genome-folding-in-specific-conditions.html"><a href="6-6-towards-studying-genome-folding-in-specific-conditions.html"><i class="fa fa-check"></i><b>6.6</b> Towards studying genome folding in specific conditions</a><ul>
<li class="chapter" data-level="6.6.1" data-path="6-6-towards-studying-genome-folding-in-specific-conditions.html"><a href="6-6-towards-studying-genome-folding-in-specific-conditions.html#interpretation-of-non-coding-variants-by-interacting-genes"><i class="fa fa-check"></i><b>6.6.1</b> Interpretation of non-coding variants by interacting genes</a></li>
<li class="chapter" data-level="6.6.2" data-path="6-6-towards-studying-genome-folding-in-specific-conditions.html"><a href="6-6-towards-studying-genome-folding-in-specific-conditions.html#cell-type-specific-regulatory-interactions-between-enhancers-and-genes"><i class="fa fa-check"></i><b>6.6.2</b> Cell type-specific regulatory interactions between enhancers and genes</a></li>
<li class="chapter" data-level="6.6.3" data-path="6-6-towards-studying-genome-folding-in-specific-conditions.html"><a href="6-6-towards-studying-genome-folding-in-specific-conditions.html#variability-of-long-range-interactions-across-individuals"><i class="fa fa-check"></i><b>6.6.3</b> Variability of long-range interactions across individuals</a></li>
<li class="chapter" data-level="6.6.4" data-path="6-6-towards-studying-genome-folding-in-specific-conditions.html"><a href="6-6-towards-studying-genome-folding-in-specific-conditions.html#single-cell"><i class="fa fa-check"></i><b>6.6.4</b> Genome folding in single cells</a></li>
<li class="chapter" data-level="6.6.5" data-path="6-6-towards-studying-genome-folding-in-specific-conditions.html"><a href="6-6-towards-studying-genome-folding-in-specific-conditions.html#constantly-improving-targeted-experimental-methods."><i class="fa fa-check"></i><b>6.6.5</b> Constantly improving targeted experimental methods.</a></li>
</ul></li>
<li class="chapter" data-level="6.7" data-path="6-7-molecular-mechanisms-driving-genome-folding.html"><a href="6-7-molecular-mechanisms-driving-genome-folding.html"><i class="fa fa-check"></i><b>6.7</b> Molecular mechanisms driving genome folding</a></li>
<li class="chapter" data-level="6.8" data-path="6-8-conclusions.html"><a href="6-8-conclusions.html"><i class="fa fa-check"></i><b>6.8</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="appendix"><span><b>Appendix</b></span></li>
<li class="chapter" data-level="A" data-path="A-sup-paralog-regulation.html"><a href="A-sup-paralog-regulation.html"><i class="fa fa-check"></i><b>A</b> Supporting Information: Co-regulation of paralog genes</a></li>
<li class="chapter" data-level="B" data-path="B-sup-tad-evolution.html"><a href="B-sup-tad-evolution.html"><i class="fa fa-check"></i><b>B</b> Supplementary Data: Stability of TADs in evolution</a><ul>
<li class="chapter" data-level="B.1" data-path="B-1-TadEvoSupTab.html"><a href="B-1-TadEvoSupTab.html"><i class="fa fa-check"></i><b>B.1</b> Supplementary Tables</a></li>
<li class="chapter" data-level="B.2" data-path="B-2-supplementary-figures.html"><a href="B-2-supplementary-figures.html"><i class="fa fa-check"></i><b>B.2</b> Supplementary Figures</a></li>
</ul></li>
<li class="chapter" data-level="C" data-path="C-sup-position-effect.html"><a href="C-sup-position-effect.html"><i class="fa fa-check"></i><b>C</b> Supplemental Data: Position effects of rearrangements in disease genomes</a><ul>
<li class="chapter" data-level="C.1" data-path="C-1-supplemental-note.html"><a href="C-1-supplemental-note.html"><i class="fa fa-check"></i><b>C.1</b> Supplemental Note</a><ul>
<li class="chapter" data-level="C.1.1" data-path="C-1-supplemental-note.html"><a href="C-1-supplemental-note.html#case-reports"><i class="fa fa-check"></i><b>C.1.1</b> Case Reports</a></li>
<li class="chapter" data-level="C.1.2" data-path="C-1-supplemental-note.html"><a href="C-1-supplemental-note.html#nucleotide-level-nomenclature-for-dgap-karyotypes"><i class="fa fa-check"></i><b>C.1.2</b> Nucleotide Level Nomenclature for DGAP karyotypes</a></li>
</ul></li>
<li class="chapter" data-level="C.2" data-path="C-2-supplemental-figure.html"><a href="C-2-supplemental-figure.html"><i class="fa fa-check"></i><b>C.2</b> Supplemental Figure</a></li>
<li class="chapter" data-level="C.3" data-path="C-3-position-effect-sub-tab.html"><a href="C-3-position-effect-sub-tab.html"><i class="fa fa-check"></i><b>C.3</b> Supplemental Table Legends</a></li>
</ul></li>
<li class="chapter" data-level="D" data-path="D-loop-support.html"><a href="D-loop-support.html"><i class="fa fa-check"></i><b>D</b> Supplemental Information: Prediction of chromatin looping interactions</a><ul>
<li class="chapter" data-level="D.1" data-path="D-1-LoopPredSupTab.html"><a href="D-1-LoopPredSupTab.html"><i class="fa fa-check"></i><b>D.1</b> Supplementary Tables</a></li>
<li class="chapter" data-level="D.2" data-path="D-2-supplementary-figures-1.html"><a href="D-2-supplementary-figures-1.html"><i class="fa fa-check"></i><b>D.2</b> Supplementary Figures</a></li>
</ul></li>
<li class="chapter" data-level="E" data-path="E-contribution-to-individual-publications.html"><a href="E-contribution-to-individual-publications.html"><i class="fa fa-check"></i><b>E</b> Contribution to individual publications</a></li>
<li class="chapter" data-level="" data-path="zusammenfassung.html"><a href="zusammenfassung.html"><i class="fa fa-check"></i>Zusammenfassung</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Build with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Genome folding in evolution and disease</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="results-1" class="section level2">
<h2><span class="header-section-number">3.2</span> Results</h2>
<div id="identification-of-evolutionary-rearrangement-breakpoints-from-whole-genome-alignments" class="section level3">
<h3><span class="header-section-number">3.2.1</span> Identification of evolutionary rearrangement breakpoints from whole-genome alignments</h3>
<p>To analyze the stability of TADs in evolution, we first identified
evolutionary rearrangements by using whole-genome alignment data from
the UCSC Genome Browser <span class="citation">(Kent et al. <a href="references.html#ref-Kent2003">2003</a>, <a href="references.html#ref-Kent2002">2002</a>)</span> to compare the human genome to 12
other species. These species where selected to have genome assemblies of
good quality and to span several hundred million years of evolution.
They range from chimpanzee to zebrafish (Fig <a href="3-2-results-1.html#fig:TadEvo1">3.1</a>). The whole-genome
data consists of consecutive alignment blocks that are chained and
hierarchically ordered into so-called net files as fills <span class="citation">(Kent et al. <a href="references.html#ref-Kent2003">2003</a>)</span>. To
overcome alignment artifacts and smaller local variations between
genomes we only considered top-level fills or non-syntenic fills and
additionally applied a size threshold to use only fills that are larger
than 10 kb, 100 kb, or 1000 kb, respectively. Start and end coordinates
of such fills represent borders of syntenic regions and were extracted
as rearrangement breakpoints for further analysis (see Methods for
details).</p>
<!-- ---------------------------------------------------------------- -->
<div class="figure" style="text-align: center"><span id="fig:TadEvo1"></span>
<img src="figures/TAD_evolution/fig1_v02.png" alt="Number and size distributions of fill sizes of whole-genome alignments between human and 12 other species. (A) Number of syntenic alignment blocks (fills) between human (hg38) and 12 other species. Top-level fills are the largest and highest scoring chains and occur at the top level in the hierarchy in net files (top panel). Non-syn fills map to different chromosomes as their parent fills in the net files (bottom panel). (B) Size distribution of top-level (top panel) and non-syntenic (bottom panel) fills as violin plot. (C) Number of identified rearrangement breakpoints between human and 12 other species. Breakpoints are borders of top-level or non-syn fills that are larger or equal than a given size threshold (x-axis). (D) Phylogenetic tree with estimated divergence times according to http://timetree.org/." width="50%" />
<p class="caption">
Figure 3.1: <strong>Number and size distributions of fill sizes of whole-genome alignments between human and 12 other species. </strong> <strong>(A)</strong> Number of syntenic alignment blocks (fills) between human (hg38) and 12 other species. Top-level fills are the largest and highest scoring chains and occur at the top level in the hierarchy in net files (top panel). Non-syn fills map to different chromosomes as their parent fills in the net files (bottom panel). <strong>(B)</strong> Size distribution of top-level (top panel) and non-syntenic (bottom panel) fills as violin plot. <strong>(C)</strong> Number of identified rearrangement breakpoints between human and 12 other species. Breakpoints are borders of top-level or non-syn fills that are larger or equal than a given size threshold (x-axis). <strong>(D)</strong> Phylogenetic tree with estimated divergence times according to <a href="http://timetree.org/"><em>http://timetree.org/</em></a>.
</p>
</div>

<!-- ---------------------------------------------------------------- -->
<p>First, we analyzed the number and size distributions of top-level and
non-syntenic fills between human and other species (Fig <a href="3-2-results-1.html#fig:TadEvo1">3.1</a>). As
expected, closely related species such as chimpanzee and gorilla have in
general fewer fills but larger fill sizes (mean length ≥1 kb), whereas
species which are more distant to human, such as chicken and zebrafish,
tend to have more but smaller fills (mean length ≤ 1 kb, Fig <a href="3-2-results-1.html#fig:TadEvo1">3.1</a>A,B).
However, we also observe many small non-syntenic fills in closely
related species, likely arising from transposon insertions <span class="citation">(Mills et al. <a href="references.html#ref-Mills2006">2006</a>)</span>. As a
consequence of the number of fills and size distributions, we identify
different breakpoint numbers depending on species and size threshold
applied. For example, the whole-genome alignment between human and mouse
results in 2182, 655, and 302 rearrangement breakpoints for size
thresholds, 10 kb, 100 kb, and 1000 kb, respectively (Fig <a href="3-2-results-1.html#fig:TadEvo1">3.1</a>C).
Together, the number and size distributions of syntenic regions reflect
the evolutionary divergence time from human and allow us to identify
thousands of evolutionary rearrangement breakpoints for enrichment
analysis at TADs.</p>
</div>
<div id="rearrangement-breakpoints-are-enriched-at-tad-boundaries" class="section level3">
<h3><span class="header-section-number">3.2.2</span> Rearrangement breakpoints are enriched at TAD boundaries</h3>
<p>Next, we analyzed how the identified rearrangement breakpoints are
distributed in the human genome with respect to TADs. We obtained 3,062
TADs identified in human embryonic stem cells (hESC) <span class="citation">(Dixon et al. <a href="references.html#ref-Dixon2012">2012</a>)</span> and 9,274
contact domains from high-resolution <em>in situ</em> Hi-C in human
B-lymphoblastoid cells (GM12878) <span class="citation">(Rao et al. <a href="references.html#ref-Rao2014">2014</a>)</span>. To calculate the number of
breakpoints around TADs, we enlarged each TAD region by +/-50% of its
size and divided the region in 20 equal sized bins. For each bin we
computed the number of overlapping rearrangement breakpoints. This
results in a size-normalized distribution of rearrangement breakpoints
along TAD regions.</p>
<!-- ---------------------------------------------------------------- -->
<div class="figure" style="text-align: center"><span id="fig:TadEvo2"></span>
<img src="figures/TAD_evolution/fig2_v02.png" alt="Evolutionary rearrangements are enriched at TAD boundaries. (A) Distribution of evolutionary rearrangement breakpoints between human and mouse around hESC TADs. Each TAD and 50% of its adjacent sequence was subdivided into 20 bins of equal size, the breakpoints were assigned to the bins and their number summed up over the corresponding bins in all TADs. Blue color scale represents breakpoints from different fill-size thresholds. Dotted lines in gray show simulated background controls of randomly placed breakpoints. (B) Distribution of rearrangement breakpoints between human and: chimpanzee, cattle, opossum, and zebrafish, at 10 kb size threshold around hESC TADs. Dotted lines in gray show simulated background controls of randomly placed breakpoints. (C) Enrichment of breakpoints at TAD boundaries as log-odds-ratio between actual breakpoints at TAD boundaries and randomly placed breakpoints. Enrichment is shown for three different fill size thresholds (blue color scale) and TADs in hESC from (Dixon et al. 2012) (top) and contact domains in human GM12878 cells from (Rao et al. 2014) (bottom), respectively. Asterisks indicate significance of the enrichment using Fisher’s exact test (*p &lt;= 0.05; **p &lt;= 0.01; ***p &lt;= 0.001)." width="50%" />
<p class="caption">
Figure 3.2: <strong>Evolutionary rearrangements are enriched at TAD boundaries.</strong> <strong>(A)</strong> Distribution of evolutionary rearrangement breakpoints between human and mouse around hESC TADs. Each TAD and 50% of its adjacent sequence was subdivided into 20 bins of equal size, the breakpoints were assigned to the bins and their number summed up over the corresponding bins in all TADs. Blue color scale represents breakpoints from different fill-size thresholds. Dotted lines in gray show simulated background controls of randomly placed breakpoints. <strong>(B)</strong> Distribution of rearrangement breakpoints between human and: chimpanzee, cattle, opossum, and zebrafish, at 10 kb size threshold around hESC TADs. Dotted lines in gray show simulated background controls of randomly placed breakpoints. <strong>(C)</strong> Enrichment of breakpoints at TAD boundaries as log-odds-ratio between actual breakpoints at TAD boundaries and randomly placed breakpoints. Enrichment is shown for three different fill size thresholds (blue color scale) and TADs in hESC from <span class="citation">(Dixon et al. <a href="references.html#ref-Dixon2012">2012</a>)</span> (top) and contact domains in human GM12878 cells from <span class="citation">(Rao et al. <a href="references.html#ref-Rao2014">2014</a>)</span> (bottom), respectively. Asterisks indicate significance of the enrichment using Fisher’s exact test (*p &lt;= 0.05; **p &lt;= 0.01; ***p &lt;= 0.001).
</p>
</div>

<!-- ---------------------------------------------------------------- -->
<p>First, we analyzed the distribution of breakpoints at
different size thresholds between human and mouse at hESC TADs (Fig. <a href="3-2-results-1.html#fig:TadEvo2">3.2</a>A). Rearrangement breakpoints are clearly enriched at TAD boundaries
and depleted within TAD regions. Notably, this enrichment is observed
for all size thresholds applied in the identification of rearrangement
breakpoints. Next, we also analyzed the breakpoints from chimpanzee,
cattle, opossum, and zebrafish (Fig <a href="3-2-results-1.html#fig:TadEvo2">3.2</a>B) at the 10 kb size threshold.
Interestingly, we observed for all species a clear enrichment of
breakpoints at TAD boundaries and depletion within TAD regions. To
quantify this enrichment, we simulated an expected background
distribution of breakpoints by placing each breakpoint 100 times at a
random position of the respective chromosome. We than calculated the
fraction of observed and expected breakpoints that are closer than 40 kb
to a TAD boundary. For all size thresholds and analyzed species, we
computed the log-fold-ratio of actual breakpoints over random
breakpoints at domain boundaries (Fig <a href="3-2-results-1.html#fig:TadEvo2">3.2</a>C). For virtually all species
and size thresholds analyzed, we found breakpoints significantly
enriched at boundaries of TADs and contact domains (Fig <a href="3-2-results-1.html#fig:TadEvo2">3.2</a>C, <a href="B-2-supplementary-figures.html#fig:TadEvoS1">B.1</a>).
Depletion was only observed for some combinations of species and size
thresholds which have only very few breakpoints (see Fig <a href="3-2-results-1.html#fig:TadEvo1">3.1</a>C).
Furthermore, we compared the distance of each breakpoint to the closest
TAD boundary and observed nearly always significantly shorter distances
for actual breakpoints compared to random controls (Fig <a href="B-2-supplementary-figures.html#fig:TadEvoS2">B.2</a>). Overall,
the enrichment was stronger for TADs in hESC compared to the contact
domains in GM12878. However, these differences were likely due to
different sizes of TADs and contact domains and the nested structure of
contact domains, which overlap each other <span class="citation">(Rao et al. <a href="references.html#ref-Rao2014">2014</a>)</span>. Rearrangements between
human and both closely and distantly related species are highly enriched
at TAD boundaries and depleted within TADs. These results show (i) that
rearrangements are not randomly distributed in the genome, in agreement
with <span class="citation">(Farré et al. <a href="references.html#ref-Farre2015">2015</a>)</span>, and (ii) strong conservation of TAD regions over large
evolutionary time scales, indicating selective pressure against
disruption of TADs, presumably because of their functional role in gene
expression regulation.</p>
</div>
<div id="clusters-of-conserved-non-coding-elements-are-depleted-for-rearrangement-breakpoints" class="section level3">
<h3><span class="header-section-number">3.2.3</span> Clusters of conserved non-coding elements are depleted for rearrangement breakpoints</h3>
<p>Another interesting feature that can be extracted from whole-genome
alignments are highly conserved non-coding elements (CNEs) <span class="citation">(Polychronopoulos et al. <a href="references.html#ref-Polychronopoulos2017">2017</a>)</span>. CNEs
are defined as non-protein-coding sequences of at least 50 bp with over
70% sequence identity between distantly related species such as human
and chicken <span class="citation">(Polychronopoulos et al. <a href="references.html#ref-Polychronopoulos2017">2017</a>)</span>. In the human genome, CNEs cluster around
developmental genes in so-called genomic regulatory blocks (GRBs)
<span class="citation">(Kikuta et al. <a href="references.html#ref-Kikuta2007">2007</a>)</span>. It has been shown recently that many GRBs coincide with TADs in
human and <em>Drosophila</em> genomes <span class="citation">(Harmston et al. <a href="references.html#ref-Harmston2017">2017</a>)</span>. Therefore, we asked whether
evolutionary breakpoints are also enriched at boundaries of GRBs. This
would support the idea of a conserved regulatory environment around
important developmental genes. Indeed we saw a strong enrichment around
GRBs (Fig <a href="3-2-results-1.html#fig:TadEvo3">3.3</a>A). This is consistent with previous studies in <em>Drosophila</em>
and Fish where CNE arrays often correspond to syntenic blocks <span class="citation">(Engström et al. <a href="references.html#ref-Engstrom2007">2007</a>; Dimitrieva and Bucher <a href="references.html#ref-Dimitrieva2013">2013</a>)</span>.</p>
<!-- ---------------------------------------------------------------- -->
<div class="figure" style="text-align: center"><span id="fig:TadEvo3"></span>
<img src="figures/TAD_evolution/fig3_v02.png" alt="Rearrangement breakpoint distribution around GRBs and GRB-TADs. (A) Rearrangement breakpoints between mouse and human around 816 GRBs. (B) Breakpoint distribution around GRB-TADs and non-GRB-TADs. GRB-TADs are defined as TADs overlapping more than 80% with GRBs and non-GRB-TADs have less than 20% overlap with GRBs. Breakpoints using a 10 kb fill size threshold are shown." width="50%" />
<p class="caption">
Figure 3.3: <strong>Rearrangement breakpoint distribution around GRBs and GRB-TADs. </strong> <strong>(A)</strong> Rearrangement breakpoints between mouse and human around 816 GRBs. <strong>(B)</strong> Breakpoint distribution around GRB-TADs and non-GRB-TADs. GRB-TADs are defined as TADs overlapping more than 80% with GRBs and non-GRB-TADs have less than 20% overlap with GRBs. Breakpoints using a 10 kb fill size threshold are shown.
</p>
</div>

<!-- ---------------------------------------------------------------- -->
<p>Next, we subdivided TADs according to their overlap with GRBs in
GRB-TADs (&gt; 80% overlap) and non-GRB-TADs (&lt; 20% overlap) as in
the original study <span class="citation">(Harmston et al. <a href="references.html#ref-Harmston2017">2017</a>)</span>. As expected, we observed a higher
accumulation of breakpoints at boundaries and stronger depletion within
TADs for GRB-TADs compared to non-GRB-TADs (Fig <a href="3-2-results-1.html#fig:TadEvo3">3.3</a>B). However, also the
non-GRB-TADs, that have less than 20% overlap with GRBs, are enriched
for rearrangements at TAD boundaries. This indicates that not only TADs
overlapping GRBs are evolutionary conserved. In summary, we show that
human TADs overlapping clusters of non-coding conserved elements are
strongly depleted for rearrangements, likely due to strong selective
pressure on the conserved regulatory environment around important
developmental genes.</p>
</div>
<div id="rearranged-tads-are-associated-with-divergent-gene-expression-between-species" class="section level3">
<h3><span class="header-section-number">3.2.4</span> Rearranged TADs are associated with divergent gene expression between species</h3>
<p>The enrichment of rearrangement breakpoints at TAD boundaries indicates
that TADs are stable across large evolutionary time scales. However, the
reason for this strong conservation of TAD regions is unclear. A
mechanistic explanation could be that certain chromatin features at TAD
boundaries promote or prevent DNA double strand breaks (DSBs) <span class="citation">(Farré et al. <a href="references.html#ref-Farre2015">2015</a>; Canela et al. <a href="references.html#ref-Canela2017">2017</a>)</span>.
Alternatively, selective pressure might act against the disruption of
TADs due to their functional importance, for example in developmental
gene regulation <span class="citation">(Nora et al. <a href="references.html#ref-Nora2013">2013</a>; Farré et al. <a href="references.html#ref-Farre2015">2015</a>)</span>. TADs constitute a structural framework
determining possible interactions between promoters and cis-regulatory
sequences while prohibiting the influence of other sequences <span class="citation">(Symmons et al. <a href="references.html#ref-Symmons2014">2014</a>; Lupiáñez et al. <a href="references.html#ref-Lupianez2015">2015</a>)</span>.
TAD disruption would prevent formerly established contacts.
Rearrangements of TADs might also enable the recruitment of new
cis-regulatory sequences which would alter the expression patterns of
genes in rearranged TADs <span class="citation">(Lupiáñez et al. <a href="references.html#ref-Lupianez2015">2015</a>; Redin et al. <a href="references.html#ref-Redin2017">2017</a>)</span>. Because of these detrimental effects,
rearranged TADs should largely be eliminated by purifying selection.
However, rearrangement of TADs could also enable the expression of genes
in a new context and be selected if conferring an advantage. Therefore,
we hypothesized that genes within conserved TADs might have a more
stable gene expression pattern across tissues, whereas genes in
rearranged TADs between two species might have a more divergent
expression between species.</p>
<p>To test this, we analyzed the conservation of gene expression of
ortholog genes between human and mouse across 19 matched tissues from
the FANTOM5 project (<a href="B-1-TadEvoSupTab.html#TadEvoSupTab">Table S1</a>) <span class="citation">(Forrest et al. <a href="references.html#ref-Forrest2014">2014</a>)</span>. If a human gene and its mouse
ortholog have high correlation across matching tissues, they are likely
to have the same regulation and eventually similar functions.
Conversely, low correlation of expression across tissues can indicate
functional divergence during evolution, potentially due to altered gene
regulation.</p>
<p>First, we separated human genes according to their location within TADs
or outside of TADs. From 12,696 human genes with expression data and a
unique one-to-one ortholog in mouse (<a href="B-1-TadEvoSupTab.html#TadEvoSupTab">Table S2</a>), 1,525 have a
transcription start site (TSS) located outside hESC TADs and 11,171
within. Next, we computed for each gene its expression correlation with
mouse orthologs across 19 matching tissues. Genes within TADs have
significantly higher expression correlation with their mouse ortholog
(median R = 0,340) compared to genes outside TADs (mean R = 0,308, p =
0.0015, Fig <a href="3-2-results-1.html#fig:TadEvo4">3.4</a>A). This indicates higher conservation of gene regulation
in TADs and is consistent with the observation of housekeeping genes at
TAD boundaries <span class="citation">(Dixon et al. <a href="references.html#ref-Dixon2012">2012</a>)</span> and the role of TADs in providing conserved
regulatory environments for gene regulation <span class="citation">(Harmston et al. <a href="references.html#ref-Harmston2017">2017</a>; Ibn-Salem et al. <a href="references.html#ref-Ibn-Salem2017">2017</a>)</span>.</p>
<!-- ---------------------------------------------------------------- -->
<div class="figure" style="text-align: center"><span id="fig:TadEvo4"></span>
<img src="figures/TAD_evolution/fig4_v04.png" alt="Ortholog gene expression correlation across tissues in conserved and rearranged TADs. (A) Expression correlation of orthologs across 19 matching tissues in human and mouse for human genes within or outside of hESC TADs. (B) Expression correlation of orthologs across 19 matching tissues in human and mouse for genes in conserved or rearranged TADs. (C) Expression correlation of orthologs across 19 matching tissues in human and mouse for genes in GRB-TADs and non-GRB TADs. All P-values according to Wilcoxon rank-sum test." width="80%" />
<p class="caption">
Figure 3.4: <strong>Ortholog gene expression correlation across tissues in conserved and rearranged TADs.</strong> <strong>(A)</strong> Expression correlation of orthologs across 19 matching tissues in human and mouse for human genes within or outside of hESC TADs. <strong>(B)</strong> Expression correlation of orthologs across 19 matching tissues in human and mouse for genes in conserved or rearranged TADs. <strong>(C)</strong> Expression correlation of orthologs across 19 matching tissues in human and mouse for genes in GRB-TADs and non-GRB TADs. All P-values according to Wilcoxon rank-sum test.
</p>
</div>

<!-- ---------------------------------------------------------------- -->
<p>Next, we further subdivided TADs in two groups, rearranged and
conserved, according to syntenic blocks and rearrangements between human
and mouse genomes. In brief, a TAD is defined as conserved, if it is
completely enclosed by a syntenic alignment block and does not overlap
any rearrangement breakpoint. Conversely, a rearranged TAD is not
enclosed by a syntenic alignment block and overlaps at least one
breakpoint that is farther than 80 kb from its boundary (see Methods).
For the hESC TAD data set, this leads to 2,542 conserved and 137
rearranged TADs. The low number of rearranged TADs is consistent with
the depletion of rearrangement breakpoints within TADs in general (Fig.
2). In total 8,740 genes in conserved and 645 genes in rearranged TADs
could be assigned to a one-to-one ortholog in mouse and are contained in
the expression data set. The expression correlation with mouse orthologs
were significantly higher for genes in conserved TADs (median R = 0.316)
compared to genes in rearranged TADs (median R = 0.237, p = 0.0013)
(Fig <a href="3-2-results-1.html#fig:TadEvo4">3.4</a>B). This shows that disruptions of TADs by evolutionary
rearrangements are associated with less conserved gene expression
profiles across tissues. Although not significant, we also observed a
slightly higher expression correlation for 1,003 genes in GRB-TADs
compared to 8,038 genes in non-GRB TADs (Fig <a href="3-2-results-1.html#fig:TadEvo4">3.4</a>C, p = 0.13).</p>
<p>In summary, we observed higher expression correlation between orthologs<br />
for human genes inside TADs than outside. Moreover, we saw that genes in
rearranged TADs show lower gene expression conservation than those in
conserved TADs. These results not only support a functional role of TADs
in gene regulation, but further support the hypothesis that TAD regions
are subjected to purifying selection against their disruption by
structural variations such as rearrangements.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="3-1-introduction-1.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="3-3-discussion.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": true,
"facebook": false,
"twitter": true,
"google": false,
"weibo": false,
"instapper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/ibn-salem/thesis/edit/master/03-TAD_evolution.Rmd",
"text": "Edit"
},
"download": ["thesis.pdf", "thesis.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://cdn.bootcss.com/mathjax/2.7.1/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:" && /^https?:/.test(src))
      src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
