<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>Genome folding in evolution and disease</title>
  <meta name="description" content="A PhD thesis on genome folding in evoultion and disease.">
  <meta name="generator" content="bookdown 0.6.1 and GitBook 2.6.7">

  <meta property="og:title" content="Genome folding in evolution and disease" />
  <meta property="og:type" content="book" />
  <meta property="og:url" content="https://ibn-salem.github.io/thesis/" />
  
  <meta property="og:description" content="A PhD thesis on genome folding in evoultion and disease." />
  <meta name="github-repo" content="ibn-salem/thesis" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Genome folding in evolution and disease" />
  
  <meta name="twitter:description" content="A PhD thesis on genome folding in evoultion and disease." />
  

<meta name="author" content="Jonas Ibn-Salem">


<meta name="date" content="2018-08-19">

  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  
<link rel="prev" href="4-1-introduction-2.html">
<link rel="next" href="4-3-results-2.html">
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />










<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Thesis</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="1" data-path="1-intro.html"><a href="1-intro.html"><i class="fa fa-check"></i><b>1</b> Introduction</a><ul>
<li class="chapter" data-level="1.1" data-path="1-1-regulation-of-expression.html"><a href="1-1-regulation-of-expression.html"><i class="fa fa-check"></i><b>1.1</b> Regulation of gene expression</a></li>
<li class="chapter" data-level="1.2" data-path="1-2-distal-regulation-by-enhancers.html"><a href="1-2-distal-regulation-by-enhancers.html"><i class="fa fa-check"></i><b>1.2</b> Distal regulation by enhancers</a></li>
<li class="chapter" data-level="1.3" data-path="1-3-methods-to-probe-the-3d-chromatin-architecture.html"><a href="1-3-methods-to-probe-the-3d-chromatin-architecture.html"><i class="fa fa-check"></i><b>1.3</b> Methods to probe the 3D chromatin architecture</a><ul>
<li class="chapter" data-level="1.3.1" data-path="1-3-methods-to-probe-the-3d-chromatin-architecture.html"><a href="1-3-methods-to-probe-the-3d-chromatin-architecture.html#microscopy-based-techniques-to-visualize-the-genome-in-3d"><i class="fa fa-check"></i><b>1.3.1</b> Microscopy-based techniques to visualize the genome in 3D</a></li>
<li class="chapter" data-level="1.3.2" data-path="1-3-methods-to-probe-the-3d-chromatin-architecture.html"><a href="1-3-methods-to-probe-the-3d-chromatin-architecture.html#proximity-ligation"><i class="fa fa-check"></i><b>1.3.2</b> Proximity-ligation based method to quantify chromatin interactions</a></li>
</ul></li>
<li class="chapter" data-level="1.4" data-path="1-4-hierarchy-of-chromatin-3d-structure.html"><a href="1-4-hierarchy-of-chromatin-3d-structure.html"><i class="fa fa-check"></i><b>1.4</b> Hierarchy of chromatin 3D structure</a><ul>
<li class="chapter" data-level="1.4.1" data-path="1-4-hierarchy-of-chromatin-3d-structure.html"><a href="1-4-hierarchy-of-chromatin-3d-structure.html#chromosomal-territories-and-inter-chromosomal-contacts"><i class="fa fa-check"></i><b>1.4.1</b> Chromosomal territories and inter-chromosomal contacts</a></li>
<li class="chapter" data-level="1.4.2" data-path="1-4-hierarchy-of-chromatin-3d-structure.html"><a href="1-4-hierarchy-of-chromatin-3d-structure.html#ab-compartments"><i class="fa fa-check"></i><b>1.4.2</b> A/B compartments</a></li>
<li class="chapter" data-level="1.4.3" data-path="1-4-hierarchy-of-chromatin-3d-structure.html"><a href="1-4-hierarchy-of-chromatin-3d-structure.html#topologically-associating-domains-tads"><i class="fa fa-check"></i><b>1.4.3</b> Topologically associating domains (TADs)</a></li>
<li class="chapter" data-level="1.4.4" data-path="1-4-hierarchy-of-chromatin-3d-structure.html"><a href="1-4-hierarchy-of-chromatin-3d-structure.html#hierarchy-of-domain-structures-across-genomic-length-scales"><i class="fa fa-check"></i><b>1.4.4</b> Hierarchy of domain structures across genomic length scales</a></li>
<li class="chapter" data-level="1.4.5" data-path="1-4-hierarchy-of-chromatin-3d-structure.html"><a href="1-4-hierarchy-of-chromatin-3d-structure.html#chromatin-looping-interactions"><i class="fa fa-check"></i><b>1.4.5</b> Chromatin looping interactions</a></li>
<li class="chapter" data-level="1.4.6" data-path="1-4-hierarchy-of-chromatin-3d-structure.html"><a href="1-4-hierarchy-of-chromatin-3d-structure.html#loop-mechanism"><i class="fa fa-check"></i><b>1.4.6</b> TAD and loop formation by architectural proteins</a></li>
</ul></li>
<li class="chapter" data-level="1.5" data-path="1-5-dynamics-of-chromatin-structure.html"><a href="1-5-dynamics-of-chromatin-structure.html"><i class="fa fa-check"></i><b>1.5</b> Dynamics of chromatin structure</a><ul>
<li class="chapter" data-level="1.5.1" data-path="1-5-dynamics-of-chromatin-structure.html"><a href="1-5-dynamics-of-chromatin-structure.html#dynamics-across-the-cell-cycle"><i class="fa fa-check"></i><b>1.5.1</b> Dynamics across the cell cycle</a></li>
<li class="chapter" data-level="1.5.2" data-path="1-5-dynamics-of-chromatin-structure.html"><a href="1-5-dynamics-of-chromatin-structure.html#dynamics-across-cell-types-and-differntiation"><i class="fa fa-check"></i><b>1.5.2</b> Dynamics across cell types and differntiation</a></li>
</ul></li>
<li class="chapter" data-level="1.6" data-path="1-6-evolution-of-chromatin-organization.html"><a href="1-6-evolution-of-chromatin-organization.html"><i class="fa fa-check"></i><b>1.6</b> Evolution of chromatin organization</a></li>
<li class="chapter" data-level="1.7" data-path="1-7-disruption-of-chromatin-architecture-in-disease.html"><a href="1-7-disruption-of-chromatin-architecture-in-disease.html"><i class="fa fa-check"></i><b>1.7</b> Disruption of chromatin architecture in disease</a></li>
<li class="chapter" data-level="1.8" data-path="1-8-aims-of-this-thesis.html"><a href="1-8-aims-of-this-thesis.html"><i class="fa fa-check"></i><b>1.8</b> Aims of this thesis</a></li>
<li class="chapter" data-level="1.9" data-path="1-9-structure-of-this-thesis.html"><a href="1-9-structure-of-this-thesis.html"><i class="fa fa-check"></i><b>1.9</b> Structure of this thesis</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="2-paralog-regulation.html"><a href="2-paralog-regulation.html"><i class="fa fa-check"></i><b>2</b> Paralog genes in the 3D genome architecture</a><ul>
<li class="chapter" data-level="" data-path="2-paralog-regulation.html"><a href="2-paralog-regulation.html#preamble"><i class="fa fa-check"></i>Preamble</a></li>
<li class="chapter" data-level="" data-path="2-paralog-regulation.html"><a href="2-paralog-regulation.html#abstract-1"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="2.1" data-path="2-1-introduction.html"><a href="2-1-introduction.html"><i class="fa fa-check"></i><b>2.1</b> Introduction</a></li>
<li class="chapter" data-level="2.2" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html"><i class="fa fa-check"></i><b>2.2</b> Materials and methods</a><ul>
<li class="chapter" data-level="2.2.1" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html#selection-of-pairs-of-paralog-genes"><i class="fa fa-check"></i><b>2.2.1</b> Selection of pairs of paralog genes</a></li>
<li class="chapter" data-level="2.2.2" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html#enhancers-to-gene-association"><i class="fa fa-check"></i><b>2.2.2</b> Enhancers to gene association</a></li>
<li class="chapter" data-level="2.2.3" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html#topological-associating-domains"><i class="fa fa-check"></i><b>2.2.3</b> Topological associating domains</a></li>
<li class="chapter" data-level="2.2.4" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html#hi-c-interaction-maps"><i class="fa fa-check"></i><b>2.2.4</b> Hi-C interaction maps</a></li>
<li class="chapter" data-level="2.2.5" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html#randomization"><i class="fa fa-check"></i><b>2.2.5</b> Randomization</a></li>
<li class="chapter" data-level="2.2.6" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html#statistical-tests"><i class="fa fa-check"></i><b>2.2.6</b> Statistical tests</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="2-3-results.html"><a href="2-3-results.html"><i class="fa fa-check"></i><b>2.3</b> Results</a><ul>
<li class="chapter" data-level="2.3.1" data-path="2-3-results.html"><a href="2-3-results.html#distribution-of-paralog-genes-in-the-human-genome"><i class="fa fa-check"></i><b>2.3.1</b> Distribution of paralog genes in the human genome</a></li>
<li class="chapter" data-level="2.3.2" data-path="2-3-results.html"><a href="2-3-results.html#co-expression-of-paralog-gene-pairs-across-tissues"><i class="fa fa-check"></i><b>2.3.2</b> Co-expression of paralog gene pairs across tissues</a></li>
<li class="chapter" data-level="2.3.3" data-path="2-3-results.html"><a href="2-3-results.html#paralog-genes-share-enhancers"><i class="fa fa-check"></i><b>2.3.3</b> Paralog genes share enhancers</a></li>
<li class="chapter" data-level="2.3.4" data-path="2-3-results.html"><a href="2-3-results.html#co-localization-of-paralogs-in-tads"><i class="fa fa-check"></i><b>2.3.4</b> Co-localization of paralogs in TADs</a></li>
<li class="chapter" data-level="2.3.5" data-path="2-3-results.html"><a href="2-3-results.html#distal-paralog-pairs-are-enriched-for-long-range-chromatin-contacts"><i class="fa fa-check"></i><b>2.3.5</b> Distal paralog pairs are enriched for long-range chromatin contacts</a></li>
<li class="chapter" data-level="2.3.6" data-path="2-3-results.html"><a href="2-3-results.html#close-paralogs-have-fewer-contacts-than-expected"><i class="fa fa-check"></i><b>2.3.6</b> Close paralogs have fewer contacts than expected</a></li>
<li class="chapter" data-level="2.3.7" data-path="2-3-results.html"><a href="2-3-results.html#paralogs-in-mouse-and-dog-genome"><i class="fa fa-check"></i><b>2.3.7</b> Paralogs in mouse and dog genome</a></li>
<li class="chapter" data-level="2.3.8" data-path="2-3-results.html"><a href="2-3-results.html#orthologs-of-human-paralogs-show-conserved-co-localization"><i class="fa fa-check"></i><b>2.3.8</b> Orthologs of human paralogs show conserved co-localization</a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="2-4-discussions.html"><a href="2-4-discussions.html"><i class="fa fa-check"></i><b>2.4</b> Discussions</a></li>
<li class="chapter" data-level="2.5" data-path="2-5-conclusion.html"><a href="2-5-conclusion.html"><i class="fa fa-check"></i><b>2.5</b> Conclusion</a></li>
<li class="chapter" data-level="" data-path="acknowledgements.html"><a href="acknowledgements.html"><i class="fa fa-check"></i>Acknowledgements</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="3-TAD-evolution.html"><a href="3-TAD-evolution.html"><i class="fa fa-check"></i><b>3</b> Stability of TADs in evolution</a><ul>
<li class="chapter" data-level="" data-path="3-TAD-evolution.html"><a href="3-TAD-evolution.html#preamble-1"><i class="fa fa-check"></i>Preamble</a></li>
<li class="chapter" data-level="" data-path="3-TAD-evolution.html"><a href="3-TAD-evolution.html#abstract-2"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="3.1" data-path="3-1-introduction-1.html"><a href="3-1-introduction-1.html"><i class="fa fa-check"></i><b>3.1</b> Introduction</a></li>
<li class="chapter" data-level="3.2" data-path="3-2-results-1.html"><a href="3-2-results-1.html"><i class="fa fa-check"></i><b>3.2</b> Results</a><ul>
<li class="chapter" data-level="3.2.1" data-path="3-2-results-1.html"><a href="3-2-results-1.html#identification-of-evolutionary-rearrangement-breakpoints-from-whole-genome-alignments"><i class="fa fa-check"></i><b>3.2.1</b> Identification of evolutionary rearrangement breakpoints from whole-genome alignments</a></li>
<li class="chapter" data-level="3.2.2" data-path="3-2-results-1.html"><a href="3-2-results-1.html#rearrangement-breakpoints-are-enriched-at-tad-boundaries"><i class="fa fa-check"></i><b>3.2.2</b> Rearrangement breakpoints are enriched at TAD boundaries</a></li>
<li class="chapter" data-level="3.2.3" data-path="3-2-results-1.html"><a href="3-2-results-1.html#clusters-of-conserved-non-coding-elements-are-depleted-for-rearrangement-breakpoints"><i class="fa fa-check"></i><b>3.2.3</b> Clusters of conserved non-coding elements are depleted for rearrangement breakpoints</a></li>
<li class="chapter" data-level="3.2.4" data-path="3-2-results-1.html"><a href="3-2-results-1.html#rearranged-tads-are-associated-with-divergent-gene-expression-between-species"><i class="fa fa-check"></i><b>3.2.4</b> Rearranged TADs are associated with divergent gene expression between species</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="3-3-discussion.html"><a href="3-3-discussion.html"><i class="fa fa-check"></i><b>3.3</b> Discussion</a></li>
<li class="chapter" data-level="3.4" data-path="3-4-conclusion-1.html"><a href="3-4-conclusion-1.html"><i class="fa fa-check"></i><b>3.4</b> Conclusion</a></li>
<li class="chapter" data-level="3.5" data-path="3-5-methods.html"><a href="3-5-methods.html"><i class="fa fa-check"></i><b>3.5</b> Methods</a><ul>
<li class="chapter" data-level="3.5.1" data-path="3-5-methods.html"><a href="3-5-methods.html#rearrangement-breakpoints-from-whole-genome-alignments"><i class="fa fa-check"></i><b>3.5.1</b> Rearrangement breakpoints from whole-genome alignments</a></li>
<li class="chapter" data-level="3.5.2" data-path="3-5-methods.html"><a href="3-5-methods.html#topologically-associating-domains-and-contact-domains"><i class="fa fa-check"></i><b>3.5.2</b> Topologically associating domains and contact domains</a></li>
<li class="chapter" data-level="3.5.3" data-path="3-5-methods.html"><a href="3-5-methods.html#breakpoint-distributions-at-tads"><i class="fa fa-check"></i><b>3.5.3</b> Breakpoint distributions at TADs</a></li>
<li class="chapter" data-level="3.5.4" data-path="3-5-methods.html"><a href="3-5-methods.html#quantification-of-breakpoint-enrichment"><i class="fa fa-check"></i><b>3.5.4</b> Quantification of breakpoint enrichment</a></li>
<li class="chapter" data-level="3.5.5" data-path="3-5-methods.html"><a href="3-5-methods.html#expression-data-for-mouse-and-human-orthologs"><i class="fa fa-check"></i><b>3.5.5</b> Expression data for mouse and human orthologs</a></li>
<li class="chapter" data-level="3.5.6" data-path="3-5-methods.html"><a href="3-5-methods.html#classification-of-tads-and-genes-according-to-rearrangements-and-grbs"><i class="fa fa-check"></i><b>3.5.6</b> Classification of TADs and genes according to rearrangements and GRBs</a></li>
<li class="chapter" data-level="3.5.7" data-path="3-5-methods.html"><a href="3-5-methods.html#source-code-and-implementation-details"><i class="fa fa-check"></i><b>3.5.7</b> Source code and implementation details</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="declarations.html"><a href="declarations.html"><i class="fa fa-check"></i>Declarations</a><ul>
<li class="chapter" data-level="" data-path="declarations.html"><a href="declarations.html#availability-of-data-and-material"><i class="fa fa-check"></i>Availability of data and material</a></li>
<li class="chapter" data-level="" data-path="declarations.html"><a href="declarations.html#competing-interests"><i class="fa fa-check"></i>Competing interests</a></li>
<li class="chapter" data-level="" data-path="declarations.html"><a href="declarations.html#authors-contributions"><i class="fa fa-check"></i>Authors‘ contributions</a></li>
<li class="chapter" data-level="" data-path="declarations.html"><a href="declarations.html#acknowledgments"><i class="fa fa-check"></i>Acknowledgments</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="4-position-effect.html"><a href="4-position-effect.html"><i class="fa fa-check"></i><b>4</b> Position effects of rearrangements in disease genomes</a><ul>
<li class="chapter" data-level="" data-path="4-position-effect.html"><a href="4-position-effect.html#preamble-2"><i class="fa fa-check"></i>Preamble</a></li>
<li class="chapter" data-level="" data-path="4-position-effect.html"><a href="4-position-effect.html#abstract-3"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="4.1" data-path="4-1-introduction-2.html"><a href="4-1-introduction-2.html"><i class="fa fa-check"></i><b>4.1</b> Introduction</a></li>
<li class="chapter" data-level="4.2" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html"><i class="fa fa-check"></i><b>4.2</b> Materials and Methods</a><ul>
<li class="chapter" data-level="4.2.1" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html#selection-of-subjects-with-apparently-balanced-chromosome-abnormalities"><i class="fa fa-check"></i><b>4.2.1</b> Selection of subjects with apparently balanced chromosome abnormalities</a></li>
<li class="chapter" data-level="4.2.2" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html#clinical-descriptions-of-dgap-cases"><i class="fa fa-check"></i><b>4.2.2</b> Clinical descriptions of DGAP cases</a></li>
<li class="chapter" data-level="4.2.3" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html#analysis-of-genes-bordering-the-rearrangement-breakpoints"><i class="fa fa-check"></i><b>4.2.3</b> Analysis of genes bordering the rearrangement breakpoints</a></li>
<li class="chapter" data-level="4.2.4" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html#assessment-of-disrupted-functional-elements-and-chromatin-interactions-bordering-rearrangement-breakpoints"><i class="fa fa-check"></i><b>4.2.4</b> Assessment of disrupted functional elements and chromatin interactions bordering rearrangement breakpoints</a></li>
<li class="chapter" data-level="4.2.5" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html#ontological-analysis-of-genes-neighboring-breakpoints"><i class="fa fa-check"></i><b>4.2.5</b> Ontological analysis of genes neighboring breakpoints</a></li>
<li class="chapter" data-level="4.2.6" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html#quantitative-real-time-pcr"><i class="fa fa-check"></i><b>4.2.6</b> Quantitative real-time PCR</a></li>
<li class="chapter" data-level="4.2.7" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html#assessment-of-dgap-breakpoints-overlapping-with-non-coding-structural-variants-in-public-databases"><i class="fa fa-check"></i><b>4.2.7</b> Assessment of DGAP breakpoints overlapping with non-coding structural variants in public databases</a></li>
</ul></li>
<li class="chapter" data-level="4.3" data-path="4-3-results-2.html"><a href="4-3-results-2.html"><i class="fa fa-check"></i><b>4.3</b> Results</a><ul>
<li class="chapter" data-level="4.3.1" data-path="4-3-results-2.html"><a href="4-3-results-2.html#genomic-characterization-of-non-coding-breakpoints"><i class="fa fa-check"></i><b>4.3.1</b> Genomic characterization of non-coding breakpoints</a></li>
<li class="chapter" data-level="4.3.2" data-path="4-3-results-2.html"><a href="4-3-results-2.html#identification-of-genes-with-potential-position-effects"><i class="fa fa-check"></i><b>4.3.2</b> Identification of genes with potential position effects</a></li>
<li class="chapter" data-level="4.3.3" data-path="4-3-results-2.html"><a href="4-3-results-2.html#identification-of-subjects-with-shared-non-coding-chromosome-alterations-and-phenotypes"><i class="fa fa-check"></i><b>4.3.3</b> Identification of subjects with shared non-coding chromosome alterations and phenotypes</a></li>
</ul></li>
<li class="chapter" data-level="4.4" data-path="4-4-discussion-1.html"><a href="4-4-discussion-1.html"><i class="fa fa-check"></i><b>4.4</b> Discussion</a></li>
<li class="chapter" data-level="" data-path="declarations-1.html"><a href="declarations-1.html"><i class="fa fa-check"></i>Declarations</a><ul>
<li class="chapter" data-level="" data-path="declarations-1.html"><a href="declarations-1.html#acknowledgements-1"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="" data-path="declarations-1.html"><a href="declarations-1.html#web-resources"><i class="fa fa-check"></i>Web Resources</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="5-loop.html"><a href="5-loop.html"><i class="fa fa-check"></i><b>5</b> Prediction of chromatin looping interactions</a><ul>
<li class="chapter" data-level="" data-path="5-loop.html"><a href="5-loop.html#preamble-3"><i class="fa fa-check"></i>Preamble</a></li>
<li class="chapter" data-level="" data-path="5-loop.html"><a href="5-loop.html#abstract-4"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="5.1" data-path="5-1-introduction-3.html"><a href="5-1-introduction-3.html"><i class="fa fa-check"></i><b>5.1</b> Introduction</a></li>
<li class="chapter" data-level="5.2" data-path="5-2-results-3.html"><a href="5-2-results-3.html"><i class="fa fa-check"></i><b>5.2</b> Results</a><ul>
<li class="chapter" data-level="5.2.1" data-path="5-2-results-3.html"><a href="5-2-results-3.html#ctcf-motif-pairs-as-candidate-chromatin-loop-anchors"><i class="fa fa-check"></i><b>5.2.1</b> CTCF motif pairs as candidate chromatin loop anchors</a></li>
<li class="chapter" data-level="5.2.2" data-path="5-2-results-3.html"><a href="5-2-results-3.html#similarity-of-chip-seq-signals-at-looping-ctcf-motifs"><i class="fa fa-check"></i><b>5.2.2</b> Similarity of ChIP-seq signals at looping CTCF motifs</a></li>
<li class="chapter" data-level="5.2.3" data-path="5-2-results-3.html"><a href="5-2-results-3.html#genomic-sequence-features-of-ctcf-motif-pairs-are-associated-with-looping"><i class="fa fa-check"></i><b>5.2.3</b> Genomic sequence features of CTCF motif pairs are associated with looping</a></li>
<li class="chapter" data-level="5.2.4" data-path="5-2-results-3.html"><a href="5-2-results-3.html#chromatin-loop-prediction-using-7c"><i class="fa fa-check"></i><b>5.2.4</b> Chromatin loop prediction using 7C</a></li>
<li class="chapter" data-level="5.2.5" data-path="5-2-results-3.html"><a href="5-2-results-3.html#prediction-performance-evaluation"><i class="fa fa-check"></i><b>5.2.5</b> Prediction performance evaluation</a></li>
<li class="chapter" data-level="5.2.6" data-path="5-2-results-3.html"><a href="5-2-results-3.html#prediction-performance-of-7c-with-sequence-features-and-single-tf-chip-seq-data-sets"><i class="fa fa-check"></i><b>5.2.6</b> Prediction performance of 7C with sequence features and single TF ChIP-seq data sets</a></li>
<li class="chapter" data-level="5.2.7" data-path="5-2-results-3.html"><a href="5-2-results-3.html#comparison-of-transcription-factors-by-prediction-performance"><i class="fa fa-check"></i><b>5.2.7</b> Comparison of transcription factors by prediction performance</a></li>
<li class="chapter" data-level="5.2.8" data-path="5-2-results-3.html"><a href="5-2-results-3.html#prediction-performance-in-other-cell-types-and-for-different-tfs"><i class="fa fa-check"></i><b>5.2.8</b> Prediction performance in other cell types and for different TFs</a></li>
<li class="chapter" data-level="5.2.9" data-path="5-2-results-3.html"><a href="5-2-results-3.html#the-high-resolution-of-chip-nexus-improves-prediction-performance"><i class="fa fa-check"></i><b>5.2.9</b> The high resolution of ChIP-nexus improves prediction performance</a></li>
</ul></li>
<li class="chapter" data-level="5.3" data-path="5-3-discussion-2.html"><a href="5-3-discussion-2.html"><i class="fa fa-check"></i><b>5.3</b> Discussion</a></li>
<li class="chapter" data-level="5.4" data-path="5-4-conclusion-2.html"><a href="5-4-conclusion-2.html"><i class="fa fa-check"></i><b>5.4</b> Conclusion</a></li>
<li class="chapter" data-level="5.5" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html"><i class="fa fa-check"></i><b>5.5</b> Methods</a><ul>
<li class="chapter" data-level="5.5.1" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#ctcf-motifs-in-the-human-genome"><i class="fa fa-check"></i><b>5.5.1</b> CTCF motifs in the human genome</a></li>
<li class="chapter" data-level="5.5.2" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#loop-interaction-data-for-training-and-validation"><i class="fa fa-check"></i><b>5.5.2</b> Loop interaction data for training and validation</a></li>
<li class="chapter" data-level="5.5.3" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#chip-seq-datasets-in-gm12878-cells"><i class="fa fa-check"></i><b>5.5.3</b> ChIP-seq datasets in GM12878 cells</a></li>
<li class="chapter" data-level="5.5.4" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#chip-seq-data-types"><i class="fa fa-check"></i><b>5.5.4</b> ChIP-seq data types</a></li>
<li class="chapter" data-level="5.5.5" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#chip-nexus-data-processing-for-rad21-and-scm3"><i class="fa fa-check"></i><b>5.5.5</b> ChIP-nexus data processing for RAD21 and SCM3</a></li>
<li class="chapter" data-level="5.5.6" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#similarity-of-chip-seq-profiles-as-correlation-of-coverage-around-motifs"><i class="fa fa-check"></i><b>5.5.6</b> Similarity of ChIP-seq profiles as correlation of coverage around motifs</a></li>
<li class="chapter" data-level="5.5.7" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#genomic-sequence-features-of-chromatin-loops"><i class="fa fa-check"></i><b>5.5.7</b> Genomic sequence features of chromatin loops</a></li>
<li class="chapter" data-level="5.5.8" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#chromatin-loop-prediction-model"><i class="fa fa-check"></i><b>5.5.8</b> Chromatin Loop prediction model</a></li>
<li class="chapter" data-level="5.5.9" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#training-and-validation-of-prediction-model"><i class="fa fa-check"></i><b>5.5.9</b> Training and validation of prediction model</a></li>
<li class="chapter" data-level="5.5.10" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#analysis-of-prediction-performance"><i class="fa fa-check"></i><b>5.5.10</b> Analysis of prediction performance</a></li>
<li class="chapter" data-level="5.5.11" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#implementation-of-7c-and-compatibility-to-other-tools"><i class="fa fa-check"></i><b>5.5.11</b> Implementation of 7C and compatibility to other tools</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="declarations-2.html"><a href="declarations-2.html"><i class="fa fa-check"></i>Declarations</a><ul>
<li class="chapter" data-level="" data-path="declarations-2.html"><a href="declarations-2.html#availability-of-data-and-material-1"><i class="fa fa-check"></i>Availability of data and material</a></li>
<li class="chapter" data-level="" data-path="declarations-2.html"><a href="declarations-2.html#competing-interests-1"><i class="fa fa-check"></i>Competing interests</a></li>
<li class="chapter" data-level="" data-path="declarations-2.html"><a href="declarations-2.html#authors-contributions-1"><i class="fa fa-check"></i>Authors‘ contributions</a></li>
<li class="chapter" data-level="" data-path="declarations-2.html"><a href="declarations-2.html#acknowledgments-1"><i class="fa fa-check"></i>Acknowledgments</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="6-thesis-discussion.html"><a href="6-thesis-discussion.html"><i class="fa fa-check"></i><b>6</b> Discussion</a><ul>
<li class="chapter" data-level="6.1" data-path="6-1-co-regulation-of-functionally-related-genes-in-tads.html"><a href="6-1-co-regulation-of-functionally-related-genes-in-tads.html"><i class="fa fa-check"></i><b>6.1</b> Co-regulation of functionally related genes in TADs</a></li>
<li class="chapter" data-level="6.2" data-path="6-2-evolution-by-gene-duplications-and-altered-regulatory-environments.html"><a href="6-2-evolution-by-gene-duplications-and-altered-regulatory-environments.html"><i class="fa fa-check"></i><b>6.2</b> Evolution by gene duplications and altered regulatory environments</a></li>
<li class="chapter" data-level="6.3" data-path="6-3-tads-are-stable-across-large-evolutionary-time-scales.html"><a href="6-3-tads-are-stable-across-large-evolutionary-time-scales.html"><i class="fa fa-check"></i><b>6.3</b> TADs are stable across large evolutionary time-scales</a></li>
<li class="chapter" data-level="6.4" data-path="6-4-TAD-disease.html"><a href="6-4-TAD-disease.html"><i class="fa fa-check"></i><b>6.4</b> Gene expression changes by altered TADs in disease</a></li>
<li class="chapter" data-level="6.5" data-path="6-5-towards-predicting-regulatory-pathomechanisms-of-structural-variants.html"><a href="6-5-towards-predicting-regulatory-pathomechanisms-of-structural-variants.html"><i class="fa fa-check"></i><b>6.5</b> Towards predicting regulatory pathomechanisms of structural variants</a></li>
<li class="chapter" data-level="6.6" data-path="6-6-towards-studying-genome-folding-in-specific-conditions.html"><a href="6-6-towards-studying-genome-folding-in-specific-conditions.html"><i class="fa fa-check"></i><b>6.6</b> Towards studying genome folding in specific conditions</a><ul>
<li class="chapter" data-level="6.6.1" data-path="6-6-towards-studying-genome-folding-in-specific-conditions.html"><a href="6-6-towards-studying-genome-folding-in-specific-conditions.html#interpretation-of-non-coding-variants-by-interacting-genes"><i class="fa fa-check"></i><b>6.6.1</b> Interpretation of non-coding variants by interacting genes</a></li>
<li class="chapter" data-level="6.6.2" data-path="6-6-towards-studying-genome-folding-in-specific-conditions.html"><a href="6-6-towards-studying-genome-folding-in-specific-conditions.html#cell-type-specific-regulatory-interactions-between-enhancers-and-genes"><i class="fa fa-check"></i><b>6.6.2</b> Cell type-specific regulatory interactions between enhancers and genes</a></li>
<li class="chapter" data-level="6.6.3" data-path="6-6-towards-studying-genome-folding-in-specific-conditions.html"><a href="6-6-towards-studying-genome-folding-in-specific-conditions.html#variability-of-long-range-interactions-across-individuals"><i class="fa fa-check"></i><b>6.6.3</b> Variability of long-range interactions across individuals</a></li>
<li class="chapter" data-level="6.6.4" data-path="6-6-towards-studying-genome-folding-in-specific-conditions.html"><a href="6-6-towards-studying-genome-folding-in-specific-conditions.html#single-cell"><i class="fa fa-check"></i><b>6.6.4</b> Genome folding in single cells</a></li>
<li class="chapter" data-level="6.6.5" data-path="6-6-towards-studying-genome-folding-in-specific-conditions.html"><a href="6-6-towards-studying-genome-folding-in-specific-conditions.html#constantly-improving-targeted-experimental-methods."><i class="fa fa-check"></i><b>6.6.5</b> Constantly improving targeted experimental methods.</a></li>
</ul></li>
<li class="chapter" data-level="6.7" data-path="6-7-molecular-mechanisms-driving-genome-folding.html"><a href="6-7-molecular-mechanisms-driving-genome-folding.html"><i class="fa fa-check"></i><b>6.7</b> Molecular mechanisms driving genome folding</a></li>
<li class="chapter" data-level="6.8" data-path="6-8-conclusions.html"><a href="6-8-conclusions.html"><i class="fa fa-check"></i><b>6.8</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="appendix"><span><b>Appendix</b></span></li>
<li class="chapter" data-level="A" data-path="A-sup-paralog-regulation.html"><a href="A-sup-paralog-regulation.html"><i class="fa fa-check"></i><b>A</b> Supporting Information: Co-regulation of paralog genes</a></li>
<li class="chapter" data-level="B" data-path="B-sup-tad-evolution.html"><a href="B-sup-tad-evolution.html"><i class="fa fa-check"></i><b>B</b> Supplementary Data: Stability of TADs in evolution</a><ul>
<li class="chapter" data-level="B.1" data-path="B-1-TadEvoSupTab.html"><a href="B-1-TadEvoSupTab.html"><i class="fa fa-check"></i><b>B.1</b> Supplementary Tables</a></li>
<li class="chapter" data-level="B.2" data-path="B-2-supplementary-figures.html"><a href="B-2-supplementary-figures.html"><i class="fa fa-check"></i><b>B.2</b> Supplementary Figures</a></li>
</ul></li>
<li class="chapter" data-level="C" data-path="C-sup-position-effect.html"><a href="C-sup-position-effect.html"><i class="fa fa-check"></i><b>C</b> Supplemental Data: Position effects of rearrangements in disease genomes</a><ul>
<li class="chapter" data-level="C.1" data-path="C-1-supplemental-note.html"><a href="C-1-supplemental-note.html"><i class="fa fa-check"></i><b>C.1</b> Supplemental Note</a><ul>
<li class="chapter" data-level="C.1.1" data-path="C-1-supplemental-note.html"><a href="C-1-supplemental-note.html#case-reports"><i class="fa fa-check"></i><b>C.1.1</b> Case Reports</a></li>
<li class="chapter" data-level="C.1.2" data-path="C-1-supplemental-note.html"><a href="C-1-supplemental-note.html#nucleotide-level-nomenclature-for-dgap-karyotypes"><i class="fa fa-check"></i><b>C.1.2</b> Nucleotide Level Nomenclature for DGAP karyotypes</a></li>
</ul></li>
<li class="chapter" data-level="C.2" data-path="C-2-supplemental-figure.html"><a href="C-2-supplemental-figure.html"><i class="fa fa-check"></i><b>C.2</b> Supplemental Figure</a></li>
<li class="chapter" data-level="C.3" data-path="C-3-position-effect-sub-tab.html"><a href="C-3-position-effect-sub-tab.html"><i class="fa fa-check"></i><b>C.3</b> Supplemental Table Legends</a></li>
</ul></li>
<li class="chapter" data-level="D" data-path="D-loop-support.html"><a href="D-loop-support.html"><i class="fa fa-check"></i><b>D</b> Supplemental Information: Prediction of chromatin looping interactions</a><ul>
<li class="chapter" data-level="D.1" data-path="D-1-LoopPredSupTab.html"><a href="D-1-LoopPredSupTab.html"><i class="fa fa-check"></i><b>D.1</b> Supplementary Tables</a></li>
<li class="chapter" data-level="D.2" data-path="D-2-supplementary-figures-1.html"><a href="D-2-supplementary-figures-1.html"><i class="fa fa-check"></i><b>D.2</b> Supplementary Figures</a></li>
</ul></li>
<li class="chapter" data-level="E" data-path="E-contribution-to-individual-publications.html"><a href="E-contribution-to-individual-publications.html"><i class="fa fa-check"></i><b>E</b> Contribution to individual publications</a></li>
<li class="chapter" data-level="" data-path="zusammenfassung.html"><a href="zusammenfassung.html"><i class="fa fa-check"></i>Zusammenfassung</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Build with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Genome folding in evolution and disease</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="materials-and-methods" class="section level2">
<h2><span class="header-section-number">4.2</span> Materials and Methods</h2>
<div id="selection-of-subjects-with-apparently-balanced-chromosome-abnormalities" class="section level3">
<h3><span class="header-section-number">4.2.1</span> Selection of subjects with apparently balanced chromosome abnormalities</h3>
<p>BCA breakpoints and clinical data were obtained from DGAP cases for
which whole-genome sequencing was performed using a previously described
large-insert jumping library approach.<span class="citation">(Higgins et al. <a href="references.html#ref-Higgins2008">2008</a>; Ligon et al. <a href="references.html#ref-Ligon2005">2005</a>; Kim and Marcotte <a href="references.html#ref-Kim2008">2008</a>; Lu et al. <a href="references.html#ref-Lu2007">2007</a>; Redin et al. <a href="references.html#ref-Redin2017">2017</a>; Talkowski et al. <a href="references.html#ref-Talkowski2011">2011</a>)</span> A total of 151 cases
were filtered to select only subjects whose translocation or inversion
breakpoints fall within intergenic regions (GRCh37) and did not overlap
known long intergenic non-coding RNAs (lincRNAs) or pseudogenes, as
these elements have been shown to exert functional roles (reviewed in
<span class="citation">(Quinn and Chang <a href="references.html#ref-Quinn2016">2016</a>)</span> <span class="citation">(Pink et al. <a href="references.html#ref-Pink2011">2011</a>; Muro and Andrade-Navarro <a href="references.html#ref-Muro2010">2010</a>)</span>). Of 151 DGAP subjects, only 17 fulfilled our selection
criteria, 12 of whom had available and reportedly normal clinical array
results, suggesting lack of large duplications or deletions.</p>
</div>
<div id="clinical-descriptions-of-dgap-cases" class="section level3">
<h3><span class="header-section-number">4.2.2</span> Clinical descriptions of DGAP cases</h3>
<p>The clinical presentation of the 17 subjects varied, ranging from
developmental delay to neurological conditions, offering the opportunity
to assess long-range position effects in different phenotypes. Subjects’
karyotypes are presented in the main text using the International System
for Human Cytogenetic Nomenclature (ISCN2016) (Table <a href="4-2-materials-and-methods.html#tab:position-effect-subjects">4.1</a>). Detailed case
descriptions are included in the Supplemental Note: Case Reports, as
well as a nomenclature developed to describe chromosome rearrangements
using next-generation sequencing.<span class="citation">(Ordulu et al. <a href="references.html#ref-Ordulu2014">2014</a>)</span> Reported ages of DGAP subjects are
from time of enrollment. All reported genomic coordinates use GRCh37.</p>
<table>
<caption><span id="tab:position-effect-subjects">Table 4.1: </span> <strong>Description of the 17 analyzed DGAP cases with non-coding
BCAs.</strong> Corresponding clinical karyotypes are reported, with overlap of
breakpoints with regulatory elements (E = enhancer, DHS = DNaseI
hypersensitive sites, CTCF = CTCF binding sites), and TADs from H1-hESC,
IMR90, and GM12878 (1= one breakpoint within TAD, 2=both BCA breakpoints
are located within TAD). Top-ranking position effect genes are provided
for the +-1 Mb windows surrounding the BCA breakpoints; each gene is
highlighted with different evidence supporting its inclusion (a =
ClinGen known recessive genes, b= ClinGen genes with emerging and
sufficient evidence suggesting haploinsufficiency is associated with
clinical phenotype, c = HI scores less than 10, d = within H1-ESC TAD, e
= DHS enhancer-promoter disrupted interactions).</caption>
<colgroup>
<col width="28%" />
<col width="21%" />
<col width="29%" />
<col width="20%" />
</colgroup>
<thead>
<tr class="header">
<th align="left"><strong>Subject ID and Reported Karyotype</strong></th>
<th><strong>Disruption of Functional Elements</strong></th>
<th><strong>Breakpoints within TADs (hESC / IMR90 / GM12878)</strong></th>
<th align="right"><strong>Top-ranking Candidates +-1 Mb </strong></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><strong>DGAP017</strong> <em>46,X,t(X;10)(p11.2;q24.3)</em></td>
<td>DHS</td>
<td>2/2/1</td>
<td align="right">-</td>
</tr>
<tr class="even">
<td align="left"><strong>DGAP111</strong> <em>46,XY,t(16;20)(q11.2;q13.2)dn</em></td>
<td>CTCF</td>
<td>1/1/2</td>
<td align="right"><em>ORC6<sup>a</sup></em></td>
</tr>
<tr class="odd">
<td align="left"><strong>DGAP113</strong> *46,XY,t(1;3)(q32.1;q13.2)dn</td>
<td>-</td>
<td>2/2/2</td>
<td align="right"><em>ASPM<sup>a</sup></em></td>
</tr>
<tr class="even">
<td align="left"><strong>DGAP126</strong> <em>46,XX,t(5;10)(p13.3;q21.1)dn</em></td>
<td>-</td>
<td>2/1/2</td>
<td align="right">-</td>
</tr>
<tr class="odd">
<td align="left"><strong>DGAP138</strong> <em>46,XY,t(1;6)(q23;q13)dn</em></td>
<td>-</td>
<td>2/2/2</td>
<td align="right"><em>GRIK2<sup>ac</sup></em></td>
</tr>
<tr class="even">
<td align="left"><strong>DGAP153</strong> <em>46,X,t(X;17)(p11.23;p11.2)dn</em></td>
<td>-</td>
<td>1/1/1</td>
<td align="right">-</td>
</tr>
<tr class="odd">
<td align="left"><strong>DGAP163</strong> <em>46,XY,t(2;14)(p23;q13)dn</em></td>
<td>-</td>
<td>2/2/2</td>
<td align="right"><em>SOS1<sup>cde</sup>, COCH<sup>de</sup></em></td>
</tr>
<tr class="even">
<td align="left"><strong>DGAP176</strong> <em>46,Y,inv(X)(q13q24)mat</em></td>
<td>DHS, CTCF</td>
<td>2/1/2</td>
<td align="right"><em>ACSL4<sup>bd</sup>, COL4A5<sup>bcde</sup></em></td>
</tr>
<tr class="odd">
<td align="left"><strong>DGAP249</strong> <em>46,XX,t(2;11)(q33;q23)dn</em></td>
<td>E, DHS</td>
<td>2/2/2</td>
<td align="right"><em>SATB2<sup>bcde</sup>, SORL1<sup>e</sup></em></td>
</tr>
<tr class="even">
<td align="left"><strong>DGAP252</strong> <em>46,XY,t(3;18)(q13.2;q11.2)dn</em></td>
<td>-</td>
<td>2/2/2</td>
<td align="right"><em>RBBP8<sup>a</sup>,GATA6<sup>bcde</sup></em></td>
</tr>
<tr class="odd">
<td align="left"><strong>DGAP275</strong> <em>46,XX,t(7;12)(p13;q24.33)dn</em></td>
<td>DHS</td>
<td>1/1/2</td>
<td align="right"><em>ANKLE2<sup>e</sup>, POLE<sup>e</sup></em></td>
</tr>
<tr class="even">
<td align="left"><strong>DGAP287</strong> <em>46,XY,t(10;14)(p13;q32.1)dn</em></td>
<td>CTCF</td>
<td>2/2/2</td>
<td align="right">-</td>
</tr>
<tr class="odd">
<td align="left"><strong>DGAP288</strong> <em>46,XX,t(6:17)(q13;q21)dn</em></td>
<td>DHS</td>
<td>2/2/2</td>
<td align="right"><em>SOX9<sup>bcd</sup></em></td>
</tr>
<tr class="even">
<td align="left"><strong>DGAP315</strong> <em>46,XX,inv(6)(p24q11)dn</em></td>
<td>-</td>
<td>1/1/2</td>
<td align="right">-</td>
</tr>
<tr class="odd">
<td align="left"><strong>DGAP319</strong> <em>46,XX,t(4;13)(q31.3;q14.3)dn</em></td>
<td>-</td>
<td>2/1/2</td>
<td align="right">-</td>
</tr>
<tr class="even">
<td align="left"><strong>DGAP322</strong> <em>46,XY,t(1;18)(q32.1;q22.1)</em></td>
<td>DHS</td>
<td>1/2/2</td>
<td align="right"><em>IRF6<sup>bcd</sup></em></td>
</tr>
<tr class="odd">
<td align="left"><strong>DGAP329</strong> <em>46,XX,t(2;14)(q21;q24.3)dn</em></td>
<td>-</td>
<td>1/2/2</td>
<td align="right"><em>ZEB2<sup>bcde</sup></em></td>
</tr>
</tbody>
</table>
<p><!-- **Subject ID**   **Reported Karyotype**          **Disruption of Functional Elements**   **Breakpoints within <br>TADs (hESC / IMR90 / GM12878)**   **Top-ranking Candidates +-1 Mb ** -->
<!-- ---------------- ------------------------------- ------------------------------------- ----------------------------------------------------   ----------------------------------- -->
<!-- **DGAP017**      46,X,t(X;10)(p11.2;q24.3)       DHS                                   2 / 2 / 1                                              - -->
<!-- **DGAP111**      46,XY,t(16;20)(q11.2;q13.2)dn   CTCF                                  1 / 1 / 2                                              *ORC6^a^* -->
<!-- **DGAP113**      46,XY,t(1;3)(q32.1;q13.2)dn     -                                     2 / 2 / 2                                              *ASPM^a^* -->
<!-- **DGAP126**      46,XX,t(5;10)(p13.3;q21.1)dn    -                                     2 / 1 / 2                                               - -->
<!-- **DGAP138**      46,XY,t(1;6)(q23;q13)dn         -                                     2 / 2 / 2                                               *GRIK2^ac^* -->
<!-- **DGAP153**      46,X,t(X;17)(p11.23;p11.2)dn    -                                     1 / 1 / 1                                               - -->
<!-- **DGAP163**      46,XY,t(2;14)(p23;q13)dn        -                                     2 / 2 / 2                                               *SOS1^cde^, COCH^de^* -->
<!-- **DGAP176**      46,Y,inv(X)(q13q24)mat          DHS, CTCF                             2 / 1 / 2                                               *ACSL4^bd^, COL4A5^bcde^* -->
<!-- **DGAP249**      46,XX,t(2;11)(q33;q23)dn        E, DHS                                2 / 2 / 2                                               *SATB2^bcde^, SORL1^e^* -->
<!-- **DGAP252**      46,XY,t(3;18)(q13.2;q11.2)dn    -                                     2 / 2 / 2                                               *RBBP8^a^,GATA6^bcde^* -->
<!-- **DGAP275**      46,XX,t(7;12)(p13;q24.33)dn     DHS                                   1 / 1 / 2                                               *ANKLE2^e^, POLE^e^* -->
<!-- **DGAP287**      46,XY,t(10;14)(p13;q32.1)dn     CTCF                                  2 / 2 / 2                                               - -->
<!-- **DGAP288**      46,XX,t(6:17)(q13;q21)dn        DHS                                   2 / 2 / 2                                               *SOX9^bcd^* -->
<!-- **DGAP315**      46,XX,inv(6)(p24q11)dn          -                                     1 / 1 / 2                                               - -->
<!-- **DGAP319**      46,XX,t(4;13)(q31.3;q14.3)dn    -                                     2 / 1 / 2                                               - -->
<!-- **DGAP322**      46,XY,t(1;18)(q32.1;q22.1)      DHS                                   1 / 2 / 2                                               *IRF6^bcd^* -->
<!-- **DGAP329**      46,XX,t(2;14)(q21;q24.3)dn      -                                     1 / 2 / 2                                               *ZEB2^bcde^* --></p>
</div>
<div id="analysis-of-genes-bordering-the-rearrangement-breakpoints" class="section level3">
<h3><span class="header-section-number">4.2.3</span> Analysis of genes bordering the rearrangement breakpoints</h3>
<p>The presence of annotated genes or pseudogenes and lincRNAs was assessed
in windows of +-3 and +-1 Mb neighboring each subject’s translocation and
inversion breakpoints, and within reported H1-hESC topologically
associated domains (TADs)<span class="citation">(Dixon et al. <a href="references.html#ref-Dixon2012">2012</a>)</span> where the breakpoints were located. The
gene annotation file was obtained from Ensembl GRCh37 archive,<span class="citation">(Flicek et al. <a href="references.html#ref-Flicek2014">2014</a>)</span> and
we used the Human Body Map lincRNAs catalog.<span class="citation">(Cabili et al. <a href="references.html#ref-Cabili2011">2011</a>)</span> Haploinsufficiency (HI)
and triplosensitivity scores were assigned using Huang <em>et al</em>.,
2010<span class="citation">(Huang et al. <a href="references.html#ref-Huang2010">2010</a>)</span> and version hg19 of ClinGen<span class="citation">(Rehm et al. <a href="references.html#ref-Rehm2015">2015</a>)</span> data downloaded on 9/20/2016.</p>
</div>
<div id="assessment-of-disrupted-functional-elements-and-chromatin-interactions-bordering-rearrangement-breakpoints" class="section level3">
<h3><span class="header-section-number">4.2.4</span> Assessment of disrupted functional elements and chromatin interactions bordering rearrangement breakpoints</h3>
<p>The disruption of regulatory elements such as enhancers, promoters,
locus control regions, and insulators can lead to disease-related gene
expression changes; DNase I hypersensitive (DHS) sites have been used as
markers for the identification of such elements.<span class="citation">(Thurman et al. <a href="references.html#ref-Thurman2012">2012</a>)</span> In addition, the
alteration of TAD boundaries has been previously shown to cause a
rewiring of enhancers with pathological consequences;<span class="citation">(Lupiáñez et al. <a href="references.html#ref-Lupianez2015">2015</a>; Giorgio et al. <a href="references.html#ref-Giorgio2015">2015</a>; Narendra et al. <a href="references.html#ref-Narendra2015">2015</a>)</span>
CCCTC-Binding Factor (CTCF) binding sites have been found to be enriched
in TAD boundaries,<span class="citation">(Dixon et al. <a href="references.html#ref-Dixon2012">2012</a>)</span> and several mutations of boundary-defining sites
have been associated with cancer.<span class="citation">(Flavahan et al. <a href="references.html#ref-Flavahan2016">2016</a>; Hnisz et al. <a href="references.html#ref-Hnisz2016">2016</a><a href="references.html#ref-Hnisz2016">b</a>)</span> Based on these observations,
we assessed the number of regulatory elements that were potentially
disrupted by the analyzed DGAP breakpoints. We compared the breakpoint
positions of the selected DGAP subjects against data corresponding to
CTCF binding sites, DHS sites, and chromatin segmentation
classifications (Broad ChromHMM) derived from a lymphoblastoid cell line
(GM12878) and human stem cells (H1-hESC), obtained from the Encyclopedia
of DNA Elements (ENCODE) project<span class="citation">(Dunham et al. <a href="references.html#ref-Dunham2012">2012</a>)</span> and accessed through the University
of California Santa Cruz Genome Browser.<span class="citation">(Kent et al. <a href="references.html#ref-Kent2002">2002</a>)</span> Enhancer positions were
additionally obtained from Andersson <em>et al</em>., 2014<span class="citation">(Andersson et al. <a href="references.html#ref-Andersson2014">2014</a>)</span> for tissue and
primary cells, and the VISTA Enhancer browser, human version hg19.<span class="citation">(Visel et al. <a href="references.html#ref-Visel2007">2007</a>)</span>
Finally, lists of transcription factor (TF) binding sites and gene
promoters were obtained from the Ensembl database human version
GRCh37.<span class="citation">(Flicek et al. <a href="references.html#ref-Flicek2014">2014</a>)</span> Hi-C interaction data and TAD positions for H1-hESC,
GM06990, and IMR90 at 20 Kb, 40 Kb, 100 Kb, and 1 Mb resolution were
obtained from Dixon <em>et al</em>., 2012<span class="citation">(Dixon et al. <a href="references.html#ref-Dixon2012">2012</a>)</span> and the WashU EpiGenome
Browser.<span class="citation">(Zhou and Wang <a href="references.html#ref-Zhou2012">2012</a>)</span> A high-resolution dataset of chromatin loops and domains
was obtained from Rao <em>et al.</em>, 2014 for IMR90 and GM12878 cells.<span class="citation">(Rao et al. <a href="references.html#ref-Rao2014">2014</a>)</span>
Lastly, distal DHS/enhancer–promoter connections<span class="citation">(Thurman et al. <a href="references.html#ref-Thurman2012">2012</a>)</span> were used to assess
disrupted predicted cis-regulatory interactions by the BCAs. Genomic
overlaps between the rearrangement breakpoints, functional elements and
disrupted chromatin interactions were calculated using custom Perl
scripts, the BEDtools suite<span class="citation">(Quinlan and Hall <a href="references.html#ref-Quinlan2010">2010</a>)</span> and the genomic association tester (GAT)
tool.<span class="citation">(Heger et al. <a href="references.html#ref-Heger2013">2013</a>)</span></p>
</div>
<div id="ontological-analysis-of-genes-neighboring-breakpoints" class="section level3">
<h3><span class="header-section-number">4.2.5</span> Ontological analysis of genes neighboring breakpoints</h3>
<p>Phenotype similarity between potential position effect genes and DGAP
cases was calculated by converting the phenotypes of the 17 subjects to
Human Phenotype Ontology (HPO)<span class="citation">(Köhler et al. <a href="references.html#ref-Kohler2014">2014</a>)</span> terms and calculating their
phenomatch score as described in Ibn-Salem <em>et al</em>., 2014.<span class="citation">(Ibn-Salem et al. <a href="references.html#ref-Ibn-Salem2014">2014</a>)</span> The
phenomatch score quantiﬁes the information content of the most speciﬁc
HPO term that is part of or a common ancestor (more general term) of a
set of phenotypes. Our set of phenotypes is constituted by the HPO terms
associated to DGAP cases and the ones annotated to candidate position
effect genes within windows of +-3 and +-1 Mb of sequence in proximity to
the breakpoints. We used two background models to assess signiﬁcance of
this similarity. The ﬁrst is based on randomly permuting the
associations of phenotypes to genes; to this effect, the phenotype-gene
associations are shuffled 100 times randomly and the similarity of these
random phenotypes to the studied case clinical findings is calculated.
The second background control is based on shifting the breakpoint
location along the chromosome; each breakpoint is shifted by -9, -6, -3,
+3, +6, and +9 Mb and the similarity of genes in proximity to the
shifted breakpoints is computed.</p>
</div>
<div id="quantitative-real-time-pcr" class="section level3">
<h3><span class="header-section-number">4.2.6</span> Quantitative real-time PCR</h3>
<p>LCLs derived from DGAP236-02m, DGAP244-02m and DGAP245-02m were used as
karyotypically normal male controls. These are karyotypically normal
fathers of enrolled DGAP cases with no history of disease. LCL 17402
(DGAP163) was used to test differential gene expression for SOS Ras/Rac
guanine nucleotide exchange factor 1 (<em>SOS1</em> [MIM: 182530]), and LCL
18060 (DGAP176) was used to test midline 2 (<em>MID2</em> [MIM: 300204]), p21
(RAC1) activated kinase 3 (<em>PAK3</em> [MIM: 300142]), and POU class 3
homeobox 4 (<em>POU3F4</em> [MIM: 300039]) expression using quantitative
polymerase chain reaction (qPCR). Glucuronidase beta (<em>GUSB</em> [MIM:
611499]) was used as a housekeeping control. qPCR experiments were
performed by the Harvard Biopolymers Facility using TaqMan probes
Hs00264887_s1 (<em>POU3F4</em>), Hs00201978_m1 (<em>MID2</em>), Hs00176828_m1
(<em>PAK3</em>), Hs00893134_m1 (<em>SOS1</em>), and Hs00939627_m1 (<em>GUSB</em>). Data
were analyzed using the ΔCT method.</p>
</div>
<div id="assessment-of-dgap-breakpoints-overlapping-with-non-coding-structural-variants-in-public-databases" class="section level3">
<h3><span class="header-section-number">4.2.7</span> Assessment of DGAP breakpoints overlapping with non-coding structural variants in public databases</h3>
<p>To find similar non-coding structural rearrangement subjects and compare
their annotated clinical phenotypes to those observed in DGAP cases, we
searched the DatabasE of genomiC varIation and Phenotype in Humans using
Ensembl Resources (DECIPHER)<span class="citation">(Firth et al. <a href="references.html#ref-Firth2009">2009</a>)</span> version 2015-07-13, as well as the
dbVar database from the National Center for Biotechnology Information
(NCBI) Variation Viewer 1.5.<span class="citation">(Lappalainen et al. <a href="references.html#ref-Lappalainen2013">2012</a>)</span> Both databases are comprehensive
community-supported repositories of clinical cases with novel and
extremely rare genomic variants.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="4-1-introduction-2.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="4-3-results-2.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": true,
"facebook": false,
"twitter": true,
"google": false,
"weibo": false,
"instapper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/ibn-salem/thesis/edit/master/04-position_effect.Rmd",
"text": "Edit"
},
"download": ["thesis.pdf", "thesis.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://cdn.bootcss.com/mathjax/2.7.1/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:" && /^https?:/.test(src))
      src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
