<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>Genome folding in evolution and disease</title>
  <meta name="description" content="This is a draft PhD thesis.">
  <meta name="generator" content="bookdown 0.6.1 and GitBook 2.6.7">

  <meta property="og:title" content="Genome folding in evolution and disease" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="This is a draft PhD thesis." />
  <meta name="github-repo" content="ibn-salem/thesis" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Genome folding in evolution and disease" />
  
  <meta name="twitter:description" content="This is a draft PhD thesis." />
  

<meta name="author" content="Jonas Ibn-Salem">


<meta name="date" content="2018-01-28">

  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  
<link rel="prev" href="4-1-introduction-2.html">
<link rel="next" href="4-3-results-2.html">
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />










<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Thesis</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="1" data-path="1-thesis-intro.html"><a href="1-thesis-intro.html"><i class="fa fa-check"></i><b>1</b> Introduction</a><ul>
<li class="chapter" data-level="1.1" data-path="1-1-structure-of-the-intorduction.html"><a href="1-1-structure-of-the-intorduction.html"><i class="fa fa-check"></i><b>1.1</b> Structure of the Intorduction</a></li>
<li class="chapter" data-level="1.2" data-path="1-2-cell-diversity-and-regulation-of-gene-expression.html"><a href="1-2-cell-diversity-and-regulation-of-gene-expression.html"><i class="fa fa-check"></i><b>1.2</b> Cell diversity and regulation of gene expression</a></li>
<li class="chapter" data-level="1.3" data-path="1-3-enhancers.html"><a href="1-3-enhancers.html"><i class="fa fa-check"></i><b>1.3</b> Enhancers</a></li>
<li class="chapter" data-level="1.4" data-path="1-4-methods-to-probe-the-3d-chromatin-archtiecture.html"><a href="1-4-methods-to-probe-the-3d-chromatin-archtiecture.html"><i class="fa fa-check"></i><b>1.4</b> Methods to probe the 3D chromatin archtiecture</a><ul>
<li class="chapter" data-level="1.4.1" data-path="1-4-methods-to-probe-the-3d-chromatin-archtiecture.html"><a href="1-4-methods-to-probe-the-3d-chromatin-archtiecture.html#microscopy-based-techniques-to-visuallize-the-genome-in-3d"><i class="fa fa-check"></i><b>1.4.1</b> Microscopy-based techniques to visuallize the genome in 3D</a></li>
<li class="chapter" data-level="1.4.2" data-path="1-4-methods-to-probe-the-3d-chromatin-archtiecture.html"><a href="1-4-methods-to-probe-the-3d-chromatin-archtiecture.html#proximity-ligation-based-method-to-quantify-chromatin-interactions"><i class="fa fa-check"></i><b>1.4.2</b> Proximity-ligation based method to quantify chromatin interactions</a></li>
</ul></li>
<li class="chapter" data-level="1.5" data-path="1-5-hierarchy-of-chromatin-3d-structure.html"><a href="1-5-hierarchy-of-chromatin-3d-structure.html"><i class="fa fa-check"></i><b>1.5</b> Hierarchy of chromatin 3D structure</a><ul>
<li class="chapter" data-level="1.5.1" data-path="1-5-hierarchy-of-chromatin-3d-structure.html"><a href="1-5-hierarchy-of-chromatin-3d-structure.html#chromosomal-territories-and-inter-chromosomal-contacts"><i class="fa fa-check"></i><b>1.5.1</b> Chromosomal territories and inter-chromosomal contacts</a></li>
<li class="chapter" data-level="1.5.2" data-path="1-5-hierarchy-of-chromatin-3d-structure.html"><a href="1-5-hierarchy-of-chromatin-3d-structure.html#ab-compartments"><i class="fa fa-check"></i><b>1.5.2</b> A/B compartments</a></li>
<li class="chapter" data-level="1.5.3" data-path="1-5-hierarchy-of-chromatin-3d-structure.html"><a href="1-5-hierarchy-of-chromatin-3d-structure.html#topologically-associating-domains-tads"><i class="fa fa-check"></i><b>1.5.3</b> Topologically associating domains (TADs)</a></li>
<li class="chapter" data-level="1.5.4" data-path="1-5-hierarchy-of-chromatin-3d-structure.html"><a href="1-5-hierarchy-of-chromatin-3d-structure.html#chromatin-loops"><i class="fa fa-check"></i><b>1.5.4</b> Chromatin loops</a></li>
</ul></li>
<li class="chapter" data-level="1.6" data-path="1-6-dynamics-of-chromatin-structure.html"><a href="1-6-dynamics-of-chromatin-structure.html"><i class="fa fa-check"></i><b>1.6</b> Dynamics of chromatin structure</a></li>
<li class="chapter" data-level="1.7" data-path="1-7-changes-of-genome-folding-in-evoltuion-and-disease-genomes.html"><a href="1-7-changes-of-genome-folding-in-evoltuion-and-disease-genomes.html"><i class="fa fa-check"></i><b>1.7</b> Changes of genome folding in evoltuion and disease genomes</a></li>
<li class="chapter" data-level="1.8" data-path="1-8-aims-of-this-thesis.html"><a href="1-8-aims-of-this-thesis.html"><i class="fa fa-check"></i><b>1.8</b> Aims of this thesis</a><ul>
<li class="chapter" data-level="" data-path="1-8-aims-of-this-thesis.html"><a href="1-8-aims-of-this-thesis.html#is-the-three-dimensional-folding-of-genomes-associated-with-co-regulation-of-functionally-related-genes"><i class="fa fa-check"></i>Is the three-dimensional folding of genomes associated with co-regulation of functionally related genes? </a></li>
<li class="chapter" data-level="" data-path="1-8-aims-of-this-thesis.html"><a href="1-8-aims-of-this-thesis.html#are-tads-functional-building-blocks-of-genomes-and-subjected-to-selective-pressure-during-evolution"><i class="fa fa-check"></i>Are TADs functional building blocks of genomes and subjected to selective pressure during evolution?</a></li>
<li class="chapter" data-level="" data-path="1-8-aims-of-this-thesis.html"><a href="1-8-aims-of-this-thesis.html#is-the-disruption-of-tads-by-rearrangements-also-associated-to-genetic-diseases"><i class="fa fa-check"></i>Is the disruption of TADs by rearrangements also associated to genetic diseases?</a></li>
<li class="chapter" data-level="" data-path="1-8-aims-of-this-thesis.html"><a href="1-8-aims-of-this-thesis.html#can-chromatin-looping-interactions-be-predicted-by-chip-seq-and-sequence-features"><i class="fa fa-check"></i>Can chromatin looping interactions be predicted by ChIP-seq and sequence features?</a></li>
</ul></li>
<li class="chapter" data-level="1.9" data-path="1-9-structure-of-this-thesis.html"><a href="1-9-structure-of-this-thesis.html"><i class="fa fa-check"></i><b>1.9</b> Structure of this thesis</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="2-paralog-regulation.html"><a href="2-paralog-regulation.html"><i class="fa fa-check"></i><b>2</b> Paralog genes in the 3D genome architecture</a><ul>
<li class="chapter" data-level="" data-path="2-paralog-regulation.html"><a href="2-paralog-regulation.html#preamble"><i class="fa fa-check"></i>Preamble</a></li>
<li class="chapter" data-level="" data-path="2-paralog-regulation.html"><a href="2-paralog-regulation.html#abstract-1"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="2.1" data-path="2-1-introduction.html"><a href="2-1-introduction.html"><i class="fa fa-check"></i><b>2.1</b> Introduction</a></li>
<li class="chapter" data-level="2.2" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html"><i class="fa fa-check"></i><b>2.2</b> Materials and methods</a><ul>
<li class="chapter" data-level="2.2.1" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html#selection-of-pairs-of-paralog-genes"><i class="fa fa-check"></i><b>2.2.1</b> Selection of pairs of paralog genes</a></li>
<li class="chapter" data-level="2.2.2" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html#enhancers-to-gene-association"><i class="fa fa-check"></i><b>2.2.2</b> Enhancers to gene association</a></li>
<li class="chapter" data-level="2.2.3" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html#topological-associating-domains"><i class="fa fa-check"></i><b>2.2.3</b> Topological associating domains</a></li>
<li class="chapter" data-level="2.2.4" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html#hi-c-interaction-maps"><i class="fa fa-check"></i><b>2.2.4</b> Hi-C interaction maps</a></li>
<li class="chapter" data-level="2.2.5" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html#randomization"><i class="fa fa-check"></i><b>2.2.5</b> Randomization</a></li>
<li class="chapter" data-level="2.2.6" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html#statistical-tests"><i class="fa fa-check"></i><b>2.2.6</b> Statistical tests</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="2-3-results.html"><a href="2-3-results.html"><i class="fa fa-check"></i><b>2.3</b> Results</a><ul>
<li class="chapter" data-level="2.3.1" data-path="2-3-results.html"><a href="2-3-results.html#distribution-of-paralog-genes-in-the-human-genome"><i class="fa fa-check"></i><b>2.3.1</b> Distribution of paralog genes in the human genome</a></li>
<li class="chapter" data-level="2.3.2" data-path="2-3-results.html"><a href="2-3-results.html#co-expression-of-paralog-gene-pairs-across-tissues"><i class="fa fa-check"></i><b>2.3.2</b> Co-expression of paralog gene pairs across tissues</a></li>
<li class="chapter" data-level="2.3.3" data-path="2-3-results.html"><a href="2-3-results.html#paralog-genes-share-enhancers"><i class="fa fa-check"></i><b>2.3.3</b> Paralog genes share enhancers</a></li>
<li class="chapter" data-level="2.3.4" data-path="2-3-results.html"><a href="2-3-results.html#co-localization-of-paralogs-in-tads"><i class="fa fa-check"></i><b>2.3.4</b> Co-localization of paralogs in TADs</a></li>
<li class="chapter" data-level="2.3.5" data-path="2-3-results.html"><a href="2-3-results.html#distal-paralog-pairs-are-enriched-for-long-range-chromatin-contacts"><i class="fa fa-check"></i><b>2.3.5</b> Distal paralog pairs are enriched for long-range chromatin contacts</a></li>
<li class="chapter" data-level="2.3.6" data-path="2-3-results.html"><a href="2-3-results.html#close-paralogs-have-fewer-contacts-than-expected"><i class="fa fa-check"></i><b>2.3.6</b> Close paralogs have fewer contacts than expected</a></li>
<li class="chapter" data-level="2.3.7" data-path="2-3-results.html"><a href="2-3-results.html#paralogs-in-mouse-and-dog-genome"><i class="fa fa-check"></i><b>2.3.7</b> Paralogs in mouse and dog genome</a></li>
<li class="chapter" data-level="2.3.8" data-path="2-3-results.html"><a href="2-3-results.html#orthologs-of-human-paralogs-show-conserved-co-localization"><i class="fa fa-check"></i><b>2.3.8</b> Orthologs of human paralogs show conserved co-localization</a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="2-4-discussions.html"><a href="2-4-discussions.html"><i class="fa fa-check"></i><b>2.4</b> Discussions</a></li>
<li class="chapter" data-level="2.5" data-path="2-5-conclusion.html"><a href="2-5-conclusion.html"><i class="fa fa-check"></i><b>2.5</b> Conclusion</a></li>
<li class="chapter" data-level="2.6" data-path="2-6-acknowledgements.html"><a href="2-6-acknowledgements.html"><i class="fa fa-check"></i><b>2.6</b> Acknowledgements</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="3-TAD-evolution.html"><a href="3-TAD-evolution.html"><i class="fa fa-check"></i><b>3</b> Stability of TADs in evolution</a><ul>
<li class="chapter" data-level="" data-path="3-TAD-evolution.html"><a href="3-TAD-evolution.html#preamble-1"><i class="fa fa-check"></i>Preamble</a></li>
<li class="chapter" data-level="" data-path="3-TAD-evolution.html"><a href="3-TAD-evolution.html#abstract-2"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="" data-path="3-TAD-evolution.html"><a href="3-TAD-evolution.html#keywords"><i class="fa fa-check"></i>Keywords</a></li>
<li class="chapter" data-level="3.1" data-path="3-1-introduction-1.html"><a href="3-1-introduction-1.html"><i class="fa fa-check"></i><b>3.1</b> Introduction</a></li>
<li class="chapter" data-level="3.2" data-path="3-2-results-1.html"><a href="3-2-results-1.html"><i class="fa fa-check"></i><b>3.2</b> Results</a><ul>
<li class="chapter" data-level="3.2.1" data-path="3-2-results-1.html"><a href="3-2-results-1.html#identification-of-evolutionary-rearrangement-breakpoints-from-whole-genome-alignments"><i class="fa fa-check"></i><b>3.2.1</b> Identification of evolutionary rearrangement breakpoints from whole-genome alignments</a></li>
<li class="chapter" data-level="3.2.2" data-path="3-2-results-1.html"><a href="3-2-results-1.html#rearrangement-breakpoints-are-enriched-at-tad-boundaries"><i class="fa fa-check"></i><b>3.2.2</b> Rearrangement breakpoints are enriched at TAD boundaries</a></li>
<li class="chapter" data-level="3.2.3" data-path="3-2-results-1.html"><a href="3-2-results-1.html#clusters-of-conserved-non-coding-elements-are-depleted-for-rearrangement-breakpoints"><i class="fa fa-check"></i><b>3.2.3</b> Clusters of conserved non-coding elements are depleted for rearrangement breakpoints</a></li>
<li class="chapter" data-level="3.2.4" data-path="3-2-results-1.html"><a href="3-2-results-1.html#rearranged-tads-are-associated-with-divergent-gene-expression-between-species"><i class="fa fa-check"></i><b>3.2.4</b> Rearranged TADs are associated with divergent gene expression between species</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="3-3-discussion.html"><a href="3-3-discussion.html"><i class="fa fa-check"></i><b>3.3</b> Discussion</a></li>
<li class="chapter" data-level="3.4" data-path="3-4-conclusion-1.html"><a href="3-4-conclusion-1.html"><i class="fa fa-check"></i><b>3.4</b> Conclusion</a></li>
<li class="chapter" data-level="3.5" data-path="3-5-methods.html"><a href="3-5-methods.html"><i class="fa fa-check"></i><b>3.5</b> Methods</a><ul>
<li class="chapter" data-level="3.5.1" data-path="3-5-methods.html"><a href="3-5-methods.html#rearrangement-breakpoints-from-whole-genome-alignments"><i class="fa fa-check"></i><b>3.5.1</b> Rearrangement breakpoints from whole-genome alignments</a></li>
<li class="chapter" data-level="3.5.2" data-path="3-5-methods.html"><a href="3-5-methods.html#topologically-associating-domains-and-contact-domains"><i class="fa fa-check"></i><b>3.5.2</b> Topologically associating domains and contact domains</a></li>
<li class="chapter" data-level="3.5.3" data-path="3-5-methods.html"><a href="3-5-methods.html#breakpoint-distributions-at-tads"><i class="fa fa-check"></i><b>3.5.3</b> Breakpoint distributions at TADs</a></li>
<li class="chapter" data-level="3.5.4" data-path="3-5-methods.html"><a href="3-5-methods.html#quantification-of-breakpoint-enrichment"><i class="fa fa-check"></i><b>3.5.4</b> Quantification of breakpoint enrichment</a></li>
<li class="chapter" data-level="3.5.5" data-path="3-5-methods.html"><a href="3-5-methods.html#expression-data-for-mouse-and-human-orthologs"><i class="fa fa-check"></i><b>3.5.5</b> Expression data for mouse and human orthologs</a></li>
<li class="chapter" data-level="3.5.6" data-path="3-5-methods.html"><a href="3-5-methods.html#classification-of-tads-and-genes-according-to-rearrangements-and-grbs"><i class="fa fa-check"></i><b>3.5.6</b> Classification of TADs and genes according to rearrangements and GRBs</a></li>
<li class="chapter" data-level="3.5.7" data-path="3-5-methods.html"><a href="3-5-methods.html#source-code-and-implementation-details"><i class="fa fa-check"></i><b>3.5.7</b> Source code and implementation details</a></li>
</ul></li>
<li class="chapter" data-level="3.6" data-path="3-6-declarations.html"><a href="3-6-declarations.html"><i class="fa fa-check"></i><b>3.6</b> Declarations</a><ul>
<li class="chapter" data-level="3.6.1" data-path="3-6-declarations.html"><a href="3-6-declarations.html#ethics-approval-and-consent-to-participate"><i class="fa fa-check"></i><b>3.6.1</b> Ethics approval and consent to participate</a></li>
<li class="chapter" data-level="3.6.2" data-path="3-6-declarations.html"><a href="3-6-declarations.html#consent-for-publication"><i class="fa fa-check"></i><b>3.6.2</b> Consent for publication</a></li>
<li class="chapter" data-level="3.6.3" data-path="3-6-declarations.html"><a href="3-6-declarations.html#availability-of-data-and-material"><i class="fa fa-check"></i><b>3.6.3</b> Availability of data and material</a></li>
<li class="chapter" data-level="3.6.4" data-path="3-6-declarations.html"><a href="3-6-declarations.html#competing-interests"><i class="fa fa-check"></i><b>3.6.4</b> Competing interests</a></li>
<li class="chapter" data-level="3.6.5" data-path="3-6-declarations.html"><a href="3-6-declarations.html#funding"><i class="fa fa-check"></i><b>3.6.5</b> Funding</a></li>
<li class="chapter" data-level="3.6.6" data-path="3-6-declarations.html"><a href="3-6-declarations.html#authors-contributions"><i class="fa fa-check"></i><b>3.6.6</b> Authors‘ contributions</a></li>
<li class="chapter" data-level="3.6.7" data-path="3-6-declarations.html"><a href="3-6-declarations.html#acknowledgments"><i class="fa fa-check"></i><b>3.6.7</b> Acknowledgments</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="4-position-effect.html"><a href="4-position-effect.html"><i class="fa fa-check"></i><b>4</b> Position effects of rearrangements in disease genomes</a><ul>
<li class="chapter" data-level="" data-path="4-position-effect.html"><a href="4-position-effect.html#preamble-2"><i class="fa fa-check"></i>Preamble</a></li>
<li class="chapter" data-level="" data-path="4-position-effect.html"><a href="4-position-effect.html#abstract-3"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="4.1" data-path="4-1-introduction-2.html"><a href="4-1-introduction-2.html"><i class="fa fa-check"></i><b>4.1</b> Introduction</a></li>
<li class="chapter" data-level="4.2" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html"><i class="fa fa-check"></i><b>4.2</b> Materials and Methods</a><ul>
<li class="chapter" data-level="4.2.1" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html#selection-of-subjects-with-apparently-balanced-chromosome-abnormalities"><i class="fa fa-check"></i><b>4.2.1</b> Selection of subjects with apparently balanced chromosome abnormalities</a></li>
<li class="chapter" data-level="4.2.2" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html#clinical-descriptions-of-dgap-cases"><i class="fa fa-check"></i><b>4.2.2</b> Clinical descriptions of DGAP cases</a></li>
<li class="chapter" data-level="4.2.3" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html#analysis-of-genes-bordering-the-rearrangement-breakpoints"><i class="fa fa-check"></i><b>4.2.3</b> Analysis of genes bordering the rearrangement breakpoints</a></li>
<li class="chapter" data-level="4.2.4" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html#assessment-of-disrupted-functional-elements-and-chromatin-interactions-bordering-rearrangement-breakpoints"><i class="fa fa-check"></i><b>4.2.4</b> Assessment of disrupted functional elements and chromatin interactions bordering rearrangement breakpoints</a></li>
<li class="chapter" data-level="4.2.5" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html#ontological-analysis-of-genes-neighboring-breakpoints"><i class="fa fa-check"></i><b>4.2.5</b> Ontological analysis of genes neighboring breakpoints</a></li>
<li class="chapter" data-level="4.2.6" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html#quantitative-real-time-pcr"><i class="fa fa-check"></i><b>4.2.6</b> Quantitative real-time PCR</a></li>
<li class="chapter" data-level="4.2.7" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html#assessment-of-dgap-breakpoints-overlapping-with-non-coding-structural-variants-in-public-databases"><i class="fa fa-check"></i><b>4.2.7</b> Assessment of DGAP breakpoints overlapping with non-coding structural variants in public databases</a></li>
</ul></li>
<li class="chapter" data-level="4.3" data-path="4-3-results-2.html"><a href="4-3-results-2.html"><i class="fa fa-check"></i><b>4.3</b> Results</a><ul>
<li class="chapter" data-level="4.3.1" data-path="4-3-results-2.html"><a href="4-3-results-2.html#genomic-characterization-of-non-coding-breakpoints"><i class="fa fa-check"></i><b>4.3.1</b> Genomic characterization of non-coding breakpoints</a></li>
<li class="chapter" data-level="4.3.2" data-path="4-3-results-2.html"><a href="4-3-results-2.html#identification-of-genes-with-potential-position-effects"><i class="fa fa-check"></i><b>4.3.2</b> Identification of genes with potential position effects</a></li>
<li class="chapter" data-level="4.3.3" data-path="4-3-results-2.html"><a href="4-3-results-2.html#identification-of-subjects-with-shared-non-coding-chromosome-alterations-and-phenotypes"><i class="fa fa-check"></i><b>4.3.3</b> Identification of subjects with shared non-coding chromosome alterations and phenotypes</a></li>
</ul></li>
<li class="chapter" data-level="4.4" data-path="4-4-discussion-1.html"><a href="4-4-discussion-1.html"><i class="fa fa-check"></i><b>4.4</b> Discussion</a><ul>
<li class="chapter" data-level="" data-path="4-4-discussion-1.html"><a href="4-4-discussion-1.html#acknowledgements-1"><i class="fa fa-check"></i>Acknowledgements</a></li>
</ul></li>
<li class="chapter" data-level="4.5" data-path="4-5-figures.html"><a href="4-5-figures.html"><i class="fa fa-check"></i><b>4.5</b> Figures</a><ul>
<li class="chapter" data-level="4.5.1" data-path="4-5-figures.html"><a href="4-5-figures.html#web-resources"><i class="fa fa-check"></i><b>4.5.1</b> Web Resources</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="5-loop-prediction.html"><a href="5-loop-prediction.html"><i class="fa fa-check"></i><b>5</b> Prediction of chromatin looping interactions</a></li>
<li class="chapter" data-level="6" data-path="6-thesis-discussion.html"><a href="6-thesis-discussion.html"><i class="fa fa-check"></i><b>6</b> Discussion</a><ul>
<li class="chapter" data-level="6.1" data-path="6-1-discussion-on-tad-evolution.html"><a href="6-1-discussion-on-tad-evolution.html"><i class="fa fa-check"></i><b>6.1</b> Discussion on TAD evolution</a></li>
<li class="chapter" data-level="6.2" data-path="6-2-further-directions.html"><a href="6-2-further-directions.html"><i class="fa fa-check"></i><b>6.2</b> Further directions</a></li>
<li class="chapter" data-level="6.3" data-path="6-3-conclusions.html"><a href="6-3-conclusions.html"><i class="fa fa-check"></i><b>6.3</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="appendix"><span><b>Appendix</b></span></li>
<li class="chapter" data-level="A" data-path="A-sup-paralog-regulation.html"><a href="A-sup-paralog-regulation.html"><i class="fa fa-check"></i><b>A</b> Supporting Information: Co-regulation of paralog genes in the three-dimensional chromatin architecture</a></li>
<li class="chapter" data-level="B" data-path="B-sup-TAD-evolution.html"><a href="B-sup-TAD-evolution.html"><i class="fa fa-check"></i><b>B</b> Supplementary Data: Evolutionary stability of topologically associating domains is associated with conserved gene regulation</a><ul>
<li class="chapter" data-level="B.1" data-path="B-1-TadEvoSupTab.html"><a href="B-1-TadEvoSupTab.html"><i class="fa fa-check"></i><b>B.1</b> Supplementary Tables</a></li>
<li class="chapter" data-level="B.2" data-path="B-2-supplementary-figures.html"><a href="B-2-supplementary-figures.html"><i class="fa fa-check"></i><b>B.2</b> Supplementary Figures</a></li>
</ul></li>
<li class="chapter" data-level="C" data-path="C-contribution-to-individual-publications.html"><a href="C-contribution-to-individual-publications.html"><i class="fa fa-check"></i><b>C</b> Contribution to individual publications</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Build with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Genome folding in evolution and disease</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="materials-and-methods" class="section level2">
<h2><span class="header-section-number">4.2</span> Materials and Methods</h2>
<div id="selection-of-subjects-with-apparently-balanced-chromosome-abnormalities" class="section level3">
<h3><span class="header-section-number">4.2.1</span> Selection of subjects with apparently balanced chromosome abnormalities</h3>
<p>BCA breakpoints and clinical data were obtained from DGAP cases for which whole-genome sequencing was performed using a previously described large-insert jumping library approach.<span class="citation">(Higgins et al., <a href="#ref-Higgins2008">2008</a>; Kim and Marcotte, <a href="#ref-Kim2008">2008</a>; Ligon et al., <a href="#ref-Ligon2005">2005</a>; Lu et al., <a href="#ref-Lu2007">2007</a>; Redin et al., <a href="#ref-Redin2017">2017</a>)</span> A total of 151 cases were filtered to select only subjects whose translocation or inversion breakpoints fall within intergenic regions (GRCh37) and did not overlap known long intergenic non-coding RNAs (lincRNAs) or pseudogenes, as these elements have been shown to exert functional roles (reviewed in <span class="citation">(Quinn and Chang, <a href="#ref-Quinn2016">2016</a>)</span> <span class="citation">(Muro and Andrade-Navarro, <a href="#ref-Muro2010">2010</a>; Pink et al., <a href="#ref-Pink2011">2011</a>)</span>). Of 151 DGAP subjects, only 17 fulfilled our selection criteria, 12 of whom had available and reportedly normal clinical array results, suggesting lack of large duplications or deletions.</p>
</div>
<div id="clinical-descriptions-of-dgap-cases" class="section level3">
<h3><span class="header-section-number">4.2.2</span> Clinical descriptions of DGAP cases</h3>
<p>The clinical presentation of the 17 subjects varied, ranging from developmental delay to neurological conditions, offering the opportunity to assess long-range position effects in different phenotypes. Subjects’ karyotypes are presented in the main text using the International System for Human Cytogenetic Nomenclature (ISCN2016) (Table <a href="4-2-materials-and-methods.html#tab:position-effect-subjects">4.1</a>). Detailed case descriptions are included in the Supplemental Note: Case Reports, as well as a nomenclature developed to describe chromosome rearrangements using next-generation sequencing.<span class="citation">(Ordulu et al., <a href="#ref-Ordulu2014">2014</a>)</span> Reported ages of DGAP subjects are from time of enrollment. All reported genomic coordinates use GRCh37.</p>
<table>
<caption><span id="tab:position-effect-subjects">Table 4.1: </span> <strong>Description of the 17 analyzed DGAP cases with non-coding BCAs.</strong> Corresponding clinical karyotypes are reported, with overlap of breakpoints with regulatory elements (E = enhancer, DHS = DNaseI hypersensitive sites, CTCF = CTCF binding sites), and TADs from H1-hESC, IMR90, and GM12878 (1= one breakpoint within TAD, 2=both BCA breakpoints are located within TAD). Top-ranking position effect genes are provided for the ±1 Mb windows surrounding the BCA breakpoints; each gene is highlighted with different evidence supporting its inclusion (a = ClinGen known recessive genes, b= ClinGen genes with emerging and sufficient evidence suggesting haploinsufficiency is associated with clinical phenotype, c = HI scores less than 10, d = within H1-ESC TAD, e = DHS enhancer-promoter disrupted interactions).</caption>
<thead>
<tr class="header">
<th align="left"><strong>Subject ID</strong></th>
<th align="left"><strong>Reported Karyotype</strong></th>
<th align="left"><strong>Disruption of Functional Elemen</strong></th>
<th><strong>Breakpoints within TADs (hESC / IMR90 / GM12878)</strong></th>
<th align="left"><strong>Top-ranking Candidates ±1 Mb </strong></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><strong>DGAP017</strong></td>
<td align="left">46,X,t(X;10)(p11.2;q24.3)</td>
<td align="left">DHS</td>
<td>2 / 2 / 1</td>
<td align="left">-</td>
</tr>
<tr class="even">
<td align="left"><strong>DGAP111</strong></td>
<td align="left">46,XY,t(16;20)(q11.2;q13.2)dn</td>
<td align="left">CTCF</td>
<td>1 / 1 / 2</td>
<td align="left"><em>ORC6<sup>a</sup></em></td>
</tr>
<tr class="odd">
<td align="left"><strong>DGAP113</strong></td>
<td align="left">46,XY,t(1;3)(q32.1;q13.2)dn</td>
<td align="left">-</td>
<td>2 / 2 / 2</td>
<td align="left"><em>ASPM<sup>a</sup></em></td>
</tr>
<tr class="even">
<td align="left"><strong>DGAP126</strong></td>
<td align="left">46,XX,t(5;10)(p13.3;q21.1)dn</td>
<td align="left">-</td>
<td>2 / 1 / 2</td>
<td align="left">-</td>
</tr>
<tr class="odd">
<td align="left"><strong>DGAP138</strong></td>
<td align="left">46,XY,t(1;6)(q23;q13)dn</td>
<td align="left">-</td>
<td>2 / 2 / 2</td>
<td align="left"><em>GRIK2<sup>ac</sup></em></td>
</tr>
<tr class="even">
<td align="left"><strong>DGAP153</strong></td>
<td align="left">46,X,t(X;17)(p11.23;p11.2)dn</td>
<td align="left">-</td>
<td>1 / 1 / 1</td>
<td align="left">-</td>
</tr>
<tr class="odd">
<td align="left"><strong>DGAP163</strong></td>
<td align="left">46,XY,t(2;14)(p23;q13)dn</td>
<td align="left">-</td>
<td>2 / 2 / 2</td>
<td align="left"><em>SOS1<sup>cde</sup>, COCH<sup>de</sup></em></td>
</tr>
<tr class="even">
<td align="left"><strong>DGAP176</strong></td>
<td align="left">46,Y,inv(X)(q13q24)mat</td>
<td align="left">DHS, CTCF</td>
<td>2 / 1 / 2</td>
<td align="left"><em>ACSL4<sup>bd</sup>, COL4A5<sup>bcde</sup></em></td>
</tr>
<tr class="odd">
<td align="left"><strong>DGAP249</strong></td>
<td align="left">46,XX,t(2;11)(q33;q23)dn</td>
<td align="left">E, DHS</td>
<td>2 / 2 / 2</td>
<td align="left"><em>SATB2<sup>bcde</sup>, SORL1<sup>e</sup></em></td>
</tr>
<tr class="even">
<td align="left"><strong>DGAP252</strong></td>
<td align="left">46,XY,t(3;18)(q13.2;q11.2)dn</td>
<td align="left">-</td>
<td>2 / 2 / 2</td>
<td align="left"><em>RBBP8<sup>a</sup>,GATA6<sup>bcde</sup></em></td>
</tr>
<tr class="odd">
<td align="left"><strong>DGAP275</strong></td>
<td align="left">46,XX,t(7;12)(p13;q24.33)dn</td>
<td align="left">DHS</td>
<td>1 / 1 / 2</td>
<td align="left"><em>ANKLE2<sup>e</sup>, POLE<sup>e</sup></em></td>
</tr>
<tr class="even">
<td align="left"><strong>DGAP287</strong></td>
<td align="left">46,XY,t(10;14)(p13;q32.1)dn</td>
<td align="left">CTCF</td>
<td>2 / 2 / 2</td>
<td align="left">-</td>
</tr>
<tr class="odd">
<td align="left"><strong>DGAP288</strong></td>
<td align="left">46,XX,t(6:17)(q13;q21)dn</td>
<td align="left">DHS</td>
<td>2 / 2 / 2</td>
<td align="left"><em>SOX9<sup>bcd</sup></em></td>
</tr>
<tr class="even">
<td align="left"><strong>DGAP315</strong></td>
<td align="left">46,XX,inv(6)(p24q11)dn</td>
<td align="left">-</td>
<td>1 / 1 / 2</td>
<td align="left">-</td>
</tr>
<tr class="odd">
<td align="left"><strong>DGAP319</strong></td>
<td align="left">46,XX,t(4;13)(q31.3;q14.3)dn</td>
<td align="left">-</td>
<td>2 / 1 / 2</td>
<td align="left">-</td>
</tr>
<tr class="even">
<td align="left"><strong>DGAP322</strong></td>
<td align="left">46,XY,t(1;18)(q32.1;q22.1)</td>
<td align="left">DHS</td>
<td>1 / 2 / 2</td>
<td align="left"><em>IRF6<sup>bcd</sup></em></td>
</tr>
<tr class="odd">
<td align="left"><strong>DGAP329</strong></td>
<td align="left">46,XX,t(2;14)(q21;q24.3)dn</td>
<td align="left">-</td>
<td>1 / 2 / 2</td>
<td align="left"><em>ZEB2<sup>bcde</sup></em></td>
</tr>
</tbody>
</table>
</div>
<div id="analysis-of-genes-bordering-the-rearrangement-breakpoints" class="section level3">
<h3><span class="header-section-number">4.2.3</span> Analysis of genes bordering the rearrangement breakpoints</h3>
<p>The presence of annotated genes or pseudogenes and lincRNAs was assessed in windows of ±3 and ±1 Mb neighboring each subject’s translocation and inversion breakpoints, and within reported H1-hESC topologically associated domains (TADs)<span class="citation">(Dixon et al., <a href="#ref-Dixon2012">2012</a>)</span> where the breakpoints were located. The gene annotation file was obtained from Ensembl GRCh37 archive,<span class="citation">(Flicek et al., <a href="#ref-Flicek2014">2014</a>)</span> and we used the Human Body Map lincRNAs catalog.<span class="citation">(Cabili et al., <a href="#ref-Cabili2011">2011</a>)</span> Haploinsufficiency (HI) and triplosensitivity scores were assigned using Huang <em>et al</em>., 2010<span class="citation">(Huang et al., <a href="#ref-Huang2010">2010</a>)</span> and version hg19 of ClinGen<span class="citation">(Rehm et al., <a href="#ref-Rehm2015">2015</a>)</span> data downloaded on 9/20/2016.</p>
</div>
<div id="assessment-of-disrupted-functional-elements-and-chromatin-interactions-bordering-rearrangement-breakpoints" class="section level3">
<h3><span class="header-section-number">4.2.4</span> Assessment of disrupted functional elements and chromatin interactions bordering rearrangement breakpoints</h3>
<p>The disruption of regulatory elements such as enhancers, promoters, locus control regions, and insulators can lead to disease-related gene expression changes; DNase I hypersensitive (DHS) sites have been used as markers for the identification of such elements.<span class="citation">(Thurman et al., <a href="#ref-Thurman2012">2012</a>)</span> In addition, the alteration of TAD boundaries has been previously shown to cause a rewiring of enhancers with pathological consequences;<span class="citation">(Giorgio et al., <a href="#ref-Giorgio2015">2015</a>; Lupiáñez et al., <a href="#ref-Lupianez2015">2015</a>; Narendra et al., <a href="#ref-Narendra2015">2015</a>)</span> CCCTC-Binding Factor (CTCF) binding sites have been found to be enriched in TAD boundaries,<span class="citation">(Dixon et al., <a href="#ref-Dixon2012">2012</a>)</span> and several mutations of boundary-defining sites have been associated with cancer.<span class="citation">(Flavahan et al., <a href="#ref-Flavahan2016">2016</a>; Hnisz et al., <a href="#ref-Hnisz2016">2016</a>)</span> Based on these observations, we assessed the number of regulatory elements that were potentially disrupted by the analyzed DGAP breakpoints. We compared the breakpoint positions of the selected DGAP subjects against data corresponding to CTCF binding sites, DHS sites, and chromatin segmentation classifications (Broad ChromHMM) derived from a lymphoblastoid cell line (GM12878) and human stem cells (H1-hESC), obtained from the Encyclopedia of DNA Elements (ENCODE) project<span class="citation">(Dunham et al., <a href="#ref-Dunham2012">2012</a>)</span> and accessed through the University of California Santa Cruz Genome Browser.<span class="citation">(Kent et al., <a href="#ref-Kent2002">2002</a>)</span> Enhancer positions were additionally obtained from Andersson <em>et al</em>., 2014<span class="citation">(Andersson et al., <a href="#ref-Andersson2014">2014</a>)</span> for tissue and primary cells, and the VISTA Enhancer browser, human version hg19.<span class="citation">(Visel et al., <a href="#ref-Visel2007">2007</a>)</span> Finally, lists of transcription factor (TF) binding sites and gene promoters were obtained from the Ensembl database human version GRCh37.<span class="citation">(Flicek et al., <a href="#ref-Flicek2014">2014</a>)</span> Hi-C interaction data and TAD positions for H1-hESC, GM06990, and IMR90 at 20 Kb, 40 Kb, 100 Kb, and 1 Mb resolution were obtained from Dixon <em>et al</em>., 2012<span class="citation">(Dixon et al., <a href="#ref-Dixon2012">2012</a>)</span> and the WashU EpiGenome Browser.<span class="citation">(Zhou and Wang, <a href="#ref-Zhou2012">2012</a>)</span> A high-resolution dataset of chromatin loops and domains was obtained from Rao <em>et al.</em>, 2014 for IMR90 and GM12878 cells.<span class="citation">(Rao et al., <a href="#ref-Rao2014">2014</a>)</span> Lastly, distal DHS/enhancer–promoter connections<span class="citation">(Thurman et al., <a href="#ref-Thurman2012">2012</a>)</span> were used to assess disrupted predicted cis-regulatory interactions by the BCAs. Genomic overlaps between the rearrangement breakpoints, functional elements and disrupted chromatin interactions were calculated using custom Perl scripts, the BEDtools suite<span class="citation">(Quinlan and Hall, <a href="#ref-Quinlan2010">2010</a>)</span> and the genomic association tester (GAT) tool.<span class="citation">(Heger et al., <a href="#ref-Heger2013">2013</a>)</span></p>
</div>
<div id="ontological-analysis-of-genes-neighboring-breakpoints" class="section level3">
<h3><span class="header-section-number">4.2.5</span> Ontological analysis of genes neighboring breakpoints</h3>
<p>Phenotype similarity between potential position effect genes and DGAP cases was calculated by converting the phenotypes of the 17 subjects to Human Phenotype Ontology (HPO)<span class="citation">(Köhler et al., <a href="#ref-Kohler2014">2014</a>)</span> terms and calculating their phenomatch score as described in Ibn-Salem <em>et al</em>., 2014.<span class="citation">(Ibn-Salem et al., <a href="#ref-Ibn-Salem2014">2014</a>)</span> The phenomatch score quantiﬁes the information content of the most speciﬁc HPO term that is part of or a common ancestor (more general term) of a set of phenotypes. Our set of phenotypes is constituted by the HPO terms associated to DGAP cases and the ones annotated to candidate position effect genes within windows of ±3 and ±1 Mb of sequence in proximity to the breakpoints. We used two background models to assess signiﬁcance of this similarity. The ﬁrst is based on randomly permuting the associations of phenotypes to genes; to this effect, the phenotype-gene associations are shuffled 100 times randomly and the similarity of these random phenotypes to the studied case clinical findings is calculated. The second background control is based on shifting the breakpoint location along the chromosome; each breakpoint is shifted by -9, -6, -3, +3, +6, and +9 Mb and the similarity of genes in proximity to the shifted breakpoints is computed.</p>
</div>
<div id="quantitative-real-time-pcr" class="section level3">
<h3><span class="header-section-number">4.2.6</span> Quantitative real-time PCR</h3>
<p>LCLs derived from DGAP236-02m, DGAP244-02m and DGAP245-02m were used as karyotypically normal male controls. These are karyotypically normal fathers of enrolled DGAP cases with no history of disease. LCL 17402 (DGAP163) was used to test differential gene expression for SOS Ras/Rac guanine nucleotide exchange factor 1 (<em>SOS1</em> [MIM: 182530]), and LCL 18060 (DGAP176) was used to test midline 2 (<em>MID2</em> [MIM: 300204]), p21 (RAC1) activated kinase 3 (<em>PAK3</em> [MIM: 300142]), and POU class 3 homeobox 4 (<em>POU3F4</em> [MIM: 300039]) expression using quantitative polymerase chain reaction (qPCR). Glucuronidase beta (<em>GUSB</em> [MIM: 611499]) was used as a housekeeping control. qPCR experiments were performed by the Harvard Biopolymers Facility using TaqMan probes Hs00264887_s1 (<em>POU3F4</em>), Hs00201978_m1 (<em>MID2</em>), Hs00176828_m1 (<em>PAK3</em>), Hs00893134_m1 (<em>SOS1</em>), and Hs00939627_m1 (<em>GUSB</em>). Data were analyzed using the ΔCT method.</p>
</div>
<div id="assessment-of-dgap-breakpoints-overlapping-with-non-coding-structural-variants-in-public-databases" class="section level3">
<h3><span class="header-section-number">4.2.7</span> Assessment of DGAP breakpoints overlapping with non-coding structural variants in public databases</h3>
<p>To find similar non-coding structural rearrangement subjects and compare their annotated clinical phenotypes to those observed in DGAP cases, we searched the DatabasE of genomiC varIation and Phenotype in Humans using Ensembl Resources (DECIPHER)<span class="citation">(Firth et al., <a href="#ref-Firth2009">2009</a>)</span> version 2015-07-13, as well as the dbVar database from the National Center for Biotechnology Information (NCBI) Variation Viewer 1.5.<span class="citation">(Lappalainen et al., <a href="#ref-Lappalainen2013">2012</a>)</span> Both databases are comprehensive community-supported repositories of clinical cases with novel and extremely rare genomic variants.</p>
</div>
</div>
<h3>References</h3>
<div id="refs" class="references">
<div id="ref-Higgins2008">
<p>Higgins, A.W., Alkuraya, F.S., Bosco, A.F., Brown, K.K., Bruns, G.A., Donovan, D.J., Eisenman, R., Fan, Y., Farra, C.G., Ferguson, H.L., et al. (2008). Characterization of Apparently Balanced Chromosomal Rearrangements from the Developmental Genome Anatomy Project. American Journal of Human Genetics <em>82</em>, 712–722.</p>
</div>
<div id="ref-Kim2008">
<p>Kim, W.K., and Marcotte, E.M. (2008). Age-dependent evolution of the yeast protein interaction network suggests a limited role of gene duplication and divergence. PLoS Computational Biology <em>4</em>, e1000232.</p>
</div>
<div id="ref-Ligon2005">
<p>Ligon, A.H., Moore, S.D., Parisi, M.A., Mealiffe, M.E., Harris, D.J., Ferguson, H.L., Quade, B.J., and Morton, C.C. (2005). Constitutional Rearrangement of the Architectural Factor HMGA2: A Novel Human Phenotype Including Overgrowth and Lipomas. The American Journal of Human Genetics <em>76</em>, 340–348.</p>
</div>
<div id="ref-Lu2007">
<p>Lu, W., Quintero-Rivera, F., Fan, Y., Alkuraya, F.S., Donovan, D.J., Xi, Q., Turbe-Doan, A., Li, Q.-G., Campbell, C.G., Shanske, A.L., et al. (2007). NFIA Haploinsufficiency Is Associated with a CNS Malformation Syndrome and Urinary Tract Defects. PLoS Genetics <em>3</em>, e80.</p>
</div>
<div id="ref-Redin2017">
<p>Redin, C., Brand, H., Collins, R.L., Kammin, T., Mitchell, E., Hodge, J.C., Hanscom, C., Pillalamarri, V., Seabra, C.M., Abbott, M.-A., et al. (2017). The genomic landscape of balanced cytogenetic abnormalities associated with human congenital anomalies. Nature Genetics <em>49</em>, 36–45.</p>
</div>
<div id="ref-Quinn2016">
<p>Quinn, J.J., and Chang, H.Y. (2016). Unique features of long non-coding RNA biogenesis and function. Nature Reviews Genetics <em>17</em>, 47–62.</p>
</div>
<div id="ref-Muro2010">
<p>Muro, E.M., and Andrade-Navarro, M.A. (2010). Pseudogenes as an alternative source of natural antisense transcripts. BMC Evolutionary Biology <em>10</em>, 338.</p>
</div>
<div id="ref-Pink2011">
<p>Pink, R.C., Wicks, K., Caley, D.P., Punch, E.K., Jacobs, L., and Carter, D.R.F. (2011). Pseudogenes: pseudo-functional or key regulators in health and disease? RNA (New York, N.Y.) <em>17</em>, 792–798.</p>
</div>
<div id="ref-Ordulu2014">
<p>Ordulu, Z., Wong, K.E., Currall, B.B., Ivanov, A.R., Pereira, S., Althari, S., Gusella, J.F., Talkowski, M.E., and Morton, C.C. (2014). Describing Sequencing Results of Structural Chromosome Rearrangements with a Suggested Next-Generation Cytogenetic Nomenclature. The American Journal of Human Genetics <em>94</em>, 695–709.</p>
</div>
<div id="ref-Dixon2012">
<p>Dixon, J.R., Selvaraj, S., Yue, F., Kim, A., Li, Y., Shen, Y., Hu, M., Liu, J.S., and Ren, B. (2012). Topological domains in mammalian genomes identified by analysis of chromatin interactions. Nature <em>485</em>, 376–380.</p>
</div>
<div id="ref-Flicek2014">
<p>Flicek, P., Amode, M.R., Barrell, D., Beal, K., Billis, K., Brent, S., Carvalho-Silva, D., Clapham, P., Coates, G., Fitzgerald, S., et al. (2014). Ensembl 2014. Nucleic Acids Research <em>42</em>, D749–D755.</p>
</div>
<div id="ref-Cabili2011">
<p>Cabili, M.N., Trapnell, C., Goff, L., Koziol, M., Tazon-Vega, B., Regev, A., and Rinn, J.L. (2011). Integrative annotation of human large intergenic noncoding RNAs reveals global properties and specific subclasses. Genes &amp; Development <em>25</em>, 1915–1927.</p>
</div>
<div id="ref-Huang2010">
<p>Huang, N., Lee, I., Marcotte, E.M., and Hurles, M.E. (2010). Characterising and Predicting Haploinsufficiency in the Human Genome. PLoS Genetics <em>6</em>, e1001154.</p>
</div>
<div id="ref-Rehm2015">
<p>Rehm, H.L., Berg, J.S., Brooks, L.D., Bustamante, C.D., Evans, J.P., Landrum, M.J., Ledbetter, D.H., Maglott, D.R., Martin, C.L., Nussbaum, R.L., et al. (2015). ClinGen — The Clinical Genome Resource. New England Journal of Medicine <em>372</em>, 2235–2242.</p>
</div>
<div id="ref-Thurman2012">
<p>Thurman, R.E., Rynes, E., Humbert, R., Vierstra, J., Maurano, M.T., Haugen, E., Sheffield, N.C., Stergachis, A.B., Wang, H., Vernot, B., et al. (2012). The accessible chromatin landscape of the human genome. Nature <em>489</em>, 75–82.</p>
</div>
<div id="ref-Giorgio2015">
<p>Giorgio, E., Robyr, D., Spielmann, M., Ferrero, E., Di Gregorio, E., Imperiale, D., Vaula, G., Stamoulis, G., Santoni, F., Atzori, C., et al. (2015). A large genomic deletion leads to enhancer adoption by the lamin B1 gene: a second path to autosomal dominant adult-onset demyelinating leukodystrophy (ADLD). Human Molecular Genetics <em>24</em>, 3143–3154.</p>
</div>
<div id="ref-Lupianez2015">
<p>Lupiáñez, D.G., Kraft, K., Heinrich, V., Krawitz, P., Brancati, F., Klopocki, E., Horn, D., Kayserili, H., Opitz, J.M., Laxova, R., et al. (2015). Disruptions of Topological Chromatin Domains Cause Pathogenic Rewiring of Gene-Enhancer Interactions. Cell <em>161</em>, 1012–1025.</p>
</div>
<div id="ref-Narendra2015">
<p>Narendra, V., Rocha, P.P., An, D., Raviram, R., Skok, J.A., Mazzoni, E.O., and Reinberg, D. (2015). Transcription. CTCF establishes discrete functional chromatin domains at the Hox clusters during differentiation. Science (New York, N.Y.) <em>347</em>, 1017–1021.</p>
</div>
<div id="ref-Flavahan2016">
<p>Flavahan, W. a, Drier, Y., Liau, B.B., Gillespie, S.M., Venteicher, a S., Stemmer-Rachamimov, a O., Suva, M.L., and Bernstein, B.E. (2016). Insulator dysfunction and oncogene activation in IDH mutant gliomas. Nature <em>529</em>, 110–114.</p>
</div>
<div id="ref-Hnisz2016">
<p>Hnisz, D., Weintraub, A.S., Day, D.S., Valton, A.-l., Bak, R.O., Li, C.H., Goldmann, J., Lajoie, B.R., Fan, Z.P., Sigova, A.A., et al. (2016). Activation of proto-oncogenes by disruption of chromosome neighborhoods. Science (New York, N.Y.) <em>351</em>, 1454–1458.</p>
</div>
<div id="ref-Dunham2012">
<p>Dunham, I., Kundaje, A., Aldred, S.F., Collins, P.J., Davis, C. a, Doyle, F., Epstein, C.B., Frietze, S., Harrow, J., Kaul, R., et al. (2012). An integrated encyclopedia of DNA elements in the human genome. Nature <em>489</em>, 57–74.</p>
</div>
<div id="ref-Kent2002">
<p>Kent, W.J., Sugnet, C.W., Furey, T.S., Roskin, K.M., Pringle, T.H., Zahler, A.M., and Haussler, D. (2002). The human genome browser at UCSC. Genome Research <em>12</em>, 996–1006.</p>
</div>
<div id="ref-Andersson2014">
<p>Andersson, R., Gebhard, C., Miguel-Escalada, I., Hoof, I., Bornholdt, J., Boyd, M., Chen, Y., Zhao, X., Schmidl, C., Suzuki, T., et al. (2014). An atlas of active enhancers across human cell types and tissues. Nature <em>507</em>, 455–461.</p>
</div>
<div id="ref-Visel2007">
<p>Visel, A., Minovitsky, S., Dubchak, I., and Pennacchio, L. a (2007). VISTA Enhancer Browser–a database of tissue-specific human enhancers. Nucleic Acids Research <em>35</em>, D88–92.</p>
</div>
<div id="ref-Zhou2012">
<p>Zhou, X., and Wang, T. (2012). Using the Wash U Epigenome Browser to Examine Genome-Wide Sequencing Data. In Current Protocols in Bioinformatics, (Hoboken, NJ, USA: John Wiley &amp; Sons, Inc.), p. Unit10.10.</p>
</div>
<div id="ref-Rao2014">
<p>Rao, S.S.P., Huntley, M.H., Durand, N.C., Stamenova, E.K., Bochkov, I.D., Robinson, J.T., Sanborn, A.L., Machol, I., Omer, A.D., Lander, E.S., et al. (2014). A 3D map of the human genome at kilobase resolution reveals principles of chromatin looping. Cell <em>159</em>, 1665–1680.</p>
</div>
<div id="ref-Quinlan2010">
<p>Quinlan, A.R., and Hall, I.M. (2010). BEDTools: a flexible suite of utilities for comparing genomic features. Bioinformatics <em>26</em>, 841–842.</p>
</div>
<div id="ref-Heger2013">
<p>Heger, A., Webber, C., Goodson, M., Ponting, C.P., and Lunter, G. (2013). GAT: a simulation framework for testing the association of genomic intervals. Bioinformatics (Oxford, England) <em>29</em>, 2046–2048.</p>
</div>
<div id="ref-Kohler2014">
<p>Köhler, S., Doelken, S.C., Mungall, C.J., Bauer, S., Firth, H.V., Bailleul-Forestier, I., Black, G.C.M., Brown, D.L., Brudno, M., Campbell, J., et al. (2014). The Human Phenotype Ontology project: linking molecular biology and disease through phenotype data. Nucleic Acids Research <em>42</em>, D966–74.</p>
</div>
<div id="ref-Ibn-Salem2014">
<p>Ibn-Salem, J., Köhler, S., Love, M.I., Chung, H.-R., Huang, N., Hurles, M.E., Haendel, M., Washington, N.L., Smedley, D., Mungall, C.J., et al. (2014). Deletions of chromosomal regulatory boundaries are associated with congenital disease. Genome Biology <em>15</em>, 423.</p>
</div>
<div id="ref-Firth2009">
<p>Firth, H.V., Richards, S.M., Bevan, a P., Clayton, S., Corpas, M., Rajan, D., Van Vooren, S., Moreau, Y., Pettett, R.M., and Carter, N.P. (2009). DECIPHER: Database of Chromosomal Imbalance and Phenotype in Humans Using Ensembl Resources. American Journal of Human Genetics <em>84</em>, 524–533.</p>
</div>
<div id="ref-Lappalainen2013">
<p>Lappalainen, I., Lopez, J., Skipper, L., Hefferon, T., Spalding, J.D., Garner, J., Chen, C., Maguire, M., Corbett, M., Zhou, G., et al. (2012). dbVar and DGVa: public archives for genomic structural variation. Nucleic Acids Research <em>41</em>, D936–D941.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="4-1-introduction-2.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="4-3-results-2.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"google": false,
"weibo": false,
"instapper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/ibn-salem/thesis/edit/master/04-position_effect.Rmd",
"text": "Edit"
},
"download": ["thesis.pdf", "thesis.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "";
    if (src === "" || src === "true") src = "https://cdn.bootcss.com/mathjax/2.7.1/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:" && /^https?:/.test(src))
      src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
