<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>Genome folding in evolution and disease</title>
  <meta name="description" content="This is a draft PhD thesis.">
  <meta name="generator" content="bookdown 0.6.1 and GitBook 2.6.7">

  <meta property="og:title" content="Genome folding in evolution and disease" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="This is a draft PhD thesis." />
  <meta name="github-repo" content="ibn-salem/thesis" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Genome folding in evolution and disease" />
  
  <meta name="twitter:description" content="This is a draft PhD thesis." />
  

<meta name="author" content="Jonas Ibn-Salem">


<meta name="date" content="2018-01-28">

  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  
<link rel="prev" href="3-6-declarations.html">
<link rel="next" href="5-loop-prediction.html">
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />










<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Thesis</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="1" data-path="1-thesis-intro.html"><a href="1-thesis-intro.html"><i class="fa fa-check"></i><b>1</b> Introduction</a><ul>
<li class="chapter" data-level="1.1" data-path="1-1-structure-of-the-intorduction.html"><a href="1-1-structure-of-the-intorduction.html"><i class="fa fa-check"></i><b>1.1</b> Structure of the Intorduction</a></li>
<li class="chapter" data-level="1.2" data-path="1-2-cell-diversity-and-regulation-of-gene-expression.html"><a href="1-2-cell-diversity-and-regulation-of-gene-expression.html"><i class="fa fa-check"></i><b>1.2</b> Cell diversity and regulation of gene expression</a></li>
<li class="chapter" data-level="1.3" data-path="1-3-enhancers.html"><a href="1-3-enhancers.html"><i class="fa fa-check"></i><b>1.3</b> Enhancers</a></li>
<li class="chapter" data-level="1.4" data-path="1-4-methods-to-probe-the-3d-chromatin-archtiecture.html"><a href="1-4-methods-to-probe-the-3d-chromatin-archtiecture.html"><i class="fa fa-check"></i><b>1.4</b> Methods to probe the 3D chromatin archtiecture</a><ul>
<li class="chapter" data-level="1.4.1" data-path="1-4-methods-to-probe-the-3d-chromatin-archtiecture.html"><a href="1-4-methods-to-probe-the-3d-chromatin-archtiecture.html#microscopy-based-techniques-to-visuallize-the-genome-in-3d"><i class="fa fa-check"></i><b>1.4.1</b> Microscopy-based techniques to visuallize the genome in 3D</a></li>
<li class="chapter" data-level="1.4.2" data-path="1-4-methods-to-probe-the-3d-chromatin-archtiecture.html"><a href="1-4-methods-to-probe-the-3d-chromatin-archtiecture.html#proximity-ligation-based-method-to-quantify-chromatin-interactions"><i class="fa fa-check"></i><b>1.4.2</b> Proximity-ligation based method to quantify chromatin interactions</a></li>
</ul></li>
<li class="chapter" data-level="1.5" data-path="1-5-hierarchy-of-chromatin-3d-structure.html"><a href="1-5-hierarchy-of-chromatin-3d-structure.html"><i class="fa fa-check"></i><b>1.5</b> Hierarchy of chromatin 3D structure</a><ul>
<li class="chapter" data-level="1.5.1" data-path="1-5-hierarchy-of-chromatin-3d-structure.html"><a href="1-5-hierarchy-of-chromatin-3d-structure.html#chromosomal-territories-and-inter-chromosomal-contacts"><i class="fa fa-check"></i><b>1.5.1</b> Chromosomal territories and inter-chromosomal contacts</a></li>
<li class="chapter" data-level="1.5.2" data-path="1-5-hierarchy-of-chromatin-3d-structure.html"><a href="1-5-hierarchy-of-chromatin-3d-structure.html#ab-compartments"><i class="fa fa-check"></i><b>1.5.2</b> A/B compartments</a></li>
<li class="chapter" data-level="1.5.3" data-path="1-5-hierarchy-of-chromatin-3d-structure.html"><a href="1-5-hierarchy-of-chromatin-3d-structure.html#topologically-associating-domains-tads"><i class="fa fa-check"></i><b>1.5.3</b> Topologically associating domains (TADs)</a></li>
<li class="chapter" data-level="1.5.4" data-path="1-5-hierarchy-of-chromatin-3d-structure.html"><a href="1-5-hierarchy-of-chromatin-3d-structure.html#chromatin-loops"><i class="fa fa-check"></i><b>1.5.4</b> Chromatin loops</a></li>
</ul></li>
<li class="chapter" data-level="1.6" data-path="1-6-dynamics-of-chromatin-structure.html"><a href="1-6-dynamics-of-chromatin-structure.html"><i class="fa fa-check"></i><b>1.6</b> Dynamics of chromatin structure</a></li>
<li class="chapter" data-level="1.7" data-path="1-7-changes-of-genome-folding-in-evoltuion-and-disease-genomes.html"><a href="1-7-changes-of-genome-folding-in-evoltuion-and-disease-genomes.html"><i class="fa fa-check"></i><b>1.7</b> Changes of genome folding in evoltuion and disease genomes</a></li>
<li class="chapter" data-level="1.8" data-path="1-8-aims-of-this-thesis.html"><a href="1-8-aims-of-this-thesis.html"><i class="fa fa-check"></i><b>1.8</b> Aims of this thesis</a><ul>
<li class="chapter" data-level="" data-path="1-8-aims-of-this-thesis.html"><a href="1-8-aims-of-this-thesis.html#is-the-three-dimensional-folding-of-genomes-associated-with-co-regulation-of-functionally-related-genes"><i class="fa fa-check"></i>Is the three-dimensional folding of genomes associated with co-regulation of functionally related genes? </a></li>
<li class="chapter" data-level="" data-path="1-8-aims-of-this-thesis.html"><a href="1-8-aims-of-this-thesis.html#are-tads-functional-building-blocks-of-genomes-and-subjected-to-selective-pressure-during-evolution"><i class="fa fa-check"></i>Are TADs functional building blocks of genomes and subjected to selective pressure during evolution?</a></li>
<li class="chapter" data-level="" data-path="1-8-aims-of-this-thesis.html"><a href="1-8-aims-of-this-thesis.html#is-the-disruption-of-tads-by-rearrangements-also-associated-to-genetic-diseases"><i class="fa fa-check"></i>Is the disruption of TADs by rearrangements also associated to genetic diseases?</a></li>
<li class="chapter" data-level="" data-path="1-8-aims-of-this-thesis.html"><a href="1-8-aims-of-this-thesis.html#can-chromatin-looping-interactions-be-predicted-by-chip-seq-and-sequence-features"><i class="fa fa-check"></i>Can chromatin looping interactions be predicted by ChIP-seq and sequence features?</a></li>
</ul></li>
<li class="chapter" data-level="1.9" data-path="1-9-structure-of-this-thesis.html"><a href="1-9-structure-of-this-thesis.html"><i class="fa fa-check"></i><b>1.9</b> Structure of this thesis</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="2-paralog-regulation.html"><a href="2-paralog-regulation.html"><i class="fa fa-check"></i><b>2</b> Paralog genes in the 3D genome architecture</a><ul>
<li class="chapter" data-level="" data-path="abstract-1.html"><a href="abstract-1.html"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="2.1" data-path="2-1-introduction.html"><a href="2-1-introduction.html"><i class="fa fa-check"></i><b>2.1</b> Introduction</a></li>
<li class="chapter" data-level="2.2" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html"><i class="fa fa-check"></i><b>2.2</b> MATERIALS AND METHODS</a><ul>
<li class="chapter" data-level="2.2.1" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html#selection-of-pairs-of-paralog-genes"><i class="fa fa-check"></i><b>2.2.1</b> Selection of pairs of paralog genes</a></li>
<li class="chapter" data-level="2.2.2" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html#enhancers-to-gene-association"><i class="fa fa-check"></i><b>2.2.2</b> Enhancers to gene association</a></li>
<li class="chapter" data-level="2.2.3" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html#topological-associating-domains"><i class="fa fa-check"></i><b>2.2.3</b> Topological associating domains</a></li>
<li class="chapter" data-level="2.2.4" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html#hi-c-interaction-maps"><i class="fa fa-check"></i><b>2.2.4</b> Hi-C interaction maps</a></li>
<li class="chapter" data-level="2.2.5" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html#randomization"><i class="fa fa-check"></i><b>2.2.5</b> Randomization</a></li>
<li class="chapter" data-level="2.2.6" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html#statistical-tests"><i class="fa fa-check"></i><b>2.2.6</b> Statistical tests</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="2-3-results.html"><a href="2-3-results.html"><i class="fa fa-check"></i><b>2.3</b> RESULTS</a><ul>
<li class="chapter" data-level="2.3.1" data-path="2-3-results.html"><a href="2-3-results.html#distribution-of-paralog-genes-in-the-human-genome"><i class="fa fa-check"></i><b>2.3.1</b> Distribution of paralog genes in the human genome</a></li>
<li class="chapter" data-level="2.3.2" data-path="2-3-results.html"><a href="2-3-results.html#co-expression-of-paralog-gene-pairs-across-tissues"><i class="fa fa-check"></i><b>2.3.2</b> Co-expression of paralog gene pairs across tissues</a></li>
<li class="chapter" data-level="2.3.3" data-path="2-3-results.html"><a href="2-3-results.html#paralog-genes-share-enhancers"><i class="fa fa-check"></i><b>2.3.3</b> Paralog genes share enhancers</a></li>
<li class="chapter" data-level="2.3.4" data-path="2-3-results.html"><a href="2-3-results.html#co-localization-of-paralogs-in-tads"><i class="fa fa-check"></i><b>2.3.4</b> Co-localization of paralogs in TADs</a></li>
<li class="chapter" data-level="2.3.5" data-path="2-3-results.html"><a href="2-3-results.html#distal-paralog-pairs-are-enriched-for-long-range-chromatin-contacts"><i class="fa fa-check"></i><b>2.3.5</b> Distal paralog pairs are enriched for long-range chromatin contacts</a></li>
<li class="chapter" data-level="2.3.6" data-path="2-3-results.html"><a href="2-3-results.html#close-paralogs-have-fewer-contacts-than-expected"><i class="fa fa-check"></i><b>2.3.6</b> Close paralogs have fewer contacts than expected</a></li>
<li class="chapter" data-level="2.3.7" data-path="2-3-results.html"><a href="2-3-results.html#paralogs-in-mouse-and-dog-genome"><i class="fa fa-check"></i><b>2.3.7</b> Paralogs in mouse and dog genome</a></li>
<li class="chapter" data-level="2.3.8" data-path="2-3-results.html"><a href="2-3-results.html#orthologs-of-human-paralogs-show-conserved-co-localization"><i class="fa fa-check"></i><b>2.3.8</b> Orthologs of human paralogs show conserved co-localization</a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="2-4-discussion.html"><a href="2-4-discussion.html"><i class="fa fa-check"></i><b>2.4</b> DISCUSSION</a></li>
<li class="chapter" data-level="2.5" data-path="2-5-conclusion.html"><a href="2-5-conclusion.html"><i class="fa fa-check"></i><b>2.5</b> CONCLUSION</a></li>
<li class="chapter" data-level="2.6" data-path="2-6-acknowledgements.html"><a href="2-6-acknowledgements.html"><i class="fa fa-check"></i><b>2.6</b> ACKNOWLEDGEMENTS</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="3-TAD-evolution.html"><a href="3-TAD-evolution.html"><i class="fa fa-check"></i><b>3</b> Stability of TADs in evolution</a><ul>
<li class="chapter" data-level="" data-path="3-TAD-evolution.html"><a href="3-TAD-evolution.html#abstract-2"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="" data-path="3-TAD-evolution.html"><a href="3-TAD-evolution.html#keywords"><i class="fa fa-check"></i>Keywords</a></li>
<li class="chapter" data-level="3.1" data-path="3-1-introduction-1.html"><a href="3-1-introduction-1.html"><i class="fa fa-check"></i><b>3.1</b> Introduction</a></li>
<li class="chapter" data-level="3.2" data-path="3-2-results-1.html"><a href="3-2-results-1.html"><i class="fa fa-check"></i><b>3.2</b> Results</a><ul>
<li class="chapter" data-level="3.2.1" data-path="3-2-results-1.html"><a href="3-2-results-1.html#identification-of-evolutionary-rearrangement-breakpoints-from-whole-genome-alignments"><i class="fa fa-check"></i><b>3.2.1</b> Identification of evolutionary rearrangement breakpoints from whole-genome alignments</a></li>
<li class="chapter" data-level="3.2.2" data-path="3-2-results-1.html"><a href="3-2-results-1.html#rearrangement-breakpoints-are-enriched-at-tad-boundaries"><i class="fa fa-check"></i><b>3.2.2</b> Rearrangement breakpoints are enriched at TAD boundaries</a></li>
<li class="chapter" data-level="3.2.3" data-path="3-2-results-1.html"><a href="3-2-results-1.html#clusters-of-conserved-non-coding-elements-are-depleted-for-rearrangement-breakpoints"><i class="fa fa-check"></i><b>3.2.3</b> Clusters of conserved non-coding elements are depleted for rearrangement breakpoints</a></li>
<li class="chapter" data-level="3.2.4" data-path="3-2-results-1.html"><a href="3-2-results-1.html#rearranged-tads-are-associated-with-divergent-gene-expression-between-species"><i class="fa fa-check"></i><b>3.2.4</b> Rearranged TADs are associated with divergent gene expression between species</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="3-3-discussion-1.html"><a href="3-3-discussion-1.html"><i class="fa fa-check"></i><b>3.3</b> Discussion</a></li>
<li class="chapter" data-level="3.4" data-path="3-4-conclusion-1.html"><a href="3-4-conclusion-1.html"><i class="fa fa-check"></i><b>3.4</b> Conclusion</a></li>
<li class="chapter" data-level="3.5" data-path="3-5-methods.html"><a href="3-5-methods.html"><i class="fa fa-check"></i><b>3.5</b> Methods</a><ul>
<li class="chapter" data-level="3.5.1" data-path="3-5-methods.html"><a href="3-5-methods.html#rearrangement-breakpoints-from-whole-genome-alignments"><i class="fa fa-check"></i><b>3.5.1</b> Rearrangement breakpoints from whole-genome alignments</a></li>
<li class="chapter" data-level="3.5.2" data-path="3-5-methods.html"><a href="3-5-methods.html#topologically-associating-domains-and-contact-domains"><i class="fa fa-check"></i><b>3.5.2</b> Topologically associating domains and contact domains</a></li>
<li class="chapter" data-level="3.5.3" data-path="3-5-methods.html"><a href="3-5-methods.html#breakpoint-distributions-at-tads"><i class="fa fa-check"></i><b>3.5.3</b> Breakpoint distributions at TADs</a></li>
<li class="chapter" data-level="3.5.4" data-path="3-5-methods.html"><a href="3-5-methods.html#quantification-of-breakpoint-enrichment"><i class="fa fa-check"></i><b>3.5.4</b> Quantification of breakpoint enrichment</a></li>
<li class="chapter" data-level="3.5.5" data-path="3-5-methods.html"><a href="3-5-methods.html#expression-data-for-mouse-and-human-orthologs"><i class="fa fa-check"></i><b>3.5.5</b> Expression data for mouse and human orthologs</a></li>
<li class="chapter" data-level="3.5.6" data-path="3-5-methods.html"><a href="3-5-methods.html#classification-of-tads-and-genes-according-to-rearrangements-and-grbs"><i class="fa fa-check"></i><b>3.5.6</b> Classification of TADs and genes according to rearrangements and GRBs</a></li>
<li class="chapter" data-level="3.5.7" data-path="3-5-methods.html"><a href="3-5-methods.html#source-code-and-implementation-details"><i class="fa fa-check"></i><b>3.5.7</b> Source code and implementation details</a></li>
</ul></li>
<li class="chapter" data-level="3.6" data-path="3-6-declarations.html"><a href="3-6-declarations.html"><i class="fa fa-check"></i><b>3.6</b> Declarations</a><ul>
<li class="chapter" data-level="3.6.1" data-path="3-6-declarations.html"><a href="3-6-declarations.html#ethics-approval-and-consent-to-participate"><i class="fa fa-check"></i><b>3.6.1</b> Ethics approval and consent to participate</a></li>
<li class="chapter" data-level="3.6.2" data-path="3-6-declarations.html"><a href="3-6-declarations.html#consent-for-publication"><i class="fa fa-check"></i><b>3.6.2</b> Consent for publication</a></li>
<li class="chapter" data-level="3.6.3" data-path="3-6-declarations.html"><a href="3-6-declarations.html#availability-of-data-and-material"><i class="fa fa-check"></i><b>3.6.3</b> Availability of data and material</a></li>
<li class="chapter" data-level="3.6.4" data-path="3-6-declarations.html"><a href="3-6-declarations.html#competing-interests"><i class="fa fa-check"></i><b>3.6.4</b> Competing interests</a></li>
<li class="chapter" data-level="3.6.5" data-path="3-6-declarations.html"><a href="3-6-declarations.html#funding"><i class="fa fa-check"></i><b>3.6.5</b> Funding</a></li>
<li class="chapter" data-level="3.6.6" data-path="3-6-declarations.html"><a href="3-6-declarations.html#authors-contributions"><i class="fa fa-check"></i><b>3.6.6</b> Authors‘ contributions</a></li>
<li class="chapter" data-level="3.6.7" data-path="3-6-declarations.html"><a href="3-6-declarations.html#acknowledgments"><i class="fa fa-check"></i><b>3.6.7</b> Acknowledgments</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="4-position-effect.html"><a href="4-position-effect.html"><i class="fa fa-check"></i><b>4</b> Position effects of rearrangements in disease genomes</a></li>
<li class="chapter" data-level="5" data-path="5-loop-prediction.html"><a href="5-loop-prediction.html"><i class="fa fa-check"></i><b>5</b> Prediction of chromatin looping interactions</a></li>
<li class="chapter" data-level="6" data-path="6-thesis-discussion.html"><a href="6-thesis-discussion.html"><i class="fa fa-check"></i><b>6</b> Discussion</a><ul>
<li class="chapter" data-level="6.1" data-path="6-1-discussion-on-tad-evolution.html"><a href="6-1-discussion-on-tad-evolution.html"><i class="fa fa-check"></i><b>6.1</b> Discussion on TAD evolution</a></li>
<li class="chapter" data-level="6.2" data-path="6-2-further-directions.html"><a href="6-2-further-directions.html"><i class="fa fa-check"></i><b>6.2</b> Further directions</a></li>
<li class="chapter" data-level="6.3" data-path="6-3-conclusions.html"><a href="6-3-conclusions.html"><i class="fa fa-check"></i><b>6.3</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="appendix"><span><b>Appendix</b></span></li>
<li class="chapter" data-level="A" data-path="A-supporting-information-co-regulation-of-paralog-genes-in-the-three-dimensional-chromatin-architecture.html"><a href="A-supporting-information-co-regulation-of-paralog-genes-in-the-three-dimensional-chromatin-architecture.html"><i class="fa fa-check"></i><b>A</b> Supporting Information: Co-regulation of paralog genes in the three-dimensional chromatin architecture</a></li>
<li class="chapter" data-level="B" data-path="B-supplementary-data-evolutionary-stability-of-topologically-associating-domains-is-associated-with-conserved-gene-regulation.html"><a href="B-supplementary-data-evolutionary-stability-of-topologically-associating-domains-is-associated-with-conserved-gene-regulation.html"><i class="fa fa-check"></i><b>B</b> Supplementary Data: Evolutionary stability of topologically associating domains is associated with conserved gene regulation</a><ul>
<li class="chapter" data-level="B.1" data-path="B-1-TadEvoSupTab.html"><a href="B-1-TadEvoSupTab.html"><i class="fa fa-check"></i><b>B.1</b> Supplementary Tables</a></li>
<li class="chapter" data-level="B.2" data-path="B-2-supplementary-figures.html"><a href="B-2-supplementary-figures.html"><i class="fa fa-check"></i><b>B.2</b> Supplementary Figures</a></li>
<li class="chapter" data-level="B.3" data-path="B-3-contribution-to-individual-publications.html"><a href="B-3-contribution-to-individual-publications.html"><i class="fa fa-check"></i><b>B.3</b> Contribution to individual publications</a></li>
</ul></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Build with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Genome folding in evolution and disease</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="position-effect" class="section level1">
<h1><span class="header-section-number">Chapter 4</span> Position effects of rearrangements in disease genomes</h1>
<p><strong>Computational Prediction of Position Effects of Apparently Balanced Human Chromosome Rearrangements</strong></p>
<p>Cinthya J. Zepeda-Mendoza<sup>1,3,*</sup>, Jonas Ibn-Salem<sup>4,*</sup>, Tammy Kammin<sup>1</sup>, David J. Harris<sup>3,5</sup>, Debra Rita<sup>6</sup>, Karen W. Gripp<sup>7</sup>, Jennifer J. MacKenzie<sup>8</sup>, Andrea Gropman<sup>9</sup>, Brett Graham<sup>10</sup>, Ranad Shaheen<sup>11</sup>, Fowzan S. Alkuraya<sup>11,12</sup>, Campbell K. Brasington<sup>13</sup>, Edward J. Spence<sup>13</sup>, Diane Masser-Frye<sup>14</sup>, Lynne M. Bird<sup>14,15</sup>, Erica Spiegel<sup>16</sup>, Rebecca L. Sparkes<sup>17</sup>, Zehra Ordulu<sup>18</sup>, Michael E. Talkowski<sup>18-24</sup>, Miguel A. Andrade-Navarro<sup>4</sup>, Peter N. Robinson<sup>25</sup>, Cynthia C. Morton<sup>1-3,23,26†</sup></p>
<p>Departments of <sup>1</sup>Obstetrics, Gynecology and Reproductive Biology and <sup>2</sup>Pathology, Brigham and Women’s Hospital, Boston, MA 02115, USA</p>
<p><sup>3</sup>Harvard Medical School, Boston, MA 02115, USA</p>
<p><sup>4</sup>Johannes Gutenberg University, Mainz 55122, Germany</p>
<p><sup>5</sup>Boston Children’s Hospital, Boston, MA 02115, USA</p>
<p><sup>6</sup>ACL laboratories. Cytogenetics Lab, Rosemont, IL 60018, USA</p>
<p><sup>7</sup>Nemours Alfred I. DuPont Hospital for Children; Wilmington, DE 19803, USA</p>
<p><sup>8</sup>Department of Pediatrics, McMaster University, Hamilton, ON L8S 4L8, Canada</p>
<p><sup>9</sup>Children’s National Medical Center, Washington, DC 20010, USA</p>
<p><sup>10</sup>Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA</p>
<p><sup>11</sup>Department of Genetics, King Faisal Specialist Hospital and Research Center, Riyadh 12713, Saudi Arabia</p>
<p><sup>12</sup>Department of Anatomy and Cell Biology, College of Medicine, Alfaisal University, Riyadh 11533, Saudi Arabia</p>
<p><sup>13</sup>Clinical Genetics Division, Department of Pediatrics. Levine Children’s Hospital at Carolinas Medical Center. Charlotte, NC 28203, USA</p>
<p><sup>14</sup>Genetics and Dysmorphology, Rady Children’s Hospital San Diego, San Diego, CA 92123, USA</p>
<p><sup>15</sup>University of California, San Diego, La Jolla, CA 92093, USA</p>
<p><sup>16</sup>Maternal Fetal Medicine, Columbia University Medical Center, New York, NY 10032, USA</p>
<p><sup>17</sup>Department of Medical Genetics, Cumming School of Medicine, University of Calgary, Calgary, AB T2N 1N4, Canada</p>
<p>Departments of <sup>18</sup>Pathology, <sup>19</sup>Neurology and <sup>20</sup>Psychiatry and <sup>21</sup>Center for Genomic Medicine and, Massachusetts General Hospital, Boston, MA 02114, USA</p>
<p><sup>22</sup>Department of Neurology, Harvard Medical School, Boston, MA 02115, USA</p>
<p><sup>23</sup>Program in Medical and Population Genetics and <sup>24</sup>Stanley Center for Psychiatric Genetics, Broad Institute of Harvard and MIT, Cambridge, MA 02142, USA</p>
<p><sup>25</sup>The Jackson Laboratory for Genomic Medicine, Farmington, CT 06032, USA</p>
<p><sup>26</sup>Division of Evolution and Genomic Science, School of Biological Sciences, Manchester Academic Health Science Centre, Manchester M13 9NT, UK</p>
<p>*These authors contributed equally to this work</p>
<p><sup>†</sup>Corresponding author: Cynthia C. Morton, Ph.D. Brigham and Women’s Hospital, New Research Building, Rm. 160D, 77 Avenue Louis Pasteur. Boston, MA 02115. Tel: 617-525-4535; Fax: 617-525-4533. Email: <a href="mailto:cmorton@partners.org">cmorton@partners.org</a></p>
<p><strong>Abstract</strong></p>
<p>Interpretation of variants of uncertain significance, especially chromosome rearrangements in non-coding regions of the human genome, remains one of the biggest challenges in modern molecular diagnosis. To improve our understanding and interpretation of such variants, we used high-resolution 3-dimensional chromosome structure data and transcriptional regulatory information to predict position effects and their association with pathogenic phenotypes in 17 subjects with apparently balanced chromosome abnormalities. We find that the rearrangements predict disruption of long-range chromatin interactions between several enhancers and genes whose annotated clinical features are strongly associated with the subjects’ phenotypes. We confirm gene expression changes for a couple of candidate genes to exemplify the utility of our position effect analysis. These results highlight the important interplay between chromosome structure and disease, and demonstrate the need to utilize chromatin conformation data for the prediction of position effects in the clinical interpretation of cases of non-coding chromosome rearrangements.</p>
<p><strong>Introduction</strong></p>
<p>The importance of the integrity of chromosome structure and its association with human disease is one of the oldest and most studied topics in clinical genetics. As early as 1959, cytogenetic studies in humans linked specific genetic or genomic disorders and intellectual disability syndromes to changes in chromosomal ploidy, translocations, and DNA duplications and deletions.<span class="citation">(<span class="citeproc-not-found" data-reference-id="Conrad2010"><strong>???</strong></span>; Conrad et al., <a href="#ref-Conrad2006">2006</a>; Hinds et al., <a href="#ref-Hinds2006">2006</a>; Iafrate et al., <a href="#ref-Iafrate2004">2004</a>; Sebat et al., <a href="#ref-Sebat2004">2004</a>)</span> The discovery of copy-number variants (CNVs) by microarray and sequencing technologies expanded the catalogue of genetic variation between individuals to test such associations at higher resolution.<span class="citation">(<span class="citeproc-not-found" data-reference-id="McCarroll2006"><strong>???</strong></span>, <span class="citeproc-no-output"><strong>???</strong></span>, <span class="citeproc-no-output"><strong>???</strong></span>, <span class="citeproc-no-output"><strong>???</strong></span>, <span class="citeproc-no-output"><strong>???</strong></span>, <span class="citeproc-no-output"><strong>???</strong></span>, <span class="citeproc-no-output"><strong>???</strong></span> HapMap2010; Conrad et al., <a href="#ref-Conrad2006">2006</a>; Hinds et al., <a href="#ref-Hinds2006">2006</a>; Iafrate et al., <a href="#ref-Iafrate2004">2004</a>; Korbel et al., <a href="#ref-Korbel2007">2007</a>; Sebat et al., <a href="#ref-Sebat2004">2004</a>)</span> Over the years, analysis of disease-related structural rearrangements has illuminated genes that are mutated in various human developmental disorders.<span class="citation">(Nambiar and Raghavan, <a href="#ref-Nambiar2011">2011</a>; Theisen and Shaffer, <a href="#ref-Theisen2010">2010</a>; Zhang et al., <a href="#ref-Zhang2009">2009</a>)</span> Such chromosome aberrations can directly disrupt gene sequences, affect gene dosage, generate gene fusions, unmask recessive alleles, reveal imprinted genes, or result in alterations of gene expression through additional mechanisms such as position effects.<span class="citation">(Zhang et al., <a href="#ref-Zhang2009">2009</a>)</span> The latter is particularly important for the study of apparently balanced chromosome abnormalities (BCAs), such as translocations and inversions, often found outside of the hypothesized disease-causing genes (reviewed in <span class="citation">(Kleinjan and Van Heyningen, <a href="#ref-Kleinjan2005">2005</a>)</span>).</p>
<p>Position effects were first identified in <em>Drosophila melanogaster</em>, where chromosomal inversions placing <em>white+</em> near centric heterochromatin caused mosaic red/white eye patterns.<span class="citation">(Weiler and Wakimoto, <a href="#ref-Weiler1995">1995</a>)</span> In humans, BCAs can induce position effects through disruption of a gene’s long-range transcriptional control (<em>i.e</em>., enhancer-promoter interactions, insulator influence, etc.), or its placement in regions with different local chromatin environments as observed in the classical <em>Drosophila</em> position effect variegation (reviewed in <span class="citation">(Kleinjan and Van Heyningen, <a href="#ref-Kleinjan2005">2005</a>; Spielmann and Mundlos, <a href="#ref-Spielmann2016">2016</a>; Zhang and Wolynes, <a href="#ref-Zhang2015">2015</a>)</span>). Examples of position effect genes include paired box gene 6 (<em>PAX6</em> [MIM: 607108])<em>,</em> for which downstream chromosome translocations affect its <em>cis</em>-regulatory control and produce aniridia (AN [MIM: 106210]);<span class="citation">(Fantes et al., <a href="#ref-Fantes1995">1995</a>; Kleinjan et al., <a href="#ref-Kleinjan2001">2001</a>)</span> twist family bHLH transcription factor 1 (<em>TWIST1</em> [MIM: 601622]), where downstream translocations and inversions are associated with Saethre-Chotzen syndrome (SCS [MIM: 101400]);<span class="citation">(Cai et al., <a href="#ref-Cai2003">2003</a>)</span> paired like homeodomain 2 (<em>PITX2</em> [MIM: 601542]) for which translocations are associated with Axenfeld-Rieger syndrome type 1 (RIEG1 [MIM: 180500]);<span class="citation">(Flomen et al., <a href="#ref-Flomen1998">1998</a>; Trembath et al., <a href="#ref-Trembath2004">2004</a>)</span> SRY-box 9 (<em>SOX9</em> [MIM: 608160]), where translocation breakpoints located up to 900 Kilobases (Kb) upstream and 1.3 Megabases (Mb) downstream are associated with campomelic dysplasia (CMPD [MIM: 114290]),<span class="citation">(Velagaleti et al., <a href="#ref-Velagaleti2005">2005</a>)</span> in addition to several others.<span class="citation">(Kleinjan and Heyningen, <a href="#ref-Kleinjan1998">1998</a>; Kleinjan and Van Heyningen, <a href="#ref-Kleinjan2005">2005</a>; Lupski and Stankiewicz, <a href="#ref-Lupski2005">2005</a>)</span></p>
<p>The availability of genome sequencing in the clinical setting has generated a need for rapid prediction and interpretation of structural variants, especially those pertaining to <em>de novo</em> non-coding rearrangements in individual subjects. With the development and subsequent branching of the chromosome conformation capture (3C) technique (<span class="citation">(Dekker et al., <a href="#ref-Dekker2002">2002</a>)</span>, reviewed in <span class="citation">(Wit and Laat, <a href="#ref-DeWit2012">2012</a>)</span>), regulatory issues such as alteration of long-range transcriptional control and position effects can now be predicted in terms of chromosome organization. The high resolution view of chromosome architecture in diverse human cell lines and tissues<span class="citation">(Dixon et al., <a href="#ref-Dixon2012">2012</a>; Fullwood et al., <a href="#ref-Fullwood2009">2009</a>; Lieberman-Aiden et al., <a href="#ref-Lieberman-Aiden2009">2009</a>; Mifsud et al., <a href="#ref-Mifsud2015">2015</a>; Phillips-Cremins et al., <a href="#ref-Phillips-Cremins2013">2013</a>; Rao et al., <a href="#ref-Rao2014">2014</a>; Sanyal et al., <a href="#ref-Sanyal2012">2012</a>)</span> has allowed molecular assessment of the disruption of regulatory chromatin contacts by pathogenic structural variants and single nucleotide changes; examples include the study of limb malformations,<span class="citation">(Lupiáñez et al., <a href="#ref-Lupianez2015">2015</a>)</span> leukemia,<span class="citation">(Gröschel et al., <a href="#ref-Groschel2014">2014</a>)</span> and obesity,<span class="citation">(Claussnitzer et al., <a href="#ref-Claussnitzer2015">2015</a>)</span> among others.<span class="citation">(Giorgio et al., <a href="#ref-Giorgio2015">2015</a>; Ibn-Salem et al., <a href="#ref-Ibn-Salem2014">2014</a>; Oldridge et al., <a href="#ref-Oldridge2015">2015</a>; Roussos et al., <a href="#ref-Roussos2014">2014</a>; Visser et al., <a href="#ref-Visser2012">2012</a>)</span> These examples underscore the importance of chromatin interactions in quantitative and temporal control of gene expression, which can greatly enhance our power to predict pathologic consequences.</p>
<p>To test the feasibility of prediction and clinical interpretation of position effects of non-coding chromosome rearrangements, we analyzed 17 subjects from the Developmental Gene Anatomy Project (DGAP)<span class="citation">(Higgins et al., <a href="#ref-Higgins2008">2008</a>; Kim and Marcotte, <a href="#ref-Kim2008">2008</a>; Ligon et al., <a href="#ref-Ligon2005">2005</a>; Lu et al., <a href="#ref-Lu2007">2007</a>)</span> with <em>de novo</em> non-coding BCAs classified as variants of uncertain significance (VUS). Using publicly available chromatin contact information, annotated and predicted regulatory elements, and correlation between phenotypes observed in DGAP subjects and those associated with neighboring genes, we reliably predicted candidate genes exhibiting mis-regulated expression in DGAP-derived lymphoblastoid cell lines (LCLs). These results suggest that many VUS are likely to be further interpretable via long-range effects, and warrant their routine assessment and integration in clinical diagnosis.</p>
<p><strong>Materials and Methods</strong></p>
<p><strong>Selection of subjects with apparently balanced chromosome abnormalities</strong></p>
<p>BCA breakpoints and clinical data were obtained from DGAP cases for which whole-genome sequencing was performed using a previously described large-insert jumping library approach.<span class="citation">(Higgins et al., <a href="#ref-Higgins2008">2008</a>; Kim and Marcotte, <a href="#ref-Kim2008">2008</a>; Ligon et al., <a href="#ref-Ligon2005">2005</a>; Lu et al., <a href="#ref-Lu2007">2007</a>; Redin et al., <a href="#ref-Redin2017">2017</a>)</span> A total of 151 cases were filtered to select only subjects whose translocation or inversion breakpoints fall within intergenic regions (GRCh37) and did not overlap known long intergenic non-coding RNAs (lincRNAs) or pseudogenes, as these elements have been shown to exert functional roles (reviewed in <span class="citation">(Quinn and Chang, <a href="#ref-Quinn2016">2016</a>)</span> <span class="citation">(Muro and Andrade-Navarro, <a href="#ref-Muro2010">2010</a>; Pink et al., <a href="#ref-Pink2011">2011</a>)</span>). Of 151 DGAP subjects, only 17 fulfilled our selection criteria, 12 of whom had available and reportedly normal clinical array results, suggesting lack of large duplications or deletions.</p>
<p><strong>Clinical descriptions of DGAP cases</strong></p>
<p>The clinical presentation of the 17 subjects varied, ranging from developmental delay to neurological conditions, offering the opportunity to assess long-range position effects in different phenotypes. Subjects’ karyotypes are presented in the main text using the International System for Human Cytogenetic Nomenclature (ISCN2016) (Table 1). Detailed case descriptions are included in the Supplemental Note: Case Reports, as well as a nomenclature developed to describe chromosome rearrangements using next-generation sequencing.<span class="citation">(Ordulu et al., <a href="#ref-Ordulu2014">2014</a>)</span> Reported ages of DGAP subjects are from time of enrollment. All reported genomic coordinates use GRCh37.</p>
<p><strong>Analysis of genes bordering the rearrangement breakpoints</strong></p>
<p>The presence of annotated genes or pseudogenes and lincRNAs was assessed in windows of ±3 and ±1 Mb neighboring each subject’s translocation and inversion breakpoints, and within reported H1-hESC topologically associated domains (TADs)<span class="citation">(Dixon et al., <a href="#ref-Dixon2012">2012</a>)</span> where the breakpoints were located. The gene annotation file was obtained from Ensembl GRCh37 archive,<span class="citation">(Flicek et al., <a href="#ref-Flicek2014">2014</a>)</span> and we used the Human Body Map lincRNAs catalog.<span class="citation">(Cabili et al., <a href="#ref-Cabili2011">2011</a>)</span> Haploinsufficiency (HI) and triplosensitivity scores were assigned using Huang <em>et al</em>., 2010<span class="citation">(Huang et al., <a href="#ref-Huang2010">2010</a>)</span> and version hg19 of ClinGen<span class="citation">(Rehm et al., <a href="#ref-Rehm2015">2015</a>)</span> data downloaded on 9/20/2016.</p>
<p><strong>Assessment of disrupted functional elements and chromatin interactions bordering rearrangement breakpoints</strong></p>
<p>The disruption of regulatory elements such as enhancers, promoters, locus control regions, and insulators can lead to disease-related gene expression changes; DNase I hypersensitive (DHS) sites have been used as markers for the identification of such elements.<span class="citation">(Thurman et al., <a href="#ref-Thurman2012">2012</a>)</span> In addition, the alteration of TAD boundaries has been previously shown to cause a rewiring of enhancers with pathological consequences;<span class="citation">(Giorgio et al., <a href="#ref-Giorgio2015">2015</a>; Lupiáñez et al., <a href="#ref-Lupianez2015">2015</a>; Narendra et al., <a href="#ref-Narendra2015">2015</a>)</span> CCCTC-Binding Factor (CTCF) binding sites have been found to be enriched in TAD boundaries,<span class="citation">(Dixon et al., <a href="#ref-Dixon2012">2012</a>)</span> and several mutations of boundary-defining sites have been associated with cancer.<span class="citation">(Flavahan et al., <a href="#ref-Flavahan2016">2016</a>; Hnisz et al., <a href="#ref-Hnisz2016">2016</a>)</span> Based on these observations, we assessed the number of regulatory elements that were potentially disrupted by the analyzed DGAP breakpoints. We compared the breakpoint positions of the selected DGAP subjects against data corresponding to CTCF binding sites, DHS sites, and chromatin segmentation classifications (Broad ChromHMM) derived from a lymphoblastoid cell line (GM12878) and human stem cells (H1-hESC), obtained from the Encyclopedia of DNA Elements (ENCODE) project<span class="citation">(Dunham et al., <a href="#ref-Dunham2012">2012</a>)</span> and accessed through the University of California Santa Cruz Genome Browser.<span class="citation">(Kent et al., <a href="#ref-Kent2002">2002</a>)</span> Enhancer positions were additionally obtained from Andersson <em>et al</em>., 2014<span class="citation">(Andersson et al., <a href="#ref-Andersson2014">2014</a>)</span> for tissue and primary cells, and the VISTA Enhancer browser, human version hg19.<span class="citation">(Visel et al., <a href="#ref-Visel2007">2007</a>)</span> Finally, lists of transcription factor (TF) binding sites and gene promoters were obtained from the Ensembl database human version GRCh37.<span class="citation">(Flicek et al., <a href="#ref-Flicek2014">2014</a>)</span> Hi-C interaction data and TAD positions for H1-hESC, GM06990, and IMR90 at 20 Kb, 40 Kb, 100 Kb, and 1 Mb resolution were obtained from Dixon <em>et al</em>., 2012<span class="citation">(Dixon et al., <a href="#ref-Dixon2012">2012</a>)</span> and the WashU EpiGenome Browser.<span class="citation">(Zhou and Wang, <a href="#ref-Zhou2012">2012</a>)</span> A high-resolution dataset of chromatin loops and domains was obtained from Rao <em>et al.</em>, 2014 for IMR90 and GM12878 cells.<span class="citation">(Rao et al., <a href="#ref-Rao2014">2014</a>)</span> Lastly, distal DHS/enhancer–promoter connections<span class="citation">(Thurman et al., <a href="#ref-Thurman2012">2012</a>)</span> were used to assess disrupted predicted cis-regulatory interactions by the BCAs. Genomic overlaps between the rearrangement breakpoints, functional elements and disrupted chromatin interactions were calculated using custom Perl scripts, the BEDtools suite<span class="citation">(Quinlan and Hall, <a href="#ref-Quinlan2010">2010</a>)</span> and the genomic association tester (GAT) tool.<span class="citation">(Heger et al., <a href="#ref-Heger2013">2013</a>)</span></p>
<p><strong>Ontological analysis of genes neighboring breakpoints</strong></p>
<p>Phenotype similarity between potential position effect genes and DGAP cases was calculated by converting the phenotypes of the 17 subjects to Human Phenotype Ontology (HPO)<span class="citation">(Köhler et al., <a href="#ref-Kohler2014">2014</a>)</span> terms and calculating their phenomatch score as described in Ibn-Salem <em>et al</em>., 2014.<span class="citation">(Ibn-Salem et al., <a href="#ref-Ibn-Salem2014">2014</a>)</span> The phenomatch score quantiﬁes the information content of the most speciﬁc HPO term that is part of or a common ancestor (more general term) of a set of phenotypes. Our set of phenotypes is constituted by the HPO terms associated to DGAP cases and the ones annotated to candidate position effect genes within windows of ±3 and ±1 Mb of sequence in proximity to the breakpoints. We used two background models to assess signiﬁcance of this similarity. The ﬁrst is based on randomly permuting the associations of phenotypes to genes; to this effect, the phenotype-gene associations are shuffled 100 times randomly and the similarity of these random phenotypes to the studied case clinical findings is calculated. The second background control is based on shifting the breakpoint location along the chromosome; each breakpoint is shifted by -9, -6, -3, +3, +6, and +9 Mb and the similarity of genes in proximity to the shifted breakpoints is computed.</p>
<p><strong>Quantitative real-time PCR</strong></p>
<p>LCLs derived from DGAP236-02m, DGAP244-02m and DGAP245-02m were used as karyotypically normal male controls. These are karyotypically normal fathers of enrolled DGAP cases with no history of disease. LCL 17402 (DGAP163) was used to test differential gene expression for SOS Ras/Rac guanine nucleotide exchange factor 1 (<em>SOS1</em> [MIM: 182530]), and LCL 18060 (DGAP176) was used to test midline 2 (<em>MID2</em> [MIM: 300204]), p21 (RAC1) activated kinase 3 (<em>PAK3</em> [MIM: 300142]), and POU class 3 homeobox 4 (<em>POU3F4</em> [MIM: 300039]) expression using quantitative polymerase chain reaction (qPCR). Glucuronidase beta (<em>GUSB</em> [MIM: 611499]) was used as a housekeeping control. qPCR experiments were performed by the Harvard Biopolymers Facility using TaqMan probes Hs00264887_s1 (<em>POU3F4</em>), Hs00201978_m1 (<em>MID2</em>), Hs00176828_m1 (<em>PAK3</em>), Hs00893134_m1 (<em>SOS1</em>), and Hs00939627_m1 (<em>GUSB</em>). Data were analyzed using the ΔCT method.</p>
<p><strong>Assessment of DGAP breakpoints overlapping with non-coding structural variants in public databases</strong></p>
<p>To find similar non-coding structural rearrangement subjects and compare their annotated clinical phenotypes to those observed in DGAP cases, we searched the DatabasE of genomiC varIation and Phenotype in Humans using Ensembl Resources (DECIPHER)<span class="citation">(Firth et al., <a href="#ref-Firth2009">2009</a>)</span> version 2015-07-13, as well as the dbVar database from the National Center for Biotechnology Information (NCBI) Variation Viewer 1.5.<span class="citation">(Lappalainen et al., <a href="#ref-Lappalainen2013">2012</a>)</span> Both databases are comprehensive community-supported repositories of clinical cases with novel and extremely rare genomic variants.</p>
<p><strong>Results</strong></p>
<p><strong>Genomic characterization of non-coding breakpoints</strong></p>
<p>To study the structural and evolutionary context of BCAs and their impact on nuclear architecture and gene expression, we used data generated by DGAP,<span class="citation">(Higgins et al., <a href="#ref-Higgins2008">2008</a>; Kim and Marcotte, <a href="#ref-Kim2008">2008</a>; Ligon et al., <a href="#ref-Ligon2005">2005</a>; Lu et al., <a href="#ref-Lu2007">2007</a>)</span> the largest collection of sequenced balanced chromosome rearrangements from individuals with abnormal developmental and cognitive phenotypes, many of which have yet to be investigated in detail. Each studied DGAP BCA has two breakpoint positions (as two distinct chromosome regions are involved in their generation), which we labeled with the DGAP#_A and DGAP#_B identifiers. We filtered DGAP data to select cases with both breakpoints in non-coding regions only, and excluding lincRNAs and pseudogenes; a total of 17 cases fulfilled our criteria, 15 translocations and 2 inversions (Figure 1 and Table S1). These subjects are phenotypically distinct, and most of them presented with congenital developmental and neurological conditions not recognized as a known syndrome or genomic disorder (see clinical descriptions in Supplemental Note: Case Reports).</p>
<p>Further analysis revealed that BCA breakpoints were significantly depleted for overlapping annotated promoters or transcription factor (TF) binding sites (GAT TF p=0.0003, promoter p=0.0001, Table S2,3). Only one breakpoint (DGAP249_B) overlapped a ChromHMM enhancer in GM12878 cells (Table 1); the others had no overlap with annotated or predicted enhancers in the analyzed datasets, and this depletion was significant for VISTA (GAT p=0.0364) and Hi-ESC (GAT p=0.0036) but not for the annotated tissue and primary cell enhancers from Andersson <em>et al</em>., 2014<span class="citation">(Andersson et al., <a href="#ref-Andersson2014">2014</a>)</span> (Table S4). Eight breakpoints overlapped cell-type specific DHS sites (Table 1 and Table S5); these corresponded to DGAP cases 017, 176, 249, 275, 288 and 322; of these, DGAP176 and DGAP275 overlapped DHS sites at both BCA breakpoint sites. In addition, three DGAP cases overlapped CTCF binding sites in H1-hESC (DGAP cases 111, 176, and 287) and none in GM12878 cells (Table 1 and Table S6). Except for two cases in H1-hESC (DGAP17 and DGAP176), and four cases in GM12878 (DGAP 017, 126, 163 and 176), all rearrangements fall within ChromHMM repressed chromatin regions, but this association was not significant (GAT p= 0.40 for GM12878 and p= 0.15 for H1-hESC, Table S2F). Interestingly, 22 of the 34 breakpoints (~65%) overlap repeated elements at a significant level (GAT p=0.0002, Table S8), which may indicate a non-allelic homologous recombination process in their generation.<span class="citation">(Cardoso et al., <a href="#ref-Cardoso2016">2016</a>; Gu et al., <a href="#ref-Gu2008">2008</a>)</span></p>
<p>Noticeably, either one or two breakpoints from all the non-coding DGAP BCAs fall within previously reported TADs in H1-hESC and IMR90 cell lines (Table 1 and Table S9).<span class="citation">(Dixon et al., <a href="#ref-Dixon2012">2012</a>)</span> However, this overlap was not significant for both cell lines (GAT H1-ESC p=0.0537 and IMR90 p=0.28). We found that the breakpoints disrupt dozens, hundreds, or even thousands of chromatin contacts when assessed at the 20 and 40 Kb resolution in Hi-C data of H1-hESC and IMR90 cells, as well as chromatin contacts at 100 Kb and 1 Mb resolution in GM06990 cells (Table S11). Breakpoint DGAP111_A had a consistent absence of disrupted chromatin contacts, which is expected as it overlaps a repetitive satellite region so no chromatin contacts could be mapped to the segment (Table S9 and Table S11). With the availability of higher resolution data, it is possible to detect whether BCA breakpoints disrupt smaller chromatin domains and loops not detected in previous studies. When analyzing high resolution IMR90 and GM12878 Hi-C data,<span class="citation">(Rao et al., <a href="#ref-Rao2014">2014</a>)</span> we discovered that 32 out of 34 breakpoints are contained within GM12878 sub-compartments (Table 1 and Table S10); interestingly, 28 of these are classified as members of the B compartment, which is less gene dense and less expressed compared to the A compartment. On the other hand, 18 and 24 breakpoints are contained within GM12878 and IMR90 arrowhead domains, respectively (Table S10), which are regions of enhanced contact frequency that tile the diagonal of each chromatin contact matrix. In addition, the breakpoints disrupt several significant short and long-range chromatin interactions in the GM12878 Hi-C data (Table S12).</p>
<p>Overall, the observation of breakpoint-associated DHS sites suggests the alteration of underlying regulatory elements with potential pathogenic outcomes, while the predicted extensive disruption of chromatin contacts and the alteration of TAD boundaries by the BCAs may affect long-range regulatory interactions of neighboring genes (see Discussion).</p>
<p><strong>Identification of genes with potential position effects</strong></p>
<p>To identify genes which could be generating the complex DGAP phenotypes via position effects from chromosome rearrangements, we analyzed all annotated genes within windows of ±3 and ±1 Mb proximal and distal to the breakpoints, and within the BCA-containing H1-hESC reported TAD positions. A total of 3081 genes were contained within the ±3 and ±1 Mb windows for all cases; 106 of these genes (~3.4%) have an HI score of &lt;10%, which is a predictor of haploinsufficiency,<span class="citation">(Huang et al., <a href="#ref-Huang2010">2010</a>)</span> and 55 and two genes have ClinGen emerging evidence suggesting that dosage haplo/triplo-sensitivity, respectively, is associated with clinical phenotype (Table S15).</p>
<p>To further refine our search for genes which may exhibit position effects, we performed an unbiased correlation between DGAP case phenotypes and the clinical traits associated with genes bordering each breakpoint. To this end, we used the HPO dataset,<span class="citation">(Köhler et al., <a href="#ref-Kohler2014">2014</a>)</span> which provides a standardized vocabulary of phenotypic abnormalities encountered in human disease, and currently contains ~11,000 terms and over 115,000 annotations to hereditary diseases. We translated DGAP clinical features to HPO terms (Table S16), and calculated phenotype similarity between DGAP cases and neighboring genes using the phenomatch score.<span class="citation">(Ibn-Salem et al., <a href="#ref-Ibn-Salem2014">2014</a>)</span> The phenomatch score distinguishes between general and very specific phenotypic descriptions by quantifying the information content of the most speciﬁc HPO terms that are common to, or a common ancestor of, the DGAP case and neighboring gene phenotypes. The similarity significance is then calculated based on randomly permuting the associations of phenotypes to genes, and in shifting the DGAP translocation and inversion breakpoint positions along the chromosome. We obtained phenomatch scores ranging from 0.003 to 91.48 for 179 genes within the ±3 and ±1 Mb windows, as well as within the TAD positions (Table S15).</p>
<p>In addition to dosage sensitivity and phenotypic similarity information, we complemented our analysis with assessment of enhancer-promoter interactions to make our candidate selection more specific. A typical mechanism by which chromosome rearrangements cause position effects is through disruptions in the association of genes with their regulatory regions.<span class="citation">(Kleinjan and Heyningen, <a href="#ref-Kleinjan1998">1998</a>; Kleinjan and Van Heyningen, <a href="#ref-Kleinjan2005">2005</a>)</span> We therefore reasoned that genes and enhancers included in predicted enhancer-promoter interactions would be strong position effect candidates. We used the ENCODE distal DHS/enhancer–promoter connections<span class="citation">(Thurman et al., <a href="#ref-Thurman2012">2012</a>)</span> to assess disrupted predicted <em>cis</em>-regulatory interactions by the DGAP breakpoints within a ±500 Kb window. The analysis revealed 193 genes that were separated from their predicted candidate enhancers, potentially altering gene expression (Table S13). A total of 133 candidate genes were separated from &lt;10 of their predicted enhancers, while 60 genes were separated from their predicted interactions with 10 or up to 91 enhancers (Table S14).</p>
<p>For the 17 analyzed DGAP BCAs, there are a total of 645 genes with either evidence of dosage sensitivity, disrupted enhancer-promoter interactions, or significant phenotypic similarity. This represents ~21% of the genes contained within the ±3 Mb windows, clearly an undesirable number for timely clinical interpretation and functional analyses. To filter the most promising candidates, we ranked them using their reported dosage sensitivity, disrupted regulatory interactions, and by selecting a phenomatch cut-off value capable of detecting pathogenic and likely pathogenic genes in 57 published DGAP cases from Redin <em>et al</em>., 2017.<span class="citation">(Redin et al., <a href="#ref-Redin2017">2017</a>)</span> By taking into consideration the top quartile values of the reported phenomatch scores per case and adding up their dosage sensitivity and disrupted regulatory interaction data, we consistently ranked the reported pathogenic and likely pathogenic genes in the upper decile for 52 out of the 57 control DGAP cases (~91%) when considering candidates within the TAD and ±1 Mb analysis windows (Table S17). 32 of these genes were the top-ranking candidates in their corresponding DGAP case, while 19 of them were positioned in the second-tier rank. Only five genes could not be found in the top decile ranking positions as they had one or no lines of evidence supporting their inclusion.</p>
<p>Applying this ranking strategy to the 17 non-coding BCAs, we predict 16 top-ranking candidates for 11 DGAP cases and 102 second-tier candidates for the 17 analyzed DGAP cases within ±1 Mb analysis windows (Table 1 and Table S15). This is a significant reduction compared to the initial 645 possible candidates (~3.8% of the neighboring genes in the ±3 Mb windows considering top and second-tier candidates, and only 0.05% considering top candidates only). Of note, only nine of the 16 top-ranking candidates are included within the same TAD as the BCA breakpoint (H1-hESC TADs from <span class="citation">(Dixon et al., <a href="#ref-Dixon2012">2012</a>)</span>), while the rest are located farther away. Nine top-ranking genes had an HI score &lt;10%,<span class="citation">(Huang et al., <a href="#ref-Huang2010">2010</a>)</span> while ClinGen HI data revealed that four of these 16 genes are associated with autosomal recessive phenotypes, and an additional seven have sufficient or some evidence for haploinsufficiency. Only one candidate gene for DGAP138, glutamate ionotropic receptor kainate type subunit 2 (<em>GRIK2</em> [MIM: 138244]) was a confirmed triplosensitive annotated gene in ClinGen (Table S15).</p>
<p>Taken together, these cases represent more plausible candidates in the search for position effect genes with functional consequences in the subjects’ phenotypes. Examples include <em>GRIK2</em> which could explain the intellectual disability observed in DGAP138; <em>SOS1</em>, forkhead box G1 (<em>FOXG1</em> [MIM: 164874]) and cochlin (<em>COCH</em> [MIM: 603196]) may be related to the neurological and developmental delay as well as hearing loss of DGAP163; acyl-CoA synthetase long-chain family member 4 (<em>ACSL4</em> [MIM: 300157]) and <em>POU3F4</em> could be involved in DGAP176’s cognitive impairment and hearing loss; SATB homeobox 2 (<em>SATB2</em> [MIM: 608148]) may underlie the delayed speech and language development observed in DGAP249; RB binding protein 8 endonuclease (<em>RBBP8</em> [MIM: 604124]) may be involved in DGAP252’s craniofacial dysmorphic features; <em>SOX9</em> most likely explains the cleft palate observed in DGAP288; DNA polymerase epsilon catalytic subunit (<em>POLE</em> [MIM: 174762]) may contribute to the extreme short stature observed in DGAP275, and zinc finger E-box binding homeobox 2 (<em>ZEB2</em> [MIM: 605802]) can potentially explain the hypotonia and neurological features observed in DGAP329. <em>SOX9</em> had been previously proposed to explain DGAP288’s phenotype, and as predicted by our method, a decrease in its expression was observed in RNA derived from DGAP288’s umbilical cord blood.<span class="citation">(Ordulu et al., <a href="#ref-Ordulu2016">2016</a>)</span> Additional quantitative real-time PCR analyses revealed <em>SOS1</em> as having reduced expression in DGAP163-derived LCLs compared to three normal sex-matched controls (Figure 2).Expression assessment for second-tier candidates <em>PAK3</em>, <em>MID2</em> and <em>POU3F4</em> in DGAP176 LCLs did not deviate substantially from their control expression values (Figure S1); further searches into the Genotype-Tissue Expression (GTEx) project<span class="citation">(Lonsdale et al., <a href="#ref-Lonsdale2013">2013</a>)</span> reveal that <em>PAK3</em>, <em>MID2</em> and <em>POU3F4</em> have low expression in LCLs, which would have made assessing changes in expression of these genes technically difficult. This points to the importance of the availability of tissues and cell lines relevant to the studied phenotypes, or the capacity to generate animal models that reproduce the observed BCAs for further analysis.</p>
<p><strong>Identification of subjects with shared non-coding chromosome alterations and phenotypes</strong></p>
<p>The identification of subjects with shared non-coding chromosome alterations and phenotypes as described herein would further support our idea of these rearrangements exerting their pathogenic outcomes through long-range position effects. To identify such subjects, we searched the DECIPHER<span class="citation">(Firth et al., <a href="#ref-Firth2009">2009</a>)</span> and dbVar databases,<span class="citation">(Lappalainen et al., <a href="#ref-Lappalainen2013">2012</a>)</span> both comprehensive community-supported repositories of clinical cases with novel or extremely rare genomic variants.</p>
<p>We found 494 DECIPHER cases overlapping our 34 non-coding BCA breakpoints (Table S19). Of these, 489 had rearrangements that overlapped one or more annotated genes (Table S20). Only five DECIPHER cases fulfilled our non-coding selection criteria (Table S21): cases 1985 and 1989, both of which overlap one of DGAP017’s breakpoints in chromosome 10, but which have several other gene-altering genomic rearrangements; case 289720, a subject with a 161.44 Kb deletion in chromosome 10 described as likely benign and sharing a sequence breakpoint with DGAP126; case 289865 overlapping a breakpoint in DGAP126 in chromosome 10, very similar to case 289720, however with the presence of an additional pathogenic gene-altering rearrangement; and lastly case 293610, a pathogenic duplication of 364.43 Kb in chromosome 17 sharing a breakpoint with DGAP288. Only two of the five DECIPHER cases have reported clinical phenotypes. DECIPHER case 289720 presents with intellectual disability and psychosis, both pertaining to the superclasses of behavioral and neurodevelopmental abnormalities under the HPO classification. Interestingly, DGAP126 has abnormal aggressive, impulsive or violent behavior and auto-aggression, as well as language and motor delays, which also fall under the classification of behavioral and neurodevelopmental abnormalities. DECIPHER case 293610 has reported gonadal tissue discordant for external genitalia or chromosomal sex as well as a non-obstructive azoospermia clinical phenotype;<span class="citation">(Vetro et al., <a href="#ref-Vetro2015">2015</a>)</span> both features are not observed until puberty, and are associated with the female-to-male sex disorder observed for CNVs altering the <em>SOX9</em> genomic landscape. Although DGAP288 is still an infant, there is no report of sex reversal.</p>
<p>From the dbVar database, 675 non-coding structural rearrangements including CNVs, deletions, inversions, and translocations overlap DGAP breakpoints (Table S22). Of these, only five variants had associated clinical information, including variant nsv534336, a 530 Kb duplication overlapping the DGAP017 breakpoint in chromosome 10, classified as “uncertain significance”<span class="citation">(Miller et al., <a href="#ref-Miller2010">2010</a>)</span> and exhibiting a growth delay phenotype; nsv931775, a benign ~381.8 Kb deletion overlapping the DGAP113 breakpoint on chromosome 3, associated with developmental delay and/or other significant developmental or morphological phenotypes;<span class="citation">(Miller et al., <a href="#ref-Miller2010">2010</a>)</span> nsv534571, an ~639.7 Kb duplication of uncertain significance associated with muscular hypotonia and overlapping the DGAP287 breakpoint on chromosome 10; and variants nsv532026 and nsv917014, two duplications of ~613 Kb classified as “uncertain significance” and “likely benign,” respectively, overlapping the DGAP315 breakpoint in chromosome 6, and associated with developmental delay and/or other significant developmental or morphological phenotypes as well as autism and global developmental delay. All the detected variants are associated with phenotypes observed in the DGAP cases, especially DGAP017’s hypoplasia, the developmental delay observed in DGAP113, and DGAP315’s significant developmental or morphological phenotypes.</p>
<p>Strictly speaking, these phenotypes are disparate, but fall under similar phenotypic categories, which could enable identification of long-range effect genes between different cases with similar clinical features and chromosome rearrangements. These comparisons highlight the importance of establishing detailed, specific, and unbiased guidelines for assigning phenotypes when performing computational phenotype comparisons.</p>
<p><strong>Discussion</strong></p>
<p>Structural variation of the human genome, either inherited or arising by <em>de novo</em> germline or somatic mutations, can give rise to different phenotypes through several mechanisms. Chromosome rearrangements can alter gene dosage, promote gene fusions, unmask recessive alleles, or disrupt associations between genes and their regulatory elements. The traditional clinical focus of studying genes disrupted by chromosome rearrangements has shifted to also assess regions neighboring these variants.<span class="citation">(Ordulu et al., <a href="#ref-Ordulu2016">2016</a>)</span> This search for positional effects has been particularly important in the analysis of chromosome rearrangements associated with different clinical conditions and disrupting non-annotated genomic regions.<span class="citation">(Spielmann and Mundlos, <a href="#ref-Spielmann2016">2016</a>; Zhang and Wolynes, <a href="#ref-Zhang2015">2015</a>)</span></p>
<p>The study of chromatin conformation has been requisite in the analysis of such non-coding rearrangements. DNA is organized in the three-dimensional nucleus at varying hierarchical levels that are important for the regulation of gene expression,<span class="citation">(Wit and Laat, <a href="#ref-DeWit2012">2012</a>)</span> with primary roles in embryonic development and disease.<span class="citation">(Bonev and Cavalli, <a href="#ref-Bonev2016">2016</a>)</span> Several studies have analyzed the impact of structural variants in disruption of the regulatory chromatin environment leading to disease;<span class="citation">(Gröschel et al., <a href="#ref-Groschel2014">2014</a>; Ibn-Salem et al., <a href="#ref-Ibn-Salem2014">2014</a>; Lupiáñez et al., <a href="#ref-Lupianez2015">2015</a>; Roussos et al., <a href="#ref-Roussos2014">2014</a>; Visser et al., <a href="#ref-Visser2012">2012</a>)</span> these studies have set the precedent for integrative analyses of disrupted chromatin conformation to expedite functional annotations of non-coding chromosome rearrangements.</p>
<p>We tested the possibility of utilizing chromatin contact information to dissect chromosome rearrangements which disrupt non-coding chromosome regions in clinical cases. We focused on 17 subjects from DGAP, 12 with available clinical microarray information, with different rare presentations and <em>de novo</em> non-coding BCAs classified as VUS. Of these, 15 corresponded to translocations and two were inversions. These cases represent ~11% of the total number of sequenced DGAP cases, which makes our predictions even more significant for future potential treatment or management of subjects who would not otherwise obtain a clinical diagnosis. Utilizing publicly available annotated genomic and regulatory elements, chromatin conformation capture information, predicted enhancer-promoter interactions, phenomatch scores, as well as haploinsufficiency and triplosensitivity information for all genes surrounding the BCA breakpoints at different window sizes (±3 and ±1 Mb as well as BCA-containing TAD positions), we discovered 16 genes for 11 DGAP cases that are top-ranking position effect candidates for the subjects’ clinical phenotypes (Table 1).</p>
<p>We observed that eight of the sequenced DGAP BCA breakpoints, corresponding to six DGAP cases (DGAP017, 176, 249, 275, 288 and 322), overlapped reported annotated and predicted enhancers and DHS sites. Disruption of these regulatory elements could potentially cause improper gene expression or repression through altered enhancer-promoter interactions or interactions with other DHS-associated elements such as insulators and locus control regions, among others. In fact, four of the breakpoints that disrupt annotated DHS sites and enhancers have been shown to establish chromatin contacts with our top position effect candidate genes in the region in Hi-C data of H1-hESC cells at 40 Kb resolution (Table S18). For example, the DGAP275_B breakpoint is involved in a chromatin interaction that puts it into physical proximity with <em>POLE</em> and <em>ANKLE2,</em> DGAP288_B contacts <em>SOX9</em>, and DGAP176_B interacts with <em>ACSL4</em>. Three additional breakpoints from DGAP111, 249 and 287 overlap CTCF binding sites. CTCF binding sites are enriched in TAD boundaries,<span class="citation">(Dixon et al., <a href="#ref-Dixon2012">2012</a>)</span> and the elimination of these binding sites could potentially induce gene expression or other functional changes through alteration of the structural regulatory landscape of the region.<span class="citation">(Lupiáñez et al., <a href="#ref-Lupianez2015">2015</a>)</span></p>
<p>There are nine DGAP cases (DGAP113, 126, 138, 153, 163, 252, 315, 319 and 329), six with normal arrays and two with benign CNVs, for which no overlap with genomic or other regulatory elements was detected. These cases thus represent events in which position effects are most likely caused by alteration of the underlying chromatin structure itself. This hypothesis is supported by detection of a vast number of disrupted chromatin contacts in four different cell lines (H1-hESC, IMR90, GM06990, GM12878) at different Hi-C window resolutions, 32 breakpoints included in H1-hESC TADs,<span class="citation">(Dixon et al., <a href="#ref-Dixon2012">2012</a>)</span> and the separation of 193 genes from one and up to 91 of their predicted enhancers after the occurrence of the BCAs (Table S14). For example, <em>SOS1</em>, one of the most significant candidates in explaining DGAP163’s global developmental delay, dysmorphic/distinctive facies and hearing loss, as observed in Noonan Syndrome 1 (NS1 [MIM: 163950]), is separated from its interaction with 88 predicted enhancers (Figure 3), and exhibited a decrease in expression in DGAP163-derived LCLs. However, NS1 is caused by autosomal dominant mutations in <em>SOS1</em>; we hypothesize that the reduced expression of <em>SOS1</em> might affect the RAS/MAPK signaling pathway and generate clinical features not completely overlapping those of NS1; however, this possibility remains to be functionally tested and complimented with analyses of genomic single nucleotide variants. A similar approach could be explored for DGAP275, where we hypothesize that <em>POLE</em>, associated with the facial dysmorphism, immunodeficiency, livedo, and short stature syndrome (FILS [MIM: 615139]) in an autosomal recessive manner,<span class="citation">(Pachlopnik Schmid et al., <a href="#ref-PachlopnikSchmid2012">2012</a>)</span> may contribute to the extreme short stature observed in this DGAP subject; and <em>ZEB2</em>, etiologic for Mowat-Wilson syndrome (MOWS [MIM: 235730]) in an autosomal dominant manner (OMIM#235730), may potentially explain the hypotonia and neurological features observed in DGAP329 but not present all of the dysmorphic features or medical/non-neurologic phenotype of MOWS. Overall, more candidate genes will need to be analyzed rigorously to assess the validity of our position effect predictions and the disruption of important chromatin regulatory elements. Nonetheless, insight into the molecular pathway of disorders may be forthcoming from our approach and of value in the management of some individuals.</p>
<p>All predicted candidate genes have different lines of evidence supporting their selection, starting with a significant phenomatch score that correlates annotated gene phenotypes to those observed in the DGAP cases. HI and triplosensitivity evidence, inclusion in TAD regions, as well as HI scores build upon this selection, and can help laboratories and clinicians focus in subsequent analyses on candidates of their interest. As of now, the “top-ranking” candidates have the highest number of evidence supporting their selection; however, there are also 102 second-tier candidates for the 17 analyzed DGAP cases within ±1 Mb analysis windows which may well play a functional role. Presently, we are unable to give “weights” to any of these selection criteria (<em>i.e</em>., a gene with a high phenomatch score and no evidence of HI is “more significant” than a gene with a medium phenomatch score and evidence of HI) mainly for two reasons: (i) we would need to collect more examples, which might not be easy to find and require a tremendous curation effort, and (ii) we need to understand the possibility, suggested by our results, that more than one gene may be contributory in the clinical presentation of the DGAP subjects, either acting simultaneously or throughout development. Moreover, many of the candidates have recessive inheritance modes, which make it necessary to assess the mutational status of both alleles as well as additional sequence variants not captured by our BCA breakpoint sequencing and the microarrays. Future in-depth exome, DNA and RNA sequencing as well as Hi-C experiments will provide a comprehensive view of the contribution of sequence variants, disruption of chromatin contacts, and changes in gene expression in the DGAP disease etiologies, such that guidelines might be developed as to which candidates should be followed up first and further studied with comprehensive functional validation using animal models and human cell lines that reproduce the BCA breakpoints.</p>
<p>Overall our results suggest that the integration of phenomatch scores, altered chromatin contacts, and other clinical gene annotations provide valuable interpretation to many variants of uncertain significance through long-range position effects. The correct prediction of 52 out of 57 known pathogenic genes in DGAP cases used as positive controls supports such integration. Our computational analysis is rapid and can provide additional information to benefit the clinical assessment of both coding and non-coding genome variants. The latter is an important step towards prediction of pathogenic consequences of non-coding variantion observed in prenatal samples. For example, based on its position and chromatin contact alterations, we correctly predicted the involvement and decreased expression of <em>SOX9</em> in the cleft palate Pierre-Robin sequence (PRBNS [MIM: 261800]) association in DGAP288.<span class="citation">(Ordulu et al., <a href="#ref-Ordulu2016">2016</a>)</span></p>
<p>Lastly, we would like to note that predicting the pathogenic outcome of disrupted chromatin contacts is not a straightforward endeavor: it has been shown that a single gene promoter can be targeted by several enhancers,<span class="citation">(Thurman et al., <a href="#ref-Thurman2012">2012</a>)</span> therefore compensating for the perturbed interactions by the chromosome rearrangements. In addition, rearrangements can reposition gene promoters and enhancers outside of their preferred chromatin environments, leading to improper gene activation by enhancer adoption.<span class="citation">(Lupiáñez et al., <a href="#ref-Lupianez2015">2015</a>)</span> Our method currently identifies instances in which known and predicted enhancer/promoter interactions are disrupted by the rearrangement breakpoints and thus lead to decreased candidate gene expression. Enhancer adoption prediction will be incorporated once mathematical models of TAD formation upon changes in genomic sequence are refined and available to the greater scientific community. Presently, our predictions are as good as the availability of pathogenic gene annotations, chromatin conformation data, clinical phenotype information, and the presence of similar rearrangements in databases such as DECIPHER and dbVar. While the existence of other subjects with related phenotypes to the DGAP cases does not prove the involvement of neighboring genes in the etiology of these phenotypes, it is a step forward towards prediction of pathogenic effects starting from a simple computational analysis, pointing to a better phenotypic categorization when clinically examining affected individuals. By making our position effect prediction method available to the human genetics community, we hope to study additional cases with complete phenotypic information and be able to refine better the rules for the prediction of position effects on gene expression and discover new mechanisms of pathogenicity.</p>
<p><strong>Acknowledgements</strong></p>
<p>We offer heartfelt gratitude to all DGAP research participants and their families, and to countless genetic counselors, clinical geneticists, cytogeneticists, and physicians for their ongoing support of our study and for referrals to our project. This study was funded by the National Institutes of Health (GM061354 to CCM and MET). The authors declare no conflicts of interest.</p>
<p><strong>Tables</strong></p>
<p><strong>Table 1.</strong> <strong>Description of the 17 analyzed DGAP cases with non-coding BCAs.</strong> Corresponding clinical karyotypes are reported, with overlap of breakpoints with regulatory elements (E = enhancer, DHS = DNaseI hypersensitive sites, CTCF = CTCF binding sites), and TADs from H1-hESC, IMR90, and GM12878 (1= one breakpoint within TAD, 2=both BCA breakpoints are located within TAD). Top-ranking position effect genes are provided for the ±1 Mb windows surrounding the BCA breakpoints; each gene is highlighted with different evidence supporting its inclusion (a = ClinGen known recessive genes, b= ClinGen genes with emerging and sufficient evidence suggesting haploinsufficiency is associated with clinical phenotype, c = HI scores less than 10, d = within H1-ESC TAD, e = DHS enhancer-promoter disrupted interactions).</p>
<table>
<thead>
<tr class="header">
<th align="left"><strong>Subject ID</strong></th>
<th align="left"><strong>Reported Karyotype</strong></th>
<th align="left"><strong>Disruption of Functional Elemen</strong></th>
<th align="left"><strong>Breakpoints within TADs (hESC)</strong></th>
<th align="left"><strong>Breakpoints within TADs (IMR90)</strong></th>
<th align="left"><strong>Breakpoints within TADs (GM12878)</strong></th>
<th align="left"><strong>Top-ranking Candidates ±1 Mb </strong></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><strong>DGAP017</strong></td>
<td align="left">46,X,t(X;10)(p11.2;q24.3)</td>
<td align="left">DHS</td>
<td align="left">2</td>
<td align="left">2</td>
<td align="left">1</td>
<td align="left">-</td>
</tr>
<tr class="even">
<td align="left"><strong>DGAP111</strong></td>
<td align="left">46,XY,t(16;20)(q11.2;q13.2)dn</td>
<td align="left">CTCF</td>
<td align="left">1</td>
<td align="left">1</td>
<td align="left">2</td>
<td align="left"><em>ORC6<sup>a</sup></em></td>
</tr>
<tr class="odd">
<td align="left"><strong>DGAP113</strong></td>
<td align="left">46,XY,t(1;3)(q32.1;q13.2)dn</td>
<td align="left">-</td>
<td align="left">2</td>
<td align="left">2</td>
<td align="left">2</td>
<td align="left"><em>ASPM<sup>a</sup></em></td>
</tr>
<tr class="even">
<td align="left"><strong>DGAP126</strong></td>
<td align="left">46,XX,t(5;10)(p13.3;q21.1)dn</td>
<td align="left">-</td>
<td align="left">2</td>
<td align="left">1</td>
<td align="left">2</td>
<td align="left">-</td>
</tr>
<tr class="odd">
<td align="left"><strong>DGAP138</strong></td>
<td align="left">46,XY,t(1;6)(q23;q13)dn</td>
<td align="left">-</td>
<td align="left">2</td>
<td align="left">2</td>
<td align="left">2</td>
<td align="left"><em>GRIK2<sup>ac</sup></em></td>
</tr>
<tr class="even">
<td align="left"><strong>DGAP153</strong></td>
<td align="left">46,X,t(X;17)(p11.23;p11.2)dn</td>
<td align="left">-</td>
<td align="left">1</td>
<td align="left">1</td>
<td align="left">1</td>
<td align="left">-</td>
</tr>
<tr class="odd">
<td align="left"><strong>DGAP163</strong></td>
<td align="left">46,XY,t(2;14)(p23;q13)dn</td>
<td align="left">-</td>
<td align="left">2</td>
<td align="left">2</td>
<td align="left">2</td>
<td align="left"><em>SOS1<sup>cde</sup>, COCH<sup>de</sup></em></td>
</tr>
<tr class="even">
<td align="left"><strong>DGAP176</strong></td>
<td align="left">46,Y,inv(X)(q13q24)mat</td>
<td align="left">DHS, CTCF</td>
<td align="left">2</td>
<td align="left">1</td>
<td align="left">2</td>
<td align="left"><em>ACSL4<sup>bd</sup>, COL4A5<sup>bcde</sup></em></td>
</tr>
<tr class="odd">
<td align="left"><strong>DGAP249</strong></td>
<td align="left">46,XX,t(2;11)(q33;q23)dn</td>
<td align="left">E, DHS</td>
<td align="left">2</td>
<td align="left">2</td>
<td align="left">2</td>
<td align="left"><em>SATB2<sup>bcde</sup>, SORL1<sup>e</sup></em></td>
</tr>
<tr class="even">
<td align="left"><strong>DGAP252</strong></td>
<td align="left">46,XY,t(3;18)(q13.2;q11.2)dn</td>
<td align="left">-</td>
<td align="left">2</td>
<td align="left">2</td>
<td align="left">2</td>
<td align="left"><em>RBBP8<sup>a</sup>,GATA6<sup>bcde</sup></em></td>
</tr>
<tr class="odd">
<td align="left"><strong>DGAP275</strong></td>
<td align="left">46,XX,t(7;12)(p13;q24.33)dn</td>
<td align="left">DHS</td>
<td align="left">1</td>
<td align="left">1</td>
<td align="left">2</td>
<td align="left"><em>ANKLE2<sup>e</sup>, POLE<sup>e</sup></em></td>
</tr>
<tr class="even">
<td align="left"><strong>DGAP287</strong></td>
<td align="left">46,XY,t(10;14)(p13;q32.1)dn</td>
<td align="left">CTCF</td>
<td align="left">2</td>
<td align="left">2</td>
<td align="left">2</td>
<td align="left">-</td>
</tr>
<tr class="odd">
<td align="left"><strong>DGAP288</strong></td>
<td align="left">46,XX,t(6:17)(q13;q21)dn</td>
<td align="left">DHS</td>
<td align="left">2</td>
<td align="left">2</td>
<td align="left">2</td>
<td align="left"><em>SOX9<sup>bcd</sup></em></td>
</tr>
<tr class="even">
<td align="left"><strong>DGAP315</strong></td>
<td align="left">46,XX,inv(6)(p24q11)dn</td>
<td align="left">-</td>
<td align="left">1</td>
<td align="left">1</td>
<td align="left">2</td>
<td align="left">-</td>
</tr>
<tr class="odd">
<td align="left"><strong>DGAP319</strong></td>
<td align="left">46,XX,t(4;13)(q31.3;q14.3)dn</td>
<td align="left">-</td>
<td align="left">2</td>
<td align="left">1</td>
<td align="left">2</td>
<td align="left">-</td>
</tr>
<tr class="even">
<td align="left"><strong>DGAP322</strong></td>
<td align="left">46,XY,t(1;18)(q32.1;q22.1)</td>
<td align="left">DHS</td>
<td align="left">1</td>
<td align="left">2</td>
<td align="left">2</td>
<td align="left"><em>IRF6<sup>bcd</sup></em></td>
</tr>
<tr class="odd">
<td align="left"><strong>DGAP329</strong></td>
<td align="left">46,XX,t(2;14)(q21;q24.3)dn</td>
<td align="left">-</td>
<td align="left">1</td>
<td align="left">2</td>
<td align="left">2</td>
<td align="left"><em>ZEB2<sup>bcde</sup></em></td>
</tr>
</tbody>
</table>
<p><strong>Figures</strong></p>
<p><strong>Figure 1.</strong> Chromosome locations of the 17 analyzed DGAP cases with non-coding BCAs. Breakpoint positions are marked with a blue line and the corresponding DGAP number. All chromosomes are aligned by the centromere (marked in pink) and are indicated above by their corresponding chromosome number.</p>
<p><strong>Figure 2.</strong> Assessment of gene expression changes for DGAP163-derived LCLs. Each column represents the ΔCT results of three culture replicates, with four technical replicates each, compared to three sex-matched control cell lines. Error bars indicate the standard deviation calculated from the biological replicates. The Mann-Whitney U test p-value is provided for the comparison between expression values of <em>SOS1</em> and the control <em>GUSB</em>.</p>
<p><strong>Figure 3.</strong> Disrupted enhancer-promoter DHS interactions predicted for <em>SOS1</em> (gene position indicated by asterisk). The color graded rectangle represents the correlation values for the interactions as reported by ENCODE. The dashed line indicates the translocation breakpoint position in chromosome 2. Lilac colored rectangles represent genes, and pink rectangles show TAD positions annotated in H1-hESC.</p>
<p><strong>Web Resources</strong></p>
<p>The scripts used in this study to predict position effects can be downloaded from: <a href="https://github.com/ibn-salem/position_effect"><em>https://github.com/ibn-salem/position_effect</em></a></p>
<p>OMIM, <a href="http://www.omim.org/"><em>http://www.omim.org</em></a></p>
<p>Ensembl GRCh37 archive, <a href="http://grch37.ensembl.org/"><em>http://grch37.ensembl.org</em></a></p>
<p>Human lincRNAs catalog, <a href="http://portals.broadinstitute.org/genome_bio/human_lincrnas"><em>http://portals.broadinstitute.org/genome_bio/human_lincrnas</em></a></p>
<p>Haploinsufficiency scores, <a href="https://decipher.sanger.ac.uk/"><em>https://decipher.sanger.ac.uk</em></a></p>
<p>ClinGen GRCh37 data, <a href="ftp://ftp.ncbi.nlm.nih.gov/pub/dbVar/clingen"><em>ftp://ftp.ncbi.nlm.nih.gov/pub/dbVar/clingen</em></a></p>
<p>University of California Santa Cruz Genome Browser, <a href="https://genome.ucsc.edu/"><em>https://genome.ucsc.edu</em></a></p>
<p>Human Phenotype Ontology, <a href="http://human-phenotype-ontology.github.io/"><em>http://human-phenotype-ontology.github.io</em></a></p>
<p>Harvard Biopolymers Facility, <a href="https://genome.med.harvard.edu/"><em>https://genome.med.harvard.edu</em></a></p>
<p>dbVar Variation Viewer, <a href="https://www.ncbi.nlm.nih.gov/variation/view"><em>https://www.ncbi.nlm.nih.gov/variation/view</em></a></p>
<p>3D Genome Browser, <a href="http://promoter.bx.psu.edu/hi-c"><em>http://promoter.bx.psu.edu/hi-c</em></a></p>
<p>ENCODE, <a href="https://www.encodeproject.org/"><em>https://www.encodeproject.org</em></a></p>
<p>WashU EpiGenome Browser, <a href="http://epigenomegateway.wustl.edu/"><em>http://epigenomegateway.wustl.edu/</em></a></p>
<p>GTEx portal, <a href="https://www.gtexportal.org/home"><em>https://www.gtexportal.org/home</em></a></p>

</div>
<h3>References</h3>
<div id="refs" class="references">
<div id="ref-Conrad2006">
<p>Conrad, D.F., Andrews, T.D., Carter, N.P., Hurles, M.E., and Pritchard, J.K. (2006). A high-resolution survey of deletion polymorphism in the human genome. Nature Genetics <em>38</em>, 75–81.</p>
</div>
<div id="ref-Hinds2006">
<p>Hinds, D.A., Kloek, A.P., Jen, M., Chen, X., and Frazer, K.A. (2006). Common deletions and SNPs are in linkage disequilibrium in the human genome. Nature Genetics <em>38</em>, 82–85.</p>
</div>
<div id="ref-Iafrate2004">
<p>Iafrate, A.J., Feuk, L., Rivera, M.N., Listewnik, M.L., Donahoe, P.K., Qi, Y., Scherer, S.W., and Lee, C. (2004). Detection of large-scale variation in the human genome. Nature Genetics <em>36</em>, 949–951.</p>
</div>
<div id="ref-Sebat2004">
<p>Sebat, J., Lakshmi, B., Troge, J., Alexander, J., Young, J., Lundin, P., Manér, S., Massa, H., Walker, M., Chi, M., et al. (2004). Large-Scale Copy Number Polymorphism in the Human Genome. Science <em>305</em>, 525–528.</p>
</div>
<div id="ref-Korbel2007">
<p>Korbel, J.O., Urban, A.E., Affourtit, J.P., Godwin, B., Grubert, F., Simons, J.F., Kim, P.M., Palejev, D., Carriero, N.J., Du, L., et al. (2007). Paired-end mapping reveals extensive structural variation in the human genome. Science (New York, N.Y.) <em>318</em>, 420–426.</p>
</div>
<div id="ref-Nambiar2011">
<p>Nambiar, M., and Raghavan, S.C. (2011). How does DNA break during chromosomal translocations? Nucleic Acids Research <em>39</em>, 5813–5825.</p>
</div>
<div id="ref-Theisen2010">
<p>Theisen, A., and Shaffer, L.G. (2010). Disorders caused by chromosome abnormalities. The Application of Clinical Genetics <em>3</em>, 159–174.</p>
</div>
<div id="ref-Zhang2009">
<p>Zhang, F., Gu, W., Hurles, M.E., and Lupski, J.R. (2009). Copy Number Variation in Human Health, Disease, and Evolution. Annual Review of Genomics and Human Genetics <em>10</em>, 451–481.</p>
</div>
<div id="ref-Kleinjan2005">
<p>Kleinjan, D.A., and Van Heyningen, V. (2005). Long-Range Control of Gene Expression: Emerging Mechanisms and Disruption in Disease. Am. J. Hum. Genet <em>76</em>, 8–32.</p>
</div>
<div id="ref-Weiler1995">
<p>Weiler, K.S., and Wakimoto, B.T. (1995). Heterochromatin and gene expression in Drosophila. Annual Review of Genetics <em>29</em>, 577–605.</p>
</div>
<div id="ref-Spielmann2016">
<p>Spielmann, M., and Mundlos, S. (2016). Looking beyond the genes: the role of non-coding variants in human disease. Human Molecular Genetics ddw205.</p>
</div>
<div id="ref-Zhang2015">
<p>Zhang, B., and Wolynes, P.G. (2015). Topology, structures, and energy landscapes of human chromosomes. Proceedings of the National Academy of Sciences <em>112</em>, 201506257.</p>
</div>
<div id="ref-Fantes1995">
<p>Fantes, J., Redeker, B., Breen, M., Boyle, S., Brown, J., Fletcher, J., Jones, S., Bickmore, W., Fukushima, Y., Mannens, M., et al. (1995). Aniridia-associated cytogenetic rearrangements suggest that a position effect may cause the mutant phenotype. Human Molecular Genetics <em>4</em>, 415–422.</p>
</div>
<div id="ref-Kleinjan2001">
<p>Kleinjan, D.A., Seawright, A., Schedl, A., Quinlan, R.A., Danes, S., and Heyningen, V. van (2001). Aniridia-associated translocations, DNase hypersensitivity, sequence comparison and transgenic analysis redefine the functional domain of PAX6. Human Molecular Genetics <em>10</em>, 2049–2059.</p>
</div>
<div id="ref-Cai2003">
<p>Cai, J., Goodman, B.K., Patel, A.S., Mulliken, J.B., Van Maldergem, L., Hoganson, G.E., Paznekas, W.A., Ben-Neriah, Z., Sheffer, R., Cunningham, M.L., et al. (2003). Increased risk for developmental delay in Saethre-Chotzen syndrome is associated with TWIST deletions: an improved strategy for TWIST mutation screening. Human Genetics <em>114</em>, 68–76.</p>
</div>
<div id="ref-Flomen1998">
<p>Flomen, R.H., Vatcheva, R., Gorman, P.A., Baptista, P.R., Groet, J., Barišić, I., Ligutic, I., and Nižetić, D. (1998). Construction and Analysis of a Sequence-Ready Map in 4q25: Rieger Syndrome Can Be Caused by Haploinsufficiency ofRIEG, but Also by Chromosome Breaks ≈90 kb Upstream of This Gene. Genomics <em>47</em>, 409–413.</p>
</div>
<div id="ref-Trembath2004">
<p>Trembath, D.G., Semina, E.V., Jones, D.H., Patil, S.R., Qian, Q., Amendt, B.A., Russo, A.F., and Murray, J.C. (2004). Analysis of two translocation breakpoints and identification of a negative regulatory element in patients with Rieger’s syndrome. Birth Defects Research Part A: Clinical and Molecular Teratology <em>70</em>, 82–91.</p>
</div>
<div id="ref-Velagaleti2005">
<p>Velagaleti, G.V., Bien-Willner, G.A., Northup, J.K., Lockhart, L.H., Hawkins, J.C., Jalal, S.M., Withers, M., Lupski, J.R., and Stankiewicz, P. (2005). Position Effects Due to Chromosome Breakpoints that Map ∼900 Kb Upstream and ∼1.3 Mb Downstream of SOX9 in Two Patients with Campomelic Dysplasia. The American Journal of Human Genetics <em>76</em>, 652–662.</p>
</div>
<div id="ref-Kleinjan1998">
<p>Kleinjan, D., and Heyningen, V. van (1998). Position effect in human genetic disease. Human Molecular Genetics <em>7</em>, 1611–1618.</p>
</div>
<div id="ref-Lupski2005">
<p>Lupski, J.R., and Stankiewicz, P. (2005). Genomic Disorders: Molecular Mechanisms for Rearrangements and Conveyed Phenotypes. PLoS Genetics <em>1</em>, e49.</p>
</div>
<div id="ref-Dekker2002">
<p>Dekker, J., Rippe, K., Dekker, M., and Kleckner, N. (2002). Capturing chromosome conformation. Science (New York, N.Y.) <em>295</em>, 1306–1311.</p>
</div>
<div id="ref-DeWit2012">
<p>Wit, E. de, and Laat, W. de (2012). A decade of 3C technologies: insights into nuclear organization. Genes &amp; Development <em>26</em>, 11–24.</p>
</div>
<div id="ref-Dixon2012">
<p>Dixon, J.R., Selvaraj, S., Yue, F., Kim, A., Li, Y., Shen, Y., Hu, M., Liu, J.S., and Ren, B. (2012). Topological domains in mammalian genomes identified by analysis of chromatin interactions. Nature <em>485</em>, 376–380.</p>
</div>
<div id="ref-Fullwood2009">
<p>Fullwood, M.J., Liu, M.H., Pan, Y.F., Liu, J., Xu, H., Mohamed, Y.B., Orlov, Y.L., Velkov, S., Ho, A., Mei, P.H., et al. (2009). An oestrogen-receptor-<span class="math inline">\(\alpha\)</span>-bound human chromatin interactome. Nature <em>462</em>, 58–64.</p>
</div>
<div id="ref-Lieberman-Aiden2009">
<p>Lieberman-Aiden, E., Berkum, N.L. van, Williams, L., Imakaev, M., Ragoczy, T., Telling, A., Amit, I., Lajoie, B.R., Sabo, P.J., Dorschner, M.O., et al. (2009). Comprehensive mapping of long-range interactions reveals folding principles of the human genome. Science (New York, N.Y.) <em>326</em>, 289–293.</p>
</div>
<div id="ref-Mifsud2015">
<p>Mifsud, B., Tavares-Cadete, F., Young, A.N., Sugar, R., Schoenfelder, S., Ferreira, L., Wingett, S.W., Andrews, S., Grey, W., Ewels, P. a, et al. (2015). Mapping long-range promoter contacts in human cells with high-resolution capture Hi-C. Nature Genetics <em>47</em>.</p>
</div>
<div id="ref-Phillips-Cremins2013">
<p>Phillips-Cremins, J.E., Sauria, M.E.G., Sanyal, A., Gerasimova, T.I., Lajoie, B.R., Bell, J.S.K., Ong, C.-T., Hookway, T. a, Guo, C., Sun, Y., et al. (2013). Architectural protein subclasses shape 3D organization of genomes during lineage commitment. Cell <em>153</em>, 1281–1295.</p>
</div>
<div id="ref-Rao2014">
<p>Rao, S.S.P., Huntley, M.H., Durand, N.C., Stamenova, E.K., Bochkov, I.D., Robinson, J.T., Sanborn, A.L., Machol, I., Omer, A.D., Lander, E.S., et al. (2014). A 3D map of the human genome at kilobase resolution reveals principles of chromatin looping. Cell <em>159</em>, 1665–1680.</p>
</div>
<div id="ref-Sanyal2012">
<p>Sanyal, A., Lajoie, B.R., Jain, G., and Dekker, J. (2012). The long-range interaction landscape of gene promoters. Nature <em>489</em>, 109–113.</p>
</div>
<div id="ref-Lupianez2015">
<p>Lupiáñez, D.G., Kraft, K., Heinrich, V., Krawitz, P., Brancati, F., Klopocki, E., Horn, D., Kayserili, H., Opitz, J.M., Laxova, R., et al. (2015). Disruptions of Topological Chromatin Domains Cause Pathogenic Rewiring of Gene-Enhancer Interactions. Cell <em>161</em>, 1012–1025.</p>
</div>
<div id="ref-Groschel2014">
<p>Gröschel, S., Sanders, M.A., Hoogenboezem, R., Wit, E. de, Bouwman, B.A.M., Erpelinck, C., Velden, V.H.J. van der, Havermans, M., Avellino, R., Lom, K. van, et al. (2014). A single oncogenic enhancer rearrangement causes concomitant EVI1 and GATA2 deregulation in leukemia. Cell <em>157</em>, 369–381.</p>
</div>
<div id="ref-Claussnitzer2015">
<p>Claussnitzer, M., Dankel, S.N., Kim, K.-H., Quon, G., Meuleman, W., Haugen, C., Glunk, V., Sousa, I.S., Beaudry, J.L., Puviindran, V., et al. (2015). &lt;i&gt;FTO&lt;/i&gt; Obesity Variant Circuitry and Adipocyte Browning in Humans. New England Journal of Medicine <em>373</em>, 895–907.</p>
</div>
<div id="ref-Giorgio2015">
<p>Giorgio, E., Robyr, D., Spielmann, M., Ferrero, E., Di Gregorio, E., Imperiale, D., Vaula, G., Stamoulis, G., Santoni, F., Atzori, C., et al. (2015). A large genomic deletion leads to enhancer adoption by the lamin B1 gene: a second path to autosomal dominant adult-onset demyelinating leukodystrophy (ADLD). Human Molecular Genetics <em>24</em>, 3143–3154.</p>
</div>
<div id="ref-Ibn-Salem2014">
<p>Ibn-Salem, J., Köhler, S., Love, M.I., Chung, H.-R., Huang, N., Hurles, M.E., Haendel, M., Washington, N.L., Smedley, D., Mungall, C.J., et al. (2014). Deletions of chromosomal regulatory boundaries are associated with congenital disease. Genome Biology <em>15</em>, 423.</p>
</div>
<div id="ref-Oldridge2015">
<p>Oldridge, D.A., Wood, A.C., Weichert-Leahey, N., Crimmins, I., Sussman, R., Winter, C., McDaniel, L.D., Diamond, M., Hart, L.S., Zhu, S., et al. (2015). Genetic predisposition to neuroblastoma mediated by a LMO1 super-enhancer polymorphism. Nature <em>528</em>, 418–421.</p>
</div>
<div id="ref-Roussos2014">
<p>Roussos, P., Mitchell, A.C., Voloudakis, G., Fullard, J.F., Pothula, V.M., Tsang, J., Stahl, E.A., Georgakopoulos, A., Ruderfer, D.M., Charney, A., et al. (2014). A role for noncoding variation in schizophrenia. Cell Reports <em>9</em>, 1417–1429.</p>
</div>
<div id="ref-Visser2012">
<p>Visser, M., Kayser, M., and Palstra, R.-J. (2012). HERC2 rs12913832 modulates human pigmentation by attenuating chromatin-loop formation between a long-range enhancer and the OCA2 promoter. Genome Research <em>22</em>, 446–455.</p>
</div>
<div id="ref-Higgins2008">
<p>Higgins, A.W., Alkuraya, F.S., Bosco, A.F., Brown, K.K., Bruns, G.A., Donovan, D.J., Eisenman, R., Fan, Y., Farra, C.G., Ferguson, H.L., et al. (2008). Characterization of Apparently Balanced Chromosomal Rearrangements from the Developmental Genome Anatomy Project. American Journal of Human Genetics <em>82</em>, 712–722.</p>
</div>
<div id="ref-Kim2008">
<p>Kim, W.K., and Marcotte, E.M. (2008). Age-dependent evolution of the yeast protein interaction network suggests a limited role of gene duplication and divergence. PLoS Computational Biology <em>4</em>, e1000232.</p>
</div>
<div id="ref-Ligon2005">
<p>Ligon, A.H., Moore, S.D., Parisi, M.A., Mealiffe, M.E., Harris, D.J., Ferguson, H.L., Quade, B.J., and Morton, C.C. (2005). Constitutional Rearrangement of the Architectural Factor HMGA2: A Novel Human Phenotype Including Overgrowth and Lipomas. The American Journal of Human Genetics <em>76</em>, 340–348.</p>
</div>
<div id="ref-Lu2007">
<p>Lu, W., Quintero-Rivera, F., Fan, Y., Alkuraya, F.S., Donovan, D.J., Xi, Q., Turbe-Doan, A., Li, Q.-G., Campbell, C.G., Shanske, A.L., et al. (2007). NFIA Haploinsufficiency Is Associated with a CNS Malformation Syndrome and Urinary Tract Defects. PLoS Genetics <em>3</em>, e80.</p>
</div>
<div id="ref-Redin2017">
<p>Redin, C., Brand, H., Collins, R.L., Kammin, T., Mitchell, E., Hodge, J.C., Hanscom, C., Pillalamarri, V., Seabra, C.M., Abbott, M.-A., et al. (2017). The genomic landscape of balanced cytogenetic abnormalities associated with human congenital anomalies. Nature Genetics <em>49</em>, 36–45.</p>
</div>
<div id="ref-Quinn2016">
<p>Quinn, J.J., and Chang, H.Y. (2016). Unique features of long non-coding RNA biogenesis and function. Nature Reviews Genetics <em>17</em>, 47–62.</p>
</div>
<div id="ref-Muro2010">
<p>Muro, E.M., and Andrade-Navarro, M.A. (2010). Pseudogenes as an alternative source of natural antisense transcripts. BMC Evolutionary Biology <em>10</em>, 338.</p>
</div>
<div id="ref-Pink2011">
<p>Pink, R.C., Wicks, K., Caley, D.P., Punch, E.K., Jacobs, L., and Carter, D.R.F. (2011). Pseudogenes: pseudo-functional or key regulators in health and disease? RNA (New York, N.Y.) <em>17</em>, 792–798.</p>
</div>
<div id="ref-Ordulu2014">
<p>Ordulu, Z., Wong, K.E., Currall, B.B., Ivanov, A.R., Pereira, S., Althari, S., Gusella, J.F., Talkowski, M.E., and Morton, C.C. (2014). Describing Sequencing Results of Structural Chromosome Rearrangements with a Suggested Next-Generation Cytogenetic Nomenclature. The American Journal of Human Genetics <em>94</em>, 695–709.</p>
</div>
<div id="ref-Flicek2014">
<p>Flicek, P., Amode, M.R., Barrell, D., Beal, K., Billis, K., Brent, S., Carvalho-Silva, D., Clapham, P., Coates, G., Fitzgerald, S., et al. (2014). Ensembl 2014. Nucleic Acids Research <em>42</em>, D749–D755.</p>
</div>
<div id="ref-Cabili2011">
<p>Cabili, M.N., Trapnell, C., Goff, L., Koziol, M., Tazon-Vega, B., Regev, A., and Rinn, J.L. (2011). Integrative annotation of human large intergenic noncoding RNAs reveals global properties and specific subclasses. Genes &amp; Development <em>25</em>, 1915–1927.</p>
</div>
<div id="ref-Huang2010">
<p>Huang, N., Lee, I., Marcotte, E.M., and Hurles, M.E. (2010). Characterising and Predicting Haploinsufficiency in the Human Genome. PLoS Genetics <em>6</em>, e1001154.</p>
</div>
<div id="ref-Rehm2015">
<p>Rehm, H.L., Berg, J.S., Brooks, L.D., Bustamante, C.D., Evans, J.P., Landrum, M.J., Ledbetter, D.H., Maglott, D.R., Martin, C.L., Nussbaum, R.L., et al. (2015). ClinGen — The Clinical Genome Resource. New England Journal of Medicine <em>372</em>, 2235–2242.</p>
</div>
<div id="ref-Thurman2012">
<p>Thurman, R.E., Rynes, E., Humbert, R., Vierstra, J., Maurano, M.T., Haugen, E., Sheffield, N.C., Stergachis, A.B., Wang, H., Vernot, B., et al. (2012). The accessible chromatin landscape of the human genome. Nature <em>489</em>, 75–82.</p>
</div>
<div id="ref-Narendra2015">
<p>Narendra, V., Rocha, P.P., An, D., Raviram, R., Skok, J.A., Mazzoni, E.O., and Reinberg, D. (2015). Transcription. CTCF establishes discrete functional chromatin domains at the Hox clusters during differentiation. Science (New York, N.Y.) <em>347</em>, 1017–1021.</p>
</div>
<div id="ref-Flavahan2016">
<p>Flavahan, W. a, Drier, Y., Liau, B.B., Gillespie, S.M., Venteicher, a S., Stemmer-Rachamimov, a O., Suva, M.L., and Bernstein, B.E. (2016). Insulator dysfunction and oncogene activation in IDH mutant gliomas. Nature <em>529</em>, 110–114.</p>
</div>
<div id="ref-Hnisz2016">
<p>Hnisz, D., Weintraub, A.S., Day, D.S., Valton, A.-l., Bak, R.O., Li, C.H., Goldmann, J., Lajoie, B.R., Fan, Z.P., Sigova, A.A., et al. (2016). Activation of proto-oncogenes by disruption of chromosome neighborhoods. Science (New York, N.Y.) <em>351</em>, 1454–1458.</p>
</div>
<div id="ref-Dunham2012">
<p>Dunham, I., Kundaje, A., Aldred, S.F., Collins, P.J., Davis, C. a, Doyle, F., Epstein, C.B., Frietze, S., Harrow, J., Kaul, R., et al. (2012). An integrated encyclopedia of DNA elements in the human genome. Nature <em>489</em>, 57–74.</p>
</div>
<div id="ref-Kent2002">
<p>Kent, W.J., Sugnet, C.W., Furey, T.S., Roskin, K.M., Pringle, T.H., Zahler, A.M., and Haussler, D. (2002). The human genome browser at UCSC. Genome Research <em>12</em>, 996–1006.</p>
</div>
<div id="ref-Andersson2014">
<p>Andersson, R., Gebhard, C., Miguel-Escalada, I., Hoof, I., Bornholdt, J., Boyd, M., Chen, Y., Zhao, X., Schmidl, C., Suzuki, T., et al. (2014). An atlas of active enhancers across human cell types and tissues. Nature <em>507</em>, 455–461.</p>
</div>
<div id="ref-Visel2007">
<p>Visel, A., Minovitsky, S., Dubchak, I., and Pennacchio, L. a (2007). VISTA Enhancer Browser–a database of tissue-specific human enhancers. Nucleic Acids Research <em>35</em>, D88–92.</p>
</div>
<div id="ref-Zhou2012">
<p>Zhou, X., and Wang, T. (2012). Using the Wash U Epigenome Browser to Examine Genome-Wide Sequencing Data. In Current Protocols in Bioinformatics, (Hoboken, NJ, USA: John Wiley &amp; Sons, Inc.), p. Unit10.10.</p>
</div>
<div id="ref-Quinlan2010">
<p>Quinlan, A.R., and Hall, I.M. (2010). BEDTools: a flexible suite of utilities for comparing genomic features. Bioinformatics <em>26</em>, 841–842.</p>
</div>
<div id="ref-Heger2013">
<p>Heger, A., Webber, C., Goodson, M., Ponting, C.P., and Lunter, G. (2013). GAT: a simulation framework for testing the association of genomic intervals. Bioinformatics (Oxford, England) <em>29</em>, 2046–2048.</p>
</div>
<div id="ref-Kohler2014">
<p>Köhler, S., Doelken, S.C., Mungall, C.J., Bauer, S., Firth, H.V., Bailleul-Forestier, I., Black, G.C.M., Brown, D.L., Brudno, M., Campbell, J., et al. (2014). The Human Phenotype Ontology project: linking molecular biology and disease through phenotype data. Nucleic Acids Research <em>42</em>, D966–74.</p>
</div>
<div id="ref-Firth2009">
<p>Firth, H.V., Richards, S.M., Bevan, a P., Clayton, S., Corpas, M., Rajan, D., Van Vooren, S., Moreau, Y., Pettett, R.M., and Carter, N.P. (2009). DECIPHER: Database of Chromosomal Imbalance and Phenotype in Humans Using Ensembl Resources. American Journal of Human Genetics <em>84</em>, 524–533.</p>
</div>
<div id="ref-Lappalainen2013">
<p>Lappalainen, I., Lopez, J., Skipper, L., Hefferon, T., Spalding, J.D., Garner, J., Chen, C., Maguire, M., Corbett, M., Zhou, G., et al. (2012). dbVar and DGVa: public archives for genomic structural variation. Nucleic Acids Research <em>41</em>, D936–D941.</p>
</div>
<div id="ref-Cardoso2016">
<p>Cardoso, A.R., Oliveira, M., Amorim, A., and Azevedo, L. (2016). Major influence of repetitive elements on disease-associated copy number variants (CNVs). Human Genomics <em>10</em>, 30.</p>
</div>
<div id="ref-Gu2008">
<p>Gu, W., Zhang, F., and Lupski, J.R. (2008). Mechanisms for human genomic rearrangements. PathoGenetics <em>1</em>, 4.</p>
</div>
<div id="ref-Ordulu2016">
<p>Ordulu, Z., Kammin, T., Brand, H., Pillalamarri, V., Redin, C.E., Collins, R.L., Blumenthal, I., Hanscom, C., Pereira, S., Crandall, B.F., et al. (2016). Structural Chromosomal Rearrangements Require Nucleotide-Level Resolution : Lessons from Next-Generation Sequencing in Prenatal Diagnosis. The American Journal of Human Genetics 1–19.</p>
</div>
<div id="ref-Lonsdale2013">
<p>Lonsdale, J., Thomas, J., Salvatore, M., Phillips, R., Lo, E., Shad, S., Hasz, R., Walters, G., Garcia, F., Young, N., et al. (2013). The Genotype-Tissue Expression (GTEx) project. Nature Genetics <em>45</em>, 580–585.</p>
</div>
<div id="ref-Vetro2015">
<p>Vetro, A., Dehghani, M.R., Kraoua, L., Giorda, R., Beri, S., Cardarelli, L., Merico, M., Manolakos, E., Parada-Bustamante, A., Castro, A., et al. (2015). Testis development in the absence of SRY: chromosomal rearrangements at SOX9 and SOX3. European Journal of Human Genetics <em>23</em>, 1025–1032.</p>
</div>
<div id="ref-Miller2010">
<p>Miller, D.T., Adam, M.P., Aradhya, S., Biesecker, L.G., Brothman, A.R., Carter, N.P., Church, D.M., Crolla, J.A., Eichler, E.E., Epstein, C.J., et al. (2010). Consensus Statement: Chromosomal Microarray Is a First-Tier Clinical Diagnostic Test for Individuals with Developmental Disabilities or Congenital Anomalies. The American Journal of Human Genetics <em>86</em>, 749–764.</p>
</div>
<div id="ref-Bonev2016">
<p>Bonev, B., and Cavalli, G. (2016). Organization and function of the 3D genome. Nature Reviews Genetics <em>17</em>, 661–678.</p>
</div>
<div id="ref-PachlopnikSchmid2012">
<p>Pachlopnik Schmid, J., Lemoine, R., Nehme, N., Cormier-Daire, V., Revy, P., Debeurme, F., Debré, M., Nitschke, P., Bole-Feysot, C., Legeai-Mallet, L., et al. (2012). Polymerase <span class="math inline">\(\epsilon\)</span>1 mutation in a human syndrome with facial dysmorphism, immunodeficiency, livedo, and short stature (“FILS syndrome”). The Journal of Experimental Medicine <em>209</em>, 2323–2330.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="3-6-declarations.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="5-loop-prediction.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"google": false,
"weibo": false,
"instapper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/ibn-salem/thesis/edit/master/04-position_effect.Rmd",
"text": "Edit"
},
"download": ["thesis.pdf", "thesis.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "";
    if (src === "" || src === "true") src = "https://cdn.bootcss.com/mathjax/2.7.1/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:" && /^https?:/.test(src))
      src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
