<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>Genome folding in evolution and disease</title>
  <meta name="description" content="This is a draft PhD thesis.">
  <meta name="generator" content="bookdown 0.6.1 and GitBook 2.6.7">

  <meta property="og:title" content="Genome folding in evolution and disease" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="This is a draft PhD thesis." />
  <meta name="github-repo" content="ibn-salem/thesis" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Genome folding in evolution and disease" />
  
  <meta name="twitter:description" content="This is a draft PhD thesis." />
  

<meta name="author" content="Jonas Ibn-Salem">


<meta name="date" content="2018-02-13">

  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  
<link rel="prev" href="5-1-introduction-3.html">
<link rel="next" href="5-3-discussion-2.html">
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />










<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Thesis</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="1" data-path="1-thesis-intro.html"><a href="1-thesis-intro.html"><i class="fa fa-check"></i><b>1</b> Introduction</a><ul>
<li class="chapter" data-level="1.1" data-path="1-1-regulation-of-gene-expression.html"><a href="1-1-regulation-of-gene-expression.html"><i class="fa fa-check"></i><b>1.1</b> Regulation of gene expression</a></li>
<li class="chapter" data-level="1.2" data-path="1-2-enhancers.html"><a href="1-2-enhancers.html"><i class="fa fa-check"></i><b>1.2</b> Enhancers</a></li>
<li class="chapter" data-level="1.3" data-path="1-3-methods-to-probe-the-3d-chromatin-archtiecture.html"><a href="1-3-methods-to-probe-the-3d-chromatin-archtiecture.html"><i class="fa fa-check"></i><b>1.3</b> Methods to probe the 3D chromatin archtiecture</a><ul>
<li class="chapter" data-level="1.3.1" data-path="1-3-methods-to-probe-the-3d-chromatin-archtiecture.html"><a href="1-3-methods-to-probe-the-3d-chromatin-archtiecture.html#microscopy-based-techniques-to-visuallize-the-genome-in-3d"><i class="fa fa-check"></i><b>1.3.1</b> Microscopy-based techniques to visuallize the genome in 3D</a></li>
<li class="chapter" data-level="1.3.2" data-path="1-3-methods-to-probe-the-3d-chromatin-archtiecture.html"><a href="1-3-methods-to-probe-the-3d-chromatin-archtiecture.html#proximity-ligation-based-method-to-quantify-chromatin-interactions"><i class="fa fa-check"></i><b>1.3.2</b> Proximity-ligation based method to quantify chromatin interactions</a></li>
</ul></li>
<li class="chapter" data-level="1.4" data-path="1-4-hierarchy-of-chromatin-3d-structure.html"><a href="1-4-hierarchy-of-chromatin-3d-structure.html"><i class="fa fa-check"></i><b>1.4</b> Hierarchy of chromatin 3D structure</a><ul>
<li class="chapter" data-level="1.4.1" data-path="1-4-hierarchy-of-chromatin-3d-structure.html"><a href="1-4-hierarchy-of-chromatin-3d-structure.html#chromosomal-territories-and-inter-chromosomal-contacts"><i class="fa fa-check"></i><b>1.4.1</b> Chromosomal territories and inter-chromosomal contacts</a></li>
<li class="chapter" data-level="1.4.2" data-path="1-4-hierarchy-of-chromatin-3d-structure.html"><a href="1-4-hierarchy-of-chromatin-3d-structure.html#ab-compartments"><i class="fa fa-check"></i><b>1.4.2</b> A/B compartments</a></li>
<li class="chapter" data-level="1.4.3" data-path="1-4-hierarchy-of-chromatin-3d-structure.html"><a href="1-4-hierarchy-of-chromatin-3d-structure.html#topologically-associating-domains-tads"><i class="fa fa-check"></i><b>1.4.3</b> Topologically associating domains (TADs)</a></li>
<li class="chapter" data-level="1.4.4" data-path="1-4-hierarchy-of-chromatin-3d-structure.html"><a href="1-4-hierarchy-of-chromatin-3d-structure.html#chromatin-loops"><i class="fa fa-check"></i><b>1.4.4</b> Chromatin loops</a></li>
</ul></li>
<li class="chapter" data-level="1.5" data-path="1-5-dynamics-of-chromatin-structure.html"><a href="1-5-dynamics-of-chromatin-structure.html"><i class="fa fa-check"></i><b>1.5</b> Dynamics of chromatin structure</a><ul>
<li class="chapter" data-level="1.5.1" data-path="1-5-dynamics-of-chromatin-structure.html"><a href="1-5-dynamics-of-chromatin-structure.html#variation-of-chromatin-structure-across-cell-types"><i class="fa fa-check"></i><b>1.5.1</b> Variation of chromatin structure across cell types</a></li>
</ul></li>
<li class="chapter" data-level="1.6" data-path="1-6-evolution-of-chromatin-organization.html"><a href="1-6-evolution-of-chromatin-organization.html"><i class="fa fa-check"></i><b>1.6</b> Evolution of chromatin organization</a></li>
<li class="chapter" data-level="1.7" data-path="1-7-disruption-of-chromatin-architecture-in-disease.html"><a href="1-7-disruption-of-chromatin-architecture-in-disease.html"><i class="fa fa-check"></i><b>1.7</b> Disruption of chromatin architecture in disease</a></li>
<li class="chapter" data-level="1.8" data-path="1-8-aims-of-this-thesis.html"><a href="1-8-aims-of-this-thesis.html"><i class="fa fa-check"></i><b>1.8</b> Aims of this thesis</a></li>
<li class="chapter" data-level="1.9" data-path="1-9-structure-of-this-thesis.html"><a href="1-9-structure-of-this-thesis.html"><i class="fa fa-check"></i><b>1.9</b> Structure of this thesis</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="2-paralog-regulation.html"><a href="2-paralog-regulation.html"><i class="fa fa-check"></i><b>2</b> Paralog genes in the 3D genome architecture</a><ul>
<li class="chapter" data-level="" data-path="2-paralog-regulation.html"><a href="2-paralog-regulation.html#preamble"><i class="fa fa-check"></i>Preamble</a></li>
<li class="chapter" data-level="" data-path="2-paralog-regulation.html"><a href="2-paralog-regulation.html#abstract-1"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="2.1" data-path="2-1-introduction.html"><a href="2-1-introduction.html"><i class="fa fa-check"></i><b>2.1</b> Introduction</a></li>
<li class="chapter" data-level="2.2" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html"><i class="fa fa-check"></i><b>2.2</b> Materials and methods</a><ul>
<li class="chapter" data-level="2.2.1" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html#selection-of-pairs-of-paralog-genes"><i class="fa fa-check"></i><b>2.2.1</b> Selection of pairs of paralog genes</a></li>
<li class="chapter" data-level="2.2.2" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html#enhancers-to-gene-association"><i class="fa fa-check"></i><b>2.2.2</b> Enhancers to gene association</a></li>
<li class="chapter" data-level="2.2.3" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html#topological-associating-domains"><i class="fa fa-check"></i><b>2.2.3</b> Topological associating domains</a></li>
<li class="chapter" data-level="2.2.4" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html#hi-c-interaction-maps"><i class="fa fa-check"></i><b>2.2.4</b> Hi-C interaction maps</a></li>
<li class="chapter" data-level="2.2.5" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html#randomization"><i class="fa fa-check"></i><b>2.2.5</b> Randomization</a></li>
<li class="chapter" data-level="2.2.6" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html#statistical-tests"><i class="fa fa-check"></i><b>2.2.6</b> Statistical tests</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="2-3-results.html"><a href="2-3-results.html"><i class="fa fa-check"></i><b>2.3</b> Results</a><ul>
<li class="chapter" data-level="2.3.1" data-path="2-3-results.html"><a href="2-3-results.html#distribution-of-paralog-genes-in-the-human-genome"><i class="fa fa-check"></i><b>2.3.1</b> Distribution of paralog genes in the human genome</a></li>
<li class="chapter" data-level="2.3.2" data-path="2-3-results.html"><a href="2-3-results.html#co-expression-of-paralog-gene-pairs-across-tissues"><i class="fa fa-check"></i><b>2.3.2</b> Co-expression of paralog gene pairs across tissues</a></li>
<li class="chapter" data-level="2.3.3" data-path="2-3-results.html"><a href="2-3-results.html#paralog-genes-share-enhancers"><i class="fa fa-check"></i><b>2.3.3</b> Paralog genes share enhancers</a></li>
<li class="chapter" data-level="2.3.4" data-path="2-3-results.html"><a href="2-3-results.html#co-localization-of-paralogs-in-tads"><i class="fa fa-check"></i><b>2.3.4</b> Co-localization of paralogs in TADs</a></li>
<li class="chapter" data-level="2.3.5" data-path="2-3-results.html"><a href="2-3-results.html#distal-paralog-pairs-are-enriched-for-long-range-chromatin-contacts"><i class="fa fa-check"></i><b>2.3.5</b> Distal paralog pairs are enriched for long-range chromatin contacts</a></li>
<li class="chapter" data-level="2.3.6" data-path="2-3-results.html"><a href="2-3-results.html#close-paralogs-have-fewer-contacts-than-expected"><i class="fa fa-check"></i><b>2.3.6</b> Close paralogs have fewer contacts than expected</a></li>
<li class="chapter" data-level="2.3.7" data-path="2-3-results.html"><a href="2-3-results.html#paralogs-in-mouse-and-dog-genome"><i class="fa fa-check"></i><b>2.3.7</b> Paralogs in mouse and dog genome</a></li>
<li class="chapter" data-level="2.3.8" data-path="2-3-results.html"><a href="2-3-results.html#orthologs-of-human-paralogs-show-conserved-co-localization"><i class="fa fa-check"></i><b>2.3.8</b> Orthologs of human paralogs show conserved co-localization</a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="2-4-discussions.html"><a href="2-4-discussions.html"><i class="fa fa-check"></i><b>2.4</b> Discussions</a></li>
<li class="chapter" data-level="2.5" data-path="2-5-conclusion.html"><a href="2-5-conclusion.html"><i class="fa fa-check"></i><b>2.5</b> Conclusion</a></li>
<li class="chapter" data-level="2.6" data-path="2-6-acknowledgements.html"><a href="2-6-acknowledgements.html"><i class="fa fa-check"></i><b>2.6</b> Acknowledgements</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="3-TAD-evolution.html"><a href="3-TAD-evolution.html"><i class="fa fa-check"></i><b>3</b> Stability of TADs in evolution</a><ul>
<li class="chapter" data-level="" data-path="3-TAD-evolution.html"><a href="3-TAD-evolution.html#preamble-1"><i class="fa fa-check"></i>Preamble</a></li>
<li class="chapter" data-level="" data-path="3-TAD-evolution.html"><a href="3-TAD-evolution.html#abstract-2"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="" data-path="3-TAD-evolution.html"><a href="3-TAD-evolution.html#keywords"><i class="fa fa-check"></i>Keywords</a></li>
<li class="chapter" data-level="3.1" data-path="3-1-introduction-1.html"><a href="3-1-introduction-1.html"><i class="fa fa-check"></i><b>3.1</b> Introduction</a></li>
<li class="chapter" data-level="3.2" data-path="3-2-results-1.html"><a href="3-2-results-1.html"><i class="fa fa-check"></i><b>3.2</b> Results</a><ul>
<li class="chapter" data-level="3.2.1" data-path="3-2-results-1.html"><a href="3-2-results-1.html#identification-of-evolutionary-rearrangement-breakpoints-from-whole-genome-alignments"><i class="fa fa-check"></i><b>3.2.1</b> Identification of evolutionary rearrangement breakpoints from whole-genome alignments</a></li>
<li class="chapter" data-level="3.2.2" data-path="3-2-results-1.html"><a href="3-2-results-1.html#rearrangement-breakpoints-are-enriched-at-tad-boundaries"><i class="fa fa-check"></i><b>3.2.2</b> Rearrangement breakpoints are enriched at TAD boundaries</a></li>
<li class="chapter" data-level="3.2.3" data-path="3-2-results-1.html"><a href="3-2-results-1.html#clusters-of-conserved-non-coding-elements-are-depleted-for-rearrangement-breakpoints"><i class="fa fa-check"></i><b>3.2.3</b> Clusters of conserved non-coding elements are depleted for rearrangement breakpoints</a></li>
<li class="chapter" data-level="3.2.4" data-path="3-2-results-1.html"><a href="3-2-results-1.html#rearranged-tads-are-associated-with-divergent-gene-expression-between-species"><i class="fa fa-check"></i><b>3.2.4</b> Rearranged TADs are associated with divergent gene expression between species</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="3-3-discussion.html"><a href="3-3-discussion.html"><i class="fa fa-check"></i><b>3.3</b> Discussion</a></li>
<li class="chapter" data-level="3.4" data-path="3-4-conclusion-1.html"><a href="3-4-conclusion-1.html"><i class="fa fa-check"></i><b>3.4</b> Conclusion</a></li>
<li class="chapter" data-level="3.5" data-path="3-5-methods.html"><a href="3-5-methods.html"><i class="fa fa-check"></i><b>3.5</b> Methods</a><ul>
<li class="chapter" data-level="3.5.1" data-path="3-5-methods.html"><a href="3-5-methods.html#rearrangement-breakpoints-from-whole-genome-alignments"><i class="fa fa-check"></i><b>3.5.1</b> Rearrangement breakpoints from whole-genome alignments</a></li>
<li class="chapter" data-level="3.5.2" data-path="3-5-methods.html"><a href="3-5-methods.html#topologically-associating-domains-and-contact-domains"><i class="fa fa-check"></i><b>3.5.2</b> Topologically associating domains and contact domains</a></li>
<li class="chapter" data-level="3.5.3" data-path="3-5-methods.html"><a href="3-5-methods.html#breakpoint-distributions-at-tads"><i class="fa fa-check"></i><b>3.5.3</b> Breakpoint distributions at TADs</a></li>
<li class="chapter" data-level="3.5.4" data-path="3-5-methods.html"><a href="3-5-methods.html#quantification-of-breakpoint-enrichment"><i class="fa fa-check"></i><b>3.5.4</b> Quantification of breakpoint enrichment</a></li>
<li class="chapter" data-level="3.5.5" data-path="3-5-methods.html"><a href="3-5-methods.html#expression-data-for-mouse-and-human-orthologs"><i class="fa fa-check"></i><b>3.5.5</b> Expression data for mouse and human orthologs</a></li>
<li class="chapter" data-level="3.5.6" data-path="3-5-methods.html"><a href="3-5-methods.html#classification-of-tads-and-genes-according-to-rearrangements-and-grbs"><i class="fa fa-check"></i><b>3.5.6</b> Classification of TADs and genes according to rearrangements and GRBs</a></li>
<li class="chapter" data-level="3.5.7" data-path="3-5-methods.html"><a href="3-5-methods.html#source-code-and-implementation-details"><i class="fa fa-check"></i><b>3.5.7</b> Source code and implementation details</a></li>
</ul></li>
<li class="chapter" data-level="3.6" data-path="3-6-declarations.html"><a href="3-6-declarations.html"><i class="fa fa-check"></i><b>3.6</b> Declarations</a><ul>
<li class="chapter" data-level="3.6.1" data-path="3-6-declarations.html"><a href="3-6-declarations.html#ethics-approval-and-consent-to-participate"><i class="fa fa-check"></i><b>3.6.1</b> Ethics approval and consent to participate</a></li>
<li class="chapter" data-level="3.6.2" data-path="3-6-declarations.html"><a href="3-6-declarations.html#consent-for-publication"><i class="fa fa-check"></i><b>3.6.2</b> Consent for publication</a></li>
<li class="chapter" data-level="3.6.3" data-path="3-6-declarations.html"><a href="3-6-declarations.html#availability-of-data-and-material"><i class="fa fa-check"></i><b>3.6.3</b> Availability of data and material</a></li>
<li class="chapter" data-level="3.6.4" data-path="3-6-declarations.html"><a href="3-6-declarations.html#competing-interests"><i class="fa fa-check"></i><b>3.6.4</b> Competing interests</a></li>
<li class="chapter" data-level="3.6.5" data-path="3-6-declarations.html"><a href="3-6-declarations.html#funding"><i class="fa fa-check"></i><b>3.6.5</b> Funding</a></li>
<li class="chapter" data-level="3.6.6" data-path="3-6-declarations.html"><a href="3-6-declarations.html#authors-contributions"><i class="fa fa-check"></i><b>3.6.6</b> Authors‘ contributions</a></li>
<li class="chapter" data-level="3.6.7" data-path="3-6-declarations.html"><a href="3-6-declarations.html#acknowledgments"><i class="fa fa-check"></i><b>3.6.7</b> Acknowledgments</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="4-position-effect.html"><a href="4-position-effect.html"><i class="fa fa-check"></i><b>4</b> Position effects of rearrangements in disease genomes</a><ul>
<li class="chapter" data-level="" data-path="4-position-effect.html"><a href="4-position-effect.html#preamble-2"><i class="fa fa-check"></i>Preamble</a></li>
<li class="chapter" data-level="" data-path="4-position-effect.html"><a href="4-position-effect.html#abstract-3"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="4.1" data-path="4-1-introduction-2.html"><a href="4-1-introduction-2.html"><i class="fa fa-check"></i><b>4.1</b> Introduction</a></li>
<li class="chapter" data-level="4.2" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html"><i class="fa fa-check"></i><b>4.2</b> Materials and Methods</a><ul>
<li class="chapter" data-level="4.2.1" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html#selection-of-subjects-with-apparently-balanced-chromosome-abnormalities"><i class="fa fa-check"></i><b>4.2.1</b> Selection of subjects with apparently balanced chromosome abnormalities</a></li>
<li class="chapter" data-level="4.2.2" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html#clinical-descriptions-of-dgap-cases"><i class="fa fa-check"></i><b>4.2.2</b> Clinical descriptions of DGAP cases</a></li>
<li class="chapter" data-level="4.2.3" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html#analysis-of-genes-bordering-the-rearrangement-breakpoints"><i class="fa fa-check"></i><b>4.2.3</b> Analysis of genes bordering the rearrangement breakpoints</a></li>
<li class="chapter" data-level="4.2.4" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html#assessment-of-disrupted-functional-elements-and-chromatin-interactions-bordering-rearrangement-breakpoints"><i class="fa fa-check"></i><b>4.2.4</b> Assessment of disrupted functional elements and chromatin interactions bordering rearrangement breakpoints</a></li>
<li class="chapter" data-level="4.2.5" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html#ontological-analysis-of-genes-neighboring-breakpoints"><i class="fa fa-check"></i><b>4.2.5</b> Ontological analysis of genes neighboring breakpoints</a></li>
<li class="chapter" data-level="4.2.6" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html#quantitative-real-time-pcr"><i class="fa fa-check"></i><b>4.2.6</b> Quantitative real-time PCR</a></li>
<li class="chapter" data-level="4.2.7" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html#assessment-of-dgap-breakpoints-overlapping-with-non-coding-structural-variants-in-public-databases"><i class="fa fa-check"></i><b>4.2.7</b> Assessment of DGAP breakpoints overlapping with non-coding structural variants in public databases</a></li>
</ul></li>
<li class="chapter" data-level="4.3" data-path="4-3-results-2.html"><a href="4-3-results-2.html"><i class="fa fa-check"></i><b>4.3</b> Results</a><ul>
<li class="chapter" data-level="4.3.1" data-path="4-3-results-2.html"><a href="4-3-results-2.html#genomic-characterization-of-non-coding-breakpoints"><i class="fa fa-check"></i><b>4.3.1</b> Genomic characterization of non-coding breakpoints</a></li>
<li class="chapter" data-level="4.3.2" data-path="4-3-results-2.html"><a href="4-3-results-2.html#identification-of-genes-with-potential-position-effects"><i class="fa fa-check"></i><b>4.3.2</b> Identification of genes with potential position effects</a></li>
<li class="chapter" data-level="4.3.3" data-path="4-3-results-2.html"><a href="4-3-results-2.html#identification-of-subjects-with-shared-non-coding-chromosome-alterations-and-phenotypes"><i class="fa fa-check"></i><b>4.3.3</b> Identification of subjects with shared non-coding chromosome alterations and phenotypes</a></li>
</ul></li>
<li class="chapter" data-level="4.4" data-path="4-4-discussion-1.html"><a href="4-4-discussion-1.html"><i class="fa fa-check"></i><b>4.4</b> Discussion</a><ul>
<li class="chapter" data-level="" data-path="4-4-discussion-1.html"><a href="4-4-discussion-1.html#acknowledgements-1"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="4.4.1" data-path="4-4-discussion-1.html"><a href="4-4-discussion-1.html#web-resources"><i class="fa fa-check"></i><b>4.4.1</b> Web Resources</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="5-loop.html"><a href="5-loop.html"><i class="fa fa-check"></i><b>5</b> Prediction of chromatin looping interactions</a><ul>
<li class="chapter" data-level="" data-path="5-loop.html"><a href="5-loop.html#preamble-3"><i class="fa fa-check"></i>Preamble</a></li>
<li class="chapter" data-level="" data-path="5-loop.html"><a href="5-loop.html#abstract-4"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="" data-path="5-loop.html"><a href="5-loop.html#keywords-1"><i class="fa fa-check"></i>Keywords</a></li>
<li class="chapter" data-level="5.1" data-path="5-1-introduction-3.html"><a href="5-1-introduction-3.html"><i class="fa fa-check"></i><b>5.1</b> Introduction</a></li>
<li class="chapter" data-level="5.2" data-path="5-2-results-3.html"><a href="5-2-results-3.html"><i class="fa fa-check"></i><b>5.2</b> Results</a><ul>
<li class="chapter" data-level="5.2.1" data-path="5-2-results-3.html"><a href="5-2-results-3.html#ctcf-motif-pairs-as-candidate-chromatin-loop-anchors"><i class="fa fa-check"></i><b>5.2.1</b> CTCF motif pairs as candidate chromatin loop anchors</a></li>
<li class="chapter" data-level="5.2.2" data-path="5-2-results-3.html"><a href="5-2-results-3.html#similarity-of-chip-seq-signals-at-looping-ctcf-motifs"><i class="fa fa-check"></i><b>5.2.2</b> Similarity of ChIP-seq signals at looping CTCF motifs</a></li>
<li class="chapter" data-level="5.2.3" data-path="5-2-results-3.html"><a href="5-2-results-3.html#genomic-sequence-features-of-ctcf-motif-pairs-are-associated-with-looping"><i class="fa fa-check"></i><b>5.2.3</b> Genomic sequence features of CTCF motif pairs are associated with looping</a></li>
<li class="chapter" data-level="5.2.4" data-path="5-2-results-3.html"><a href="5-2-results-3.html#chromatin-loop-prediction-using-7c"><i class="fa fa-check"></i><b>5.2.4</b> Chromatin loop prediction using 7C</a></li>
<li class="chapter" data-level="5.2.5" data-path="5-2-results-3.html"><a href="5-2-results-3.html#prediction-performance-evaluation"><i class="fa fa-check"></i><b>5.2.5</b> Prediction performance evaluation</a></li>
<li class="chapter" data-level="5.2.6" data-path="5-2-results-3.html"><a href="5-2-results-3.html#prediction-performance-of-7c-with-sequence-features-and-single-tf-chip-seq-data-sets"><i class="fa fa-check"></i><b>5.2.6</b> Prediction performance of 7C with sequence features and single TF ChIP-seq data sets</a></li>
<li class="chapter" data-level="5.2.7" data-path="5-2-results-3.html"><a href="5-2-results-3.html#comparison-of-transcription-factors-by-prediction-performance"><i class="fa fa-check"></i><b>5.2.7</b> Comparison of transcription factors by prediction performance</a></li>
<li class="chapter" data-level="5.2.8" data-path="5-2-results-3.html"><a href="5-2-results-3.html#prediction-performance-in-other-cell-types-and-for-different-tfs"><i class="fa fa-check"></i><b>5.2.8</b> Prediction performance in other cell types and for different TFs</a></li>
<li class="chapter" data-level="5.2.9" data-path="5-2-results-3.html"><a href="5-2-results-3.html#the-high-resolution-of-chip-nexus-improves-prediction-performance"><i class="fa fa-check"></i><b>5.2.9</b> The high resolution of ChIP-nexus improves prediction performance</a></li>
</ul></li>
<li class="chapter" data-level="5.3" data-path="5-3-discussion-2.html"><a href="5-3-discussion-2.html"><i class="fa fa-check"></i><b>5.3</b> Discussion</a></li>
<li class="chapter" data-level="5.4" data-path="5-4-conclusion-2.html"><a href="5-4-conclusion-2.html"><i class="fa fa-check"></i><b>5.4</b> Conclusion</a></li>
<li class="chapter" data-level="5.5" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html"><i class="fa fa-check"></i><b>5.5</b> Methods</a><ul>
<li class="chapter" data-level="5.5.1" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#ctcf-motifs-in-the-human-genome"><i class="fa fa-check"></i><b>5.5.1</b> CTCF motifs in the human genome</a></li>
<li class="chapter" data-level="5.5.2" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#loop-interaction-data-for-training-and-validation"><i class="fa fa-check"></i><b>5.5.2</b> Loop interaction data for training and validation</a></li>
<li class="chapter" data-level="5.5.3" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#chip-seq-datasets-in-gm12878-cells"><i class="fa fa-check"></i><b>5.5.3</b> ChIP-seq datasets in GM12878 cells</a></li>
<li class="chapter" data-level="5.5.4" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#chip-seq-data-types"><i class="fa fa-check"></i><b>5.5.4</b> ChIP-seq data types</a></li>
<li class="chapter" data-level="5.5.5" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#chip-nexus-data-processing-for-rad21-and-scm3"><i class="fa fa-check"></i><b>5.5.5</b> ChIP-nexus data processing for RAD21 and SCM3</a></li>
<li class="chapter" data-level="5.5.6" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#similarity-of-chip-seq-profiles-as-correlation-of-coverage-around-motifs"><i class="fa fa-check"></i><b>5.5.6</b> Similarity of ChIP-seq profiles as correlation of coverage around motifs</a></li>
<li class="chapter" data-level="5.5.7" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#genomic-sequence-features-of-chromatin-loops"><i class="fa fa-check"></i><b>5.5.7</b> Genomic sequence features of chromatin loops</a></li>
<li class="chapter" data-level="5.5.8" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#chromatin-loop-prediction-model"><i class="fa fa-check"></i><b>5.5.8</b> Chromatin Loop prediction model</a></li>
<li class="chapter" data-level="5.5.9" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#training-and-validation-of-prediction-model"><i class="fa fa-check"></i><b>5.5.9</b> Training and validation of prediction model</a></li>
<li class="chapter" data-level="5.5.10" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#analysis-of-prediction-performance"><i class="fa fa-check"></i><b>5.5.10</b> Analysis of prediction performance</a></li>
<li class="chapter" data-level="5.5.11" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#implementation-of-7c-and-compatibility-to-other-tools"><i class="fa fa-check"></i><b>5.5.11</b> Implementation of 7C and compatibility to other tools</a></li>
</ul></li>
<li class="chapter" data-level="5.6" data-path="5-6-declarations-1.html"><a href="5-6-declarations-1.html"><i class="fa fa-check"></i><b>5.6</b> Declarations</a><ul>
<li class="chapter" data-level="5.6.1" data-path="5-6-declarations-1.html"><a href="5-6-declarations-1.html#ethics-approval-and-consent-to-participate-1"><i class="fa fa-check"></i><b>5.6.1</b> Ethics approval and consent to participate</a></li>
<li class="chapter" data-level="5.6.2" data-path="5-6-declarations-1.html"><a href="5-6-declarations-1.html#consent-for-publication-1"><i class="fa fa-check"></i><b>5.6.2</b> Consent for publication</a></li>
<li class="chapter" data-level="5.6.3" data-path="5-6-declarations-1.html"><a href="5-6-declarations-1.html#availability-of-data-and-material-1"><i class="fa fa-check"></i><b>5.6.3</b> Availability of data and material</a></li>
<li class="chapter" data-level="5.6.4" data-path="5-6-declarations-1.html"><a href="5-6-declarations-1.html#competing-interests-1"><i class="fa fa-check"></i><b>5.6.4</b> Competing interests</a></li>
<li class="chapter" data-level="5.6.5" data-path="5-6-declarations-1.html"><a href="5-6-declarations-1.html#funding-1"><i class="fa fa-check"></i><b>5.6.5</b> Funding</a></li>
<li class="chapter" data-level="5.6.6" data-path="5-6-declarations-1.html"><a href="5-6-declarations-1.html#authors-contributions-1"><i class="fa fa-check"></i><b>5.6.6</b> Authors‘ contributions</a></li>
<li class="chapter" data-level="5.6.7" data-path="5-6-declarations-1.html"><a href="5-6-declarations-1.html#acknowledgments-1"><i class="fa fa-check"></i><b>5.6.7</b> Acknowledgments</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="6-thesis-discussion.html"><a href="6-thesis-discussion.html"><i class="fa fa-check"></i><b>6</b> Discussion</a><ul>
<li class="chapter" data-level="6.1" data-path="6-1-discussion-paralog-co-regulation-in-tads.html"><a href="6-1-discussion-paralog-co-regulation-in-tads.html"><i class="fa fa-check"></i><b>6.1</b> Discussion paralog co-regulation in TADs</a></li>
<li class="chapter" data-level="6.2" data-path="6-2-discussion-on-tad-evolution.html"><a href="6-2-discussion-on-tad-evolution.html"><i class="fa fa-check"></i><b>6.2</b> Discussion on TAD evolution</a></li>
<li class="chapter" data-level="6.3" data-path="6-3-further-directions.html"><a href="6-3-further-directions.html"><i class="fa fa-check"></i><b>6.3</b> Further directions</a></li>
<li class="chapter" data-level="6.4" data-path="6-4-conclusions.html"><a href="6-4-conclusions.html"><i class="fa fa-check"></i><b>6.4</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="appendix"><span><b>Appendix</b></span></li>
<li class="chapter" data-level="A" data-path="A-sup-paralog-regulation.html"><a href="A-sup-paralog-regulation.html"><i class="fa fa-check"></i><b>A</b> Co-regulation of paralog genes: Supporting Information</a></li>
<li class="chapter" data-level="B" data-path="B-sup-tad-evolution.html"><a href="B-sup-tad-evolution.html"><i class="fa fa-check"></i><b>B</b> TAD evolution: Supporting Information</a><ul>
<li class="chapter" data-level="B.1" data-path="B-1-TadEvoSupTab.html"><a href="B-1-TadEvoSupTab.html"><i class="fa fa-check"></i><b>B.1</b> Supplementary Tables</a></li>
<li class="chapter" data-level="B.2" data-path="B-2-supplementary-figures.html"><a href="B-2-supplementary-figures.html"><i class="fa fa-check"></i><b>B.2</b> Supplementary Figures</a></li>
</ul></li>
<li class="chapter" data-level="C" data-path="C-sup-position-effect.html"><a href="C-sup-position-effect.html"><i class="fa fa-check"></i><b>C</b> Position Effect: Supplemental Data</a><ul>
<li class="chapter" data-level="C.1" data-path="C-1-supplemental-note-case-reports.html"><a href="C-1-supplemental-note-case-reports.html"><i class="fa fa-check"></i><b>C.1</b> Supplemental Note: Case Reports</a></li>
<li class="chapter" data-level="C.2" data-path="C-2-supplemental-note-nucleotide-level-nomenclature-for-dgap-karyotypes.html"><a href="C-2-supplemental-note-nucleotide-level-nomenclature-for-dgap-karyotypes.html"><i class="fa fa-check"></i><b>C.2</b> Supplemental Note: Nucleotide Level Nomenclature for DGAP karyotypes</a></li>
<li class="chapter" data-level="C.3" data-path="C-3-supplemental-figure.html"><a href="C-3-supplemental-figure.html"><i class="fa fa-check"></i><b>C.3</b> Supplemental Figure</a></li>
<li class="chapter" data-level="C.4" data-path="C-4-position-effect-sub-tab.html"><a href="C-4-position-effect-sub-tab.html"><i class="fa fa-check"></i><b>C.4</b> Supplemental Table Legends</a></li>
</ul></li>
<li class="chapter" data-level="D" data-path="D-sup-loop.html"><a href="D-sup-loop.html"><i class="fa fa-check"></i><b>D</b> Loop prediction: Supplemental Information</a><ul>
<li class="chapter" data-level="D.1" data-path="D-1-LoopPredSupTab.html"><a href="D-1-LoopPredSupTab.html"><i class="fa fa-check"></i><b>D.1</b> Supplementary Tables</a></li>
<li class="chapter" data-level="D.2" data-path="D-2-supplementary-figures-1.html"><a href="D-2-supplementary-figures-1.html"><i class="fa fa-check"></i><b>D.2</b> Supplementary Figures</a></li>
</ul></li>
<li class="chapter" data-level="E" data-path="E-contribution-to-individual-publications.html"><a href="E-contribution-to-individual-publications.html"><i class="fa fa-check"></i><b>E</b> Contribution to individual publications</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Build with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Genome folding in evolution and disease</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="results-3" class="section level2">
<h2><span class="header-section-number">5.2</span> Results</h2>
<div id="ctcf-motif-pairs-as-candidate-chromatin-loop-anchors" class="section level3">
<h3><span class="header-section-number">5.2.1</span> CTCF motif pairs as candidate chromatin loop anchors</h3>
<p>In order to predict chromatin looping interactions from ChIP-seq data,
we first analyzed which features at looping anchors correlate with
interaction signals. As a starting point for all analyses we used 38,316
CTCF motif sites in the human genome as potential chromatin loop
anchors. We built a dataset of all CTCF motif pairs located within a
genomic distance of 1 Mb to each other. This resulted in 717,137
potential looping interactions; we expect that only a minority of these
motif pairs will be in physical contact for a given cell type and
condition. To label motif pairs as true loops, we used chromatin loops
from published high-resolution <em>in-situ</em> Hi-C data and ChIA-PET data for
CTCF and Pol2 in human GM12878 cells <span class="citation">[Rao et al., <a href="#ref-Rao2014">2014</a>; Tang et al., <a href="#ref-Tang2015">2015</a>]</span>. If a motif pair was
measured to interact in one of the data sets, we labeled it as true
interaction. Overall 30,025 (4.19 %) of CTCF motif pairs were considered
as true loops using these data sets.</p>
</div>
<div id="similarity-of-chip-seq-signals-at-looping-ctcf-motifs" class="section level3">
<h3><span class="header-section-number">5.2.2</span> Similarity of ChIP-seq signals at looping CTCF motifs</h3>
<p>The ChIP-seq protocol involves a cross-linking step, in which
formaldehyde treatment results in covalent bonds between DNA and
proteins <span class="citation">[Orlando et al., <a href="#ref-Orlando1997">1997</a>]</span>. This allows the pull-down and detection of sites
directly bound by the targeted protein. However, cross-linking occurs
also between proteins, which results in detection of sites that are
indirectly bound through protein-protein interactions or chromatin
looping interactions <span class="citation">[Hoffman et al., <a href="#ref-Hoffman2015">2015</a>; Starick et al., <a href="#ref-Starick2015">2015</a>]</span>.</p>
<p>We hypothesized that if a protein binds directly to a genomic region in
chromatin contact with other genomic regions, DNA from both loci might
be pulled out in the cross-linking and DNA-purification step of ChIP-seq
protocols. As a result, we expect ChIP-seq signals (e.g. mapped reads)
at both genomic regions: the directly bound one and the chromatin loop
interaction partner locus (Fig. <a href="5-1-introduction-3.html#fig:LoopPred1">5.1</a>A).</p>
<p>To test this hypothesis, we used CTCF motif pairs as anchors and
compared the ChIP-seq signal from one anchor to the (reversed) signal of
the corresponding anchor. We found similar ChIP-seq coverage patterns
around CTCF motifs more often when the two sites perform looping
interactions than when they do not (Fig. <a href="5-1-introduction-3.html#fig:LoopPred1">5.1</a>B). To quantify the similarity
of ChIP-seq coverage from any two CTCF sites, we correlated their
ChIP-seq signals at +/- 500 bp around the CTCF motif (Fig. <a href="5-1-introduction-3.html#fig:LoopPred1">5.1</a>C) (see
Methods for details). Measuring ChIP-seq profile similarity by
correlation has the advantage that the correlation can be high even if
the anchor that is not bound directly has a much lower ChIP-seq signal
(which is often the case).</p>
<p>Next, we compared ChIP-seq similarity at looping and non-looping CTCF
motif pairs for six selected TF ChIP-seq data sets (Fig. <a href="5-1-introduction-3.html#fig:LoopPred1">5.1</a>D). Compared
to non-interacting CTCF sites the ChIP-seq correlation is significantly
higher at looping interactions (Fig. <a href="5-2-results-3.html#fig:LoopPred2">5.2</a>A). However, the overall
correlation as well as the difference between looping and non-looping
CTCF sites varies between TF ChIP-seq datasets (Fig. <a href="5-2-results-3.html#fig:LoopPred2">5.2</a>A). As expected,
we observed a large difference for the CTCF ChIP-seq dataset but,
interestingly, also for other known architectural proteins, such as
Rad21 and Znf143. Moreover, other TFs, such as STAT1 have significantly
higher ChIP-seq signal similarity at CTCF motifs that interact via
chromatin looping. Overall, this analysis shows that ChIP-seq signals
are more similar at interacting CTCF sites, indicating that this
similarity can be used to predict looping interactions.</p>
<!-- ---------------------------------------------------------------- -->
<div class="figure" style="text-align: center"><span id="fig:LoopPred2"></span>
<img src="figures/loop_prediction/fig2_v04.png" alt="ChIP-seq similarity and genomic features of looping and non-looping CTCF motif pairs. (A) Boxplot of Pearson correlation coefficient of ChIP-seq signals between CTCF motif pairs for all CTCF motif pairs within 1 Mb genome-wide. The correlation is shown separately for non-looping and looping motif pairs (according to HI-C and ChIA-PET data in GM12878 cells), and for six selected ChIP-seq data sets in GM12878 cells. (B) Distance distribution between looping (green) and non-looping CTCF motif pairs. (C) Number of looping and non-looping CTCF motif pairs in convergent, divergent, both forward, or both reverse orientation. (D) Distribution of CTCF motif hit significance as \(-\log_{10}\) transformed p-value for looping and non-looping CTCF motif pairs. For each motif pair only the less significant motif is considered." width="100%" />
<p class="caption">
Figure 5.2: <strong>ChIP-seq similarity and genomic features of looping and non-looping CTCF motif pairs.</strong> <strong>(A)</strong> Boxplot of Pearson correlation coefficient of ChIP-seq signals between CTCF motif pairs for all CTCF motif pairs within 1 Mb genome-wide. The correlation is shown separately for non-looping and looping motif pairs (according to HI-C and ChIA-PET data in GM12878 cells), and for six selected ChIP-seq data sets in GM12878 cells. <strong>(B)</strong> Distance distribution between looping (green) and non-looping CTCF motif pairs. <strong>(C)</strong> Number of looping and non-looping CTCF motif pairs in convergent, divergent, both forward, or both reverse orientation. <strong>(D)</strong> Distribution of CTCF motif hit significance as <span class="math inline">\(-\log_{10}\)</span> transformed p-value for looping and non-looping CTCF motif pairs. For each motif pair only the less significant motif is considered.
</p>
</div>

<!-- ---------------------------------------------------------------- -->
</div>
<div id="genomic-sequence-features-of-ctcf-motif-pairs-are-associated-with-looping" class="section level3">
<h3><span class="header-section-number">5.2.3</span> Genomic sequence features of CTCF motif pairs are associated with looping</h3>
<p>The frequency of two genomic regions to physically interact depends on
their genomic distance <span class="citation">[Lieberman-Aiden et al., <a href="#ref-Lieberman-Aiden2009">2009</a>]</span>. Consequently, we observed that CTCF motif
pairs are more often in contact when they are close to each other in the
genomic sequence (Fig. <a href="5-2-results-3.html#fig:LoopPred2">5.2</a>B). Recent studies on 3D chromatin structure led
to an increased understanding of the molecular mechanism of chromatin
loop formation and suggested a functional role of CTCF proteins, which
bind specific DNA sequences <span class="citation">[Merkenschlager and Nora, <a href="#ref-Merkenschlager2016">2016</a>]</span>. The canonical CTCF motif is
non-palindromic and therefore occurs either in the positive or in the
negative DNA strand. Importantly, it is known that CTCF motifs occur
predominantly in convergent orientation to each other at chromatin loop
anchors <span class="citation">[Rao et al., <a href="#ref-Rao2014">2014</a>; Vietri Rudan et al., <a href="#ref-VietriRudan2015">2015</a>]</span>. Experimental inversions of CTCF motifs lead to
changes of the interactions and expression of the associated genes
<span class="citation">[Guo et al., <a href="#ref-Guo2015">2015</a>; Wit et al., <a href="#ref-deWit2015">2015</a>]</span>. Accordingly, we observed that 55.6% of the looping CTCF pairs
have convergent orientation versus only 25.3% of the non-looping pairs
(Fig. <a href="5-2-results-3.html#fig:LoopPred2">5.2</a>C). We also observed that the motif match strength, as measured
by the significance of a motif location to match the canonical CTCF
motif <span class="citation">[Khan et al., <a href="#ref-Khan2018">2018</a>]</span>, is higher for motifs involved in looping interactions
(Fig. <a href="5-2-results-3.html#fig:LoopPred2">5.2</a>D). Together, the linear genome encodes several features, such as
motif strength, orientation, and distance, that correlate with chromatin
looping and can be used to predict such interactions.</p>
</div>
<div id="chromatin-loop-prediction-using-7c" class="section level3">
<h3><span class="header-section-number">5.2.4</span> Chromatin loop prediction using 7C</h3>
<p>To make use of both the condition specific ChIP-seq signals and the
genomic features of CTCF motifs to predict chromatin loops, we
trained a prediction model that takes only ChIP-seq data as input.
To this end, we built a logistic regression model that takes into
account only four features: the correlation coefficient between the
ChIP-seq signals of the paired CTCF motifs (in a window of 1000 bp
around the motif), the genomic distance between motifs, the
orientation, and the (minimum) motif hit significance score (see
Methods for details). For each ChIP-seq data set, we trained and
evaluated a separate model. The method is implemented as the R
package ‘sevenC’, which predicts chromatin loops using as only input
a bigWig file from a ChIP-seq experiment.</p>
</div>
<div id="prediction-performance-evaluation" class="section level3">
<h3><span class="header-section-number">5.2.5</span> Prediction performance evaluation</h3>
<p>We used 10-fold cross-validation to assess the performance of the
predictions on independent data that was not seen in the training phase.
For each cutoff on the predicted interaction probability score, we
computed the sensitivity, specificity, precision and recall to plot
receiver operator characteristic (ROC) and precision recall curves
(PRC). Since only 4.2% of CTCF pairs are measured to interact, we mainly
used the area under the PRC (auPRC) to evaluate prediction performance
since, compared to ROC, the PRC gives a more accurate classification
performance in imbalanced datasets in which the number of negatives
outweighs the number of positives significantly <span class="citation">[Saito and Rehmsmeier, <a href="#ref-Saito2015">2015</a>]</span>. Furthermore, we
defined an optimal cutoff for the prediction probability <em>p</em> based on
optimizing the f1-score. The six selected TF ChIP-seq data sets have
optimal f1-scores at about <em>p</em> = 0.15 (Figure S1B). For binary
prediction, we provide a default prediction score threshold as the
average of thresholds with optimal f1-score for the 10 best performing
TF ChIP-seq datasets.</p>
</div>
<div id="prediction-performance-of-7c-with-sequence-features-and-single-tf-chip-seq-data-sets" class="section level3">
<h3><span class="header-section-number">5.2.6</span> Prediction performance of 7C with sequence features and single TF ChIP-seq data sets</h3>
<p>First, we evaluated how the sequence-encoded features can predict
chromatin interactions. For this, we built regression models that use
only these features. Each of these features alone, CTCF motif hit
significance, motif orientation or distance, were very poor predictors,
and resulted in auROC between 0.67 and 0.74 (Fig. <a href="5-2-results-3.html#fig:LoopPred3">5.3</a>A) and auPRC scores
between 0.08 and 0.09 (Fig. <a href="5-2-results-3.html#fig:LoopPred3">5.3</a>C). Using the three sequence features
together improved prediction performance (auROC = 0.85, auPRC = 0.22).</p>
<!-- ---------------------------------------------------------------- -->
<div class="figure" style="text-align: center"><span id="fig:LoopPred3"></span>
<img src="figures/loop_prediction/fig3_v05.png" alt="Prediction performance using cross-validation. (A) ROC plot for different models to predict chromatin looping interactions. The sensitivity (y-axis) is shown against the false discovery rate (1 – specificity, x-axis) for thresholds of the prediction score. Curves show averages of 10-fold cross-validation experiments. The models “Dist”, “Orientation”, and “Motif” contain only a single feature as indicated and all three genomic features are combined in the model “Dist+Orientation+Motif”. The models “RAD21”, CTCF”, “ZNF143”, “STAT1”, “EP300”, and “POLR2A”, contain the genomic features and the ChIP-seq correlation of the indicated factor. The model “all_TF” contains the genomic features and correlation of all indicated TFs. The model “across_TFs” contains the genomic features and a single correlation feature across the six ChIP-seq datasets as described in the main text. (B) PRC plot of precision against the recall for different prediction models. Color code as in (A). (C) Values of the area under the ROC (top) and PRC curves (bottom) as prediction performance. Error bars indicate standard deviation in 10-fold cross-validation experiments. (D) Example region on chromosome 11 in the genome browser showing: human genes, RAD21 ChIP-seq data in GM12878, CTCF motifs, CTCF motif pairs with that interact according to Hi-C or ChIA-PET data (green arcs) and predicted chromatin loops from RAD21 ChIPseq data using 7C (blue arcs). (E) Prediction performance of 7C as auPRC values for models with 124 TF ChIP-seq data sets from ENCODE. Error bars as in (C)." width="100%" />
<p class="caption">
Figure 5.3: <strong>Prediction performance using cross-validation. (A)</strong> ROC plot for different models to predict chromatin looping interactions. The sensitivity (y-axis) is shown against the false discovery rate (1 – specificity, x-axis) for thresholds of the prediction score. Curves show averages of 10-fold cross-validation experiments. The models “Dist”, “Orientation”, and “Motif” contain only a single feature as indicated and all three genomic features are combined in the model “Dist+Orientation+Motif”. The models “RAD21”, CTCF”, “ZNF143”, “STAT1”, “EP300”, and “POLR2A”, contain the genomic features and the ChIP-seq correlation of the indicated factor. The model “all_TF” contains the genomic features and correlation of all indicated TFs. The model “across_TFs” contains the genomic features and a single correlation feature across the six ChIP-seq datasets as described in the main text. <strong>(B)</strong> PRC plot of precision against the recall for different prediction models. Color code as in (A). <strong>(C)</strong> Values of the area under the ROC (top) and PRC curves (bottom) as prediction performance. Error bars indicate standard deviation in 10-fold cross-validation experiments. <strong>(D)</strong> Example region on chromosome 11 in the genome browser showing: human genes, RAD21 ChIP-seq data in GM12878, CTCF motifs, CTCF motif pairs with that interact according to Hi-C or ChIA-PET data (green arcs) and predicted chromatin loops from RAD21 ChIPseq data using 7C (blue arcs). <strong>(E)</strong> Prediction performance of 7C as auPRC values for models with 124 TF ChIP-seq data sets from ENCODE. Error bars as in (C).
</p>
</div>

<!-- ---------------------------------------------------------------- -->
<p>Next, we tested the addition of ChIP-seq data as feature in the
prediction model using ChIP-seq data for each of six different TFs.
Three of them, CTCF, RAD21, and ZNF143, have known function in chromatin
loop formation <span class="citation">[Merkenschlager and Nora, <a href="#ref-Merkenschlager2016">2016</a>; Ye et al., <a href="#ref-Ye2016b">2016</a>; Bailey et al., <a href="#ref-Bailey2015">2015</a>]</span>, while STAT1, P300, and POL2, are to our
knowledge not directly involved in chromatin loop formation (Fig. <a href="5-1-introduction-3.html#fig:LoopPred1">5.1</a>D).
Adding any of these TF ChIP-seq datasets to the model increased
prediction performance. STAT1, EP300, and POL2 only moderately increased
prediction performance with auROC values between 0.86 and 0.87 (Fig. <a href="5-2-results-3.html#fig:LoopPred3">5.3</a>A)
and auPRC between 0.24 and 0.26 (Fig. <a href="5-2-results-3.html#fig:LoopPred3">5.3</a>B, C). However, ChIP-seq of the
known architectural proteins CTCF, RAD21, and ZNF143 resulted in
markedly increased prediction performance with auPRCs of 0.31, 0.37, and
0.38 for CTCF, RAD21, and ZNF143, respectively (Fig. <a href="5-2-results-3.html#fig:LoopPred3">5.3</a>B, C). For visual
comparison, we show the actual looping interactions and 7C predictions
on an example region at chromosome 11 (Fig. <a href="5-2-results-3.html#fig:LoopPred3">5.3</a>D).</p>
<p>Finally, we built a full model using the sequence based features and the
ChIP-seq data of all six selected TFs. This only resulted in a slight
increase of prediction performance to auPRC = 0.42 (Fig. <a href="5-2-results-3.html#fig:LoopPred3">5.3</a>B, C),
indicating that a single ChIP-seq experiment might be sufficient for
accurate prediction of chromatin loops.</p>
<p>We also tested if a single value of correlation of ChIP-seq signal at
both loop anchors across the six different TFs is predictive. Indeed, we
find high prediction performance of auPRC = 0.34 for this approach.
However, this was lower than using the correlation from single TF
ChIP-seq experiments for RAD21 or ZNF143 and has the disadvantage of
relying on ChIP-seq data from multiple experiments.</p>
<p>Together, these results show that sequenced based features alone have
only a limited loop prediction performance, but integrating them with a
single ChIP-seq experiment, 7C can predict chromatin loops with high
accuracy.</p>
</div>
<div id="comparison-of-transcription-factors-by-prediction-performance" class="section level3">
<h3><span class="header-section-number">5.2.7</span> Comparison of transcription factors by prediction performance</h3>
<p>Our results can be used to better understand the molecular mechanisms of
chromatin loop formation. We hypothesize that TFs whose ChIP-seq
provides high prediction performance are likely to be functionally
involved in chromatin looping. These TFs would be therefore interesting
targets for further investigation of their potential function in
chromatin looping.</p>
<p>To investigate this for as many TFs as possible, we used all available
124 TF ChIP-seq datasets from ENCODE for the human cell line GM12878 and
compared transcription factors by their prediction performance. Notably,
nearly all TF ChIP-seq data sets could increase the prediction
performance of sequence-based features alone (Fig. <a href="5-2-results-3.html#fig:LoopPred3">5.3</a>E). However, there
was a large variance in performance between TFs and a subset of TFs with
high predictive power could be identified. These include for example the
known architectural proteins mentioned above, CTCF, cohesin (RAD21 and
SMC3), and ZNF143, but also factors, such as TRIM22, RUNX3, BHLHE40, or
RELA, which might be interesting candidate factors with functional roles
in chromatin loop formation.</p>
</div>
<div id="prediction-performance-in-other-cell-types-and-for-different-tfs" class="section level3">
<h3><span class="header-section-number">5.2.8</span> Prediction performance in other cell types and for different TFs</h3>
<p>Next, we wanted to test if 7C is general enough to predict looping
interactions in a cell type different to the one used to train it. To
test this, we used the models presented above (trained with data from
human GM12878 cells) to predict loops using as input ChIP-seq data from
human HeLa cells. The prediction performance was assessed using as
positives 12,480 loops (1.74 % of all motif pairs) identified in HeLa
cells <span class="citation">[Rao et al., <a href="#ref-Rao2014">2014</a>; Tang et al., <a href="#ref-Tang2015">2015</a>]</span>. While prediction performance in HeLa cells is slightly
lower as compared to the cross-validation in GM12878 cells, we see
overall very good prediction performance also in HeLa cells by ROC
curves (auPRC up to 0.91, Fig. <a href="5-2-results-3.html#fig:LoopPred4">5.4</a>A) and PRC curves (auPRC up to 0.27,
Fig. <a href="5-2-results-3.html#fig:LoopPred4">5.4</a>B,C).</p>
<!-- ---------------------------------------------------------------- -->
<div class="figure" style="text-align: center"><span id="fig:LoopPred4"></span>
<img src="figures/loop_prediction/fig4_v04.png" alt="Prediction performance in HeLa cells using 7C trained in GM12878 cells. (A) ROC curve of prediction performance of six selected TF ChIP-seq data sets. The 7C model was trained using ChIP-seq and true loop data in human GM1287 but loops were predicted using ChIP-seq data of the same TFs in HeLa cells and true loop data in HeLa cells. (B) Precision-Recall curves for the same analysis as in (A). (C) Prediction performance as auPRC (top) and auROC (bottom) in HeLa for the six TF ChIP-seq data sets (x-axis) and 7C models trained for the specific TF (left), 7C with parameters averaged across all 124 TF models (center), and 7C with parameters as average of the 10 best performing TF ChIP-seq data sets (right). (D) Example region on human chromosome 21 with genes, RAD21 ChIP-seq data in HeLa, CTCF motifs, true loops in HeLa cells according to Hi-C and ChIA-PET (green arcs) and predicted chromatin loops from RAD21 ChIP-seq data in HeLa (blue arcs)." width="100%" />
<p class="caption">
Figure 5.4: <strong>Prediction performance in HeLa cells using 7C trained in GM12878 cells. (A)</strong> ROC curve of prediction performance of six selected TF ChIP-seq data sets. The 7C model was trained using ChIP-seq and true loop data in human GM1287 but loops were predicted using ChIP-seq data of the same TFs in HeLa cells and true loop data in HeLa cells. <strong>(B)</strong> Precision-Recall curves for the same analysis as in (A). <strong>(C)</strong> Prediction performance as auPRC (top) and auROC (bottom) in HeLa for the six TF ChIP-seq data sets (x-axis) and 7C models trained for the specific TF (left), 7C with parameters averaged across all 124 TF models (center), and 7C with parameters as average of the 10 best performing TF ChIP-seq data sets (right). <strong>(D)</strong> Example region on human chromosome 21 with genes, RAD21 ChIP-seq data in HeLa, CTCF motifs, true loops in HeLa cells according to Hi-C and ChIA-PET (green arcs) and predicted chromatin loops from RAD21 ChIP-seq data in HeLa (blue arcs).
</p>
</div>

<!-- ---------------------------------------------------------------- -->
<p>In this analysis, we compared the prediction performance of each
specific TF model. However, in an application use case, one might not be
able to train the model for a specific TF of interest and the model
should predict loops for TFs that were not used in the training.
Therefore, we built default 7C models by either averaging model
parameters from all 124 TF models or by averaging across the model
parameters of only the 10 best performing TFs.</p>
<p>While all three approaches result in good prediction performance for the
six selected TFs (Fig. <a href="5-2-results-3.html#fig:LoopPred4">5.4</a>C), the model averaging parameters across all
TFs performs poorer than the ones of only the best 10 models, which are
actually nearly as good as the specific TF models. This is consistent
with similar results from cross-validation analysis in GM12878 data
(Fig. S1C). Furthermore, we visually inspected chromatin loop
predictions from RAD21 ChIP-seq data in HeLa at an example loci on
chromosome 21 (Fig. <a href="5-2-results-3.html#fig:LoopPred4">5.4</a>D). In summary, these results show that 7C can
predict chromatin looping interactions in different cell types that were
not used to train it. Similarly, the 7C default prediction model
performs nearly as good as a TF specific model. This makes 7C applicable
for ChIP-seq data from diverse TFs in many different cell types and
conditions.</p>
</div>
<div id="the-high-resolution-of-chip-nexus-improves-prediction-performance" class="section level3">
<h3><span class="header-section-number">5.2.9</span> The high resolution of ChIP-nexus improves prediction performance</h3>
We wondered if the similarity of other genomics signals at loop anchors
could potentially indicate looping interactions. Therefore, we used
different genomic assays, such as DNase hypersensitivity (DNase-seq),
ChIP-nexus and only ChIP-seq input material as input to our prediction
methods (Fig. <a href="5-2-results-3.html#fig:LoopPred5">5.5</a>).
<!-- ---------------------------------------------------------------- -->
<div class="figure" style="text-align: center"><span id="fig:LoopPred5"></span>
<img src="figures/loop_prediction/fig5_v04.png" alt="Higher resolution of ChIP-exo and ChIP-nexus improves prediction performance. Prediction performance as area under the precision recall curve (auPRC, x-axis) for 7C models with different input data sets to predict chromatin looping (y-axis). Input data sets are grouped by signal-type (middle panel) and assay-type (right panel) and colored according to the TF (if any) used in the experiment." width="80%" />
<p class="caption">
Figure 5.5: <strong>Higher resolution of ChIP-exo and ChIP-nexus improves prediction performance.</strong> Prediction performance as area under the precision recall curve (auPRC, x-axis) for 7C models with different input data sets to predict chromatin looping (y-axis). Input data sets are grouped by signal-type (middle panel) and assay-type (right panel) and colored according to the TF (if any) used in the experiment.
</p>
</div>

<!-- ---------------------------------------------------------------- -->
<p>Furthermore, we compared different signal types of
ChIP-seq. During computational processing of ChIP-seq raw data, reads
are shifted in 5’ direction by the estimated average fragment size
<span class="citation">[Zhang et al., <a href="#ref-Zhang2008">2008</a>; Hansen et al., <a href="#ref-Hansen2015">2015</a>]</span>. The coverage of these shifted reads is then compared to
coverage of input control experiment (fold change over control).
Furthermore, a recent study quantified read pairs (qfrags) in a specific
distance to each other as estimate for the actual fragment numbers
detected by ChIP-seq <span class="citation">[Hansen et al., <a href="#ref-Hansen2015">2015</a>]</span>. For most of the TFs tested here, we
observed that the ChIP-seq signal types ‘shifted reads’ and ‘qfrag’ have
better loop prediction performance than the ‘fold change over control’
(Fig. <a href="5-2-results-3.html#fig:LoopPred5">5.5</a>). Interestingly, also the ChIP-seq input signal alone results in
better prediction performance than sequence features alone, indicating
that cross-linking efficiency and density of chromatin itself is
specifically distributed at chromatin loop anchors (Fig. <a href="5-2-results-3.html#fig:LoopPred5">5.5</a>). Also,
DNase-seq, which measures chromatin accessibility, predicts looping
interactions with similar accuracy than ChIP-seq input control (Fig. <a href="5-2-results-3.html#fig:LoopPred5">5.5</a>).
This is consistent with specific open-chromatin profiles at TF binding
sites <span class="citation">[Pique-Regi et al., <a href="#ref-Pique-Regi2011">2011</a>; Yardımcı et al., <a href="#ref-Yardmc2014">2014</a>]</span>.</p>
<p>However, using ChIP-nexus data for RAD21 and SMC3 <span class="citation">[Tang et al., <a href="#ref-Tang2015">2015</a>]</span>, we could
markedly improve chromatin loop predictions using 7C (Fig. <a href="5-2-results-3.html#fig:LoopPred5">5.5</a>).
ChIP-nexus and ChIP-exo are variations of the ChIP-seq protocol, in
which additionally, an exonuclease digestion step is applied to trim the
DNA from the 5’ end until the actual bound protein <span class="citation">[Rhee and Pugh, <a href="#ref-Rhee2011">2011</a>; He et al., <a href="#ref-He2015a">2015</a>]</span>. These
signals result in high-resolution binding footprints that can be used to
identify different TF binding modes and cooperation with co-factors
<span class="citation">[Starick et al., <a href="#ref-Starick2015">2015</a>]</span>. Therefore, we conclude, that the high-resolution binding
profiles from ChIP-nexus allow to compute a more predictive binding
signal similarity at chromatin loop anchors.</p>
<p>In summary, the comparison of different genomic signal types shows that
cross-linking effect and chromatin density at chromatin anchors are
predictive signals for long-range chromatin interactions and higher
resolution TF binding assays, such as ChIP-nexus, result in improved
prediction performance.</p>
</div>
</div>
<h3>References</h3>
<div id="refs" class="references">
<div id="ref-Rao2014">
<p>Rao SSP, Huntley MH, Durand NC, Stamenova EK, Bochkov ID, Robinson JT, Sanborn AL, Machol I, Omer AD, Lander ES, Aiden EL. 2014. A 3D map of the human genome at kilobase resolution reveals principles of chromatin looping. Cell 159: 1665–80.</p>
</div>
<div id="ref-Tang2015">
<p>Tang Z, Luo OJ, Li X, Zheng M, Zhu JJ, Szalaj P, Trzaskoma P, Magalska A, Wlodarczyk J, Ruszczycki B, Michalski P, Piecuch E, Wang P, Wang D, Tian SZ, Penrad-mobayed M, Sachs LM, Ruan X, Wei C-l, Liu ET, Wilczynski GM, Plewczynski D, Li G, Ruan Y. 2015. CTCF-Mediated Human 3D Genome Architecture Reveals Chromatin Topology for Transcription. Cell: 1–17.</p>
</div>
<div id="ref-Orlando1997">
<p>Orlando V, Strutt H, Paro R. 1997. Analysis of Chromatin Structure byin VivoFormaldehyde Cross-Linking. Methods 11: 205–214.</p>
</div>
<div id="ref-Hoffman2015">
<p>Hoffman EA, Frey BL, Smith LM, Auble DT. 2015. Formaldehyde crosslinking: a tool for the study of chromatin complexes. The Journal of biological chemistry 290: 26404–11.</p>
</div>
<div id="ref-Starick2015">
<p>Starick SR, Ibn-Salem J, Jurk M, Hernandez C, Love MI, Chung H-R, Vingron M, Thomas-Chollier M, Meijsing SH, Hernandez C, Love MI, Chung H-R, Vingron M, Thomas-Chollier M, Meijsing SH. 2015. ChIP-exo signal associated with DNA-binding motifs provide insights into the genomic binding of the glucocorticoid receptor and cooperating transcription factors. Genome research 25: 825–35.</p>
</div>
<div id="ref-Lieberman-Aiden2009">
<p>Lieberman-Aiden E, Berkum NL van, Williams L, Imakaev M, Ragoczy T, Telling A, Amit I, Lajoie BR, Sabo PJ, Dorschner MO, Sandstrom R, Bernstein B, Bender MA, Groudine M, Gnirke A, Stamatoyannopoulos J, Mirny L a, Lander ES, Dekker J. 2009. Comprehensive mapping of long-range interactions reveals folding principles of the human genome. Science (New York, N.Y.) 326: 289–93.</p>
</div>
<div id="ref-Merkenschlager2016">
<p>Merkenschlager M, Nora EP. 2016. CTCF and Cohesin in Genome Folding and Transcriptional Gene Regulation. Annual review of genomics and human genetics 17: 17–43.</p>
</div>
<div id="ref-VietriRudan2015">
<p>Vietri Rudan M, Barrington C, Henderson S, Ernst C, Odom DT, Tanay A, Hadjur S. 2015. Comparative Hi-C reveals that CTCF underlies evolution of chromosomal domain architecture. Cell reports 10: 1297–309.</p>
</div>
<div id="ref-Guo2015">
<p>Guo Y, Xu Q, Canzio D, Shou J, Li J, Gorkin DU, Jung I, Wu H, Zhai Y, Tang Y, Lu Y, Wu Y, Jia Z, Li W, Zhang MQ, Ren B, Krainer AR, Maniatis T, Wu Q. 2015. CRISPR Inversion of CTCF Sites Alters Genome Topology and Enhancer/Promoter Function. Cell 162: 900–910.</p>
</div>
<div id="ref-deWit2015">
<p>Wit E de, Vos ESM, Holwerda SJB, Valdes-Quezada C, Verstegen MJAM, Teunissen H, Splinter E, Wijchers PJ, Krijger PHL, Laat W de. 2015. CTCF Binding Polarity Determines Chromatin Looping. Molecular Cell 60: 676–684.</p>
</div>
<div id="ref-Khan2018">
<p>Khan A, Fornes O, Stigliani A. 2018. JASPAR 2018: update of the open-access database of transcription factor binding profiles and its web framework. Nucleic Acids Research (In print): 1–7.</p>
</div>
<div id="ref-Saito2015">
<p>Saito T, Rehmsmeier M. 2015. The precision-recall plot is more informative than the ROC plot when evaluating binary classifiers on imbalanced datasets. PLoS ONE 10: 1–21.</p>
</div>
<div id="ref-Ye2016b">
<p>Ye B-Y, Shen W-L, Wang D, Li P, Zhang Z, Shi M-L, Zhang Y, Zhang F-X, Zhao Z-H. 2016. ZNF143 is involved in CTCF-mediated chromatin interactions by cooperation with cohesin and other partners. Molecular Biology 50: 431–437.</p>
</div>
<div id="ref-Bailey2015">
<p>Bailey SD, Zhang X, Desai K, Aid M, Corradin O, Cowper-Sal Lari R, Akhtar-Zaidi B, Scacheri PC, Haibe-Kains B, Lupien M, Cowper-Sallari R, Akhtar-Zaidi B, Scacheri PC, Haibe-Kains B, Lupien M. 2015. ZNF143 provides sequence specificity to secure chromatin interactions at gene promoters. Nature Communications 2: 6186.</p>
</div>
<div id="ref-Zhang2008">
<p>Zhang B, Park B-H, Karpinets T, Samatova NF. 2008. From pull-down data to protein interaction networks and complexes with biological relevance. Bioinformatics (Oxford, England) 24: 979–86.</p>
</div>
<div id="ref-Hansen2015">
<p>Hansen P, Hecht J, Ibrahim DM, Krannich A, Truss M, Robinson PN. 2015. Saturation analysis of ChIP-seq data for reproducible identification of binding peaks. Genome research 25: 1391–400.</p>
</div>
<div id="ref-Pique-Regi2011">
<p>Pique-Regi R, Degner JF, Pai A a, Gaffney DJ, Gilad Y, Pritchard JK. 2011. Accurate inference of transcription factor binding from DNA sequence and chromatin accessibility data. Genome research 21: 447–55.</p>
</div>
<div id="ref-Yardmc2014">
<p>Yardımcı GG, Frank CL, Crawford GE, Ohler U. 2014. Explicit DNase sequence bias modeling enables high-resolution transcription factor footprint detection. Nucleic acids research: 1–14.</p>
</div>
<div id="ref-Rhee2011">
<p>Rhee HS, Pugh BF. 2011. Comprehensive genome-wide protein-DNA interactions detected at single-nucleotide resolution. Cell 147: 1408–19.</p>
</div>
<div id="ref-He2015a">
<p>He Q, Johnston J, Zeitlinger J. 2015. ChIP-nexus enables improved detection of in vivo transcription factor binding footprints. Nature Biotechnology 33: 395–401.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="5-1-introduction-3.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="5-3-discussion-2.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"google": false,
"weibo": false,
"instapper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/ibn-salem/thesis/edit/master/05-loop_prediction.Rmd",
"text": "Edit"
},
"download": ["thesis.pdf", "thesis.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://cdn.bootcss.com/mathjax/2.7.1/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:" && /^https?:/.test(src))
      src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
