<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>Genome folding in evolution and disease</title>
  <meta name="description" content="A draft PhD thesis on genome folding in evoultion and disease.">
  <meta name="generator" content="bookdown 0.6.1 and GitBook 2.6.7">

  <meta property="og:title" content="Genome folding in evolution and disease" />
  <meta property="og:type" content="book" />
  <meta property="og:url" content="https://ibn-salem.github.io/thesis/" />
  
  <meta property="og:description" content="A draft PhD thesis on genome folding in evoultion and disease." />
  <meta name="github-repo" content="ibn-salem/thesis" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Genome folding in evolution and disease" />
  
  <meta name="twitter:description" content="A draft PhD thesis on genome folding in evoultion and disease." />
  

<meta name="author" content="Jonas Ibn-Salem">


<meta name="date" content="2018-02-22">

  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  
<link rel="prev" href="5-2-results-3.html">
<link rel="next" href="5-4-conclusion-2.html">
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />










<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Thesis</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="1" data-path="1-intro.html"><a href="1-intro.html"><i class="fa fa-check"></i><b>1</b> Introduction</a><ul>
<li class="chapter" data-level="1.1" data-path="1-1-regulation-of-expression.html"><a href="1-1-regulation-of-expression.html"><i class="fa fa-check"></i><b>1.1</b> Regulation of gene expression</a></li>
<li class="chapter" data-level="1.2" data-path="1-2-distal-regulation-by-enhancers.html"><a href="1-2-distal-regulation-by-enhancers.html"><i class="fa fa-check"></i><b>1.2</b> Distal regulation by enhancers</a></li>
<li class="chapter" data-level="1.3" data-path="1-3-methods-to-probe-the-3d-chromatin-architecture.html"><a href="1-3-methods-to-probe-the-3d-chromatin-architecture.html"><i class="fa fa-check"></i><b>1.3</b> Methods to probe the 3D chromatin architecture</a><ul>
<li class="chapter" data-level="1.3.1" data-path="1-3-methods-to-probe-the-3d-chromatin-architecture.html"><a href="1-3-methods-to-probe-the-3d-chromatin-architecture.html#microscopy-based-techniques-to-visualize-the-genome-in-3d"><i class="fa fa-check"></i><b>1.3.1</b> Microscopy-based techniques to visualize the genome in 3D</a></li>
<li class="chapter" data-level="1.3.2" data-path="1-3-methods-to-probe-the-3d-chromatin-architecture.html"><a href="1-3-methods-to-probe-the-3d-chromatin-architecture.html#proximity-ligation"><i class="fa fa-check"></i><b>1.3.2</b> Proximity-ligation based method to quantify chromatin interactions</a></li>
</ul></li>
<li class="chapter" data-level="1.4" data-path="1-4-hierarchy-of-chromatin-3d-structure.html"><a href="1-4-hierarchy-of-chromatin-3d-structure.html"><i class="fa fa-check"></i><b>1.4</b> Hierarchy of chromatin 3D structure</a><ul>
<li class="chapter" data-level="1.4.1" data-path="1-4-hierarchy-of-chromatin-3d-structure.html"><a href="1-4-hierarchy-of-chromatin-3d-structure.html#chromosomal-territories-and-inter-chromosomal-contacts"><i class="fa fa-check"></i><b>1.4.1</b> Chromosomal territories and inter-chromosomal contacts</a></li>
<li class="chapter" data-level="1.4.2" data-path="1-4-hierarchy-of-chromatin-3d-structure.html"><a href="1-4-hierarchy-of-chromatin-3d-structure.html#ab-compartments"><i class="fa fa-check"></i><b>1.4.2</b> A/B compartments</a></li>
<li class="chapter" data-level="1.4.3" data-path="1-4-hierarchy-of-chromatin-3d-structure.html"><a href="1-4-hierarchy-of-chromatin-3d-structure.html#topologically-associating-domains-tads"><i class="fa fa-check"></i><b>1.4.3</b> Topologically associating domains (TADs)</a></li>
<li class="chapter" data-level="1.4.4" data-path="1-4-hierarchy-of-chromatin-3d-structure.html"><a href="1-4-hierarchy-of-chromatin-3d-structure.html#hierarchy-of-domain-like-structures-across-genomic-length-scales"><i class="fa fa-check"></i><b>1.4.4</b> Hierarchy of domain-like structures across genomic-length scales</a></li>
<li class="chapter" data-level="1.4.5" data-path="1-4-hierarchy-of-chromatin-3d-structure.html"><a href="1-4-hierarchy-of-chromatin-3d-structure.html#chromatin-looping-interactions"><i class="fa fa-check"></i><b>1.4.5</b> Chromatin looping interactions</a></li>
<li class="chapter" data-level="1.4.6" data-path="1-4-hierarchy-of-chromatin-3d-structure.html"><a href="1-4-hierarchy-of-chromatin-3d-structure.html#loop-mechanism"><i class="fa fa-check"></i><b>1.4.6</b> TAD and loop formation by architectural proteins</a></li>
</ul></li>
<li class="chapter" data-level="1.5" data-path="1-5-dynamics-of-chromatin-structure.html"><a href="1-5-dynamics-of-chromatin-structure.html"><i class="fa fa-check"></i><b>1.5</b> Dynamics of chromatin structure</a><ul>
<li class="chapter" data-level="1.5.1" data-path="1-5-dynamics-of-chromatin-structure.html"><a href="1-5-dynamics-of-chromatin-structure.html#dynamics-across-cell-cycle"><i class="fa fa-check"></i><b>1.5.1</b> Dynamics across cell cycle</a></li>
<li class="chapter" data-level="1.5.2" data-path="1-5-dynamics-of-chromatin-structure.html"><a href="1-5-dynamics-of-chromatin-structure.html#dynamics-across-cell-types-and-differntiation"><i class="fa fa-check"></i><b>1.5.2</b> Dynamics across cell types and differntiation</a></li>
</ul></li>
<li class="chapter" data-level="1.6" data-path="1-6-evolution-of-chromatin-organization.html"><a href="1-6-evolution-of-chromatin-organization.html"><i class="fa fa-check"></i><b>1.6</b> Evolution of chromatin organization</a></li>
<li class="chapter" data-level="1.7" data-path="1-7-disruption-of-chromatin-architecture-in-disease.html"><a href="1-7-disruption-of-chromatin-architecture-in-disease.html"><i class="fa fa-check"></i><b>1.7</b> Disruption of chromatin architecture in disease</a></li>
<li class="chapter" data-level="1.8" data-path="1-8-aims-of-this-thesis.html"><a href="1-8-aims-of-this-thesis.html"><i class="fa fa-check"></i><b>1.8</b> Aims of this thesis</a></li>
<li class="chapter" data-level="1.9" data-path="1-9-structure-of-this-thesis.html"><a href="1-9-structure-of-this-thesis.html"><i class="fa fa-check"></i><b>1.9</b> Structure of this thesis</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="2-paralog-regulation.html"><a href="2-paralog-regulation.html"><i class="fa fa-check"></i><b>2</b> Paralog genes in the 3D genome architecture</a><ul>
<li class="chapter" data-level="" data-path="2-paralog-regulation.html"><a href="2-paralog-regulation.html#preamble"><i class="fa fa-check"></i>Preamble</a></li>
<li class="chapter" data-level="" data-path="2-paralog-regulation.html"><a href="2-paralog-regulation.html#abstract-1"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="2.1" data-path="2-1-introduction.html"><a href="2-1-introduction.html"><i class="fa fa-check"></i><b>2.1</b> Introduction</a></li>
<li class="chapter" data-level="2.2" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html"><i class="fa fa-check"></i><b>2.2</b> Materials and methods</a><ul>
<li class="chapter" data-level="2.2.1" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html#selection-of-pairs-of-paralog-genes"><i class="fa fa-check"></i><b>2.2.1</b> Selection of pairs of paralog genes</a></li>
<li class="chapter" data-level="2.2.2" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html#enhancers-to-gene-association"><i class="fa fa-check"></i><b>2.2.2</b> Enhancers to gene association</a></li>
<li class="chapter" data-level="2.2.3" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html#topological-associating-domains"><i class="fa fa-check"></i><b>2.2.3</b> Topological associating domains</a></li>
<li class="chapter" data-level="2.2.4" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html#hi-c-interaction-maps"><i class="fa fa-check"></i><b>2.2.4</b> Hi-C interaction maps</a></li>
<li class="chapter" data-level="2.2.5" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html#randomization"><i class="fa fa-check"></i><b>2.2.5</b> Randomization</a></li>
<li class="chapter" data-level="2.2.6" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html#statistical-tests"><i class="fa fa-check"></i><b>2.2.6</b> Statistical tests</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="2-3-results.html"><a href="2-3-results.html"><i class="fa fa-check"></i><b>2.3</b> Results</a><ul>
<li class="chapter" data-level="2.3.1" data-path="2-3-results.html"><a href="2-3-results.html#distribution-of-paralog-genes-in-the-human-genome"><i class="fa fa-check"></i><b>2.3.1</b> Distribution of paralog genes in the human genome</a></li>
<li class="chapter" data-level="2.3.2" data-path="2-3-results.html"><a href="2-3-results.html#co-expression-of-paralog-gene-pairs-across-tissues"><i class="fa fa-check"></i><b>2.3.2</b> Co-expression of paralog gene pairs across tissues</a></li>
<li class="chapter" data-level="2.3.3" data-path="2-3-results.html"><a href="2-3-results.html#paralog-genes-share-enhancers"><i class="fa fa-check"></i><b>2.3.3</b> Paralog genes share enhancers</a></li>
<li class="chapter" data-level="2.3.4" data-path="2-3-results.html"><a href="2-3-results.html#co-localization-of-paralogs-in-tads"><i class="fa fa-check"></i><b>2.3.4</b> Co-localization of paralogs in TADs</a></li>
<li class="chapter" data-level="2.3.5" data-path="2-3-results.html"><a href="2-3-results.html#distal-paralog-pairs-are-enriched-for-long-range-chromatin-contacts"><i class="fa fa-check"></i><b>2.3.5</b> Distal paralog pairs are enriched for long-range chromatin contacts</a></li>
<li class="chapter" data-level="2.3.6" data-path="2-3-results.html"><a href="2-3-results.html#close-paralogs-have-fewer-contacts-than-expected"><i class="fa fa-check"></i><b>2.3.6</b> Close paralogs have fewer contacts than expected</a></li>
<li class="chapter" data-level="2.3.7" data-path="2-3-results.html"><a href="2-3-results.html#paralogs-in-mouse-and-dog-genome"><i class="fa fa-check"></i><b>2.3.7</b> Paralogs in mouse and dog genome</a></li>
<li class="chapter" data-level="2.3.8" data-path="2-3-results.html"><a href="2-3-results.html#orthologs-of-human-paralogs-show-conserved-co-localization"><i class="fa fa-check"></i><b>2.3.8</b> Orthologs of human paralogs show conserved co-localization</a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="2-4-discussions.html"><a href="2-4-discussions.html"><i class="fa fa-check"></i><b>2.4</b> Discussions</a></li>
<li class="chapter" data-level="2.5" data-path="2-5-conclusion.html"><a href="2-5-conclusion.html"><i class="fa fa-check"></i><b>2.5</b> Conclusion</a></li>
<li class="chapter" data-level="2.6" data-path="2-6-acknowledgements.html"><a href="2-6-acknowledgements.html"><i class="fa fa-check"></i><b>2.6</b> Acknowledgements</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="3-TAD-evolution.html"><a href="3-TAD-evolution.html"><i class="fa fa-check"></i><b>3</b> Stability of TADs in evolution</a><ul>
<li class="chapter" data-level="" data-path="3-TAD-evolution.html"><a href="3-TAD-evolution.html#preamble-1"><i class="fa fa-check"></i>Preamble</a></li>
<li class="chapter" data-level="" data-path="3-TAD-evolution.html"><a href="3-TAD-evolution.html#abstract-2"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="" data-path="3-TAD-evolution.html"><a href="3-TAD-evolution.html#keywords"><i class="fa fa-check"></i>Keywords</a></li>
<li class="chapter" data-level="3.1" data-path="3-1-introduction-1.html"><a href="3-1-introduction-1.html"><i class="fa fa-check"></i><b>3.1</b> Introduction</a></li>
<li class="chapter" data-level="3.2" data-path="3-2-results-1.html"><a href="3-2-results-1.html"><i class="fa fa-check"></i><b>3.2</b> Results</a><ul>
<li class="chapter" data-level="3.2.1" data-path="3-2-results-1.html"><a href="3-2-results-1.html#identification-of-evolutionary-rearrangement-breakpoints-from-whole-genome-alignments"><i class="fa fa-check"></i><b>3.2.1</b> Identification of evolutionary rearrangement breakpoints from whole-genome alignments</a></li>
<li class="chapter" data-level="3.2.2" data-path="3-2-results-1.html"><a href="3-2-results-1.html#rearrangement-breakpoints-are-enriched-at-tad-boundaries"><i class="fa fa-check"></i><b>3.2.2</b> Rearrangement breakpoints are enriched at TAD boundaries</a></li>
<li class="chapter" data-level="3.2.3" data-path="3-2-results-1.html"><a href="3-2-results-1.html#clusters-of-conserved-non-coding-elements-are-depleted-for-rearrangement-breakpoints"><i class="fa fa-check"></i><b>3.2.3</b> Clusters of conserved non-coding elements are depleted for rearrangement breakpoints</a></li>
<li class="chapter" data-level="3.2.4" data-path="3-2-results-1.html"><a href="3-2-results-1.html#rearranged-tads-are-associated-with-divergent-gene-expression-between-species"><i class="fa fa-check"></i><b>3.2.4</b> Rearranged TADs are associated with divergent gene expression between species</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="3-3-discussion.html"><a href="3-3-discussion.html"><i class="fa fa-check"></i><b>3.3</b> Discussion</a></li>
<li class="chapter" data-level="3.4" data-path="3-4-conclusion-1.html"><a href="3-4-conclusion-1.html"><i class="fa fa-check"></i><b>3.4</b> Conclusion</a></li>
<li class="chapter" data-level="3.5" data-path="3-5-methods.html"><a href="3-5-methods.html"><i class="fa fa-check"></i><b>3.5</b> Methods</a><ul>
<li class="chapter" data-level="3.5.1" data-path="3-5-methods.html"><a href="3-5-methods.html#rearrangement-breakpoints-from-whole-genome-alignments"><i class="fa fa-check"></i><b>3.5.1</b> Rearrangement breakpoints from whole-genome alignments</a></li>
<li class="chapter" data-level="3.5.2" data-path="3-5-methods.html"><a href="3-5-methods.html#topologically-associating-domains-and-contact-domains"><i class="fa fa-check"></i><b>3.5.2</b> Topologically associating domains and contact domains</a></li>
<li class="chapter" data-level="3.5.3" data-path="3-5-methods.html"><a href="3-5-methods.html#breakpoint-distributions-at-tads"><i class="fa fa-check"></i><b>3.5.3</b> Breakpoint distributions at TADs</a></li>
<li class="chapter" data-level="3.5.4" data-path="3-5-methods.html"><a href="3-5-methods.html#quantification-of-breakpoint-enrichment"><i class="fa fa-check"></i><b>3.5.4</b> Quantification of breakpoint enrichment</a></li>
<li class="chapter" data-level="3.5.5" data-path="3-5-methods.html"><a href="3-5-methods.html#expression-data-for-mouse-and-human-orthologs"><i class="fa fa-check"></i><b>3.5.5</b> Expression data for mouse and human orthologs</a></li>
<li class="chapter" data-level="3.5.6" data-path="3-5-methods.html"><a href="3-5-methods.html#classification-of-tads-and-genes-according-to-rearrangements-and-grbs"><i class="fa fa-check"></i><b>3.5.6</b> Classification of TADs and genes according to rearrangements and GRBs</a></li>
<li class="chapter" data-level="3.5.7" data-path="3-5-methods.html"><a href="3-5-methods.html#source-code-and-implementation-details"><i class="fa fa-check"></i><b>3.5.7</b> Source code and implementation details</a></li>
</ul></li>
<li class="chapter" data-level="3.6" data-path="3-6-declarations.html"><a href="3-6-declarations.html"><i class="fa fa-check"></i><b>3.6</b> Declarations</a><ul>
<li class="chapter" data-level="3.6.1" data-path="3-6-declarations.html"><a href="3-6-declarations.html#ethics-approval-and-consent-to-participate"><i class="fa fa-check"></i><b>3.6.1</b> Ethics approval and consent to participate</a></li>
<li class="chapter" data-level="3.6.2" data-path="3-6-declarations.html"><a href="3-6-declarations.html#consent-for-publication"><i class="fa fa-check"></i><b>3.6.2</b> Consent for publication</a></li>
<li class="chapter" data-level="3.6.3" data-path="3-6-declarations.html"><a href="3-6-declarations.html#availability-of-data-and-material"><i class="fa fa-check"></i><b>3.6.3</b> Availability of data and material</a></li>
<li class="chapter" data-level="3.6.4" data-path="3-6-declarations.html"><a href="3-6-declarations.html#competing-interests"><i class="fa fa-check"></i><b>3.6.4</b> Competing interests</a></li>
<li class="chapter" data-level="3.6.5" data-path="3-6-declarations.html"><a href="3-6-declarations.html#funding"><i class="fa fa-check"></i><b>3.6.5</b> Funding</a></li>
<li class="chapter" data-level="3.6.6" data-path="3-6-declarations.html"><a href="3-6-declarations.html#authors-contributions"><i class="fa fa-check"></i><b>3.6.6</b> Authors‘ contributions</a></li>
<li class="chapter" data-level="3.6.7" data-path="3-6-declarations.html"><a href="3-6-declarations.html#acknowledgments"><i class="fa fa-check"></i><b>3.6.7</b> Acknowledgments</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="4-position-effect.html"><a href="4-position-effect.html"><i class="fa fa-check"></i><b>4</b> Position effects of rearrangements in disease genomes</a><ul>
<li class="chapter" data-level="" data-path="4-position-effect.html"><a href="4-position-effect.html#preamble-2"><i class="fa fa-check"></i>Preamble</a></li>
<li class="chapter" data-level="" data-path="4-position-effect.html"><a href="4-position-effect.html#abstract-3"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="4.1" data-path="4-1-introduction-2.html"><a href="4-1-introduction-2.html"><i class="fa fa-check"></i><b>4.1</b> Introduction</a></li>
<li class="chapter" data-level="4.2" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html"><i class="fa fa-check"></i><b>4.2</b> Materials and Methods</a><ul>
<li class="chapter" data-level="4.2.1" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html#selection-of-subjects-with-apparently-balanced-chromosome-abnormalities"><i class="fa fa-check"></i><b>4.2.1</b> Selection of subjects with apparently balanced chromosome abnormalities</a></li>
<li class="chapter" data-level="4.2.2" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html#clinical-descriptions-of-dgap-cases"><i class="fa fa-check"></i><b>4.2.2</b> Clinical descriptions of DGAP cases</a></li>
<li class="chapter" data-level="4.2.3" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html#analysis-of-genes-bordering-the-rearrangement-breakpoints"><i class="fa fa-check"></i><b>4.2.3</b> Analysis of genes bordering the rearrangement breakpoints</a></li>
<li class="chapter" data-level="4.2.4" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html#assessment-of-disrupted-functional-elements-and-chromatin-interactions-bordering-rearrangement-breakpoints"><i class="fa fa-check"></i><b>4.2.4</b> Assessment of disrupted functional elements and chromatin interactions bordering rearrangement breakpoints</a></li>
<li class="chapter" data-level="4.2.5" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html#ontological-analysis-of-genes-neighboring-breakpoints"><i class="fa fa-check"></i><b>4.2.5</b> Ontological analysis of genes neighboring breakpoints</a></li>
<li class="chapter" data-level="4.2.6" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html#quantitative-real-time-pcr"><i class="fa fa-check"></i><b>4.2.6</b> Quantitative real-time PCR</a></li>
<li class="chapter" data-level="4.2.7" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html#assessment-of-dgap-breakpoints-overlapping-with-non-coding-structural-variants-in-public-databases"><i class="fa fa-check"></i><b>4.2.7</b> Assessment of DGAP breakpoints overlapping with non-coding structural variants in public databases</a></li>
</ul></li>
<li class="chapter" data-level="4.3" data-path="4-3-results-2.html"><a href="4-3-results-2.html"><i class="fa fa-check"></i><b>4.3</b> Results</a><ul>
<li class="chapter" data-level="4.3.1" data-path="4-3-results-2.html"><a href="4-3-results-2.html#genomic-characterization-of-non-coding-breakpoints"><i class="fa fa-check"></i><b>4.3.1</b> Genomic characterization of non-coding breakpoints</a></li>
<li class="chapter" data-level="4.3.2" data-path="4-3-results-2.html"><a href="4-3-results-2.html#identification-of-genes-with-potential-position-effects"><i class="fa fa-check"></i><b>4.3.2</b> Identification of genes with potential position effects</a></li>
<li class="chapter" data-level="4.3.3" data-path="4-3-results-2.html"><a href="4-3-results-2.html#identification-of-subjects-with-shared-non-coding-chromosome-alterations-and-phenotypes"><i class="fa fa-check"></i><b>4.3.3</b> Identification of subjects with shared non-coding chromosome alterations and phenotypes</a></li>
</ul></li>
<li class="chapter" data-level="4.4" data-path="4-4-discussion-1.html"><a href="4-4-discussion-1.html"><i class="fa fa-check"></i><b>4.4</b> Discussion</a><ul>
<li class="chapter" data-level="" data-path="4-4-discussion-1.html"><a href="4-4-discussion-1.html#acknowledgements-1"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="4.4.1" data-path="4-4-discussion-1.html"><a href="4-4-discussion-1.html#web-resources"><i class="fa fa-check"></i><b>4.4.1</b> Web Resources</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="5-loop.html"><a href="5-loop.html"><i class="fa fa-check"></i><b>5</b> Prediction of chromatin looping interactions</a><ul>
<li class="chapter" data-level="" data-path="5-loop.html"><a href="5-loop.html#preamble-3"><i class="fa fa-check"></i>Preamble</a></li>
<li class="chapter" data-level="" data-path="5-loop.html"><a href="5-loop.html#abstract-4"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="" data-path="5-loop.html"><a href="5-loop.html#keywords-1"><i class="fa fa-check"></i>Keywords</a></li>
<li class="chapter" data-level="5.1" data-path="5-1-introduction-3.html"><a href="5-1-introduction-3.html"><i class="fa fa-check"></i><b>5.1</b> Introduction</a></li>
<li class="chapter" data-level="5.2" data-path="5-2-results-3.html"><a href="5-2-results-3.html"><i class="fa fa-check"></i><b>5.2</b> Results</a><ul>
<li class="chapter" data-level="5.2.1" data-path="5-2-results-3.html"><a href="5-2-results-3.html#ctcf-motif-pairs-as-candidate-chromatin-loop-anchors"><i class="fa fa-check"></i><b>5.2.1</b> CTCF motif pairs as candidate chromatin loop anchors</a></li>
<li class="chapter" data-level="5.2.2" data-path="5-2-results-3.html"><a href="5-2-results-3.html#similarity-of-chip-seq-signals-at-looping-ctcf-motifs"><i class="fa fa-check"></i><b>5.2.2</b> Similarity of ChIP-seq signals at looping CTCF motifs</a></li>
<li class="chapter" data-level="5.2.3" data-path="5-2-results-3.html"><a href="5-2-results-3.html#genomic-sequence-features-of-ctcf-motif-pairs-are-associated-with-looping"><i class="fa fa-check"></i><b>5.2.3</b> Genomic sequence features of CTCF motif pairs are associated with looping</a></li>
<li class="chapter" data-level="5.2.4" data-path="5-2-results-3.html"><a href="5-2-results-3.html#chromatin-loop-prediction-using-7c"><i class="fa fa-check"></i><b>5.2.4</b> Chromatin loop prediction using 7C</a></li>
<li class="chapter" data-level="5.2.5" data-path="5-2-results-3.html"><a href="5-2-results-3.html#prediction-performance-evaluation"><i class="fa fa-check"></i><b>5.2.5</b> Prediction performance evaluation</a></li>
<li class="chapter" data-level="5.2.6" data-path="5-2-results-3.html"><a href="5-2-results-3.html#prediction-performance-of-7c-with-sequence-features-and-single-tf-chip-seq-data-sets"><i class="fa fa-check"></i><b>5.2.6</b> Prediction performance of 7C with sequence features and single TF ChIP-seq data sets</a></li>
<li class="chapter" data-level="5.2.7" data-path="5-2-results-3.html"><a href="5-2-results-3.html#comparison-of-transcription-factors-by-prediction-performance"><i class="fa fa-check"></i><b>5.2.7</b> Comparison of transcription factors by prediction performance</a></li>
<li class="chapter" data-level="5.2.8" data-path="5-2-results-3.html"><a href="5-2-results-3.html#prediction-performance-in-other-cell-types-and-for-different-tfs"><i class="fa fa-check"></i><b>5.2.8</b> Prediction performance in other cell types and for different TFs</a></li>
<li class="chapter" data-level="5.2.9" data-path="5-2-results-3.html"><a href="5-2-results-3.html#the-high-resolution-of-chip-nexus-improves-prediction-performance"><i class="fa fa-check"></i><b>5.2.9</b> The high resolution of ChIP-nexus improves prediction performance</a></li>
</ul></li>
<li class="chapter" data-level="5.3" data-path="5-3-discussion-2.html"><a href="5-3-discussion-2.html"><i class="fa fa-check"></i><b>5.3</b> Discussion</a></li>
<li class="chapter" data-level="5.4" data-path="5-4-conclusion-2.html"><a href="5-4-conclusion-2.html"><i class="fa fa-check"></i><b>5.4</b> Conclusion</a></li>
<li class="chapter" data-level="5.5" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html"><i class="fa fa-check"></i><b>5.5</b> Methods</a><ul>
<li class="chapter" data-level="5.5.1" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#ctcf-motifs-in-the-human-genome"><i class="fa fa-check"></i><b>5.5.1</b> CTCF motifs in the human genome</a></li>
<li class="chapter" data-level="5.5.2" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#loop-interaction-data-for-training-and-validation"><i class="fa fa-check"></i><b>5.5.2</b> Loop interaction data for training and validation</a></li>
<li class="chapter" data-level="5.5.3" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#chip-seq-datasets-in-gm12878-cells"><i class="fa fa-check"></i><b>5.5.3</b> ChIP-seq datasets in GM12878 cells</a></li>
<li class="chapter" data-level="5.5.4" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#chip-seq-data-types"><i class="fa fa-check"></i><b>5.5.4</b> ChIP-seq data types</a></li>
<li class="chapter" data-level="5.5.5" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#chip-nexus-data-processing-for-rad21-and-scm3"><i class="fa fa-check"></i><b>5.5.5</b> ChIP-nexus data processing for RAD21 and SCM3</a></li>
<li class="chapter" data-level="5.5.6" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#similarity-of-chip-seq-profiles-as-correlation-of-coverage-around-motifs"><i class="fa fa-check"></i><b>5.5.6</b> Similarity of ChIP-seq profiles as correlation of coverage around motifs</a></li>
<li class="chapter" data-level="5.5.7" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#genomic-sequence-features-of-chromatin-loops"><i class="fa fa-check"></i><b>5.5.7</b> Genomic sequence features of chromatin loops</a></li>
<li class="chapter" data-level="5.5.8" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#chromatin-loop-prediction-model"><i class="fa fa-check"></i><b>5.5.8</b> Chromatin Loop prediction model</a></li>
<li class="chapter" data-level="5.5.9" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#training-and-validation-of-prediction-model"><i class="fa fa-check"></i><b>5.5.9</b> Training and validation of prediction model</a></li>
<li class="chapter" data-level="5.5.10" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#analysis-of-prediction-performance"><i class="fa fa-check"></i><b>5.5.10</b> Analysis of prediction performance</a></li>
<li class="chapter" data-level="5.5.11" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#implementation-of-7c-and-compatibility-to-other-tools"><i class="fa fa-check"></i><b>5.5.11</b> Implementation of 7C and compatibility to other tools</a></li>
</ul></li>
<li class="chapter" data-level="5.6" data-path="5-6-declarations-1.html"><a href="5-6-declarations-1.html"><i class="fa fa-check"></i><b>5.6</b> Declarations</a><ul>
<li class="chapter" data-level="5.6.1" data-path="5-6-declarations-1.html"><a href="5-6-declarations-1.html#ethics-approval-and-consent-to-participate-1"><i class="fa fa-check"></i><b>5.6.1</b> Ethics approval and consent to participate</a></li>
<li class="chapter" data-level="5.6.2" data-path="5-6-declarations-1.html"><a href="5-6-declarations-1.html#consent-for-publication-1"><i class="fa fa-check"></i><b>5.6.2</b> Consent for publication</a></li>
<li class="chapter" data-level="5.6.3" data-path="5-6-declarations-1.html"><a href="5-6-declarations-1.html#availability-of-data-and-material-1"><i class="fa fa-check"></i><b>5.6.3</b> Availability of data and material</a></li>
<li class="chapter" data-level="5.6.4" data-path="5-6-declarations-1.html"><a href="5-6-declarations-1.html#competing-interests-1"><i class="fa fa-check"></i><b>5.6.4</b> Competing interests</a></li>
<li class="chapter" data-level="5.6.5" data-path="5-6-declarations-1.html"><a href="5-6-declarations-1.html#funding-1"><i class="fa fa-check"></i><b>5.6.5</b> Funding</a></li>
<li class="chapter" data-level="5.6.6" data-path="5-6-declarations-1.html"><a href="5-6-declarations-1.html#authors-contributions-1"><i class="fa fa-check"></i><b>5.6.6</b> Authors‘ contributions</a></li>
<li class="chapter" data-level="5.6.7" data-path="5-6-declarations-1.html"><a href="5-6-declarations-1.html#acknowledgments-1"><i class="fa fa-check"></i><b>5.6.7</b> Acknowledgments</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="6-thesis-discussion.html"><a href="6-thesis-discussion.html"><i class="fa fa-check"></i><b>6</b> Discussion</a><ul>
<li class="chapter" data-level="6.1" data-path="6-1-co-regulation-of-functionally-related-genes-in-tads.html"><a href="6-1-co-regulation-of-functionally-related-genes-in-tads.html"><i class="fa fa-check"></i><b>6.1</b> Co-regulation of functionally related genes in TADs</a></li>
<li class="chapter" data-level="6.2" data-path="6-2-evolution-by-gene-duplications-and-altered-regulatory-environments.html"><a href="6-2-evolution-by-gene-duplications-and-altered-regulatory-environments.html"><i class="fa fa-check"></i><b>6.2</b> Evolution by gene duplications and altered regulatory environments</a></li>
<li class="chapter" data-level="6.3" data-path="6-3-tads-are-stable-across-large-evolutionary-time-scales.html"><a href="6-3-tads-are-stable-across-large-evolutionary-time-scales.html"><i class="fa fa-check"></i><b>6.3</b> TADs are stable across large evolutionary time-scales</a></li>
<li class="chapter" data-level="6.4" data-path="6-4-TAD-disease.html"><a href="6-4-TAD-disease.html"><i class="fa fa-check"></i><b>6.4</b> Gene expression changes by altered TADs in disease</a></li>
<li class="chapter" data-level="6.5" data-path="6-5-towards-predicting-regulatory-pathomechanisms-of-structural-variants.html"><a href="6-5-towards-predicting-regulatory-pathomechanisms-of-structural-variants.html"><i class="fa fa-check"></i><b>6.5</b> Towards predicting regulatory pathomechanisms of structural variants</a></li>
<li class="chapter" data-level="6.6" data-path="6-6-towards-studying-genome-folding-in-specific-conditions.html"><a href="6-6-towards-studying-genome-folding-in-specific-conditions.html"><i class="fa fa-check"></i><b>6.6</b> Towards studying genome folding in specific conditions</a><ul>
<li class="chapter" data-level="6.6.1" data-path="6-6-towards-studying-genome-folding-in-specific-conditions.html"><a href="6-6-towards-studying-genome-folding-in-specific-conditions.html#interpretation-of-non-coding-variants-by-interacting-genes"><i class="fa fa-check"></i><b>6.6.1</b> Interpretation of non-coding variants by interacting genes</a></li>
<li class="chapter" data-level="6.6.2" data-path="6-6-towards-studying-genome-folding-in-specific-conditions.html"><a href="6-6-towards-studying-genome-folding-in-specific-conditions.html#cell-type-specific-regulatory-interactions-between-enhancers-and-genes"><i class="fa fa-check"></i><b>6.6.2</b> Cell type-specific regulatory interactions between enhancers and genes</a></li>
<li class="chapter" data-level="6.6.3" data-path="6-6-towards-studying-genome-folding-in-specific-conditions.html"><a href="6-6-towards-studying-genome-folding-in-specific-conditions.html#variability-of-long-range-interactions-across-individuals"><i class="fa fa-check"></i><b>6.6.3</b> Variability of long-range interactions across individuals</a></li>
<li class="chapter" data-level="6.6.4" data-path="6-6-towards-studying-genome-folding-in-specific-conditions.html"><a href="6-6-towards-studying-genome-folding-in-specific-conditions.html#genome-folding-in-single-cells"><i class="fa fa-check"></i><b>6.6.4</b> Genome folding in single cells</a></li>
<li class="chapter" data-level="6.6.5" data-path="6-6-towards-studying-genome-folding-in-specific-conditions.html"><a href="6-6-towards-studying-genome-folding-in-specific-conditions.html#constantly-improving-targeted-experimental-methods."><i class="fa fa-check"></i><b>6.6.5</b> Constantly improving targeted experimental methods.</a></li>
</ul></li>
<li class="chapter" data-level="6.7" data-path="6-7-molecular-mechanisms-driving-genome-folding.html"><a href="6-7-molecular-mechanisms-driving-genome-folding.html"><i class="fa fa-check"></i><b>6.7</b> Molecular mechanisms driving genome folding</a></li>
<li class="chapter" data-level="6.8" data-path="6-8-conclusions.html"><a href="6-8-conclusions.html"><i class="fa fa-check"></i><b>6.8</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="appendix"><span><b>Appendix</b></span></li>
<li class="chapter" data-level="A" data-path="A-sup-paralog-regulation.html"><a href="A-sup-paralog-regulation.html"><i class="fa fa-check"></i><b>A</b> Supporting Information: Co-regulation of paralog genes</a></li>
<li class="chapter" data-level="B" data-path="B-sup-tad-evolution.html"><a href="B-sup-tad-evolution.html"><i class="fa fa-check"></i><b>B</b> Supplementary Data: Stability of TADs in evolution</a><ul>
<li class="chapter" data-level="B.1" data-path="B-1-TadEvoSupTab.html"><a href="B-1-TadEvoSupTab.html"><i class="fa fa-check"></i><b>B.1</b> Supplementary Tables</a></li>
<li class="chapter" data-level="B.2" data-path="B-2-supplementary-figures.html"><a href="B-2-supplementary-figures.html"><i class="fa fa-check"></i><b>B.2</b> Supplementary Figures</a></li>
</ul></li>
<li class="chapter" data-level="C" data-path="C-sup-position-effect.html"><a href="C-sup-position-effect.html"><i class="fa fa-check"></i><b>C</b> Supplemental Data: Position effects of rearrangements in disease genomes</a><ul>
<li class="chapter" data-level="C.1" data-path="C-1-supplemental-note.html"><a href="C-1-supplemental-note.html"><i class="fa fa-check"></i><b>C.1</b> Supplemental Note</a><ul>
<li class="chapter" data-level="C.1.1" data-path="C-1-supplemental-note.html"><a href="C-1-supplemental-note.html#case-reports"><i class="fa fa-check"></i><b>C.1.1</b> Case Reports</a></li>
<li class="chapter" data-level="C.1.2" data-path="C-1-supplemental-note.html"><a href="C-1-supplemental-note.html#nucleotide-level-nomenclature-for-dgap-karyotypes"><i class="fa fa-check"></i><b>C.1.2</b> Nucleotide Level Nomenclature for DGAP karyotypes</a></li>
</ul></li>
<li class="chapter" data-level="C.2" data-path="C-2-supplemental-figure.html"><a href="C-2-supplemental-figure.html"><i class="fa fa-check"></i><b>C.2</b> Supplemental Figure</a></li>
<li class="chapter" data-level="C.3" data-path="C-3-position-effect-sub-tab.html"><a href="C-3-position-effect-sub-tab.html"><i class="fa fa-check"></i><b>C.3</b> Supplemental Table Legends</a></li>
</ul></li>
<li class="chapter" data-level="D" data-path="D-loop-support.html"><a href="D-loop-support.html"><i class="fa fa-check"></i><b>D</b> Supplemental Information: Prediction of chromatin looping interactions</a><ul>
<li class="chapter" data-level="D.1" data-path="D-1-LoopPredSupTab.html"><a href="D-1-LoopPredSupTab.html"><i class="fa fa-check"></i><b>D.1</b> Supplementary Tables</a></li>
<li class="chapter" data-level="D.2" data-path="D-2-supplementary-figures-1.html"><a href="D-2-supplementary-figures-1.html"><i class="fa fa-check"></i><b>D.2</b> Supplementary Figures</a></li>
</ul></li>
<li class="chapter" data-level="E" data-path="E-contribution-to-individual-publications.html"><a href="E-contribution-to-individual-publications.html"><i class="fa fa-check"></i><b>E</b> Contribution to individual publications</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Build with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Genome folding in evolution and disease</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="discussion-2" class="section level2">
<h2><span class="header-section-number">5.3</span> Discussion</h2>
<p>We have developed 7C to reuse ChIP-seq data, profiling the interactions
of proteins with genomes, for the prediction of chromatin looping
interactions. We present this method as an alternative to dedicated
techniques like Hi-C that directly measure genomic contacts. Since the
results of ChIP-seq experiments are increasingly available for a large
number of proteins, species, tissues, cell types, and conditions, our
method offers a valid alternative when Hi-C data is not available, or
cannot be produced due to cost or material limitations. Another major
advantage of our method over Hi-C is that the predictions are at a base
pair resolution, while Hi-C only reaches resolutions of at best kilo
base pairs at a high cost.</p>
<p>Other computational approaches were developed to predict genomic
contacts or assign regulatory regions to target genes. A commonly used
approach is to compare activity signals at enhancers and promoters
across many different conditions or tissues <span class="citation">(Sheffield et al. <a href="#ref-Sheffield2013">2013</a>; Fishilevich et al. <a href="#ref-Fishilevich2017">2017</a>; Andersson et al. <a href="#ref-Andersson2014">2014</a>; O’Connor and Bailey <a href="#ref-OConnor2014">2014</a>)</span>: high correlation
indicates association and potential physical interactions between
enhancers and genes. However, these approaches lose the tissue
specificity of the interactions. Other approaches integrate many diverse
chromatin signals such as post-translational histone modifications,
chromatin accessibility, or transcriptional activity <span class="citation">(Roy et al. <a href="#ref-Roy2015">2015</a>; Whalen et al. <a href="#ref-Whalen2016">2015</a>; Zhu et al. <a href="#ref-Zhu2016">2016</a>; Schreiber et al. <a href="#ref-Schreiber2017">2017</a>; Dzida et al. <a href="#ref-Dzida2017">2017</a>)</span>, and
combine them with sequence features <span class="citation">(Zhao et al. <a href="#ref-Zhao2016">2016</a>)</span>, or evolutionary constrains
<span class="citation">(Naville et al. <a href="#ref-Naville2015">2015</a>)</span>. While these methods predict enhancer-gene association with good
performance, they require for each specific condition of interest a
multiplicity of input datasets, which are often not available.</p>
<p>Further computational approaches try to directly predict chromatin
interactions by using diverse sequence features <span class="citation">(Nikumbh and Pfeifer <a href="#ref-Nikumbh2017">2017</a>)</span> or multiple
chromatin features such as histone modifications <span class="citation">(Brackley et al. <a href="#ref-Brackley2016">2016</a>; Chen et al. <a href="#ref-Chen2016">2016</a>)</span> or
transcription <span class="citation">(Rowley et al. <a href="#ref-Rowley2017">2017</a>)</span>. One study makes use of the more recently
discovered CTCF motif directionality to predict loop interactions from
CTCF ChIP-seq peak locations <span class="citation">(Oti et al. <a href="#ref-Oti2016">2016</a>)</span>. Another study combines CTCF binding
locations and motif orientation with polymer modeling to predict Hi-C
interaction maps <span class="citation">(Sanborn et al. <a href="#ref-Sanborn2015">2015</a>)</span>. However, none of these studies predicts
chromatin loops from ChIP-seq signals of TFs different from CTCF by
taking the CTCF motif orientation into account. Furthermore, CTCF
binding sites are often only considered, when the signal is strong
enough for peak calling algorithms to identify binding sites. In
contrast, 7C takes the distribution of ChIP-seq signals from all TFs
into account without a peak-calling step. Furthermore, the other studies
do not provide a tool for the direct prediction of pairwise interactions
from single ChIP-seq experiments. Interestingly, shadow peaks in
ChIP-seq data of insulator proteins in <em>Drosophila</em> were previously
associated to long-range interactions <span class="citation">(Liang et al. <a href="#ref-Liang2014">2014</a>)</span> and used to study the
contribution of sequence motifs and co-factors in loop formation <span class="citation">(Mourad et al. <a href="#ref-Mourad2017">2017</a>)</span>,
but not to directly predict chromatin loop interactions.</p>
<p>Compared to the predictive methods mentioned above, our approach has the
clear advantage to directly predict chromatin looping interactions, and
not enhancer-promoter associations, by making use of ChIP-seq signals
from a single experiment with respect to CTCF motifs. This gives the
prediction a base pair resolution since it relies in the alignment of a
pair of CTCF motifs. In fact, given several CTCF motifs within a 1kb
genomic bin, our looping prediction approach can be used to decide which
of the CTCF sites is actually involved in the measured interactions and
thus increase resolution even when Hi-C data is available. We showed
that our approach, 7C, can work with just a single ChIP-seq experiment
for many different TFs, making it usable for many diverse conditions of
interest. Therefore, 7C can be used complementary to existing
enhancer-promoter association tools or can be integrated in such
predictive models to improve them.</p>
<p>Association of distal cis-regulatory elements, such as TF binding sites
or enhancers, to their regulated target gene is a common problem in
genomic studies <span class="citation">(Mora et al. <a href="#ref-Mora2015">2015</a>)</span>, and this can be addressed by methods mapping
contacts at a base pair resolution. While Hi-C measures pairwise
contacts genome-wide in an unbiased manner and experimentally measuring
genomic interactions is now becoming feasible due to the recent advances
in 3C based technologies <span class="citation">(Sati and Cavalli <a href="#ref-Sati2016">2016</a>)</span>, a main drawback of the Hi-C method is
the limitation of resolution. While the first Hi-C study analyzed
chromatin interactions at bin sizes of 1Mb <span class="citation">(Lieberman-Aiden et al. <a href="#ref-Lieberman-Aiden2009">2009</a>)</span>, structural features
such as topologically associating domains (TADs) were later called at
40kb bin resolution <span class="citation">(Dixon et al. <a href="#ref-Dixon2012">2012</a>)</span>, and the highest resolution for the human
genome, reached only recently, is 1kb <span class="citation">(Rao et al. <a href="#ref-Rao2014">2014</a>)</span>. This higher resolution
requires largely increased sequencing depths <span class="citation">(Rao et al. <a href="#ref-Rao2014">2014</a>; Bonev et al. <a href="#ref-Bonev2017">2017</a>)</span>. Capture Hi-C
identifies only the interactions of per-defined target regions such as
promoters <span class="citation">(Dryden et al. <a href="#ref-Dryden2014">2014</a>; Mifsud et al. <a href="#ref-Mifsud2015">2015</a>)</span>. ChIA-PET restricts the interactions to those where
a specific protein of interested is involved <span class="citation">(Heidari et al. <a href="#ref-Heidari2014">2014</a>; Fullwood et al. <a href="#ref-Fullwood2009">2009</a>; Tang et al. <a href="#ref-Tang2015">2015</a>)</span>. Therefore
these experiments are not always applicable and require a large amount
of sample material. Ultimately, even in high resolution Hi-C it is not
trivial to connect enhancers to interacting genes in a unique way, since
enhancers are often found in clusters within short distances in the
genome <span class="citation">(Whyte et al. <a href="#ref-Whyte2013">2013</a>; Parker et al. <a href="#ref-Parker2013">2013</a>)</span>.</p>
<p>Currently, our method, by using CTCF motifs, focuses on CTCF mediated
chromatin loops. It is very likely that other DNA binding proteins
mediate loops: for example, recent studies suggest that other TFs are
involved in enhancer promoter interactions during differentiation <span class="citation">(Bonev et al. <a href="#ref-Bonev2017">2017</a>)</span>
and knockout of transcriptional repressor YY1 and other candidate
factors result in loss of chromatin loops <span class="citation">(Weintraub et al. <a href="#ref-Weintraub2018">2018</a>)</span>. Using motifs predicted
for these different transcription factors, or combinations thereof, are
open avenues for the future extension of our method.</p>
</div>
<h3>References</h3>
<div id="refs" class="references">
<div id="ref-Sheffield2013">
<p>Sheffield NC, Thurman RE, Song L, Safi A, Stamatoyannopoulos J a, Lenhard B, Crawford GE, Furey TS. 2013. Patterns of regulatory activity across diverse human cell types predict tissue identity , transcription factor binding , and long-range interactions predict tissue identity , transcription factor binding , and long-range interactions. <em>Genome Research</em> <strong>23</strong>: 777–788.</p>
</div>
<div id="ref-Fishilevich2017">
<p>Fishilevich S, Nudel R, Rappaport N, Hadar R, Plaschkes I, Iny Stein T, Rosen N, Kohn A, Twik M, Safran M, et al. 2017. GeneHancer: Genome-wide integration of enhancers and target genes in genecards. <em>Database</em> <strong>2017</strong>: 1665–1680.</p>
</div>
<div id="ref-Andersson2014">
<p>Andersson R, Gebhard C, Miguel-Escalada I, Hoof I, Bornholdt J, Boyd M, Chen Y, Zhao X, Schmidl C, Suzuki T, et al. 2014. An atlas of active enhancers across human cell types and tissues. <em>Nature</em> <strong>507</strong>: 455–61.</p>
</div>
<div id="ref-OConnor2014">
<p>O’Connor TR, Bailey TL. 2014. Creating and validating cis-regulatory maps of tissue-specific gene expression regulation. <em>Nucleic acids research</em> 1–11.</p>
</div>
<div id="ref-Roy2015">
<p>Roy S, Siahpirani AF, Chasman D, Knaack S, Ay F, Stewart R, Wilson M, Sridharan R. 2015. A predictive modeling approach for cell line-specific long-range regulatory interactions. <em>Nucleic Acids Research</em> <strong>43</strong>: 8694–8712.</p>
</div>
<div id="ref-Whalen2016">
<p>Whalen S, Truty RM, Pollard KS. 2015. Enhancer-promoter interactions are encoded by complex genomic signatures on looping chromatin. <em>Nature Genetics</em> <strong>48</strong>: 1–10.</p>
</div>
<div id="ref-Zhu2016">
<p>Zhu Y, Chen Z, Zhang K, Wang M, Medovoy D, Whitaker JW, Ding B, Li N, Zheng L, Wang W. 2016. Constructing 3D interaction maps from 1D epigenomes. <em>Nature Communications</em> <strong>7</strong>: 10812.</p>
</div>
<div id="ref-Schreiber2017">
<p>Schreiber J, Libbrecht M, Bilmes J, Noble W. 2017. Nucleotide sequence and dnasei sensitivity are predictive of 3D chromatin architecture. <em>bioRxiv</em> 1–15.</p>
</div>
<div id="ref-Dzida2017">
<p>Dzida T, Iqbal M, Charapitsa I, Reid G, Stunnenberg H, Matarese F, Grote K, Honkela A, Rattray M. 2017. Predicting stimulation-dependent enhancer-promoter interactions from chip-seq time course data. <em>PeerJ</em> <strong>5</strong>: e3742.</p>
</div>
<div id="ref-Zhao2016">
<p>Zhao C, Li X, Hu H. 2016. PETModule: A motif module based approach for enhancer target gene prediction. <em>Scientific Reports</em> <strong>6</strong>: 30043.</p>
</div>
<div id="ref-Naville2015">
<p>Naville M, Ishibashi M, Ferg M, Bengani H, Rinkwitz S, Krecsmarik M, Hawkins Ta, Wilson SW, Manning E, Chilamakuri CSR, et al. 2015. Long-range evolutionary constraints reveal cis-regulatory interactions on the human x chromosome. <em>Nature Communications</em> <strong>6</strong>: 6904.</p>
</div>
<div id="ref-Nikumbh2017">
<p>Nikumbh S, Pfeifer N. 2017. Genetic sequence-based prediction of long-range chromatin interactions suggests a potential role of short tandem repeat sequences in genome organization. <em>BMC Bioinformatics</em> <strong>18</strong>: 218.</p>
</div>
<div id="ref-Brackley2016">
<p>Brackley CA, Brown JM, Waithe D, Babbs C, Davies J, Hughes JR, Buckle VJ, Marenduzzo D. 2016. Predicting the three-dimensional folding of cis -regulatory regions in mammalian genomes using bioinformatic data and polymer models. <em>Genome Biology</em> 31–36.</p>
</div>
<div id="ref-Chen2016">
<p>Chen Y, Wang Y, Xuan Z, Chen M, Zhang MQ. 2016. De novo deciphering three-dimensional chromatin interaction and topological domains by wavelet transformation of epigenetic profiles. <em>Nucleic Acids Research</em> <strong>44</strong>: e106–e106.</p>
</div>
<div id="ref-Rowley2017">
<p>Rowley MJ, Nichols MH, Lyu X, Ando-Kuri M, Rivera ISM, Hermetz K, Wang P, Ruan Y, Corces VG. 2017. Evolutionarily conserved principles predict 3D chromatin organization. <em>Molecular Cell</em> 1–16.</p>
</div>
<div id="ref-Oti2016">
<p>Oti M, Falck J, Huynen MA, Zhou H. 2016. CTCF-mediated chromatin loops enclose inducible gene regulatory domains. <em>BMC Genomics</em> <strong>17</strong>: 252.</p>
</div>
<div id="ref-Sanborn2015">
<p>Sanborn AL, Rao SSP, Huang S-C, Durand NC, Huntley MH, Jewett AI, Bochkov ID, Chinnappan D, Cutkosky A, Li J, et al. 2015. Chromatin extrusion explains key features of loop and domain formation in wild-type and engineered genomes. <em>Proceedings of the National Academy of Sciences</em> <strong>112</strong>: 201518552.</p>
</div>
<div id="ref-Liang2014">
<p>Liang J, Lacroix L, Gamot A, Cuddapah S, Queille S, Lhoumaud P, Lepetit P, Martin PGP, Vogelmann J, Court F, et al. 2014. Chromatin immunoprecipitation indirect peaks highlight long-range interactions of insulator proteins and pol ii pausing. <em>Molecular Cell</em> <strong>53</strong>: 672–681.</p>
</div>
<div id="ref-Mourad2017">
<p>Mourad R, Li L, Cuvier O. 2017. Uncovering direct and indirect molecular determinants of chromatin loops using a computational integrative approach. <em>PLOS Computational Biology</em> <strong>13</strong>: e1005538.</p>
</div>
<div id="ref-Mora2015">
<p>Mora A, Sandve GK, Gabrielsen OS, Eskeland R. 2015. In the loop: Promoter-enhancer interactions and bioinformatics. <em>Briefings in bioinformatics</em> 1–16.</p>
</div>
<div id="ref-Sati2016">
<p>Sati S, Cavalli G. 2016. Chromosome conformation capture technologies and their impact in understanding genome function. <em>Chromosoma</em>.</p>
</div>
<div id="ref-Lieberman-Aiden2009">
<p>Lieberman-Aiden E, Berkum NL van, Williams L, Imakaev M, Ragoczy T, Telling A, Amit I, Lajoie BR, Sabo PJ, Dorschner MO, et al. 2009. Comprehensive mapping of long-range interactions reveals folding principles of the human genome. <em>Science (New York, NY)</em> <strong>326</strong>: 289–93.</p>
</div>
<div id="ref-Dixon2012">
<p>Dixon JR, Selvaraj S, Yue F, Kim A, Li Y, Shen Y, Hu M, Liu JS, Ren B. 2012. Topological domains in mammalian genomes identified by analysis of chromatin interactions. <em>Nature</em> <strong>485</strong>: 376–380.</p>
</div>
<div id="ref-Rao2014">
<p>Rao SSP, Huntley MH, Durand NC, Stamenova EK, Bochkov ID, Robinson JT, Sanborn AL, Machol I, Omer AD, Lander ES, et al. 2014. A 3D map of the human genome at kilobase resolution reveals principles of chromatin looping. <em>Cell</em> <strong>159</strong>: 1665–80.</p>
</div>
<div id="ref-Bonev2017">
<p>Bonev B, Cohen NM, Szabo Q, Hugnot J-p, Tanay A, Cavalli G, Bonev B, Cohen NM, Szabo Q, Fritsch L, et al. 2017. Multiscale 3D genome rewiring during mouse article multiscale 3D genome rewiring during mouse neural development. <em>Cell</em> <strong>171</strong>: 557.e1–557.e24.</p>
</div>
<div id="ref-Dryden2014">
<p>Dryden NH, Broome LR, Dudbridge F, Johnson N, Orr N, Schoenfelder S, Nagano T, Andrews S, Wingett S, Kozarewa I, et al. 2014. Unbiased analysis of potential targets of breast cancer susceptibility loci by capture hi-c. <em>Genome research</em> 1854–1868.</p>
</div>
<div id="ref-Mifsud2015">
<p>Mifsud B, Tavares-Cadete F, Young AN, Sugar R, Schoenfelder S, Ferreira L, Wingett SW, Andrews S, Grey W, Ewels P a, et al. 2015. Mapping long-range promoter contacts in human cells with high-resolution capture hi-c. <em>Nature Genetics</em> <strong>47</strong>.</p>
</div>
<div id="ref-Heidari2014">
<p>Heidari N, Phanstiel DHD, He C, Grubert F, Jahanbani F, Kasowski M, Zhang MQ, Snyder MP. 2014. Genome-wide map of regulatory interactions in the human genome. <em>Genome research</em> <strong>24</strong>: 1905–17.</p>
</div>
<div id="ref-Fullwood2009">
<p>Fullwood MJ, Liu MH, Pan YF, Liu J, Xu H, Mohamed YB, Orlov YL, Velkov S, Ho A, Mei PH, et al. 2009. An oestrogen-receptor-<span class="math inline">\(\alpha\)</span>-bound human chromatin interactome. <em>Nature</em> <strong>462</strong>: 58–64.</p>
</div>
<div id="ref-Tang2015">
<p>Tang Z, Luo OJ, Li X, Zheng M, Zhu JJ, Szalaj P, Trzaskoma P, Magalska A, Wlodarczyk J, Ruszczycki B, et al. 2015. CTCF-mediated human 3D genome architecture reveals chromatin topology for transcription. <em>Cell</em> 1–17.</p>
</div>
<div id="ref-Whyte2013">
<p>Whyte WA, Orlando DA, Hnisz D, Abraham BJ, Lin CY, Kagey MH, Rahl PB, Lee TI, Young RA. 2013. Master transcription factors and mediator establish super-enhancers at key cell identity genes. <em>Cell</em> <strong>153</strong>: 307–19.</p>
</div>
<div id="ref-Parker2013">
<p>Parker SCJ, Stitzel ML, Taylor DL, Orozco JM, Erdos MR, Akiyama JA, Bueren KL van, Chines PS, Narisu N, NISC Comparative Sequencing Program NCS, et al. 2013. Chromatin stretch enhancer states drive cell-specific gene regulation and harbor human disease risk variants. <em>Proceedings of the National Academy of Sciences of the United States of America</em> <strong>110</strong>: 17921–6.</p>
</div>
<div id="ref-Weintraub2018">
<p>Weintraub AS, Li CH, Zamudio AV, Sigova AA, Hannet NM, Day DS, Abraham BJ, Cohen MA, Nabet B, Buckley DL, et al. 2018. YY1 is a structural regulator of enhancer-promoter loops. <em>Cell</em> 1–16.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="5-2-results-3.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="5-4-conclusion-2.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": true,
"facebook": false,
"twitter": true,
"google": false,
"weibo": false,
"instapper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/ibn-salem/thesis/edit/master/05-loop_prediction.Rmd",
"text": "Edit"
},
"download": ["thesis.pdf", "thesis.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://cdn.bootcss.com/mathjax/2.7.1/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:" && /^https?:/.test(src))
      src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
