<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>Genome folding in evolution and disease</title>
  <meta name="description" content="A draft PhD thesis on genome folding in evoultion and disease.">
  <meta name="generator" content="bookdown 0.6.1 and GitBook 2.6.7">

  <meta property="og:title" content="Genome folding in evolution and disease" />
  <meta property="og:type" content="book" />
  <meta property="og:url" content="https://ibn-salem.github.io/thesis/" />
  
  <meta property="og:description" content="A draft PhD thesis on genome folding in evoultion and disease." />
  <meta name="github-repo" content="ibn-salem/thesis" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Genome folding in evolution and disease" />
  
  <meta name="twitter:description" content="A draft PhD thesis on genome folding in evoultion and disease." />
  

<meta name="author" content="Jonas Ibn-Salem">


<meta name="date" content="2018-02-20">

  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  
<link rel="prev" href="5-4-conclusion-2.html">
<link rel="next" href="5-6-declarations-1.html">
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />










<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Thesis</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="1" data-path="1-intro.html"><a href="1-intro.html"><i class="fa fa-check"></i><b>1</b> Introduction</a><ul>
<li class="chapter" data-level="1.1" data-path="1-1-regulation-of-expression.html"><a href="1-1-regulation-of-expression.html"><i class="fa fa-check"></i><b>1.1</b> Regulation of gene expression</a></li>
<li class="chapter" data-level="1.2" data-path="1-2-enhancers.html"><a href="1-2-enhancers.html"><i class="fa fa-check"></i><b>1.2</b> Enhancers</a></li>
<li class="chapter" data-level="1.3" data-path="1-3-methods-to-probe-the-3d-chromatin-architecture.html"><a href="1-3-methods-to-probe-the-3d-chromatin-architecture.html"><i class="fa fa-check"></i><b>1.3</b> Methods to probe the 3D chromatin architecture</a><ul>
<li class="chapter" data-level="1.3.1" data-path="1-3-methods-to-probe-the-3d-chromatin-architecture.html"><a href="1-3-methods-to-probe-the-3d-chromatin-architecture.html#microscopy-based-techniques-to-visualize-the-genome-in-3d"><i class="fa fa-check"></i><b>1.3.1</b> Microscopy-based techniques to visualize the genome in 3D</a></li>
<li class="chapter" data-level="1.3.2" data-path="1-3-methods-to-probe-the-3d-chromatin-architecture.html"><a href="1-3-methods-to-probe-the-3d-chromatin-architecture.html#proximity-ligation"><i class="fa fa-check"></i><b>1.3.2</b> Proximity-ligation based method to quantify chromatin interactions</a></li>
</ul></li>
<li class="chapter" data-level="1.4" data-path="1-4-hierarchy-of-chromatin-3d-structure.html"><a href="1-4-hierarchy-of-chromatin-3d-structure.html"><i class="fa fa-check"></i><b>1.4</b> Hierarchy of chromatin 3D structure</a><ul>
<li class="chapter" data-level="1.4.1" data-path="1-4-hierarchy-of-chromatin-3d-structure.html"><a href="1-4-hierarchy-of-chromatin-3d-structure.html#chromosomal-territories-and-inter-chromosomal-contacts"><i class="fa fa-check"></i><b>1.4.1</b> Chromosomal territories and inter-chromosomal contacts</a></li>
<li class="chapter" data-level="1.4.2" data-path="1-4-hierarchy-of-chromatin-3d-structure.html"><a href="1-4-hierarchy-of-chromatin-3d-structure.html#ab-compartments"><i class="fa fa-check"></i><b>1.4.2</b> A/B compartments</a></li>
<li class="chapter" data-level="1.4.3" data-path="1-4-hierarchy-of-chromatin-3d-structure.html"><a href="1-4-hierarchy-of-chromatin-3d-structure.html#topologically-associating-domains-tads"><i class="fa fa-check"></i><b>1.4.3</b> Topologically associating domains (TADs)</a></li>
<li class="chapter" data-level="1.4.4" data-path="1-4-hierarchy-of-chromatin-3d-structure.html"><a href="1-4-hierarchy-of-chromatin-3d-structure.html#hierarchy-of-domain-like-structures-across-genomic-length-scales"><i class="fa fa-check"></i><b>1.4.4</b> Hierarchy of domain-like structures across genomic-length scales</a></li>
<li class="chapter" data-level="1.4.5" data-path="1-4-hierarchy-of-chromatin-3d-structure.html"><a href="1-4-hierarchy-of-chromatin-3d-structure.html#chromatin-looping-interactions"><i class="fa fa-check"></i><b>1.4.5</b> Chromatin looping interactions</a></li>
<li class="chapter" data-level="1.4.6" data-path="1-4-hierarchy-of-chromatin-3d-structure.html"><a href="1-4-hierarchy-of-chromatin-3d-structure.html#loop-mechanism"><i class="fa fa-check"></i><b>1.4.6</b> TAD and loop formation by architectural proteins</a></li>
</ul></li>
<li class="chapter" data-level="1.5" data-path="1-5-dynamics-of-chromatin-structure.html"><a href="1-5-dynamics-of-chromatin-structure.html"><i class="fa fa-check"></i><b>1.5</b> Dynamics of chromatin structure</a><ul>
<li class="chapter" data-level="1.5.1" data-path="1-5-dynamics-of-chromatin-structure.html"><a href="1-5-dynamics-of-chromatin-structure.html#variation-of-chromatin-structure-across-cell-types"><i class="fa fa-check"></i><b>1.5.1</b> Variation of chromatin structure across cell types</a></li>
</ul></li>
<li class="chapter" data-level="1.6" data-path="1-6-evolution-of-chromatin-organization.html"><a href="1-6-evolution-of-chromatin-organization.html"><i class="fa fa-check"></i><b>1.6</b> Evolution of chromatin organization</a></li>
<li class="chapter" data-level="1.7" data-path="1-7-disruption-of-chromatin-architecture-in-disease.html"><a href="1-7-disruption-of-chromatin-architecture-in-disease.html"><i class="fa fa-check"></i><b>1.7</b> Disruption of chromatin architecture in disease</a></li>
<li class="chapter" data-level="1.8" data-path="1-8-aims-of-this-thesis.html"><a href="1-8-aims-of-this-thesis.html"><i class="fa fa-check"></i><b>1.8</b> Aims of this thesis</a></li>
<li class="chapter" data-level="1.9" data-path="1-9-structure-of-this-thesis.html"><a href="1-9-structure-of-this-thesis.html"><i class="fa fa-check"></i><b>1.9</b> Structure of this thesis</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="2-paralog-regulation.html"><a href="2-paralog-regulation.html"><i class="fa fa-check"></i><b>2</b> Paralog genes in the 3D genome architecture</a><ul>
<li class="chapter" data-level="" data-path="2-paralog-regulation.html"><a href="2-paralog-regulation.html#preamble"><i class="fa fa-check"></i>Preamble</a></li>
<li class="chapter" data-level="" data-path="2-paralog-regulation.html"><a href="2-paralog-regulation.html#abstract-1"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="2.1" data-path="2-1-introduction.html"><a href="2-1-introduction.html"><i class="fa fa-check"></i><b>2.1</b> Introduction</a></li>
<li class="chapter" data-level="2.2" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html"><i class="fa fa-check"></i><b>2.2</b> Materials and methods</a><ul>
<li class="chapter" data-level="2.2.1" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html#selection-of-pairs-of-paralog-genes"><i class="fa fa-check"></i><b>2.2.1</b> Selection of pairs of paralog genes</a></li>
<li class="chapter" data-level="2.2.2" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html#enhancers-to-gene-association"><i class="fa fa-check"></i><b>2.2.2</b> Enhancers to gene association</a></li>
<li class="chapter" data-level="2.2.3" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html#topological-associating-domains"><i class="fa fa-check"></i><b>2.2.3</b> Topological associating domains</a></li>
<li class="chapter" data-level="2.2.4" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html#hi-c-interaction-maps"><i class="fa fa-check"></i><b>2.2.4</b> Hi-C interaction maps</a></li>
<li class="chapter" data-level="2.2.5" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html#randomization"><i class="fa fa-check"></i><b>2.2.5</b> Randomization</a></li>
<li class="chapter" data-level="2.2.6" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html#statistical-tests"><i class="fa fa-check"></i><b>2.2.6</b> Statistical tests</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="2-3-results.html"><a href="2-3-results.html"><i class="fa fa-check"></i><b>2.3</b> Results</a><ul>
<li class="chapter" data-level="2.3.1" data-path="2-3-results.html"><a href="2-3-results.html#distribution-of-paralog-genes-in-the-human-genome"><i class="fa fa-check"></i><b>2.3.1</b> Distribution of paralog genes in the human genome</a></li>
<li class="chapter" data-level="2.3.2" data-path="2-3-results.html"><a href="2-3-results.html#co-expression-of-paralog-gene-pairs-across-tissues"><i class="fa fa-check"></i><b>2.3.2</b> Co-expression of paralog gene pairs across tissues</a></li>
<li class="chapter" data-level="2.3.3" data-path="2-3-results.html"><a href="2-3-results.html#paralog-genes-share-enhancers"><i class="fa fa-check"></i><b>2.3.3</b> Paralog genes share enhancers</a></li>
<li class="chapter" data-level="2.3.4" data-path="2-3-results.html"><a href="2-3-results.html#co-localization-of-paralogs-in-tads"><i class="fa fa-check"></i><b>2.3.4</b> Co-localization of paralogs in TADs</a></li>
<li class="chapter" data-level="2.3.5" data-path="2-3-results.html"><a href="2-3-results.html#distal-paralog-pairs-are-enriched-for-long-range-chromatin-contacts"><i class="fa fa-check"></i><b>2.3.5</b> Distal paralog pairs are enriched for long-range chromatin contacts</a></li>
<li class="chapter" data-level="2.3.6" data-path="2-3-results.html"><a href="2-3-results.html#close-paralogs-have-fewer-contacts-than-expected"><i class="fa fa-check"></i><b>2.3.6</b> Close paralogs have fewer contacts than expected</a></li>
<li class="chapter" data-level="2.3.7" data-path="2-3-results.html"><a href="2-3-results.html#paralogs-in-mouse-and-dog-genome"><i class="fa fa-check"></i><b>2.3.7</b> Paralogs in mouse and dog genome</a></li>
<li class="chapter" data-level="2.3.8" data-path="2-3-results.html"><a href="2-3-results.html#orthologs-of-human-paralogs-show-conserved-co-localization"><i class="fa fa-check"></i><b>2.3.8</b> Orthologs of human paralogs show conserved co-localization</a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="2-4-discussions.html"><a href="2-4-discussions.html"><i class="fa fa-check"></i><b>2.4</b> Discussions</a></li>
<li class="chapter" data-level="2.5" data-path="2-5-conclusion.html"><a href="2-5-conclusion.html"><i class="fa fa-check"></i><b>2.5</b> Conclusion</a></li>
<li class="chapter" data-level="2.6" data-path="2-6-acknowledgements.html"><a href="2-6-acknowledgements.html"><i class="fa fa-check"></i><b>2.6</b> Acknowledgements</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="3-TAD-evolution.html"><a href="3-TAD-evolution.html"><i class="fa fa-check"></i><b>3</b> Stability of TADs in evolution</a><ul>
<li class="chapter" data-level="" data-path="3-TAD-evolution.html"><a href="3-TAD-evolution.html#preamble-1"><i class="fa fa-check"></i>Preamble</a></li>
<li class="chapter" data-level="" data-path="3-TAD-evolution.html"><a href="3-TAD-evolution.html#abstract-2"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="" data-path="3-TAD-evolution.html"><a href="3-TAD-evolution.html#keywords"><i class="fa fa-check"></i>Keywords</a></li>
<li class="chapter" data-level="3.1" data-path="3-1-introduction-1.html"><a href="3-1-introduction-1.html"><i class="fa fa-check"></i><b>3.1</b> Introduction</a></li>
<li class="chapter" data-level="3.2" data-path="3-2-results-1.html"><a href="3-2-results-1.html"><i class="fa fa-check"></i><b>3.2</b> Results</a><ul>
<li class="chapter" data-level="3.2.1" data-path="3-2-results-1.html"><a href="3-2-results-1.html#identification-of-evolutionary-rearrangement-breakpoints-from-whole-genome-alignments"><i class="fa fa-check"></i><b>3.2.1</b> Identification of evolutionary rearrangement breakpoints from whole-genome alignments</a></li>
<li class="chapter" data-level="3.2.2" data-path="3-2-results-1.html"><a href="3-2-results-1.html#rearrangement-breakpoints-are-enriched-at-tad-boundaries"><i class="fa fa-check"></i><b>3.2.2</b> Rearrangement breakpoints are enriched at TAD boundaries</a></li>
<li class="chapter" data-level="3.2.3" data-path="3-2-results-1.html"><a href="3-2-results-1.html#clusters-of-conserved-non-coding-elements-are-depleted-for-rearrangement-breakpoints"><i class="fa fa-check"></i><b>3.2.3</b> Clusters of conserved non-coding elements are depleted for rearrangement breakpoints</a></li>
<li class="chapter" data-level="3.2.4" data-path="3-2-results-1.html"><a href="3-2-results-1.html#rearranged-tads-are-associated-with-divergent-gene-expression-between-species"><i class="fa fa-check"></i><b>3.2.4</b> Rearranged TADs are associated with divergent gene expression between species</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="3-3-discussion.html"><a href="3-3-discussion.html"><i class="fa fa-check"></i><b>3.3</b> Discussion</a></li>
<li class="chapter" data-level="3.4" data-path="3-4-conclusion-1.html"><a href="3-4-conclusion-1.html"><i class="fa fa-check"></i><b>3.4</b> Conclusion</a></li>
<li class="chapter" data-level="3.5" data-path="3-5-methods.html"><a href="3-5-methods.html"><i class="fa fa-check"></i><b>3.5</b> Methods</a><ul>
<li class="chapter" data-level="3.5.1" data-path="3-5-methods.html"><a href="3-5-methods.html#rearrangement-breakpoints-from-whole-genome-alignments"><i class="fa fa-check"></i><b>3.5.1</b> Rearrangement breakpoints from whole-genome alignments</a></li>
<li class="chapter" data-level="3.5.2" data-path="3-5-methods.html"><a href="3-5-methods.html#topologically-associating-domains-and-contact-domains"><i class="fa fa-check"></i><b>3.5.2</b> Topologically associating domains and contact domains</a></li>
<li class="chapter" data-level="3.5.3" data-path="3-5-methods.html"><a href="3-5-methods.html#breakpoint-distributions-at-tads"><i class="fa fa-check"></i><b>3.5.3</b> Breakpoint distributions at TADs</a></li>
<li class="chapter" data-level="3.5.4" data-path="3-5-methods.html"><a href="3-5-methods.html#quantification-of-breakpoint-enrichment"><i class="fa fa-check"></i><b>3.5.4</b> Quantification of breakpoint enrichment</a></li>
<li class="chapter" data-level="3.5.5" data-path="3-5-methods.html"><a href="3-5-methods.html#expression-data-for-mouse-and-human-orthologs"><i class="fa fa-check"></i><b>3.5.5</b> Expression data for mouse and human orthologs</a></li>
<li class="chapter" data-level="3.5.6" data-path="3-5-methods.html"><a href="3-5-methods.html#classification-of-tads-and-genes-according-to-rearrangements-and-grbs"><i class="fa fa-check"></i><b>3.5.6</b> Classification of TADs and genes according to rearrangements and GRBs</a></li>
<li class="chapter" data-level="3.5.7" data-path="3-5-methods.html"><a href="3-5-methods.html#source-code-and-implementation-details"><i class="fa fa-check"></i><b>3.5.7</b> Source code and implementation details</a></li>
</ul></li>
<li class="chapter" data-level="3.6" data-path="3-6-declarations.html"><a href="3-6-declarations.html"><i class="fa fa-check"></i><b>3.6</b> Declarations</a><ul>
<li class="chapter" data-level="3.6.1" data-path="3-6-declarations.html"><a href="3-6-declarations.html#ethics-approval-and-consent-to-participate"><i class="fa fa-check"></i><b>3.6.1</b> Ethics approval and consent to participate</a></li>
<li class="chapter" data-level="3.6.2" data-path="3-6-declarations.html"><a href="3-6-declarations.html#consent-for-publication"><i class="fa fa-check"></i><b>3.6.2</b> Consent for publication</a></li>
<li class="chapter" data-level="3.6.3" data-path="3-6-declarations.html"><a href="3-6-declarations.html#availability-of-data-and-material"><i class="fa fa-check"></i><b>3.6.3</b> Availability of data and material</a></li>
<li class="chapter" data-level="3.6.4" data-path="3-6-declarations.html"><a href="3-6-declarations.html#competing-interests"><i class="fa fa-check"></i><b>3.6.4</b> Competing interests</a></li>
<li class="chapter" data-level="3.6.5" data-path="3-6-declarations.html"><a href="3-6-declarations.html#funding"><i class="fa fa-check"></i><b>3.6.5</b> Funding</a></li>
<li class="chapter" data-level="3.6.6" data-path="3-6-declarations.html"><a href="3-6-declarations.html#authors-contributions"><i class="fa fa-check"></i><b>3.6.6</b> Authors‘ contributions</a></li>
<li class="chapter" data-level="3.6.7" data-path="3-6-declarations.html"><a href="3-6-declarations.html#acknowledgments"><i class="fa fa-check"></i><b>3.6.7</b> Acknowledgments</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="4-position-effect.html"><a href="4-position-effect.html"><i class="fa fa-check"></i><b>4</b> Position effects of rearrangements in disease genomes</a><ul>
<li class="chapter" data-level="" data-path="4-position-effect.html"><a href="4-position-effect.html#preamble-2"><i class="fa fa-check"></i>Preamble</a></li>
<li class="chapter" data-level="" data-path="4-position-effect.html"><a href="4-position-effect.html#abstract-3"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="4.1" data-path="4-1-introduction-2.html"><a href="4-1-introduction-2.html"><i class="fa fa-check"></i><b>4.1</b> Introduction</a></li>
<li class="chapter" data-level="4.2" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html"><i class="fa fa-check"></i><b>4.2</b> Materials and Methods</a><ul>
<li class="chapter" data-level="4.2.1" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html#selection-of-subjects-with-apparently-balanced-chromosome-abnormalities"><i class="fa fa-check"></i><b>4.2.1</b> Selection of subjects with apparently balanced chromosome abnormalities</a></li>
<li class="chapter" data-level="4.2.2" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html#clinical-descriptions-of-dgap-cases"><i class="fa fa-check"></i><b>4.2.2</b> Clinical descriptions of DGAP cases</a></li>
<li class="chapter" data-level="4.2.3" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html#analysis-of-genes-bordering-the-rearrangement-breakpoints"><i class="fa fa-check"></i><b>4.2.3</b> Analysis of genes bordering the rearrangement breakpoints</a></li>
<li class="chapter" data-level="4.2.4" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html#assessment-of-disrupted-functional-elements-and-chromatin-interactions-bordering-rearrangement-breakpoints"><i class="fa fa-check"></i><b>4.2.4</b> Assessment of disrupted functional elements and chromatin interactions bordering rearrangement breakpoints</a></li>
<li class="chapter" data-level="4.2.5" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html#ontological-analysis-of-genes-neighboring-breakpoints"><i class="fa fa-check"></i><b>4.2.5</b> Ontological analysis of genes neighboring breakpoints</a></li>
<li class="chapter" data-level="4.2.6" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html#quantitative-real-time-pcr"><i class="fa fa-check"></i><b>4.2.6</b> Quantitative real-time PCR</a></li>
<li class="chapter" data-level="4.2.7" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html#assessment-of-dgap-breakpoints-overlapping-with-non-coding-structural-variants-in-public-databases"><i class="fa fa-check"></i><b>4.2.7</b> Assessment of DGAP breakpoints overlapping with non-coding structural variants in public databases</a></li>
</ul></li>
<li class="chapter" data-level="4.3" data-path="4-3-results-2.html"><a href="4-3-results-2.html"><i class="fa fa-check"></i><b>4.3</b> Results</a><ul>
<li class="chapter" data-level="4.3.1" data-path="4-3-results-2.html"><a href="4-3-results-2.html#genomic-characterization-of-non-coding-breakpoints"><i class="fa fa-check"></i><b>4.3.1</b> Genomic characterization of non-coding breakpoints</a></li>
<li class="chapter" data-level="4.3.2" data-path="4-3-results-2.html"><a href="4-3-results-2.html#identification-of-genes-with-potential-position-effects"><i class="fa fa-check"></i><b>4.3.2</b> Identification of genes with potential position effects</a></li>
<li class="chapter" data-level="4.3.3" data-path="4-3-results-2.html"><a href="4-3-results-2.html#identification-of-subjects-with-shared-non-coding-chromosome-alterations-and-phenotypes"><i class="fa fa-check"></i><b>4.3.3</b> Identification of subjects with shared non-coding chromosome alterations and phenotypes</a></li>
</ul></li>
<li class="chapter" data-level="4.4" data-path="4-4-discussion-1.html"><a href="4-4-discussion-1.html"><i class="fa fa-check"></i><b>4.4</b> Discussion</a><ul>
<li class="chapter" data-level="" data-path="4-4-discussion-1.html"><a href="4-4-discussion-1.html#acknowledgements-1"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="4.4.1" data-path="4-4-discussion-1.html"><a href="4-4-discussion-1.html#web-resources"><i class="fa fa-check"></i><b>4.4.1</b> Web Resources</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="5-loop.html"><a href="5-loop.html"><i class="fa fa-check"></i><b>5</b> Prediction of chromatin looping interactions</a><ul>
<li class="chapter" data-level="" data-path="5-loop.html"><a href="5-loop.html#preamble-3"><i class="fa fa-check"></i>Preamble</a></li>
<li class="chapter" data-level="" data-path="5-loop.html"><a href="5-loop.html#abstract-4"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="" data-path="5-loop.html"><a href="5-loop.html#keywords-1"><i class="fa fa-check"></i>Keywords</a></li>
<li class="chapter" data-level="5.1" data-path="5-1-introduction-3.html"><a href="5-1-introduction-3.html"><i class="fa fa-check"></i><b>5.1</b> Introduction</a></li>
<li class="chapter" data-level="5.2" data-path="5-2-results-3.html"><a href="5-2-results-3.html"><i class="fa fa-check"></i><b>5.2</b> Results</a><ul>
<li class="chapter" data-level="5.2.1" data-path="5-2-results-3.html"><a href="5-2-results-3.html#ctcf-motif-pairs-as-candidate-chromatin-loop-anchors"><i class="fa fa-check"></i><b>5.2.1</b> CTCF motif pairs as candidate chromatin loop anchors</a></li>
<li class="chapter" data-level="5.2.2" data-path="5-2-results-3.html"><a href="5-2-results-3.html#similarity-of-chip-seq-signals-at-looping-ctcf-motifs"><i class="fa fa-check"></i><b>5.2.2</b> Similarity of ChIP-seq signals at looping CTCF motifs</a></li>
<li class="chapter" data-level="5.2.3" data-path="5-2-results-3.html"><a href="5-2-results-3.html#genomic-sequence-features-of-ctcf-motif-pairs-are-associated-with-looping"><i class="fa fa-check"></i><b>5.2.3</b> Genomic sequence features of CTCF motif pairs are associated with looping</a></li>
<li class="chapter" data-level="5.2.4" data-path="5-2-results-3.html"><a href="5-2-results-3.html#chromatin-loop-prediction-using-7c"><i class="fa fa-check"></i><b>5.2.4</b> Chromatin loop prediction using 7C</a></li>
<li class="chapter" data-level="5.2.5" data-path="5-2-results-3.html"><a href="5-2-results-3.html#prediction-performance-evaluation"><i class="fa fa-check"></i><b>5.2.5</b> Prediction performance evaluation</a></li>
<li class="chapter" data-level="5.2.6" data-path="5-2-results-3.html"><a href="5-2-results-3.html#prediction-performance-of-7c-with-sequence-features-and-single-tf-chip-seq-data-sets"><i class="fa fa-check"></i><b>5.2.6</b> Prediction performance of 7C with sequence features and single TF ChIP-seq data sets</a></li>
<li class="chapter" data-level="5.2.7" data-path="5-2-results-3.html"><a href="5-2-results-3.html#comparison-of-transcription-factors-by-prediction-performance"><i class="fa fa-check"></i><b>5.2.7</b> Comparison of transcription factors by prediction performance</a></li>
<li class="chapter" data-level="5.2.8" data-path="5-2-results-3.html"><a href="5-2-results-3.html#prediction-performance-in-other-cell-types-and-for-different-tfs"><i class="fa fa-check"></i><b>5.2.8</b> Prediction performance in other cell types and for different TFs</a></li>
<li class="chapter" data-level="5.2.9" data-path="5-2-results-3.html"><a href="5-2-results-3.html#the-high-resolution-of-chip-nexus-improves-prediction-performance"><i class="fa fa-check"></i><b>5.2.9</b> The high resolution of ChIP-nexus improves prediction performance</a></li>
</ul></li>
<li class="chapter" data-level="5.3" data-path="5-3-discussion-2.html"><a href="5-3-discussion-2.html"><i class="fa fa-check"></i><b>5.3</b> Discussion</a></li>
<li class="chapter" data-level="5.4" data-path="5-4-conclusion-2.html"><a href="5-4-conclusion-2.html"><i class="fa fa-check"></i><b>5.4</b> Conclusion</a></li>
<li class="chapter" data-level="5.5" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html"><i class="fa fa-check"></i><b>5.5</b> Methods</a><ul>
<li class="chapter" data-level="5.5.1" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#ctcf-motifs-in-the-human-genome"><i class="fa fa-check"></i><b>5.5.1</b> CTCF motifs in the human genome</a></li>
<li class="chapter" data-level="5.5.2" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#loop-interaction-data-for-training-and-validation"><i class="fa fa-check"></i><b>5.5.2</b> Loop interaction data for training and validation</a></li>
<li class="chapter" data-level="5.5.3" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#chip-seq-datasets-in-gm12878-cells"><i class="fa fa-check"></i><b>5.5.3</b> ChIP-seq datasets in GM12878 cells</a></li>
<li class="chapter" data-level="5.5.4" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#chip-seq-data-types"><i class="fa fa-check"></i><b>5.5.4</b> ChIP-seq data types</a></li>
<li class="chapter" data-level="5.5.5" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#chip-nexus-data-processing-for-rad21-and-scm3"><i class="fa fa-check"></i><b>5.5.5</b> ChIP-nexus data processing for RAD21 and SCM3</a></li>
<li class="chapter" data-level="5.5.6" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#similarity-of-chip-seq-profiles-as-correlation-of-coverage-around-motifs"><i class="fa fa-check"></i><b>5.5.6</b> Similarity of ChIP-seq profiles as correlation of coverage around motifs</a></li>
<li class="chapter" data-level="5.5.7" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#genomic-sequence-features-of-chromatin-loops"><i class="fa fa-check"></i><b>5.5.7</b> Genomic sequence features of chromatin loops</a></li>
<li class="chapter" data-level="5.5.8" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#chromatin-loop-prediction-model"><i class="fa fa-check"></i><b>5.5.8</b> Chromatin Loop prediction model</a></li>
<li class="chapter" data-level="5.5.9" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#training-and-validation-of-prediction-model"><i class="fa fa-check"></i><b>5.5.9</b> Training and validation of prediction model</a></li>
<li class="chapter" data-level="5.5.10" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#analysis-of-prediction-performance"><i class="fa fa-check"></i><b>5.5.10</b> Analysis of prediction performance</a></li>
<li class="chapter" data-level="5.5.11" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#implementation-of-7c-and-compatibility-to-other-tools"><i class="fa fa-check"></i><b>5.5.11</b> Implementation of 7C and compatibility to other tools</a></li>
</ul></li>
<li class="chapter" data-level="5.6" data-path="5-6-declarations-1.html"><a href="5-6-declarations-1.html"><i class="fa fa-check"></i><b>5.6</b> Declarations</a><ul>
<li class="chapter" data-level="5.6.1" data-path="5-6-declarations-1.html"><a href="5-6-declarations-1.html#ethics-approval-and-consent-to-participate-1"><i class="fa fa-check"></i><b>5.6.1</b> Ethics approval and consent to participate</a></li>
<li class="chapter" data-level="5.6.2" data-path="5-6-declarations-1.html"><a href="5-6-declarations-1.html#consent-for-publication-1"><i class="fa fa-check"></i><b>5.6.2</b> Consent for publication</a></li>
<li class="chapter" data-level="5.6.3" data-path="5-6-declarations-1.html"><a href="5-6-declarations-1.html#availability-of-data-and-material-1"><i class="fa fa-check"></i><b>5.6.3</b> Availability of data and material</a></li>
<li class="chapter" data-level="5.6.4" data-path="5-6-declarations-1.html"><a href="5-6-declarations-1.html#competing-interests-1"><i class="fa fa-check"></i><b>5.6.4</b> Competing interests</a></li>
<li class="chapter" data-level="5.6.5" data-path="5-6-declarations-1.html"><a href="5-6-declarations-1.html#funding-1"><i class="fa fa-check"></i><b>5.6.5</b> Funding</a></li>
<li class="chapter" data-level="5.6.6" data-path="5-6-declarations-1.html"><a href="5-6-declarations-1.html#authors-contributions-1"><i class="fa fa-check"></i><b>5.6.6</b> Authors‘ contributions</a></li>
<li class="chapter" data-level="5.6.7" data-path="5-6-declarations-1.html"><a href="5-6-declarations-1.html#acknowledgments-1"><i class="fa fa-check"></i><b>5.6.7</b> Acknowledgments</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="6-thesis-discussion.html"><a href="6-thesis-discussion.html"><i class="fa fa-check"></i><b>6</b> Discussion</a><ul>
<li class="chapter" data-level="6.1" data-path="6-1-co-regulation-of-functionally-related-genes-in-tads.html"><a href="6-1-co-regulation-of-functionally-related-genes-in-tads.html"><i class="fa fa-check"></i><b>6.1</b> Co-regulation of functionally related genes in TADs</a></li>
<li class="chapter" data-level="6.2" data-path="6-2-evolutionary-novelties-by-gene-duplications-and-altered-regulatory-environments.html"><a href="6-2-evolutionary-novelties-by-gene-duplications-and-altered-regulatory-environments.html"><i class="fa fa-check"></i><b>6.2</b> Evolutionary novelties by gene duplications and altered regulatory environments</a></li>
<li class="chapter" data-level="6.3" data-path="6-3-tads-are-stable-across-large-evolutionary-time-scales.html"><a href="6-3-tads-are-stable-across-large-evolutionary-time-scales.html"><i class="fa fa-check"></i><b>6.3</b> TADs are stable across large evolutionary time-scales</a></li>
<li class="chapter" data-level="6.4" data-path="6-4-TAD-disease.html"><a href="6-4-TAD-disease.html"><i class="fa fa-check"></i><b>6.4</b> Gene miss-expression by altered TADs in disease</a></li>
<li class="chapter" data-level="6.5" data-path="6-5-integrating-genome-folding-data-can-explain-the-pathomechanism-structural-variants.html"><a href="6-5-integrating-genome-folding-data-can-explain-the-pathomechanism-structural-variants.html"><i class="fa fa-check"></i><b>6.5</b> Integrating genome folding data can explain the pathomechanism structural variants</a></li>
<li class="chapter" data-level="6.6" data-path="6-6-condition-specific-genome-folding.html"><a href="6-6-condition-specific-genome-folding.html"><i class="fa fa-check"></i><b>6.6</b> Condition specific genome folding</a><ul>
<li class="chapter" data-level="6.6.1" data-path="6-6-condition-specific-genome-folding.html"><a href="6-6-condition-specific-genome-folding.html#interpretation-of-genetic-variants-by-interacting-genes"><i class="fa fa-check"></i><b>6.6.1</b> Interpretation of genetic variants by interacting genes</a></li>
<li class="chapter" data-level="6.6.2" data-path="6-6-condition-specific-genome-folding.html"><a href="6-6-condition-specific-genome-folding.html#cell-type-specific-regulatory-interactions-between-enhancers-and-genes"><i class="fa fa-check"></i><b>6.6.2</b> Cell type-specific regulatory interactions between enhancers and genes</a></li>
<li class="chapter" data-level="6.6.3" data-path="6-6-condition-specific-genome-folding.html"><a href="6-6-condition-specific-genome-folding.html#variability-of-long-range-interactions-across-individuals"><i class="fa fa-check"></i><b>6.6.3</b> Variability of long-range interactions across individuals</a></li>
<li class="chapter" data-level="6.6.4" data-path="6-6-condition-specific-genome-folding.html"><a href="6-6-condition-specific-genome-folding.html#genome-folding-in-single-cells"><i class="fa fa-check"></i><b>6.6.4</b> Genome folding in single cells</a></li>
<li class="chapter" data-level="6.6.5" data-path="6-6-condition-specific-genome-folding.html"><a href="6-6-condition-specific-genome-folding.html#constantly-improving-targeted-experimental-methods."><i class="fa fa-check"></i><b>6.6.5</b> Constantly improving targeted experimental methods.</a></li>
</ul></li>
<li class="chapter" data-level="6.7" data-path="6-7-architectural-proteins-mediate-chromatin-looping.html"><a href="6-7-architectural-proteins-mediate-chromatin-looping.html"><i class="fa fa-check"></i><b>6.7</b> Architectural proteins mediate chromatin looping</a></li>
<li class="chapter" data-level="6.8" data-path="6-8-further-directions.html"><a href="6-8-further-directions.html"><i class="fa fa-check"></i><b>6.8</b> Further directions</a><ul>
<li class="chapter" data-level="6.8.1" data-path="6-8-further-directions.html"><a href="6-8-further-directions.html#limitation-of-the-analysis"><i class="fa fa-check"></i><b>6.8.1</b> Limitation of the analysis</a></li>
</ul></li>
<li class="chapter" data-level="6.9" data-path="6-9-conclusions.html"><a href="6-9-conclusions.html"><i class="fa fa-check"></i><b>6.9</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="appendix"><span><b>Appendix</b></span></li>
<li class="chapter" data-level="A" data-path="A-sup-paralog-regulation.html"><a href="A-sup-paralog-regulation.html"><i class="fa fa-check"></i><b>A</b> Co-regulation of paralog genes: Supporting Information</a></li>
<li class="chapter" data-level="B" data-path="B-sup-tad-evolution.html"><a href="B-sup-tad-evolution.html"><i class="fa fa-check"></i><b>B</b> TAD evolution: Supporting Information</a><ul>
<li class="chapter" data-level="B.1" data-path="B-1-TadEvoSupTab.html"><a href="B-1-TadEvoSupTab.html"><i class="fa fa-check"></i><b>B.1</b> Supplementary Tables</a></li>
<li class="chapter" data-level="B.2" data-path="B-2-supplementary-figures.html"><a href="B-2-supplementary-figures.html"><i class="fa fa-check"></i><b>B.2</b> Supplementary Figures</a></li>
</ul></li>
<li class="chapter" data-level="C" data-path="C-sup-position-effect.html"><a href="C-sup-position-effect.html"><i class="fa fa-check"></i><b>C</b> Position Effect: Supplemental Data</a><ul>
<li class="chapter" data-level="C.1" data-path="C-1-supplemental-note.html"><a href="C-1-supplemental-note.html"><i class="fa fa-check"></i><b>C.1</b> Supplemental Note</a><ul>
<li class="chapter" data-level="C.1.1" data-path="C-1-supplemental-note.html"><a href="C-1-supplemental-note.html#case-reports"><i class="fa fa-check"></i><b>C.1.1</b> Case Reports</a></li>
<li class="chapter" data-level="C.1.2" data-path="C-1-supplemental-note.html"><a href="C-1-supplemental-note.html#nucleotide-level-nomenclature-for-dgap-karyotypes"><i class="fa fa-check"></i><b>C.1.2</b> Nucleotide Level Nomenclature for DGAP karyotypes</a></li>
</ul></li>
<li class="chapter" data-level="C.2" data-path="C-2-supplemental-figure.html"><a href="C-2-supplemental-figure.html"><i class="fa fa-check"></i><b>C.2</b> Supplemental Figure</a></li>
<li class="chapter" data-level="C.3" data-path="C-3-position-effect-sub-tab.html"><a href="C-3-position-effect-sub-tab.html"><i class="fa fa-check"></i><b>C.3</b> Supplemental Table Legends</a></li>
</ul></li>
<li class="chapter" data-level="D" data-path="D-loop-support.html"><a href="D-loop-support.html"><i class="fa fa-check"></i><b>D</b> Loop prediction: Supplemental Information</a><ul>
<li class="chapter" data-level="D.1" data-path="D-1-LoopPredSupTab.html"><a href="D-1-LoopPredSupTab.html"><i class="fa fa-check"></i><b>D.1</b> Supplementary Tables</a></li>
<li class="chapter" data-level="D.2" data-path="D-2-supplementary-figures-1.html"><a href="D-2-supplementary-figures-1.html"><i class="fa fa-check"></i><b>D.2</b> Supplementary Figures</a></li>
</ul></li>
<li class="chapter" data-level="E" data-path="E-contribution-to-individual-publications.html"><a href="E-contribution-to-individual-publications.html"><i class="fa fa-check"></i><b>E</b> Contribution to individual publications</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Build with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Genome folding in evolution and disease</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="methods-1" class="section level2">
<h2><span class="header-section-number">5.5</span> Methods</h2>
<div id="ctcf-motifs-in-the-human-genome" class="section level3">
<h3><span class="header-section-number">5.5.1</span> CTCF motifs in the human genome</h3>
<p>The recognition motif of CTCF is well defined and available from the
JASPAR database (MA0139.1) <span class="citation">(Mathelier et al. <a href="#ref-Mathelier2015">2015</a>)</span>. We downloaded TF binding site
predictions with the CTCF motif (MA0139.1) in the human genome hg19 from
the JASPAR database
(<a href="http://expdata.cmmt.ubc.ca/JASPAR/downloads/UCSC_tracks/2018/hg19/tsv/MA0139.1.tsv.gz"><em>http://expdata.cmmt.ubc.ca/JASPAR/downloads/UCSC_tracks/2018/hg19/tsv/MA0139.1.tsv.gz</em></a>).
Motif hits were filtered for p-value ≤ 2.5 x 10<sup>-6</sup>, resulting in 38,316
highly significant CTCF motif hits genome-wide and 717,137 motif pairs
within 1 Mb genomic distance that are considered as potential loop
interaction anchors in this study.</p>
</div>
<div id="loop-interaction-data-for-training-and-validation" class="section level3">
<h3><span class="header-section-number">5.5.2</span> Loop interaction data for training and validation</h3>
<p>For training and validating the prediction model we used 9,448 published
loops derived from high-resolution <em>in-situ</em> Hi-C experiments <span class="citation">(Rao et al. <a href="#ref-Rao2014">2014</a>)</span> and
206,399 CTCF and Pol2 ChIA-PET interactions <span class="citation">(Tang et al. <a href="#ref-Tang2015">2015</a>)</span> in human GM12878
cells. We considered each CTCF motif pair as positive (true looping
interaction) if there was at least one measured looping interaction for
which each loop anchor overlapped one of the CTCF motifs. Overlaps were
calculated using the R package <em>InteractionSet</em> <span class="citation">(Lun et al. <a href="#ref-Lun2016">2016</a>)</span>. This resulted in
30,025 (4.2%) of 717,137 candidate motif pairs that were labeled as true
looping interactions in GM12878. For the prediction validation in HeLa
cells we used the 3,094 Hi-C loops and 402,722 ChIA-PET interactions for
CTCF and Pol2 in HeLa from the same studies <span class="citation">(Rao et al. <a href="#ref-Rao2014">2014</a>; Tang et al. <a href="#ref-Tang2015">2015</a>)</span> and labeled 12,480
(1.7 %) of motif pairs as true loops in HeLa cells.</p>
</div>
<div id="chip-seq-datasets-in-gm12878-cells" class="section level3">
<h3><span class="header-section-number">5.5.3</span> ChIP-seq datasets in GM12878 cells</h3>
<p>We downloaded publicly available ChIP-seq data from the ENCODE data
portal <span class="citation">(Dunham et al. <a href="#ref-Dunham2012">2012</a>; Davis et al. <a href="#ref-Davis2017">2017</a>)</span> by requiring the assay to be ChIP-seq, the target to be
a transcription factor, the biosample term name to be GM12878, the
genome assembly to be hg19, and the file-type to be bigWig. Furthermore,
we filtered the data to have output type ‘fold change over control’ or
‘signal’ and to be built from two replicates. Then we selected for each
TF only one unique experiment as bigWig file with either output type
‘fold change over control’ or, if unavailable, output type ‘signal’.
This resulted in 124 ChIP-seq experiments for different TFs (<a href="D-1-LoopPredSupTab.html#LoopPredSupTab">Table S1</a>).
ChIP-seq data for HeLa were retrieved analogously and filtered for the
selected targets: RAD21, CTCF, ZNF143, STAT1, EP300, and ZNF143 (<a href="D-1-LoopPredSupTab.html#LoopPredSupTab">Table S2</a>).</p>
</div>
<div id="chip-seq-data-types" class="section level3">
<h3><span class="header-section-number">5.5.4</span> ChIP-seq data types</h3>
<p>To analyze the effect of different ChIP-seq signal types and other
genomic assays on loop prediction performance, we selected five TFs
(ZNF143, STAT1, SMC3, RAD21, and CTCF) and downloaded the mapped reads
of ChIP-seq experiments as BAM files from the ENCODE data portal <span class="citation">(Davis et al. <a href="#ref-Davis2017">2017</a>)</span>
and from the UCSC Genome Browser <span class="citation">(Hinrichs et al. <a href="#ref-Hinrichs2006">2006</a>)</span>. Furthermore, we downloaded
signal tracks as bigWig files for ChIP-seq input control experiment and
DNase-seq experiments in GM12878 cells. File accession identifiers and
download links are provided in <a href="D-1-LoopPredSupTab.html#LoopPredSupTab">Table S3</a>. We used the ChIP-seq peak
caller <em>Q</em> <span class="citation">(Hansen et al. <a href="#ref-Hansen2015">2015</a>)</span> with option ‘-w’ for each human chromosome to generate
signal tracks in BED format of shifted reads and qfrags. ‘Shifted reads’
are counts of mapped reads that are shifted in 5’ direction by half of
the estimated fragment size. ‘qfrags’ are pairs of forward and reverse
mapped reads within a given distance <span class="citation">(Hansen et al. <a href="#ref-Hansen2015">2015</a>)</span> and are shown to improve
signal to noise ratio in ChIP-seq peak calling <span class="citation">(Hansen et al. <a href="#ref-Hansen2015">2015</a>)</span>. We then combined
resulting BED files from all chromosomes and converted them to the
bedGraph and bigWig formats using the <em>bedtools</em> <span class="citation">(Quinlan and Hall <a href="#ref-Quinlan2010">2010</a>)</span> and
<em>bedGrpahtoBigWig</em> tools from the UCSC Genome Browser <span class="citation">(Kent et al. <a href="#ref-Kent2010">2010</a>)</span>.</p>
</div>
<div id="chip-nexus-data-processing-for-rad21-and-scm3" class="section level3">
<h3><span class="header-section-number">5.5.5</span> ChIP-nexus data processing for RAD21 and SCM3</h3>
<p>ChIP-nexus data for RAD21 and SMC3 in GM12878 cells were published
recently <span class="citation">(Tang et al. <a href="#ref-Tang2015">2015</a>)</span>. We downloaded the corresponding raw reads from the
Sequence Read Archive (SRA) (Run IDs SRR2312570 and SRR2312571).
Reads were processed using <em>felxcut</em> for barcode removal and adapter
trimming as recommended in the user guide of the <em>Q-nexus</em> tool
<span class="citation">(Hansen et al. <a href="#ref-Hansen2016">2016</a>)</span>. Reads were than mapped to human genome hg19 using <em>Bowtie</em>
version 2.3.2 with default settings. Duplicate reads were removed
using <em>nexcat</em> <span class="citation">(Hansen et al. <a href="#ref-Hansen2016">2016</a>)</span>. Finally, we created shifted-reads and qfraq
profiles using <em>Q-nexus</em> <span class="citation">(Hansen et al. <a href="#ref-Hansen2016">2016</a>)</span> with options ‘–nexus-mode’ and ‘-w’
for each chromosome and combined them to bigWig files as described
above.</p>
</div>
<div id="similarity-of-chip-seq-profiles-as-correlation-of-coverage-around-motifs" class="section level3">
<h3><span class="header-section-number">5.5.6</span> Similarity of ChIP-seq profiles as correlation of coverage around motifs</h3>
<p>For each CTCF sequence motif in the human genome, we quantified the
number of reads overlapping each base within +/- 500 bp around the motif
center. This results in a vector <span class="math inline">\(x_i = (x_{i,1}, x_{i,2}, ..., x_{i,n})\)</span> where <span class="math inline">\(x_{i,k}\)</span> is the coverage signal at position
<span class="math inline">\(k\)</span> around CTCF motif <span class="math inline">\(i\)</span>. Coverage vectors for motif hits reported on
the minus strand were reversed because CTCF motif sites are assumed to
be symmetrically aligned to each other when cooperating at loop anchors
(Fig. <a href="5-1-introduction-3.html#fig:LoopPred1">5.1</a>A) <span class="citation">(Rao et al. <a href="#ref-Rao2014">2014</a>; Tang et al. <a href="#ref-Tang2015">2015</a>; Guo et al. <a href="#ref-Guo2015">2015</a>; Wit et al. <a href="#ref-deWit2015">2015</a>; Sanborn et al. <a href="#ref-Sanborn2015">2015</a>)</span>.
For all considered pairs of CTCF motifs <span class="math inline">\(i\)</span> and <span class="math inline">\(j\)</span>,
we calculated the ChIP-seq profile similarity as Pearson correlation
coefficient <span class="math inline">\(r_{i,j}\)</span> of the corresponding coverage vectors <span class="math inline">\(x_i\)</span> and <span class="math inline">\(x_j\)</span>.</p>
</div>
<div id="genomic-sequence-features-of-chromatin-loops" class="section level3">
<h3><span class="header-section-number">5.5.7</span> Genomic sequence features of chromatin loops</h3>
<p>Besides the correlation of ChIP-seq profiles, we used genomic features
of motif pairs as features to predict interactions. The distance <em>d</em> is
the number of bp between the two motif centers. The categorical variable
orientation <em>o</em> is either, <em>convergent</em>, <em>forward</em>, <em>reverse</em>, or
<em>divergent</em>, depending on the orientation of CTCF motifs in the pair
(+-, ++, –, and -+, respectively). The motif hit similarity <em>s</em> is the
minimum of the two motif hit scores in each pair; we derived these motif
scores from the JASPAR motif hit tracks as <span class="math inline">\(-\log_{10}\)</span> transformed p-values
<span class="citation">(Khan et al. <a href="#ref-Khan2018">2018</a>)</span>.</p>
</div>
<div id="chromatin-loop-prediction-model" class="section level3">
<h3><span class="header-section-number">5.5.8</span> Chromatin Loop prediction model</h3>
<p>We used a logistic regression model to predict the log-likelihood
probability of CTCF motif pairs to perform chromatin looping
interactions. The probability <span class="math inline">\(p\)</span> that two sites interact is modeled as:
<span class="math display">\[
\ln (\frac{p}{1-p}) = \beta_0 + \beta_1 x_1, + ... + \beta_k x_k
\]</span>
where <span class="math inline">\(\beta\)</span> are the unknown model parameters and <span class="math inline">\(x_1, ..., x_k\)</span> the features.</p>
<p>More specifically, for the 7C model with a single ChIP-seq experiment as
input, the logistic regression model for the interaction probability <em>p</em>
is:
<span class="math display">\[
\ln (\frac{p}{1-p}) = \beta_0 + \beta_1 d, + \beta_2 o + \beta_3 s + \beta_4 r
\]</span></p>
<p>Parameters were estimated using the function ‘glm()’ with option
‘family=binomial()’ in R during model training as described below.</p>
</div>
<div id="training-and-validation-of-prediction-model" class="section level3">
<h3><span class="header-section-number">5.5.9</span> Training and validation of prediction model</h3>
<p>We used the R package <em>rsample</em> for 10-fold cross-validation. Thereby,
we randomly split the dataset of CTCF motif pairs into ten equal sized
subsets. For each round of cross-validation one subset is held out (test
dataset) and the model parameters are trained on the remaining 90% of
the samples (training dataset). The model parameters are shown for six
selected TFs and combined models in Figure S1A. For each split, the
performance of the model is than evaluated on the test dataset. For
prediction performance in HeLa cells, we trained on all motif pairs
using ChIP-seq and true loops from GM12878 cells and evaluated
performance on all motif pairs using the true loop data in HeLa.</p>
</div>
<div id="analysis-of-prediction-performance" class="section level3">
<h3><span class="header-section-number">5.5.10</span> Analysis of prediction performance</h3>
<p>We quantified prediction performance using the receiver operating
characteristic (ROC) and precision recall curves (PRC) as implemented in
the R package <em>precrec</em> <span class="citation">(Saito and Rehmsmeier <a href="#ref-Saito2017">2016</a>)</span>.</p>
<p>Given the number of true positives (TP), true negatives (TN), false
positives (FP), and false negatives (FN), the sensitivity is defined as
TP/(TP+FN), specificity as TN/(TN+FP), precision as TP/(TP+FP), and
recall as TP/(TP+FN). For each cross-validation split, the area under
the curve is computed separately, and the mean across splits together
with the standard deviation reported. To get binary prediction outputs,
we computed the f1-score as harmonic mean of precision and recall for
all prediction scores on all cross-validation folds using the R package
<em>ROCR</em> <span class="citation">(Sing et al. <a href="#ref-Sing2005">2005</a>)</span>. Then we computed the prediction score that maximizes the
f1-score as default cutoff for binary prediction output (Fig. <a href="D-2-supplementary-figures-1.html#fig:LoopPredS1">C.2</a>B).</p>
</div>
<div id="implementation-of-7c-and-compatibility-to-other-tools" class="section level3">
<h3><span class="header-section-number">5.5.11</span> Implementation of 7C and compatibility to other tools</h3>
<p>We implemented 7C as R package, termed <em>sevenC</em>, by using existing
infrastructure for chromatin interaction data from the <em>interactionSet</em>
package <span class="citation">(Lun et al. <a href="#ref-Lun2016">2016</a>)</span> and functionality for reading bigWig files from the
<em>rtracklayer</em> package <span class="citation">(Lawrence et al. <a href="#ref-Lawrence2009">2009</a>)</span> from the Bioconductor project <span class="citation">(Huber et al. <a href="#ref-Huber2015">2015</a>)</span>.
Predicted loops can be written as interaction tracks for visualization
in the WashU Epigenome Browser <span class="citation">(Zhou et al. <a href="#ref-Zhou2013">2013</a>)</span> or as BEDPE format using the
<em>GenomicInteractions</em> package <span class="citation">(Harmston et al. <a href="#ref-Harmston2015">2015</a>)</span> for visualization in the Juicebox
tool <span class="citation">(Durand et al. <a href="#ref-Durand2016">2016</a>)</span>. The package is freely available and easy to install from
GitHub under
<a href="https://ibn-salem.github.io/sevenC/"><em>https://ibn-salem.github.io/sevenC/</em></a>
and has been submitted to the Bioconductor project <span class="citation">(Huber et al. <a href="#ref-Huber2015">2015</a>)</span>. All analysis
presented in this work were implemented in R and all scripts used have
been made available in a separate GitHub repository:
<a href="https://github.com/ibn-salem/chromloop_analysis"><em>https://github.com/ibn-salem/sevenC_analysis</em></a>.</p>
</div>
</div>
<h3>References</h3>
<div id="refs" class="references">
<div id="ref-Mathelier2015">
<p>Mathelier A, Fornes O, Arenillas DJ, Chen C-y, Denay G, Lee J, Shi W, Shyr C, Tan G, Worsley-Hunt R, et al. 2015. JASPAR 2016: A major expansion and update of the open-access database of transcription factor binding profiles. <em>Nucleic acids research</em> <strong>44</strong>: gkv1176.</p>
</div>
<div id="ref-Rao2014">
<p>Rao SSP, Huntley MH, Durand NC, Stamenova EK, Bochkov ID, Robinson JT, Sanborn AL, Machol I, Omer AD, Lander ES, et al. 2014. A 3D map of the human genome at kilobase resolution reveals principles of chromatin looping. <em>Cell</em> <strong>159</strong>: 1665–80.</p>
</div>
<div id="ref-Tang2015">
<p>Tang Z, Luo OJ, Li X, Zheng M, Zhu JJ, Szalaj P, Trzaskoma P, Magalska A, Wlodarczyk J, Ruszczycki B, et al. 2015. CTCF-mediated human 3D genome architecture reveals chromatin topology for transcription. <em>Cell</em> 1–17.</p>
</div>
<div id="ref-Lun2016">
<p>Lun ATL, Perry M, Ing-Simmons E. 2016. Infrastructure for genomic interactions : Bioconductor classes for hi-c , chia-pet and related experiments [version 1; referees: 2 approved]. <em>F1000Research</em> <strong>5</strong>: 1–6.</p>
</div>
<div id="ref-Dunham2012">
<p>Dunham I, Kundaje A, Aldred SF, Collins PJ, Davis C a, Doyle F, Epstein CB, Frietze S, Harrow J, Kaul R, et al. 2012. An integrated encyclopedia of dna elements in the human genome. <em>Nature</em> <strong>489</strong>: 57–74.</p>
</div>
<div id="ref-Davis2017">
<p>Davis AP, Grondin CJ, Johnson RJ, Sciaky D, King BL, McMorran R, Wiegers J, Wiegers TC, Mattingly CJ. 2017. The comparative toxicogenomics database: Update 2017. <em>Nucleic Acids Research</em> <strong>45</strong>: D972–D978.</p>
</div>
<div id="ref-Hinrichs2006">
<p>Hinrichs AS, Karolchik D, Baertsch R, Barber GP, Bejerano G, Clawson H, Diekhans M, Furey TS, Harte RA, Hsu F, et al. 2006. The ucsc genome browser database: Update 2006. <em>Nucleic Acids Res</em> <strong>34</strong>: D590—–D598.</p>
</div>
<div id="ref-Hansen2015">
<p>Hansen P, Hecht J, Ibrahim DM, Krannich A, Truss M, Robinson PN. 2015. Saturation analysis of chip-seq data for reproducible identification of binding peaks. <em>Genome research</em> <strong>25</strong>: 1391–400.</p>
</div>
<div id="ref-Quinlan2010">
<p>Quinlan AR, Hall IM. 2010. BEDTools: A flexible suite of utilities for comparing genomic features. <em>Bioinformatics</em> <strong>26</strong>: 841–842.</p>
</div>
<div id="ref-Kent2010">
<p>Kent WJ, Zweig AS, Barber G, Hinrichs AS, Karolchik D. 2010. BigWig and bigbed: Enabling browsing of large distributed datasets. <em>Bioinformatics</em> <strong>26</strong>: 2204–2207.</p>
</div>
<div id="ref-Hansen2016">
<p>Hansen P, Hecht J, Ibn-Salem J, Menkuec BS, Roskosch S, Truss M, Robinson PN. 2016. Q-nexus: A comprehensive and efficient analysis pipeline designed for chip-nexus. <em>BMC Genomics</em> <strong>17</strong>: 873.</p>
</div>
<div id="ref-Guo2015">
<p>Guo Y, Xu Q, Canzio D, Shou J, Li J, Gorkin DU, Jung I, Wu H, Zhai Y, Tang Y, et al. 2015. CRISPR inversion of ctcf sites alters genome topology and enhancer/promoter function. <em>Cell</em> <strong>162</strong>: 900–910.</p>
</div>
<div id="ref-deWit2015">
<p>Wit E de, Vos ESM, Holwerda SJB, Valdes-Quezada C, Verstegen MJAM, Teunissen H, Splinter E, Wijchers PJ, Krijger PHL, Laat W de. 2015. CTCF binding polarity determines chromatin looping. <em>Molecular Cell</em> <strong>60</strong>: 676–684.</p>
</div>
<div id="ref-Sanborn2015">
<p>Sanborn AL, Rao SSP, Huang S-C, Durand NC, Huntley MH, Jewett AI, Bochkov ID, Chinnappan D, Cutkosky A, Li J, et al. 2015. Chromatin extrusion explains key features of loop and domain formation in wild-type and engineered genomes. <em>Proceedings of the National Academy of Sciences</em> <strong>112</strong>: 201518552.</p>
</div>
<div id="ref-Khan2018">
<p>Khan A, Fornes O, Stigliani A. 2018. JASPAR 2018: Update of the open-access database of transcription factor binding profiles and its web framework. <em>Nucleic Acids Research</em> <strong>(In print)</strong>: 1–7.</p>
</div>
<div id="ref-Saito2017">
<p>Saito T, Rehmsmeier M. 2016. Precrec: Fast and accurate precision-recall and roc curve calculations in r. <em>Bioinformatics</em> <strong>33</strong>: btw570.</p>
</div>
<div id="ref-Sing2005">
<p>Sing T, Sander O, Beerenwinkel N, Lengauer T. 2005. ROCR: Visualizing classifier performance in r. <em>Bioinformatics</em> <strong>21</strong>: 3940–3941.</p>
</div>
<div id="ref-Lawrence2009">
<p>Lawrence M, Gentleman R, Carey V. 2009. Rtracklayer: An r package for interfacing with genome browsers. <em>Bioinformatics</em> <strong>25</strong>: 1841–1842.</p>
</div>
<div id="ref-Huber2015">
<p>Huber W, Carey VJ, Gentleman R, Anders S, Carlson M, Carvalho BS, Bravo HC, Davis S, Gatto L, Girke T, et al. 2015. Orchestrating high-throughput genomic analysis with bioconductor. <em>Nature Methods</em> <strong>12</strong>: 115–121.</p>
</div>
<div id="ref-Zhou2013">
<p>Zhou X, Lowdon RF, Li D, Lawson H a, Madden P a F, Costello JF, Wang T. 2013. Exploring long-range genome interactions using the washu epigenome browser. <em>Nature methods</em> <strong>10</strong>: 375–6.</p>
</div>
<div id="ref-Harmston2015">
<p>Harmston N, Ing-Simmons E, Perry M, Barešić A, Lenhard B. 2015. GenomicInteractions: An r/bioconductor package for manipulating and investigating chromatin interaction data. <em>BMC Genomics</em> <strong>16</strong>: 963.</p>
</div>
<div id="ref-Durand2016">
<p>Durand NC, Shamim MS, Machol I, Rao SS, Huntley MH, Lander ES, Aiden EL. 2016. Juicer provides a one-click system for analyzing loop-resolution hi-c experiments. <em>Cell Systems</em> <strong>3</strong>: 95–98.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="5-4-conclusion-2.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="5-6-declarations-1.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": true,
"facebook": false,
"twitter": true,
"google": false,
"weibo": false,
"instapper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/ibn-salem/thesis/edit/master/05-loop_prediction.Rmd",
"text": "Edit"
},
"download": ["thesis.pdf", "thesis.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://cdn.bootcss.com/mathjax/2.7.1/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:" && /^https?:/.test(src))
      src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
