<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>Genome folding in evolution and disease</title>
  <meta name="description" content="This is a draft PhD thesis.">
  <meta name="generator" content="bookdown 0.6.1 and GitBook 2.6.7">

  <meta property="og:title" content="Genome folding in evolution and disease" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="This is a draft PhD thesis." />
  <meta name="github-repo" content="ibn-salem/thesis" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Genome folding in evolution and disease" />
  
  <meta name="twitter:description" content="This is a draft PhD thesis." />
  

<meta name="author" content="Jonas Ibn-Salem">


<meta name="date" content="2018-01-28">

  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  
<link rel="prev" href="B-2-supplementary-figures.html">

<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />










<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Thesis</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="1" data-path="1-thesis-intro.html"><a href="1-thesis-intro.html"><i class="fa fa-check"></i><b>1</b> Introduction</a><ul>
<li class="chapter" data-level="1.1" data-path="1-1-structure-of-the-intorduction.html"><a href="1-1-structure-of-the-intorduction.html"><i class="fa fa-check"></i><b>1.1</b> Structure of the Intorduction</a></li>
<li class="chapter" data-level="1.2" data-path="1-2-cell-diversity-and-regulation-of-gene-expression.html"><a href="1-2-cell-diversity-and-regulation-of-gene-expression.html"><i class="fa fa-check"></i><b>1.2</b> Cell diversity and regulation of gene expression</a></li>
<li class="chapter" data-level="1.3" data-path="1-3-enhancers.html"><a href="1-3-enhancers.html"><i class="fa fa-check"></i><b>1.3</b> Enhancers</a></li>
<li class="chapter" data-level="1.4" data-path="1-4-methods-to-probe-the-3d-chromatin-archtiecture.html"><a href="1-4-methods-to-probe-the-3d-chromatin-archtiecture.html"><i class="fa fa-check"></i><b>1.4</b> Methods to probe the 3D chromatin archtiecture</a><ul>
<li class="chapter" data-level="1.4.1" data-path="1-4-methods-to-probe-the-3d-chromatin-archtiecture.html"><a href="1-4-methods-to-probe-the-3d-chromatin-archtiecture.html#microscopy-based-techniques-to-visuallize-the-genome-in-3d"><i class="fa fa-check"></i><b>1.4.1</b> Microscopy-based techniques to visuallize the genome in 3D</a></li>
<li class="chapter" data-level="1.4.2" data-path="1-4-methods-to-probe-the-3d-chromatin-archtiecture.html"><a href="1-4-methods-to-probe-the-3d-chromatin-archtiecture.html#proximity-ligation-based-method-to-quantify-chromatin-interactions"><i class="fa fa-check"></i><b>1.4.2</b> Proximity-ligation based method to quantify chromatin interactions</a></li>
</ul></li>
<li class="chapter" data-level="1.5" data-path="1-5-hierarchy-of-chromatin-3d-structure.html"><a href="1-5-hierarchy-of-chromatin-3d-structure.html"><i class="fa fa-check"></i><b>1.5</b> Hierarchy of chromatin 3D structure</a><ul>
<li class="chapter" data-level="1.5.1" data-path="1-5-hierarchy-of-chromatin-3d-structure.html"><a href="1-5-hierarchy-of-chromatin-3d-structure.html#chromosomal-territories-and-inter-chromosomal-contacts"><i class="fa fa-check"></i><b>1.5.1</b> Chromosomal territories and inter-chromosomal contacts</a></li>
<li class="chapter" data-level="1.5.2" data-path="1-5-hierarchy-of-chromatin-3d-structure.html"><a href="1-5-hierarchy-of-chromatin-3d-structure.html#ab-compartments"><i class="fa fa-check"></i><b>1.5.2</b> A/B compartments</a></li>
<li class="chapter" data-level="1.5.3" data-path="1-5-hierarchy-of-chromatin-3d-structure.html"><a href="1-5-hierarchy-of-chromatin-3d-structure.html#topologically-associating-domains-tads"><i class="fa fa-check"></i><b>1.5.3</b> Topologically associating domains (TADs)</a></li>
<li class="chapter" data-level="1.5.4" data-path="1-5-hierarchy-of-chromatin-3d-structure.html"><a href="1-5-hierarchy-of-chromatin-3d-structure.html#chromatin-loops"><i class="fa fa-check"></i><b>1.5.4</b> Chromatin loops</a></li>
</ul></li>
<li class="chapter" data-level="1.6" data-path="1-6-dynamics-of-chromatin-structure.html"><a href="1-6-dynamics-of-chromatin-structure.html"><i class="fa fa-check"></i><b>1.6</b> Dynamics of chromatin structure</a></li>
<li class="chapter" data-level="1.7" data-path="1-7-changes-of-genome-folding-in-evoltuion-and-disease-genomes.html"><a href="1-7-changes-of-genome-folding-in-evoltuion-and-disease-genomes.html"><i class="fa fa-check"></i><b>1.7</b> Changes of genome folding in evoltuion and disease genomes</a></li>
<li class="chapter" data-level="1.8" data-path="1-8-aims-of-this-thesis.html"><a href="1-8-aims-of-this-thesis.html"><i class="fa fa-check"></i><b>1.8</b> Aims of this thesis</a><ul>
<li class="chapter" data-level="" data-path="1-8-aims-of-this-thesis.html"><a href="1-8-aims-of-this-thesis.html#is-the-three-dimensional-folding-of-genomes-associated-with-co-regulation-of-functionally-related-genes"><i class="fa fa-check"></i>Is the three-dimensional folding of genomes associated with co-regulation of functionally related genes? </a></li>
<li class="chapter" data-level="" data-path="1-8-aims-of-this-thesis.html"><a href="1-8-aims-of-this-thesis.html#are-tads-functional-building-blocks-of-genomes-and-subjected-to-selective-pressure-during-evolution"><i class="fa fa-check"></i>Are TADs functional building blocks of genomes and subjected to selective pressure during evolution?</a></li>
<li class="chapter" data-level="" data-path="1-8-aims-of-this-thesis.html"><a href="1-8-aims-of-this-thesis.html#is-the-disruption-of-tads-by-rearrangements-also-associated-to-genetic-diseases"><i class="fa fa-check"></i>Is the disruption of TADs by rearrangements also associated to genetic diseases?</a></li>
<li class="chapter" data-level="" data-path="1-8-aims-of-this-thesis.html"><a href="1-8-aims-of-this-thesis.html#can-chromatin-looping-interactions-be-predicted-by-chip-seq-and-sequence-features"><i class="fa fa-check"></i>Can chromatin looping interactions be predicted by ChIP-seq and sequence features?</a></li>
</ul></li>
<li class="chapter" data-level="1.9" data-path="1-9-structure-of-this-thesis.html"><a href="1-9-structure-of-this-thesis.html"><i class="fa fa-check"></i><b>1.9</b> Structure of this thesis</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="2-paralog-regulation.html"><a href="2-paralog-regulation.html"><i class="fa fa-check"></i><b>2</b> Paralog genes in the 3D genome architecture</a><ul>
<li class="chapter" data-level="" data-path="preamble.html"><a href="preamble.html"><i class="fa fa-check"></i>Preamble</a></li>
<li class="chapter" data-level="" data-path="abstract-1.html"><a href="abstract-1.html"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="2.1" data-path="2-1-introduction.html"><a href="2-1-introduction.html"><i class="fa fa-check"></i><b>2.1</b> Introduction</a></li>
<li class="chapter" data-level="2.2" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html"><i class="fa fa-check"></i><b>2.2</b> MATERIALS AND METHODS</a><ul>
<li class="chapter" data-level="2.2.1" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html#selection-of-pairs-of-paralog-genes"><i class="fa fa-check"></i><b>2.2.1</b> Selection of pairs of paralog genes</a></li>
<li class="chapter" data-level="2.2.2" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html#enhancers-to-gene-association"><i class="fa fa-check"></i><b>2.2.2</b> Enhancers to gene association</a></li>
<li class="chapter" data-level="2.2.3" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html#topological-associating-domains"><i class="fa fa-check"></i><b>2.2.3</b> Topological associating domains</a></li>
<li class="chapter" data-level="2.2.4" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html#hi-c-interaction-maps"><i class="fa fa-check"></i><b>2.2.4</b> Hi-C interaction maps</a></li>
<li class="chapter" data-level="2.2.5" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html#randomization"><i class="fa fa-check"></i><b>2.2.5</b> Randomization</a></li>
<li class="chapter" data-level="2.2.6" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html#statistical-tests"><i class="fa fa-check"></i><b>2.2.6</b> Statistical tests</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="2-3-results.html"><a href="2-3-results.html"><i class="fa fa-check"></i><b>2.3</b> RESULTS</a><ul>
<li class="chapter" data-level="2.3.1" data-path="2-3-results.html"><a href="2-3-results.html#distribution-of-paralog-genes-in-the-human-genome"><i class="fa fa-check"></i><b>2.3.1</b> Distribution of paralog genes in the human genome</a></li>
<li class="chapter" data-level="2.3.2" data-path="2-3-results.html"><a href="2-3-results.html#co-expression-of-paralog-gene-pairs-across-tissues"><i class="fa fa-check"></i><b>2.3.2</b> Co-expression of paralog gene pairs across tissues</a></li>
<li class="chapter" data-level="2.3.3" data-path="2-3-results.html"><a href="2-3-results.html#paralog-genes-share-enhancers"><i class="fa fa-check"></i><b>2.3.3</b> Paralog genes share enhancers</a></li>
<li class="chapter" data-level="2.3.4" data-path="2-3-results.html"><a href="2-3-results.html#co-localization-of-paralogs-in-tads"><i class="fa fa-check"></i><b>2.3.4</b> Co-localization of paralogs in TADs</a></li>
<li class="chapter" data-level="2.3.5" data-path="2-3-results.html"><a href="2-3-results.html#distal-paralog-pairs-are-enriched-for-long-range-chromatin-contacts"><i class="fa fa-check"></i><b>2.3.5</b> Distal paralog pairs are enriched for long-range chromatin contacts</a></li>
<li class="chapter" data-level="2.3.6" data-path="2-3-results.html"><a href="2-3-results.html#close-paralogs-have-fewer-contacts-than-expected"><i class="fa fa-check"></i><b>2.3.6</b> Close paralogs have fewer contacts than expected</a></li>
<li class="chapter" data-level="2.3.7" data-path="2-3-results.html"><a href="2-3-results.html#paralogs-in-mouse-and-dog-genome"><i class="fa fa-check"></i><b>2.3.7</b> Paralogs in mouse and dog genome</a></li>
<li class="chapter" data-level="2.3.8" data-path="2-3-results.html"><a href="2-3-results.html#orthologs-of-human-paralogs-show-conserved-co-localization"><i class="fa fa-check"></i><b>2.3.8</b> Orthologs of human paralogs show conserved co-localization</a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="2-4-discussion.html"><a href="2-4-discussion.html"><i class="fa fa-check"></i><b>2.4</b> DISCUSSION</a></li>
<li class="chapter" data-level="2.5" data-path="2-5-conclusion.html"><a href="2-5-conclusion.html"><i class="fa fa-check"></i><b>2.5</b> CONCLUSION</a></li>
<li class="chapter" data-level="2.6" data-path="2-6-acknowledgements.html"><a href="2-6-acknowledgements.html"><i class="fa fa-check"></i><b>2.6</b> ACKNOWLEDGEMENTS</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="3-TAD-evolution.html"><a href="3-TAD-evolution.html"><i class="fa fa-check"></i><b>3</b> Stability of TADs in evolution</a><ul>
<li class="chapter" data-level="" data-path="3-TAD-evolution.html"><a href="3-TAD-evolution.html#abstract-2"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="" data-path="3-TAD-evolution.html"><a href="3-TAD-evolution.html#keywords"><i class="fa fa-check"></i>Keywords</a></li>
<li class="chapter" data-level="3.1" data-path="3-1-introduction-1.html"><a href="3-1-introduction-1.html"><i class="fa fa-check"></i><b>3.1</b> Introduction</a></li>
<li class="chapter" data-level="3.2" data-path="3-2-results-1.html"><a href="3-2-results-1.html"><i class="fa fa-check"></i><b>3.2</b> Results</a><ul>
<li class="chapter" data-level="3.2.1" data-path="3-2-results-1.html"><a href="3-2-results-1.html#identification-of-evolutionary-rearrangement-breakpoints-from-whole-genome-alignments"><i class="fa fa-check"></i><b>3.2.1</b> Identification of evolutionary rearrangement breakpoints from whole-genome alignments</a></li>
<li class="chapter" data-level="3.2.2" data-path="3-2-results-1.html"><a href="3-2-results-1.html#rearrangement-breakpoints-are-enriched-at-tad-boundaries"><i class="fa fa-check"></i><b>3.2.2</b> Rearrangement breakpoints are enriched at TAD boundaries</a></li>
<li class="chapter" data-level="3.2.3" data-path="3-2-results-1.html"><a href="3-2-results-1.html#clusters-of-conserved-non-coding-elements-are-depleted-for-rearrangement-breakpoints"><i class="fa fa-check"></i><b>3.2.3</b> Clusters of conserved non-coding elements are depleted for rearrangement breakpoints</a></li>
<li class="chapter" data-level="3.2.4" data-path="3-2-results-1.html"><a href="3-2-results-1.html#rearranged-tads-are-associated-with-divergent-gene-expression-between-species"><i class="fa fa-check"></i><b>3.2.4</b> Rearranged TADs are associated with divergent gene expression between species</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="3-3-discussion-1.html"><a href="3-3-discussion-1.html"><i class="fa fa-check"></i><b>3.3</b> Discussion</a></li>
<li class="chapter" data-level="3.4" data-path="3-4-conclusion-1.html"><a href="3-4-conclusion-1.html"><i class="fa fa-check"></i><b>3.4</b> Conclusion</a></li>
<li class="chapter" data-level="3.5" data-path="3-5-methods.html"><a href="3-5-methods.html"><i class="fa fa-check"></i><b>3.5</b> Methods</a><ul>
<li class="chapter" data-level="3.5.1" data-path="3-5-methods.html"><a href="3-5-methods.html#rearrangement-breakpoints-from-whole-genome-alignments"><i class="fa fa-check"></i><b>3.5.1</b> Rearrangement breakpoints from whole-genome alignments</a></li>
<li class="chapter" data-level="3.5.2" data-path="3-5-methods.html"><a href="3-5-methods.html#topologically-associating-domains-and-contact-domains"><i class="fa fa-check"></i><b>3.5.2</b> Topologically associating domains and contact domains</a></li>
<li class="chapter" data-level="3.5.3" data-path="3-5-methods.html"><a href="3-5-methods.html#breakpoint-distributions-at-tads"><i class="fa fa-check"></i><b>3.5.3</b> Breakpoint distributions at TADs</a></li>
<li class="chapter" data-level="3.5.4" data-path="3-5-methods.html"><a href="3-5-methods.html#quantification-of-breakpoint-enrichment"><i class="fa fa-check"></i><b>3.5.4</b> Quantification of breakpoint enrichment</a></li>
<li class="chapter" data-level="3.5.5" data-path="3-5-methods.html"><a href="3-5-methods.html#expression-data-for-mouse-and-human-orthologs"><i class="fa fa-check"></i><b>3.5.5</b> Expression data for mouse and human orthologs</a></li>
<li class="chapter" data-level="3.5.6" data-path="3-5-methods.html"><a href="3-5-methods.html#classification-of-tads-and-genes-according-to-rearrangements-and-grbs"><i class="fa fa-check"></i><b>3.5.6</b> Classification of TADs and genes according to rearrangements and GRBs</a></li>
<li class="chapter" data-level="3.5.7" data-path="3-5-methods.html"><a href="3-5-methods.html#source-code-and-implementation-details"><i class="fa fa-check"></i><b>3.5.7</b> Source code and implementation details</a></li>
</ul></li>
<li class="chapter" data-level="3.6" data-path="3-6-declarations.html"><a href="3-6-declarations.html"><i class="fa fa-check"></i><b>3.6</b> Declarations</a><ul>
<li class="chapter" data-level="3.6.1" data-path="3-6-declarations.html"><a href="3-6-declarations.html#ethics-approval-and-consent-to-participate"><i class="fa fa-check"></i><b>3.6.1</b> Ethics approval and consent to participate</a></li>
<li class="chapter" data-level="3.6.2" data-path="3-6-declarations.html"><a href="3-6-declarations.html#consent-for-publication"><i class="fa fa-check"></i><b>3.6.2</b> Consent for publication</a></li>
<li class="chapter" data-level="3.6.3" data-path="3-6-declarations.html"><a href="3-6-declarations.html#availability-of-data-and-material"><i class="fa fa-check"></i><b>3.6.3</b> Availability of data and material</a></li>
<li class="chapter" data-level="3.6.4" data-path="3-6-declarations.html"><a href="3-6-declarations.html#competing-interests"><i class="fa fa-check"></i><b>3.6.4</b> Competing interests</a></li>
<li class="chapter" data-level="3.6.5" data-path="3-6-declarations.html"><a href="3-6-declarations.html#funding"><i class="fa fa-check"></i><b>3.6.5</b> Funding</a></li>
<li class="chapter" data-level="3.6.6" data-path="3-6-declarations.html"><a href="3-6-declarations.html#authors-contributions"><i class="fa fa-check"></i><b>3.6.6</b> Authors‘ contributions</a></li>
<li class="chapter" data-level="3.6.7" data-path="3-6-declarations.html"><a href="3-6-declarations.html#acknowledgments"><i class="fa fa-check"></i><b>3.6.7</b> Acknowledgments</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="4-position-effect.html"><a href="4-position-effect.html"><i class="fa fa-check"></i><b>4</b> Position effects of rearrangements in disease genomes</a></li>
<li class="chapter" data-level="5" data-path="5-loop-prediction.html"><a href="5-loop-prediction.html"><i class="fa fa-check"></i><b>5</b> Prediction of chromatin looping interactions</a></li>
<li class="chapter" data-level="6" data-path="6-thesis-discussion.html"><a href="6-thesis-discussion.html"><i class="fa fa-check"></i><b>6</b> Discussion</a><ul>
<li class="chapter" data-level="6.1" data-path="6-1-discussion-on-tad-evolution.html"><a href="6-1-discussion-on-tad-evolution.html"><i class="fa fa-check"></i><b>6.1</b> Discussion on TAD evolution</a></li>
<li class="chapter" data-level="6.2" data-path="6-2-further-directions.html"><a href="6-2-further-directions.html"><i class="fa fa-check"></i><b>6.2</b> Further directions</a></li>
<li class="chapter" data-level="6.3" data-path="6-3-conclusions.html"><a href="6-3-conclusions.html"><i class="fa fa-check"></i><b>6.3</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="appendix"><span><b>Appendix</b></span></li>
<li class="chapter" data-level="A" data-path="A-supporting-information-co-regulation-of-paralog-genes-in-the-three-dimensional-chromatin-architecture.html"><a href="A-supporting-information-co-regulation-of-paralog-genes-in-the-three-dimensional-chromatin-architecture.html"><i class="fa fa-check"></i><b>A</b> Supporting Information: Co-regulation of paralog genes in the three-dimensional chromatin architecture</a></li>
<li class="chapter" data-level="B" data-path="B-supplementary-data-evolutionary-stability-of-topologically-associating-domains-is-associated-with-conserved-gene-regulation.html"><a href="B-supplementary-data-evolutionary-stability-of-topologically-associating-domains-is-associated-with-conserved-gene-regulation.html"><i class="fa fa-check"></i><b>B</b> Supplementary Data: Evolutionary stability of topologically associating domains is associated with conserved gene regulation</a><ul>
<li class="chapter" data-level="B.1" data-path="B-1-TadEvoSupTab.html"><a href="B-1-TadEvoSupTab.html"><i class="fa fa-check"></i><b>B.1</b> Supplementary Tables</a></li>
<li class="chapter" data-level="B.2" data-path="B-2-supplementary-figures.html"><a href="B-2-supplementary-figures.html"><i class="fa fa-check"></i><b>B.2</b> Supplementary Figures</a></li>
<li class="chapter" data-level="B.3" data-path="B-3-contribution-to-individual-publications.html"><a href="B-3-contribution-to-individual-publications.html"><i class="fa fa-check"></i><b>B.3</b> Contribution to individual publications</a></li>
</ul></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Build with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Genome folding in evolution and disease</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="contribution-to-individual-publications" class="section level2">
<h2><span class="header-section-number">B.3</span> Contribution to individual publications</h2>
<table>
<caption><span id="tab:contribution">Table A.1: </span>Contribution to individual publications. Contributions in percent from all authors for each contribution role and publication. Author contribution definitions according to the CRediT Taxonomy (<a href="http://journals.plos.org/plosone/s/authorship" class="uri">http://journals.plos.org/plosone/s/authorship</a>).</caption>
<thead>
<tr class="header">
<th align="left">Contributor Role</th>
<th align="left">Role Definition</th>
<th align="right">Paralog regulation (Chapter <a href="2-paralog-regulation.html#paralog-regulation"><strong>??</strong></a>)</th>
<th align="right">TAD evolution (Chapter <a href="3-TAD-evolution.html#TAD-evolution">3</a>)</th>
<th align="right">Position effect (Chapter <a href="4-position-effect.html#position-effect">4</a>)</th>
<th align="right">Loop prediction (Chapter <a href="5-loop-prediction.html#loop-prediction">5</a>)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">Conceptualization</td>
<td align="left">Ideas; formulation or evolution of overarching research goals and aims.</td>
<td align="right">80</td>
<td align="right">90</td>
<td align="right">20</td>
<td align="right">90</td>
</tr>
<tr class="even">
<td align="left">Data Curation</td>
<td align="left">Management activities to annotate (produce metadata), scrub data and maintain research data (including software code, where it is necessary for interpreting the data itself) for initial use and later reuse.</td>
<td align="right">100</td>
<td align="right">80</td>
<td align="right">50</td>
<td align="right">100</td>
</tr>
<tr class="odd">
<td align="left">Formal Analysis</td>
<td align="left">Application of statistical, mathematical, computational, or other formal techniques to analyze or synthesize study data.</td>
<td align="right">100</td>
<td align="right">50</td>
<td align="right">40</td>
<td align="right">100</td>
</tr>
<tr class="even">
<td align="left">Funding Acquisition</td>
<td align="left">Acquisition of the financial support for the project leading to this publication.</td>
<td align="right">0</td>
<td align="right">0</td>
<td align="right">0</td>
<td align="right">0</td>
</tr>
<tr class="odd">
<td align="left">Investigation</td>
<td align="left">Conducting a research and investigation process, specifically performing the experiments, or data/evidence collection.</td>
<td align="right">100</td>
<td align="right">80</td>
<td align="right">20</td>
<td align="right">100</td>
</tr>
<tr class="even">
<td align="left">Methodology</td>
<td align="left">Development or design of methodology; creation of models</td>
<td align="right">90</td>
<td align="right">80</td>
<td align="right">30</td>
<td align="right">100</td>
</tr>
<tr class="odd">
<td align="left">Project Administration</td>
<td align="left">Management and coordination responsibility for the research activity planning and execution.</td>
<td align="right">50</td>
<td align="right">80</td>
<td align="right">0</td>
<td align="right">80</td>
</tr>
<tr class="even">
<td align="left">Resources</td>
<td align="left">Provision of study materials, reagents, materials, patients, laboratory samples, animals, instrumentation, computing resources, or other analysis tools.</td>
<td align="right">0</td>
<td align="right">20</td>
<td align="right">0</td>
<td align="right">0</td>
</tr>
<tr class="odd">
<td align="left">Software</td>
<td align="left">Programming, software development; designing computer programs; implementation of the computer code and supporting algorithms; testing of existing code components.</td>
<td align="right">100</td>
<td align="right">50</td>
<td align="right">50</td>
<td align="right">100</td>
</tr>
<tr class="even">
<td align="left">Supervision</td>
<td align="left">Oversight and leadership responsibility for the research activity planning and execution, including mentorship external to the core team.</td>
<td align="right">0</td>
<td align="right">80</td>
<td align="right">0</td>
<td align="right">0</td>
</tr>
<tr class="odd">
<td align="left">Validation</td>
<td align="left">Verification, whether as a part of the activity or separate, of the overall replication/reproducibility of results/experiments and other research outputs.</td>
<td align="right">100</td>
<td align="right">80</td>
<td align="right">20</td>
<td align="right">100</td>
</tr>
<tr class="even">
<td align="left">Visualization</td>
<td align="left">Preparation, creation and/or presentation of the published work, specifically visualization/data presentation.</td>
<td align="right">100</td>
<td align="right">50</td>
<td align="right">0</td>
<td align="right">100</td>
</tr>
<tr class="odd">
<td align="left">Writing – Original Draft Preparation</td>
<td align="left">Creation and/or presentation of the published work, specifically writing the initial draft (including substantive translation).</td>
<td align="right">100</td>
<td align="right">60</td>
<td align="right">20</td>
<td align="right">100</td>
</tr>
<tr class="even">
<td align="left">Writing – Review &amp; Editing</td>
<td align="left">Preparation, creation and/or presentation of the published work by those from the original research group, specifically critical review, commentary or revision – including pre- or post-publication stages.</td>
<td align="right">50</td>
<td align="right">80</td>
<td align="right">20</td>
<td align="right">50</td>
</tr>
</tbody>
</table>

<div id="refs" class="references">
<div>
<p>Acemel, R.D., Maeso, I., and Gómez‐Skarmeta, J.L. (2017). Topologically associated domains: a successful scaffold for the evolution of gene regulation in animals. Wiley Interdisciplinary Reviews: Developmental Biology 1–19.</p>
</div>
<div>
<p>Andersson, R., Gebhard, C., Miguel-Escalada, I., Hoof, I., Bornholdt, J., Boyd, M., Chen, Y., Zhao, X., Schmidl, C., Suzuki, T., et al. (2014). An atlas of active enhancers across human cell types and tissues. Nature <em>507</em>, 455–461.</p>
</div>
<div>
<p>Andrey, G., and Mundlos, S. (2017). The three-dimensional genome: regulating gene expression during pluripotency and development. 3646–3658.</p>
</div>
<div>
<p>Ardlie, K.G., DeLuca, D.S., Segrè, A.V., Sullivan, T.J., Young, T.R., Gelfand, E.T., Trowbridge, C.A., Maller, J.B., Tukiainen, T., Lek, M., et al. (2015). The Genotype-Tissue Expression (GTEx) pilot analysis: Multitissue gene regulation in humans. Science <em>348</em>, 648–660.</p>
</div>
<div>
<p>Ay, F., and Noble, W.S. (2015). Analysis methods for studying the 3D architecture of the genome. Genome Biology <em>16</em>, 183.</p>
</div>
<div>
<p>Banerji, J., Rusconi, S., and Schaffner, W. (1981). Expression of a <span class="math inline">\(\beta\)</span>-globin gene is enhanced by remote SV40 DNA sequences. Cell <em>27</em>, 299–308.</p>
</div>
<div>
<p>Becker, M., Mah, N., Zdzieblo, D., Li, X., Mer, A., Andrade-navarro, M.A., and Mu, A.M. (2015). Epigenetic Mechanisms in Cellular Reprogramming. In Epigenetics and Human Health, A. Meissner, and J. Walter, eds. (Berlin, Heidelberg: Springer Berlin Heidelberg), pp. pp 141–166.</p>
</div>
<div>
<p>Bonev, B., and Cavalli, G. (2016). Organization and function of the 3D genome. Nature Reviews Genetics <em>17</em>, 661–678.</p>
</div>
<div>
<p>Boutanaev, a M., Kalmykova, a I., Shevelyov, Y.Y., Nurminsky, D.I., Smith, M., Evolution, T., Masterpiece, T., Helm, C., and Selection, N. (2002). Large clusters of co-expressed genes in the Drosophila genome. Nature <em>420</em>, 666–669.</p>
</div>
<div>
<p>Branco, M.R., and Pombo, A. (2006). Intermingling of chromosome territories in interphase suggests role in translocations and transcription-dependent associations. PLoS Biology <em>4</em>, 780–788.</p>
</div>
<div>
<p>Buenrostro, J.D., Giresi, P.G., Zaba, L.C., Chang, H.Y., and Greenleaf, W.J. (2013). Transposition of native chromatin for fast and sensitive epigenomic profiling of open chromatin, DNA-binding proteins and nucleosome position. Nature Methods <em>10</em>, 1213–1218.</p>
</div>
<div>
<p>Cabili, M.N., Trapnell, C., Goff, L., Koziol, M., Tazon-Vega, B., Regev, A., and Rinn, J.L. (2011). Integrative annotation of human large intergenic noncoding RNAs reveals global properties and specific subclasses. Genes &amp; Development <em>25</em>, 1915–1927.</p>
</div>
<div>
<p>Cai, J., Goodman, B.K., Patel, A.S., Mulliken, J.B., Van Maldergem, L., Hoganson, G.E., Paznekas, W.A., Ben-Neriah, Z., Sheffer, R., Cunningham, M.L., et al. (2003). Increased risk for developmental delay in Saethre-Chotzen syndrome is associated with TWIST deletions: an improved strategy for TWIST mutation screening. Human Genetics <em>114</em>, 68–76.</p>
</div>
<div>
<p>Canela, A., Maman, Y., Jung, S., Wong, N., Callen, E., Day, A., Kieffer-Kwon, K.-R., Pekowska, A., Zhang, H., Rao, S.S., et al. (2017). Genome Organization Drives Chromosome Fragility. Cell 1–15.</p>
</div>
<div>
<p>Cardoso, A.R., Oliveira, M., Amorim, A., and Azevedo, L. (2016). Major influence of repetitive elements on disease-associated copy number variants (CNVs). Human Genomics <em>10</em>, 30.</p>
</div>
<div>
<p>Carroll, S.B. (2008). Evo-Devo and an Expanding Evolutionary Synthesis: A Genetic Theory of Morphological Evolution. Cell <em>134</em>, 25–36.</p>
</div>
<div>
<p>Carter, D., Chakalova, L., Osborne, C.S., Dai, Y.F., and Fraser, P. (2002). Long-range chromatin regulatory interactions in vivo. Nat Genet <em>32</em>, 623–626.</p>
</div>
<div>
<p>Claussnitzer, M., Dankel, S.N., Kim, K.-H., Quon, G., Meuleman, W., Haugen, C., Glunk, V., Sousa, I.S., Beaudry, J.L., Puviindran, V., et al. (2015). &lt;i&gt;FTO&lt;/i&gt; Obesity Variant Circuitry and Adipocyte Browning in Humans. New England Journal of Medicine <em>373</em>, 895–907.</p>
</div>
<div>
<p>Conrad, D.F., Andrews, T.D., Carter, N.P., Hurles, M.E., and Pritchard, J.K. (2006). A high-resolution survey of deletion polymorphism in the human genome. Nature Genetics <em>38</em>, 75–81.</p>
</div>
<div>
<p>Crane, E., Bian, Q., McCord, R.P., Lajoie, B.R., Wheeler, B.S., Ralston, E.J., Uzawa, S., Dekker, J., and Meyer, B.J. (2015). Condensin-driven remodelling of X chromosome topology during dosage compensation. Nature.</p>
</div>
<div>
<p>Cremer, T., and Cremer, C. (2001). Chromosome territories, nuclear architecture and gene regulation in mammalian cells. Nature Reviews Genetics <em>2</em>, 292–301.</p>
</div>
<div>
<p>Cremer, T., and Cremer, M. (2010). Chromosome territories. Cold Spring Harbor Perspectives in Biology <em>2</em>, a003889.</p>
</div>
<div>
<p>Cremer, T., Cremer, M., Hübner, B., Strickfaden, H., Smeets, D., Popken, J., Sterr, M., Markaki, Y., Rippe, K., and Cremer, C. (2015). The 4D nucleome: Evidence for a dynamic nuclear landscape based on co-aligned active and inactive nuclear compartments. FEBS Letters.</p>
</div>
<div>
<p>Creyghton, M.P., Cheng, A.W., Welstead, G.G., Kooistra, T., Carey, B.W., Steine, E.J., Hanna, J., Lodato, M.A., Frampton, G.M., Sharp, P.A., et al. (2010). Histone H3K27ac separates active from poised enhancers and predicts developmental state. Proceedings of the National Academy of Sciences of the United States of America <em>107</em>, 21931–21936.</p>
</div>
<div>
<p>Dekker, J., Rippe, K., Dekker, M., and Kleckner, N. (2002). Capturing chromosome conformation. Science (New York, N.Y.) <em>295</em>, 1306–1311.</p>
</div>
<div>
<p>Dekker, J., Marti-Renom, M.A., and Mirny, L.A. (2013). Exploring the three-dimensional organization of genomes: interpreting chromatin interaction data. Nature Reviews. Genetics <em>14</em>, 390–403.</p>
</div>
<div>
<p>Deng, W., Lee, J., Wang, H., Miller, J., Reik, A., Gregory, P.D., Dean, A., and Blobel, G. a (2012). Controlling long-range genomic interactions at a native locus by targeted tethering of a looping factor. Cell <em>149</em>, 1233–1244.</p>
</div>
<div>
<p>Dileep, V., Ay, F., Sima, J., Vera, D.L., Noble, W.S., and Gilbert, D.M. (2015). Topologically associating domains and their long-range contacts are established during early G1 coincident with the establishment of the replication-timing program. Genome Research <em>25</em>, 1104–1113.</p>
</div>
<div>
<p>Dimitrieva, S., and Bucher, P. (2013). UCNEbase—a database of ultraconserved non-coding elements and genomic regulatory blocks. Nucleic Acids Research <em>41</em>, D101–D109.</p>
</div>
<div>
<p>Dixon, J.R., Selvaraj, S., Yue, F., Kim, A., Li, Y., Shen, Y., Hu, M., Liu, J.S., and Ren, B. (2012). Topological domains in mammalian genomes identified by analysis of chromatin interactions. Nature <em>485</em>, 376–380.</p>
</div>
<div>
<p>Dixon, J.R., Jung, I., Selvaraj, S., Shen, Y., Antosiewicz-Bourget, J.E., Lee, A.Y., Ye, Z., Kim, A., Rajagopal, N., Xie, W., et al. (2015). Chromatin architecture reorganization during stem cell differentiation. Nature <em>518</em>, 331–336.</p>
</div>
<div>
<p>Dixon, J.R., Gorkin, D.U., and Ren, B. (2016). Chromatin Domains: The Unit of Chromosome Organization. Molecular Cell <em>62</em>, 668–680.</p>
</div>
<div>
<p>Djebali, S., Davis, C.a., Merkel, A., Dobin, A., Lassmann, T., Mortazavi, A., Tanzer, A., Lagarde, J., Lin, W., Schlesinger, F., et al. (2012). Landscape of transcription in human cells. Nature <em>489</em>, 101–108.</p>
</div>
<div>
<p>Dostie, J., Richmond, T.A., Arnaout, R.A., Selzer, R.R., Lee, W.L., Honan, T.A., Rubio, E.D., Krumm, A., Lamb, J., Nusbaum, C., et al. (2006). Chromosome Conformation Capture Carbon Copy (5C): A massively parallel solution for mapping interactions between genomic elements. Genome Research <em>16</em>, 1299–1309.</p>
</div>
<div>
<p>Dunham, I., Kundaje, A., Aldred, S.F., Collins, P.J., Davis, C. a, Doyle, F., Epstein, C.B., Frietze, S., Harrow, J., Kaul, R., et al. (2012). An integrated encyclopedia of DNA elements in the human genome. Nature <em>489</em>, 57–74.</p>
</div>
<div>
<p>Durinck, S., Moreau, Y., Kasprzyk, A., Davis, S., De Moor, B., Brazma, A., and Huber, W. (2005). BioMart and Bioconductor: A powerful link between biological databases and microarray data analysis. Bioinformatics <em>21</em>, 3439–3440.</p>
</div>
<div>
<p>Durinck, S., Spellman, P.T., Birney, E., and Huber, W. (2009a). Mapping identifiers for the integration of genomic datasets with the R/Bioconductor package biomaRt. Nature Protocols <em>4</em>, 1184–1191.</p>
</div>
<div>
<p>Durinck, S., Spellman, P.T., Birney, E., and Huber, W. (2009b). Mapping identifiers for the integration of genomic datasets with the R/Bioconductor package biomaRt. Nature Protocols <em>4</em>, 1184–1191.</p>
</div>
<div>
<p>Eagen, K.P., Hartl, T.A., and Kornberg, R.D. (2015). Stable Chromosome Condensation Revealed by Chromosome Conformation Capture. Cell <em>163</em>, 934–946.</p>
</div>
<div>
<p>Eisenberg, E., and Levanon, E.Y. (2013). Human housekeeping genes, revisited. Trends in Genetics <em>29</em>, 569–574.</p>
</div>
<div>
<p>Engström, P.G., Sui, S.J.H., Drivenes, Ø., Becker, T.S., and Lenhard, B. (2007). Genomic regulatory blocks underlie extensive microsynteny conservation in insects. Genome Research <em>17</em>, 1898–1908.</p>
</div>
<div>
<p>Fantes, J., Redeker, B., Breen, M., Boyle, S., Brown, J., Fletcher, J., Jones, S., Bickmore, W., Fukushima, Y., Mannens, M., et al. (1995). Aniridia-associated cytogenetic rearrangements suggest that a position effect may cause the mutant phenotype. Human Molecular Genetics <em>4</em>, 415–422.</p>
</div>
<div>
<p>Farré, M., Robinson, T.J., and Ruiz-Herrera, A. (2015). An Integrative Breakage Model of genome architecture, reshuffling and evolution. BioEssays n/a–n/a.</p>
</div>
<div>
<p>Filippova, D., Patro, R., Duggal, G., and Kingsford, C. (2014). Identification of alternative topological domains in chromatin. Algorithms for Molecular Biology : AMB <em>9</em>, 14.</p>
</div>
<div>
<p>Firth, H.V., Richards, S.M., Bevan, a P., Clayton, S., Corpas, M., Rajan, D., Van Vooren, S., Moreau, Y., Pettett, R.M., and Carter, N.P. (2009). DECIPHER: Database of Chromosomal Imbalance and Phenotype in Humans Using Ensembl Resources. American Journal of Human Genetics <em>84</em>, 524–533.</p>
</div>
<div>
<p>Flavahan, W. a, Drier, Y., Liau, B.B., Gillespie, S.M., Venteicher, a S., Stemmer-Rachamimov, a O., Suva, M.L., and Bernstein, B.E. (2016). Insulator dysfunction and oncogene activation in IDH mutant gliomas. Nature <em>529</em>, 110–114.</p>
</div>
<div>
<p>Flicek, P., Amode, M.R., Barrell, D., Beal, K., Billis, K., Brent, S., Carvalho-Silva, D., Clapham, P., Coates, G., Fitzgerald, S., et al. (2014). Ensembl 2014. Nucleic Acids Research <em>42</em>, D749–D755.</p>
</div>
<div>
<p>Flomen, R.H., Vatcheva, R., Gorman, P.A., Baptista, P.R., Groet, J., Barišić, I., Ligutic, I., and Nižetić, D. (1998). Construction and Analysis of a Sequence-Ready Map in 4q25: Rieger Syndrome Can Be Caused by Haploinsufficiency ofRIEG, but Also by Chromosome Breaks ≈90 kb Upstream of This Gene. Genomics <em>47</em>, 409–413.</p>
</div>
<div>
<p>Flyamer, I.M., Gassler, J., Imakaev, M., Ulyanov, S.V., Abdennur, N., Razin, S.V., Mirny, L., and Tachibana-Konwalski, K. (2017). Single-cell Hi-C reveals unique chromatin reorganization at oocyte-tozygote transition. Nature Publishing Group <em>544</em>, 1–17.</p>
</div>
<div>
<p>Forcato, M., Nicoletti, C., Pal, K., Livi, C.M., Ferrari, F., and Bicciato, S. (2017). Comparison of computational methods for Hi-C data analysis. Nature Methods 14–19.</p>
</div>
<div>
<p>Forrest, A.R.R., Kawaji, H., Rehli, M., Baillie, J.K., Hoon, M.J.L. de, Lassmann, T., Itoh, M., Summers, K.M., Suzuki, H., Daub, C.O., et al. (2014). A promoter-level mammalian expression atlas. Nature <em>507</em>, 462–470.</p>
</div>
<div>
<p>Franke, M., Ibrahim, D.M., Andrey, G., Schwarzer, W., Heinrich, V., Schöpflin, R., Kraft, K., Kempfer, R., Jerković, I., Chan, W.-L., et al. (2016). Formation of new chromatin domains determines pathogenicity of genomic duplications. Nature 1–15.</p>
</div>
<div>
<p>Fraser, J., Ferrai, C., Chiariello, A.M., Schueler, M., Rito, T., Laudanno, G., Barbieri, M., Moore, B.L., Kraemer, D.C., Aitken, S., et al. (2015). Hierarchical folding and reorganization of chromosomes are linked to transcriptional changes in cellular differentiation. Mol Syst Biol <em>11</em>, 1–14.</p>
</div>
<div>
<p>Fullwood, M.J., Liu, M.H., Pan, Y.F., Liu, J., Xu, H., Mohamed, Y.B., Orlov, Y.L., Velkov, S., Ho, A., Mei, P.H., et al. (2009). An oestrogen-receptor-<span class="math inline">\(\alpha\)</span>-bound human chromatin interactome. Nature <em>462</em>, 58–64.</p>
</div>
<div>
<p>Galil, Z. (1986). Efficient algorithms for finding maximum matching in graphs. ACM Computing Surveys <em>18</em>, 23–38.</p>
</div>
<div>
<p>Gibcus, J.H., and Dekker, J. (2013). The hierarchy of the 3D genome. Molecular Cell <em>49</em>, 773–782.</p>
</div>
<div>
<p>Giorgio, E., Robyr, D., Spielmann, M., Ferrero, E., Di Gregorio, E., Imperiale, D., Vaula, G., Stamoulis, G., Santoni, F., Atzori, C., et al. (2015). A large genomic deletion leads to enhancer adoption by the lamin B1 gene: a second path to autosomal dominant adult-onset demyelinating leukodystrophy (ADLD). Human Molecular Genetics <em>24</em>, 3143–3154.</p>
</div>
<div>
<p>Gómez-Marín, C., Tena, J.J., Acemel, R.D., López-Mayorga, M., Naranjo, S., Calle-Mustienes, E. de la, Maeso, I., Beccari, L., Aneas, I., Vielmas, E., et al. (2015). Evolutionary comparison reveals that diverging CTCF sites are signatures of ancestral topological associating domains borders. Proceedings of the National Academy of Sciences <em>112</em>, 201505463.</p>
</div>
<div>
<p>Gröschel, S., Sanders, M.A., Hoogenboezem, R., Wit, E. de, Bouwman, B.A.M., Erpelinck, C., Velden, V.H.J. van der, Havermans, M., Avellino, R., Lom, K. van, et al. (2014). A single oncogenic enhancer rearrangement causes concomitant EVI1 and GATA2 deregulation in leukemia. Cell <em>157</em>, 369–381.</p>
</div>
<div>
<p>Gu, W., Zhang, F., and Lupski, J.R. (2008). Mechanisms for human genomic rearrangements. PathoGenetics <em>1</em>, 4.</p>
</div>
<div>
<p>Harmston, N., Ing-Simmons, E., Tan, G., Perry, M., Merkenschlager, M., and Lenhard, B. (2017). Topologically associating domains are ancient features that coincide with Metazoan clusters of extreme noncoding conservation. Nature Communications <em>8</em>, 441.</p>
</div>
<div>
<p>Hay, D., Hughes, J.R., Babbs, C., Davies, J.O.J., Graham, B.J., Hanssen, L.L.P., Kassouf, M.T., Oudelaar, A.M., Sharpe, J.A., Suciu, M.C., et al. (2016). Genetic dissection of the <span class="math inline">\(\alpha\)</span>-globin super-enhancer in vivo. Nature Genetics <em>48</em>, 895–903.</p>
</div>
<div>
<p>He, X., and Zhang, J. (2005). Gene complexity and gene duplicability. Current Biology <em>15</em>, 1016–1021.</p>
</div>
<div>
<p>Heger, A., Webber, C., Goodson, M., Ponting, C.P., and Lunter, G. (2013). GAT: a simulation framework for testing the association of genomic intervals. Bioinformatics (Oxford, England) <em>29</em>, 2046–2048.</p>
</div>
<div>
<p>Herrero, J., Muffato, M., Beal, K., Fitzgerald, S., Gordon, L., Pignatelli, M., Vilella, A.J., Searle, S.M.J., Amode, R., Brent, S., et al. (2016). Ensembl comparative genomics resources. Database <em>2016</em>, bav096.</p>
</div>
<div>
<p>Higgins, A.W., Alkuraya, F.S., Bosco, A.F., Brown, K.K., Bruns, G.A., Donovan, D.J., Eisenman, R., Fan, Y., Farra, C.G., Ferguson, H.L., et al. (2008). Characterization of Apparently Balanced Chromosomal Rearrangements from the Developmental Genome Anatomy Project. American Journal of Human Genetics <em>82</em>, 712–722.</p>
</div>
<div>
<p>Hinds, D.A., Kloek, A.P., Jen, M., Chen, X., and Frazer, K.A. (2006). Common deletions and SNPs are in linkage disequilibrium in the human genome. Nature Genetics <em>38</em>, 82–85.</p>
</div>
<div>
<p>Hinrichs, A.S., Karolchik, D., Baertsch, R., Barber, G.P., Bejerano, G., Clawson, H., Diekhans, M., Furey, T.S., Harte, R.A., Hsu, F., et al. (2006). The UCSC Genome Browser Database: update 2006. Nucleic Acids Res <em>34</em>, D590—–D598.</p>
</div>
<div>
<p>Hinsch, H., and Hannenhalli, S. (2006). Recurring genomic breaks in independent lineages support genomic fragility. BMC Evolutionary Biology <em>6</em>, 90.</p>
</div>
<div>
<p>Hnisz, D., Weintraub, A.S., Day, D.S., Valton, A.-l., Bak, R.O., Li, C.H., Goldmann, J., Lajoie, B.R., Fan, Z.P., Sigova, A.A., et al. (2016). Activation of proto-oncogenes by disruption of chromosome neighborhoods. Science (New York, N.Y.) <em>351</em>, 1454–1458.</p>
</div>
<div>
<p>Hou, C., Li, L., Qin, Z.S., and Corces, V.G. (2012). Gene Density, Transcription, and Insulators Contribute to the Partition of the Drosophila Genome into Physical Domains. Molecular Cell <em>48</em>, 471–484.</p>
</div>
<div>
<p>Hsieh, T.-H.S., Weiner, A., Lajoie, B., Dekker, J., Friedman, N., and Rando, O.J. (2015). Mapping Nucleosome Resolution Chromosome Folding in Yeast by Micro-C. Cell 1–12.</p>
</div>
<div>
<p>Huang, N., Lee, I., Marcotte, E.M., and Hurles, M.E. (2010). Characterising and Predicting Haploinsufficiency in the Human Genome. PLoS Genetics <em>6</em>, e1001154.</p>
</div>
<div>
<p>Huber, W., Carey, V.J., Gentleman, R., Anders, S., Carlson, M., Carvalho, B.S., Bravo, H.C., Davis, S., Gatto, L., Girke, T., et al. (2015). Orchestrating high-throughput genomic analysis with Bioconductor. Nature Methods <em>12</em>, 115–121.</p>
</div>
<div>
<p>Huminiecki, L. (2004). Divergence of Spatial Gene Expression Profiles Following Species-Specific Gene Duplications in Human and Mouse. Genome Research <em>14</em>, 1870–1879.</p>
</div>
<div>
<p>Iafrate, A.J., Feuk, L., Rivera, M.N., Listewnik, M.L., Donahoe, P.K., Qi, Y., Scherer, S.W., and Lee, C. (2004). Detection of large-scale variation in the human genome. Nature Genetics <em>36</em>, 949–951.</p>
</div>
<div>
<p>Ibn-Salem, J., Köhler, S., Love, M.I., Chung, H.-R., Huang, N., Hurles, M.E., Haendel, M., Washington, N.L., Smedley, D., Mungall, C.J., et al. (2014). Deletions of chromosomal regulatory boundaries are associated with congenital disease. Genome Biology <em>15</em>, 423.</p>
</div>
<div>
<p>Ibn-Salem, J., Muro, E.M., and Andrade-Navarro, M.A. (2017). Co-regulation of paralog genes in the three-dimensional chromatin architecture. Nucleic Acids Research <em>45</em>, 81–91.</p>
</div>
<div>
<p>Kent, W.J., Sugnet, C.W., Furey, T.S., Roskin, K.M., Pringle, T.H., Zahler, A.M., and Haussler, D. (2002). The human genome browser at UCSC. Genome Research <em>12</em>, 996–1006.</p>
</div>
<div>
<p>Kent, W.J., Baertsch, R., Hinrichs, A., Miller, W., and Haussler, D. (2003). Evolution’s cauldron: duplication, deletion, and rearrangement in the mouse and human genomes. Proceedings of the National Academy of Sciences of the United States of America <em>100</em>, 11484–11489.</p>
</div>
<div>
<p>Kikuta, H., Laplante, M., Navratilova, P., Komisarczuk, A.Z., Engström, P.G., Fredman, D., Akalin, A., Caccamo, M., Sealy, I., Howe, K., et al. (2007). Genomic regulatory blocks encompass multiple neighboring genes and maintain conserved synteny in vertebrates. Genome Research <em>17</em>, 545–555.</p>
</div>
<div>
<p>Kim, W.K., and Marcotte, E.M. (2008). Age-dependent evolution of the yeast protein interaction network suggests a limited role of gene duplication and divergence. PLoS Computational Biology <em>4</em>, e1000232.</p>
</div>
<div>
<p>Kleinjan, D., and Heyningen, V. van (1998). Position effect in human genetic disease. Human Molecular Genetics <em>7</em>, 1611–1618.</p>
</div>
<div>
<p>Kleinjan, D.A., and Van Heyningen, V. (2005). Long-Range Control of Gene Expression: Emerging Mechanisms and Disruption in Disease. Am. J. Hum. Genet <em>76</em>, 8–32.</p>
</div>
<div>
<p>Kleinjan, D.A., Seawright, A., Schedl, A., Quinlan, R.A., Danes, S., and Heyningen, V. van (2001). Aniridia-associated translocations, DNase hypersensitivity, sequence comparison and transgenic analysis redefine the functional domain of PAX6. Human Molecular Genetics <em>10</em>, 2049–2059.</p>
</div>
<div>
<p>Knight, P.a., and Ruiz, D. (2013). A fast algorithm for matrix balancing. IMA Journal of Numerical Analysis <em>33</em>, 1029–1047.</p>
</div>
<div>
<p>Koonin, E.V. (2005). Orthologs, paralogs, and evolutionary genomics. Annual Review of Genetics <em>39</em>, 309–338.</p>
</div>
<div>
<p>Korbel, J.O., Urban, A.E., Affourtit, J.P., Godwin, B., Grubert, F., Simons, J.F., Kim, P.M., Palejev, D., Carriero, N.J., Du, L., et al. (2007). Paired-end mapping reveals extensive structural variation in the human genome. Science (New York, N.Y.) <em>318</em>, 420–426.</p>
</div>
<div>
<p>Köhler, S., Doelken, S.C., Mungall, C.J., Bauer, S., Firth, H.V., Bailleul-Forestier, I., Black, G.C.M., Brown, D.L., Brudno, M., Campbell, J., et al. (2014). The Human Phenotype Ontology project: linking molecular biology and disease through phenotype data. Nucleic Acids Research <em>42</em>, D966–74.</p>
</div>
<div>
<p>Laat, W. de, and Grosveld, F. (2007). Inter-chromosomal gene regulation in the mammalian cell nucleus. Current Opinion in Genetics &amp; Development <em>17</em>, 456–464.</p>
</div>
<div>
<p>Lan, X., and Pritchard, J.K. (2016). Coregulation of tandem duplicate genes slows evolution of subfunctionalization in mammals. Science (New York, N.Y.) <em>352</em>, 1009–1013.</p>
</div>
<div>
<p>Lappalainen, I., Lopez, J., Skipper, L., Hefferon, T., Spalding, J.D., Garner, J., Chen, C., Maguire, M., Corbett, M., Zhou, G., et al. (2012). dbVar and DGVa: public archives for genomic structural variation. Nucleic Acids Research <em>41</em>, D936–D941.</p>
</div>
<div>
<p>Lawrence, M., Gentleman, R., and Carey, V. (2009). rtracklayer: an R package for interfacing with genome browsers. Bioinformatics <em>25</em>, 1841–1842.</p>
</div>
<div>
<p>Lawrence, M., Huber, W., Pagès, H., Aboyoun, P., Carlson, M., Gentleman, R., Morgan, M.T., and Carey, V.J. (2013). Software for computing and annotating genomic ranges. PLoS Computational Biology <em>9</em>, e1003118.</p>
</div>
<div>
<p>Le Dily, F., Bau, D., Pohl, a., Vicent, G.P., Serra, F., Soronellas, D., Castellano, G., Wright, R.H.G., Ballare, C., Filion, G., et al. (2014). Distinct structural transitions of chromatin topological domains correlate with coordinated hormone-induced gene regulation. Genes &amp; Development <em>28</em>, 2151–2162.</p>
</div>
<div>
<p>Lieberman-Aiden, E., Berkum, N.L. van, Williams, L., Imakaev, M., Ragoczy, T., Telling, A., Amit, I., Lajoie, B.R., Sabo, P.J., Dorschner, M.O., et al. (2009). Comprehensive mapping of long-range interactions reveals folding principles of the human genome. Science (New York, N.Y.) <em>326</em>, 289–293.</p>
</div>
<div>
<p>Ligon, A.H., Moore, S.D., Parisi, M.A., Mealiffe, M.E., Harris, D.J., Ferguson, H.L., Quade, B.J., and Morton, C.C. (2005). Constitutional Rearrangement of the Architectural Factor HMGA2: A Novel Human Phenotype Including Overgrowth and Lipomas. The American Journal of Human Genetics <em>76</em>, 340–348.</p>
</div>
<div>
<p>Long, H.K., Prescott, S.L., and Wysocka, J. (2016). Ever-Changing Landscapes: Transcriptional Enhancers in Development and Evolution. Cell <em>167</em>, 1170–1187.</p>
</div>
<div>
<p>Lonsdale, J., Thomas, J., Salvatore, M., Phillips, R., Lo, E., Shad, S., Hasz, R., Walters, G., Garcia, F., Young, N., et al. (2013). The Genotype-Tissue Expression (GTEx) project. Nature Genetics <em>45</em>, 580–585.</p>
</div>
<div>
<p>Lu, W., Quintero-Rivera, F., Fan, Y., Alkuraya, F.S., Donovan, D.J., Xi, Q., Turbe-Doan, A., Li, Q.-G., Campbell, C.G., Shanske, A.L., et al. (2007). NFIA Haploinsufficiency Is Associated with a CNS Malformation Syndrome and Urinary Tract Defects. PLoS Genetics <em>3</em>, e80.</p>
</div>
<div>
<p>Lupiáñez, D.G., Kraft, K., Heinrich, V., Krawitz, P., Brancati, F., Klopocki, E., Horn, D., Kayserili, H., Opitz, J.M., Laxova, R., et al. (2015). Disruptions of Topological Chromatin Domains Cause Pathogenic Rewiring of Gene-Enhancer Interactions. Cell <em>161</em>, 1012–1025.</p>
</div>
<div>
<p>Lupski, J.R., and Stankiewicz, P. (2005). Genomic Disorders: Molecular Mechanisms for Rearrangements and Conveyed Phenotypes. PLoS Genetics <em>1</em>, e49.</p>
</div>
<div>
<p>Makova, K.D.K., and Li, W.-H.W. (2003). Divergence in the spatial pattern of gene expression between human duplicate genes. Genome Research <em>13</em>, 1638–1645.</p>
</div>
<div>
<p>Merkenschlager, M., and Nora, E.P. (2016). CTCF and Cohesin in Genome Folding and Transcriptional Gene Regulation. Annual Review of Genomics and Human Genetics <em>17</em>, 17–43.</p>
</div>
<div>
<p>Mifsud, B., Tavares-Cadete, F., Young, A.N., Sugar, R., Schoenfelder, S., Ferreira, L., Wingett, S.W., Andrews, S., Grey, W., Ewels, P. a, et al. (2015). Mapping long-range promoter contacts in human cells with high-resolution capture Hi-C. Nature Genetics <em>47</em>.</p>
</div>
<div>
<p>Miller, D.T., Adam, M.P., Aradhya, S., Biesecker, L.G., Brothman, A.R., Carter, N.P., Church, D.M., Crolla, J.A., Eichler, E.E., Epstein, C.J., et al. (2010). Consensus Statement: Chromosomal Microarray Is a First-Tier Clinical Diagnostic Test for Individuals with Developmental Disabilities or Congenital Anomalies. The American Journal of Human Genetics <em>86</em>, 749–764.</p>
</div>
<div>
<p>Mills, R.E., Bennett, E.A., Iskow, R.C., Luttig, C.T., Tsui, C., Pittard, W.S., and Devine, S.E. (2006). Recently Mobilized Transposons in the Human and Chimpanzee Genomes. The American Journal of Human Genetics <em>78</em>, 671–679.</p>
</div>
<div>
<p>Mizuguchi, T., Fudenberg, G., Mehta, S., Belton, J.-M., Taneja, N., Folco, H.D., FitzGerald, P., Dekker, J., Mirny, L., Barrowman, J., et al. (2014). Cohesin-dependent globules and heterochromatin shape 3D genome architecture in S. pombe. Nature.</p>
</div>
<div>
<p>Monahan, K., and Lomvardas, S. (2015). Monoallelic Expression of Olfactory Receptors. Annual Review of Cell and Developmental Biology <em>31</em>, annurev–cellbio–100814–125308.</p>
</div>
<div>
<p>Montavon, T., Thevenet, L., and Duboule, D. (2012). Impact of copy number variations (CNVs) on long-range gene regulation at the HoxD locus. Proceedings of the National Academy of Sciences of the United States of America <em>109</em>, 20204–20211.</p>
</div>
<div>
<p>Muro, E.M., and Andrade-Navarro, M.A. (2010). Pseudogenes as an alternative source of natural antisense transcripts. BMC Evolutionary Biology <em>10</em>, 338.</p>
</div>
<div>
<p>Murphy, W.J., Larkin, D.M., Everts-van der Wind, A., Bourque, G., Tesler, G., Auvil, L., Beever, J.E., Chowdhary, B.P., Galibert, F., Gatzke, L., et al. (2005). Dynamics of mammalian chromosome evolution inferred from multispecies comparative maps. Science (New York, N.Y.) <em>309</em>, 613–617.</p>
</div>
<div>
<p>Nagano, T., Lubling, Y., Várnai, C., Dudley, C., Leung, W., Baran, Y., and Mendelson-cohen, N. (2017). Cell-cycle dynamics of chromosomal organization at single-cell resolution. Nature Publishing Group <em>547</em>, 61–67.</p>
</div>
<div>
<p>Nambiar, M., and Raghavan, S.C. (2011). How does DNA break during chromosomal translocations? Nucleic Acids Research <em>39</em>, 5813–5825.</p>
</div>
<div>
<p>Narendra, V., Rocha, P.P., An, D., Raviram, R., Skok, J.A., Mazzoni, E.O., and Reinberg, D. (2015). Transcription. CTCF establishes discrete functional chromatin domains at the Hox clusters during differentiation. Science (New York, N.Y.) <em>347</em>, 1017–1021.</p>
</div>
<div>
<p>Newman, S., Hermetz, K.E., Weckselblatt, B., and Rudd, M.K. (2015). Next-Generation Sequencing of Duplication CNVs Reveals that Most Are Tandem and Some Create Fusion Genes at Breakpoints. The American Journal of Human Genetics <em>96</em>, 1–13.</p>
</div>
<div>
<p>Noordermeer, D., Wit, E. de, Klous, P., Werken, H. van de, Simonis, M., Lopez-Jones, M., Eussen, B., Klein, A. de, Singer, R.H., and Laat, W. de (2011). Variegated gene expression caused by cell-specific long-range DNA interactions. Nature Cell Biology <em>13</em>, 944–951.</p>
</div>
<div>
<p>Nora, E.P., Lajoie, B.R., Schulz, E.G., Giorgetti, L., Okamoto, I., Servant, N., Piolot, T., Berkum, N.L. van, Meisig, J., Sedat, J., et al. (2012). Spatial partitioning of the regulatory landscape of the X-inactivation centre. Nature <em>485</em>, 381–385.</p>
</div>
<div>
<p>Nora, E.P., Dekker, J., and Heard, E. (2013). Segmental folding of chromosomes: A basis for structural and regulatory chromosomal neighborhoods? BioEssays <em>35</em>, 818–828.</p>
</div>
<div>
<p>Northcott, P.a., Lee, C., Zichner, T., Stütz, A.M., Erkek, S., Kawauchi, D., Shih, D.J.H., Hovestadt, V., Zapatka, M., Sturm, D., et al. (2014). Enhancer hijacking activates GFI1 family oncogenes in medulloblastoma. Nature.</p>
</div>
<div>
<p>Oldridge, D.A., Wood, A.C., Weichert-Leahey, N., Crimmins, I., Sussman, R., Winter, C., McDaniel, L.D., Diamond, M., Hart, L.S., Zhu, S., et al. (2015). Genetic predisposition to neuroblastoma mediated by a LMO1 super-enhancer polymorphism. Nature <em>528</em>, 418–421.</p>
</div>
<div>
<p>Ordulu, Z., Wong, K.E., Currall, B.B., Ivanov, A.R., Pereira, S., Althari, S., Gusella, J.F., Talkowski, M.E., and Morton, C.C. (2014). Describing Sequencing Results of Structural Chromosome Rearrangements with a Suggested Next-Generation Cytogenetic Nomenclature. The American Journal of Human Genetics <em>94</em>, 695–709.</p>
</div>
<div>
<p>Ordulu, Z., Kammin, T., Brand, H., Pillalamarri, V., Redin, C.E., Collins, R.L., Blumenthal, I., Hanscom, C., Pereira, S., Crandall, B.F., et al. (2016). Structural Chromosomal Rearrangements Require Nucleotide-Level Resolution : Lessons from Next-Generation Sequencing in Prenatal Diagnosis. The American Journal of Human Genetics 1–19.</p>
</div>
<div>
<p>Pachlopnik Schmid, J., Lemoine, R., Nehme, N., Cormier-Daire, V., Revy, P., Debeurme, F., Debré, M., Nitschke, P., Bole-Feysot, C., Legeai-Mallet, L., et al. (2012). Polymerase <span class="math inline">\(\epsilon\)</span>1 mutation in a human syndrome with facial dysmorphism, immunodeficiency, livedo, and short stature (“FILS syndrome”). The Journal of Experimental Medicine <em>209</em>, 2323–2330.</p>
</div>
<div>
<p>Petryszak, R., Keays, M., Tang, Y.A., Fonseca, N.A., Barrera, E., Burdett, T., Füllgrabe, A., Fuentes, A.M.-P., Jupp, S., Koskinen, S., et al. (2015). Expression Atlas update—an integrated database of gene and protein expression in humans, animals and plants. Nucleic Acids Research <em>44</em>, gkv1045.</p>
</div>
<div>
<p>Pevzner, P., and Tesler, G. (2003). Human and mouse genomic sequences reveal extensive breakpoint reuse in mammalian evolution. Proceedings of the National Academy of Sciences of the United States of America <em>100</em>, 7672–7677.</p>
</div>
<div>
<p>Phillips-Cremins, J.E., Sauria, M.E.G., Sanyal, A., Gerasimova, T.I., Lajoie, B.R., Bell, J.S.K., Ong, C.-T., Hookway, T. a, Guo, C., Sun, Y., et al. (2013). Architectural protein subclasses shape 3D organization of genomes during lineage commitment. Cell <em>153</em>, 1281–1295.</p>
</div>
<div>
<p>Pink, R.C., Wicks, K., Caley, D.P., Punch, E.K., Jacobs, L., and Carter, D.R.F. (2011). Pseudogenes: pseudo-functional or key regulators in health and disease? RNA (New York, N.Y.) <em>17</em>, 792–798.</p>
</div>
<div>
<p>Polychronopoulos, D., King, J.W.D., Nash, A.J., Tan, G., and Lenhard, B. (2017). Conserved non-coding elements: developmental gene regulation meets genome organization. Nucleic Acids Research 1–14.</p>
</div>
<div>
<p>Pope, B.D., Ryba, T., Dileep, V., Yue, F., Wu, W., Denas, O., Vera, D.L., Wang, Y., Hansen, R.S., Canfield, T.K., et al. (2014). Topologically associating domains are stable units of replication-timing regulation. Nature <em>515</em>, 402–405.</p>
</div>
<div>
<p>Ptashne, M. (1986). Gene regulation by proteins acting nearby and at a distance. Nature <em>322</em>, 697–701.</p>
</div>
<div>
<p>Purmann, A., Toedling, J., Schueler, M., Carninci, P., Lehrach, H., Hayashizaki, Y., Huber, W., and Sperling, S. (2007). Genomic organization of transcriptomes in mammals: Coregulation and cofunctionality. Genomics <em>89</em>, 580–587.</p>
</div>
<div>
<p>Quinlan, A.R., and Hall, I.M. (2010). BEDTools: a flexible suite of utilities for comparing genomic features. Bioinformatics <em>26</em>, 841–842.</p>
</div>
<div>
<p>Quinn, J.J., and Chang, H.Y. (2016). Unique features of long non-coding RNA biogenesis and function. Nature Reviews Genetics <em>17</em>, 47–62.</p>
</div>
<div>
<p>Rao, S.S.P., Huntley, M.H., Durand, N.C., Stamenova, E.K., Bochkov, I.D., Robinson, J.T., Sanborn, A.L., Machol, I., Omer, A.D., Lander, E.S., et al. (2014). A 3D map of the human genome at kilobase resolution reveals principles of chromatin looping. Cell <em>159</em>, 1665–1680.</p>
</div>
<div>
<p>Redin, C., Brand, H., Collins, R.L., Kammin, T., Mitchell, E., Hodge, J.C., Hanscom, C., Pillalamarri, V., Seabra, C.M., Abbott, M.-A., et al. (2016). The genomic landscape of balanced cytogenetic abnormalities associated with human congenital anomalies. Nature Genetics.</p>
</div>
<div>
<p>Redin, C., Brand, H., Collins, R.L., Kammin, T., Mitchell, E., Hodge, J.C., Hanscom, C., Pillalamarri, V., Seabra, C.M., Abbott, M.-A., et al. (2017). The genomic landscape of balanced cytogenetic abnormalities associated with human congenital anomalies. Nature Genetics <em>49</em>, 36–45.</p>
</div>
<div>
<p>Rehm, H.L., Berg, J.S., Brooks, L.D., Bustamante, C.D., Evans, J.P., Landrum, M.J., Ledbetter, D.H., Maglott, D.R., Martin, C.L., Nussbaum, R.L., et al. (2015). ClinGen — The Clinical Genome Resource. New England Journal of Medicine <em>372</em>, 2235–2242.</p>
</div>
<div>
<p>Rogozin, I.B., Managadze, D., Shabalina, S.A., and Koonin, E.V. (2014). Gene family level comparative analysis of gene expression n mammals validates the ortholog conjecture. Genome Biology and Evolution <em>6</em>, 754–762.</p>
</div>
<div>
<p>Roukos, V., and Misteli, T. (2014). The biogenesis of chromosome translocations. Nature Cell Biology <em>16</em>, 293–300.</p>
</div>
<div>
<p>Roukos, V., Voss, T.C., Schmidt, C.K., Lee, S., Wangsa, D., and Misteli, T. (2013). Spatial dynamics of chromosome translocations in living cells. Science <em>341</em>, 660–664.</p>
</div>
<div>
<p>Roussos, P., Mitchell, A.C., Voloudakis, G., Fullard, J.F., Pothula, V.M., Tsang, J., Stahl, E.A., Georgakopoulos, A., Ruderfer, D.M., Charney, A., et al. (2014). A role for noncoding variation in schizophrenia. Cell Reports <em>9</em>, 1417–1429.</p>
</div>
<div>
<p>Sanyal, A., Lajoie, B.R., Jain, G., and Dekker, J. (2012). The long-range interaction landscape of gene promoters. Nature <em>489</em>, 109–113.</p>
</div>
<div>
<p>Sati, S., and Cavalli, G. (2017). Chromosome conformation capture technologies and their impact in understanding genome function. Chromosoma <em>126</em>, 33–44.</p>
</div>
<div>
<p>Schaefer, M.H., Fontaine, J.-F., Vinayagam, A., Porras, P., Wanker, E.E., and Andrade-Navarro, M. a (2012). HIPPIE: Integrating protein interaction networks with experiment based quality scores. PloS One <em>7</em>, e31826.</p>
</div>
<div>
<p>Schmitt, A.D., Hu, M., and Ren, B. (2016). Genome-wide mapping and analysis of chromosome architecture. Nature Reviews Molecular Cell Biology.</p>
</div>
<div>
<p>Schoenfelder, S., Furlan-magaril, M., Mifsud, B., Tavares-cadete, F., Sugar, R., Javierre, B.-m., Nagano, T., Katsman, Y., Sakthidevi, M., Wingett, S.W., et al. (2015). The pluripotent regulatory circuitry connecting promoters to their long-range interacting elements. Genome Research 1–16.</p>
</div>
<div>
<p>Sebat, J., Lakshmi, B., Troge, J., Alexander, J., Young, J., Lundin, P., Manér, S., Massa, H., Walker, M., Chi, M., et al. (2004). Large-Scale Copy Number Polymorphism in the Human Genome. Science <em>305</em>, 525–528.</p>
</div>
<div>
<p>Sekelja, M., Paulsen, J., and Collas, P. (2016). 4D nucleomes in single cells: what can computational modeling reveal about spatial chromatin conformation? Genome Biology <em>17</em>, 54.</p>
</div>
<div>
<p>Sexton, T., Yaffe, E., Kenigsberg, E., Bantignies, F., Leblanc, B., Hoichman, M., Parrinello, H., Tanay, A., and Cavalli, G. (2012). Three-dimensional folding and functional organization principles of the Drosophila genome. Cell <em>148</em>, 458–472.</p>
</div>
<div>
<p>Shen, Y., Yue, F., McCleary, D.F., Ye, Z., Edsall, L., Kuan, S., Wagner, U., Dixon, J., Lee, L., Lobanenkov, V.V., et al. (2012). A map of the cis-regulatory sequences in the mouse genome. Nature <em>488</em>, 116–120.</p>
</div>
<div>
<p>Shlyueva, D., Stampfel, G., and Stark, A. (2014). Transcriptional enhancers: from properties to genome-wide predictions. Nature Reviews Genetics <em>15</em>, 272–286.</p>
</div>
<div>
<p>Simonis, M., Klous, P., Splinter, E., Moshkin, Y., Willemsen, R., Wit, E. de, Steensel, B. van, and Laat, W. de (2006). Nuclear organization of active and inactive chromatin domains uncovered by chromosome conformation capture–on-chip (4C). Nature Genetics <em>38</em>, 1348–1354.</p>
</div>
<div>
<p>Song, L., and Crawford, G.E. (2010). DNase-seq: a high-resolution technique for mapping active gene regulatory elements across the genome from mammalian cells. Cold Spring Harbor Protocols <em>2010</em>, pdb.prot5384.</p>
</div>
<div>
<p>Spielmann, M., and Mundlos, S. (2016). Looking beyond the genes: the role of non-coding variants in human disease. Human Molecular Genetics ddw205.</p>
</div>
<div>
<p>Spielmann, M., Brancati, F., Krawitz, P.M., Robinson, P.N., Ibrahim, D.M., Franke, M., Hecht, J., Lohan, S., Dathe, K., Nardone, A.M., et al. (2012). Homeotic arm-to-leg transformation associated with genomic rearrangements at the PITX1 locus. American Journal of Human Genetics <em>91</em>, 629–635.</p>
</div>
<div>
<p>Spitz, F., and Furlong, E.E.M. (2012). Transcription factors: from enhancer binding to developmental control. Nature Reviews. Genetics <em>13</em>, 613–626.</p>
</div>
<div>
<p>Sproul, D., Gilbert, N., and Bickmore, W. a (2005). The role of chromatin structure in regulating the expression of clustered genes. Nature Reviews. Genetics <em>6</em>, 775–781.</p>
</div>
<div>
<p>Stevens, T.J., Lando, D., Basu, S., Liam, P., Cao, Y., Lee, S.F., Leeb, M., Wohlfahrt, K.J., Boucher, W., Shaughnessy-kirwan, A.O., et al. (2017). 3D structures of individual mammalian genomes studied by single-cell Hi-C. Nature 1–21.</p>
</div>
<div>
<p>Symmons, O., Uslu, V.V., Tsujimura, T., Ruf, S., Nassari, S., Schwarzer, W., Ettwiller, L., and Spitz, F. (2014). Functional and topological characteristics of mammalian regulatory domains. Genome Research <em>24</em>, 390–400.</p>
</div>
<div>
<p>Tang, Z., Luo, O.J., Li, X., Zheng, M., Zhu, J.J., Szalaj, P., Trzaskoma, P., Magalska, A., Wlodarczyk, J., Ruszczycki, B., et al. (2015). CTCF-Mediated Human 3D Genome Architecture Reveals Chromatin Topology for Transcription. Cell 1–17.</p>
</div>
<div>
<p>Theisen, A., and Shaffer, L.G. (2010). Disorders caused by chromosome abnormalities. The Application of Clinical Genetics <em>3</em>, 159–174.</p>
</div>
<div>
<p>Thurman, R.E., Rynes, E., Humbert, R., Vierstra, J., Maurano, M.T., Haugen, E., Sheffield, N.C., Stergachis, A.B., Wang, H., Vernot, B., et al. (2012). The accessible chromatin landscape of the human genome. Nature <em>489</em>, 75–82.</p>
</div>
<div>
<p>Tolhuis, B., Palstra, R.J., Splinter, E., Grosveld, F., and Laat, W. de (2002). Looping and interaction between hypersensitive sites in the active beta-globin locus. Mol Cell <em>10</em>, 1453–1465.</p>
</div>
<div>
<p>Trembath, D.G., Semina, E.V., Jones, D.H., Patil, S.R., Qian, Q., Amendt, B.A., Russo, A.F., and Murray, J.C. (2004). Analysis of two translocation breakpoints and identification of a negative regulatory element in patients with Rieger’s syndrome. Birth Defects Research Part A: Clinical and Molecular Teratology <em>70</em>, 82–91.</p>
</div>
<div>
<p>Velagaleti, G.V., Bien-Willner, G.A., Northup, J.K., Lockhart, L.H., Hawkins, J.C., Jalal, S.M., Withers, M., Lupski, J.R., and Stankiewicz, P. (2005). Position Effects Due to Chromosome Breakpoints that Map ∼900 Kb Upstream and ∼1.3 Mb Downstream of SOX9 in Two Patients with Campomelic Dysplasia. The American Journal of Human Genetics <em>76</em>, 652–662.</p>
</div>
<div>
<p>Vetro, A., Dehghani, M.R., Kraoua, L., Giorda, R., Beri, S., Cardarelli, L., Merico, M., Manolakos, E., Parada-Bustamante, A., Castro, A., et al. (2015). Testis development in the absence of SRY: chromosomal rearrangements at SOX9 and SOX3. European Journal of Human Genetics <em>23</em>, 1025–1032.</p>
</div>
<div>
<p>Vietri Rudan, M., Barrington, C., Henderson, S., Ernst, C., Odom, D.T., Tanay, A., and Hadjur, S. (2015). Comparative Hi-C reveals that CTCF underlies evolution of chromosomal domain architecture. Cell Reports <em>10</em>, 1297–1309.</p>
</div>
<div>
<p>Vilella, A.J., Severin, J., Ureta-Vidal, A., Heng, L., Durbin, R., and Birney, E. (2009). EnsemblCompara GeneTrees: Complete, duplication-aware phylogenetic trees in vertebrates. Genome Research <em>19</em>, 327–335.</p>
</div>
<div>
<p>Visel, A., Minovitsky, S., Dubchak, I., and Pennacchio, L. a (2007). VISTA Enhancer Browser–a database of tissue-specific human enhancers. Nucleic Acids Research <em>35</em>, D88–92.</p>
</div>
<div>
<p>Visser, M., Kayser, M., and Palstra, R.-J. (2012). HERC2 rs12913832 modulates human pigmentation by attenuating chromatin-loop formation between a long-range enhancer and the OCA2 promoter. Genome Research <em>22</em>, 446–455.</p>
</div>
<div>
<p>Weiler, K.S., and Wakimoto, B.T. (1995). Heterochromatin and gene expression in Drosophila. Annual Review of Genetics <em>29</em>, 577–605.</p>
</div>
<div>
<p>Weischenfeldt, J., Dubash, T., Drainas, A.P., Mardin, B.R., Chen, Y., Stütz, A.M., Waszak, S.M., Bosco, G., Halvorsen, A.R., Raeder, B., et al. (2016). Pan-cancer analysis of somatic copy-number alterations implicates IRS4 and IGF2 in enhancer hijacking. Nature Genetics.</p>
</div>
<div>
<p>Wickham, H., and Grolemund, G. R for data science : import, tidy, transform, visualize, and model data.</p>
</div>
<div>
<p>Will, A.J., Cova, G., Osterwalder, M., Chan, W.-L., Wittler, L., Brieske, N., Heinrich, V., Villartay, J.-P. de, Vingron, M., Klopocki, E., et al. (2017). Composition and dosage of a multipartite enhancer cluster control developmental expression of Ihh (Indian hedgehog). Nature Genetics.</p>
</div>
<div>
<p>Wit, E. de, and Laat, W. de (2012). A decade of 3C technologies: insights into nuclear organization. Genes &amp; Development <em>26</em>, 11–24.</p>
</div>
<div>
<p>Xie, S., Duan, J., Li, B., Zhou, P., and Hon, G.C. (2017). Multiplexed Engineering and Analysis of Combinatorial Enhancer Activity in Single Cells. Molecular Cell <em>66</em>, 285–299.e5.</p>
</div>
<div>
<p>Zepeda-Mendoza, C.J., Ibn-Salem, J., Kammin, T., Harris, D.J., Rita, D., Gripp, K.W., MacKenzie, J.J., Gropman, A., Graham, B., Shaheen, R., et al. (2017). Computational Prediction of Position Effects of Apparently Balanced Human Chromosomal Rearrangements. American Journal of Human Genetics <em>101</em>, 206–217.</p>
</div>
<div>
<p>Zhan, Y., Mariani, L., Barozzi, I., Schulz, E.G., Blüthgen, N., Stadler, M., Tiana, G., Giorgetti, L., Bluthgen, N., Stadler, M., et al. (2017). Reciprocal insulation analysis of Hi-C data shows that TADs represent a functionally but not structurally privileged scale in the hierarchical folding of chromosomes. Genome Research <em>27</em>, gr.212803.116.</p>
</div>
<div>
<p>Zhang, B., and Wolynes, P.G. (2015). Topology, structures, and energy landscapes of human chromosomes. Proceedings of the National Academy of Sciences <em>112</em>, 201506257.</p>
</div>
<div>
<p>Zhang, F., Gu, W., Hurles, M.E., and Lupski, J.R. (2009). Copy Number Variation in Human Health, Disease, and Evolution. Annual Review of Genomics and Human Genetics <em>10</em>, 451–481.</p>
</div>
<div>
<p>Zhou, X., and Wang, T. (2012). Using the Wash U Epigenome Browser to Examine Genome-Wide Sequencing Data. In Current Protocols in Bioinformatics, (Hoboken, NJ, USA: John Wiley &amp; Sons, Inc.), p. Unit10.10.</p>
</div>
</div>
</div>
<!-- </div> -->
            </section>

          </div>
        </div>
      </div>
<a href="B-2-supplementary-figures.html" class="navigation navigation-prev navigation-unique" aria-label="Previous page"><i class="fa fa-angle-left"></i></a>

    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"google": false,
"weibo": false,
"instapper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/ibn-salem/thesis/edit/master/08-appendix.Rmd",
"text": "Edit"
},
"download": ["thesis.pdf", "thesis.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "";
    if (src === "" || src === "true") src = "https://cdn.bootcss.com/mathjax/2.7.1/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:" && /^https?:/.test(src))
      src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
