<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>Genome folding in evolution and disease</title>
  <meta name="description" content="A draft PhD thesis on genome folding in evoultion and disease.">
  <meta name="generator" content="bookdown 0.6.1 and GitBook 2.6.7">

  <meta property="og:title" content="Genome folding in evolution and disease" />
  <meta property="og:type" content="book" />
  <meta property="og:url" content="https://ibn-salem.github.io/thesis/" />
  
  <meta property="og:description" content="A draft PhD thesis on genome folding in evoultion and disease." />
  <meta name="github-repo" content="ibn-salem/thesis" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Genome folding in evolution and disease" />
  
  <meta name="twitter:description" content="A draft PhD thesis on genome folding in evoultion and disease." />
  

<meta name="author" content="Jonas Ibn-Salem">


<meta name="date" content="2018-02-21">

  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  
<link rel="prev" href="C-sup-position-effect.html">
<link rel="next" href="C-2-supplemental-figure.html">
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />










<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Thesis</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="1" data-path="1-intro.html"><a href="1-intro.html"><i class="fa fa-check"></i><b>1</b> Introduction</a><ul>
<li class="chapter" data-level="1.1" data-path="1-1-regulation-of-expression.html"><a href="1-1-regulation-of-expression.html"><i class="fa fa-check"></i><b>1.1</b> Regulation of gene expression</a></li>
<li class="chapter" data-level="1.2" data-path="1-2-distal-regulation-by-enhancers.html"><a href="1-2-distal-regulation-by-enhancers.html"><i class="fa fa-check"></i><b>1.2</b> Distal regulation by enhancers</a></li>
<li class="chapter" data-level="1.3" data-path="1-3-methods-to-probe-the-3d-chromatin-architecture.html"><a href="1-3-methods-to-probe-the-3d-chromatin-architecture.html"><i class="fa fa-check"></i><b>1.3</b> Methods to probe the 3D chromatin architecture</a><ul>
<li class="chapter" data-level="1.3.1" data-path="1-3-methods-to-probe-the-3d-chromatin-architecture.html"><a href="1-3-methods-to-probe-the-3d-chromatin-architecture.html#microscopy-based-techniques-to-visualize-the-genome-in-3d"><i class="fa fa-check"></i><b>1.3.1</b> Microscopy-based techniques to visualize the genome in 3D</a></li>
<li class="chapter" data-level="1.3.2" data-path="1-3-methods-to-probe-the-3d-chromatin-architecture.html"><a href="1-3-methods-to-probe-the-3d-chromatin-architecture.html#proximity-ligation"><i class="fa fa-check"></i><b>1.3.2</b> Proximity-ligation based method to quantify chromatin interactions</a></li>
</ul></li>
<li class="chapter" data-level="1.4" data-path="1-4-hierarchy-of-chromatin-3d-structure.html"><a href="1-4-hierarchy-of-chromatin-3d-structure.html"><i class="fa fa-check"></i><b>1.4</b> Hierarchy of chromatin 3D structure</a><ul>
<li class="chapter" data-level="1.4.1" data-path="1-4-hierarchy-of-chromatin-3d-structure.html"><a href="1-4-hierarchy-of-chromatin-3d-structure.html#chromosomal-territories-and-inter-chromosomal-contacts"><i class="fa fa-check"></i><b>1.4.1</b> Chromosomal territories and inter-chromosomal contacts</a></li>
<li class="chapter" data-level="1.4.2" data-path="1-4-hierarchy-of-chromatin-3d-structure.html"><a href="1-4-hierarchy-of-chromatin-3d-structure.html#ab-compartments"><i class="fa fa-check"></i><b>1.4.2</b> A/B compartments</a></li>
<li class="chapter" data-level="1.4.3" data-path="1-4-hierarchy-of-chromatin-3d-structure.html"><a href="1-4-hierarchy-of-chromatin-3d-structure.html#topologically-associating-domains-tads"><i class="fa fa-check"></i><b>1.4.3</b> Topologically associating domains (TADs)</a></li>
<li class="chapter" data-level="1.4.4" data-path="1-4-hierarchy-of-chromatin-3d-structure.html"><a href="1-4-hierarchy-of-chromatin-3d-structure.html#hierarchy-of-domain-like-structures-across-genomic-length-scales"><i class="fa fa-check"></i><b>1.4.4</b> Hierarchy of domain-like structures across genomic-length scales</a></li>
<li class="chapter" data-level="1.4.5" data-path="1-4-hierarchy-of-chromatin-3d-structure.html"><a href="1-4-hierarchy-of-chromatin-3d-structure.html#chromatin-looping-interactions"><i class="fa fa-check"></i><b>1.4.5</b> Chromatin looping interactions</a></li>
<li class="chapter" data-level="1.4.6" data-path="1-4-hierarchy-of-chromatin-3d-structure.html"><a href="1-4-hierarchy-of-chromatin-3d-structure.html#loop-mechanism"><i class="fa fa-check"></i><b>1.4.6</b> TAD and loop formation by architectural proteins</a></li>
</ul></li>
<li class="chapter" data-level="1.5" data-path="1-5-dynamics-of-chromatin-structure.html"><a href="1-5-dynamics-of-chromatin-structure.html"><i class="fa fa-check"></i><b>1.5</b> Dynamics of chromatin structure</a><ul>
<li class="chapter" data-level="1.5.1" data-path="1-5-dynamics-of-chromatin-structure.html"><a href="1-5-dynamics-of-chromatin-structure.html#dynamics-across-cell-cycle"><i class="fa fa-check"></i><b>1.5.1</b> Dynamics across cell cycle</a></li>
<li class="chapter" data-level="1.5.2" data-path="1-5-dynamics-of-chromatin-structure.html"><a href="1-5-dynamics-of-chromatin-structure.html#dynamics-across-cell-types-and-differntiation"><i class="fa fa-check"></i><b>1.5.2</b> Dynamics across cell types and differntiation</a></li>
</ul></li>
<li class="chapter" data-level="1.6" data-path="1-6-evolution-of-chromatin-organization.html"><a href="1-6-evolution-of-chromatin-organization.html"><i class="fa fa-check"></i><b>1.6</b> Evolution of chromatin organization</a></li>
<li class="chapter" data-level="1.7" data-path="1-7-disruption-of-chromatin-architecture-in-disease.html"><a href="1-7-disruption-of-chromatin-architecture-in-disease.html"><i class="fa fa-check"></i><b>1.7</b> Disruption of chromatin architecture in disease</a></li>
<li class="chapter" data-level="1.8" data-path="1-8-aims-of-this-thesis.html"><a href="1-8-aims-of-this-thesis.html"><i class="fa fa-check"></i><b>1.8</b> Aims of this thesis</a></li>
<li class="chapter" data-level="1.9" data-path="1-9-structure-of-this-thesis.html"><a href="1-9-structure-of-this-thesis.html"><i class="fa fa-check"></i><b>1.9</b> Structure of this thesis</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="2-paralog-regulation.html"><a href="2-paralog-regulation.html"><i class="fa fa-check"></i><b>2</b> Paralog genes in the 3D genome architecture</a><ul>
<li class="chapter" data-level="" data-path="2-paralog-regulation.html"><a href="2-paralog-regulation.html#preamble"><i class="fa fa-check"></i>Preamble</a></li>
<li class="chapter" data-level="" data-path="2-paralog-regulation.html"><a href="2-paralog-regulation.html#abstract-1"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="2.1" data-path="2-1-introduction.html"><a href="2-1-introduction.html"><i class="fa fa-check"></i><b>2.1</b> Introduction</a></li>
<li class="chapter" data-level="2.2" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html"><i class="fa fa-check"></i><b>2.2</b> Materials and methods</a><ul>
<li class="chapter" data-level="2.2.1" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html#selection-of-pairs-of-paralog-genes"><i class="fa fa-check"></i><b>2.2.1</b> Selection of pairs of paralog genes</a></li>
<li class="chapter" data-level="2.2.2" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html#enhancers-to-gene-association"><i class="fa fa-check"></i><b>2.2.2</b> Enhancers to gene association</a></li>
<li class="chapter" data-level="2.2.3" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html#topological-associating-domains"><i class="fa fa-check"></i><b>2.2.3</b> Topological associating domains</a></li>
<li class="chapter" data-level="2.2.4" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html#hi-c-interaction-maps"><i class="fa fa-check"></i><b>2.2.4</b> Hi-C interaction maps</a></li>
<li class="chapter" data-level="2.2.5" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html#randomization"><i class="fa fa-check"></i><b>2.2.5</b> Randomization</a></li>
<li class="chapter" data-level="2.2.6" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html#statistical-tests"><i class="fa fa-check"></i><b>2.2.6</b> Statistical tests</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="2-3-results.html"><a href="2-3-results.html"><i class="fa fa-check"></i><b>2.3</b> Results</a><ul>
<li class="chapter" data-level="2.3.1" data-path="2-3-results.html"><a href="2-3-results.html#distribution-of-paralog-genes-in-the-human-genome"><i class="fa fa-check"></i><b>2.3.1</b> Distribution of paralog genes in the human genome</a></li>
<li class="chapter" data-level="2.3.2" data-path="2-3-results.html"><a href="2-3-results.html#co-expression-of-paralog-gene-pairs-across-tissues"><i class="fa fa-check"></i><b>2.3.2</b> Co-expression of paralog gene pairs across tissues</a></li>
<li class="chapter" data-level="2.3.3" data-path="2-3-results.html"><a href="2-3-results.html#paralog-genes-share-enhancers"><i class="fa fa-check"></i><b>2.3.3</b> Paralog genes share enhancers</a></li>
<li class="chapter" data-level="2.3.4" data-path="2-3-results.html"><a href="2-3-results.html#co-localization-of-paralogs-in-tads"><i class="fa fa-check"></i><b>2.3.4</b> Co-localization of paralogs in TADs</a></li>
<li class="chapter" data-level="2.3.5" data-path="2-3-results.html"><a href="2-3-results.html#distal-paralog-pairs-are-enriched-for-long-range-chromatin-contacts"><i class="fa fa-check"></i><b>2.3.5</b> Distal paralog pairs are enriched for long-range chromatin contacts</a></li>
<li class="chapter" data-level="2.3.6" data-path="2-3-results.html"><a href="2-3-results.html#close-paralogs-have-fewer-contacts-than-expected"><i class="fa fa-check"></i><b>2.3.6</b> Close paralogs have fewer contacts than expected</a></li>
<li class="chapter" data-level="2.3.7" data-path="2-3-results.html"><a href="2-3-results.html#paralogs-in-mouse-and-dog-genome"><i class="fa fa-check"></i><b>2.3.7</b> Paralogs in mouse and dog genome</a></li>
<li class="chapter" data-level="2.3.8" data-path="2-3-results.html"><a href="2-3-results.html#orthologs-of-human-paralogs-show-conserved-co-localization"><i class="fa fa-check"></i><b>2.3.8</b> Orthologs of human paralogs show conserved co-localization</a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="2-4-discussions.html"><a href="2-4-discussions.html"><i class="fa fa-check"></i><b>2.4</b> Discussions</a></li>
<li class="chapter" data-level="2.5" data-path="2-5-conclusion.html"><a href="2-5-conclusion.html"><i class="fa fa-check"></i><b>2.5</b> Conclusion</a></li>
<li class="chapter" data-level="2.6" data-path="2-6-acknowledgements.html"><a href="2-6-acknowledgements.html"><i class="fa fa-check"></i><b>2.6</b> Acknowledgements</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="3-TAD-evolution.html"><a href="3-TAD-evolution.html"><i class="fa fa-check"></i><b>3</b> Stability of TADs in evolution</a><ul>
<li class="chapter" data-level="" data-path="3-TAD-evolution.html"><a href="3-TAD-evolution.html#preamble-1"><i class="fa fa-check"></i>Preamble</a></li>
<li class="chapter" data-level="" data-path="3-TAD-evolution.html"><a href="3-TAD-evolution.html#abstract-2"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="" data-path="3-TAD-evolution.html"><a href="3-TAD-evolution.html#keywords"><i class="fa fa-check"></i>Keywords</a></li>
<li class="chapter" data-level="3.1" data-path="3-1-introduction-1.html"><a href="3-1-introduction-1.html"><i class="fa fa-check"></i><b>3.1</b> Introduction</a></li>
<li class="chapter" data-level="3.2" data-path="3-2-results-1.html"><a href="3-2-results-1.html"><i class="fa fa-check"></i><b>3.2</b> Results</a><ul>
<li class="chapter" data-level="3.2.1" data-path="3-2-results-1.html"><a href="3-2-results-1.html#identification-of-evolutionary-rearrangement-breakpoints-from-whole-genome-alignments"><i class="fa fa-check"></i><b>3.2.1</b> Identification of evolutionary rearrangement breakpoints from whole-genome alignments</a></li>
<li class="chapter" data-level="3.2.2" data-path="3-2-results-1.html"><a href="3-2-results-1.html#rearrangement-breakpoints-are-enriched-at-tad-boundaries"><i class="fa fa-check"></i><b>3.2.2</b> Rearrangement breakpoints are enriched at TAD boundaries</a></li>
<li class="chapter" data-level="3.2.3" data-path="3-2-results-1.html"><a href="3-2-results-1.html#clusters-of-conserved-non-coding-elements-are-depleted-for-rearrangement-breakpoints"><i class="fa fa-check"></i><b>3.2.3</b> Clusters of conserved non-coding elements are depleted for rearrangement breakpoints</a></li>
<li class="chapter" data-level="3.2.4" data-path="3-2-results-1.html"><a href="3-2-results-1.html#rearranged-tads-are-associated-with-divergent-gene-expression-between-species"><i class="fa fa-check"></i><b>3.2.4</b> Rearranged TADs are associated with divergent gene expression between species</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="3-3-discussion.html"><a href="3-3-discussion.html"><i class="fa fa-check"></i><b>3.3</b> Discussion</a></li>
<li class="chapter" data-level="3.4" data-path="3-4-conclusion-1.html"><a href="3-4-conclusion-1.html"><i class="fa fa-check"></i><b>3.4</b> Conclusion</a></li>
<li class="chapter" data-level="3.5" data-path="3-5-methods.html"><a href="3-5-methods.html"><i class="fa fa-check"></i><b>3.5</b> Methods</a><ul>
<li class="chapter" data-level="3.5.1" data-path="3-5-methods.html"><a href="3-5-methods.html#rearrangement-breakpoints-from-whole-genome-alignments"><i class="fa fa-check"></i><b>3.5.1</b> Rearrangement breakpoints from whole-genome alignments</a></li>
<li class="chapter" data-level="3.5.2" data-path="3-5-methods.html"><a href="3-5-methods.html#topologically-associating-domains-and-contact-domains"><i class="fa fa-check"></i><b>3.5.2</b> Topologically associating domains and contact domains</a></li>
<li class="chapter" data-level="3.5.3" data-path="3-5-methods.html"><a href="3-5-methods.html#breakpoint-distributions-at-tads"><i class="fa fa-check"></i><b>3.5.3</b> Breakpoint distributions at TADs</a></li>
<li class="chapter" data-level="3.5.4" data-path="3-5-methods.html"><a href="3-5-methods.html#quantification-of-breakpoint-enrichment"><i class="fa fa-check"></i><b>3.5.4</b> Quantification of breakpoint enrichment</a></li>
<li class="chapter" data-level="3.5.5" data-path="3-5-methods.html"><a href="3-5-methods.html#expression-data-for-mouse-and-human-orthologs"><i class="fa fa-check"></i><b>3.5.5</b> Expression data for mouse and human orthologs</a></li>
<li class="chapter" data-level="3.5.6" data-path="3-5-methods.html"><a href="3-5-methods.html#classification-of-tads-and-genes-according-to-rearrangements-and-grbs"><i class="fa fa-check"></i><b>3.5.6</b> Classification of TADs and genes according to rearrangements and GRBs</a></li>
<li class="chapter" data-level="3.5.7" data-path="3-5-methods.html"><a href="3-5-methods.html#source-code-and-implementation-details"><i class="fa fa-check"></i><b>3.5.7</b> Source code and implementation details</a></li>
</ul></li>
<li class="chapter" data-level="3.6" data-path="3-6-declarations.html"><a href="3-6-declarations.html"><i class="fa fa-check"></i><b>3.6</b> Declarations</a><ul>
<li class="chapter" data-level="3.6.1" data-path="3-6-declarations.html"><a href="3-6-declarations.html#ethics-approval-and-consent-to-participate"><i class="fa fa-check"></i><b>3.6.1</b> Ethics approval and consent to participate</a></li>
<li class="chapter" data-level="3.6.2" data-path="3-6-declarations.html"><a href="3-6-declarations.html#consent-for-publication"><i class="fa fa-check"></i><b>3.6.2</b> Consent for publication</a></li>
<li class="chapter" data-level="3.6.3" data-path="3-6-declarations.html"><a href="3-6-declarations.html#availability-of-data-and-material"><i class="fa fa-check"></i><b>3.6.3</b> Availability of data and material</a></li>
<li class="chapter" data-level="3.6.4" data-path="3-6-declarations.html"><a href="3-6-declarations.html#competing-interests"><i class="fa fa-check"></i><b>3.6.4</b> Competing interests</a></li>
<li class="chapter" data-level="3.6.5" data-path="3-6-declarations.html"><a href="3-6-declarations.html#funding"><i class="fa fa-check"></i><b>3.6.5</b> Funding</a></li>
<li class="chapter" data-level="3.6.6" data-path="3-6-declarations.html"><a href="3-6-declarations.html#authors-contributions"><i class="fa fa-check"></i><b>3.6.6</b> Authors‘ contributions</a></li>
<li class="chapter" data-level="3.6.7" data-path="3-6-declarations.html"><a href="3-6-declarations.html#acknowledgments"><i class="fa fa-check"></i><b>3.6.7</b> Acknowledgments</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="4-position-effect.html"><a href="4-position-effect.html"><i class="fa fa-check"></i><b>4</b> Position effects of rearrangements in disease genomes</a><ul>
<li class="chapter" data-level="" data-path="4-position-effect.html"><a href="4-position-effect.html#preamble-2"><i class="fa fa-check"></i>Preamble</a></li>
<li class="chapter" data-level="" data-path="4-position-effect.html"><a href="4-position-effect.html#abstract-3"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="4.1" data-path="4-1-introduction-2.html"><a href="4-1-introduction-2.html"><i class="fa fa-check"></i><b>4.1</b> Introduction</a></li>
<li class="chapter" data-level="4.2" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html"><i class="fa fa-check"></i><b>4.2</b> Materials and Methods</a><ul>
<li class="chapter" data-level="4.2.1" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html#selection-of-subjects-with-apparently-balanced-chromosome-abnormalities"><i class="fa fa-check"></i><b>4.2.1</b> Selection of subjects with apparently balanced chromosome abnormalities</a></li>
<li class="chapter" data-level="4.2.2" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html#clinical-descriptions-of-dgap-cases"><i class="fa fa-check"></i><b>4.2.2</b> Clinical descriptions of DGAP cases</a></li>
<li class="chapter" data-level="4.2.3" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html#analysis-of-genes-bordering-the-rearrangement-breakpoints"><i class="fa fa-check"></i><b>4.2.3</b> Analysis of genes bordering the rearrangement breakpoints</a></li>
<li class="chapter" data-level="4.2.4" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html#assessment-of-disrupted-functional-elements-and-chromatin-interactions-bordering-rearrangement-breakpoints"><i class="fa fa-check"></i><b>4.2.4</b> Assessment of disrupted functional elements and chromatin interactions bordering rearrangement breakpoints</a></li>
<li class="chapter" data-level="4.2.5" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html#ontological-analysis-of-genes-neighboring-breakpoints"><i class="fa fa-check"></i><b>4.2.5</b> Ontological analysis of genes neighboring breakpoints</a></li>
<li class="chapter" data-level="4.2.6" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html#quantitative-real-time-pcr"><i class="fa fa-check"></i><b>4.2.6</b> Quantitative real-time PCR</a></li>
<li class="chapter" data-level="4.2.7" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html#assessment-of-dgap-breakpoints-overlapping-with-non-coding-structural-variants-in-public-databases"><i class="fa fa-check"></i><b>4.2.7</b> Assessment of DGAP breakpoints overlapping with non-coding structural variants in public databases</a></li>
</ul></li>
<li class="chapter" data-level="4.3" data-path="4-3-results-2.html"><a href="4-3-results-2.html"><i class="fa fa-check"></i><b>4.3</b> Results</a><ul>
<li class="chapter" data-level="4.3.1" data-path="4-3-results-2.html"><a href="4-3-results-2.html#genomic-characterization-of-non-coding-breakpoints"><i class="fa fa-check"></i><b>4.3.1</b> Genomic characterization of non-coding breakpoints</a></li>
<li class="chapter" data-level="4.3.2" data-path="4-3-results-2.html"><a href="4-3-results-2.html#identification-of-genes-with-potential-position-effects"><i class="fa fa-check"></i><b>4.3.2</b> Identification of genes with potential position effects</a></li>
<li class="chapter" data-level="4.3.3" data-path="4-3-results-2.html"><a href="4-3-results-2.html#identification-of-subjects-with-shared-non-coding-chromosome-alterations-and-phenotypes"><i class="fa fa-check"></i><b>4.3.3</b> Identification of subjects with shared non-coding chromosome alterations and phenotypes</a></li>
</ul></li>
<li class="chapter" data-level="4.4" data-path="4-4-discussion-1.html"><a href="4-4-discussion-1.html"><i class="fa fa-check"></i><b>4.4</b> Discussion</a><ul>
<li class="chapter" data-level="" data-path="4-4-discussion-1.html"><a href="4-4-discussion-1.html#acknowledgements-1"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="4.4.1" data-path="4-4-discussion-1.html"><a href="4-4-discussion-1.html#web-resources"><i class="fa fa-check"></i><b>4.4.1</b> Web Resources</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="5-loop.html"><a href="5-loop.html"><i class="fa fa-check"></i><b>5</b> Prediction of chromatin looping interactions</a><ul>
<li class="chapter" data-level="" data-path="5-loop.html"><a href="5-loop.html#preamble-3"><i class="fa fa-check"></i>Preamble</a></li>
<li class="chapter" data-level="" data-path="5-loop.html"><a href="5-loop.html#abstract-4"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="" data-path="5-loop.html"><a href="5-loop.html#keywords-1"><i class="fa fa-check"></i>Keywords</a></li>
<li class="chapter" data-level="5.1" data-path="5-1-introduction-3.html"><a href="5-1-introduction-3.html"><i class="fa fa-check"></i><b>5.1</b> Introduction</a></li>
<li class="chapter" data-level="5.2" data-path="5-2-results-3.html"><a href="5-2-results-3.html"><i class="fa fa-check"></i><b>5.2</b> Results</a><ul>
<li class="chapter" data-level="5.2.1" data-path="5-2-results-3.html"><a href="5-2-results-3.html#ctcf-motif-pairs-as-candidate-chromatin-loop-anchors"><i class="fa fa-check"></i><b>5.2.1</b> CTCF motif pairs as candidate chromatin loop anchors</a></li>
<li class="chapter" data-level="5.2.2" data-path="5-2-results-3.html"><a href="5-2-results-3.html#similarity-of-chip-seq-signals-at-looping-ctcf-motifs"><i class="fa fa-check"></i><b>5.2.2</b> Similarity of ChIP-seq signals at looping CTCF motifs</a></li>
<li class="chapter" data-level="5.2.3" data-path="5-2-results-3.html"><a href="5-2-results-3.html#genomic-sequence-features-of-ctcf-motif-pairs-are-associated-with-looping"><i class="fa fa-check"></i><b>5.2.3</b> Genomic sequence features of CTCF motif pairs are associated with looping</a></li>
<li class="chapter" data-level="5.2.4" data-path="5-2-results-3.html"><a href="5-2-results-3.html#chromatin-loop-prediction-using-7c"><i class="fa fa-check"></i><b>5.2.4</b> Chromatin loop prediction using 7C</a></li>
<li class="chapter" data-level="5.2.5" data-path="5-2-results-3.html"><a href="5-2-results-3.html#prediction-performance-evaluation"><i class="fa fa-check"></i><b>5.2.5</b> Prediction performance evaluation</a></li>
<li class="chapter" data-level="5.2.6" data-path="5-2-results-3.html"><a href="5-2-results-3.html#prediction-performance-of-7c-with-sequence-features-and-single-tf-chip-seq-data-sets"><i class="fa fa-check"></i><b>5.2.6</b> Prediction performance of 7C with sequence features and single TF ChIP-seq data sets</a></li>
<li class="chapter" data-level="5.2.7" data-path="5-2-results-3.html"><a href="5-2-results-3.html#comparison-of-transcription-factors-by-prediction-performance"><i class="fa fa-check"></i><b>5.2.7</b> Comparison of transcription factors by prediction performance</a></li>
<li class="chapter" data-level="5.2.8" data-path="5-2-results-3.html"><a href="5-2-results-3.html#prediction-performance-in-other-cell-types-and-for-different-tfs"><i class="fa fa-check"></i><b>5.2.8</b> Prediction performance in other cell types and for different TFs</a></li>
<li class="chapter" data-level="5.2.9" data-path="5-2-results-3.html"><a href="5-2-results-3.html#the-high-resolution-of-chip-nexus-improves-prediction-performance"><i class="fa fa-check"></i><b>5.2.9</b> The high resolution of ChIP-nexus improves prediction performance</a></li>
</ul></li>
<li class="chapter" data-level="5.3" data-path="5-3-discussion-2.html"><a href="5-3-discussion-2.html"><i class="fa fa-check"></i><b>5.3</b> Discussion</a></li>
<li class="chapter" data-level="5.4" data-path="5-4-conclusion-2.html"><a href="5-4-conclusion-2.html"><i class="fa fa-check"></i><b>5.4</b> Conclusion</a></li>
<li class="chapter" data-level="5.5" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html"><i class="fa fa-check"></i><b>5.5</b> Methods</a><ul>
<li class="chapter" data-level="5.5.1" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#ctcf-motifs-in-the-human-genome"><i class="fa fa-check"></i><b>5.5.1</b> CTCF motifs in the human genome</a></li>
<li class="chapter" data-level="5.5.2" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#loop-interaction-data-for-training-and-validation"><i class="fa fa-check"></i><b>5.5.2</b> Loop interaction data for training and validation</a></li>
<li class="chapter" data-level="5.5.3" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#chip-seq-datasets-in-gm12878-cells"><i class="fa fa-check"></i><b>5.5.3</b> ChIP-seq datasets in GM12878 cells</a></li>
<li class="chapter" data-level="5.5.4" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#chip-seq-data-types"><i class="fa fa-check"></i><b>5.5.4</b> ChIP-seq data types</a></li>
<li class="chapter" data-level="5.5.5" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#chip-nexus-data-processing-for-rad21-and-scm3"><i class="fa fa-check"></i><b>5.5.5</b> ChIP-nexus data processing for RAD21 and SCM3</a></li>
<li class="chapter" data-level="5.5.6" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#similarity-of-chip-seq-profiles-as-correlation-of-coverage-around-motifs"><i class="fa fa-check"></i><b>5.5.6</b> Similarity of ChIP-seq profiles as correlation of coverage around motifs</a></li>
<li class="chapter" data-level="5.5.7" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#genomic-sequence-features-of-chromatin-loops"><i class="fa fa-check"></i><b>5.5.7</b> Genomic sequence features of chromatin loops</a></li>
<li class="chapter" data-level="5.5.8" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#chromatin-loop-prediction-model"><i class="fa fa-check"></i><b>5.5.8</b> Chromatin Loop prediction model</a></li>
<li class="chapter" data-level="5.5.9" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#training-and-validation-of-prediction-model"><i class="fa fa-check"></i><b>5.5.9</b> Training and validation of prediction model</a></li>
<li class="chapter" data-level="5.5.10" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#analysis-of-prediction-performance"><i class="fa fa-check"></i><b>5.5.10</b> Analysis of prediction performance</a></li>
<li class="chapter" data-level="5.5.11" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#implementation-of-7c-and-compatibility-to-other-tools"><i class="fa fa-check"></i><b>5.5.11</b> Implementation of 7C and compatibility to other tools</a></li>
</ul></li>
<li class="chapter" data-level="5.6" data-path="5-6-declarations-1.html"><a href="5-6-declarations-1.html"><i class="fa fa-check"></i><b>5.6</b> Declarations</a><ul>
<li class="chapter" data-level="5.6.1" data-path="5-6-declarations-1.html"><a href="5-6-declarations-1.html#ethics-approval-and-consent-to-participate-1"><i class="fa fa-check"></i><b>5.6.1</b> Ethics approval and consent to participate</a></li>
<li class="chapter" data-level="5.6.2" data-path="5-6-declarations-1.html"><a href="5-6-declarations-1.html#consent-for-publication-1"><i class="fa fa-check"></i><b>5.6.2</b> Consent for publication</a></li>
<li class="chapter" data-level="5.6.3" data-path="5-6-declarations-1.html"><a href="5-6-declarations-1.html#availability-of-data-and-material-1"><i class="fa fa-check"></i><b>5.6.3</b> Availability of data and material</a></li>
<li class="chapter" data-level="5.6.4" data-path="5-6-declarations-1.html"><a href="5-6-declarations-1.html#competing-interests-1"><i class="fa fa-check"></i><b>5.6.4</b> Competing interests</a></li>
<li class="chapter" data-level="5.6.5" data-path="5-6-declarations-1.html"><a href="5-6-declarations-1.html#funding-1"><i class="fa fa-check"></i><b>5.6.5</b> Funding</a></li>
<li class="chapter" data-level="5.6.6" data-path="5-6-declarations-1.html"><a href="5-6-declarations-1.html#authors-contributions-1"><i class="fa fa-check"></i><b>5.6.6</b> Authors‘ contributions</a></li>
<li class="chapter" data-level="5.6.7" data-path="5-6-declarations-1.html"><a href="5-6-declarations-1.html#acknowledgments-1"><i class="fa fa-check"></i><b>5.6.7</b> Acknowledgments</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="6-thesis-discussion.html"><a href="6-thesis-discussion.html"><i class="fa fa-check"></i><b>6</b> Discussion</a><ul>
<li class="chapter" data-level="6.1" data-path="6-1-co-regulation-of-functionally-related-genes-in-tads.html"><a href="6-1-co-regulation-of-functionally-related-genes-in-tads.html"><i class="fa fa-check"></i><b>6.1</b> Co-regulation of functionally related genes in TADs</a></li>
<li class="chapter" data-level="6.2" data-path="6-2-evolution-by-gene-duplications-and-altered-regulatory-environments.html"><a href="6-2-evolution-by-gene-duplications-and-altered-regulatory-environments.html"><i class="fa fa-check"></i><b>6.2</b> Evolution by gene duplications and altered regulatory environments</a></li>
<li class="chapter" data-level="6.3" data-path="6-3-tads-are-stable-across-large-evolutionary-time-scales.html"><a href="6-3-tads-are-stable-across-large-evolutionary-time-scales.html"><i class="fa fa-check"></i><b>6.3</b> TADs are stable across large evolutionary time-scales</a></li>
<li class="chapter" data-level="6.4" data-path="6-4-TAD-disease.html"><a href="6-4-TAD-disease.html"><i class="fa fa-check"></i><b>6.4</b> Gene expression changes by altered TADs in disease</a></li>
<li class="chapter" data-level="6.5" data-path="6-5-genome-folding-data-can-explain-the-pathomechanism-of-structural-variants.html"><a href="6-5-genome-folding-data-can-explain-the-pathomechanism-of-structural-variants.html"><i class="fa fa-check"></i><b>6.5</b> Genome folding data can explain the pathomechanism of structural variants</a></li>
<li class="chapter" data-level="6.6" data-path="6-6-towards-studying-condition-specific-genome-folding.html"><a href="6-6-towards-studying-condition-specific-genome-folding.html"><i class="fa fa-check"></i><b>6.6</b> Towards studying condition specific genome folding</a><ul>
<li class="chapter" data-level="6.6.1" data-path="6-6-towards-studying-condition-specific-genome-folding.html"><a href="6-6-towards-studying-condition-specific-genome-folding.html#interpretation-of-genetic-variants-by-interacting-genes"><i class="fa fa-check"></i><b>6.6.1</b> Interpretation of genetic variants by interacting genes</a></li>
<li class="chapter" data-level="6.6.2" data-path="6-6-towards-studying-condition-specific-genome-folding.html"><a href="6-6-towards-studying-condition-specific-genome-folding.html#cell-type-specific-regulatory-interactions-between-enhancers-and-genes"><i class="fa fa-check"></i><b>6.6.2</b> Cell type-specific regulatory interactions between enhancers and genes</a></li>
<li class="chapter" data-level="6.6.3" data-path="6-6-towards-studying-condition-specific-genome-folding.html"><a href="6-6-towards-studying-condition-specific-genome-folding.html#variability-of-long-range-interactions-across-individuals"><i class="fa fa-check"></i><b>6.6.3</b> Variability of long-range interactions across individuals</a></li>
<li class="chapter" data-level="6.6.4" data-path="6-6-towards-studying-condition-specific-genome-folding.html"><a href="6-6-towards-studying-condition-specific-genome-folding.html#genome-folding-in-single-cells"><i class="fa fa-check"></i><b>6.6.4</b> Genome folding in single cells</a></li>
<li class="chapter" data-level="6.6.5" data-path="6-6-towards-studying-condition-specific-genome-folding.html"><a href="6-6-towards-studying-condition-specific-genome-folding.html#constantly-improving-targeted-experimental-methods."><i class="fa fa-check"></i><b>6.6.5</b> Constantly improving targeted experimental methods.</a></li>
</ul></li>
<li class="chapter" data-level="6.7" data-path="6-7-architectural-proteins-mediate-chromatin-looping.html"><a href="6-7-architectural-proteins-mediate-chromatin-looping.html"><i class="fa fa-check"></i><b>6.7</b> Architectural proteins mediate chromatin looping</a></li>
<li class="chapter" data-level="6.8" data-path="6-8-further-directions.html"><a href="6-8-further-directions.html"><i class="fa fa-check"></i><b>6.8</b> Further directions</a><ul>
<li class="chapter" data-level="6.8.1" data-path="6-8-further-directions.html"><a href="6-8-further-directions.html#limitation-of-the-analysis"><i class="fa fa-check"></i><b>6.8.1</b> Limitation of the analysis</a></li>
</ul></li>
<li class="chapter" data-level="6.9" data-path="6-9-conclusions.html"><a href="6-9-conclusions.html"><i class="fa fa-check"></i><b>6.9</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="appendix"><span><b>Appendix</b></span></li>
<li class="chapter" data-level="A" data-path="A-sup-paralog-regulation.html"><a href="A-sup-paralog-regulation.html"><i class="fa fa-check"></i><b>A</b> Supporting Information: Co-regulation of paralog genes</a></li>
<li class="chapter" data-level="B" data-path="B-sup-tad-evolution.html"><a href="B-sup-tad-evolution.html"><i class="fa fa-check"></i><b>B</b> Supplementary Data: Stability of TADs in evolution</a><ul>
<li class="chapter" data-level="B.1" data-path="B-1-TadEvoSupTab.html"><a href="B-1-TadEvoSupTab.html"><i class="fa fa-check"></i><b>B.1</b> Supplementary Tables</a></li>
<li class="chapter" data-level="B.2" data-path="B-2-supplementary-figures.html"><a href="B-2-supplementary-figures.html"><i class="fa fa-check"></i><b>B.2</b> Supplementary Figures</a></li>
</ul></li>
<li class="chapter" data-level="C" data-path="C-sup-position-effect.html"><a href="C-sup-position-effect.html"><i class="fa fa-check"></i><b>C</b> Supplemental Data: Position effects of rearrangements in disease genomes</a><ul>
<li class="chapter" data-level="C.1" data-path="C-1-supplemental-note.html"><a href="C-1-supplemental-note.html"><i class="fa fa-check"></i><b>C.1</b> Supplemental Note</a><ul>
<li class="chapter" data-level="C.1.1" data-path="C-1-supplemental-note.html"><a href="C-1-supplemental-note.html#case-reports"><i class="fa fa-check"></i><b>C.1.1</b> Case Reports</a></li>
<li class="chapter" data-level="C.1.2" data-path="C-1-supplemental-note.html"><a href="C-1-supplemental-note.html#nucleotide-level-nomenclature-for-dgap-karyotypes"><i class="fa fa-check"></i><b>C.1.2</b> Nucleotide Level Nomenclature for DGAP karyotypes</a></li>
</ul></li>
<li class="chapter" data-level="C.2" data-path="C-2-supplemental-figure.html"><a href="C-2-supplemental-figure.html"><i class="fa fa-check"></i><b>C.2</b> Supplemental Figure</a></li>
<li class="chapter" data-level="C.3" data-path="C-3-position-effect-sub-tab.html"><a href="C-3-position-effect-sub-tab.html"><i class="fa fa-check"></i><b>C.3</b> Supplemental Table Legends</a></li>
</ul></li>
<li class="chapter" data-level="D" data-path="D-loop-support.html"><a href="D-loop-support.html"><i class="fa fa-check"></i><b>D</b> Supplemental Information: Prediction of chromatin looping interactions</a><ul>
<li class="chapter" data-level="D.1" data-path="D-1-LoopPredSupTab.html"><a href="D-1-LoopPredSupTab.html"><i class="fa fa-check"></i><b>D.1</b> Supplementary Tables</a></li>
<li class="chapter" data-level="D.2" data-path="D-2-supplementary-figures-1.html"><a href="D-2-supplementary-figures-1.html"><i class="fa fa-check"></i><b>D.2</b> Supplementary Figures</a></li>
</ul></li>
<li class="chapter" data-level="E" data-path="E-contribution-to-individual-publications.html"><a href="E-contribution-to-individual-publications.html"><i class="fa fa-check"></i><b>E</b> Contribution to individual publications</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Build with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Genome folding in evolution and disease</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="supplemental-note" class="section level2">
<h2><span class="header-section-number">C.1</span> Supplemental Note</h2>
<div id="case-reports" class="section level3">
<h3><span class="header-section-number">C.1.1</span> Case Reports</h3>
<p><strong>DGAP017</strong></p>
<p>46,X,t(X;10)(p11.2;q24.3)dn.arr(1-22,X)x2</p>
<p>Newborn female with a bicornuate uterus, diaphragmatic hernia, thenar
hypoplasia, pulmonary hypoplasia, absent right olfactory lobe, loose
skin, scoliosis, small thorax, hypoplastic labia, right clinodactyly and
camptodactyly, as well as a scaphoid abdomen. This collection of
features was reminiscent of Fryns syndrome (FRNS [MIM: 229850]). This
case was obtained from the NIGMS Human Genetic Cell Repository at the
Coriell Institute for Medical Research (GM00972) <span class="citation">(Tang et al. <a href="#ref-Tang2013">2013</a>)</span>. An Affymetrix
Genome-Wide Human SNP Array 6.0 performed at Coriell is reportedly
normal.</p>
<p><strong>DGAP111</strong></p>
<p>46,XY,t(16;20)(q11.2;q13.2)dn.arr[hg18] 1q23.3(159763523_159905125)x3</p>
<p>Six-year-old male with congenital heart disease (one atrial septal
defect, seven small ventricular septal defects), eye anomaly (Duane
syndrome), poor growth, developmental delay, chronic constipation, left
undescended testis, history of scoliosis (resolved), history of weak
ankles and feet requiring braces (resolved), and asthma. Microarray
analysis of DNA extracted from the DGAP111 EBV-transformed cell line
contributed from DGAP to the NIGMS Human Genetic Cell Repository
(GM22709, Coriell) was performed on the Affymetrix Genome-Wide Human SNP
Array 6.0 and revealed a duplication of ~141.6 Kb in 1q23.3
(159763523-159905125) that was interpreted as likely benign.</p>
<p><strong>DGAP113</strong></p>
<p>46,XY,t(1;3)(q32.1;q13.2)dn</p>
<p>One-year-old male with bilateral congenital cataracts (TORCH screen,
positive IgG and negative IgM for rubella, cytomegalovirus, herpes
simplex virus; rubella virus isolation from urine and lens was
negative), and mild developmental delay. Cranial magnetic resonance
imaging (MRI) revealed prominent extra-axial cerebrospinal fluid spaces
of uncertain significance, and the subject has marked macrocephaly (head
circumference &gt;95th percentile) <span class="citation">(Lachke et al. <a href="#ref-Lachke2012">2012</a>)</span>. No microarray was performed.</p>
<p><strong>DGAP126</strong></p>
<p>46,XX,t(5;10)(p13.3;q21.1)dn.arr[hg18] 7q34(142030226_142154515)x1</p>
<p>Ten-year-old female with significant developmental delay with
regression, autistic tendencies, and receptive and expressive language
delay, disruptive behavior disorder, enuresis, dysthymia, sleep
disturbance, self-injurious behaviors, and agitation. She had delays in
gross and fine motor skills. No dysmorphic features were observed.
Microarray analysis of DNA extracted from the DGAP126 EBV-transformed
cell line contributed from DGAP to the NIGMS Human Genetic Cell
Repository (GM18825, Coriell) was performed on the Affymetrix
Genome-Wide Human SNP Array 6.0 and revealed a deletion of ~124.3 Kb in
region 7q34 (142030226-142154515) that was interpreted to be benign.</p>
<p><strong>DGAP138</strong></p>
<p>46,XY,t(1;6)(q23;q13)dn.arr(1-22)x2,(X,Y)x1</p>
<p>Seven-year-old male with intellectual disability, fat distribution
around trunk, gastroesophageal reflux, feeding problems (gastrostomy),
seizure disorder, movement disorder (random, writhing type movements),
wheelchair-dependence, Pierre-Robin sequence (mild micrognathia and
cleft of the soft palate) (PRBNS [MIM: 261800]), microcephaly,
pseudogynecomastia, and low growth hormone and high cortisone levels.
Normal microarray results were reported from of DNA extracted from the
DGAP138 EBV-transformed cell line contributed from DGAP to the NIGMS
Human Genetic Cell Repository (GM20568, Coriell) on the Affymetrix
Genome-Wide Human SNP Array 6.0.</p>
<p><strong>DGAP153</strong></p>
<p>46,X,t(X;17)(p11.23;p11.2)dn.arr(1-22,X)x2</p>
<p>Eight-year-old female with dysmorphic features (including mild
synophrys, a flat philtrum and thin upper lip vermilion), mild
developmental delay, sleep disturbance, and behavior problems (including
temper tantrums, self-biting, and agitation). Deletion testing was
negative for Smith-Magenis syndrome (SMS [MIM: 182290]). No cryptic
aneusomies were reported to be detected by clinical aCGH. The DGAP153
EBV-transformed cell line was contributed to the NIGMS Human Genetic
Cell Repository (GM20572, Coriell).</p>
<p><strong>DGAP163</strong></p>
<p>46,XY,t(2;14)(p23;q13)dn.arr(1-22)x2,(X,Y)x1</p>
<p>Four-year-old male with severe global developmental delay, absent
speech, dysmorphic/distinctive facies, hypospadias (repaired), seizures
as an infant (now seizure free), myopia, nystagmus, small left retinal
coloboma, and conductive hearing loss (history of otitis media). MRI
showed periventricular white matter changes of unknown origin (no record
of anoxic event), and recent electroencephalograms (EEGs) were normal.
Fluorescence <em>in situ</em> hybridization (FISH) for SMS, DiGeorge syndrome
(DGS [MIM: 188400]) and Velocardiofacial syndrome (VCFS [MIM:
192430]) was reportedly normal, as was aCGH using a 1M Agilent array
with a resolution of 6.3 Kb.</p>
<p><strong>DGAP176</strong></p>
<p>46,Y,inv(X)(q13q24)mat</p>
<p>Four-year-old male with congenital, severe, bilateral sensorineural
hearing loss, cognitive impairment, plagiocephaly, lax joints, and
coordination difficulties. Dysmorphic features include macrocephaly,
broad forehead, hypertelorism, downslanting palpebral fissures,
epicanthic folds, flat midface, rounded nasal tip, flat nasal root,
downturned corners of the mouth, simple helix of left ear, and full
lips. He also had fifth finger clinodactyly and bridged palmar creases.
No mutations were detected in the coding regions of gap junction protein
beta 2 (<em>GJB2</em> [MIM: 121011]) or gap junction protein beta 6 (<em>GJB6</em>
[MIM: 604418]). The mother is mosaic for inv(X)(q13q24) and 45,X but
is reportedly healthy <span class="citation">(Anger et al. <a href="#ref-Anger2014">2014</a>)</span>. No microarray was performed.</p>
<p><strong>DGAP249</strong></p>
<p>46,XX,t(2;11)(q33;q23)dn.arr(1-22,X)x2</p>
<p>Seven-year-old female with a history of global developmental delay. She
has gross and fine motor delays, atypical oral motor skills and limited
exploration of sensory materials. At four years she had an abnormal
sleep-deprived EEG and increased bilateral electrocortical excitability;
at six years EEG results were significantly abnormal with bifrontal
symptoms consistent with epileptiform disturbance recorded in the
interictal state. She has decreased visual motor integration, and a
composite intellectual coefficient (IQ) of 71. Normal clinical
microarray results were reported.</p>
<p><strong>DGAP252</strong></p>
<p>46,XY,t(3;18)(q13.2;q11.2)dn.arr(1-22)x2,(X,Y)x1</p>
<p>Four-month-old male whose prenatal course was complicated by
polyhydramnios with an accompanying abnormal prenatal ultrasound and
MRI, revealing an abnormal cerebellum, dilated cisterna magna, right
lung apex cyst, intra-abdominal cysts and bilateral abnormal feet.
Delivery was at term with two right posterior mediastinal cysts
identified as a foregut duplication cyst and a bronchogenic cyst by
pathology after surgical excision. Three ileal cysts were identified as
duplication cysts with complete muscularis propria, small bowel/colon,
and gastric oxyntic type mucosa by pathologic examination after
excision. Cerebellar hypoplasia was noted by MRI of his brain at one day
of age. A wide anterior fontanelle (three finger widths) was observed,
and his head was reportedly mildly turricephalic with a high forehead
and a round bony protrusion of his skull at the occipital base. Normal
clinical microarray results (CMA-HR + SNP (v.8.3)) were reported.</p>
<p><strong>DGAP275</strong></p>
<p>46,XX,t(7;12)(p13;q24.33)dn.arr(1-22,X)x2</p>
<p>Nine-year-old female with severe unexplained short stature (&lt;4 SDs)
and normal radiographs. An extensive endocrine workup revealed a normal
growth hormone axis and no evidence of precocious puberty. She was
non-dysmorphic and had normal cognitive development. A normal clinical
Affymetrix Cytoscan SNP microarray was reported.</p>
<p><strong>DGAP287</strong></p>
<p>46,XY,t(10;14)(p13;q32.1)dn.arr(1-22)x2,(X,Y)x1</p>
<p>Four-year-old male with a history of global developmental delay and
asymmetric spastic diplegia. He is ataxic, non-verbal, and drools
frequently. He is non-dysmorphic, and a brain MRI was normal. Normal
clinical Affymetrix Cytoscan HD SNP microarray results were reported.</p>
<p><strong>DGAP288</strong></p>
<p>46,XX,t(6;17)(q13;q21)dn.arr(1-22,X)x2</p>
<p>Prenatal case enrolled in study at 15 weeks, following ultrasound at 11
weeks revealing a cystic hygroma and chorionic villus sampling (CVS) at
12 weeks revealing the t(6;17) apparently balanced chromosome
translocation. Normal clinical Affymetrix Cytoscan HD SNP microarray
results were reported at 13 weeks. Micrognathia was seen on ultrasound
at 18 weeks. At 19 weeks, DGAP sequencing results revealed no genes
disrupted by the translocation, and the pregnancy was continued.
Polyhydramnios and micrognathia were noted at 28 weeks. Fetal MRI at 34
weeks revealed a small jaw index consistent with micrognathia and
retrognathia, glossoptosis, and cleft palate without cleft lip; findings
were suspicious for PRBNS. Following delivery at 39 weeks, initial exams
revealed a cleft palate. She was placed on continuous positive airway
pressure, but otherwise was considered well.</p>
<p><strong>DGAP315</strong></p>
<p>46,XX,inv(6)(p24q11)dn.arr(1-22,X)x2</p>
<p>Fifteen-year-old female with severe static encephalopathy of unknown
etiology. She uses a wheelchair, is microcephalic, nonverbal, and has
severe generalized spasticity with poorly controlled epilepsy. She had a
normal echo and eye examination and reportedly normal aCGH results.</p>
<p><strong>DGAP319</strong></p>
<p>46,XX,t(4;13)(q31.3;q14.3)dn.arr(1-22,X)x2</p>
<p>Thirteen-year-old female with intellectual disability, and height,
weight, and head circumference below the 3rd percentile. She has a grade
II-IV systolic murmur, abnormal facies, finger and toe abnormalities.
This case was obtained from the NIGMS Human Genetic Cell Repository at
the Coriell Institute for Medical Research (GM00972) <span class="citation">(Tang et al. <a href="#ref-Tang2013">2013</a>)</span>. This case was
previously reported <span class="citation">(Jenkins et al. <a href="#ref-Jenkins1975">1975</a>)</span>. The Affymetrix Genome-Wide Human SNP Array 6.0
performed at Coriell is reportedly normal.</p>
<p><strong>DGAP322</strong></p>
<p>46,XY,t(1;18)(q32.1;q22.1).arr(1-22)x2,(X,Y)x1</p>
<p>Male subject of unknown age with genitourinary malformations, third
degree hypospadias, labialized scrotum with palpable descended testes,
mild developmental delay, growth delay, and apparently intact hormonal
axis. This case was obtained from the NIGMS Human Genetic Cell
Repository at the Coriell Institute for Medical Research (GM16438) <span class="citation">(Tang et al. <a href="#ref-Tang2013">2013</a>; Frizell et al. <a href="#ref-Frizell1998">1998</a>)</span>.
The Affymetrix Genome-Wide Human SNP Array 6.0 performed at Coriell
is reportedly normal.</p>
<p><strong>DGAP329</strong></p>
<p>46,XX,t(2;14)(q21;q24.3)dn.arr[GRCh37/hg19]
18q22.3(72545050_72692202)x1 pat</p>
<p>Five-year-old female with a progressive neurologic disorder. She has
nearly constant choreoathetosis, dystonia (including painful neck
dystonia), and myoclonic movements, which are exacerbated by fatigue and
emotional stress and are worsening with time. She is profoundly
hypotonic and non-ambulatory. She is nonverbal but able to follow simple
commands. She had a reported normal clinical CytoSure ISCA 8x60K v2.0
microarray, although a paternally inherited 150 Kb deletion at 18q22.3
from her phenotypically normal father was detected.</p>
</div>
<div id="nucleotide-level-nomenclature-for-dgap-karyotypes" class="section level3">
<h3><span class="header-section-number">C.1.2</span> Nucleotide Level Nomenclature for DGAP karyotypes</h3>
<p>Karyotypes of DGAP cases are described using a revised nomenclature that
incorporates next-generation sequencing positions from Ordulu <em>et al</em>.,
2014.</p>
<p><strong>DGAP017</strong></p>
<p>46,X,t(X;10)(p11.2;q24.3)dn.arr(1-22,X)x2.seq[GRCh37/hg19]
t(X;10)(10pter-&gt;10q25.1(107,711,256)::TATCCTTTG::Xp11.22(51,702,992)-&gt;Xpter;10qter-&gt;10q25.1(107,714,387)::GAGAAAAC::Xp11.22(51,707,815)-&gt;Xqter)dn</p>
<p><strong>DGAP111</strong></p>
<p>46,XY,t(16;20)(q11.2;q13.2)dn.arr[hg18]
1q23.3(159763523_159905125)x3.seq[GRCh37/hg19]
(16,20)cx,der(16)(16pter-&gt;16q11.2(46,396,774)::16q11.2(46,397,625-46,397,900)::16q11.2(46,408,942-464093{69-70})::20q13.2(53,969,64{0-1}-53,970,162)::20q13.2(53,970,203)-&gt;20qter),der(20)(20pter-&gt;20q13.2(53,969,63{5-6})::16q11.2(46,403,29{1-2})-&gt;16qter)dn</p>
<p><strong>DGAP113</strong></p>
<p>46,XY,t(1;3)(q32.1;q13.2)dn.seq[GRCh37/hg19]
t(1;3)(1pter-&gt;1q31.3(198,076,14{1})::3q13.13(110,275,76{4})-&gt;3qter;3pter-&gt;3q13.13(110,275,769)::AGAA::1q31.3(198,076,137)-&gt;1qter)dn</p>
<p><strong>DGAP126</strong></p>
<p>46,XX,t(5;10)(p13.3;q21.1)dn.arr[hg18]
7q34(142030226_142154515)x1.seq[GRCh37/hg19]
t(5;10)(10qter-&gt;10q21.3(67,539,99{7-5})::5p13.3(29,658,44{0-2})-&gt;5qter;10pter-&gt;10q21.3(67,539,99{0})::5p13.3(29,658,42{6})-&gt;5pter)dn</p>
<p><strong>DGAP138</strong></p>
<p>46,XY,t(1;6)(q23;q13)dn.arr(1-22)x2,(X,Y)x1.seq[GRCh37/hg19]
t(1;6)(1pter-&gt;1q31.2(193,491,602)::6q16.2(100,159,181)-&gt;6qter;6pter-&gt;6q16.2(100,159,182)::A::1q31.2(193,491,602)-&gt;1qter)dn</p>
<p><strong>DGAP153</strong></p>
<p>46,X,t(X;17)(p11.23;p11.2)dn.arr(1-22,X)x2.seq[GRCh37/hg19]
t(X;17)(17pter-&gt;17p11.2(20,682,69{0-1})::Xp11.3(44,372,16{4-5})-&gt;Xqter;17qter-&gt;17p11.2(20,682,68{7-4})::Xp11.3(44,372,1{72-69})-&gt;Xpter)dn</p>
<p><strong>DGAP163</strong></p>
<p>46,XY,t(2;14)(p23;q13)dn.arr(1-22)x2,(X,Y)x1.seq[GRCh37/hg19]
t(2;14)(14qter-&gt;14q13(31,717,834)::G::2p23(39,206,240-39,206,384)::2p23(39,206,414)-&gt;2qter;14pter-&gt;14q13(31,717,73{3})::2p23(39,206,24{2})-&gt;2pter)dn</p>
<p><strong>DGAP176</strong></p>
<p>46,Y,inv(X)(q13q24)mat.seq[GRCh37/hg19]
inv(X)(pter-&gt;q13(82,275,014)::ATCAATTTA::q24q13(108,129,970-82,320,86{7-5})::q24(108,149,24{9-7})-&gt;qter)mat</p>
<p><strong>DGAP249</strong></p>
<p>46,XX,t(2;11)(q33;q23)dn.arr(1-22,X)x2.seq[GRCh37/hg19]
t(2;11)(2pter-&gt;2q33.1(199,943,78{1-9})::11q24.1(121,642,3{46-54})-&gt;11qter;11pter-&gt;11q24.1(121,638,616)::AGATCT::2q33.1(199,943,805)-&gt;2qter)dn</p>
<p><strong>DGAP252</strong></p>
<p>46,XY,t(3;18)(q13.2;q11.2)dn.arr(1-22)x2,(X,Y)x1.seq[GRCh37/hg19]
t(3;18)(3pter-&gt;3q13.11(104,627,622)::TCAATACCTTTTA::18q11.2(19,498,398)-&gt;18qter;18pter-&gt;18q11.2(19,498,400)::AAAAATGGC::3q13.11(104,627,629)-&gt;3qter)dn</p>
<p><strong>DGAP275</strong></p>
<p>46,XX,t(7;12)(p13;q24.33)dn.arr(1-22,X)x2.seq[GRCh37/hg19]
t(7;12)(12qter-&gt;12q24.33(132,983,131)::TC::7p12.3(46,111,841)-&gt;7qter;12pter-&gt;12q24.33(132,983,129)::7p12.3(46,111,839)-&gt;7pter)dn</p>
<p><strong>DGAP287</strong></p>
<p>46,XY,t(10;14)(p13;q32.1)dn.arr(1-22)x2,(X,Y)x1.seq[GRCh37/hg19]
t(10;14)(14qter-&gt;14q32.13(95,212,573)::AGTAAAGGGTTGGGTTAC::10p14(10,161,500-10,161,740)::TCG::10p14(10,161,685)-&gt;10qter;14pter-&gt;14q32.13(95,212,572)::TATCAG::10p14(10,161,498)-&gt;10pter)dn</p>
<p><strong>DGAP288</strong></p>
<p>46,XX,t(6;17)(q13;q21)dn.arr(1-22,X)x2.seq[GRCh37/hg19]
t(6;17)(6pter-&gt;6q21(112,976,04{2-4})::17q24.3(69,728,01{7-9}-&gt;17qter;17pter-&gt;17q24.3(69,728,006)::CCCTTTA::6q21(112,976,031)-&gt;6qter)dn</p>
<p><strong>DGAP315</strong></p>
<p>46,XX,inv(6)(p24q11)dn.arr(1-22,X)x2.seq[GRCh37/hg19]
inv(6)(qter-&gt;q11.1(63,115,715)::p24.3q11.1(9,394,991-63,115,685)::T::p24.3(9,394,994)-&gt;pter)dn</p>
<p><strong>DGAP319</strong></p>
<p>46,XX,t(4;13)(q31.3;q14.3)dn.arr(1-22,X)x2.seq[GRCh37/hg19]
t(4;13)(4pter-&gt;4q32.2(161,913,247)::13q21.1(59,345,837)-&gt;13qter;13pter-&gt;13q21.1(59,345,83{5-6})::4q32.2(161,913,24{7-8})-&gt;4qter)dn</p>
<p><strong>DGAP322</strong></p>
<p>46,XY,t(1;18)(q32.1;q22.1)dn.arr(1-22)x2,(X,Y)x1.seq[GRCh37/hg19]
t(1;18)(1pter-&gt;1q32.2(208,544,055)::ACTCCTCCAACTCCTATGTAGTTG::18q22.1(63,566,045)-&gt;18qter;18pter-&gt;18q22.1(63,566,053)::TACA::1q32.2(208,544,091)-&gt;1qter)dn</p>
<p><strong>DGAP329</strong></p>
<p>46,XX,t(2;14)(q21;q24.3)dn. arr[GRCh37/hg19]
18q22.3(72545050_72692202)x1 pat.seq[GRCh37/hg19]
t(2;14)(2pter-&gt;2pter-&gt;2q22.3(145,110,93{6})::14q31.1(83,574,72{4})-&gt;14qter;14pter-&gt;14q31.1(83,574,71{5-9}::2q22.3(14,511,09{37-41}-&gt;2qter)dn</p>
</div>
</div>
<h3>References</h3>
<div id="refs" class="references">
<div id="ref-Tang2013">
<p>Tang Z, Berlin DS, Toji L, Toruner GA, Beiswanger C, Kulkarni S, Martin CL, Emanuel BS, Christman M, Gerry NP. 2013. A dynamic database of microarray-characterized cell lines with various cytogenetic and genomic backgrounds. <em>G3 (Bethesda, Md)</em> <strong>3</strong>: 1143–9.</p>
</div>
<div id="ref-Lachke2012">
<p>Lachke SA, Higgins AW, Inagaki M, Saadi I, Xi Q, Long M, Quade BJ, Talkowski ME, Gusella JF, Fujimoto A, et al. 2012. The cell adhesion gene pvrl3 is associated with congenital ocular defects. <em>Human Genetics</em> <strong>131</strong>: 235–250.</p>
</div>
<div id="ref-Anger2014">
<p>Anger GJ, Crocker S, McKenzie K, Brown KK, Morton CC, Harrison K, MacKenzie JJ. 2014. X-linked deafness-2 (dfnx2) phenotype associated with a paracentric inversion upstream of &lt;i&gt;POU3F4&lt;/i&gt;. <em>American Journal of Audiology</em> <strong>23</strong>: 1.</p>
</div>
<div id="ref-Jenkins1975">
<p>Jenkins EC, Curcuru-Giordano FM, Krishna SG, Cantarella J. 1975. De novo occurrence of 46,XX,t(4;13) (q31;q14) in a mentally retarded girl. <em>Annales de genetique</em> <strong>18</strong>: 117–20.</p>
</div>
<div id="ref-Frizell1998">
<p>Frizell ER, Sutphen1 R, Diamond Jr. FB, Sherwood M, Overhauser J. 1998. T(1;18)(q32.1;q22.1) associated with genitourinary malformations. <em>Clinical Genetics</em> <strong>54</strong>: 330–333.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="C-sup-position-effect.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="C-2-supplemental-figure.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": true,
"facebook": false,
"twitter": true,
"google": false,
"weibo": false,
"instapper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/ibn-salem/thesis/edit/master/08-appendix.Rmd",
"text": "Edit"
},
"download": ["thesis.pdf", "thesis.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://cdn.bootcss.com/mathjax/2.7.1/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:" && /^https?:/.test(src))
      src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
