<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>Genome folding in evolution and disease</title>
  <meta name="description" content="This is a draft PhD thesis.">
  <meta name="generator" content="bookdown 0.6.1 and GitBook 2.6.7">

  <meta property="og:title" content="Genome folding in evolution and disease" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="This is a draft PhD thesis." />
  <meta name="github-repo" content="ibn-salem/thesis" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Genome folding in evolution and disease" />
  
  <meta name="twitter:description" content="This is a draft PhD thesis." />
  

<meta name="author" content="Jonas Ibn-Salem">


<meta name="date" content="2018-01-29">

  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  
<link rel="prev" href="C-4-supplementary-figures-1.html">
<link rel="next" href="D-contribution-to-individual-publications.html">
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />









<style type="text/css">
div.sourceCode { overflow-x: auto; }
table.sourceCode, tr.sourceCode, td.lineNumbers, td.sourceCode {
  margin: 0; padding: 0; vertical-align: baseline; border: none; }
table.sourceCode { width: 100%; line-height: 100%; }
td.lineNumbers { text-align: right; padding-right: 4px; padding-left: 4px; color: #aaaaaa; border-right: 1px solid #aaaaaa; }
td.sourceCode { padding-left: 5px; }
code > span.kw { color: #007020; font-weight: bold; } /* Keyword */
code > span.dt { color: #902000; } /* DataType */
code > span.dv { color: #40a070; } /* DecVal */
code > span.bn { color: #40a070; } /* BaseN */
code > span.fl { color: #40a070; } /* Float */
code > span.ch { color: #4070a0; } /* Char */
code > span.st { color: #4070a0; } /* String */
code > span.co { color: #60a0b0; font-style: italic; } /* Comment */
code > span.ot { color: #007020; } /* Other */
code > span.al { color: #ff0000; font-weight: bold; } /* Alert */
code > span.fu { color: #06287e; } /* Function */
code > span.er { color: #ff0000; font-weight: bold; } /* Error */
code > span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
code > span.cn { color: #880000; } /* Constant */
code > span.sc { color: #4070a0; } /* SpecialChar */
code > span.vs { color: #4070a0; } /* VerbatimString */
code > span.ss { color: #bb6688; } /* SpecialString */
code > span.im { } /* Import */
code > span.va { color: #19177c; } /* Variable */
code > span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code > span.op { color: #666666; } /* Operator */
code > span.bu { } /* BuiltIn */
code > span.ex { } /* Extension */
code > span.pp { color: #bc7a00; } /* Preprocessor */
code > span.at { color: #7d9029; } /* Attribute */
code > span.do { color: #ba2121; font-style: italic; } /* Documentation */
code > span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code > span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code > span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Thesis</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="1" data-path="1-thesis-intro.html"><a href="1-thesis-intro.html"><i class="fa fa-check"></i><b>1</b> Introduction</a><ul>
<li class="chapter" data-level="1.1" data-path="1-1-structure-of-the-intorduction.html"><a href="1-1-structure-of-the-intorduction.html"><i class="fa fa-check"></i><b>1.1</b> Structure of the Intorduction</a></li>
<li class="chapter" data-level="1.2" data-path="1-2-cell-diversity-and-regulation-of-gene-expression.html"><a href="1-2-cell-diversity-and-regulation-of-gene-expression.html"><i class="fa fa-check"></i><b>1.2</b> Cell diversity and regulation of gene expression</a></li>
<li class="chapter" data-level="1.3" data-path="1-3-enhancers.html"><a href="1-3-enhancers.html"><i class="fa fa-check"></i><b>1.3</b> Enhancers</a></li>
<li class="chapter" data-level="1.4" data-path="1-4-methods-to-probe-the-3d-chromatin-archtiecture.html"><a href="1-4-methods-to-probe-the-3d-chromatin-archtiecture.html"><i class="fa fa-check"></i><b>1.4</b> Methods to probe the 3D chromatin archtiecture</a><ul>
<li class="chapter" data-level="1.4.1" data-path="1-4-methods-to-probe-the-3d-chromatin-archtiecture.html"><a href="1-4-methods-to-probe-the-3d-chromatin-archtiecture.html#microscopy-based-techniques-to-visuallize-the-genome-in-3d"><i class="fa fa-check"></i><b>1.4.1</b> Microscopy-based techniques to visuallize the genome in 3D</a></li>
<li class="chapter" data-level="1.4.2" data-path="1-4-methods-to-probe-the-3d-chromatin-archtiecture.html"><a href="1-4-methods-to-probe-the-3d-chromatin-archtiecture.html#proximity-ligation-based-method-to-quantify-chromatin-interactions"><i class="fa fa-check"></i><b>1.4.2</b> Proximity-ligation based method to quantify chromatin interactions</a></li>
</ul></li>
<li class="chapter" data-level="1.5" data-path="1-5-hierarchy-of-chromatin-3d-structure.html"><a href="1-5-hierarchy-of-chromatin-3d-structure.html"><i class="fa fa-check"></i><b>1.5</b> Hierarchy of chromatin 3D structure</a><ul>
<li class="chapter" data-level="1.5.1" data-path="1-5-hierarchy-of-chromatin-3d-structure.html"><a href="1-5-hierarchy-of-chromatin-3d-structure.html#chromosomal-territories-and-inter-chromosomal-contacts"><i class="fa fa-check"></i><b>1.5.1</b> Chromosomal territories and inter-chromosomal contacts</a></li>
<li class="chapter" data-level="1.5.2" data-path="1-5-hierarchy-of-chromatin-3d-structure.html"><a href="1-5-hierarchy-of-chromatin-3d-structure.html#ab-compartments"><i class="fa fa-check"></i><b>1.5.2</b> A/B compartments</a></li>
<li class="chapter" data-level="1.5.3" data-path="1-5-hierarchy-of-chromatin-3d-structure.html"><a href="1-5-hierarchy-of-chromatin-3d-structure.html#topologically-associating-domains-tads"><i class="fa fa-check"></i><b>1.5.3</b> Topologically associating domains (TADs)</a></li>
<li class="chapter" data-level="1.5.4" data-path="1-5-hierarchy-of-chromatin-3d-structure.html"><a href="1-5-hierarchy-of-chromatin-3d-structure.html#chromatin-loops"><i class="fa fa-check"></i><b>1.5.4</b> Chromatin loops</a></li>
</ul></li>
<li class="chapter" data-level="1.6" data-path="1-6-dynamics-of-chromatin-structure.html"><a href="1-6-dynamics-of-chromatin-structure.html"><i class="fa fa-check"></i><b>1.6</b> Dynamics of chromatin structure</a></li>
<li class="chapter" data-level="1.7" data-path="1-7-changes-of-genome-folding-in-evoltuion-and-disease-genomes.html"><a href="1-7-changes-of-genome-folding-in-evoltuion-and-disease-genomes.html"><i class="fa fa-check"></i><b>1.7</b> Changes of genome folding in evoltuion and disease genomes</a></li>
<li class="chapter" data-level="1.8" data-path="1-8-aims-of-this-thesis.html"><a href="1-8-aims-of-this-thesis.html"><i class="fa fa-check"></i><b>1.8</b> Aims of this thesis</a><ul>
<li class="chapter" data-level="" data-path="1-8-aims-of-this-thesis.html"><a href="1-8-aims-of-this-thesis.html#is-the-three-dimensional-folding-of-genomes-associated-with-co-regulation-of-functionally-related-genes"><i class="fa fa-check"></i>Is the three-dimensional folding of genomes associated with co-regulation of functionally related genes?Â </a></li>
<li class="chapter" data-level="" data-path="1-8-aims-of-this-thesis.html"><a href="1-8-aims-of-this-thesis.html#are-tads-functional-building-blocks-of-genomes-and-subjected-to-selective-pressure-during-evolution"><i class="fa fa-check"></i>Are TADs functional building blocks of genomes and subjected to selective pressure during evolution?</a></li>
<li class="chapter" data-level="" data-path="1-8-aims-of-this-thesis.html"><a href="1-8-aims-of-this-thesis.html#is-the-disruption-of-tads-by-rearrangements-also-associated-to-genetic-diseases"><i class="fa fa-check"></i>Is the disruption of TADs by rearrangements also associated to genetic diseases?</a></li>
<li class="chapter" data-level="" data-path="1-8-aims-of-this-thesis.html"><a href="1-8-aims-of-this-thesis.html#can-chromatin-looping-interactions-be-predicted-by-chip-seq-and-sequence-features"><i class="fa fa-check"></i>Can chromatin looping interactions be predicted by ChIP-seq and sequence features?</a></li>
</ul></li>
<li class="chapter" data-level="1.9" data-path="1-9-structure-of-this-thesis.html"><a href="1-9-structure-of-this-thesis.html"><i class="fa fa-check"></i><b>1.9</b> Structure of this thesis</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="2-paralog-regulation.html"><a href="2-paralog-regulation.html"><i class="fa fa-check"></i><b>2</b> Paralog genes in the 3D genome architecture</a><ul>
<li class="chapter" data-level="" data-path="2-paralog-regulation.html"><a href="2-paralog-regulation.html#preamble"><i class="fa fa-check"></i>Preamble</a></li>
<li class="chapter" data-level="" data-path="2-paralog-regulation.html"><a href="2-paralog-regulation.html#abstract-1"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="2.0.1" data-path="2-paralog-regulation.html"><a href="2-paralog-regulation.html#selection-of-pairs-of-paralog-genes"><i class="fa fa-check"></i><b>2.0.1</b> Selection of pairs of paralog genes</a></li>
<li class="chapter" data-level="2.0.2" data-path="2-paralog-regulation.html"><a href="2-paralog-regulation.html#enhancers-to-gene-association"><i class="fa fa-check"></i><b>2.0.2</b> Enhancers to gene association</a></li>
<li class="chapter" data-level="2.0.3" data-path="2-paralog-regulation.html"><a href="2-paralog-regulation.html#topological-associating-domains"><i class="fa fa-check"></i><b>2.0.3</b> Topological associating domains</a></li>
<li class="chapter" data-level="2.0.4" data-path="2-paralog-regulation.html"><a href="2-paralog-regulation.html#hi-c-interaction-maps"><i class="fa fa-check"></i><b>2.0.4</b> Hi-C interaction maps</a></li>
<li class="chapter" data-level="2.0.5" data-path="2-paralog-regulation.html"><a href="2-paralog-regulation.html#randomization"><i class="fa fa-check"></i><b>2.0.5</b> Randomization</a></li>
<li class="chapter" data-level="2.0.6" data-path="2-paralog-regulation.html"><a href="2-paralog-regulation.html#statistical-tests"><i class="fa fa-check"></i><b>2.0.6</b> Statistical tests</a></li>
<li class="chapter" data-level="2.0.7" data-path="2-paralog-regulation.html"><a href="2-paralog-regulation.html#distribution-of-paralog-genes-in-the-human-genome"><i class="fa fa-check"></i><b>2.0.7</b> Distribution of paralog genes in the human genome</a></li>
<li class="chapter" data-level="2.0.8" data-path="2-paralog-regulation.html"><a href="2-paralog-regulation.html#co-expression-of-paralog-gene-pairs-across-tissues"><i class="fa fa-check"></i><b>2.0.8</b> Co-expression of paralog gene pairs across tissues</a></li>
<li class="chapter" data-level="2.0.9" data-path="2-paralog-regulation.html"><a href="2-paralog-regulation.html#paralog-genes-share-enhancers"><i class="fa fa-check"></i><b>2.0.9</b> Paralog genes share enhancers</a></li>
<li class="chapter" data-level="2.0.10" data-path="2-paralog-regulation.html"><a href="2-paralog-regulation.html#co-localization-of-paralogs-in-tads"><i class="fa fa-check"></i><b>2.0.10</b> Co-localization of paralogs in TADs</a></li>
<li class="chapter" data-level="2.0.11" data-path="2-paralog-regulation.html"><a href="2-paralog-regulation.html#distal-paralog-pairs-are-enriched-for-long-range-chromatin-contacts"><i class="fa fa-check"></i><b>2.0.11</b> Distal paralog pairs are enriched for long-range chromatin contacts</a></li>
<li class="chapter" data-level="2.0.12" data-path="2-paralog-regulation.html"><a href="2-paralog-regulation.html#close-paralogs-have-fewer-contacts-than-expected"><i class="fa fa-check"></i><b>2.0.12</b> Close paralogs have fewer contacts than expected</a></li>
<li class="chapter" data-level="2.0.13" data-path="2-paralog-regulation.html"><a href="2-paralog-regulation.html#paralogs-in-mouse-and-dog-genome"><i class="fa fa-check"></i><b>2.0.13</b> Paralogs in mouse and dog genome</a></li>
<li class="chapter" data-level="2.0.14" data-path="2-paralog-regulation.html"><a href="2-paralog-regulation.html#orthologs-of-human-paralogs-show-conserved-co-localization"><i class="fa fa-check"></i><b>2.0.14</b> Orthologs of human paralogs show conserved co-localization</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="3-TAD-evolution.html"><a href="3-TAD-evolution.html"><i class="fa fa-check"></i><b>3</b> Stability of TADs in evolution</a><ul>
<li class="chapter" data-level="" data-path="3-TAD-evolution.html"><a href="3-TAD-evolution.html#preamble-1"><i class="fa fa-check"></i>Preamble</a></li>
<li class="chapter" data-level="" data-path="3-TAD-evolution.html"><a href="3-TAD-evolution.html#abstract-2"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="" data-path="3-TAD-evolution.html"><a href="3-TAD-evolution.html#keywords"><i class="fa fa-check"></i>Keywords</a></li>
<li class="chapter" data-level="3.1" data-path="3-1-introduction.html"><a href="3-1-introduction.html"><i class="fa fa-check"></i><b>3.1</b> Introduction</a></li>
<li class="chapter" data-level="3.2" data-path="3-2-results.html"><a href="3-2-results.html"><i class="fa fa-check"></i><b>3.2</b> Results</a><ul>
<li class="chapter" data-level="3.2.1" data-path="3-2-results.html"><a href="3-2-results.html#identification-of-evolutionary-rearrangement-breakpoints-from-whole-genome-alignments"><i class="fa fa-check"></i><b>3.2.1</b> Identification of evolutionary rearrangement breakpoints from whole-genome alignments</a></li>
<li class="chapter" data-level="3.2.2" data-path="3-2-results.html"><a href="3-2-results.html#rearrangement-breakpoints-are-enriched-at-tad-boundaries"><i class="fa fa-check"></i><b>3.2.2</b> Rearrangement breakpoints are enriched at TAD boundaries</a></li>
<li class="chapter" data-level="3.2.3" data-path="3-2-results.html"><a href="3-2-results.html#clusters-of-conserved-non-coding-elements-are-depleted-for-rearrangement-breakpoints"><i class="fa fa-check"></i><b>3.2.3</b> Clusters of conserved non-coding elements are depleted for rearrangement breakpoints</a></li>
<li class="chapter" data-level="3.2.4" data-path="3-2-results.html"><a href="3-2-results.html#rearranged-tads-are-associated-with-divergent-gene-expression-between-species"><i class="fa fa-check"></i><b>3.2.4</b> Rearranged TADs are associated with divergent gene expression between species</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="3-3-discussion.html"><a href="3-3-discussion.html"><i class="fa fa-check"></i><b>3.3</b> Discussion</a></li>
<li class="chapter" data-level="3.4" data-path="3-4-conclusion.html"><a href="3-4-conclusion.html"><i class="fa fa-check"></i><b>3.4</b> Conclusion</a></li>
<li class="chapter" data-level="3.5" data-path="3-5-methods.html"><a href="3-5-methods.html"><i class="fa fa-check"></i><b>3.5</b> Methods</a><ul>
<li class="chapter" data-level="3.5.1" data-path="3-5-methods.html"><a href="3-5-methods.html#rearrangement-breakpoints-from-whole-genome-alignments"><i class="fa fa-check"></i><b>3.5.1</b> Rearrangement breakpoints from whole-genome alignments</a></li>
<li class="chapter" data-level="3.5.2" data-path="3-5-methods.html"><a href="3-5-methods.html#topologically-associating-domains-and-contact-domains"><i class="fa fa-check"></i><b>3.5.2</b> Topologically associating domains and contact domains</a></li>
<li class="chapter" data-level="3.5.3" data-path="3-5-methods.html"><a href="3-5-methods.html#breakpoint-distributions-at-tads"><i class="fa fa-check"></i><b>3.5.3</b> Breakpoint distributions at TADs</a></li>
<li class="chapter" data-level="3.5.4" data-path="3-5-methods.html"><a href="3-5-methods.html#quantification-of-breakpoint-enrichment"><i class="fa fa-check"></i><b>3.5.4</b> Quantification of breakpoint enrichment</a></li>
<li class="chapter" data-level="3.5.5" data-path="3-5-methods.html"><a href="3-5-methods.html#expression-data-for-mouse-and-human-orthologs"><i class="fa fa-check"></i><b>3.5.5</b> Expression data for mouse and human orthologs</a></li>
<li class="chapter" data-level="3.5.6" data-path="3-5-methods.html"><a href="3-5-methods.html#classification-of-tads-and-genes-according-to-rearrangements-and-grbs"><i class="fa fa-check"></i><b>3.5.6</b> Classification of TADs and genes according to rearrangements and GRBs</a></li>
<li class="chapter" data-level="3.5.7" data-path="3-5-methods.html"><a href="3-5-methods.html#source-code-and-implementation-details"><i class="fa fa-check"></i><b>3.5.7</b> Source code and implementation details</a></li>
</ul></li>
<li class="chapter" data-level="3.6" data-path="3-6-declarations.html"><a href="3-6-declarations.html"><i class="fa fa-check"></i><b>3.6</b> Declarations</a><ul>
<li class="chapter" data-level="3.6.1" data-path="3-6-declarations.html"><a href="3-6-declarations.html#ethics-approval-and-consent-to-participate"><i class="fa fa-check"></i><b>3.6.1</b> Ethics approval and consent to participate</a></li>
<li class="chapter" data-level="3.6.2" data-path="3-6-declarations.html"><a href="3-6-declarations.html#consent-for-publication"><i class="fa fa-check"></i><b>3.6.2</b> Consent for publication</a></li>
<li class="chapter" data-level="3.6.3" data-path="3-6-declarations.html"><a href="3-6-declarations.html#availability-of-data-and-material"><i class="fa fa-check"></i><b>3.6.3</b> Availability of data and material</a></li>
<li class="chapter" data-level="3.6.4" data-path="3-6-declarations.html"><a href="3-6-declarations.html#competing-interests"><i class="fa fa-check"></i><b>3.6.4</b> Competing interests</a></li>
<li class="chapter" data-level="3.6.5" data-path="3-6-declarations.html"><a href="3-6-declarations.html#funding"><i class="fa fa-check"></i><b>3.6.5</b> Funding</a></li>
<li class="chapter" data-level="3.6.6" data-path="3-6-declarations.html"><a href="3-6-declarations.html#authors-contributions"><i class="fa fa-check"></i><b>3.6.6</b> Authorsâ contributions</a></li>
<li class="chapter" data-level="3.6.7" data-path="3-6-declarations.html"><a href="3-6-declarations.html#acknowledgments"><i class="fa fa-check"></i><b>3.6.7</b> Acknowledgments</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="4-position-effect.html"><a href="4-position-effect.html"><i class="fa fa-check"></i><b>4</b> Position effects of rearrangements in disease genomes</a><ul>
<li class="chapter" data-level="" data-path="4-position-effect.html"><a href="4-position-effect.html#preamble-2"><i class="fa fa-check"></i>Preamble</a></li>
<li class="chapter" data-level="" data-path="4-position-effect.html"><a href="4-position-effect.html#abstract-3"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="4.1" data-path="4-1-introduction-1.html"><a href="4-1-introduction-1.html"><i class="fa fa-check"></i><b>4.1</b> Introduction</a></li>
<li class="chapter" data-level="4.2" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html"><i class="fa fa-check"></i><b>4.2</b> Materials and Methods</a><ul>
<li class="chapter" data-level="4.2.1" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html#selection-of-subjects-with-apparently-balanced-chromosome-abnormalities"><i class="fa fa-check"></i><b>4.2.1</b> Selection of subjects with apparently balanced chromosome abnormalities</a></li>
<li class="chapter" data-level="4.2.2" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html#clinical-descriptions-of-dgap-cases"><i class="fa fa-check"></i><b>4.2.2</b> Clinical descriptions of DGAP cases</a></li>
<li class="chapter" data-level="4.2.3" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html#analysis-of-genes-bordering-the-rearrangement-breakpoints"><i class="fa fa-check"></i><b>4.2.3</b> Analysis of genes bordering the rearrangement breakpoints</a></li>
<li class="chapter" data-level="4.2.4" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html#assessment-of-disrupted-functional-elements-and-chromatin-interactions-bordering-rearrangement-breakpoints"><i class="fa fa-check"></i><b>4.2.4</b> Assessment of disrupted functional elements and chromatin interactions bordering rearrangement breakpoints</a></li>
<li class="chapter" data-level="4.2.5" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html#ontological-analysis-of-genes-neighboring-breakpoints"><i class="fa fa-check"></i><b>4.2.5</b> Ontological analysis of genes neighboring breakpoints</a></li>
<li class="chapter" data-level="4.2.6" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html#quantitative-real-time-pcr"><i class="fa fa-check"></i><b>4.2.6</b> Quantitative real-time PCR</a></li>
<li class="chapter" data-level="4.2.7" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html#assessment-of-dgap-breakpoints-overlapping-with-non-coding-structural-variants-in-public-databases"><i class="fa fa-check"></i><b>4.2.7</b> Assessment of DGAP breakpoints overlapping with non-coding structural variants in public databases</a></li>
</ul></li>
<li class="chapter" data-level="4.3" data-path="4-3-results-1.html"><a href="4-3-results-1.html"><i class="fa fa-check"></i><b>4.3</b> Results</a><ul>
<li class="chapter" data-level="4.3.1" data-path="4-3-results-1.html"><a href="4-3-results-1.html#genomic-characterization-of-non-coding-breakpoints"><i class="fa fa-check"></i><b>4.3.1</b> Genomic characterization of non-coding breakpoints</a></li>
<li class="chapter" data-level="4.3.2" data-path="4-3-results-1.html"><a href="4-3-results-1.html#identification-of-genes-with-potential-position-effects"><i class="fa fa-check"></i><b>4.3.2</b> Identification of genes with potential position effects</a></li>
<li class="chapter" data-level="4.3.3" data-path="4-3-results-1.html"><a href="4-3-results-1.html#identification-of-subjects-with-shared-non-coding-chromosome-alterations-and-phenotypes"><i class="fa fa-check"></i><b>4.3.3</b> Identification of subjects with shared non-coding chromosome alterations and phenotypes</a></li>
</ul></li>
<li class="chapter" data-level="4.4" data-path="4-4-discussion-1.html"><a href="4-4-discussion-1.html"><i class="fa fa-check"></i><b>4.4</b> Discussion</a><ul>
<li class="chapter" data-level="" data-path="4-4-discussion-1.html"><a href="4-4-discussion-1.html#acknowledgements"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="4.4.1" data-path="4-4-discussion-1.html"><a href="4-4-discussion-1.html#web-resources"><i class="fa fa-check"></i><b>4.4.1</b> Web Resources</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="5-loop-prediction.html"><a href="5-loop-prediction.html"><i class="fa fa-check"></i><b>5</b> Prediction of chromatin looping interactions</a></li>
<li class="chapter" data-level="6" data-path="6-thesis-discussion.html"><a href="6-thesis-discussion.html"><i class="fa fa-check"></i><b>6</b> Discussion</a><ul>
<li class="chapter" data-level="6.1" data-path="6-1-discussion-on-tad-evolution.html"><a href="6-1-discussion-on-tad-evolution.html"><i class="fa fa-check"></i><b>6.1</b> Discussion on TAD evolution</a></li>
<li class="chapter" data-level="6.2" data-path="6-2-further-directions.html"><a href="6-2-further-directions.html"><i class="fa fa-check"></i><b>6.2</b> Further directions</a></li>
<li class="chapter" data-level="6.3" data-path="6-3-conclusions.html"><a href="6-3-conclusions.html"><i class="fa fa-check"></i><b>6.3</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="appendix"><span><b>Appendix</b></span></li>
<li class="chapter" data-level="A" data-path="A-sup-paralog-regulation.html"><a href="A-sup-paralog-regulation.html"><i class="fa fa-check"></i><b>A</b> Supporting Information: Co-regulation of paralog genes in the three-dimensional chromatin architecture</a></li>
<li class="chapter" data-level="B" data-path="B-sup-TAD-evolution.html"><a href="B-sup-TAD-evolution.html"><i class="fa fa-check"></i><b>B</b> Supplementary Data: Evolutionary stability of topologically associating domains is associated with conserved gene regulation</a><ul>
<li class="chapter" data-level="B.1" data-path="B-1-TadEvoSupTab.html"><a href="B-1-TadEvoSupTab.html"><i class="fa fa-check"></i><b>B.1</b> Supplementary Tables</a></li>
<li class="chapter" data-level="B.2" data-path="B-2-supplementary-figures.html"><a href="B-2-supplementary-figures.html"><i class="fa fa-check"></i><b>B.2</b> Supplementary Figures</a></li>
</ul></li>
<li class="chapter" data-level="C" data-path="C-sup-position-effect.html"><a href="C-sup-position-effect.html"><i class="fa fa-check"></i><b>C</b> Supplemental Data: Computational Prediction of Position Effects of Apparently Balanced Human Chromosomal Rearrangements</a><ul>
<li class="chapter" data-level="C.1" data-path="C-1-supplemental-note-case-reports.html"><a href="C-1-supplemental-note-case-reports.html"><i class="fa fa-check"></i><b>C.1</b> Supplemental Note: Case Reports</a></li>
<li class="chapter" data-level="C.2" data-path="C-2-supplemental-note-nucleotide-level-nomenclature-for-dgap-karyotypes.html"><a href="C-2-supplemental-note-nucleotide-level-nomenclature-for-dgap-karyotypes.html"><i class="fa fa-check"></i><b>C.2</b> Supplemental Note: Nucleotide Level Nomenclature for DGAP karyotypes</a></li>
<li class="chapter" data-level="C.3" data-path="C-3-supplemental-figure.html"><a href="C-3-supplemental-figure.html"><i class="fa fa-check"></i><b>C.3</b> Supplemental Figure</a></li>
<li class="chapter" data-level="C.4" data-path="C-4-supplementary-figures-1.html"><a href="C-4-supplementary-figures-1.html"><i class="fa fa-check"></i><b>C.4</b> Supplementary Figures</a></li>
<li class="chapter" data-level="C.5" data-path="C-5-supplemental-table-legends.html"><a href="C-5-supplemental-table-legends.html"><i class="fa fa-check"></i><b>C.5</b> Supplemental Table Legends</a><ul>
<li class="chapter" data-level="C.5.1" data-path="C-5-supplemental-table-legends.html"><a href="C-5-supplemental-table-legends.html#table-s1."><i class="fa fa-check"></i><b>C.5.1</b> Table S1.</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="D" data-path="D-contribution-to-individual-publications.html"><a href="D-contribution-to-individual-publications.html"><i class="fa fa-check"></i><b>D</b> Contribution to individual publications</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Build with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Genome folding in evolution and disease</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="supplemental-table-legends" class="section level2">
<h2><span class="header-section-number">C.5</span> Supplemental Table Legends</h2>
<div id="table-s1." class="section level3">
<h3><span class="header-section-number">C.5.1</span> Table S1.</h3>
<p><strong>Table describing the 17 cases with both breakpoints in non-coding regions.</strong> Case identifiers are provided per studied subject (Subject ID), in addition to their karyotypes using the International System for Human Cytogenetic Nomenclature (ISCN2016) and array information reported in hg19 unless otherwise stated in hg18. Each case has two reported breakpoints (A and B), and for each we provide cytogenetic band and nucleotide locations in hg19 coordinates for the derivative chromosomes involved in their generation (der(A) and der(B)). We also report the sequencing reads by which the breakpoints were identified, and the overlap with known annotated genes (Disrupted gene 1 and Disrupted Gene 2), as well as the two nearest genes (Closest Gene 1 and Closest Gene 2) and their distance in base pairs (bp) to the breakpoint locations (Distance to gene 1 and Distance to Gene 2) in the derivative chromosomes. Negative distance numbers indicate genes upstream of the breakpoint position, while positive numbers indicate genes located downstream of the breakpoint.</p>
<p><strong>Table S2. Overlap of non-coding DGAP breakpoint positions with gene promoters.</strong> Table reporting the number of annotated Ensembl GRCh37 gene promoters (Ensembl_GRCh37_promoters) that overlap non-coding DGAP breakpoints (DGAP id, chr, start, end).</p>
<p><strong>Table S3. Overlap of non-coding DGAP breakpoint positions with transcription factor binding sites.</strong> Table reporting the number of annotated Ensembl GRCh37 transcription factor binding sites (Ensembl_GRCh37_tfbindingsites) that overlap non-coding DGAP breakpoints (DGAP id, chr, start, end).</p>
<p><strong>Table S4. Overlap of non-coding DGAP breakpoint positions with enhancers.</strong> Table reporting the number of primary cell (Primary_cell_enhancers), tissue (Tissue_enhancers), H1-ESC (ChromHMM_H1_ESC_enhancers), GM12878 (ChromHMM_GM12878_enhancers), and VISTA (VISTA_db_hg19) enhancers that overlap non-coding DGAP breakpoints (DGAP id, chr, start, end). Enhancer positions were obtained from Andersson <em>et al</em>., 2014, ENCODE, and the VISTA enhancer database human version hg19. Highlighted green rows indicate breakpoints which overlapped one or more of the enhancer categories analyzed.</p>
<p><strong>Table S5. Overlap of non-coding DGAP breakpoint positions with DNaseI hypersensitive sites.</strong> Table reporting the number of DNaseI hypersensitive sites from H1-hESC, GM06990, GM12878, and the master table (a compilation of 125 cell lines DNaseI clusters) from ENCODE that overlap non-coding DGAP breakpoints (DGAP id, chr, start, end). Highlighted green rows indicate breakpoints which overlapped one or more of the DNaseI hypersensitive sites in the different cell lines analyzed.</p>
<p><strong>Table S6. Overlap of non-coding DGAP breakpoint positions with CTCF binding sites.</strong> Table reporting the number of ENCODE CTCF binding sites from H1-hESC and GM12878 that overlap non-coding DGAP breakpoints (DGAP id, chr, start, end). Highlighted green rows indicate breakpoints which overlapped one or more of the CTCF binding sites in the two cell lines analyzed.</p>
<p><strong>Table S7. Overlap of non-coding DGAP breakpoint positions with ENCODE chromatin state segments.</strong> Table reporting the ENCODE chromatin state segment classifications per non-coding DGAP breakpoint (DGAP id, chr, start, end) for H1-hESC and GM12878 cell lines. Chromatin state segment coordinates and other bed file information is displayed starting from column #bin until column itemRGB. Please refer to ENCODEâs bed items description from here: <a href="http://rohsdb.cmb.usc.edu/GBshape/cgi-bin/hgTables" class="uri">http://rohsdb.cmb.usc.edu/GBshape/cgi-bin/hgTables</a>. Chromatin state names CTCF = CTCF binding site, E = enhancer, WE = weak enhancer, T = transcriptionally active, R = transcriptionally repressed.</p>
<p><strong>Table S8. Overlap of non-coding DGAP breakpoint positions with repetitive elements.</strong> Table reporting the number of repetitive elements as assessed by Repeat Masker that overlap non-coding DGAP breakpoints (DGAP id, chr, start, end). Repetitive elements information such as coordinates (Rep_chr, Rep_start, Rep_end), name, class and family are provided for each overlap.</p>
<p><strong>Table S9. Overlap of non-coding DGAP breakpoint positions with topologically associating domains (TADs).</strong> Table reporting the number of TADs in H1-hESC and IMR90 (Dixon <em>et al</em>., 2012) that overlap non-coding DGAP breakpoints (DGAP id, chr, start, end). TAD information such as coordinates (TAD_chr, TAD_start, TAD_end) are provided for each overlap.</p>
<p><strong>Table S10. Overlap of non-coding DGAP breakpoint positions with high-resolution chromatin subcompartments and arrowhead domains.</strong> Table reporting the number of high-resolution chromatin subcompartments and arrowhead domains in IMR90 and GM12878 (Rao <em>et al</em>., 2014) that overlap non-coding DGAP breakpoints (DGAP id, chr, start, end). Chromatin subcompartments and arrowhead domains information such as coordinates and class are provided for each overlap.</p>
<p><strong>Table S11. Disruption of chromatin contacts by non-coding DGAP breakpoint positions.</strong> Table reporting the number of chromatin contacts disrupted by non-coding DGAP breakpoint positions (DGAP id, chr, start, end) within their analysis windows (window_start, window_end) in Hi-C datasets of 20 and 40 Kb resolution of H1-hESC (Dixon <em>et al</em>., 2012) (Esc_20kb_HindIII_rep1, Esc_20kb_HindIII_rep2, Esc_40kb_hindIII_combined, Esc_40kb_hindIII_rep1, Esc_40kb_hindIII_rep2), 20 and 40 Kb resolution of IMR90 (Dixon <em>et al</em>., 2012) (IMR90_20kb_hindIII_rep1, IMR90_20kb_hindIII_rep2, IMR90_40kb_hindIII_combined, IMR90_40kb_hindIII_rep1, IMR90_40kb_hindIII_rep2), 100Kb and 1Mb resolution of GM06990 (<a href="http://epigenomegateway.wustl.edu/" class="uri">http://epigenomegateway.wustl.edu/</a>) (GM06990_obsexp_100kb, GM06990_obsexp_1mb) and looplists from Rao <em>et al</em>., 2014 for GM12878 and IMR90 (GSE63525_GM12878_primary+replicate_HiCCUPS_looplist, GSE63525_IMR90_HiCCUPS_looplist).</p>
<p><strong>Table S12. Disruption of GM12878 chromatin contacts at various resolution levels by non-coding DGAP breakpoint positions.</strong> Table reporting the number of chromatin contacts disrupted by non-coding DGAP breakpoint positions (DGAP id, chr, start, end) within their analysis windows (window_start, window_end) in the 50Kb, 100Kb, 250Kb, 500Kb and 1Mb resolution Hi-C datasets from Rao <em>et al</em>., 2014 for GM12878.</p>
<p><strong>Table S13. Disruption of predicted disrupted ENCODE distal DHS/enhancerâpromoter connections by non-coding DGAP breakpoint positions.</strong> Table reporting the number of predicted ENCODE distal DHS/enhancerâpromoter connections (Thurman <em>et al</em>., 2012) (promoter_DHS_chr, promoter_DHS_start, promoter_DHS_end, promoter_DHS_gene, distal_DHS_chr, distal_DHS_start, distal_DHS_end, promoter_distal_DHS_correlation) by non-coding DGAP breakpoint positions (DGAP id, chr, start, end) within their Â±500 Kb analysis windows (window_start, window_end).</p>
<p><strong>Table S14. Genes with predicted disrupted ENCODE distal DHS/enhancerâpromoter connections by the non-coding DGAP breakpoint positions.</strong> Table reporting the names of genes (Genes) separated from their predicted enhancers (Disrupted_enh_prom_interactions) (Thurman <em>et al</em>., 2012).</p>
<p><strong>Table S15. Identification of genes with potential position effects.</strong> Table reporting the candidate genes (ensembl_gene_ID, Gene_chr, Gene_start, Gene_end, Gene_name) and their various lines of selection evidence for each non-coding DGAP breakpoint position (DGAP id, chr, start, end) within their analysis windows (window_start, window_end). Evidence lines include Hi-C domain inclusion (Hi_domain, HiC_chr, HiC_start, HiC_end), haploinsufficiency (HI_chr, Gene-start, gene_end, HI_prob, Haploinsufficiency_score,), triplosensitivity (Triplosensitivity_score), phenomatch score (PhenoScore, MaxPhenoScore, Phone_percentile, count_Pheno_percentile, MaxPheno_percentile, count_MaxPheno_percentile, Percentile_final_count). All of the evidence information is summarized (6Mb, 2Mb, TAD, DHS, Count_haplo, count_triplo) and the gene rankings are presented in the PERC+DHS+TAD+HAPLO+TRIPLO and PERC+DHS+2Mb+HAPLO+TRIPLO columns which take different evidence lines into considerarion. Green row highlight indicates highest ranking gene, and yellow row highlight indicates second best ranking genes.</p>
<p><strong>Table S16. Translation of DGAP clinical features to HPO terms.</strong> Table reporting the HPO identifiers per DGAP case.</p>
<p><strong>Table S17. Identification of genes with potential position effects for known pathogenic positive controls.</strong> Table reporting the candidate genes (ensembl_gene_ID, Gene_chr, Gene_start, Gene_end, Gene_name) and their various lines of selection evidence for the set of known pathogenic rearrangement positive controls (DGAP id, chr, start, end) within their analysis windows (window_start, window_end) from Redin <em>et al</em>., 2017. Evidence lines include Hi-C domain inclusion (Hi_domain, HiC_chr, HiC_start, HiC_end), haploinsufficiency (HI_chr, Gene-start, gene_end, HI_prob, Haploinsufficiency_score,), triplosensitivity (Triplosensitivity_score), phenomatch score (PhenoScore, MaxPhenoScore, Phone_percentile, count_Pheno_percentile, MaxPheno_percentile, count_MaxPheno_percentile, Percentile_final_count). All of the evidence information is summarized (6Mb, 2Mb, TAD, DHS, Count_haplo, count_triplo) and the gene rankings are presented in the PERC+DHS+TAD+HAPLO+TRIPLO and PERC+DHS+2Mb+HAPLO+TRIPLO columns which take different evidence lines into considerarion. Yellow row highlight indicates pathogenic genes reported by Redin <em>et al</em>., 2017.</p>
<p><strong>Table S18. Identification of disrupted chromatin contacts between disrupted DHS and enhancers by the non-coding DGAP breakpoint positions.</strong> An agnostic search revealed the existence of chromatin contacts between breakpoint-disrupted sequences of DHS sites and gene enhancers in Hi-C data of H1-hESC cells at 40 Kb resolution (Dixon <em>et al</em>., 2012). The reported genes are our top position effect candidate genes in the region. Table columns report the candidate gene information (Gene_chr, Gene_start, Gene_end, Gene_name), the associated DGAP case information (DGAP_ID, DGAP_chr, DGAP_start, DGAP_end) and the disrupted Hi-C chromatin interaction (HiC_1_chr, HiC_1_start, HiC_1_end, HiC_2_chr, HiC_2_start, HiC_2_end, HiC_1_interaction).</p>
<p><strong>Table S19. Overlap of non-coding DGAP breakpoint positions with DECIPHER cases.</strong> Table reporting the number of DECIPHER cases that overlap non-coding DGAP breakpoints (DGAP_ID, DGAP_chr, DGAP_start, DGAP_end). DECIPHER case information such as ID_patient, chr_start, chr_end, chr, mean_ratio, classification_type and phenotype are provided for each overlap.</p>
<p><strong>Table S20. Genes contained within overlapped DECIPHER cases by non-coding DGAP breakpoint positions.</strong> Table reporting the number of genes contained within overlapped DECIPHER cases by the non-coding DGAP breakpoints (DGAP_ID, DGAP_chr, DGAP_start, DGAP_end). DECIPHER case and gene information such as gene_count, DECIPHER_ID, DECIPHER_chr, DECIPHER_start, DECIPHER_end, DECIPHER_value, DECIPHER_type_rearr, DECIPHER_phenotype and HG_symbol are provided for each overlapped DECIPHER case.</p>
<p><strong>Table S21. DECIPHER cases overlapped by non-coding DGAP breakpoint positions that fulfilled non-coding selection criteria.</strong> Table reporting the number of DECIPHER cases that have non-coding breakpoints. DGAP comparison case information (DGAP_ID, DGAP_chr, DGAP_start, DGAP_end) is provided, as well as overlapped DECIPHER case information containing id_patient, chr_start, chr_end, chr, mean_ratio, classification_type and phenotype.</p>
<p><strong>Table S22. Overlap of non-coding DGAP breakpoint positions with dbVar cases.</strong> Table reporting the number of dbVar cases that overlap non-coding DGAP breakpoints (DGAP_chr, DGAP_start, DGAP_end, DGAP_ID). dbVar case information such as dbVar ID, Start, End, Variant type, Gene, Molecular consequences, Most severe clinical significance, 1000G minor allele, 1000G MAF, GO-ESP minor allele, GO-ESP MAF, ExAC minor allele, ExAC MAF, Publications (PMIDs), Variant allele, Transcript change, RefSeq, Protein change, Molecular consequence, HGVS_c, HGVS_g, HGVS_ng, HGVS_p, Condition, Most severe clinical significance, Submitters, Highest review status and Last evaluated are provided for each overlap.</p>
<p><strong>Supplemental References:</strong></p>
<p>1. Tang, Z., Berlin, D.S., Toji, L., Toruner, G.A., Beiswanger, C., Kulkarni, S., Martin, C.L., Emanuel, B.S., Christman, M., and Gerry, N.P. (2013). A dynamic database of microarray-characterized cell lines with various cytogenetic and genomic backgrounds. G3 3, 1143-1149.</p>
<p>2. Lachke, S.A., Higgins, A.W., Inagaki, M., Saadi, I., Xi, Q., Long, M., Quade, B.J., Talkowski, M.E., Gusella, J.F., Fujimoto, A., et al. (2012). The cell adhesion gene PVRL3 is associated with congenital ocular defects. Hum Genet 131, 235-250.</p>
<p>3. Anger, G.J., Crocker, S., McKenzie, K., Brown, K.K., Morton, C.C., Harrison, K., and MacKenzie, J.J. (2014). X-linked deafness-2 (DFNX2) phenotype associated with a paracentric inversion upstream of POU3F4. Am J Audiol 23, 1-6.</p>
<p>4. Jenkins, E.C., Curcuru-Giordano, F.M., Krishna, S.G., and Cantarella, J. (1975). De novo occurrence of 46,XX,t(4;13) (q31;q14) in a mentally retarded girl. Ann Genet 18, 117-120.</p>
<p>5. Frizell, E.R., Sutphen, R., Diamond, F.B., Jr., Sherwood, M., and Overhauser, J. (1998). t(1;18)(q32.1;q22.1) associated with genitourinary malformations. Clin Genet 54, 330-333.</p>
<!-- ======================================================================= -->
</div>
</div>
<!-- </div> -->
            </section>

          </div>
        </div>
      </div>
<a href="C-4-supplementary-figures-1.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="D-contribution-to-individual-publications.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"google": false,
"weibo": false,
"instapper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/ibn-salem/thesis/edit/master/08-appendix.Rmd",
"text": "Edit"
},
"download": ["thesis.pdf", "thesis.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "";
    if (src === "" || src === "true") src = "https://cdn.bootcss.com/mathjax/2.7.1/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:" && /^https?:/.test(src))
      src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
