<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>Genome folding and transcriptional regulation in evolution and disease</title>
  <meta name="description" content="This is a draft PhD thesis.">
  <meta name="generator" content="bookdown 0.5.10 and GitBook 2.6.7">

  <meta property="og:title" content="Genome folding and transcriptional regulation in evolution and disease" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="This is a draft PhD thesis." />
  <meta name="github-repo" content="ibn-salem/thesis" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Genome folding and transcriptional regulation in evolution and disease" />
  
  <meta name="twitter:description" content="This is a draft PhD thesis." />
  

<meta name="author" content="Jonas Ibn-Salem">


<meta name="date" content="2018-01-24">

  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  
<link rel="prev" href="introduction.html">
<link rel="next" href="ch-TAD-evolution.html">
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />










<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Thesis</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Preface</a></li>
<li class="chapter" data-level="1" data-path="introduction.html"><a href="introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a><ul>
<li class="chapter" data-level="1.1" data-path="introduction.html"><a href="introduction.html#structure-of-the-intorduction"><i class="fa fa-check"></i><b>1.1</b> Structure of the Intorduction</a></li>
<li class="chapter" data-level="1.2" data-path="introduction.html"><a href="introduction.html#cell-diversity-and-regulation-of-gene-expression"><i class="fa fa-check"></i><b>1.2</b> Cell diversity and regulation of gene expression</a></li>
<li class="chapter" data-level="1.3" data-path="introduction.html"><a href="introduction.html#enhancers"><i class="fa fa-check"></i><b>1.3</b> Enhancers</a></li>
<li class="chapter" data-level="1.4" data-path="introduction.html"><a href="introduction.html#methods-to-probe-the-3d-chromatin-archtiecture"><i class="fa fa-check"></i><b>1.4</b> Methods to probe the 3D chromatin archtiecture</a><ul>
<li class="chapter" data-level="1.4.1" data-path="introduction.html"><a href="introduction.html#microscopy-based-techniques-to-visuallize-the-genome-in-3d"><i class="fa fa-check"></i><b>1.4.1</b> Microscopy-based techniques to visuallize the genome in 3D</a></li>
<li class="chapter" data-level="1.4.2" data-path="introduction.html"><a href="introduction.html#proximity-ligation-based-method-to-quantify-chromatin-interactions"><i class="fa fa-check"></i><b>1.4.2</b> Proximity-ligation based method to quantify chromatin interactions</a></li>
</ul></li>
<li class="chapter" data-level="1.5" data-path="introduction.html"><a href="introduction.html#hierarchy-of-chromatin-3d-structure"><i class="fa fa-check"></i><b>1.5</b> Hierarchy of chromatin 3D structure</a><ul>
<li class="chapter" data-level="1.5.1" data-path="introduction.html"><a href="introduction.html#chromosomal-territories-and-inter-chromosomal-contacts"><i class="fa fa-check"></i><b>1.5.1</b> Chromosomal territories and inter-chromosomal contacts</a></li>
<li class="chapter" data-level="1.5.2" data-path="introduction.html"><a href="introduction.html#ab-compartments"><i class="fa fa-check"></i><b>1.5.2</b> A/B compartments</a></li>
<li class="chapter" data-level="1.5.3" data-path="introduction.html"><a href="introduction.html#topologically-associating-domains-tads"><i class="fa fa-check"></i><b>1.5.3</b> Topologically associating domains (TADs)</a></li>
<li class="chapter" data-level="1.5.4" data-path="introduction.html"><a href="introduction.html#chromatin-loops"><i class="fa fa-check"></i><b>1.5.4</b> Chromatin loops</a></li>
</ul></li>
<li class="chapter" data-level="1.6" data-path="introduction.html"><a href="introduction.html#changes-of-genome-folding-in-evoltuion-and-disease-genomes"><i class="fa fa-check"></i><b>1.6</b> Changes of genome folding in evoltuion and disease genomes</a></li>
<li class="chapter" data-level="1.7" data-path="introduction.html"><a href="introduction.html#aims-of-this-thesis"><i class="fa fa-check"></i><b>1.7</b> Aims of this thesis</a><ul>
<li class="chapter" data-level="" data-path="introduction.html"><a href="introduction.html#is-the-three-dimensonal-folding-of-genomes-associated-with-co-regulation-of-functionally-related-genes"><i class="fa fa-check"></i>Is the three-dimensonal folding of genomes associated with co-regulation of functionally related genes?</a></li>
<li class="chapter" data-level="" data-path="introduction.html"><a href="introduction.html#are-tads-functional-buidling-blocks-of-genomes-and-subjected-to-selective-preasure-during-evolution"><i class="fa fa-check"></i>Are TADs functional buidling blocks of genomes and subjected to selective preasure during evolution?</a></li>
<li class="chapter" data-level="" data-path="introduction.html"><a href="introduction.html#is-the-disruption-of-tads-by-rearrangements-also-associated-to-genetic-diseases"><i class="fa fa-check"></i>Is the disruption of TADs by rearrangements also associated to genetic diseases?</a></li>
<li class="chapter" data-level="" data-path="introduction.html"><a href="introduction.html#can-chromatin-looping-interactions-be-predicted-by-chip-seq-and-sequence-features"><i class="fa fa-check"></i>Can chromatin looping interactions be predicted by ChIP-seq and sequence features?</a></li>
</ul></li>
<li class="chapter" data-level="1.8" data-path="introduction.html"><a href="introduction.html#structure-of-this-thesis"><i class="fa fa-check"></i><b>1.8</b> Structure of this thesis</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html"><i class="fa fa-check"></i><b>2</b> Paralog genes in the 3D genome architecture</a><ul>
<li class="chapter" data-level="2.1" data-path="introduction.html"><a href="introduction.html#introduction"><i class="fa fa-check"></i><b>2.1</b> Introduction</a></li>
<li class="chapter" data-level="2.2" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#sec:paralog-methods"><i class="fa fa-check"></i><b>2.2</b> MATERIALS AND METHODS</a><ul>
<li class="chapter" data-level="2.2.1" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#selection-of-pairs-of-paralog-genes"><i class="fa fa-check"></i><b>2.2.1</b> Selection of pairs of paralog genes</a></li>
<li class="chapter" data-level="2.2.2" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#enhancers-to-gene-association"><i class="fa fa-check"></i><b>2.2.2</b> Enhancers to gene association</a></li>
<li class="chapter" data-level="2.2.3" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#topological-associating-domains"><i class="fa fa-check"></i><b>2.2.3</b> Topological associating domains</a></li>
<li class="chapter" data-level="2.2.4" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#hi-c-interaction-maps"><i class="fa fa-check"></i><b>2.2.4</b> Hi-C interaction maps</a></li>
<li class="chapter" data-level="2.2.5" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#randomization"><i class="fa fa-check"></i><b>2.2.5</b> Randomization</a></li>
<li class="chapter" data-level="2.2.6" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#statistical-tests"><i class="fa fa-check"></i><b>2.2.6</b> Statistical tests</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#results"><i class="fa fa-check"></i><b>2.3</b> RESULTS</a><ul>
<li class="chapter" data-level="2.3.1" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#distribution-of-paralog-genes-in-the-human-genome"><i class="fa fa-check"></i><b>2.3.1</b> Distribution of paralog genes in the human genome</a></li>
<li class="chapter" data-level="2.3.2" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#co-expression-of-paralog-gene-pairs-across-tissues"><i class="fa fa-check"></i><b>2.3.2</b> Co-expression of paralog gene pairs across tissues</a></li>
<li class="chapter" data-level="2.3.3" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#paralog-genes-share-enhancers"><i class="fa fa-check"></i><b>2.3.3</b> Paralog genes share enhancers</a></li>
<li class="chapter" data-level="2.3.4" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#co-localization-of-paralogs-in-tads"><i class="fa fa-check"></i><b>2.3.4</b> Co-localization of paralogs in TADs</a></li>
<li class="chapter" data-level="2.3.5" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#distal-paralog-pairs-are-enriched-for-long-range-chromatin-contacts"><i class="fa fa-check"></i><b>2.3.5</b> Distal paralog pairs are enriched for long-range chromatin contacts</a></li>
<li class="chapter" data-level="2.3.6" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#close-paralogs-have-fewer-contacts-than-expected"><i class="fa fa-check"></i><b>2.3.6</b> Close paralogs have fewer contacts than expected</a></li>
<li class="chapter" data-level="2.3.7" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#paralogs-in-mouse-and-dog-genome"><i class="fa fa-check"></i><b>2.3.7</b> Paralogs in mouse and dog genome</a></li>
<li class="chapter" data-level="2.3.8" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#orthologs-of-human-paralogs-show-conserved-co-localization"><i class="fa fa-check"></i><b>2.3.8</b> Orthologs of human paralogs show conserved co-localization</a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#discussion"><i class="fa fa-check"></i><b>2.4</b> DISCUSSION</a></li>
<li class="chapter" data-level="2.5" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#conclusion"><i class="fa fa-check"></i><b>2.5</b> CONCLUSION</a></li>
<li class="chapter" data-level="2.6" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#acknowledgements"><i class="fa fa-check"></i><b>2.6</b> ACKNOWLEDGEMENTS</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html"><i class="fa fa-check"></i><b>3</b> Stability of TADs in evolution</a><ul>
<li class="chapter" data-level="" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#abstract"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#keywords"><i class="fa fa-check"></i>Keywords</a></li>
<li class="chapter" data-level="3.1" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#introduction-1"><i class="fa fa-check"></i><b>3.1</b> Introduction</a></li>
<li class="chapter" data-level="3.2" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#results-1"><i class="fa fa-check"></i><b>3.2</b> Results</a><ul>
<li class="chapter" data-level="3.2.1" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#identification-of-evolutionary-rearrangement-breakpoints-from-whole-genome-alignments"><i class="fa fa-check"></i><b>3.2.1</b> Identification of evolutionary rearrangement breakpoints from whole-genome alignments</a></li>
<li class="chapter" data-level="3.2.2" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#rearrangement-breakpoints-are-enriched-at-tad-boundaries"><i class="fa fa-check"></i><b>3.2.2</b> Rearrangement breakpoints are enriched at TAD boundaries</a></li>
<li class="chapter" data-level="3.2.3" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#clusters-of-conserved-non-coding-elements-are-depleted-for-rearrangement-breakpoints"><i class="fa fa-check"></i><b>3.2.3</b> Clusters of conserved non-coding elements are depleted for rearrangement breakpoints</a></li>
<li class="chapter" data-level="3.2.4" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#rearranged-tads-are-associated-with-divergent-gene-expression-between-species"><i class="fa fa-check"></i><b>3.2.4</b> Rearranged TADs are associated with divergent gene expression between species</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#discussion-1"><i class="fa fa-check"></i><b>3.3</b> Discussion</a></li>
<li class="chapter" data-level="3.4" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#conclusion-1"><i class="fa fa-check"></i><b>3.4</b> Conclusion</a></li>
<li class="chapter" data-level="3.5" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#methods"><i class="fa fa-check"></i><b>3.5</b> Methods</a><ul>
<li class="chapter" data-level="3.5.1" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#rearrangement-breakpoints-from-whole-genome-alignments"><i class="fa fa-check"></i><b>3.5.1</b> Rearrangement breakpoints from whole-genome alignments</a></li>
<li class="chapter" data-level="3.5.2" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#topologically-associating-domains-and-contact-domains"><i class="fa fa-check"></i><b>3.5.2</b> Topologically associating domains and contact domains</a></li>
<li class="chapter" data-level="3.5.3" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#breakpoint-distributions-at-tads"><i class="fa fa-check"></i><b>3.5.3</b> Breakpoint distributions at TADs</a></li>
<li class="chapter" data-level="3.5.4" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#quantification-of-breakpoint-enrichment"><i class="fa fa-check"></i><b>3.5.4</b> Quantification of breakpoint enrichment</a></li>
<li class="chapter" data-level="3.5.5" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#expression-data-for-mouse-and-human-orthologs"><i class="fa fa-check"></i><b>3.5.5</b> Expression data for mouse and human orthologs</a></li>
<li class="chapter" data-level="3.5.6" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#classification-of-tads-and-genes-according-to-rearrangements-and-grbs"><i class="fa fa-check"></i><b>3.5.6</b> Classification of TADs and genes according to rearrangements and GRBs</a></li>
<li class="chapter" data-level="3.5.7" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#source-code-and-implementation-details"><i class="fa fa-check"></i><b>3.5.7</b> Source code and implementation details</a></li>
</ul></li>
<li class="chapter" data-level="3.6" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#declarations"><i class="fa fa-check"></i><b>3.6</b> Declarations</a><ul>
<li class="chapter" data-level="3.6.1" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#ethics-approval-and-consent-to-participate"><i class="fa fa-check"></i><b>3.6.1</b> Ethics approval and consent to participate</a></li>
<li class="chapter" data-level="3.6.2" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#consent-for-publication"><i class="fa fa-check"></i><b>3.6.2</b> Consent for publication</a></li>
<li class="chapter" data-level="3.6.3" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#availability-of-data-and-material"><i class="fa fa-check"></i><b>3.6.3</b> Availability of data and material</a></li>
<li class="chapter" data-level="3.6.4" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#competing-interests"><i class="fa fa-check"></i><b>3.6.4</b> Competing interests</a></li>
<li class="chapter" data-level="3.6.5" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#funding"><i class="fa fa-check"></i><b>3.6.5</b> Funding</a></li>
<li class="chapter" data-level="3.6.6" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#authors-contributions"><i class="fa fa-check"></i><b>3.6.6</b> Authors‘ contributions</a></li>
<li class="chapter" data-level="3.6.7" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#acknowledgments"><i class="fa fa-check"></i><b>3.6.7</b> Acknowledgments</a></li>
</ul></li>
<li class="chapter" data-level="3.7" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#tables"><i class="fa fa-check"></i><b>3.7</b> Tables</a><ul>
<li class="chapter" data-level="3.7.1" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#table-1"><i class="fa fa-check"></i><b>3.7.1</b> Table 1</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="ch-position-effect.html"><a href="ch-position-effect.html"><i class="fa fa-check"></i><b>4</b> Position effects of rearrangements in disease genomes</a><ul>
<li class="chapter" data-level="4.1" data-path="ch-position-effect.html"><a href="ch-position-effect.html#abstract-1"><i class="fa fa-check"></i><b>4.1</b> Abstract</a></li>
<li class="chapter" data-level="4.2" data-path="ch-position-effect.html"><a href="ch-position-effect.html#introduction-2"><i class="fa fa-check"></i><b>4.2</b> Introduction</a></li>
<li class="chapter" data-level="4.3" data-path="ch-position-effect.html"><a href="ch-position-effect.html#materials-and-methods"><i class="fa fa-check"></i><b>4.3</b> Materials and Methods</a><ul>
<li class="chapter" data-level="4.3.1" data-path="ch-position-effect.html"><a href="ch-position-effect.html#selection-of-subjects-with-apparently-balanced-chromosome-abnormalities"><i class="fa fa-check"></i><b>4.3.1</b> Selection of subjects with apparently balanced chromosome abnormalities</a></li>
<li class="chapter" data-level="4.3.2" data-path="ch-position-effect.html"><a href="ch-position-effect.html#clinical-descriptions-of-dgap-cases"><i class="fa fa-check"></i><b>4.3.2</b> Clinical descriptions of DGAP cases</a></li>
<li class="chapter" data-level="4.3.3" data-path="ch-position-effect.html"><a href="ch-position-effect.html#analysis-of-genes-bordering-the-rearrangement-breakpoints"><i class="fa fa-check"></i><b>4.3.3</b> Analysis of genes bordering the rearrangement breakpoints</a></li>
<li class="chapter" data-level="4.3.4" data-path="ch-position-effect.html"><a href="ch-position-effect.html#assessment-of-disrupted-functional-elements-and-chromatin-interactions"><i class="fa fa-check"></i><b>4.3.4</b> Assessment of disrupted functional elements and chromatin interactions</a></li>
<li class="chapter" data-level="4.3.5" data-path="ch-position-effect.html"><a href="ch-position-effect.html#ontological-analysis-of-genes-neighboring-breakpoints"><i class="fa fa-check"></i><b>4.3.5</b> Ontological analysis of genes neighboring breakpoints</a></li>
<li class="chapter" data-level="4.3.6" data-path="ch-position-effect.html"><a href="ch-position-effect.html#quantitative-real-time-pcr"><i class="fa fa-check"></i><b>4.3.6</b> Quantitative real-time PCR</a></li>
<li class="chapter" data-level="4.3.7" data-path="ch-position-effect.html"><a href="ch-position-effect.html#assessment-of-dgap-breakpoints-overlapping-with-non-coding-structural-variants-in-public-databases"><i class="fa fa-check"></i><b>4.3.7</b> Assessment of DGAP breakpoints overlapping with non-coding structural variants in public databases</a></li>
</ul></li>
<li class="chapter" data-level="4.4" data-path="ch-position-effect.html"><a href="ch-position-effect.html#results-2"><i class="fa fa-check"></i><b>4.4</b> Results</a><ul>
<li class="chapter" data-level="4.4.1" data-path="ch-position-effect.html"><a href="ch-position-effect.html#genomic-characterization-of-non-coding-breakpoints"><i class="fa fa-check"></i><b>4.4.1</b> Genomic characterization of non-coding breakpoints</a></li>
<li class="chapter" data-level="4.4.2" data-path="ch-position-effect.html"><a href="ch-position-effect.html#identification-of-genes-with-potential-position-effects"><i class="fa fa-check"></i><b>4.4.2</b> Identification of genes with potential position effects</a></li>
<li class="chapter" data-level="4.4.3" data-path="ch-position-effect.html"><a href="ch-position-effect.html#identification-of-subjects-with-shared-non-coding-chromosome-alterations-and-phenotypes"><i class="fa fa-check"></i><b>4.4.3</b> Identification of subjects with shared non-coding chromosome alterations and phenotypes</a></li>
</ul></li>
<li class="chapter" data-level="4.5" data-path="ch-position-effect.html"><a href="ch-position-effect.html#discussion-2"><i class="fa fa-check"></i><b>4.5</b> Discussion</a></li>
<li class="chapter" data-level="4.6" data-path="ch-position-effect.html"><a href="ch-position-effect.html#acknowledgements-1"><i class="fa fa-check"></i><b>4.6</b> Acknowledgements</a></li>
<li class="chapter" data-level="4.7" data-path="ch-position-effect.html"><a href="ch-position-effect.html#tables-1"><i class="fa fa-check"></i><b>4.7</b> Tables</a><ul>
<li class="chapter" data-level="4.7.1" data-path="ch-position-effect.html"><a href="ch-position-effect.html#table-1."><i class="fa fa-check"></i><b>4.7.1</b> Table 1.</a></li>
</ul></li>
<li class="chapter" data-level="4.8" data-path="ch-position-effect.html"><a href="ch-position-effect.html#figures"><i class="fa fa-check"></i><b>4.8</b> Figures</a></li>
<li class="chapter" data-level="4.9" data-path="ch-position-effect.html"><a href="ch-position-effect.html#web-resources"><i class="fa fa-check"></i><b>4.9</b> Web Resources</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="ch-loop-prediction.html"><a href="ch-loop-prediction.html"><i class="fa fa-check"></i><b>5</b> Prediction of chromatin looping interactions</a></li>
<li class="chapter" data-level="6" data-path="discussion-3.html"><a href="discussion-3.html"><i class="fa fa-check"></i><b>6</b> Discussion</a><ul>
<li class="chapter" data-level="6.1" data-path="discussion-3.html"><a href="discussion-3.html#further-directions"><i class="fa fa-check"></i><b>6.1</b> Further directions</a></li>
<li class="chapter" data-level="6.2" data-path="discussion-3.html"><a href="discussion-3.html#conclusions"><i class="fa fa-check"></i><b>6.2</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="appendix"><span><b>Appendix</b></span></li>
<li class="chapter" data-level="A" data-path="supporting-information-co-regulation-of-paralog-genes-in-the-three-dimensional-chromatin-architecture.html"><a href="supporting-information-co-regulation-of-paralog-genes-in-the-three-dimensional-chromatin-architecture.html"><i class="fa fa-check"></i><b>A</b> Supporting Information: Co-regulation of paralog genes in the three-dimensional chromatin architecture</a></li>
<li class="chapter" data-level="B" data-path="supplementary-data-evolutionary-stability-of-topologically-associating-domains-is-associated-with-conserved-gene-regulation.html"><a href="supplementary-data-evolutionary-stability-of-topologically-associating-domains-is-associated-with-conserved-gene-regulation.html"><i class="fa fa-check"></i><b>B</b> Supplementary Data: Evolutionary stability of topologically associating domains is associated with conserved gene regulation</a><ul>
<li class="chapter" data-level="B.1" data-path="supplementary-data-evolutionary-stability-of-topologically-associating-domains-is-associated-with-conserved-gene-regulation.html"><a href="supplementary-data-evolutionary-stability-of-topologically-associating-domains-is-associated-with-conserved-gene-regulation.html#TadEvoSupTab"><i class="fa fa-check"></i><b>B.1</b> Supplementary Tables</a></li>
<li class="chapter" data-level="B.2" data-path="supplementary-data-evolutionary-stability-of-topologically-associating-domains-is-associated-with-conserved-gene-regulation.html"><a href="supplementary-data-evolutionary-stability-of-topologically-associating-domains-is-associated-with-conserved-gene-regulation.html#supplementary-figures"><i class="fa fa-check"></i><b>B.2</b> Supplementary Figures</a></li>
</ul></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Build with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Genome folding and transcriptional regulation in evolution and disease</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="ch:paralog_regulaion" class="section level1">
<h1><span class="header-section-number">Chapter 2</span> Paralog genes in the 3D genome architecture</h1>
<p>This chapter is published in Nucleac Acid Research <span class="citation">(Ibn-Salem et al., <a href="references.html#ref-Ibn-Salem2017">2017</a>)</span>. The surce code for the complete analysis is available at GitHub: <a href="https://github.com/ibn-salem/paralog_regulation" class="uri">https://github.com/ibn-salem/paralog_regulation</a></p>
<div id="introduction" class="section level2">
<h2><span class="header-section-number">2.1</span> Introduction</h2>
<p>Paralog genes arise from gene duplication events during evolution. The resulting sequence similarity between paralog pairs might lead to similar structure and function of encoded proteins <span class="citation">(Koonin, <a href="references.html#ref-Koonin2005">2005</a>)</span>. Since paralogs often form part of the same protein complexes and pathways, it is advantageous for the cell to coordinate their expression <span class="citation">(Makova and Li, <a href="references.html#ref-Makova2003">2003</a>)</span>.</p>
<p>In eukaryotes, genes are regulated in part by binding of transcription factors to promoter sequences and to distal regulatory regions such as enhancers. By chromatin looping, enhancer bound proteins can physically interact with the transcription machinery at the promoter of genes <span class="citation">(Carter et al., <a href="references.html#ref-Carter2002">2002</a>; Deng et al., <a href="references.html#ref-Deng2012">2012</a>; Ptashne, <a href="references.html#ref-Ptashne1986">1986</a>; Spitz and Furlong, <a href="references.html#ref-Spitz2012">2012</a>; Tolhuis et al., <a href="references.html#ref-Tolhuis2002">2002</a>)</span>. These chromatin looping events can be measured by chromatin conformation capture (3C) experiments <span class="citation">(Dekker et al., <a href="references.html#ref-Dekker2002">2002</a>)</span>, which use proximity-ligation, and more recently high-throughput sequencing (Hi-C) to measure DNA-DNA contact frequencies genome-wide <span class="citation">(Lieberman-Aiden et al., <a href="references.html#ref-Lieberman-Aiden2009">2009</a>)</span>.</p>
<p>These interaction maps revealed tissue-invariant chromatin regions, named topologically associating domains (TADs), which have more interactions within themselves than with other regions <span class="citation">(Dixon et al., <a href="references.html#ref-Dixon2012">2012</a>; Nora et al., <a href="references.html#ref-Nora2012">2012</a>; Sexton et al., <a href="references.html#ref-Sexton2012">2012</a>)</span>. TADs seem to be stable across cell types and conserved between mammals <span class="citation">(Dixon et al., <a href="references.html#ref-Dixon2012">2012</a>; Rao et al., <a href="references.html#ref-Rao2014">2014</a>; Vietri Rudan et al., <a href="references.html#ref-VietriRudan2015">2015</a>)</span>. Regions within TADs show concerted histone chromatin signatures <span class="citation">(Dixon et al., <a href="references.html#ref-Dixon2012">2012</a>; Sexton et al., <a href="references.html#ref-Sexton2012">2012</a>)</span>, gene expression <span class="citation">(Le Dily et al., <a href="references.html#ref-LeDily2014">2014</a>; Nora et al., <a href="references.html#ref-Nora2012">2012</a>)</span>, and DNA replication timing <span class="citation">(Pope et al., <a href="references.html#ref-Pope2014">2014</a>)</span>. Furthermore, disruption of TAD boundaries is associated to genetic diseases <span class="citation">(Ibn-Salem et al., <a href="references.html#ref-Ibn-Salem2014">2014</a>; Lupiáñez et al., <a href="references.html#ref-Lupianez2015">2015</a>)</span>.</p>
<p>We wondered if the Hi-C data could reveal evolutionary pressure driving paralogous expansion to favour the clustering of paralogs in the three-dimensional chromatin architecture and their regulation by common enhancer elements to enable the cell to fine-tune and coordinate their expression. To do this, we collected Hi-C data from a number of studies profiling contacts in several cell types from human <span class="citation">(Dixon et al., <a href="references.html#ref-Dixon2012">2012</a>; Rao et al., <a href="references.html#ref-Rao2014">2014</a>)</span>, mouse and dog <span class="citation">(Vietri Rudan et al., <a href="references.html#ref-VietriRudan2015">2015</a>)</span>, and we compared the properties of these data with respect to paralog genes. Our results pinpoint that pairs of paralog genes tend to be co-regulated and co-occur within TADs more often than equivalent control gene pairs. When placed in different TADs, paralogs still tend to co-occur in the same chromosome and have more contacts than control gene pairs. In contrast, close paralogs in the same TAD have significantly less contacts with each other than comparable gene pairs, which could indicate that these pairs of paralogs encode proteins that functionally replace each other.</p>
<p>These observations have relevance for the study of the evolution of chromatin structure and suggest that tandem duplications generating paralogs are under selection according to how they contribute or not to the fine structure of the genome as reflected by TADs. Thus TADs provide a favorable environment for the co-regulation of duplicated genes, which is likely followed by the evolutionary generation of additional regulatory mechanisms allowing the separation of paralogs into different TADs in the same chromosome but connected, and eventually their migration into different chromosomes.</p>
</div>
<div id="sec:paralog-methods" class="section level2">
<h2><span class="header-section-number">2.2</span> MATERIALS AND METHODS</h2>
<div id="selection-of-pairs-of-paralog-genes" class="section level3">
<h3><span class="header-section-number">2.2.1</span> Selection of pairs of paralog genes</h3>
<p>All human genes and human paralog gene pairs were retrieved from Ensembl GRCh37 (Ensembl 75) database by using the <code>biomaRt</code> package <span class="citation">(Durinck et al., <a href="references.html#ref-Durinck2005">2005</a>, <a href="references.html#ref-Durinck2009b">2009</a><a href="references.html#ref-Durinck2009b">a</a>)</span> from within the statistical programming environment R. For each gene we downloaded the Ensembl gene ID, HGNC symbol, transcription sense, transcription start site (TSS) coordinates, and gene length. We only considered protein coding genes with “KNOWN” status that are annotated in the 22 autosomes or the 2 sexual chromosomes. For each gene we used the earliest TSS coordinate. Within this set of genes, all pairs of human paralog genes were downloaded from Ensembl <span class="citation">(Vilella et al., <a href="references.html#ref-Vilella2009">2009</a>)</span>. This resulted in a total of 19,430 human genes; more than half of those had at least one human paralog gene (Fig. <a href="supporting-information-co-regulation-of-paralog-genes-in-the-three-dimensional-chromatin-architecture.html#fig:paraVSnonPara">A.1</a>A).</p>
<p>However, many human genes have more than one paralog (Fig. <a href="supporting-information-co-regulation-of-paralog-genes-in-the-three-dimensional-chromatin-architecture.html#fig:paraVSnonPara">A.1</a>B). To avoid overrepresentation of genes, we filtered the pairs such that each gene occurred only once. Thereby we selected the pairs by minimizing the rate of synonymous mutations (dS) between them using a maximum-weighted matching graph algorithm implement in the python package <code>NetworkX</code> <span class="citation">(Galil, <a href="references.html#ref-Galil1986">1986</a>)</span>. The number of synonymous mutations between paralogs has been used to approximate the duplication age <span class="citation">(Lan and Pritchard, <a href="references.html#ref-Lan2016">2016</a>)</span>. Therefore our implementation favours the selection of young paralog pairs for larger paralog families and guaranties that each gene occurs only once. This filtering strategy resulted in <span class="math inline">\(6256\)</span> unique paralog pairs for downstream analysis (Table <a href="ch-paralog-regulaion.html#tab:filter">2.1</a>). We observed that modications of this strategy to select unique paralog genes did not affect essentially the results of our study (e.g. by selecting pairs while maximising dS; Fig. <a href="supporting-information-co-regulation-of-paralog-genes-in-the-three-dimensional-chromatin-architecture.html#fig:selectOldPairs">A.2</a>).</p>
<p>Analogously to the human data we downloaded all pairs of protein coding paralog genes from the <em>Mus musculus</em> (GRCm38.p2) and <em>Canis lupus familiaris</em> (CanFam3.1) genomes from Ensembl. The numbers of filtered gene pairs are shown in Table <a href="ch-paralog-regulaion.html#tab:filter">2.1</a> . Furthermore, we related human paralog genes to orthologs in mouse and dog only if there was a unique one-to-one orthology relationship reported in the Ensembl database.</p>
<table>
<caption><span id="tab:filter">Table 2.1: </span> Filtering of human paralog gene pairs</caption>
<thead>
<tr class="header">
<th>Paralog pairs</th>
<th align="right">Human</th>
<th align="right">Mouse</th>
<th align="center">Dog</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>All paralog pairs</td>
<td align="right">46546</td>
<td align="right">110490</td>
<td align="center">28293</td>
</tr>
<tr class="even">
<td>One pair per gene</td>
<td align="right">6256</td>
<td align="right">7323</td>
<td align="center">4959</td>
</tr>
<tr class="odd">
<td>On the same chromosome</td>
<td align="right">1560</td>
<td align="right">2397</td>
<td align="center">658</td>
</tr>
<tr class="even">
<td>Close pairs (TSS distance <span class="math inline">\(\leq\)</span> 1 Mb)</td>
<td align="right">1114</td>
<td align="right">1774</td>
<td align="center">455</td>
</tr>
<tr class="odd">
<td>Distal pairs (TSS distance <span class="math inline">\(&gt;\)</span> 1 Mb)</td>
<td align="right">446</td>
<td align="right">623</td>
<td align="center">203</td>
</tr>
</tbody>
</table>
</div>
<div id="enhancers-to-gene-association" class="section level3">
<h3><span class="header-section-number">2.2.2</span> Enhancers to gene association</h3>
<p>Human enhancer annotations, including their genome locations and the corresponding genes they regulate, were obtained from the supplementary data of a recent CAGE analysis <span class="citation">(Andersson et al., <a href="references.html#ref-Andersson2014">2014</a>)</span>. In this study, the activity of enhancers and genes was correlated within 500kb over hundreds of human cell types to provide a regulatory interaction map between 27,451 enhancers and 11,604 genes consisting of 66,942 interactions.</p>
</div>
<div id="topological-associating-domains" class="section level3">
<h3><span class="header-section-number">2.2.3</span> Topological associating domains</h3>
<p>We obtained topological associating domain (TAD) calls from two recently published Hi-C studies in human cells <span class="citation">(Dixon et al., <a href="references.html#ref-Dixon2012">2012</a>; Rao et al., <a href="references.html#ref-Rao2014">2014</a>)</span>. TAD locations mapped to the hg18 genome assembly were converted to hg19 using the UCSC liftOver tool <span class="citation">(Hinrichs et al., <a href="references.html#ref-Hinrichs2006">2006</a>)</span>. A/B-compartment and sub-compartment annotations were obtained from high-resolution Hi-C experiments in human GM12878 cells <span class="citation">(Rao et al., <a href="references.html#ref-Rao2014">2014</a>)</span>.</p>
</div>
<div id="hi-c-interaction-maps" class="section level3">
<h3><span class="header-section-number">2.2.4</span> Hi-C interaction maps</h3>
<p>Individual chromatin-chromatin contact frequencies from IMR90 cells at 5 kb resolution were retrieved from <span class="citation">(Rao et al., <a href="references.html#ref-Rao2014">2014</a>)</span>(NCBI GEO accession: GSE63525). We used only reads with mapping quality <span class="math inline">\(\geq\)</span> 30 and normalized the raw contact matrices applying the provided normalization vectors for KR normalization by the matrix balancing approach <span class="citation">(Knight and Ruiz, <a href="references.html#ref-Knight2013">2013</a>)</span>. We only considered pairwise gene interactions if the TSSs of the two genes were located in different bins of the Hi-C matrix with normalized contacts <span class="math inline">\(\geq\)</span> 0. Capture Hi-C data between promoter regions in human GM12878 cells were downloaded from ArrayExpress (accession: E-MTAB-2323) <span class="citation">(Mifsud et al., <a href="references.html#ref-Mifsud2015">2015</a>)</span>.</p>
</div>
<div id="randomization" class="section level3">
<h3><span class="header-section-number">2.2.5</span> Randomization</h3>
<p>We analysed the distribution of paralog pairs over chromosomes depending on the linear distance between them. For doing so, we sampled gene pairs from all human genes with equal and independent probability and refer to them as random gene pairs.</p>
<p>For strand analysis, co-localisation in TADs, and Hi-C contact quantification between paralog pairs, we constructed a carefully sampled control set of gene pairs as null-model. Thereby we accounted for the linear distance bias observed for paralog pairs. First, we calculated all possible non-overlapping pairs of human genes on the same chromosome. From the resulting set of gene pairs we randomly sampled pairs according to the linear distance distribution of paralog gene pairs. Therefore, we assigned to each gene pair a sampling weight that is proportional to the probability to sample the pair. The sampling weight <span class="math inline">\(w(g_{i}, g_{j})\)</span> for a given pair of genes <span class="math inline">\(g_{i}\)</span> and <span class="math inline">\(g_{j}\)</span> with absolute distance <span class="math inline">\(d_{i,j}\)</span> is defined as</p>
<p><span class="math display">\[
w(g_{i}, g_{j}) = \frac{ f_{\mathrm{paralogs}}(d_{i,j}) }{f_{\mathrm{all}}(d_{i,j})}
\]</span></p>
<p>where <span class="math inline">\(f_{\mathrm{paralogs}}\)</span> is the observed frequencies of distances in the paralog genes and <span class="math inline">\(f_{\mathrm{all}}(d_{i,j})\)</span> the frequency of pairwise distances in the population of gene pairs from which we sample. We computed the observed frequencies by dividing the distances into 90 equal-sized bins after <span class="math inline">\(log_{10}\)</span> distance transformation and counted occurrences of gene pairs for each bin. The resulting sampling weights for all gene pairs are normalized to sum up 1 and were then used as probabilities for sampling:</p>
<p><span class="math display" id="eq:DistProb">\[
p_{\mathrm{dist}}(g_{i}, g_{j}) = \frac{ w(g_{i}, g_{j}) }{ \sum_{i,j} w(g_{i}, g_{j}) }
\tag{2.1}
\]</span></p>
<p>Next, for comparison of shared enhancers we slightly modified the sampling of gene pairs to account for the observation that paralogs tend to be associated to more enhancers than non-paralogs (Fig. <a href="supporting-information-co-regulation-of-paralog-genes-in-the-three-dimensional-chromatin-architecture.html#fig:paraVSnonPara">A.1</a>D). Assuming that the number of enhancers associated to genes is independent from the distance, we computed sampling probabilities by <span class="math display">\[
p_{\mathrm{dist+eh}}(g_{i}, g_{j}) = p_{\mathrm{dist}}(g_{i}, g_{j}) \cdot p_{\mathrm{eh}}(n_{i}) \cdot p_{\mathrm{eh}}(n_{j})
\]</span> whereby <span class="math inline">\(n_{i}\)</span> and <span class="math inline">\(n_{j}\)</span> are the number of enhancers associated to <span class="math inline">\(g_{i}\)</span> and <span class="math inline">\(g_{j}\)</span>, respectively and <span class="math inline">\(p_{\mathrm{eh}}(n)\)</span> is the probability to sample a gene associated to n enhancers:</p>
<p><span class="math display" id="eq:EhProb">\[
p_{\mathrm{eh}}(n) = \frac{ w_{\mathrm{eh}}(n) }{ \sum_{i=0}^{N} w_{\mathrm{eh}}(i) }
\tag{2.2}
\]</span></p>
<p>and <span class="math display">\[
w_{\mathrm{eh}}(n) = \frac{ f_{\mathrm{paralogs}}(n) }{f_{\mathrm{all}}(n)}
\]</span></p>
<p>where <span class="math inline">\(f_{\mathrm{paralogs}}(n)\)</span> and <span class="math inline">\(f_{\mathrm{all}}(n)\)</span> gives the frequency of genes associated to <span class="math inline">\(n\)</span> enhancers observed in the paralog pairs and all gene pairs, respectively.</p>
<p>Analogously, we sampled sets of pairs accounting additionally for the observed bias in paralog pairs to be in the same strand. <span class="math display">\[
p_{\mathrm{dist+eh+strand}}(g_{i}, g_{j}) = p_{\mathrm{dist}}(g_{i}, g_{j}) \cdot p_{\mathrm{eh}}(n_{i}) \cdot p_{\mathrm{eh}}(n_{j}) \cdot p_{\mathrm{strand}}(s_{i,j})
\]</span> whereby <span class="math inline">\(s_{i,j}\)</span> is <span class="math inline">\(1\)</span> if both genes, <span class="math inline">\(g_{i}\)</span> and <span class="math inline">\(g_{j}\)</span>, are transcribed from the same strand and <span class="math inline">\(0\)</span> otherwise. The probability <span class="math inline">\(p_{\mathrm{strand}}(s_{i,j})\)</span> is computed in the same way as the probability by number of enhancers <span class="math inline">\(p_{\mathrm{eh}}(n)\)</span> in equation <a href="ch-paralog-regulaion.html#eq:EhProb">(2.2)</a>.</p>
<p>Lastly, we sampled a set of gene pairs by taking additionally the gene length into account and computed sampling probabilities by <span class="math display">\[
p_{\mathrm{dist+eh+len}}(g_{i}, g_{j}) = p_{\mathrm{dist}}(g_{i}, g_{j}) \cdot p_{\mathrm{eh}}(n_{i}) \cdot p_{\mathrm{eh}}(n_{j}) \cdot p_{\mathrm{len}}(l_{i}) \cdot p_{\mathrm{len}}(l_{j})
\]</span></p>
<p>whereby <span class="math inline">\(p_{\mathrm{len}}(l)\)</span> for gene length <span class="math inline">\(l\)</span> is computed in the same way as for distances between gene pairs (equation <a href="ch-paralog-regulaion.html#eq:DistProb">(2.1)</a>) and by dividing gene lengths into 20 equal sized binds after <span class="math inline">\(log_{10}\)</span> transformation of gene lengths in bp.</p>
<p>For each paralog pair on the same chromosome within 1 Mb distance, we sampled <span class="math inline">\(10\)</span> random gene pairs with this procedures each resulting in <span class="math inline">\(n=156,000\)</span> sampled gene pairs that served as background in our statistical analysis. These sampling approaches resulted in similar distribution of linear distances (Fig. <a href="supporting-information-co-regulation-of-paralog-genes-in-the-three-dimensional-chromatin-architecture.html#fig:samplingDist">A.3</a>), associated enhancers of each gene (Fig. <a href="supporting-information-co-regulation-of-paralog-genes-in-the-three-dimensional-chromatin-architecture.html#fig:samplingDistEh">A.4</a>), same strand (Fig. <a href="supporting-information-co-regulation-of-paralog-genes-in-the-three-dimensional-chromatin-architecture.html#fig:samplingDistEhStrand">A.5</a>), and gene lengths (Fig. <a href="supporting-information-co-regulation-of-paralog-genes-in-the-three-dimensional-chromatin-architecture.html#fig:samplingDistEhLen">A.6</a>).</p>
</div>
<div id="statistical-tests" class="section level3">
<h3><span class="header-section-number">2.2.6</span> Statistical tests</h3>
<p>We compared observed fractions of gene pairs, on the same chromosome, with the same transcription sense, within the same TAD or compartment, and with at least one shared enhancer between pairs of paralogs and random or sampled pairs using the Fisher’s exact test. Hi-C contact frequencies and genomic distances between TSS of gene pairs were compared using a Wilcoxon rank-sum test. All analyses were carried out in the statistic software R version 3.2.2.</p>
</div>
</div>
<div id="results" class="section level2">
<h2><span class="header-section-number">2.3</span> RESULTS</h2>
<div id="distribution-of-paralog-genes-in-the-human-genome" class="section level3">
<h3><span class="header-section-number">2.3.1</span> Distribution of paralog genes in the human genome</h3>
<p>Paralogs are homologous genes that arise from gene duplication events. Their common ancestry and replicated sequence often leads to similar structure and function in related pathways and protein complexes. We therefore hypothesised that the transcription of paralogs should have a tendency for co-regulation, which could correspond to their position in the genome and within TADs. To test this hypothesis, we first focused on the positions of paralogs in the linear genome.</p>
<p>From all <span class="math inline">\(19,430\)</span> protein coding genes in the human genome, <span class="math inline">\(13,690~(70.5\%)\)</span> have at least one paralog (Fig. <a href="supporting-information-co-regulation-of-paralog-genes-in-the-three-dimensional-chromatin-architecture.html#fig:paraVSnonPara">A.1</a>A). However, many human genes have several paralogs (Fig. <a href="supporting-information-co-regulation-of-paralog-genes-in-the-three-dimensional-chromatin-architecture.html#fig:paraVSnonPara">A.1</a>B). From all <span class="math inline">\(46,546\)</span> paralog gene pairs we filtered for only one pair per gene <span class="math inline">\((n=6,256)\)</span> and further for non-overlapping pairs on the same chromosome <span class="math inline">\((n=1,560)\)</span> (see ). We will refer to close paralogs if their transcription start sites (TSSs) are within 1 Mb of each other <span class="math inline">\((n=1,114)\)</span> and to distal pairs for paralogs with TSSs separated by more than 1 Mb <span class="math inline">\((n=446)\)</span> (Table <a href="ch-paralog-regulaion.html#tab:filter">2.1</a>).</p>
<p>We first compared basic properties between genes that have at least one paralog copy and genes without human paralogs. Paralogs have significantly larger gene length than non-paralog genes (<span class="math inline">\(p = 1.7 \times 10^{-53}\)</span>, Wilcoxon rank-sum test, Fig. <a href="supporting-information-co-regulation-of-paralog-genes-in-the-three-dimensional-chromatin-architecture.html#fig:paraVSnonPara">A.1</a>C), which fits the observation from <span class="citation">(He and Zhang, <a href="references.html#ref-He2005">2005</a>)</span> in yeast. Furthermore, paralogs tend to be associated to more enhancers compared to non-paralog genes (on average <span class="math inline">\(3.8\)</span> vs. <span class="math inline">\(2.5\)</span> enhancers per gene, <span class="math inline">\(p=2.89\times10^{-70}\)</span>, Fig. <a href="supporting-information-co-regulation-of-paralog-genes-in-the-three-dimensional-chromatin-architecture.html#fig:paraVSnonPara">A.1</a>D) and the distance to the nearest associated enhancer is significantly shorter (<span class="math inline">\(p=2.71\times10^{-22}\)</span>,Fig. <a href="supporting-information-co-regulation-of-paralog-genes-in-the-three-dimensional-chromatin-architecture.html#fig:paraVSnonPara">A.1</a>E).</p>
<p>Since most genome duplication events in humans emerge through tandem duplications <span class="citation">(Newman et al., <a href="references.html#ref-Newman2015">2015</a>)</span>, we expected some co-localization among pairs of paralog genes. Indeed <span class="math inline">\(24.9\%\)</span> of paralog pairs are located on the same chromosome. We compared this to random expectation by sampling random gene pairs from all protein coding human genes and found only <span class="math inline">\(5.3\%\)</span> of randomly sampled gene pairs on the same chromosome (<span class="math inline">\(p&lt;10^{-16}\)</span>, Fig. <a href="ch-paralog-regulaion.html#fig:paraData">2.1</a>A).</p>
<div class="figure" style="text-align: center"><span id="fig:paraData"></span>
<img src="figures/paralog/fig1_09.png" alt="(A) Percent of paralog (red) and random (dark grey) gene pairs that are located on the same chromosome. The error bar indicates the standard deviation observed in 10 times replicated random sampling of gene pairs. (B) Genomic distance distribution of paralogs gene pairs (top), random gene pairs (center) and gene pairs sampled according to distance distribution of paralogs (bottom). Distances are measured in kilo base pairs (kb) between TSS of genes in pairs. P-values are calculated using Wilcoxon rank-sum test. (C) Percent of paralog (red) and sampled (grey) gene pairs that are transcribed from the same strand. Only pairs on the the same chromosome within 1 Mb are considered here. Error bars indicate the standard deviation observed in 10 times replicated sampling of gene pairs. (D) Boxplot of the genomic distance between paralogs and sampled gene pairs with the same or opposite strands. (E) Distribution of Pearson correlation coefficients of gene expression values in four independent data sets between paralog gene pairs (red) and sampled control gene pairs (grey). White boxes show 25th, 50th and 75th percent quantile of the data and the filled areas indicate the density distribution.." width="50%" />
<p class="caption">
Figure 2.1: <strong>(A)</strong> Percent of paralog (red) and random (dark grey) gene pairs that are located on the same chromosome. The error bar indicates the standard deviation observed in 10 times replicated random sampling of gene pairs. <strong>(B)</strong> Genomic distance distribution of paralogs gene pairs (top), random gene pairs (center) and gene pairs sampled according to distance distribution of paralogs (bottom). Distances are measured in kilo base pairs (kb) between TSS of genes in pairs. P-values are calculated using Wilcoxon rank-sum test. <strong>(C)</strong> Percent of paralog (red) and sampled (grey) gene pairs that are transcribed from the same strand. Only pairs on the the same chromosome within 1 Mb are considered here. Error bars indicate the standard deviation observed in 10 times replicated sampling of gene pairs. <strong>(D)</strong> Boxplot of the genomic distance between paralogs and sampled gene pairs with the same or opposite strands. <strong>(E)</strong> Distribution of Pearson correlation coefficients of gene expression values in four independent data sets between paralog gene pairs (red) and sampled control gene pairs (grey). White boxes show 25th, 50th and 75th percent quantile of the data and the filled areas indicate the density distribution..
</p>
</div>

<p>We further analysed whether paralog pairs tend to be located in close genomic distance on the same chromosomes. We compared the distance between paralog gene pairs to the distance of completely random genes on the same chromosome. As expected there is a strong bias of genomic co-localization among paralog gene pairs that is not observed for random gene pairs (<span class="math inline">\(p=4.3\times10^{-32}\)</span>, Fig <a href="ch-paralog-regulaion.html#fig:paraData">2.1</a>B).</p>
<p>We also observed that close paralog genes show more often than expected the same transcription orientation. From all paralog pairs within 1 Mb on the same chromosome <span class="math inline">\(66.1\%\)</span> have the same sense. This is significantly more than for randomly sampled genes with the same distance (52.6%, <span class="math inline">\(p=3.2\times10^{-18}\)</span>, Fig. <a href="ch-paralog-regulaion.html#fig:paraData">2.1</a>C).</p>
<p>Furthermore, we observed that paralogs in the same strand are closer to each other on the chromosome than pairs in opposite strands (<span class="math inline">\(p=3.48\times10^{-8}\)</span>, Fig. <a href="ch-paralog-regulaion.html#fig:paraData">2.1</a>D).</p>
<p>Together, this shows that paralogs tend to be located within short linear distance on the same chromosome and same transcription sense, which might enable coordinated regulation by shared regulatory mechanisms.</p>
</div>
<div id="co-expression-of-paralog-gene-pairs-across-tissues" class="section level3">
<h3><span class="header-section-number">2.3.2</span> Co-expression of paralog gene pairs across tissues</h3>
<p>To assess whether paralog genes tend to be indeed co-regulated we compared gene expression of paralog gene pairs over several human tissues and cell lines.</p>
<p>We compared the Pearson correlation coefficient (PCC) of gene expression values over <span class="math inline">\(n = 18\)</span> cell-lines analysed by the ENCODE consortium by RNA-seq <span class="citation">(Djebali et al., <a href="references.html#ref-Djebali2012">2012</a>)</span>. The distribution of PCC among paralog genes is bimodal with one peak around <span class="math inline">\(-0.1\)</span> and another at nearly <span class="math inline">\(1.0\)</span>, which indicates that there exists a group of paralog pairs without expression correlation and that the expression of other paralogs is highly positively correlated. Notably, we did not find the latter signal for positive correlation in our control set of carefully sampled gene pairs (Fig. <a href="ch-paralog-regulaion.html#fig:paraData">2.1</a>E).</p>
<p>We repeated the analysis with three other independent gene expression data sets form FANTOM5 (<span class="math inline">\(n=56\)</span> tissues) <span class="citation">(Forrest et al., <a href="references.html#ref-Forrest2014">2014</a>)</span>, GTEx (<span class="math inline">\(n=53\)</span> tissues) <span class="citation">(Ardlie et al., <a href="references.html#ref-GTExConsortium2015">2015</a>)</span> and the Illumina Body Map (<span class="math inline">\(n=16\)</span> tissues), which we retrieved from the EBI Expression Atlas <span class="citation">(Petryszak et al., <a href="references.html#ref-Petryszak2015">2015</a>)</span>. In all data sets we found more positively correlated paralog pairs compared to the sampled gene pairs (Fig. <a href="ch-paralog-regulaion.html#fig:paraData">2.1</a>E). This shows that many paralogs are expressed with high coordination in a tissue specific manner.</p>
</div>
<div id="paralog-genes-share-enhancers" class="section level3">
<h3><span class="header-section-number">2.3.3</span> Paralog genes share enhancers</h3>
<p>We hypothesised that common gene regulation of close paralog genes is likely to be facilitated by shared enhancer elements. Indeed we found that paralog gene pairs within 1 Mb on the same chromosome are associated to the same enhancer elements more often than expected by chance (Fig. <a href="ch-paralog-regulaion.html#fig:sharedEnhancer">2.2</a>). We estimated the expected background distribution of shared enhancers by carefully sampling gene pairs with the same distributions as paralogs in distances and associated enhancers to single genes (Fig. <a href="supporting-information-co-regulation-of-paralog-genes-in-the-three-dimensional-chromatin-architecture.html#fig:samplingDistEh">A.4</a>, section <a href="ch-paralog-regulaion.html#sec:paralog-methods">2.2</a>).</p>
<div class="figure" style="text-align: center"><span id="fig:sharedEnhancer"></span>
<img src="figures/paralog/fig2_09.png" alt="Shared enhancers among paralog gene pairs. (A) Percent of close paralog (red) and sampled control (grey) gene pairs with at least one shared enhancer. (B) Percent of gene pairs versus number of shared enhancers for paralog and sampled control gene pairs." width="50%" />
<p class="caption">
Figure 2.2: Shared enhancers among paralog gene pairs. <strong>(A)</strong> Percent of close paralog (red) and sampled control (grey) gene pairs with at least one shared enhancer. <strong>(B)</strong> Percent of gene pairs versus number of shared enhancers for paralog and sampled control gene pairs.
</p>
</div>

<p>While <span class="math inline">\(27.2\)</span>% of the paralog gene pairs have at least one enhancer in common, we observed this for only <span class="math inline">\(11.7\)</span>% of the sampled gene pairs (<span class="math inline">\(p=4.2\times10^{-40}\)</span>, Fig. <a href="ch-paralog-regulaion.html#fig:sharedEnhancer">2.2</a>A). This could be replicated when comparing against sampled gene pairs where in addition to distance and number of enhancers linked to single genes, also the transcription sense and gene length were taken into account during sampling of control gene pairs (<span class="math inline">\(p=3.4\times10^{-41}\)</span> and <span class="math inline">\(p=5\times10^{-30}\)</span>, respectively; Fig. <a href="supporting-information-co-regulation-of-paralog-genes-in-the-three-dimensional-chromatin-architecture.html#fig:ehBySampType">A.7</a>). Next, we compared the percent of gene pairs with shared enhancers as a function of the number of shared enhancers between paralogs and sampled gene pairs. We observed that paralog pairs are enriched for higher number of shared enhancers compared to the sampled gene pairs (Fig. <a href="ch-paralog-regulaion.html#fig:sharedEnhancer">2.2</a>B). Together, these results indicate that paralog genes are more often co-regulated by common enhancer elements than other genes.</p>
</div>
<div id="co-localization-of-paralogs-in-tads" class="section level3">
<h3><span class="header-section-number">2.3.4</span> Co-localization of paralogs in TADs</h3>
<p>To facilitate their function in gene regulation, distal enhancer elements need to interact physically via chromatin looping with promoter elements at the TSS of their target genes. These looping interactions occur frequently within so called topological associating domains (TADs). These are regions of hundreds of kb that show high rates of self-interactions and few interactions across domain boundaries in genome-wide Hi-C experiments <span class="citation">(Dixon et al., <a href="references.html#ref-Dixon2012">2012</a>; Rao et al., <a href="references.html#ref-Rao2014">2014</a>)</span>. Genes within the same TAD are therefore likely to have common gene regulatory programs <span class="citation">(Le Dily et al., <a href="references.html#ref-LeDily2014">2014</a>; Nora et al., <a href="references.html#ref-Nora2012">2012</a>)</span>.</p>
<p>We used TADs from Hi-C experiments in eight different human cell-types (HeLa, HUVEC, K562, KBM7, NHEK, IMR90, GM12878, and hESC) from two recent studies  <span class="citation">(Dixon et al., <a href="references.html#ref-Dixon2012">2012</a>; Rao et al., <a href="references.html#ref-Rao2014">2014</a>)</span>. Notably, the called TADs differ in size between the two publications due to different resolution of Hi-C experiments and different algorithms used to call them from Hi-C contact matrices (Fig. <a href="supporting-information-co-regulation-of-paralog-genes-in-the-three-dimensional-chromatin-architecture.html#fig:TADsize">A.8</a>). TADs from <span class="citation">(Rao et al., <a href="references.html#ref-Rao2014">2014</a>)</span> have a median size of around 240 kb and are nested, so that several small domains can occur within one ore more larger domains. In contrast TADs from <span class="citation">(Dixon et al., <a href="references.html#ref-Dixon2012">2012</a>)</span> are of 1 Mb on average and are defined as non-overlapping genomic intervals.</p>
<p>We hypothesised that paralog gene pairs might be located more often in the same TAD than expected by chance. Indeed, we found that, depending on cell-type and study, between <span class="math inline">\(35\%\)</span> and <span class="math inline">\(73\%\)</span> of close paralog pairs are located in the same TAD (Fig. <a href="ch-paralog-regulaion.html#fig:TAD">2.3</a>A). In seven out of nine data sets this difference was significant (<span class="math inline">\(p&lt;0.05\)</span>) with respect to the sampled control gene pairs with the same linear distance. We also calculated a set of <span class="math inline">\(n=2,624\)</span> stable TADs that are found in more than 50% of cell types analysed in <span class="citation">(Rao et al., <a href="references.html#ref-Rao2014">2014</a>)</span>. Notably, we found for paralog pairs a <span class="math inline">\(1.25\)</span> fold enrichment to be located in the same stable TADs compared to sampled gene pairs (<span class="math inline">\(p=0.00013\)</span>, stable_TADs in Fig. <a href="ch-paralog-regulaion.html#fig:TAD">2.3</a>A).</p>
<div class="figure" style="text-align: center"><span id="fig:TAD"></span>
<img src="figures/paralog/fig3_09.png" alt="(A) Co-localization of close paralog genes within the same TAD compared against sampled gene pairs for TAD data sets from different cell types and studies. The first seven bars show values for TADs called in HeLa, HUVEC, K562, KBM7, NHEK, IMR90, and GM12878 cells by (Rao et al., 2014). The eighth bar shows the value for stable TADs across cell types form this study (at least 90% reciprocal overlap in 50% of cells). The last two bars show data for TADs called in hESC and IMR90 cells by (Dixon et al., 2012). Error bars indicate standard deviation in 10 times replicated sampling of gene pairs. P-values are computed using Fisher’s exact test. (B) Percent of gene pairs annotated to same A/B compartment according to Hi-C data in GM12878 cells from (Rao et al., 2014). Pairs located in the very same compartment interval were excluded. (C) Percent of gene pairs annotated to same sub compartment (A1, A2, B1, B2, B3, B4) according to (Rao et al., 2014). Pairs located in the same subcompartment interval were excluded. (D) Normalized Hi-C contact frequencies between TSSs of distal paralog gene pairs and sampled background gene pairs. (E) Promoter capture-C contact frequencies between distal paralog gene pairs and sampled background gene pairs." width="50%" />
<p class="caption">
Figure 2.3: <strong>(A)</strong> Co-localization of close paralog genes within the same TAD compared against sampled gene pairs for TAD data sets from different cell types and studies. The first seven bars show values for TADs called in HeLa, HUVEC, K562, KBM7, NHEK, IMR90, and GM12878 cells by <span class="citation">(Rao et al., <a href="references.html#ref-Rao2014">2014</a>)</span>. The eighth bar shows the value for stable TADs across cell types form this study (at least 90% reciprocal overlap in 50% of cells). The last two bars show data for TADs called in hESC and IMR90 cells by <span class="citation">(Dixon et al., <a href="references.html#ref-Dixon2012">2012</a>)</span>. Error bars indicate standard deviation in 10 times replicated sampling of gene pairs. P-values are computed using Fisher’s exact test. <strong>(B)</strong> Percent of gene pairs annotated to same A/B compartment according to Hi-C data in GM12878 cells from <span class="citation">(Rao et al., <a href="references.html#ref-Rao2014">2014</a>)</span>. Pairs located in the very same compartment interval were excluded. <strong>(C)</strong> Percent of gene pairs annotated to same sub compartment (A1, A2, B1, B2, B3, B4) according to <span class="citation">(Rao et al., <a href="references.html#ref-Rao2014">2014</a>)</span>. Pairs located in the same subcompartment interval were excluded. <strong>(D)</strong> Normalized Hi-C contact frequencies between TSSs of distal paralog gene pairs and sampled background gene pairs. <strong>(E)</strong> Promoter capture-C contact frequencies between distal paralog gene pairs and sampled background gene pairs.
</p>
</div>

<p>Beside TADs, Hi-C interaction maps have revealed interaction patterns of two compartments (A and B) that alternate along chromosomes in intervals of several Mb. Thereby loci in A compartment preferentially associate with other loci in A and loci in B with others in B <span class="citation">(Dekker et al., <a href="references.html#ref-Dekker2013">2013</a>; Lieberman-Aiden et al., <a href="references.html#ref-Lieberman-Aiden2009">2009</a>; Rao et al., <a href="references.html#ref-Rao2014">2014</a>)</span>. We therefore asked whether pairs of paralogs from the same chromosome are preferentially located within the same compartment (both A or both B) whereby we excluded pairs that are in the same compartment interval. We found that <span class="math inline">\(56.4\%\)</span> of paralogs on the same chromosome but not in the same compartment interval are in compartments of the same type. This was only observed for <span class="math inline">\(49.2\%\)</span> of sampled pairs (<span class="math inline">\(p=0.0046\)</span>, Fig. <a href="ch-paralog-regulaion.html#fig:TAD">2.3</a>B). Next we tested the same for recently distinguished sub-compartment types from high-resolution Hi-C interaction maps <span class="citation">(Rao et al., <a href="references.html#ref-Rao2014">2014</a>)</span>. Again, paralogs are enriched to be located within the same sub-compartment type (<span class="math inline">\(38.9\%\)</span> vs. <span class="math inline">\(31.6\%\)</span>, <span class="math inline">\(p=0.0046\)</span>, Fig. <a href="ch-paralog-regulaion.html#fig:TAD">2.3</a>C).</p>
<p>These results show that close paralogs are enriched to be located in the same regulatory unit of the genome as defined both by TADs and compartments.</p>
</div>
<div id="distal-paralog-pairs-are-enriched-for-long-range-chromatin-contacts" class="section level3">
<h3><span class="header-section-number">2.3.5</span> Distal paralog pairs are enriched for long-range chromatin contacts</h3>
<p>Since it was shown that actively transcribed genes are localized in the same active spatial compartments and tend to contact each other frequently in the nucleus (at their promoters <span class="citation">(Cremer et al., <a href="references.html#ref-Cremer2015">2015</a>; Mifsud et al., <a href="references.html#ref-Mifsud2015">2015</a>)</span>) we hypothesised that this might be the case for distal paralogs on the same chromosome too. As spatial proximity can be approximated by Hi-C contact frequencies <span class="citation">(Lieberman-Aiden et al., <a href="references.html#ref-Lieberman-Aiden2009">2009</a>)</span> we compared the number of normalized Hi-C contacts between TSS of distal paralog genes (that have promoters separated by more than 1 Mb) to the sampled gene pairs with the same linear distances distribution. We used recently published in situ Hi-C data from IMR90 cells at 5kb bin-size resolution <span class="citation">(Rao et al., <a href="references.html#ref-Rao2014">2014</a>)</span> and observed significantly more normalized chromatin interactions between paralog genes compared to sampled control gene pairs (<span class="math inline">\(p=0.0022\)</span>, Fig. <a href="ch-paralog-regulaion.html#fig:TAD">2.3</a>D). We furthermore used an independent data set of high resolution promoter-promoter interactions measured by capture Hi-C <span class="citation">(Mifsud et al., <a href="references.html#ref-Mifsud2015">2015</a>)</span>. Again, we observed a strong enrichment of promoter-promoter interactions between distal paralogs compared to control genes pairs (<span class="math inline">\(p=0.027\)</span>, Fig. <a href="ch-paralog-regulaion.html#fig:TAD">2.3</a>E). This shows that also distal paralogs are enriched for long-range interactions, indicating that they tend to be in closer spatial proximity than other genes.</p>
</div>
<div id="close-paralogs-have-fewer-contacts-than-expected" class="section level3">
<h3><span class="header-section-number">2.3.6</span> Close paralogs have fewer contacts than expected</h3>
<p>The observed enrichment of Hi-C contacts of paralogs is distance dependent. We observe for close paralogs fewer Hi-C contacts than for equally distant sampled gene pairs (Fig. <a href="ch-paralog-regulaion.html#fig:closePairs">2.4</a>A). To analyse this in more detail we focused on only those pairs on the same chromosome that have a TSS distance of at least 10kb but less than 1Mb. This is the distance range of most paralog pairs and allows to separate genes in Hi-C interaction maps and TADs (Fig. <a href="supporting-information-co-regulation-of-paralog-genes-in-the-three-dimensional-chromatin-architecture.html#fig:closePairsInTAD">A.9</a>A). Consequently, we observe paralogs more often in the same TAD in eight out of nine data sets for this distance range (Fig. <a href="supporting-information-co-regulation-of-paralog-genes-in-the-three-dimensional-chromatin-architecture.html#fig:closePairsInTAD">A.9</a>B). For these pairs we observe significant lower Hi-C contact frequencies if pairs are within the same IMR90 TAD <span class="citation">(Rao et al., <a href="references.html#ref-Rao2014">2014</a>)</span> as compared to sampled genes (<span class="math inline">\(p=0.00094\)</span>) but not if pairs are in different TADs (<span class="math inline">\(p=0.81\)</span>, Fig. <a href="ch-paralog-regulaion.html#fig:closePairs">2.4</a>B). We got comparable results when analysing the Capture Hi-C data the same way (Fig. <a href="supporting-information-co-regulation-of-paralog-genes-in-the-three-dimensional-chromatin-architecture.html#fig:closePairsInTAD">A.9</a>C). Next, we tested whether this can be explained by the nested sub-TAD structure of TADs called from high-resolution Hi-C in IMR90 <span class="citation">(Rao et al., <a href="references.html#ref-Rao2014">2014</a>)</span>. We divided pairs into four groups, namely, ’no TAD’, if both pairs are not in any TAD, ’different TAD’, if pairs do not have at least one TAD in common, ’different sub-TADs’, if they have at least one TAD in common but are in different sub-TADs, and ’same sub-TAD’, if they overlap exactly the same set of TADs. While we saw that paralogs are more often in the no TAD group (<span class="math inline">\(p=1.4\times10^{-20}\)</span>), we found that they were highly depleted from the different TAD group (<span class="math inline">\(p=1.6\times10^{-40}\)</span>) and highly enriched to be located within the same sub-TAD (<span class="math inline">\(p=4.2\times10^{-9}\)</span>, Fig. <a href="ch-paralog-regulaion.html#fig:closePairs">2.4</a>C). However, although not always significant, paralogs have fewer Hi-C contacts than sampled gene pairs in all of these groups (Fig. <a href="ch-paralog-regulaion.html#fig:closePairs">2.4</a>D). In addition, close paralogs within the same TAD share more enhancers than close paralogs not being in the same TAD (Fig. <a href="ch-paralog-regulaion.html#fig:closePairs">2.4</a>E). However, the positive correlation of gene expression over different tissues is not significantly higher for paralogs whether they are in the same TAD or not (Fig. <a href="supporting-information-co-regulation-of-paralog-genes-in-the-three-dimensional-chromatin-architecture.html#fig:ExpByTAD">A.10</a>).</p>
<div class="figure" style="text-align: center"><span id="fig:closePairs"></span>
<img src="figures/paralog/fig4_09.png" alt="(A) Normalized Hi-C contacts by genomic distance between paralog (red) and sampled (grey) gene pairs. Lines show linear regression fit separately for paralogs (red) and sampled (grey) pairs with 95% confidence intervals in shaded areas. (B) Normalized Hi-C contacts between pairs of paralogs (red) and sampled gene pairs (grey) for the groups: $&lt;$10kb genomic distance, located in the same TAD, not in the same TAD, and with genomic distance $&gt;$1000kb. (C) Number of gene pairs located either in no TAD, in different TADs (or only one pair member in a TAD), both in a TAD but in different sub-TADs, or within the same sub-TAD, for paralogs (red) and sampled (grey) pairs. TADs from IMR90 cells from (Rao et al., 2014) were used, which nested in contrast to TAD calls from (Dixon et al., 2012). (D) Normalized Hi-C contacts between pairs of paralogs (red) and sampled gene pairs (grey) for the four groups of pairs in sub-TAD structures shown in (C). (E) Percent of gene pairs with at least one shared enhancer for paralog genes (red) and sampled control genes (grey) separated for pairs in the same IMR90 TAD (left) or not (right)." width="50%" />
<p class="caption">
Figure 2.4: <strong>(A)</strong> Normalized Hi-C contacts by genomic distance between paralog (red) and sampled (grey) gene pairs. Lines show linear regression fit separately for paralogs (red) and sampled (grey) pairs with 95% confidence intervals in shaded areas. <strong>(B)</strong> Normalized Hi-C contacts between pairs of paralogs (red) and sampled gene pairs (grey) for the groups: $&lt;$10kb genomic distance, located in the same TAD, not in the same TAD, and with genomic distance $&gt;$1000kb. <strong>(C)</strong> Number of gene pairs located either in no TAD, in different TADs (or only one pair member in a TAD), both in a TAD but in different sub-TADs, or within the same sub-TAD, for paralogs (red) and sampled (grey) pairs. TADs from IMR90 cells from <span class="citation">(Rao et al., <a href="references.html#ref-Rao2014">2014</a>)</span> were used, which nested in contrast to TAD calls from <span class="citation">(Dixon et al., <a href="references.html#ref-Dixon2012">2012</a>)</span>. <strong>(D)</strong> Normalized Hi-C contacts between pairs of paralogs (red) and sampled gene pairs (grey) for the four groups of pairs in sub-TAD structures shown in (C). <strong>(E)</strong> Percent of gene pairs with at least one shared enhancer for paralog genes (red) and sampled control genes (grey) separated for pairs in the same IMR90 TAD (left) or not (right).
</p>
</div>

<p>In summary, we observed that while close paralogs (situated at less than 1Mb) have more shared enhancers if they are in the same TAD than not, these within TAD paralog pairs have fewer contacts compared to other within TAD pairs of genes.</p>
</div>
<div id="paralogs-in-mouse-and-dog-genome" class="section level3">
<h3><span class="header-section-number">2.3.7</span> Paralogs in mouse and dog genome</h3>
<p>Next, we asked whether the co-localization and co-regulation of paralogs is conserved in other species. For this, we conducted an analogous analysis with paralog gene pairs from mouse (<em>M. musculus</em>) and dog (<em>C. familaris</em>) genomes. Similar as for human data, we found that more than two third of the genes had at least one paralog copy (Fig. <a href="supporting-information-co-regulation-of-paralog-genes-in-the-three-dimensional-chromatin-architecture.html#fig:paralogsSpecies">A.11</a>A,D), paralog pairs clustered on the same chromosome (Fig. <a href="supporting-information-co-regulation-of-paralog-genes-in-the-three-dimensional-chromatin-architecture.html#fig:paralogsSpecies">A.11</a>B,E), and had close linear distances (Fig. <a href="supporting-information-co-regulation-of-paralog-genes-in-the-three-dimensional-chromatin-architecture.html#fig:paralogsSpecies">A.11</a>C,F).</p>
<p>We sampled control gene pairs with the same distance distribution as paralogs for both species separately (Fig. <a href="supporting-information-co-regulation-of-paralog-genes-in-the-three-dimensional-chromatin-architecture.html#fig:paralogsSpecies">A.11</a>C,F). We used TADs from recently published Hi-C data in liver cells of mouse and dog <span class="citation">(Vietri Rudan et al., <a href="references.html#ref-VietriRudan2015">2015</a>)</span>, which have a size distribution comparable to TADs from human cells (Fig. <a href="supporting-information-co-regulation-of-paralog-genes-in-the-three-dimensional-chromatin-architecture.html#fig:TADsize">A.8</a>). We computed the fraction of paralog pairs that are located in the same TAD for both species. Consistent with the observation in human, we found that paralogs tend to colocalize more frequently within the same TAD in mouse (<span class="math inline">\(p=7.2 \times 10^{-22}\)</span>) and dog (<span class="math inline">\(p=0.00064\)</span>) than expected by chance (Fig. <a href="ch-paralog-regulaion.html#fig:paralogsMouseDog">2.5</a>A). We also quantified directly the contact frequencies between promoters of distal paralogs on the same chromosome and found them significantly more frequently in contact than sampled gene pairs with the same genomic distance for paralogs in mouse (<span class="math inline">\(p=7\times 10^{-7}\)</span>) and dog (<span class="math inline">\(p=0.008\)</span>) (Fig. <a href="ch-paralog-regulaion.html#fig:paralogsMouseDog">2.5</a>B). Together, these results indicate that enriched long-range interactions between paralogs are not human specific but rather a general evolutionary conserved feature of genome organization.</p>
<div class="figure" style="text-align: center"><span id="fig:paralogsMouseDog"></span>
<img src="figures/paralog/fig5_09.png" alt="(A) Co-occurrence of close paralog genes with the same TAD in mouse (left panel) and dog (right panel). (B) Hi-C contacts between promoter of distal gene pairs in Hi-C experiments in liver cells from mouse (left panel) and dog (right panel). Hi-C data and TAD calls were taken from (Vietri Rudan et al., 2015)." width="50%" />
<p class="caption">
Figure 2.5: <strong>(A)</strong> Co-occurrence of close paralog genes with the same TAD in mouse (left panel) and dog (right panel). <strong>(B)</strong> Hi-C contacts between promoter of distal gene pairs in Hi-C experiments in liver cells from mouse (left panel) and dog (right panel). Hi-C data and TAD calls were taken from <span class="citation">(Vietri Rudan et al., <a href="references.html#ref-VietriRudan2015">2015</a>)</span>.
</p>
</div>

</div>
<div id="orthologs-of-human-paralogs-show-conserved-co-localization" class="section level3">
<h3><span class="header-section-number">2.3.8</span> Orthologs of human paralogs show conserved co-localization</h3>
<p>Next, we wanted to test more directly whether the spatial co-localization of human paralogs is indeed conserved during evolution. In cases where the gene duplication event occurred before the separation of human and mouse (or human and dog) we can eventually assign each human gene of a pair of paralogs to one ortholog in mouse (or dog genomes) (Fig. <a href="supporting-information-co-regulation-of-paralog-genes-in-the-three-dimensional-chromatin-architecture.html#fig:orthologModel">A.12</a>).</p>
<p>We could map <span class="math inline">\(37.1\%\)</span> (<span class="math inline">\(n=579\)</span>) and <span class="math inline">\(34.6\%\)</span> (<span class="math inline">\(n=540\)</span>) of the close human paralogs to one-to-one orthologs in mouse and dog, respectively (Fig. <a href="supporting-information-co-regulation-of-paralog-genes-in-the-three-dimensional-chromatin-architecture.html#fig:orthologsSpecies">A.13</a>A,D). We hypothesised that the two one-to-one orthologs of human paralog pairs would also be close in the mouse and dog genomes. Indeed, we found that the orthologs of human paralogs tend to cluster on the same chromosome (Fig. <a href="supporting-information-co-regulation-of-paralog-genes-in-the-three-dimensional-chromatin-architecture.html#fig:orthologsSpecies">A.13</a>B,E) and are biased for close linear distances (Fig. <a href="supporting-information-co-regulation-of-paralog-genes-in-the-three-dimensional-chromatin-architecture.html#fig:orthologsSpecies">A.13</a>C,F).</p>
<p>We further investigated how many one-to-one orthologs of the human paralog pairs were located in the same TAD in mouse and dog genomes. Although not significant, we found that mouse orthologs of close human paralogs share more often the same TAD in mouse than orthologs of sampled human gene pairs (<span class="math inline">\(80\%\)</span> vs. <span class="math inline">\(76\%\)</span>, <span class="math inline">\(p=0.11\)</span>; Fig. <a href="ch-paralog-regulaion.html#fig:orthologsMouseDog">2.6</a>A). Significant enrichment was observed with orthologs in the dog genome (<span class="math inline">\(85\%\)</span> vs. <span class="math inline">\(77\%\)</span>, <span class="math inline">\(p=0.0016\)</span>; Fig. <a href="ch-paralog-regulaion.html#fig:orthologsMouseDog">2.6</a>A).</p>
<div class="figure" style="text-align: center"><span id="fig:orthologsMouseDog"></span>
<img src="figures/paralog/fig6_09.png" alt="One-to-one orthologs of human paralog genes in mouse and dog genome. (A) Percent of mouse (left) and dog (right) orthologs of human paralog pairs that are in the same TAD in the mouse and dog genome, respectively. (B) Normalized Hi-C contacts between promoters of one-to-one orthologs of human distal paralogs in the mouse (left) and dog (right) genome. (C) Percent of gene pairs with conserved co-localization. Orthologs in the same TAD in mouse (left) and dog (right) as percent of all orthologs of human paralog pairs that are in the same TAD in human. For human TADs from IMR90 cells from (Rao et al., 2014) were used." width="50%" />
<p class="caption">
Figure 2.6: One-to-one orthologs of human paralog genes in mouse and dog genome. <strong>(A)</strong> Percent of mouse (left) and dog (right) orthologs of human paralog pairs that are in the same TAD in the mouse and dog genome, respectively. <strong>(B)</strong> Normalized Hi-C contacts between promoters of one-to-one orthologs of human distal paralogs in the mouse (left) and dog (right) genome. <strong>(C)</strong> Percent of gene pairs with conserved co-localization. Orthologs in the same TAD in mouse (left) and dog (right) as percent of all orthologs of human paralog pairs that are in the same TAD in human. For human TADs from IMR90 cells from <span class="citation">(Rao et al., <a href="references.html#ref-Rao2014">2014</a>)</span> were used.
</p>
</div>

<p>For distal human paralogs we quantified the promoter contacts of their orthologs in mouse and dog and found enriched Hi-C contacts in mouse (<span class="math inline">\(p=0.011\)</span>) and dog (<span class="math inline">\(p=2.4\times 10^{-5}\)</span>; Fig. <a href="ch-paralog-regulaion.html#fig:orthologsMouseDog">2.6</a>B).</p>
<p>These results show that both the co-localization of paralogs in TADs and the contacts between distal paralogs are only weekly conserved at the evolutionary distances examined here. For example, we see that given a pair of human genes in the same TAD the likelihood of their orthologs being in the same TAD in mouse or dog is the same whether they are paralogs or not (Fig. <a href="ch-paralog-regulaion.html#fig:orthologsMouseDog">2.6</a>C).</p>
<p>All together, our results support the notion that tandem duplications generate paralog gene pairs that are selected if they accommodate in TADs but following evolutionary events allow their reorganization outside TADs. While within oganisms distal paralog genes are coordinated, such coordination can be eventually erased by evolution.</p>
</div>
</div>
<div id="discussion" class="section level2">
<h2><span class="header-section-number">2.4</span> DISCUSSION</h2>
<p>The generation of large datasets of gene expression across multiple tissues allowed the observation of clusters of pairs and triplets of co-expressed genes in higher eukaryotes (e.g. in <em>Drosophila</em> <span class="citation">(Boutanaev et al., <a href="references.html#ref-Boutanaev2002">2002</a>)</span> or in mammals <span class="citation">(Purmann et al., <a href="references.html#ref-Purmann2007">2007</a>)</span>) and it was previously suspected that the structure of chromatin would have to do with this <span class="citation">(Sproul et al., <a href="references.html#ref-Sproul2005">2005</a>)</span>, particularly cis-acting units <span class="citation">(Purmann et al., <a href="references.html#ref-Purmann2007">2007</a>)</span>. The discovery and characterization of topologically associating domains (TADs) has finally brought to the light the chromatin structure that could be responsible for this co-regulation.</p>
<p>To study the interplay between TADs, gene co-regulation and evolution in the human genome, we decided to focus on pairs of paralogs because they have a tendency to be produced by tandem duplication <span class="citation">(Newman et al., <a href="references.html#ref-Newman2015">2015</a>)</span> and, because of homology, result in proteins with related functions. However, the particular emergence and evolution of paralogs are probably responsible for special properties that distinguish them from non-paralog genes as we described: greater gene length, more enhancers, as well as a shorter distance to the next enhancer. These differences, which could be partially explained by the observation that paralogs are more often tissue specific (Fig. <a href="supporting-information-co-regulation-of-paralog-genes-in-the-three-dimensional-chromatin-architecture.html#fig:paraVSnonPara">A.1</a>F), complicated the methodology for choosing meaningful control pairs (see section <a href="ch-paralog-regulaion.html#sec:paralog-methods">2.2</a>).</p>
<p>Once we ensured the generation of the appropriate backgrounds, we could study the position of pairs of paralogs respect to TADs. This allowed us to test, on the one hand, the resilience of TADs to genome shuffling and, on the other hand, the rate of accommodation and gain of functionally related genes. Possibly, the generation of paralogs by tandem duplication might continuously impose a strain in the pre-existing genomic and regulatory structure, but also a chance for the evolution of new functionality.</p>
<p>On the one hand, we observed many pairs of paralogs within TADs. On the other hand, pairs of paralogs in different TADs, however distant from each other, tend to have more contacts than control gene pairs. This suggests a many-step mechanism where first tandem duplication fits TAD structure but then subsequent chromosomal rearrangements relocate paralogs at larger distances (while keeping contacts) and eventually reorganization of regulatory control allow their increased independence being eventually placed even in different chromosomes where contact is no longer necessary. Thus, TADs are units of co-regulation but do not have a strong preference for keeping co-regulated genes within during evolution. This model agrees with the recent work from Lan and Pritchard reporting that young pairs of paralogs are generally close in the genome <span class="citation">(Lan and Pritchard, <a href="references.html#ref-Lan2016">2016</a>)</span>.</p>
<p>A second effect that we observed was the existence of fewer contacts between close pairs of paralogs than in comparable pairs of non-paralog genes, particularly if they are in the same TAD (Fig. <a href="ch-paralog-regulaion.html#fig:closePairs">2.4</a>B), while sharing more enhancers (Fig. <a href="ch-paralog-regulaion.html#fig:closePairs">2.4</a>E). This result could reflect the existence of pairs of paralogs encoding proteins that replace each other, for example sub-units of a complex that occupy the same position in a protein complex but are expressed in different cells. One such case is exemplified by CBX2, CBX4 and CBX8, which occupy neighbouring positions within the same TAD in human chromosome 17 and encode replaceable subunits of the polycomb repressive complex 1 (PRC1) complex involved in epigenetic regulation of cell specification <span class="citation">(Becker et al., <a href="references.html#ref-Becker2015">2015</a>)</span>. The expression of such groups of paralogs require active coordination to ensure exclusive expression of only one gene or a subset of genes per condition, resulting in patterns of divergent expression. Since there might be also conditions where none of these genes are expressed, such divergent expression patterns are different from negative correlation.</p>
<p>Previous work studying gene expression of duplicated genes already studied how after gene duplication paralogs tend to diverge in their expression <span class="citation">(Huminiecki, <a href="references.html#ref-Huminiecki2004">2004</a>; Makova and Li, <a href="references.html#ref-Makova2003">2003</a>; Rogozin et al., <a href="references.html#ref-Rogozin2014">2014</a>)</span> but it was observed that while some paralogs are co-expressed some others have negative correlation across tissues <span class="citation">(Makova and Li, <a href="references.html#ref-Makova2003">2003</a>)</span>. Our interpretation of these observations together with our results is that the initial tandem duplication event forming a paralog is advantageous to situate the new copy in an environment that allows its controlled regulation, ideally under the same regulatory elements than the original copy, and this can be attained by duplicating both gene and surrounding regulatory elements. This would preclude the duplication of genes with very entangled regulatory associations. Once this happens, if the new protein evolves into a replacement, then the regulatory constraints on its coding gene are strong and there would be a tendency to keep it in the vicinity of the older gene so that a divergent pattern of expression can be ensured.</p>
<p>To support this hypothesis, we contrasted our data with the data collected in the HIPPIE database of experimentally verified human protein-protein interactions <span class="citation">(Schaefer et al., <a href="references.html#ref-Schaefer2012">2012</a>)</span>. We observed the well-known fact that paralog pairs generally encode for proteins that interact more often than non-paralog proteins (Fig. <a href="supporting-information-co-regulation-of-paralog-genes-in-the-three-dimensional-chromatin-architecture.html#fig:PPI">A.14</a>). But, most importantly, we observed that the chances of close pairs of genes to encode for interacting proteins raise <span class="math inline">\(2.3\)</span>-fold if they are in the same TAD, while, in contrast, if these genes are paralogs the difference is much smaller (<span class="math inline">\(1.2\)</span>-fold, Fig. <a href="supporting-information-co-regulation-of-paralog-genes-in-the-three-dimensional-chromatin-architecture.html#fig:PPI">A.14</a>). We interpret this result as evidence for a significant population of within TAD paralog pairs encoding for non-interacting proteins, which supports our hypothesis that paralog pairs within the same TAD would have a tendency to encode for proteins replacing each other.</p>
</div>
<div id="conclusion" class="section level2">
<h2><span class="header-section-number">2.5</span> CONCLUSION</h2>
<p>We propose that paralog genes generated by tandem duplication start their life coregulated within TADs, then are moved outside to other places in the chromosome and eventually to different chromosomes. TADs would then fit genomic duplications situating the new copy in a duplicated regulatory enviroment. Subsequent genomic rearrangements would create divergent regulatory circuits eventually allowing their disentanglement. An exception would be genes that precise to be strongly co-regulated with the original copy, for example, to produce a replacement protein.</p>
<p>TADs would thus act as protective nests for evolving newcomer genes. This seems to be a reasonable evolutionary mechanism, much simpler than creating from nothing a complete new regulatory environment for a new gene.</p>
</div>
<div id="acknowledgements" class="section level2">
<h2><span class="header-section-number">2.6</span> ACKNOWLEDGEMENTS</h2>
<p>The authors thank all members of the CBDM group for fruitful discussions.</p>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="introduction.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="ch-TAD-evolution.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"google": false,
"weibo": false,
"instapper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/ibn-salem/thesis/edit/master/02-paralog_regulation.Rmd",
"text": "Edit"
},
"download": ["thesis.pdf", "thesis.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "";
    if (src === "" || src === "true") src = "https://cdn.bootcss.com/mathjax/2.7.1/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:" && /^https?:/.test(src))
      src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
