<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>Genome folding in evolution and disease</title>
  <meta name="description" content="This is a draft PhD thesis.">
  <meta name="generator" content="bookdown 0.5.10 and GitBook 2.6.7">

  <meta property="og:title" content="Genome folding in evolution and disease" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="This is a draft PhD thesis." />
  <meta name="github-repo" content="ibn-salem/thesis" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Genome folding in evolution and disease" />
  
  <meta name="twitter:description" content="This is a draft PhD thesis." />
  

<meta name="author" content="Jonas Ibn-Salem">


<meta name="date" content="2018-01-25">

  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  
<link rel="prev" href="ch-paralog-regulaion.html">
<link rel="next" href="ch-position-effect.html">
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />










<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Thesis</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Preface</a></li>
<li class="chapter" data-level="1" data-path="introduction.html"><a href="introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a><ul>
<li class="chapter" data-level="1.1" data-path="introduction.html"><a href="introduction.html#structure-of-the-intorduction"><i class="fa fa-check"></i><b>1.1</b> Structure of the Intorduction</a></li>
<li class="chapter" data-level="1.2" data-path="introduction.html"><a href="introduction.html#cell-diversity-and-regulation-of-gene-expression"><i class="fa fa-check"></i><b>1.2</b> Cell diversity and regulation of gene expression</a></li>
<li class="chapter" data-level="1.3" data-path="introduction.html"><a href="introduction.html#enhancers"><i class="fa fa-check"></i><b>1.3</b> Enhancers</a></li>
<li class="chapter" data-level="1.4" data-path="introduction.html"><a href="introduction.html#methods-to-probe-the-3d-chromatin-archtiecture"><i class="fa fa-check"></i><b>1.4</b> Methods to probe the 3D chromatin archtiecture</a><ul>
<li class="chapter" data-level="1.4.1" data-path="introduction.html"><a href="introduction.html#microscopy-based-techniques-to-visuallize-the-genome-in-3d"><i class="fa fa-check"></i><b>1.4.1</b> Microscopy-based techniques to visuallize the genome in 3D</a></li>
<li class="chapter" data-level="1.4.2" data-path="introduction.html"><a href="introduction.html#proximity-ligation-based-method-to-quantify-chromatin-interactions"><i class="fa fa-check"></i><b>1.4.2</b> Proximity-ligation based method to quantify chromatin interactions</a></li>
</ul></li>
<li class="chapter" data-level="1.5" data-path="introduction.html"><a href="introduction.html#hierarchy-of-chromatin-3d-structure"><i class="fa fa-check"></i><b>1.5</b> Hierarchy of chromatin 3D structure</a><ul>
<li class="chapter" data-level="1.5.1" data-path="introduction.html"><a href="introduction.html#chromosomal-territories-and-inter-chromosomal-contacts"><i class="fa fa-check"></i><b>1.5.1</b> Chromosomal territories and inter-chromosomal contacts</a></li>
<li class="chapter" data-level="1.5.2" data-path="introduction.html"><a href="introduction.html#ab-compartments"><i class="fa fa-check"></i><b>1.5.2</b> A/B compartments</a></li>
<li class="chapter" data-level="1.5.3" data-path="introduction.html"><a href="introduction.html#topologically-associating-domains-tads"><i class="fa fa-check"></i><b>1.5.3</b> Topologically associating domains (TADs)</a></li>
<li class="chapter" data-level="1.5.4" data-path="introduction.html"><a href="introduction.html#chromatin-loops"><i class="fa fa-check"></i><b>1.5.4</b> Chromatin loops</a></li>
</ul></li>
<li class="chapter" data-level="1.6" data-path="introduction.html"><a href="introduction.html#dynamics-of-chromatin-structure"><i class="fa fa-check"></i><b>1.6</b> Dynamics of chromatin structure</a></li>
<li class="chapter" data-level="1.7" data-path="introduction.html"><a href="introduction.html#changes-of-genome-folding-in-evoltuion-and-disease-genomes"><i class="fa fa-check"></i><b>1.7</b> Changes of genome folding in evoltuion and disease genomes</a></li>
<li class="chapter" data-level="1.8" data-path="introduction.html"><a href="introduction.html#aims-of-this-thesis"><i class="fa fa-check"></i><b>1.8</b> Aims of this thesis</a><ul>
<li class="chapter" data-level="" data-path="introduction.html"><a href="introduction.html#is-the-three-dimensonal-folding-of-genomes-associated-with-co-regulation-of-functionally-related-genes"><i class="fa fa-check"></i>Is the three-dimensonal folding of genomes associated with co-regulation of functionally related genes?</a></li>
<li class="chapter" data-level="" data-path="introduction.html"><a href="introduction.html#are-tads-functional-buidling-blocks-of-genomes-and-subjected-to-selective-preasure-during-evolution"><i class="fa fa-check"></i>Are TADs functional buidling blocks of genomes and subjected to selective preasure during evolution?</a></li>
<li class="chapter" data-level="" data-path="introduction.html"><a href="introduction.html#is-the-disruption-of-tads-by-rearrangements-also-associated-to-genetic-diseases"><i class="fa fa-check"></i>Is the disruption of TADs by rearrangements also associated to genetic diseases?</a></li>
<li class="chapter" data-level="" data-path="introduction.html"><a href="introduction.html#can-chromatin-looping-interactions-be-predicted-by-chip-seq-and-sequence-features"><i class="fa fa-check"></i>Can chromatin looping interactions be predicted by ChIP-seq and sequence features?</a></li>
</ul></li>
<li class="chapter" data-level="1.9" data-path="introduction.html"><a href="introduction.html#structure-of-this-thesis"><i class="fa fa-check"></i><b>1.9</b> Structure of this thesis</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html"><i class="fa fa-check"></i><b>2</b> Paralog genes in the 3D genome architecture</a><ul>
<li class="chapter" data-level="2.1" data-path="introduction.html"><a href="introduction.html#introduction"><i class="fa fa-check"></i><b>2.1</b> Introduction</a></li>
<li class="chapter" data-level="2.2" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#sec:paralog-methods"><i class="fa fa-check"></i><b>2.2</b> MATERIALS AND METHODS</a><ul>
<li class="chapter" data-level="2.2.1" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#selection-of-pairs-of-paralog-genes"><i class="fa fa-check"></i><b>2.2.1</b> Selection of pairs of paralog genes</a></li>
<li class="chapter" data-level="2.2.2" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#enhancers-to-gene-association"><i class="fa fa-check"></i><b>2.2.2</b> Enhancers to gene association</a></li>
<li class="chapter" data-level="2.2.3" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#topological-associating-domains"><i class="fa fa-check"></i><b>2.2.3</b> Topological associating domains</a></li>
<li class="chapter" data-level="2.2.4" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#hi-c-interaction-maps"><i class="fa fa-check"></i><b>2.2.4</b> Hi-C interaction maps</a></li>
<li class="chapter" data-level="2.2.5" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#randomization"><i class="fa fa-check"></i><b>2.2.5</b> Randomization</a></li>
<li class="chapter" data-level="2.2.6" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#statistical-tests"><i class="fa fa-check"></i><b>2.2.6</b> Statistical tests</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#results"><i class="fa fa-check"></i><b>2.3</b> RESULTS</a><ul>
<li class="chapter" data-level="2.3.1" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#distribution-of-paralog-genes-in-the-human-genome"><i class="fa fa-check"></i><b>2.3.1</b> Distribution of paralog genes in the human genome</a></li>
<li class="chapter" data-level="2.3.2" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#co-expression-of-paralog-gene-pairs-across-tissues"><i class="fa fa-check"></i><b>2.3.2</b> Co-expression of paralog gene pairs across tissues</a></li>
<li class="chapter" data-level="2.3.3" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#paralog-genes-share-enhancers"><i class="fa fa-check"></i><b>2.3.3</b> Paralog genes share enhancers</a></li>
<li class="chapter" data-level="2.3.4" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#co-localization-of-paralogs-in-tads"><i class="fa fa-check"></i><b>2.3.4</b> Co-localization of paralogs in TADs</a></li>
<li class="chapter" data-level="2.3.5" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#distal-paralog-pairs-are-enriched-for-long-range-chromatin-contacts"><i class="fa fa-check"></i><b>2.3.5</b> Distal paralog pairs are enriched for long-range chromatin contacts</a></li>
<li class="chapter" data-level="2.3.6" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#close-paralogs-have-fewer-contacts-than-expected"><i class="fa fa-check"></i><b>2.3.6</b> Close paralogs have fewer contacts than expected</a></li>
<li class="chapter" data-level="2.3.7" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#paralogs-in-mouse-and-dog-genome"><i class="fa fa-check"></i><b>2.3.7</b> Paralogs in mouse and dog genome</a></li>
<li class="chapter" data-level="2.3.8" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#orthologs-of-human-paralogs-show-conserved-co-localization"><i class="fa fa-check"></i><b>2.3.8</b> Orthologs of human paralogs show conserved co-localization</a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#discussion"><i class="fa fa-check"></i><b>2.4</b> DISCUSSION</a></li>
<li class="chapter" data-level="2.5" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#conclusion"><i class="fa fa-check"></i><b>2.5</b> CONCLUSION</a></li>
<li class="chapter" data-level="2.6" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#acknowledgements"><i class="fa fa-check"></i><b>2.6</b> ACKNOWLEDGEMENTS</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html"><i class="fa fa-check"></i><b>3</b> Stability of TADs in evolution</a><ul>
<li class="chapter" data-level="" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#abstract"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#keywords"><i class="fa fa-check"></i>Keywords</a></li>
<li class="chapter" data-level="3.1" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#introduction-1"><i class="fa fa-check"></i><b>3.1</b> Introduction</a></li>
<li class="chapter" data-level="3.2" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#results-1"><i class="fa fa-check"></i><b>3.2</b> Results</a><ul>
<li class="chapter" data-level="3.2.1" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#identification-of-evolutionary-rearrangement-breakpoints-from-whole-genome-alignments"><i class="fa fa-check"></i><b>3.2.1</b> Identification of evolutionary rearrangement breakpoints from whole-genome alignments</a></li>
<li class="chapter" data-level="3.2.2" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#rearrangement-breakpoints-are-enriched-at-tad-boundaries"><i class="fa fa-check"></i><b>3.2.2</b> Rearrangement breakpoints are enriched at TAD boundaries</a></li>
<li class="chapter" data-level="3.2.3" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#clusters-of-conserved-non-coding-elements-are-depleted-for-rearrangement-breakpoints"><i class="fa fa-check"></i><b>3.2.3</b> Clusters of conserved non-coding elements are depleted for rearrangement breakpoints</a></li>
<li class="chapter" data-level="3.2.4" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#rearranged-tads-are-associated-with-divergent-gene-expression-between-species"><i class="fa fa-check"></i><b>3.2.4</b> Rearranged TADs are associated with divergent gene expression between species</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#discussion-1"><i class="fa fa-check"></i><b>3.3</b> Discussion</a></li>
<li class="chapter" data-level="3.4" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#conclusion-1"><i class="fa fa-check"></i><b>3.4</b> Conclusion</a></li>
<li class="chapter" data-level="3.5" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#methods"><i class="fa fa-check"></i><b>3.5</b> Methods</a><ul>
<li class="chapter" data-level="3.5.1" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#rearrangement-breakpoints-from-whole-genome-alignments"><i class="fa fa-check"></i><b>3.5.1</b> Rearrangement breakpoints from whole-genome alignments</a></li>
<li class="chapter" data-level="3.5.2" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#topologically-associating-domains-and-contact-domains"><i class="fa fa-check"></i><b>3.5.2</b> Topologically associating domains and contact domains</a></li>
<li class="chapter" data-level="3.5.3" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#breakpoint-distributions-at-tads"><i class="fa fa-check"></i><b>3.5.3</b> Breakpoint distributions at TADs</a></li>
<li class="chapter" data-level="3.5.4" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#quantification-of-breakpoint-enrichment"><i class="fa fa-check"></i><b>3.5.4</b> Quantification of breakpoint enrichment</a></li>
<li class="chapter" data-level="3.5.5" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#expression-data-for-mouse-and-human-orthologs"><i class="fa fa-check"></i><b>3.5.5</b> Expression data for mouse and human orthologs</a></li>
<li class="chapter" data-level="3.5.6" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#classification-of-tads-and-genes-according-to-rearrangements-and-grbs"><i class="fa fa-check"></i><b>3.5.6</b> Classification of TADs and genes according to rearrangements and GRBs</a></li>
<li class="chapter" data-level="3.5.7" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#source-code-and-implementation-details"><i class="fa fa-check"></i><b>3.5.7</b> Source code and implementation details</a></li>
</ul></li>
<li class="chapter" data-level="3.6" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#declarations"><i class="fa fa-check"></i><b>3.6</b> Declarations</a><ul>
<li class="chapter" data-level="3.6.1" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#ethics-approval-and-consent-to-participate"><i class="fa fa-check"></i><b>3.6.1</b> Ethics approval and consent to participate</a></li>
<li class="chapter" data-level="3.6.2" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#consent-for-publication"><i class="fa fa-check"></i><b>3.6.2</b> Consent for publication</a></li>
<li class="chapter" data-level="3.6.3" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#availability-of-data-and-material"><i class="fa fa-check"></i><b>3.6.3</b> Availability of data and material</a></li>
<li class="chapter" data-level="3.6.4" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#competing-interests"><i class="fa fa-check"></i><b>3.6.4</b> Competing interests</a></li>
<li class="chapter" data-level="3.6.5" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#funding"><i class="fa fa-check"></i><b>3.6.5</b> Funding</a></li>
<li class="chapter" data-level="3.6.6" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#authors-contributions"><i class="fa fa-check"></i><b>3.6.6</b> Authors‘ contributions</a></li>
<li class="chapter" data-level="3.6.7" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#acknowledgments"><i class="fa fa-check"></i><b>3.6.7</b> Acknowledgments</a></li>
</ul></li>
<li class="chapter" data-level="3.7" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#tables"><i class="fa fa-check"></i><b>3.7</b> Tables</a><ul>
<li class="chapter" data-level="3.7.1" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#table-1"><i class="fa fa-check"></i><b>3.7.1</b> Table 1</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="ch-position-effect.html"><a href="ch-position-effect.html"><i class="fa fa-check"></i><b>4</b> Position effects of rearrangements in disease genomes</a><ul>
<li class="chapter" data-level="4.1" data-path="ch-position-effect.html"><a href="ch-position-effect.html#abstract-1"><i class="fa fa-check"></i><b>4.1</b> Abstract</a></li>
<li class="chapter" data-level="4.2" data-path="ch-position-effect.html"><a href="ch-position-effect.html#introduction-2"><i class="fa fa-check"></i><b>4.2</b> Introduction</a></li>
<li class="chapter" data-level="4.3" data-path="ch-position-effect.html"><a href="ch-position-effect.html#materials-and-methods"><i class="fa fa-check"></i><b>4.3</b> Materials and Methods</a><ul>
<li class="chapter" data-level="4.3.1" data-path="ch-position-effect.html"><a href="ch-position-effect.html#selection-of-subjects-with-apparently-balanced-chromosome-abnormalities"><i class="fa fa-check"></i><b>4.3.1</b> Selection of subjects with apparently balanced chromosome abnormalities</a></li>
<li class="chapter" data-level="4.3.2" data-path="ch-position-effect.html"><a href="ch-position-effect.html#clinical-descriptions-of-dgap-cases"><i class="fa fa-check"></i><b>4.3.2</b> Clinical descriptions of DGAP cases</a></li>
<li class="chapter" data-level="4.3.3" data-path="ch-position-effect.html"><a href="ch-position-effect.html#analysis-of-genes-bordering-the-rearrangement-breakpoints"><i class="fa fa-check"></i><b>4.3.3</b> Analysis of genes bordering the rearrangement breakpoints</a></li>
<li class="chapter" data-level="4.3.4" data-path="ch-position-effect.html"><a href="ch-position-effect.html#assessment-of-disrupted-functional-elements-and-chromatin-interactions"><i class="fa fa-check"></i><b>4.3.4</b> Assessment of disrupted functional elements and chromatin interactions</a></li>
<li class="chapter" data-level="4.3.5" data-path="ch-position-effect.html"><a href="ch-position-effect.html#ontological-analysis-of-genes-neighboring-breakpoints"><i class="fa fa-check"></i><b>4.3.5</b> Ontological analysis of genes neighboring breakpoints</a></li>
<li class="chapter" data-level="4.3.6" data-path="ch-position-effect.html"><a href="ch-position-effect.html#quantitative-real-time-pcr"><i class="fa fa-check"></i><b>4.3.6</b> Quantitative real-time PCR</a></li>
<li class="chapter" data-level="4.3.7" data-path="ch-position-effect.html"><a href="ch-position-effect.html#assessment-of-dgap-breakpoints-overlapping-with-non-coding-structural-variants-in-public-databases"><i class="fa fa-check"></i><b>4.3.7</b> Assessment of DGAP breakpoints overlapping with non-coding structural variants in public databases</a></li>
</ul></li>
<li class="chapter" data-level="4.4" data-path="ch-position-effect.html"><a href="ch-position-effect.html#results-2"><i class="fa fa-check"></i><b>4.4</b> Results</a><ul>
<li class="chapter" data-level="4.4.1" data-path="ch-position-effect.html"><a href="ch-position-effect.html#genomic-characterization-of-non-coding-breakpoints"><i class="fa fa-check"></i><b>4.4.1</b> Genomic characterization of non-coding breakpoints</a></li>
<li class="chapter" data-level="4.4.2" data-path="ch-position-effect.html"><a href="ch-position-effect.html#identification-of-genes-with-potential-position-effects"><i class="fa fa-check"></i><b>4.4.2</b> Identification of genes with potential position effects</a></li>
<li class="chapter" data-level="4.4.3" data-path="ch-position-effect.html"><a href="ch-position-effect.html#identification-of-subjects-with-shared-non-coding-chromosome-alterations-and-phenotypes"><i class="fa fa-check"></i><b>4.4.3</b> Identification of subjects with shared non-coding chromosome alterations and phenotypes</a></li>
</ul></li>
<li class="chapter" data-level="4.5" data-path="ch-position-effect.html"><a href="ch-position-effect.html#discussion-2"><i class="fa fa-check"></i><b>4.5</b> Discussion</a></li>
<li class="chapter" data-level="4.6" data-path="ch-position-effect.html"><a href="ch-position-effect.html#acknowledgements-1"><i class="fa fa-check"></i><b>4.6</b> Acknowledgements</a></li>
<li class="chapter" data-level="4.7" data-path="ch-position-effect.html"><a href="ch-position-effect.html#tables-1"><i class="fa fa-check"></i><b>4.7</b> Tables</a><ul>
<li class="chapter" data-level="4.7.1" data-path="ch-position-effect.html"><a href="ch-position-effect.html#table-1."><i class="fa fa-check"></i><b>4.7.1</b> Table 1.</a></li>
</ul></li>
<li class="chapter" data-level="4.8" data-path="ch-position-effect.html"><a href="ch-position-effect.html#figures"><i class="fa fa-check"></i><b>4.8</b> Figures</a></li>
<li class="chapter" data-level="4.9" data-path="ch-position-effect.html"><a href="ch-position-effect.html#web-resources"><i class="fa fa-check"></i><b>4.9</b> Web Resources</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="ch-loop-prediction.html"><a href="ch-loop-prediction.html"><i class="fa fa-check"></i><b>5</b> Prediction of chromatin looping interactions</a></li>
<li class="chapter" data-level="6" data-path="discussion-3.html"><a href="discussion-3.html"><i class="fa fa-check"></i><b>6</b> Discussion</a><ul>
<li class="chapter" data-level="6.1" data-path="discussion-3.html"><a href="discussion-3.html#further-directions"><i class="fa fa-check"></i><b>6.1</b> Further directions</a></li>
<li class="chapter" data-level="6.2" data-path="discussion-3.html"><a href="discussion-3.html#conclusions"><i class="fa fa-check"></i><b>6.2</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="appendix"><span><b>Appendix</b></span></li>
<li class="chapter" data-level="A" data-path="supporting-information-co-regulation-of-paralog-genes-in-the-three-dimensional-chromatin-architecture.html"><a href="supporting-information-co-regulation-of-paralog-genes-in-the-three-dimensional-chromatin-architecture.html"><i class="fa fa-check"></i><b>A</b> Supporting Information: Co-regulation of paralog genes in the three-dimensional chromatin architecture</a></li>
<li class="chapter" data-level="B" data-path="supplementary-data-evolutionary-stability-of-topologically-associating-domains-is-associated-with-conserved-gene-regulation.html"><a href="supplementary-data-evolutionary-stability-of-topologically-associating-domains-is-associated-with-conserved-gene-regulation.html"><i class="fa fa-check"></i><b>B</b> Supplementary Data: Evolutionary stability of topologically associating domains is associated with conserved gene regulation</a><ul>
<li class="chapter" data-level="B.1" data-path="supplementary-data-evolutionary-stability-of-topologically-associating-domains-is-associated-with-conserved-gene-regulation.html"><a href="supplementary-data-evolutionary-stability-of-topologically-associating-domains-is-associated-with-conserved-gene-regulation.html#TadEvoSupTab"><i class="fa fa-check"></i><b>B.1</b> Supplementary Tables</a></li>
<li class="chapter" data-level="B.2" data-path="supplementary-data-evolutionary-stability-of-topologically-associating-domains-is-associated-with-conserved-gene-regulation.html"><a href="supplementary-data-evolutionary-stability-of-topologically-associating-domains-is-associated-with-conserved-gene-regulation.html#supplementary-figures"><i class="fa fa-check"></i><b>B.2</b> Supplementary Figures</a></li>
</ul></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Build with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Genome folding in evolution and disease</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="ch:TAD_evolution" class="section level1">
<h1><span class="header-section-number">Chapter 3</span> Stability of TADs in evolution</h1>
<p>Evolutionary stability of topologically associating domains is associated with conserved gene regulation</p>
<p>Jan Krefting<sup>1,2</sup>, Miguel A. Andrade-Navarro<sup>1,2</sup> and Jonas Ibn-Salem<sup>1,2,#</sup></p>
<p><sup>1</sup>Faculty of Biology, Johannes Gutenberg University of Mainz, 55128 Mainz, Germany</p>
<p><sup>2</sup>Institute of Molecular Biology, 55128 Mainz, Germany</p>
<p># corresponding author</p>
<p><a href="https://doi.org/10.1101/231431" class="uri">https://doi.org/10.1101/231431</a></p>
<div id="abstract" class="section level3 unnumbered">
<h3>Abstract</h3>
<p><strong>Background:</strong> The human genome is highly organized in the three-dimensional nucleus. Chromosomes fold locally into topologically associating domains (TADs) defined by increased intra-domain chromatin contacts. TADs contribute to gene regulation by restricting chromatin interactions of regulatory sequences, such as enhancers, with their target genes. Disruption of TADs can result in altered gene expression and is associated to genetic diseases and cancers. However, it is not clear to which extent TAD regions are conserved in evolution and whether disruption of TADs by evolutionary rearrangements can alter gene expression.</p>
<p><strong>Results:</strong> Here, we hypothesize that TADs represent essential functional units of genomes, which are selected against rearrangements during evolution. We investigate this using whole-genome alignments to identify evolutionary rearrangement breakpoints of different vertebrate species. Rearrangement breakpoints are strongly enriched at TAD boundaries and depleted within TADs across species. Furthermore, using gene expression data across many tissues in mouse and human, we show that genes within TADs have more conserved expression patterns. Disruption of TADs by evolutionary rearrangements is associated with changes in gene expression profiles, consistent with a functional role of TADs in gene expression regulation.</p>
<p><strong>Conclusions:</strong> Together, these results indicate that TADs are conserved building blocks of genomes with regulatory functions that are often reshuffled as a whole instead of being disrupted by rearrangements.</p>
</div>
<div id="keywords" class="section level3 unnumbered">
<h3>Keywords</h3>
<p>Genome rearrangements; Topologically associating domains; TAD; Chromatin interactions; 3D genome architecture; Hi-C; Evolution; Selection; Gene regulation; Structural variants</p>
</div>
<div id="introduction-1" class="section level2">
<h2><span class="header-section-number">3.1</span> Introduction</h2>
<p>The three-dimensional structure of eukaryotic genomes is organized in many hierarchical levels <span class="citation">(Bonev and Cavalli, <a href="references.html#ref-Bonev2016">2016</a>)</span>. The development of high-throughput experiments to measure pairwise chromatin-chromatin interactions, such as Hi-C <span class="citation">(Lieberman-Aiden et al., <a href="references.html#ref-Lieberman-Aiden2009">2009</a>)</span> enabled the identification of genomic domains of several hundred kilo-bases with increased self-interaction frequencies, described as topologically associating domains (TADs) <span class="citation">(Dixon et al., <a href="references.html#ref-Dixon2012">2012</a>; Nora et al., <a href="references.html#ref-Nora2012">2012</a>)</span>. Loci within TADs contact each other more frequently and TAD boundaries insulate interactions of loci in different TADs. TADs have also been shown to be important for gene regulation by restricting the interaction of cell-type specific enhancers with their target genes <span class="citation">(Nora et al., <a href="references.html#ref-Nora2012">2012</a>; Symmons et al., <a href="references.html#ref-Symmons2014">2014</a>; Zhan et al., <a href="references.html#ref-Zhan2017">2017</a>)</span>. Several studies associated disruption of TADs to ectopic regulation of important developmental genes leading to genetic diseases <span class="citation">(Ibn-Salem et al., <a href="references.html#ref-Ibn-Salem2014">2014</a>; Lupiáñez et al., <a href="references.html#ref-Lupianez2015">2015</a>)</span>. These properties of TADs suggested that they are functional genomic units of gene regulation.</p>
<p>Interestingly, TADs are largely stable across cell-types <span class="citation">(Dixon et al., <a href="references.html#ref-Dixon2012">2012</a>; Rao et al., <a href="references.html#ref-Rao2014">2014</a>)</span> and during differentiation <span class="citation">(Dixon et al., <a href="references.html#ref-Dixon2015">2015</a>)</span>. Moreover, while TADs were initially described for mammalian genomes, a similar domain organization was found in the genomes of non-mammalian species such as <em>Drosophila</em> <span class="citation">(Sexton et al., <a href="references.html#ref-Sexton2012">2012</a>)</span>, zebrafish <span class="citation">(Gómez-Marín et al., <a href="references.html#ref-Gomez-Marin2015">2015</a>)</span> <em>Caenorhabditis elegans</em> <span class="citation">(Crane et al., <a href="references.html#ref-Crane2015">2015</a>)</span> and yeast <span class="citation">(Hsieh et al., <a href="references.html#ref-Hsieh2015">2015</a>; Mizuguchi et al., <a href="references.html#ref-Mizuguchi2014">2014</a>)</span>. Evolutionary conservation of TADs together with their spatio-temporal stability within organisms, would collectively imply that TADs are robust structures.</p>
<p>This motivated the first studies comparing TAD structures across different species, which indeed suggested that individual TAD boundaries are largely conserved along evolution. More than 54% of TAD boundaries in human cells occur at homologous positions in mouse genomes <span class="citation">(Dixon et al., <a href="references.html#ref-Dixon2012">2012</a>)</span>. Similarly, 45% of contact domains called in mouse B-lymphoblasts were also identified at homologous regions in human lymphoblastoid cells <span class="citation">(Rao et al., <a href="references.html#ref-Rao2014">2014</a>)</span>. A single TAD boundary at the Six gene loci could be traced back in evolution to the origin of deuterostomes <span class="citation">(Gómez-Marín et al., <a href="references.html#ref-Gomez-Marin2015">2015</a>)</span>. However, these analyses focused only on the subset of syntenic regions that can be mapped uniquely between genomes and do not investigate systematically if TAD regions as a whole might be stable or disrupted by rearrangements during evolution.</p>
<p>A more recent study provided Hi-C interaction maps of liver cells for four mammalian genomes <span class="citation">(Vietri Rudan et al., <a href="references.html#ref-VietriRudan2015">2015</a>)</span>. Interestingly, they described three examples of rearrangements between mouse and dog, which all occurred at TAD boundaries. However, the rearrangements were identified by ortholog gene adjacencies, which might be biased by gene density. Furthermore, they did not report the total number of rearrangements identified, leaving the question open of how many TADs are actually conserved between organisms. It remains unclear to which extent TADs are selected against disruptions during evolution <span class="citation">(Nora et al., <a href="references.html#ref-Nora2013">2013</a>)</span>. All these studies underline the need to make a systematic study to verify if and how TAD regions as a whole might be stable or disrupted by rearrangements during evolution.</p>
<p>To address this issue we used whole-genome alignment data to analyze systematically whether TADs represent conserved genomic structures that are rather reshuffled as a whole than disrupted by rearrangements during evolution. Furthermore, we used gene expression data from many tissues in human and mouse to associate disruptions of TADs by evolutionary rearrangements to changes in gene expression.</p>
</div>
<div id="results-1" class="section level2">
<h2><span class="header-section-number">3.2</span> Results</h2>
<div id="identification-of-evolutionary-rearrangement-breakpoints-from-whole-genome-alignments" class="section level3">
<h3><span class="header-section-number">3.2.1</span> Identification of evolutionary rearrangement breakpoints from whole-genome alignments</h3>
<p>To analyze the stability of TADs in evolution, we first identified evolutionary rearrangements by using whole-genome alignment data from the UCSC Genome Browser <span class="citation">(Kent et al., <a href="references.html#ref-Kent2002">2002</a>, <a href="references.html#ref-Kent2003">2003</a>)</span> to compare the human genome to 12 other species. These species where selected to have genome assemblies of good quality and to span several hundred million years of evolution. They range from chimpanzee to zebrafish (Fig <a href="ch-TAD-evolution.html#fig:TadEvo1">3.1</a>). The whole-genome data consists of consecutive alignment blocks that are chained and hierarchically ordered into so-called net files as fills <span class="citation">(Kent et al., <a href="references.html#ref-Kent2003">2003</a>)</span>. To overcome alignment artifacts and smaller local variations between genomes we only considered top-level fills or non-syntenic fills and additionally applied a size threshold to use only fills that are larger than 10 kb, 100 kb, or 1000 kb, respectively. Start and end coordinates of such fills represent borders of syntenic regions and were extracted as rearrangement breakpoints for further analysis (see Methods for details).</p>
<!-- ======================================================= -->
<div class="figure" style="text-align: center"><span id="fig:TadEvo1"></span>
<img src="figures/TAD_evolution/fig1_v02.png" alt="Number and size distributions of fill sizes of whole-genome alignments between human and 12 other species. (A) Number of syntenic alignment blocks (fills) between human (hg38) and 12 other species. Top-level fills are the largest and highest scoring chains and occur at the top level in the hierarchy in net files (top panel). Non-syn fills map to different chromosomes as their parent fills in the net files (bottom panel). (B) Size distribution of top-level (top panel) and non-syntenic (bottom panel) fills as violin plot. (C) Number of identified rearrangement breakpoints between human and 12 other species. Breakpoints are borders of top-level or non-syn fills that are larger or equal than a given size threshold (x-axis). (D) Phylogenetic tree with estimated divergence times according to http://timetree.org/. " width="50%" />
<p class="caption">
Figure 3.1: <strong>Number and size distributions of fill sizes of whole-genome alignments between human and 12 other species. </strong> <strong>(A)</strong> Number of syntenic alignment blocks (fills) between human (hg38) and 12 other species. Top-level fills are the largest and highest scoring chains and occur at the top level in the hierarchy in net files (top panel). Non-syn fills map to different chromosomes as their parent fills in the net files (bottom panel). <strong>(B)</strong> Size distribution of top-level (top panel) and non-syntenic (bottom panel) fills as violin plot. <strong>(C)</strong> Number of identified rearrangement breakpoints between human and 12 other species. Breakpoints are borders of top-level or non-syn fills that are larger or equal than a given size threshold (x-axis). <strong>(D)</strong> Phylogenetic tree with estimated divergence times according to <a href="http://timetree.org/"><em>http://timetree.org/</em></a>. <!-- ======================================================= -->
</p>
</div>

<p>First, we analyzed the number and size distributions of top-level and non-syntenic fills between human and other species (Fig <a href="ch-TAD-evolution.html#fig:TadEvo1">3.1</a>). As expected, closely related species such as chimpanzee and gorilla have in general fewer fills but larger fill sizes (mean length ≥1 kb), whereas species which are more distant to human, such as chicken and zebrafish, tend to have more but smaller fills (mean length ≤ 1 kb, Fig <a href="ch-TAD-evolution.html#fig:TadEvo1">3.1</a>A,B). However, we also observe many small non-syntenic fills in closely related species, likely arising from transposon insertions <span class="citation">(Mills et al., <a href="references.html#ref-Mills2006">2006</a>)</span>. As a consequence of the number of fills and size distributions, we identify different breakpoint numbers depending on species and size threshold applied. For example, the whole-genome alignment between human and mouse results in 2182, 655, and 302 rearrangement breakpoints for size thresholds, 10 kb, 100 kb, and 1000 kb, respectively (Fig <a href="ch-TAD-evolution.html#fig:TadEvo1">3.1</a>C). Together, the number and size distributions of syntenic regions reflect the evolutionary divergence time from human and allow us to identify thousands of evolutionary rearrangement breakpoints for enrichment analysis at TADs.</p>
</div>
<div id="rearrangement-breakpoints-are-enriched-at-tad-boundaries" class="section level3">
<h3><span class="header-section-number">3.2.2</span> Rearrangement breakpoints are enriched at TAD boundaries</h3>
<p>Next, we analyzed how the identified rearrangement breakpoints are distributed in the human genome with respect to TADs. We obtained 3,062 TADs identified in human embryonic stem cells (hESC) <span class="citation">(Dixon et al., <a href="references.html#ref-Dixon2012">2012</a>)</span> and 9,274 contact domains from high-resolution <em>in situ</em> Hi-C in human B-lymphoblastoid cells (GM12878) <span class="citation">(Rao et al., <a href="references.html#ref-Rao2014">2014</a>)</span>. To calculate the number of breakpoints around TADs, we enlarged each TAD region by +/-50% of its size and divided the region in 20 equal sized bins. For each bin we computed the number of overlapping rearrangement breakpoints. This results in a size-normalized distribution of rearrangement breakpoints along TAD regions.</p>
<!-- ======================================================= -->
<div class="figure" style="text-align: center"><span id="fig:TadEvo2"></span>
<img src="figures/TAD_evolution/fig2_v02.png" alt="Evolutionary rearrangements are enriched at TAD boundaries. (A) Distribution of evolutionary rearrangement breakpoints between human and mouse around hESC TADs. Each TAD and 50% of its adjacent sequence was subdivided into 20 bins of equal size, the breakpoints were assigned to the bins and their number summed up over the corresponding bins in all TADs. Blue color scale represents breakpoints from different fill-size thresholds. Dotted lines in gray show simulated background controls of randomly placed breakpoints. (B) Distribution of rearrangement breakpoints between human and: chimpanzee, cattle, opossum, and zebrafish, at 10 kb size threshold around hESC TADs. Dotted lines in gray show simulated background controls of randomly placed breakpoints. (C) Enrichment of breakpoints at TAD boundaries as log-odds-ratio between actual breakpoints at TAD boundaries and randomly placed breakpoints. Enrichment is shown for three different fill size thresholds (blue color scale) and TADs in hESC from (Dixon et al., 2012) (top) and contact domains in human GM12878 cells from (Rao et al., 2014) (bottom), respectively. Asterisks indicate significance of the enrichment using Fisher’s exact test (*p &lt;= 0.05; **p &lt;= 0.01; ***p &lt;= 0.001). " width="50%" />
<p class="caption">
Figure 3.2: <strong>Evolutionary rearrangements are enriched at TAD boundaries.</strong> <strong>(A)</strong> Distribution of evolutionary rearrangement breakpoints between human and mouse around hESC TADs. Each TAD and 50% of its adjacent sequence was subdivided into 20 bins of equal size, the breakpoints were assigned to the bins and their number summed up over the corresponding bins in all TADs. Blue color scale represents breakpoints from different fill-size thresholds. Dotted lines in gray show simulated background controls of randomly placed breakpoints. <strong>(B)</strong> Distribution of rearrangement breakpoints between human and: chimpanzee, cattle, opossum, and zebrafish, at 10 kb size threshold around hESC TADs. Dotted lines in gray show simulated background controls of randomly placed breakpoints. <strong>(C)</strong> Enrichment of breakpoints at TAD boundaries as log-odds-ratio between actual breakpoints at TAD boundaries and randomly placed breakpoints. Enrichment is shown for three different fill size thresholds (blue color scale) and TADs in hESC from <span class="citation">(Dixon et al., <a href="references.html#ref-Dixon2012">2012</a>)</span> (top) and contact domains in human GM12878 cells from <span class="citation">(Rao et al., <a href="references.html#ref-Rao2014">2014</a>)</span> (bottom), respectively. Asterisks indicate significance of the enrichment using Fisher’s exact test (*p &lt;= 0.05; **p &lt;= 0.01; ***p &lt;= 0.001). <!-- ======================================================= -->
</p>
</div>

<p>First, we analyzed the distribution of breakpoints at different size thresholds between human and mouse at hESC TADs (Fig. 2A). Rearrangement breakpoints are clearly enriched at TAD boundaries and depleted within TAD regions. Notably, this enrichment is observed for all size thresholds applied in the identification of rearrangement breakpoints. Next, we also analyzed the breakpoints from chimpanzee, cattle, opossum, and zebrafish (Fig <a href="ch-TAD-evolution.html#fig:TadEvo2">3.2</a>B) at the 10 kb size threshold. Interestingly, we observed for all species a clear enrichment of breakpoints at TAD boundaries and depletion within TAD regions. To quantify this enrichment, we simulated an expected background distribution of breakpoints by placing each breakpoint 100 times at a random position of the respective chromosome. We than calculated the fraction of observed and expected breakpoints that are closer than 40 kb to a TAD boundary. For all size thresholds and analyzed species, we computed the log-fold-ratio of actual breakpoints over random breakpoints at domain boundaries (Fig <a href="ch-TAD-evolution.html#fig:TadEvo2">3.2</a>C). For virtually all species and size thresholds analyzed, we found breakpoints significantly enriched at boundaries of TADs and contact domains (Fig <a href="ch-TAD-evolution.html#fig:TadEvo2">3.2</a>C, <a href="supplementary-data-evolutionary-stability-of-topologically-associating-domains-is-associated-with-conserved-gene-regulation.html#fig:TadEvoS1">B.1</a>). Depletion was only observed for some combinations of species and size thresholds which have only very few breakpoints (see Fig <a href="ch-TAD-evolution.html#fig:TadEvo1">3.1</a>C). Furthermore, we compared the distance of each breakpoint to the closest TAD boundary and observed nearly always significantly shorter distances for actual breakpoints compared to random controls (Fig <a href="supplementary-data-evolutionary-stability-of-topologically-associating-domains-is-associated-with-conserved-gene-regulation.html#fig:TadEvoS2">B.2</a>). Overall, the enrichment was stronger for TADs in hESC compared to the contact domains in GM12878. However, these differences were likely due to different sizes of TADs and contact domains and the nested structure of contact domains, which overlap each other <span class="citation">(Rao et al., <a href="references.html#ref-Rao2014">2014</a>)</span>. Rearrangements between human and both closely and distantly related species are highly enriched at TAD boundaries and depleted within TADs. These results show (i) that rearrangements are not randomly distributed in the genome, in agreement with <span class="citation">(Farré et al., <a href="references.html#ref-Farre2015">2015</a>)</span>, and (ii) strong conservation of TAD regions over large evolutionary time scales, indicating selective pressure against disruption of TADs, presumably because of their functional role in gene expression regulation.</p>
</div>
<div id="clusters-of-conserved-non-coding-elements-are-depleted-for-rearrangement-breakpoints" class="section level3">
<h3><span class="header-section-number">3.2.3</span> Clusters of conserved non-coding elements are depleted for rearrangement breakpoints</h3>
<p>Another interesting feature that can be extracted from whole-genome alignments are highly conserved non-coding elements (CNEs) <span class="citation">(Polychronopoulos et al., <a href="references.html#ref-Polychronopoulos2017">2017</a>)</span>. CNEs are defined as non-protein-coding sequences of at least 50 bp with over 70% sequence identity between distantly related species such as human and chicken <span class="citation">(Polychronopoulos et al., <a href="references.html#ref-Polychronopoulos2017">2017</a>)</span>. In the human genome, CNEs cluster around developmental genes in so-called genomic regulatory blocks (GRBs) <span class="citation">(Kikuta et al., <a href="references.html#ref-Kikuta2007">2007</a>)</span>. It has been shown recently that many GRBs coincide with TADs in human and <em>Drosophila</em> genomes <span class="citation">(Harmston et al., <a href="references.html#ref-Harmston2017">2017</a>)</span>. Therefore, we asked whether evolutionary breakpoints are also enriched at boundaries of GRBs. This would support the idea of a conserved regulatory environment around important developmental genes. Indeed we saw a strong enrichment around GRBs (Fig <a href="ch-TAD-evolution.html#fig:TadEvo3">3.3</a>A). This is consistent with previous studies in <em>Drosophila</em> and Fish where CNE arrays often correspond to syntenic blocks <span class="citation">(Dimitrieva and Bucher, <a href="references.html#ref-Dimitrieva2013">2013</a>; Engström et al., <a href="references.html#ref-Engstrom2007">2007</a>)</span>.</p>
<!-- ======================================================= -->
<div class="figure" style="text-align: center"><span id="fig:TadEvo3"></span>
<img src="figures/TAD_evolution/fig3_v02.png" alt="Rearrangement breakpoint distribution around GRBs and GRB-TADs. (A) Rearrangement breakpoints between mouse and human around 816 GRBs. (B) Breakpoint distribution around GRB-TADs and non-GRB-TADs. GRB-TADs are defined as TADs overlapping more than 80% with GRBs and non-GRB-TADs have less than 20% overlap with GRBs. Breakpoints using a 10 kb fill size threshold are shown. " width="50%" />
<p class="caption">
Figure 3.3: <strong>Rearrangement breakpoint distribution around GRBs and GRB-TADs. </strong> <strong>(A)</strong> Rearrangement breakpoints between mouse and human around 816 GRBs. <strong>(B)</strong> Breakpoint distribution around GRB-TADs and non-GRB-TADs. GRB-TADs are defined as TADs overlapping more than 80% with GRBs and non-GRB-TADs have less than 20% overlap with GRBs. Breakpoints using a 10 kb fill size threshold are shown. <!-- ======================================================= -->
</p>
</div>

<p>Next, we subdivided TADs according to their overlap with GRBs in GRB-TADs (&gt; 80% overlap) and non-GRB-TADs (&lt; 20% overlap) as in the original study <span class="citation">(Harmston et al., <a href="references.html#ref-Harmston2017">2017</a>)</span>. As expected, we observed a higher accumulation of breakpoints at boundaries and stronger depletion within TADs for GRB-TADs compared to non-GRB-TADs (Fig <a href="ch-TAD-evolution.html#fig:TadEvo3">3.3</a>B). However, also the non-GRB-TADs, that have less than 20% overlap with GRBs, are enriched for rearrangements at TAD boundaries. This indicates that not only TADs overlapping GRBs are evolutionary conserved. In summary, we show that human TADs overlapping clusters of non-coding conserved elements are strongly depleted for rearrangements, likely due to strong selective pressure on the conserved regulatory environment around important developmental genes.</p>
</div>
<div id="rearranged-tads-are-associated-with-divergent-gene-expression-between-species" class="section level3">
<h3><span class="header-section-number">3.2.4</span> Rearranged TADs are associated with divergent gene expression between species</h3>
<p>The enrichment of rearrangement breakpoints at TAD boundaries indicates that TADs are stable across large evolutionary time scales. However, the reason for this strong conservation of TAD regions is unclear. A mechanistic explanation could be that certain chromatin features at TAD boundaries promote or prevent DNA double strand breaks (DSBs) <span class="citation">(Canela et al., <a href="references.html#ref-Canela2017">2017</a>; Farré et al., <a href="references.html#ref-Farre2015">2015</a>)</span>. Alternatively, selective pressure might act against the disruption of TADs due to their functional importance, for example in developmental gene regulation <span class="citation">(Farré et al., <a href="references.html#ref-Farre2015">2015</a>; Nora et al., <a href="references.html#ref-Nora2013">2013</a>)</span>. TADs constitute a structural framework determining possible interactions between promoters and cis-regulatory sequences while prohibiting the influence of other sequences <span class="citation">(Lupiáñez et al., <a href="references.html#ref-Lupianez2015">2015</a>; Symmons et al., <a href="references.html#ref-Symmons2014">2014</a>)</span>. TAD disruption would prevent formerly established contacts. Rearrangements of TADs might also enable the recruitment of new cis-regulatory sequences which would alter the expression patterns of genes in rearranged TADs <span class="citation">(Lupiáñez et al., <a href="references.html#ref-Lupianez2015">2015</a>; Redin et al., <a href="references.html#ref-Redin2016">2016</a>)</span>. Because of these detrimental effects, rearranged TADs should largely be eliminated by purifying selection. However, rearrangement of TADs could also enable the expression of genes in a new context and be selected if conferring an advantage. Therefore, we hypothesized that genes within conserved TADs might have a more stable gene expression pattern across tissues, whereas genes in rearranged TADs between two species might have a more divergent expression between species.</p>
<p>To test this, we analyzed the conservation of gene expression of ortholog genes between human and mouse across 19 matched tissues from the FANTOM5 project (<a href="supplementary-data-evolutionary-stability-of-topologically-associating-domains-is-associated-with-conserved-gene-regulation.html#TadEvoSupTab">Table S1</a>) <span class="citation">(Forrest et al., <a href="references.html#ref-Forrest2014">2014</a>)</span>. If a human gene and its mouse ortholog have high correlation across matching tissues, they are likely to have the same regulation and eventually similar functions. Conversely, low correlation of expression across tissues can indicate functional divergence during evolution, potentially due to altered gene regulation.</p>
<p>First, we separated human genes according to their location within TADs or outside of TADs. From 12,696 human genes with expression data and a unique one-to-one ortholog in mouse (<a href="supplementary-data-evolutionary-stability-of-topologically-associating-domains-is-associated-with-conserved-gene-regulation.html#TadEvoSupTab">Table S2</a>), 1,525 have a transcription start site (TSS) located outside hESC TADs and 11,171 within. Next, we computed for each gene its expression correlation with mouse orthologs across 19 matching tissues. Genes within TADs have significantly higher expression correlation with their mouse ortholog (median R = 0,340) compared to genes outside TADs (mean R = 0,308, p = 0.0015, Fig <a href="ch-TAD-evolution.html#fig:TadEvo4">3.4</a>A). This indicates higher conservation of gene regulation in TADs and is consistent with the observation of housekeeping genes at TAD boundaries <span class="citation">(Dixon et al., <a href="references.html#ref-Dixon2012">2012</a>)</span> and the role of TADs in providing conserved regulatory environments for gene regulation <span class="citation">(Harmston et al., <a href="references.html#ref-Harmston2017">2017</a>; Ibn-Salem et al., <a href="references.html#ref-Ibn-Salem2017">2017</a>)</span>.</p>
<!-- ======================================================= -->
<div class="figure" style="text-align: center"><span id="fig:TadEvo4"></span>
<img src="figures/TAD_evolution/fig4_v04.png" alt="Ortholog gene expression correlation across tissues in conserved and rearranged TADs. (A) Expression correlation of orthologs across 19 matching tissues in human and mouse for human genes within or outside of hESC TADs. (B) Expression correlation of orthologs across 19 matching tissues in human and mouse for genes in conserved or rearranged TADs. (C) Expression correlation of orthologs across 19 matching tissues in human and mouse for genes in GRB-TADs and non-GRB TADs. All P-values according to Wilcoxon rank-sum test. " width="80%" />
<p class="caption">
Figure 3.4: <strong>Ortholog gene expression correlation across tissues in conserved and rearranged TADs.</strong> <strong>(A)</strong> Expression correlation of orthologs across 19 matching tissues in human and mouse for human genes within or outside of hESC TADs. <strong>(B)</strong> Expression correlation of orthologs across 19 matching tissues in human and mouse for genes in conserved or rearranged TADs. <strong>(C)</strong> Expression correlation of orthologs across 19 matching tissues in human and mouse for genes in GRB-TADs and non-GRB TADs. All P-values according to Wilcoxon rank-sum test. <!-- ======================================================= -->
</p>
</div>

<p>Next, we further subdivided TADs in two groups, rearranged and conserved, according to syntenic blocks and rearrangements between human and mouse genomes. In brief, a TAD is defined as conserved, if it is completely enclosed by a syntenic alignment block and does not overlap any rearrangement breakpoint. Conversely, a rearranged TAD is not enclosed by a syntenic alignment block and overlaps at least one breakpoint that is farther than 80 kb from its boundary (see Methods). For the hESC TAD data set, this leads to 2,542 conserved and 137 rearranged TADs. The low number of rearranged TADs is consistent with the depletion of rearrangement breakpoints within TADs in general (Fig. 2). In total 8,740 genes in conserved and 645 genes in rearranged TADs could be assigned to a one-to-one ortholog in mouse and are contained in the expression data set. The expression correlation with mouse orthologs were significantly higher for genes in conserved TADs (median R = 0.316) compared to genes in rearranged TADs (median R = 0.237, p = 0.0013) (Fig <a href="ch-TAD-evolution.html#fig:TadEvo4">3.4</a>B). This shows that disruptions of TADs by evolutionary rearrangements are associated with less conserved gene expression profiles across tissues. Although not significant, we also observed a slightly higher expression correlation for 1,003 genes in GRB-TADs compared to 8,038 genes in non-GRB TADs (Fig <a href="ch-TAD-evolution.html#fig:TadEvo4">3.4</a>C, p = 0.13).</p>
<p>In summary, we observed higher expression correlation between orthologs<br />
for human genes inside TADs than outside. Moreover, we saw that genes in rearranged TADs show lower gene expression conservation than those in conserved TADs. These results not only support a functional role of TADs in gene regulation, but further support the hypothesis that TAD regions are subjected to purifying selection against their disruption by structural variations such as rearrangements.</p>
</div>
</div>
<div id="discussion-1" class="section level2">
<h2><span class="header-section-number">3.3</span> Discussion</h2>
<p>Our analysis of rearrangements between human and 12 diverse species shows that TADs are largely stable units of genomes, which are often reshuffled as a whole instead of disrupted by rearrangements. Furthermore, the decreased expression correlation with orthologs in mouse and human in rearranged TADs shows that disruptions of TADs are associated with changes in gene regulation over large evolutionary time scales.</p>
<p>TADs exert their influence on gene expression regulation by determining the set of possible interactions of cis-regulatory sequences with their target promoters <span class="citation">(Nora et al., <a href="references.html#ref-Nora2012">2012</a>; Schoenfelder et al., <a href="references.html#ref-Schoenfelder2015">2015</a>; Symmons et al., <a href="references.html#ref-Symmons2014">2014</a>)</span>. This might facilitate the cooperation of several sequences that is often needed for the complex spatiotemporal regulation of transcription <span class="citation">(Andrey and Mundlos, <a href="references.html#ref-Andrey2017">2017</a>)</span>. The disruption of these enclosed regulatory environments enables the recruitment of other cis-regulatory sequences and might prevent formerly established interactions <span class="citation">(Montavon et al., <a href="references.html#ref-Montavon2012">2012</a>)</span>. The detrimental effects of such events have been shown in the study of diseases <span class="citation">(Redin et al., <a href="references.html#ref-Redin2016">2016</a>; Zepeda-Mendoza et al., <a href="references.html#ref-Zepeda-Mendoza2017">2017</a>)</span>. There are also incidences where pathogenic phenotypes could be specifically attributed to enhancers establishing contacts to promoters that were formerly out of reach because of intervening TAD boundaries <span class="citation">(Ibn-Salem et al., <a href="references.html#ref-Ibn-Salem2014">2014</a>; Lupiáñez et al., <a href="references.html#ref-Lupianez2015">2015</a>; Spielmann et al., <a href="references.html#ref-Spielmann2012">2012</a>)</span>. This would explain the selective pressure to maintain TAD integrity over large evolutionary distances and why we observe higher gene expression conservation for human genes within TADs compared to genes outside TADs.</p>
<p>Disruptions of TADs by large-scale rearrangements change expression patterns of orthologs across tissues and these changes might be explained by the altered regulatory environment which genes are exposed to after rearrangement <span class="citation">(Farré et al., <a href="references.html#ref-Farre2015">2015</a>)</span>.</p>
<p>Our results are largely consistent with the reported finding that many TADs correspond to clusters of conserved non-coding elements (GRBs) <span class="citation">(Harmston et al., <a href="references.html#ref-Harmston2017">2017</a>)</span>. We observe a strong depletion of evolutionary rearrangements in GRBs and enrichment at GRB boundaries. This is consistent with comparative genome analysis revealing that GRBs largely overlap with micro-syntenic blocks in <em>Drosophila</em> <span class="citation">(Engström et al., <a href="references.html#ref-Engstrom2007">2007</a>)</span> and fish genomes <span class="citation">(Dimitrieva and Bucher, <a href="references.html#ref-Dimitrieva2013">2013</a>)</span>. However, over 60% of human hESC TADs do not overlap GRBs <span class="citation">(Harmston et al., <a href="references.html#ref-Harmston2017">2017</a>)</span>, raising the question of whether only a small subset of TADs are conserved. Interestingly, we find also depletion of rearrangements in non-GRB-TADs. This indicates that our rearrangement analysis identifies conservation also for TADs that are not enriched for CNEs. High expression correlation of orthologs in conserved TADs suggestss that the maintenance of expression regulation is important for most genes and probably even more crucial for developmental genes which are frequently found in GRBs.</p>
<p>Previous work using comparative Hi-C analysis in four mammals revealed that insulation of TAD boundaries is robustly conserved at syntenic regions, illustrating this with a few examples of rearrangements between mouse and dog genomes, which were located in both species at TAD boundaries <span class="citation">(Vietri Rudan et al., <a href="references.html#ref-VietriRudan2015">2015</a>)</span>. The results of our analysis of thousands of rearrangements between human and 12 other species confirmed and expanded these earlier observations.</p>
<p>The reliable identification of evolutionary genomic rearrangements is difficult. Especially for non-coding genomic features like TAD boundaries, it is important to use approaches that are unbiased towards coding sequence. Previous studies identified rearrangements by interrupted adjacency of ortholog genes between two organisms <span class="citation">(Pevzner and Tesler, <a href="references.html#ref-Pevzner2003">2003</a>; Vietri Rudan et al., <a href="references.html#ref-VietriRudan2015">2015</a>)</span>. However, such an approach assumes equal inter-genic distances, which is violated at TAD boundaries, which have in general higher gene density <span class="citation">(Dixon et al., <a href="references.html#ref-Dixon2012">2012</a>; Hou et al., <a href="references.html#ref-Hou2012">2012</a>)</span>. To avoid this bias we used whole-genome-alignments. However, low quality of the genome assembly of some species might introduce alignment problems and potentially false positive rearrangement breakpoints.</p>
<p>Rearrangements are created by DNA double strand breaks (DSBs), which are not uniquely distributed in the genome. Certain genomic features, such as open chromatin, active transcription and certain histone marks are shown to be enriched at DSBs in somatic translocation sites <span class="citation">(Roukos and Misteli, <a href="references.html#ref-Roukos2014">2014</a>)</span> and evolutionary rearrangements <span class="citation">(Hinsch and Hannenhalli, <a href="references.html#ref-Hinsch2006">2006</a>; Murphy et al., <a href="references.html#ref-Murphy2005">2005</a>)</span>. Furthermore, induced DSBs and somatic translocation breakpoints are enriched at chromatin loop anchors <span class="citation">(Canela et al., <a href="references.html#ref-Canela2017">2017</a>)</span>. This opens the question of whether our finding of significantly enriched evolutionary rearrangement breakpoints at TAD boundaries could be explained by the molecular properties of the chromatin at TAD boundaries, rather than by the selective pressure to keep TAD function. Although, we cannot distinguish the two explanations entirely, our gene expression analysis indicates stronger conservation of gene expression in conserved TADs and more divergent expression patterns in rearranged TADs. This supports a model in which disruption of TADs are most often disadvantageous for an organism. Structural variations disrupting TADs can lead to miss regulation of neighboring genes as shown for genetic diseases <span class="citation">(Franke et al., <a href="references.html#ref-Franke2016">2016</a>; Ibn-Salem et al., <a href="references.html#ref-Ibn-Salem2014">2014</a>; Lupiáñez et al., <a href="references.html#ref-Lupianez2015">2015</a>; Redin et al., <a href="references.html#ref-Redin2016">2016</a>)</span> and cancers <span class="citation">(Hnisz et al., <a href="references.html#ref-Hnisz2016">2016</a>; Northcott et al., <a href="references.html#ref-Northcott2014">2014</a>; Weischenfeldt et al., <a href="references.html#ref-Weischenfeldt2016">2016</a>)</span>.</p>
<p>Interestingly, we observed higher gene expression conservation for human genes within TADs compared to genes outside TADs. The larger syntenic structure of TADs might conserve the regulation likely by maintaining the proximity of promoters and cis-regulatory sequences while genes outside such frameworks are more exposed to changing genomic landscapes, presumably resulting in a greater susceptibility to the recruitment of regulatory sequences.</p>
<p>Apart from the described detrimental effects, our results suggest that TAD rearrangements occurred between genomes of human and mouse and led to changes in expression patterns of many orthologous genes. Since this is likely attributed to changing regulatory environments, it is also conceivable that some rearrangements led to a gain of function. Hence, TAD rearrangements might also provide a vehicle for evolutionary innovation. A single TAD reorganization has the potential to affect the regulation of a whole set of genes in contrast to the more confined consequences of other types of mutations <span class="citation">(Acemel et al., <a href="references.html#ref-Acemel2017">2017</a>)</span>. Since it is also believed that changes in cis-regulatory sequences of developmental genes play a big part in evolutionary innovation <span class="citation">(Carroll, <a href="references.html#ref-Carroll2008">2008</a>)</span>, the development of the enormous diversity of animal traits in evolution might have been promoted by the rearrangement of structural domains. This is consistent with a model in which new genes can arise by tandem-duplication and during evolution are then re-located to other environments <span class="citation">(Ibn-Salem et al., <a href="references.html#ref-Ibn-Salem2017">2017</a>)</span>. These changes might have facilitated significant leaps in morphological evolution explaining the emergence of features that could not appear in small gradual steps. Following this hypothesis, TADs would not only constitute structural entities that perform the function of maintaining an enclosed regulatory landscape but could also be a driving force for change by exposing many genes at once to different genomic environments following single events of genomic rearrangement.</p>
</div>
<div id="conclusion-1" class="section level2">
<h2><span class="header-section-number">3.4</span> Conclusion</h2>
<p>Our results indicate that TADs represent conserved functional building blocks of the genome. We have shown that the majority of evolutionary rearrangements do not affect the integrity of TADs and instead breakpoints are strongly clustered at TAD boundaries. This leads to the conclusion that TADs constitute conserved building blocks of the genome that are often reshuffled as a whole rather than disrupted during evolution. The conservation of TAD regions can be explained by detrimental effects of disrupting cis-regulatory environments that are essential for the spatio-temporal control of gene expression. Indeed we observe a significant association of conserved gene expression in intact TADs and divergent expression patterns in rearranged TADs explaining both why there could be selective pressure on the integrity of TADs over large evolutionary time scales, but also how TAD rearrangement can explain evolutionary leaps.</p>
</div>
<div id="methods" class="section level2">
<h2><span class="header-section-number">3.5</span> Methods</h2>
<div id="rearrangement-breakpoints-from-whole-genome-alignments" class="section level3">
<h3><span class="header-section-number">3.5.1</span> Rearrangement breakpoints from whole-genome alignments</h3>
<p>Rearrangement breakpoints were identified between human and 12 selected vertebrate species from whole-genome-alignment data (Table 1). Alignment data were downloaded as net files from UCSC Genome Browser for human genome hg38 and the genomes listed in Table 1. The whole-genome data consists of consecutive alignment blocks that are chained and hierarchically ordered in the so-called nets <span class="citation">(Kent et al., <a href="references.html#ref-Kent2003">2003</a>)</span>. Chains represent blocks of interrupted syntenic regions and may include larger gaps. When hierarchically arranged in a net file, child chains can complement their parents when they align nearby segments that fill the alignment gaps of their parents but may also break the synteny when incorporating distal segments. We implemented a computer program to extract rearrangement breakpoints from net files based on the length and type of fills. Start and end points of top-level or non-syntenic fills are reported as rearrangement breakpoint if the fill exceeds a given size threshold. We used different size thresholds to optimize both the number of identified breakpoints and to avoid biases of transposable elements that might be responsible for many small interruptions of alignment chains. In this way, we extracted rearrangement breakpoints between human and 12 genomes using size thresholds of 10 kb, 100 kb, and 1000 kb. To compare breakpoints to TADs we converted the breakpoint coordinates from hg38 to hg19 genome assembly using the liftOver tool from UCSC Genome Browser <span class="citation">(Hinrichs et al., <a href="references.html#ref-Hinrichs2006">2006</a>)</span>.</p>
</div>
<div id="topologically-associating-domains-and-contact-domains" class="section level3">
<h3><span class="header-section-number">3.5.2</span> Topologically associating domains and contact domains</h3>
<p>We obtained topologically associating domain (TAD) calls from published Hi-C experiments in human embryonic stem cells (hESC) <span class="citation">(Dixon et al., <a href="references.html#ref-Dixon2012">2012</a>)</span> and contact domains from published <em>in situ</em> Hi-C experiments in human GM12878 cells <span class="citation">(Rao et al., <a href="references.html#ref-Rao2014">2014</a>)</span>. Genomic coordinates of hESC TADs were converted from hg18 to hg19 genome assembly using the UCSC liftOver tool <span class="citation">(Hinrichs et al., <a href="references.html#ref-Hinrichs2006">2006</a>)</span>.</p>
</div>
<div id="breakpoint-distributions-at-tads" class="section level3">
<h3><span class="header-section-number">3.5.3</span> Breakpoint distributions at TADs</h3>
<p>To quantify the number of breakpoints around TADs and TAD boundaries we enlarged TAD regions by 50% of their total length on each side. The range was then subdivided into 20 equal sized bins and the number of overlapping breakpoints computed. This results in a matrix in which rows represent individual TADs and columns represent bins along TAD regions. The sum of each column indicates the number of breakpoints for corresponding bins and therefore the same relative location around TADs. For comparable visualization between different data sets, the column-wise summed breakpoint counts were further normalized as percent values of the total breakpoint number in the matrix.</p>
</div>
<div id="quantification-of-breakpoint-enrichment" class="section level3">
<h3><span class="header-section-number">3.5.4</span> Quantification of breakpoint enrichment</h3>
<p>To quantify the enrichment of breakpoints at domain boundaries, we generated random breakpoints as background control. For each chromosome, we placed the same number of actual breakpoints at a random position of the chromosome. For each breakpoint data set we simulated 100 times the same number of random breakpoints. We then computed the distribution of random breakpoints around TADs in the same way as described above for actual breakpoints. To compute enrichment of actual breakpoints compared to simulated controls, we classified each breakpoint located in a window of 400 kb around TAD borders in either close to a TAD boundary, if distance between breakpoint and TAD boundary was smaller or equal to 40 kb or as distant, when distance was larger than 40 kb. This results in a contingency table of actual and random breakpoints that are either close or distal to TAD boundaries. We computed log odds ratios as effect size of enrichment and p-values according to Fishers two-sided exact test. Additionally, we compared the distance of all actual and random breakpoints to their nearest TAD boundary using the Wilcoxon’s rank-sum test.</p>
</div>
<div id="expression-data-for-mouse-and-human-orthologs" class="section level3">
<h3><span class="header-section-number">3.5.5</span> Expression data for mouse and human orthologs</h3>
<p>Promoter based expression data from CAGE analysis in human and mouse tissues from the FANTOM5 project <span class="citation">(Forrest et al., <a href="references.html#ref-Forrest2014">2014</a>)</span> were retrieved from the EBI Expression Atlas <span class="citation">(Hinrichs et al., <a href="references.html#ref-Hinrichs2006">2006</a>)</span> as baseline expression values per gene and tissue. The meta data of samples contains tissue annotations as term IDs from Uberon, an integrated cross-species ontology covering anatomical structures in animals <span class="citation">(Herrero et al., <a href="references.html#ref-Herrero2016">2016</a>)</span>. Human and mouse samples were assigned to each other if they had the same developmental stage and matching Uberon term IDs. This resulted in 19 samples for each organism with corresponding tissues.</p>
<p>We used the R package biomaRt to retrieve all human genes in the Ensembl database (version grch37.ensembl.org) and could assign 13,065 to ortholog genes in mouse by allowing only the one-to-one orthology type <span class="citation">(Herrero et al., <a href="references.html#ref-Herrero2016">2016</a>)</span>. Of these ortholog pairs, 12,696 are contained in the expression data described above. For each pair of orthologs we computed the correlation of expression values across matching tissues as Pearson’s correlation coefficient.</p>
</div>
<div id="classification-of-tads-and-genes-according-to-rearrangements-and-grbs" class="section level3">
<h3><span class="header-section-number">3.5.6</span> Classification of TADs and genes according to rearrangements and GRBs</h3>
<p>We classified hESC TADs according to rearrangements between human and mouse genomes. We define a TAD as conserved if it is completely enclosed within a fill in the net file and no rearrangement breakpoint from any size threshold is located in the TAD region with a distance larger than 80 kb from the TAD boundary. A TAD is defined as rearranged, if the TAD is not enclosed completely by any fill in the net file, overlaps at least one breakpoint inferred using a 1000 kb fill size threshold, and this breakpoint is further than 80 kb away from each TAD boundary. TADs were also classified according to their overlap with GRBs as in <span class="citation">(Harmston et al., <a href="references.html#ref-Harmston2017">2017</a>)</span>. A given TAD is a GRB-TAD if it overlaps with more than 80% of the TAD size with a GRB. A TAD is classified as non-GRB if it has less than 20% overlap with GRBs. The 12,696 human genes with mouse ortholog and expression data were grouped according to their location with respect to hESC TADs. We used the transcription start site (TSS) of the longest transcript per gene to group each gene as within TAD if the TSS overlaps a hESC TAD or as outside TADs, if not. Furthermore, we grouped genes in TADs according to conserved or rearranged TADs and separately according to GRB and non-GRB TADs.</p>
</div>
<div id="source-code-and-implementation-details" class="section level3">
<h3><span class="header-section-number">3.5.7</span> Source code and implementation details</h3>
<p>The source code of the entire analysis described here is available on GitHub: <a href="https://github.com/Juppen/TAD-Evolution"><em>https://github.com/Juppen/TAD-Evolution</em></a>. The identification of breakpoints and extraction of fills from whole-genome alignment data was implemented in Python scripts. Reading of BED files and overlap calculations with TADs and TAD bins were computed in R with Bioconductor <span class="citation">(Huber et al., <a href="references.html#ref-Huber2015">2015</a>)</span> packages rtracklayer <span class="citation">(Lawrence et al., <a href="references.html#ref-Lawrence2009">2009</a>)</span> and GenomicRanges <span class="citation">(Lawrence et al., <a href="references.html#ref-Lawrence2013">2013</a>)</span>. Gene coordinates and ortholog assignments were retrieved from Ensemble data base (version grch37.ensembl.org) using the package biomaRt <span class="citation">(Durinck et al., <a href="references.html#ref-Durinck2009">2009</a><a href="references.html#ref-Durinck2009">b</a>)</span>. For data integration and visualization we used R packages from tidyverse <span class="citation">(Wickham and Grolemund)</span>.</p>
</div>
</div>
<div id="declarations" class="section level2">
<h2><span class="header-section-number">3.6</span> Declarations</h2>
<div id="ethics-approval-and-consent-to-participate" class="section level3">
<h3><span class="header-section-number">3.6.1</span> Ethics approval and consent to participate</h3>
<p>Not applicable</p>
</div>
<div id="consent-for-publication" class="section level3">
<h3><span class="header-section-number">3.6.2</span> Consent for publication</h3>
<p>Not applicable</p>
</div>
<div id="availability-of-data-and-material" class="section level3">
<h3><span class="header-section-number">3.6.3</span> Availability of data and material</h3>
<p>The source code of all analysis is available on GitHub: <a href="https://github.com/Juppen/TAD-Evolution"><em>https://github.com/Juppen/TAD-Evolution</em></a>. All the genomic data used for analyses are freely available to be downloaded from the UCSC Genome Browser and EBI Expression Atlas with identifiers listed in Table 1 and <a href="supplementary-data-evolutionary-stability-of-topologically-associating-domains-is-associated-with-conserved-gene-regulation.html#TadEvoSupTab">Table S1</a>.</p>
</div>
<div id="competing-interests" class="section level3">
<h3><span class="header-section-number">3.6.4</span> Competing interests</h3>
<p>The authors declare that they have no competing interests.</p>
</div>
<div id="funding" class="section level3">
<h3><span class="header-section-number">3.6.5</span> Funding</h3>
<p>Not applicable.</p>
</div>
<div id="authors-contributions" class="section level3">
<h3><span class="header-section-number">3.6.6</span> Authors‘ contributions</h3>
<p>JK and JI developed and implemented the methods and performed the analysis. JI conceived the study. JK wrote the first draft of the manuscript. JK, MA and JI wrote the manuscript. MA supervised the study.</p>
</div>
<div id="acknowledgments" class="section level3">
<h3><span class="header-section-number">3.6.7</span> Acknowledgments</h3>
<p>The authors thank all members of the CBDM group for fruitful discussions.</p>
</div>
</div>
<div id="tables" class="section level2">
<h2><span class="header-section-number">3.7</span> Tables</h2>
<div id="table-1" class="section level3">
<h3><span class="header-section-number">3.7.1</span> Table 1</h3>
<pre><code>    Species used for breakpoint identification from whole-genome
    alignments with human.</code></pre>
<table>
<tbody>
<tr class="odd">
<td align="left">Common name</td>
<td align="left">Species</td>
<td align="left">Genome Assembly</td>
<td align="left">Divergence to human (mya)</td>
</tr>
<tr class="even">
<td align="left">Chimpanzee</td>
<td align="left"><em>Pan troglodytes</em></td>
<td align="left">panTro5</td>
<td align="left">6.65</td>
</tr>
<tr class="odd">
<td align="left">Gorilla</td>
<td align="left"><em>Gorilla gorilla gorilla</em></td>
<td align="left">gorGor5</td>
<td align="left">9.06</td>
</tr>
<tr class="even">
<td align="left">Orangutan</td>
<td align="left"><em>Pongo abelii</em></td>
<td align="left">ponAbe2</td>
<td align="left">15.76</td>
</tr>
<tr class="odd">
<td align="left">Rhesus</td>
<td align="left"><em>Macaca mulatta</em></td>
<td align="left">rheMac8</td>
<td align="left">29.44</td>
</tr>
<tr class="even">
<td align="left">Mouse lemur</td>
<td align="left"><em>Microcebus murinus</em></td>
<td align="left">micMur2</td>
<td align="left">74</td>
</tr>
<tr class="odd">
<td align="left">Mouse</td>
<td align="left"><em>Mus musculus</em></td>
<td align="left">mm10</td>
<td align="left">90</td>
</tr>
<tr class="even">
<td align="left">Cattle</td>
<td align="left"><em>Bos taurus</em></td>
<td align="left">bosTau8</td>
<td align="left">96</td>
</tr>
<tr class="odd">
<td align="left">Manatee</td>
<td align="left"><em>Trichechus manatus latirostris</em></td>
<td align="left">triMan1</td>
<td align="left">105</td>
</tr>
<tr class="even">
<td align="left">Opossum</td>
<td align="left"><em>Monodelphis domestica</em></td>
<td align="left">monDom5</td>
<td align="left">159</td>
</tr>
<tr class="odd">
<td align="left">Chicken</td>
<td align="left"><em>Gallus gallus</em></td>
<td align="left">galGal5</td>
<td align="left">312</td>
</tr>
<tr class="even">
<td align="left">Clawed frog</td>
<td align="left"><em>Xenopus tropicalis</em></td>
<td align="left">xenTro7</td>
<td align="left">352</td>
</tr>
<tr class="odd">
<td align="left">Zebrafish</td>
<td align="left"><em>Danio rerio</em></td>
<td align="left">danRer10</td>
<td align="left">435</td>
</tr>
</tbody>
</table>

<!-- List of manual changes: -->
<!-- - Title without line break -->
<!-- - (TODO:) Table to proper postion in text -->
<!-- - (TODO:) Add Figures with captions -->
<!-- - (TODO:) sectionand sub-seqction header formating -->
<!-- # Computational Prediction of Position Effects of Apparently Balanced Human Chromosome Rearrangements -->
</div>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="ch-paralog-regulaion.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="ch-position-effect.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"google": false,
"weibo": false,
"instapper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/ibn-salem/thesis/edit/master/03-TAD_evolution.Rmd",
"text": "Edit"
},
"download": ["thesis.pdf", "thesis.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "";
    if (src === "" || src === "true") src = "https://cdn.bootcss.com/mathjax/2.7.1/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:" && /^https?:/.test(src))
      src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
