<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>Genome folding and transcriptional regulation in evolution and disease</title>
  <meta name="description" content="This is a draft PhD thesis.">
  <meta name="generator" content="bookdown 0.5.10 and GitBook 2.6.7">

  <meta property="og:title" content="Genome folding and transcriptional regulation in evolution and disease" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="This is a draft PhD thesis." />
  <meta name="github-repo" content="ibn-salem/thesis" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Genome folding and transcriptional regulation in evolution and disease" />
  
  <meta name="twitter:description" content="This is a draft PhD thesis." />
  

<meta name="author" content="Jonas Ibn-Salem">


<meta name="date" content="2018-01-24">

  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  
<link rel="prev" href="ch-loop-prediction.html">
<link rel="next" href="references.html">
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />










<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Thesis</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Preface</a></li>
<li class="chapter" data-level="1" data-path="introduction.html"><a href="introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a><ul>
<li class="chapter" data-level="1.1" data-path="introduction.html"><a href="introduction.html#structure-of-the-intorduction"><i class="fa fa-check"></i><b>1.1</b> Structure of the Intorduction</a></li>
<li class="chapter" data-level="1.2" data-path="introduction.html"><a href="introduction.html#cell-diversity-and-regulation-of-gene-expression"><i class="fa fa-check"></i><b>1.2</b> Cell diversity and regulation of gene expression</a></li>
<li class="chapter" data-level="1.3" data-path="introduction.html"><a href="introduction.html#enhancers"><i class="fa fa-check"></i><b>1.3</b> Enhancers</a></li>
<li class="chapter" data-level="1.4" data-path="introduction.html"><a href="introduction.html#methods-to-probe-the-3d-chromatin-archtiecture"><i class="fa fa-check"></i><b>1.4</b> Methods to probe the 3D chromatin archtiecture</a><ul>
<li class="chapter" data-level="1.4.1" data-path="introduction.html"><a href="introduction.html#microscopy-based-techniques-to-visuallize-the-genome-in-3d"><i class="fa fa-check"></i><b>1.4.1</b> Microscopy-based techniques to visuallize the genome in 3D</a></li>
<li class="chapter" data-level="1.4.2" data-path="introduction.html"><a href="introduction.html#proximity-ligation-based-method-to-quantify-chromatin-interactions"><i class="fa fa-check"></i><b>1.4.2</b> Proximity-ligation based method to quantify chromatin interactions</a></li>
</ul></li>
<li class="chapter" data-level="1.5" data-path="introduction.html"><a href="introduction.html#hierarchy-of-chromatin-3d-structure"><i class="fa fa-check"></i><b>1.5</b> Hierarchy of chromatin 3D structure</a><ul>
<li class="chapter" data-level="1.5.1" data-path="introduction.html"><a href="introduction.html#chromosomal-territories-and-inter-chromosomal-contacts"><i class="fa fa-check"></i><b>1.5.1</b> Chromosomal territories and inter-chromosomal contacts</a></li>
<li class="chapter" data-level="1.5.2" data-path="introduction.html"><a href="introduction.html#ab-compartments"><i class="fa fa-check"></i><b>1.5.2</b> A/B compartments</a></li>
<li class="chapter" data-level="1.5.3" data-path="introduction.html"><a href="introduction.html#topologically-associating-domains-tads"><i class="fa fa-check"></i><b>1.5.3</b> Topologically associating domains (TADs)</a></li>
<li class="chapter" data-level="1.5.4" data-path="introduction.html"><a href="introduction.html#chromatin-loops"><i class="fa fa-check"></i><b>1.5.4</b> Chromatin loops</a></li>
</ul></li>
<li class="chapter" data-level="1.6" data-path="introduction.html"><a href="introduction.html#changes-of-genome-folding-in-evoltuion-and-disease-genomes"><i class="fa fa-check"></i><b>1.6</b> Changes of genome folding in evoltuion and disease genomes</a></li>
<li class="chapter" data-level="1.7" data-path="introduction.html"><a href="introduction.html#aims-of-this-thesis"><i class="fa fa-check"></i><b>1.7</b> Aims of this thesis</a><ul>
<li class="chapter" data-level="" data-path="introduction.html"><a href="introduction.html#is-the-three-dimensonal-folding-of-genomes-associated-with-co-regulation-of-functionally-related-genes"><i class="fa fa-check"></i>Is the three-dimensonal folding of genomes associated with co-regulation of functionally related genes?</a></li>
<li class="chapter" data-level="" data-path="introduction.html"><a href="introduction.html#are-tads-functional-buidling-blocks-of-genomes-and-subjected-to-selective-preasure-during-evolution"><i class="fa fa-check"></i>Are TADs functional buidling blocks of genomes and subjected to selective preasure during evolution?</a></li>
<li class="chapter" data-level="" data-path="introduction.html"><a href="introduction.html#is-the-disruption-of-tads-by-rearrangements-also-associated-to-genetic-diseases"><i class="fa fa-check"></i>Is the disruption of TADs by rearrangements also associated to genetic diseases?</a></li>
<li class="chapter" data-level="" data-path="introduction.html"><a href="introduction.html#can-chromatin-looping-interactions-be-predicted-by-chip-seq-and-sequence-features"><i class="fa fa-check"></i>Can chromatin looping interactions be predicted by ChIP-seq and sequence features?</a></li>
</ul></li>
<li class="chapter" data-level="1.8" data-path="introduction.html"><a href="introduction.html#structure-of-this-thesis"><i class="fa fa-check"></i><b>1.8</b> Structure of this thesis</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html"><i class="fa fa-check"></i><b>2</b> Paralog genes in the 3D genome architecture</a><ul>
<li class="chapter" data-level="2.1" data-path="introduction.html"><a href="introduction.html#introduction"><i class="fa fa-check"></i><b>2.1</b> Introduction</a></li>
<li class="chapter" data-level="2.2" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#sec:paralog-methods"><i class="fa fa-check"></i><b>2.2</b> MATERIALS AND METHODS</a><ul>
<li class="chapter" data-level="2.2.1" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#selection-of-pairs-of-paralog-genes"><i class="fa fa-check"></i><b>2.2.1</b> Selection of pairs of paralog genes</a></li>
<li class="chapter" data-level="2.2.2" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#enhancers-to-gene-association"><i class="fa fa-check"></i><b>2.2.2</b> Enhancers to gene association</a></li>
<li class="chapter" data-level="2.2.3" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#topological-associating-domains"><i class="fa fa-check"></i><b>2.2.3</b> Topological associating domains</a></li>
<li class="chapter" data-level="2.2.4" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#hi-c-interaction-maps"><i class="fa fa-check"></i><b>2.2.4</b> Hi-C interaction maps</a></li>
<li class="chapter" data-level="2.2.5" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#randomization"><i class="fa fa-check"></i><b>2.2.5</b> Randomization</a></li>
<li class="chapter" data-level="2.2.6" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#statistical-tests"><i class="fa fa-check"></i><b>2.2.6</b> Statistical tests</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#results"><i class="fa fa-check"></i><b>2.3</b> RESULTS</a><ul>
<li class="chapter" data-level="2.3.1" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#distribution-of-paralog-genes-in-the-human-genome"><i class="fa fa-check"></i><b>2.3.1</b> Distribution of paralog genes in the human genome</a></li>
<li class="chapter" data-level="2.3.2" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#co-expression-of-paralog-gene-pairs-across-tissues"><i class="fa fa-check"></i><b>2.3.2</b> Co-expression of paralog gene pairs across tissues</a></li>
<li class="chapter" data-level="2.3.3" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#paralog-genes-share-enhancers"><i class="fa fa-check"></i><b>2.3.3</b> Paralog genes share enhancers</a></li>
<li class="chapter" data-level="2.3.4" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#co-localization-of-paralogs-in-tads"><i class="fa fa-check"></i><b>2.3.4</b> Co-localization of paralogs in TADs</a></li>
<li class="chapter" data-level="2.3.5" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#distal-paralog-pairs-are-enriched-for-long-range-chromatin-contacts"><i class="fa fa-check"></i><b>2.3.5</b> Distal paralog pairs are enriched for long-range chromatin contacts</a></li>
<li class="chapter" data-level="2.3.6" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#close-paralogs-have-fewer-contacts-than-expected"><i class="fa fa-check"></i><b>2.3.6</b> Close paralogs have fewer contacts than expected</a></li>
<li class="chapter" data-level="2.3.7" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#paralogs-in-mouse-and-dog-genome"><i class="fa fa-check"></i><b>2.3.7</b> Paralogs in mouse and dog genome</a></li>
<li class="chapter" data-level="2.3.8" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#orthologs-of-human-paralogs-show-conserved-co-localization"><i class="fa fa-check"></i><b>2.3.8</b> Orthologs of human paralogs show conserved co-localization</a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#discussion"><i class="fa fa-check"></i><b>2.4</b> DISCUSSION</a></li>
<li class="chapter" data-level="2.5" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#conclusion"><i class="fa fa-check"></i><b>2.5</b> CONCLUSION</a></li>
<li class="chapter" data-level="2.6" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#acknowledgements"><i class="fa fa-check"></i><b>2.6</b> ACKNOWLEDGEMENTS</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html"><i class="fa fa-check"></i><b>3</b> Stability of TADs in evolution</a><ul>
<li class="chapter" data-level="" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#abstract"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#keywords"><i class="fa fa-check"></i>Keywords</a></li>
<li class="chapter" data-level="3.1" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#introduction-1"><i class="fa fa-check"></i><b>3.1</b> Introduction</a></li>
<li class="chapter" data-level="3.2" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#results-1"><i class="fa fa-check"></i><b>3.2</b> Results</a><ul>
<li class="chapter" data-level="3.2.1" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#identification-of-evolutionary-rearrangement-breakpoints-from-whole-genome-alignments"><i class="fa fa-check"></i><b>3.2.1</b> Identification of evolutionary rearrangement breakpoints from whole-genome alignments</a></li>
<li class="chapter" data-level="3.2.2" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#rearrangement-breakpoints-are-enriched-at-tad-boundaries"><i class="fa fa-check"></i><b>3.2.2</b> Rearrangement breakpoints are enriched at TAD boundaries</a></li>
<li class="chapter" data-level="3.2.3" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#clusters-of-conserved-non-coding-elements-are-depleted-for-rearrangement-breakpoints"><i class="fa fa-check"></i><b>3.2.3</b> Clusters of conserved non-coding elements are depleted for rearrangement breakpoints</a></li>
<li class="chapter" data-level="3.2.4" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#rearranged-tads-are-associated-with-divergent-gene-expression-between-species"><i class="fa fa-check"></i><b>3.2.4</b> Rearranged TADs are associated with divergent gene expression between species</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#discussion-1"><i class="fa fa-check"></i><b>3.3</b> Discussion</a></li>
<li class="chapter" data-level="3.4" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#conclusion-1"><i class="fa fa-check"></i><b>3.4</b> Conclusion</a></li>
<li class="chapter" data-level="3.5" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#methods"><i class="fa fa-check"></i><b>3.5</b> Methods</a><ul>
<li class="chapter" data-level="3.5.1" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#rearrangement-breakpoints-from-whole-genome-alignments"><i class="fa fa-check"></i><b>3.5.1</b> Rearrangement breakpoints from whole-genome alignments</a></li>
<li class="chapter" data-level="3.5.2" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#topologically-associating-domains-and-contact-domains"><i class="fa fa-check"></i><b>3.5.2</b> Topologically associating domains and contact domains</a></li>
<li class="chapter" data-level="3.5.3" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#breakpoint-distributions-at-tads"><i class="fa fa-check"></i><b>3.5.3</b> Breakpoint distributions at TADs</a></li>
<li class="chapter" data-level="3.5.4" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#quantification-of-breakpoint-enrichment"><i class="fa fa-check"></i><b>3.5.4</b> Quantification of breakpoint enrichment</a></li>
<li class="chapter" data-level="3.5.5" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#expression-data-for-mouse-and-human-orthologs"><i class="fa fa-check"></i><b>3.5.5</b> Expression data for mouse and human orthologs</a></li>
<li class="chapter" data-level="3.5.6" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#classification-of-tads-and-genes-according-to-rearrangements-and-grbs"><i class="fa fa-check"></i><b>3.5.6</b> Classification of TADs and genes according to rearrangements and GRBs</a></li>
<li class="chapter" data-level="3.5.7" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#source-code-and-implementation-details"><i class="fa fa-check"></i><b>3.5.7</b> Source code and implementation details</a></li>
</ul></li>
<li class="chapter" data-level="3.6" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#declarations"><i class="fa fa-check"></i><b>3.6</b> Declarations</a><ul>
<li class="chapter" data-level="3.6.1" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#ethics-approval-and-consent-to-participate"><i class="fa fa-check"></i><b>3.6.1</b> Ethics approval and consent to participate</a></li>
<li class="chapter" data-level="3.6.2" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#consent-for-publication"><i class="fa fa-check"></i><b>3.6.2</b> Consent for publication</a></li>
<li class="chapter" data-level="3.6.3" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#availability-of-data-and-material"><i class="fa fa-check"></i><b>3.6.3</b> Availability of data and material</a></li>
<li class="chapter" data-level="3.6.4" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#competing-interests"><i class="fa fa-check"></i><b>3.6.4</b> Competing interests</a></li>
<li class="chapter" data-level="3.6.5" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#funding"><i class="fa fa-check"></i><b>3.6.5</b> Funding</a></li>
<li class="chapter" data-level="3.6.6" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#authors-contributions"><i class="fa fa-check"></i><b>3.6.6</b> Authors‘ contributions</a></li>
<li class="chapter" data-level="3.6.7" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#acknowledgments"><i class="fa fa-check"></i><b>3.6.7</b> Acknowledgments</a></li>
</ul></li>
<li class="chapter" data-level="3.7" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#tables"><i class="fa fa-check"></i><b>3.7</b> Tables</a><ul>
<li class="chapter" data-level="3.7.1" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#table-1"><i class="fa fa-check"></i><b>3.7.1</b> Table 1</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="ch-position-effect.html"><a href="ch-position-effect.html"><i class="fa fa-check"></i><b>4</b> Position effects of rearrangements in disease genomes</a><ul>
<li class="chapter" data-level="4.1" data-path="ch-position-effect.html"><a href="ch-position-effect.html#abstract-1"><i class="fa fa-check"></i><b>4.1</b> Abstract</a></li>
<li class="chapter" data-level="4.2" data-path="ch-position-effect.html"><a href="ch-position-effect.html#introduction-2"><i class="fa fa-check"></i><b>4.2</b> Introduction</a></li>
<li class="chapter" data-level="4.3" data-path="ch-position-effect.html"><a href="ch-position-effect.html#materials-and-methods"><i class="fa fa-check"></i><b>4.3</b> Materials and Methods</a><ul>
<li class="chapter" data-level="4.3.1" data-path="ch-position-effect.html"><a href="ch-position-effect.html#selection-of-subjects-with-apparently-balanced-chromosome-abnormalities"><i class="fa fa-check"></i><b>4.3.1</b> Selection of subjects with apparently balanced chromosome abnormalities</a></li>
<li class="chapter" data-level="4.3.2" data-path="ch-position-effect.html"><a href="ch-position-effect.html#clinical-descriptions-of-dgap-cases"><i class="fa fa-check"></i><b>4.3.2</b> Clinical descriptions of DGAP cases</a></li>
<li class="chapter" data-level="4.3.3" data-path="ch-position-effect.html"><a href="ch-position-effect.html#analysis-of-genes-bordering-the-rearrangement-breakpoints"><i class="fa fa-check"></i><b>4.3.3</b> Analysis of genes bordering the rearrangement breakpoints</a></li>
<li class="chapter" data-level="4.3.4" data-path="ch-position-effect.html"><a href="ch-position-effect.html#assessment-of-disrupted-functional-elements-and-chromatin-interactions"><i class="fa fa-check"></i><b>4.3.4</b> Assessment of disrupted functional elements and chromatin interactions</a></li>
<li class="chapter" data-level="4.3.5" data-path="ch-position-effect.html"><a href="ch-position-effect.html#ontological-analysis-of-genes-neighboring-breakpoints"><i class="fa fa-check"></i><b>4.3.5</b> Ontological analysis of genes neighboring breakpoints</a></li>
<li class="chapter" data-level="4.3.6" data-path="ch-position-effect.html"><a href="ch-position-effect.html#quantitative-real-time-pcr"><i class="fa fa-check"></i><b>4.3.6</b> Quantitative real-time PCR</a></li>
<li class="chapter" data-level="4.3.7" data-path="ch-position-effect.html"><a href="ch-position-effect.html#assessment-of-dgap-breakpoints-overlapping-with-non-coding-structural-variants-in-public-databases"><i class="fa fa-check"></i><b>4.3.7</b> Assessment of DGAP breakpoints overlapping with non-coding structural variants in public databases</a></li>
</ul></li>
<li class="chapter" data-level="4.4" data-path="ch-position-effect.html"><a href="ch-position-effect.html#results-2"><i class="fa fa-check"></i><b>4.4</b> Results</a><ul>
<li class="chapter" data-level="4.4.1" data-path="ch-position-effect.html"><a href="ch-position-effect.html#genomic-characterization-of-non-coding-breakpoints"><i class="fa fa-check"></i><b>4.4.1</b> Genomic characterization of non-coding breakpoints</a></li>
<li class="chapter" data-level="4.4.2" data-path="ch-position-effect.html"><a href="ch-position-effect.html#identification-of-genes-with-potential-position-effects"><i class="fa fa-check"></i><b>4.4.2</b> Identification of genes with potential position effects</a></li>
<li class="chapter" data-level="4.4.3" data-path="ch-position-effect.html"><a href="ch-position-effect.html#identification-of-subjects-with-shared-non-coding-chromosome-alterations-and-phenotypes"><i class="fa fa-check"></i><b>4.4.3</b> Identification of subjects with shared non-coding chromosome alterations and phenotypes</a></li>
</ul></li>
<li class="chapter" data-level="4.5" data-path="ch-position-effect.html"><a href="ch-position-effect.html#discussion-2"><i class="fa fa-check"></i><b>4.5</b> Discussion</a></li>
<li class="chapter" data-level="4.6" data-path="ch-position-effect.html"><a href="ch-position-effect.html#acknowledgements-1"><i class="fa fa-check"></i><b>4.6</b> Acknowledgements</a></li>
<li class="chapter" data-level="4.7" data-path="ch-position-effect.html"><a href="ch-position-effect.html#tables-1"><i class="fa fa-check"></i><b>4.7</b> Tables</a><ul>
<li class="chapter" data-level="4.7.1" data-path="ch-position-effect.html"><a href="ch-position-effect.html#table-1."><i class="fa fa-check"></i><b>4.7.1</b> Table 1.</a></li>
</ul></li>
<li class="chapter" data-level="4.8" data-path="ch-position-effect.html"><a href="ch-position-effect.html#figures"><i class="fa fa-check"></i><b>4.8</b> Figures</a></li>
<li class="chapter" data-level="4.9" data-path="ch-position-effect.html"><a href="ch-position-effect.html#web-resources"><i class="fa fa-check"></i><b>4.9</b> Web Resources</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="ch-loop-prediction.html"><a href="ch-loop-prediction.html"><i class="fa fa-check"></i><b>5</b> Prediction of chromatin looping interactions</a></li>
<li class="chapter" data-level="6" data-path="discussion-3.html"><a href="discussion-3.html"><i class="fa fa-check"></i><b>6</b> Discussion</a><ul>
<li class="chapter" data-level="6.1" data-path="discussion-3.html"><a href="discussion-3.html#further-directions"><i class="fa fa-check"></i><b>6.1</b> Further directions</a></li>
<li class="chapter" data-level="6.2" data-path="discussion-3.html"><a href="discussion-3.html#conclusions"><i class="fa fa-check"></i><b>6.2</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="appendix"><span><b>Appendix</b></span></li>
<li class="chapter" data-level="A" data-path="supporting-information-co-regulation-of-paralog-genes-in-the-three-dimensional-chromatin-architecture.html"><a href="supporting-information-co-regulation-of-paralog-genes-in-the-three-dimensional-chromatin-architecture.html"><i class="fa fa-check"></i><b>A</b> Supporting Information: Co-regulation of paralog genes in the three-dimensional chromatin architecture</a></li>
<li class="chapter" data-level="B" data-path="supplementary-data-evolutionary-stability-of-topologically-associating-domains-is-associated-with-conserved-gene-regulation.html"><a href="supplementary-data-evolutionary-stability-of-topologically-associating-domains-is-associated-with-conserved-gene-regulation.html"><i class="fa fa-check"></i><b>B</b> Supplementary Data: Evolutionary stability of topologically associating domains is associated with conserved gene regulation</a><ul>
<li class="chapter" data-level="B.1" data-path="supplementary-data-evolutionary-stability-of-topologically-associating-domains-is-associated-with-conserved-gene-regulation.html"><a href="supplementary-data-evolutionary-stability-of-topologically-associating-domains-is-associated-with-conserved-gene-regulation.html#TadEvoSupTab"><i class="fa fa-check"></i><b>B.1</b> Supplementary Tables</a></li>
<li class="chapter" data-level="B.2" data-path="supplementary-data-evolutionary-stability-of-topologically-associating-domains-is-associated-with-conserved-gene-regulation.html"><a href="supplementary-data-evolutionary-stability-of-topologically-associating-domains-is-associated-with-conserved-gene-regulation.html#supplementary-figures"><i class="fa fa-check"></i><b>B.2</b> Supplementary Figures</a></li>
</ul></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Build with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Genome folding and transcriptional regulation in evolution and disease</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="discussion-3" class="section level1">
<h1><span class="header-section-number">Chapter 6</span> Discussion</h1>
<ul>
<li>Limitations of 3C based methods</li>
<li>Plasticity and dynamics of chromatin interactions</li>
<li>Single cell resolution</li>
<li>Functional mechanism</li>
<li>Establishment of compartments / TADs / loops</li>
<li><p>Regulation of compartments / TADs / loops</p></li>
<li>Notes
<ul>
<li>Neo-TADs by tandem-duplication in evolution <span class="citation">(Franke et al., <a href="references.html#ref-Franke2016">2016</a>)</span></li>
<li>A key component and driver for new gene function in evolution or neo-functionalization can be the birth of new enhancers through acquisition of transcription factor binding and subsequent novel regulatory functions <span class="citation">(Long et al., <a href="references.html#ref-Long2016">2016</a>)</span>.</li>
<li>This data can be computationally integrated with one-dimensional measurements along the genome and lead to exciting findings of higher order organisation.</li>
<li>TADs are not only structural units of chromosomes, but also functional building blocks of genomes.</li>
</ul></li>
</ul>
<!-- Summary of whole wokr -->
<p>In this thesis, I analysed the functions of TADs for gene regulation and highlight their stability in evolution as well as their consideration when analyzing genomic variations in patient genomes. - Paralog genes as model for co-regulation in TADs - enhancer sharing - co-expression</p>
<!-- Connectio to TAD evolution -->
<p>As consequence of these association of TADs with co-regulation, enhancer sharing and co-expression, we hypothesized TADs provide regulatory environments for genes and therefore be conserved during evolution. More specifically, we asked whether genomic rearrangement between related species would more frequently occur at TAD boundaries. Furthermore, we hypothesized that disruption of TADs during evolution might be associated with changes of gene expression programs between the species.</p>
<!-- Connection from TAD-evolution to position effect -->
<p>The analysis of genomic rearrangements between human and other species during evolution lead to the conclusion that TADs are important regulatory building blocks of genomes. Indeed the changes of expression profiles are associated with the disruption of TADs during evolution. This might likely lead to severe disadvantages to the organism, as was observed for example in genetic diseases <span class="citation">(Ibn-Salem et al., <a href="references.html#ref-Ibn-Salem2014">2014</a>; Lupiáñez et al., <a href="references.html#ref-Lupianez2015">2015</a>)</span> and cancers. Therefore, we interpret the depletion of evolutionary rearrangements in TADs and the expression change associated with TAD disruption to be a consequence of selective pressure on TAD structure. Therefore selective pressure is likely to act on TAD structures. While in neutral selection</p>
<div id="further-directions" class="section level2">
<h2><span class="header-section-number">6.1</span> Further directions</h2>
<ul>
<li>Plasticity and dynamics of chromatin interactions</li>
<li>Single cell resolution
<ul>
<li>Single cell Hi-C studies: <span class="citation">(Nagano et al., <a href="references.html#ref-Nagano2017">2017</a>; Stevens et al., <a href="references.html#ref-Stevens2017">2017</a>)</span></li>
<li>Computational modelling of single cells <span class="citation">(Sekelja et al., <a href="references.html#ref-Sekelja2016">2016</a>)</span>.</li>
</ul></li>
<li>Functional mechanism</li>
<li>Establishment of compartments / TADs / loops</li>
<li>Regulation of compartment / TADs / loops</li>
</ul>
</div>
<div id="conclusions" class="section level2">
<h2><span class="header-section-number">6.2</span> Conclusions</h2>
<p>Recent methodological advances in chromatin conformation capture experiments resulted in genome-wide contact maps of genomes. These data lead to many interesting insights in the folding structures of genomes. One important discovery was that chromosomes fold locally into discreet genomic domains, called TADs.</p>
<p>In this thesis, I showed that TADs are not only structural units of genomes, but that they are also functionally important for the correct regulation of gene expression. TADs represent regulatory environment that restrict the interaction landscape of enhancers. Indeed, functionally related genes, such as paralogs, are co-regulated within TADs. During evolution, new genes can emerge by duplication and find established regulatory environments within TADs. Therefore, TADs represent productive nests for novel genes in evolution. The functional important of TADs was further stressed by their stability during hundred million years of evolution. Indeed stable TADs are associated with conserved expression profiles of genes.</p>
<p>Disruption of TADs by rearrangements is associated to changes of gene expression profiles during evolution as well as in genomes of subjects with developmental diseases. While these disruptions of TADs might be beneficial for an organism and lead to evolutionary leaps in some cases, I showed in disease genomes, that disruption of TADs can result in severe phenotypes like mental retardation. Therefore, the three-dimensonal folding structure of genomes, including TADs and enhancer-promoter interactions have to be considered for the interpretation of genomic variants of patient genomes.</p>
<p>While constantly decreasing costs of sequencing will further enable the analysis of individual genomes in many genetic syndromes or cancers, it will be increasingly important to correctly interpret these variants within their functional genomic context. To this end, we need a deeper understanding of the functional role of genome folding including its dynamics between single cells as well as its changes in specific cell types and conditions. To integrate diverse types of functional data that is measured along the genomes with the chromatin folding patterns and their interplay, we need carefully designed computational models. This will address not only fundamental questions such as evolution of genomes, mechanisms of gene regulation in differentiation and development, but also solve practical problems such as the interpretation of genetic variants in disease genomes for better developments of diagnosis and treatments.</p>

<!-- # References -->
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="ch-loop-prediction.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="references.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"google": false,
"weibo": false,
"instapper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/rstudio/bookdown-demo/edit/master/06-discussion.Rmd",
"text": "Edit"
},
"download": ["thesis.pdf", "thesis.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "";
    if (src === "" || src === "true") src = "https://cdn.bootcss.com/mathjax/2.7.1/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:" && /^https?:/.test(src))
      src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
