<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>Genome folding in evolution and disease</title>
  <meta name="description" content="This is a draft PhD thesis.">
  <meta name="generator" content="bookdown 0.6.1 and GitBook 2.6.7">

  <meta property="og:title" content="Genome folding in evolution and disease" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="This is a draft PhD thesis." />
  <meta name="github-repo" content="ibn-salem/thesis" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Genome folding in evolution and disease" />
  
  <meta name="twitter:description" content="This is a draft PhD thesis." />
  

<meta name="author" content="Jonas Ibn-Salem">


<meta name="date" content="2018-02-13">

  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  
<link rel="prev" href="D-2-supplementary-figures-1.html">

<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />










<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Thesis</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="1" data-path="1-thesis-intro.html"><a href="1-thesis-intro.html"><i class="fa fa-check"></i><b>1</b> Introduction</a><ul>
<li class="chapter" data-level="1.1" data-path="1-1-regulation-of-gene-expression.html"><a href="1-1-regulation-of-gene-expression.html"><i class="fa fa-check"></i><b>1.1</b> Regulation of gene expression</a></li>
<li class="chapter" data-level="1.2" data-path="1-2-enhancers.html"><a href="1-2-enhancers.html"><i class="fa fa-check"></i><b>1.2</b> Enhancers</a></li>
<li class="chapter" data-level="1.3" data-path="1-3-methods-to-probe-the-3d-chromatin-archtiecture.html"><a href="1-3-methods-to-probe-the-3d-chromatin-archtiecture.html"><i class="fa fa-check"></i><b>1.3</b> Methods to probe the 3D chromatin archtiecture</a><ul>
<li class="chapter" data-level="1.3.1" data-path="1-3-methods-to-probe-the-3d-chromatin-archtiecture.html"><a href="1-3-methods-to-probe-the-3d-chromatin-archtiecture.html#microscopy-based-techniques-to-visuallize-the-genome-in-3d"><i class="fa fa-check"></i><b>1.3.1</b> Microscopy-based techniques to visuallize the genome in 3D</a></li>
<li class="chapter" data-level="1.3.2" data-path="1-3-methods-to-probe-the-3d-chromatin-archtiecture.html"><a href="1-3-methods-to-probe-the-3d-chromatin-archtiecture.html#proximity-ligation-based-method-to-quantify-chromatin-interactions"><i class="fa fa-check"></i><b>1.3.2</b> Proximity-ligation based method to quantify chromatin interactions</a></li>
</ul></li>
<li class="chapter" data-level="1.4" data-path="1-4-hierarchy-of-chromatin-3d-structure.html"><a href="1-4-hierarchy-of-chromatin-3d-structure.html"><i class="fa fa-check"></i><b>1.4</b> Hierarchy of chromatin 3D structure</a><ul>
<li class="chapter" data-level="1.4.1" data-path="1-4-hierarchy-of-chromatin-3d-structure.html"><a href="1-4-hierarchy-of-chromatin-3d-structure.html#chromosomal-territories-and-inter-chromosomal-contacts"><i class="fa fa-check"></i><b>1.4.1</b> Chromosomal territories and inter-chromosomal contacts</a></li>
<li class="chapter" data-level="1.4.2" data-path="1-4-hierarchy-of-chromatin-3d-structure.html"><a href="1-4-hierarchy-of-chromatin-3d-structure.html#ab-compartments"><i class="fa fa-check"></i><b>1.4.2</b> A/B compartments</a></li>
<li class="chapter" data-level="1.4.3" data-path="1-4-hierarchy-of-chromatin-3d-structure.html"><a href="1-4-hierarchy-of-chromatin-3d-structure.html#topologically-associating-domains-tads"><i class="fa fa-check"></i><b>1.4.3</b> Topologically associating domains (TADs)</a></li>
<li class="chapter" data-level="1.4.4" data-path="1-4-hierarchy-of-chromatin-3d-structure.html"><a href="1-4-hierarchy-of-chromatin-3d-structure.html#chromatin-loops"><i class="fa fa-check"></i><b>1.4.4</b> Chromatin loops</a></li>
</ul></li>
<li class="chapter" data-level="1.5" data-path="1-5-dynamics-of-chromatin-structure.html"><a href="1-5-dynamics-of-chromatin-structure.html"><i class="fa fa-check"></i><b>1.5</b> Dynamics of chromatin structure</a><ul>
<li class="chapter" data-level="1.5.1" data-path="1-5-dynamics-of-chromatin-structure.html"><a href="1-5-dynamics-of-chromatin-structure.html#variation-of-chromatin-structure-across-cell-types"><i class="fa fa-check"></i><b>1.5.1</b> Variation of chromatin structure across cell types</a></li>
</ul></li>
<li class="chapter" data-level="1.6" data-path="1-6-evolution-of-chromatin-organization.html"><a href="1-6-evolution-of-chromatin-organization.html"><i class="fa fa-check"></i><b>1.6</b> Evolution of chromatin organization</a></li>
<li class="chapter" data-level="1.7" data-path="1-7-disruption-of-chromatin-architecture-in-disease.html"><a href="1-7-disruption-of-chromatin-architecture-in-disease.html"><i class="fa fa-check"></i><b>1.7</b> Disruption of chromatin architecture in disease</a></li>
<li class="chapter" data-level="1.8" data-path="1-8-aims-of-this-thesis.html"><a href="1-8-aims-of-this-thesis.html"><i class="fa fa-check"></i><b>1.8</b> Aims of this thesis</a></li>
<li class="chapter" data-level="1.9" data-path="1-9-structure-of-this-thesis.html"><a href="1-9-structure-of-this-thesis.html"><i class="fa fa-check"></i><b>1.9</b> Structure of this thesis</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="2-paralog-regulation.html"><a href="2-paralog-regulation.html"><i class="fa fa-check"></i><b>2</b> Paralog genes in the 3D genome architecture</a><ul>
<li class="chapter" data-level="" data-path="2-paralog-regulation.html"><a href="2-paralog-regulation.html#preamble"><i class="fa fa-check"></i>Preamble</a></li>
<li class="chapter" data-level="" data-path="2-paralog-regulation.html"><a href="2-paralog-regulation.html#abstract-1"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="2.1" data-path="2-1-introduction.html"><a href="2-1-introduction.html"><i class="fa fa-check"></i><b>2.1</b> Introduction</a></li>
<li class="chapter" data-level="2.2" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html"><i class="fa fa-check"></i><b>2.2</b> Materials and methods</a><ul>
<li class="chapter" data-level="2.2.1" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html#selection-of-pairs-of-paralog-genes"><i class="fa fa-check"></i><b>2.2.1</b> Selection of pairs of paralog genes</a></li>
<li class="chapter" data-level="2.2.2" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html#enhancers-to-gene-association"><i class="fa fa-check"></i><b>2.2.2</b> Enhancers to gene association</a></li>
<li class="chapter" data-level="2.2.3" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html#topological-associating-domains"><i class="fa fa-check"></i><b>2.2.3</b> Topological associating domains</a></li>
<li class="chapter" data-level="2.2.4" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html#hi-c-interaction-maps"><i class="fa fa-check"></i><b>2.2.4</b> Hi-C interaction maps</a></li>
<li class="chapter" data-level="2.2.5" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html#randomization"><i class="fa fa-check"></i><b>2.2.5</b> Randomization</a></li>
<li class="chapter" data-level="2.2.6" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html#statistical-tests"><i class="fa fa-check"></i><b>2.2.6</b> Statistical tests</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="2-3-results.html"><a href="2-3-results.html"><i class="fa fa-check"></i><b>2.3</b> Results</a><ul>
<li class="chapter" data-level="2.3.1" data-path="2-3-results.html"><a href="2-3-results.html#distribution-of-paralog-genes-in-the-human-genome"><i class="fa fa-check"></i><b>2.3.1</b> Distribution of paralog genes in the human genome</a></li>
<li class="chapter" data-level="2.3.2" data-path="2-3-results.html"><a href="2-3-results.html#co-expression-of-paralog-gene-pairs-across-tissues"><i class="fa fa-check"></i><b>2.3.2</b> Co-expression of paralog gene pairs across tissues</a></li>
<li class="chapter" data-level="2.3.3" data-path="2-3-results.html"><a href="2-3-results.html#paralog-genes-share-enhancers"><i class="fa fa-check"></i><b>2.3.3</b> Paralog genes share enhancers</a></li>
<li class="chapter" data-level="2.3.4" data-path="2-3-results.html"><a href="2-3-results.html#co-localization-of-paralogs-in-tads"><i class="fa fa-check"></i><b>2.3.4</b> Co-localization of paralogs in TADs</a></li>
<li class="chapter" data-level="2.3.5" data-path="2-3-results.html"><a href="2-3-results.html#distal-paralog-pairs-are-enriched-for-long-range-chromatin-contacts"><i class="fa fa-check"></i><b>2.3.5</b> Distal paralog pairs are enriched for long-range chromatin contacts</a></li>
<li class="chapter" data-level="2.3.6" data-path="2-3-results.html"><a href="2-3-results.html#close-paralogs-have-fewer-contacts-than-expected"><i class="fa fa-check"></i><b>2.3.6</b> Close paralogs have fewer contacts than expected</a></li>
<li class="chapter" data-level="2.3.7" data-path="2-3-results.html"><a href="2-3-results.html#paralogs-in-mouse-and-dog-genome"><i class="fa fa-check"></i><b>2.3.7</b> Paralogs in mouse and dog genome</a></li>
<li class="chapter" data-level="2.3.8" data-path="2-3-results.html"><a href="2-3-results.html#orthologs-of-human-paralogs-show-conserved-co-localization"><i class="fa fa-check"></i><b>2.3.8</b> Orthologs of human paralogs show conserved co-localization</a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="2-4-discussions.html"><a href="2-4-discussions.html"><i class="fa fa-check"></i><b>2.4</b> Discussions</a></li>
<li class="chapter" data-level="2.5" data-path="2-5-conclusion.html"><a href="2-5-conclusion.html"><i class="fa fa-check"></i><b>2.5</b> Conclusion</a></li>
<li class="chapter" data-level="2.6" data-path="2-6-acknowledgements.html"><a href="2-6-acknowledgements.html"><i class="fa fa-check"></i><b>2.6</b> Acknowledgements</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="3-TAD-evolution.html"><a href="3-TAD-evolution.html"><i class="fa fa-check"></i><b>3</b> Stability of TADs in evolution</a><ul>
<li class="chapter" data-level="" data-path="3-TAD-evolution.html"><a href="3-TAD-evolution.html#preamble-1"><i class="fa fa-check"></i>Preamble</a></li>
<li class="chapter" data-level="" data-path="3-TAD-evolution.html"><a href="3-TAD-evolution.html#abstract-2"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="" data-path="3-TAD-evolution.html"><a href="3-TAD-evolution.html#keywords"><i class="fa fa-check"></i>Keywords</a></li>
<li class="chapter" data-level="3.1" data-path="3-1-introduction-1.html"><a href="3-1-introduction-1.html"><i class="fa fa-check"></i><b>3.1</b> Introduction</a></li>
<li class="chapter" data-level="3.2" data-path="3-2-results-1.html"><a href="3-2-results-1.html"><i class="fa fa-check"></i><b>3.2</b> Results</a><ul>
<li class="chapter" data-level="3.2.1" data-path="3-2-results-1.html"><a href="3-2-results-1.html#identification-of-evolutionary-rearrangement-breakpoints-from-whole-genome-alignments"><i class="fa fa-check"></i><b>3.2.1</b> Identification of evolutionary rearrangement breakpoints from whole-genome alignments</a></li>
<li class="chapter" data-level="3.2.2" data-path="3-2-results-1.html"><a href="3-2-results-1.html#rearrangement-breakpoints-are-enriched-at-tad-boundaries"><i class="fa fa-check"></i><b>3.2.2</b> Rearrangement breakpoints are enriched at TAD boundaries</a></li>
<li class="chapter" data-level="3.2.3" data-path="3-2-results-1.html"><a href="3-2-results-1.html#clusters-of-conserved-non-coding-elements-are-depleted-for-rearrangement-breakpoints"><i class="fa fa-check"></i><b>3.2.3</b> Clusters of conserved non-coding elements are depleted for rearrangement breakpoints</a></li>
<li class="chapter" data-level="3.2.4" data-path="3-2-results-1.html"><a href="3-2-results-1.html#rearranged-tads-are-associated-with-divergent-gene-expression-between-species"><i class="fa fa-check"></i><b>3.2.4</b> Rearranged TADs are associated with divergent gene expression between species</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="3-3-discussion.html"><a href="3-3-discussion.html"><i class="fa fa-check"></i><b>3.3</b> Discussion</a></li>
<li class="chapter" data-level="3.4" data-path="3-4-conclusion-1.html"><a href="3-4-conclusion-1.html"><i class="fa fa-check"></i><b>3.4</b> Conclusion</a></li>
<li class="chapter" data-level="3.5" data-path="3-5-methods.html"><a href="3-5-methods.html"><i class="fa fa-check"></i><b>3.5</b> Methods</a><ul>
<li class="chapter" data-level="3.5.1" data-path="3-5-methods.html"><a href="3-5-methods.html#rearrangement-breakpoints-from-whole-genome-alignments"><i class="fa fa-check"></i><b>3.5.1</b> Rearrangement breakpoints from whole-genome alignments</a></li>
<li class="chapter" data-level="3.5.2" data-path="3-5-methods.html"><a href="3-5-methods.html#topologically-associating-domains-and-contact-domains"><i class="fa fa-check"></i><b>3.5.2</b> Topologically associating domains and contact domains</a></li>
<li class="chapter" data-level="3.5.3" data-path="3-5-methods.html"><a href="3-5-methods.html#breakpoint-distributions-at-tads"><i class="fa fa-check"></i><b>3.5.3</b> Breakpoint distributions at TADs</a></li>
<li class="chapter" data-level="3.5.4" data-path="3-5-methods.html"><a href="3-5-methods.html#quantification-of-breakpoint-enrichment"><i class="fa fa-check"></i><b>3.5.4</b> Quantification of breakpoint enrichment</a></li>
<li class="chapter" data-level="3.5.5" data-path="3-5-methods.html"><a href="3-5-methods.html#expression-data-for-mouse-and-human-orthologs"><i class="fa fa-check"></i><b>3.5.5</b> Expression data for mouse and human orthologs</a></li>
<li class="chapter" data-level="3.5.6" data-path="3-5-methods.html"><a href="3-5-methods.html#classification-of-tads-and-genes-according-to-rearrangements-and-grbs"><i class="fa fa-check"></i><b>3.5.6</b> Classification of TADs and genes according to rearrangements and GRBs</a></li>
<li class="chapter" data-level="3.5.7" data-path="3-5-methods.html"><a href="3-5-methods.html#source-code-and-implementation-details"><i class="fa fa-check"></i><b>3.5.7</b> Source code and implementation details</a></li>
</ul></li>
<li class="chapter" data-level="3.6" data-path="3-6-declarations.html"><a href="3-6-declarations.html"><i class="fa fa-check"></i><b>3.6</b> Declarations</a><ul>
<li class="chapter" data-level="3.6.1" data-path="3-6-declarations.html"><a href="3-6-declarations.html#ethics-approval-and-consent-to-participate"><i class="fa fa-check"></i><b>3.6.1</b> Ethics approval and consent to participate</a></li>
<li class="chapter" data-level="3.6.2" data-path="3-6-declarations.html"><a href="3-6-declarations.html#consent-for-publication"><i class="fa fa-check"></i><b>3.6.2</b> Consent for publication</a></li>
<li class="chapter" data-level="3.6.3" data-path="3-6-declarations.html"><a href="3-6-declarations.html#availability-of-data-and-material"><i class="fa fa-check"></i><b>3.6.3</b> Availability of data and material</a></li>
<li class="chapter" data-level="3.6.4" data-path="3-6-declarations.html"><a href="3-6-declarations.html#competing-interests"><i class="fa fa-check"></i><b>3.6.4</b> Competing interests</a></li>
<li class="chapter" data-level="3.6.5" data-path="3-6-declarations.html"><a href="3-6-declarations.html#funding"><i class="fa fa-check"></i><b>3.6.5</b> Funding</a></li>
<li class="chapter" data-level="3.6.6" data-path="3-6-declarations.html"><a href="3-6-declarations.html#authors-contributions"><i class="fa fa-check"></i><b>3.6.6</b> Authors‘ contributions</a></li>
<li class="chapter" data-level="3.6.7" data-path="3-6-declarations.html"><a href="3-6-declarations.html#acknowledgments"><i class="fa fa-check"></i><b>3.6.7</b> Acknowledgments</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="4-position-effect.html"><a href="4-position-effect.html"><i class="fa fa-check"></i><b>4</b> Position effects of rearrangements in disease genomes</a><ul>
<li class="chapter" data-level="" data-path="4-position-effect.html"><a href="4-position-effect.html#preamble-2"><i class="fa fa-check"></i>Preamble</a></li>
<li class="chapter" data-level="" data-path="4-position-effect.html"><a href="4-position-effect.html#abstract-3"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="4.1" data-path="4-1-introduction-2.html"><a href="4-1-introduction-2.html"><i class="fa fa-check"></i><b>4.1</b> Introduction</a></li>
<li class="chapter" data-level="4.2" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html"><i class="fa fa-check"></i><b>4.2</b> Materials and Methods</a><ul>
<li class="chapter" data-level="4.2.1" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html#selection-of-subjects-with-apparently-balanced-chromosome-abnormalities"><i class="fa fa-check"></i><b>4.2.1</b> Selection of subjects with apparently balanced chromosome abnormalities</a></li>
<li class="chapter" data-level="4.2.2" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html#clinical-descriptions-of-dgap-cases"><i class="fa fa-check"></i><b>4.2.2</b> Clinical descriptions of DGAP cases</a></li>
<li class="chapter" data-level="4.2.3" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html#analysis-of-genes-bordering-the-rearrangement-breakpoints"><i class="fa fa-check"></i><b>4.2.3</b> Analysis of genes bordering the rearrangement breakpoints</a></li>
<li class="chapter" data-level="4.2.4" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html#assessment-of-disrupted-functional-elements-and-chromatin-interactions-bordering-rearrangement-breakpoints"><i class="fa fa-check"></i><b>4.2.4</b> Assessment of disrupted functional elements and chromatin interactions bordering rearrangement breakpoints</a></li>
<li class="chapter" data-level="4.2.5" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html#ontological-analysis-of-genes-neighboring-breakpoints"><i class="fa fa-check"></i><b>4.2.5</b> Ontological analysis of genes neighboring breakpoints</a></li>
<li class="chapter" data-level="4.2.6" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html#quantitative-real-time-pcr"><i class="fa fa-check"></i><b>4.2.6</b> Quantitative real-time PCR</a></li>
<li class="chapter" data-level="4.2.7" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html#assessment-of-dgap-breakpoints-overlapping-with-non-coding-structural-variants-in-public-databases"><i class="fa fa-check"></i><b>4.2.7</b> Assessment of DGAP breakpoints overlapping with non-coding structural variants in public databases</a></li>
</ul></li>
<li class="chapter" data-level="4.3" data-path="4-3-results-2.html"><a href="4-3-results-2.html"><i class="fa fa-check"></i><b>4.3</b> Results</a><ul>
<li class="chapter" data-level="4.3.1" data-path="4-3-results-2.html"><a href="4-3-results-2.html#genomic-characterization-of-non-coding-breakpoints"><i class="fa fa-check"></i><b>4.3.1</b> Genomic characterization of non-coding breakpoints</a></li>
<li class="chapter" data-level="4.3.2" data-path="4-3-results-2.html"><a href="4-3-results-2.html#identification-of-genes-with-potential-position-effects"><i class="fa fa-check"></i><b>4.3.2</b> Identification of genes with potential position effects</a></li>
<li class="chapter" data-level="4.3.3" data-path="4-3-results-2.html"><a href="4-3-results-2.html#identification-of-subjects-with-shared-non-coding-chromosome-alterations-and-phenotypes"><i class="fa fa-check"></i><b>4.3.3</b> Identification of subjects with shared non-coding chromosome alterations and phenotypes</a></li>
</ul></li>
<li class="chapter" data-level="4.4" data-path="4-4-discussion-1.html"><a href="4-4-discussion-1.html"><i class="fa fa-check"></i><b>4.4</b> Discussion</a><ul>
<li class="chapter" data-level="" data-path="4-4-discussion-1.html"><a href="4-4-discussion-1.html#acknowledgements-1"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="4.4.1" data-path="4-4-discussion-1.html"><a href="4-4-discussion-1.html#web-resources"><i class="fa fa-check"></i><b>4.4.1</b> Web Resources</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="5-loop.html"><a href="5-loop.html"><i class="fa fa-check"></i><b>5</b> Prediction of chromatin looping interactions</a><ul>
<li class="chapter" data-level="" data-path="5-loop.html"><a href="5-loop.html#preamble-3"><i class="fa fa-check"></i>Preamble</a></li>
<li class="chapter" data-level="" data-path="5-loop.html"><a href="5-loop.html#abstract-4"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="" data-path="5-loop.html"><a href="5-loop.html#keywords-1"><i class="fa fa-check"></i>Keywords</a></li>
<li class="chapter" data-level="5.1" data-path="5-1-introduction-3.html"><a href="5-1-introduction-3.html"><i class="fa fa-check"></i><b>5.1</b> Introduction</a></li>
<li class="chapter" data-level="5.2" data-path="5-2-results-3.html"><a href="5-2-results-3.html"><i class="fa fa-check"></i><b>5.2</b> Results</a><ul>
<li class="chapter" data-level="5.2.1" data-path="5-2-results-3.html"><a href="5-2-results-3.html#ctcf-motif-pairs-as-candidate-chromatin-loop-anchors"><i class="fa fa-check"></i><b>5.2.1</b> CTCF motif pairs as candidate chromatin loop anchors</a></li>
<li class="chapter" data-level="5.2.2" data-path="5-2-results-3.html"><a href="5-2-results-3.html#similarity-of-chip-seq-signals-at-looping-ctcf-motifs"><i class="fa fa-check"></i><b>5.2.2</b> Similarity of ChIP-seq signals at looping CTCF motifs</a></li>
<li class="chapter" data-level="5.2.3" data-path="5-2-results-3.html"><a href="5-2-results-3.html#genomic-sequence-features-of-ctcf-motif-pairs-are-associated-with-looping"><i class="fa fa-check"></i><b>5.2.3</b> Genomic sequence features of CTCF motif pairs are associated with looping</a></li>
<li class="chapter" data-level="5.2.4" data-path="5-2-results-3.html"><a href="5-2-results-3.html#chromatin-loop-prediction-using-7c"><i class="fa fa-check"></i><b>5.2.4</b> Chromatin loop prediction using 7C</a></li>
<li class="chapter" data-level="5.2.5" data-path="5-2-results-3.html"><a href="5-2-results-3.html#prediction-performance-evaluation"><i class="fa fa-check"></i><b>5.2.5</b> Prediction performance evaluation</a></li>
<li class="chapter" data-level="5.2.6" data-path="5-2-results-3.html"><a href="5-2-results-3.html#prediction-performance-of-7c-with-sequence-features-and-single-tf-chip-seq-data-sets"><i class="fa fa-check"></i><b>5.2.6</b> Prediction performance of 7C with sequence features and single TF ChIP-seq data sets</a></li>
<li class="chapter" data-level="5.2.7" data-path="5-2-results-3.html"><a href="5-2-results-3.html#comparison-of-transcription-factors-by-prediction-performance"><i class="fa fa-check"></i><b>5.2.7</b> Comparison of transcription factors by prediction performance</a></li>
<li class="chapter" data-level="5.2.8" data-path="5-2-results-3.html"><a href="5-2-results-3.html#prediction-performance-in-other-cell-types-and-for-different-tfs"><i class="fa fa-check"></i><b>5.2.8</b> Prediction performance in other cell types and for different TFs</a></li>
<li class="chapter" data-level="5.2.9" data-path="5-2-results-3.html"><a href="5-2-results-3.html#the-high-resolution-of-chip-nexus-improves-prediction-performance"><i class="fa fa-check"></i><b>5.2.9</b> The high resolution of ChIP-nexus improves prediction performance</a></li>
</ul></li>
<li class="chapter" data-level="5.3" data-path="5-3-discussion-2.html"><a href="5-3-discussion-2.html"><i class="fa fa-check"></i><b>5.3</b> Discussion</a></li>
<li class="chapter" data-level="5.4" data-path="5-4-conclusion-2.html"><a href="5-4-conclusion-2.html"><i class="fa fa-check"></i><b>5.4</b> Conclusion</a></li>
<li class="chapter" data-level="5.5" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html"><i class="fa fa-check"></i><b>5.5</b> Methods</a><ul>
<li class="chapter" data-level="5.5.1" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#ctcf-motifs-in-the-human-genome"><i class="fa fa-check"></i><b>5.5.1</b> CTCF motifs in the human genome</a></li>
<li class="chapter" data-level="5.5.2" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#loop-interaction-data-for-training-and-validation"><i class="fa fa-check"></i><b>5.5.2</b> Loop interaction data for training and validation</a></li>
<li class="chapter" data-level="5.5.3" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#chip-seq-datasets-in-gm12878-cells"><i class="fa fa-check"></i><b>5.5.3</b> ChIP-seq datasets in GM12878 cells</a></li>
<li class="chapter" data-level="5.5.4" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#chip-seq-data-types"><i class="fa fa-check"></i><b>5.5.4</b> ChIP-seq data types</a></li>
<li class="chapter" data-level="5.5.5" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#chip-nexus-data-processing-for-rad21-and-scm3"><i class="fa fa-check"></i><b>5.5.5</b> ChIP-nexus data processing for RAD21 and SCM3</a></li>
<li class="chapter" data-level="5.5.6" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#similarity-of-chip-seq-profiles-as-correlation-of-coverage-around-motifs"><i class="fa fa-check"></i><b>5.5.6</b> Similarity of ChIP-seq profiles as correlation of coverage around motifs</a></li>
<li class="chapter" data-level="5.5.7" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#genomic-sequence-features-of-chromatin-loops"><i class="fa fa-check"></i><b>5.5.7</b> Genomic sequence features of chromatin loops</a></li>
<li class="chapter" data-level="5.5.8" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#chromatin-loop-prediction-model"><i class="fa fa-check"></i><b>5.5.8</b> Chromatin Loop prediction model</a></li>
<li class="chapter" data-level="5.5.9" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#training-and-validation-of-prediction-model"><i class="fa fa-check"></i><b>5.5.9</b> Training and validation of prediction model</a></li>
<li class="chapter" data-level="5.5.10" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#analysis-of-prediction-performance"><i class="fa fa-check"></i><b>5.5.10</b> Analysis of prediction performance</a></li>
<li class="chapter" data-level="5.5.11" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#implementation-of-7c-and-compatibility-to-other-tools"><i class="fa fa-check"></i><b>5.5.11</b> Implementation of 7C and compatibility to other tools</a></li>
</ul></li>
<li class="chapter" data-level="5.6" data-path="5-6-declarations-1.html"><a href="5-6-declarations-1.html"><i class="fa fa-check"></i><b>5.6</b> Declarations</a><ul>
<li class="chapter" data-level="5.6.1" data-path="5-6-declarations-1.html"><a href="5-6-declarations-1.html#ethics-approval-and-consent-to-participate-1"><i class="fa fa-check"></i><b>5.6.1</b> Ethics approval and consent to participate</a></li>
<li class="chapter" data-level="5.6.2" data-path="5-6-declarations-1.html"><a href="5-6-declarations-1.html#consent-for-publication-1"><i class="fa fa-check"></i><b>5.6.2</b> Consent for publication</a></li>
<li class="chapter" data-level="5.6.3" data-path="5-6-declarations-1.html"><a href="5-6-declarations-1.html#availability-of-data-and-material-1"><i class="fa fa-check"></i><b>5.6.3</b> Availability of data and material</a></li>
<li class="chapter" data-level="5.6.4" data-path="5-6-declarations-1.html"><a href="5-6-declarations-1.html#competing-interests-1"><i class="fa fa-check"></i><b>5.6.4</b> Competing interests</a></li>
<li class="chapter" data-level="5.6.5" data-path="5-6-declarations-1.html"><a href="5-6-declarations-1.html#funding-1"><i class="fa fa-check"></i><b>5.6.5</b> Funding</a></li>
<li class="chapter" data-level="5.6.6" data-path="5-6-declarations-1.html"><a href="5-6-declarations-1.html#authors-contributions-1"><i class="fa fa-check"></i><b>5.6.6</b> Authors‘ contributions</a></li>
<li class="chapter" data-level="5.6.7" data-path="5-6-declarations-1.html"><a href="5-6-declarations-1.html#acknowledgments-1"><i class="fa fa-check"></i><b>5.6.7</b> Acknowledgments</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="6-thesis-discussion.html"><a href="6-thesis-discussion.html"><i class="fa fa-check"></i><b>6</b> Discussion</a><ul>
<li class="chapter" data-level="6.1" data-path="6-1-discussion-paralog-co-regulation-in-tads.html"><a href="6-1-discussion-paralog-co-regulation-in-tads.html"><i class="fa fa-check"></i><b>6.1</b> Discussion paralog co-regulation in TADs</a></li>
<li class="chapter" data-level="6.2" data-path="6-2-discussion-on-tad-evolution.html"><a href="6-2-discussion-on-tad-evolution.html"><i class="fa fa-check"></i><b>6.2</b> Discussion on TAD evolution</a></li>
<li class="chapter" data-level="6.3" data-path="6-3-further-directions.html"><a href="6-3-further-directions.html"><i class="fa fa-check"></i><b>6.3</b> Further directions</a></li>
<li class="chapter" data-level="6.4" data-path="6-4-conclusions.html"><a href="6-4-conclusions.html"><i class="fa fa-check"></i><b>6.4</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="appendix"><span><b>Appendix</b></span></li>
<li class="chapter" data-level="A" data-path="A-sup-paralog-regulation.html"><a href="A-sup-paralog-regulation.html"><i class="fa fa-check"></i><b>A</b> Co-regulation of paralog genes: Supporting Information</a></li>
<li class="chapter" data-level="B" data-path="B-sup-tad-evolution.html"><a href="B-sup-tad-evolution.html"><i class="fa fa-check"></i><b>B</b> TAD evolution: Supporting Information</a><ul>
<li class="chapter" data-level="B.1" data-path="B-1-TadEvoSupTab.html"><a href="B-1-TadEvoSupTab.html"><i class="fa fa-check"></i><b>B.1</b> Supplementary Tables</a></li>
<li class="chapter" data-level="B.2" data-path="B-2-supplementary-figures.html"><a href="B-2-supplementary-figures.html"><i class="fa fa-check"></i><b>B.2</b> Supplementary Figures</a></li>
</ul></li>
<li class="chapter" data-level="C" data-path="C-sup-position-effect.html"><a href="C-sup-position-effect.html"><i class="fa fa-check"></i><b>C</b> Position Effect: Supplemental Data</a><ul>
<li class="chapter" data-level="C.1" data-path="C-1-supplemental-note-case-reports.html"><a href="C-1-supplemental-note-case-reports.html"><i class="fa fa-check"></i><b>C.1</b> Supplemental Note: Case Reports</a></li>
<li class="chapter" data-level="C.2" data-path="C-2-supplemental-note-nucleotide-level-nomenclature-for-dgap-karyotypes.html"><a href="C-2-supplemental-note-nucleotide-level-nomenclature-for-dgap-karyotypes.html"><i class="fa fa-check"></i><b>C.2</b> Supplemental Note: Nucleotide Level Nomenclature for DGAP karyotypes</a></li>
<li class="chapter" data-level="C.3" data-path="C-3-supplemental-figure.html"><a href="C-3-supplemental-figure.html"><i class="fa fa-check"></i><b>C.3</b> Supplemental Figure</a></li>
<li class="chapter" data-level="C.4" data-path="C-4-position-effect-sub-tab.html"><a href="C-4-position-effect-sub-tab.html"><i class="fa fa-check"></i><b>C.4</b> Supplemental Table Legends</a></li>
</ul></li>
<li class="chapter" data-level="D" data-path="D-sup-loop.html"><a href="D-sup-loop.html"><i class="fa fa-check"></i><b>D</b> Loop prediction: Supplemental Information</a><ul>
<li class="chapter" data-level="D.1" data-path="D-1-LoopPredSupTab.html"><a href="D-1-LoopPredSupTab.html"><i class="fa fa-check"></i><b>D.1</b> Supplementary Tables</a></li>
<li class="chapter" data-level="D.2" data-path="D-2-supplementary-figures-1.html"><a href="D-2-supplementary-figures-1.html"><i class="fa fa-check"></i><b>D.2</b> Supplementary Figures</a></li>
</ul></li>
<li class="chapter" data-level="E" data-path="E-contribution-to-individual-publications.html"><a href="E-contribution-to-individual-publications.html"><i class="fa fa-check"></i><b>E</b> Contribution to individual publications</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Build with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Genome folding in evolution and disease</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="contribution-to-individual-publications" class="section level1">
<h1><span class="header-section-number">E</span> Contribution to individual publications</h1>
<table>
<caption><span id="tab:contribution">Table E.1: </span>Contribution to individual publications. Contributions in percent from all authors for each contribution role and publication. Author contribution definitions according to the CRediT Taxonomy (<a href="http://journals.plos.org/plosone/s/authorship" class="uri">http://journals.plos.org/plosone/s/authorship</a>).</caption>
<thead>
<tr class="header">
<th align="left">Contributor Role</th>
<th align="left">Role Definition</th>
<th align="right">Paralog regulation (Chapter <a href="2-paralog-regulation.html#paralog-regulation">2</a>)</th>
<th align="right">TAD evolution (Chapter <a href="3-TAD-evolution.html#TAD-evolution">3</a>)</th>
<th align="right">Position effect (Chapter <a href="4-position-effect.html#position-effect">4</a>)</th>
<th align="right">Loop prediction (Chapter <a href="5-loop.html#loop">5</a>)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">Conceptualization</td>
<td align="left">Ideas; formulation or evolution of overarching research goals and aims.</td>
<td align="right">80</td>
<td align="right">90</td>
<td align="right">20</td>
<td align="right">90</td>
</tr>
<tr class="even">
<td align="left">Data Curation</td>
<td align="left">Management activities to annotate (produce metadata), scrub data and maintain research data (including software code, where it is necessary for interpreting the data itself) for initial use and later reuse.</td>
<td align="right">100</td>
<td align="right">80</td>
<td align="right">50</td>
<td align="right">100</td>
</tr>
<tr class="odd">
<td align="left">Formal Analysis</td>
<td align="left">Application of statistical, mathematical, computational, or other formal techniques to analyze or synthesize study data.</td>
<td align="right">100</td>
<td align="right">50</td>
<td align="right">40</td>
<td align="right">100</td>
</tr>
<tr class="even">
<td align="left">Funding Acquisition</td>
<td align="left">Acquisition of the financial support for the project leading to this publication.</td>
<td align="right">0</td>
<td align="right">0</td>
<td align="right">0</td>
<td align="right">0</td>
</tr>
<tr class="odd">
<td align="left">Investigation</td>
<td align="left">Conducting a research and investigation process, specifically performing the experiments, or data/evidence collection.</td>
<td align="right">100</td>
<td align="right">80</td>
<td align="right">20</td>
<td align="right">100</td>
</tr>
<tr class="even">
<td align="left">Methodology</td>
<td align="left">Development or design of methodology; creation of models</td>
<td align="right">90</td>
<td align="right">80</td>
<td align="right">30</td>
<td align="right">100</td>
</tr>
<tr class="odd">
<td align="left">Project Administration</td>
<td align="left">Management and coordination responsibility for the research activity planning and execution.</td>
<td align="right">50</td>
<td align="right">80</td>
<td align="right">0</td>
<td align="right">80</td>
</tr>
<tr class="even">
<td align="left">Resources</td>
<td align="left">Provision of study materials, reagents, materials, patients, laboratory samples, animals, instrumentation, computing resources, or other analysis tools.</td>
<td align="right">0</td>
<td align="right">20</td>
<td align="right">0</td>
<td align="right">0</td>
</tr>
<tr class="odd">
<td align="left">Software</td>
<td align="left">Programming, software development; designing computer programs; implementation of the computer code and supporting algorithms; testing of existing code components.</td>
<td align="right">100</td>
<td align="right">50</td>
<td align="right">50</td>
<td align="right">100</td>
</tr>
<tr class="even">
<td align="left">Supervision</td>
<td align="left">Oversight and leadership responsibility for the research activity planning and execution, including mentorship external to the core team.</td>
<td align="right">0</td>
<td align="right">80</td>
<td align="right">0</td>
<td align="right">0</td>
</tr>
<tr class="odd">
<td align="left">Validation</td>
<td align="left">Verification, whether as a part of the activity or separate, of the overall replication/reproducibility of results/experiments and other research outputs.</td>
<td align="right">100</td>
<td align="right">80</td>
<td align="right">20</td>
<td align="right">100</td>
</tr>
<tr class="even">
<td align="left">Visualization</td>
<td align="left">Preparation, creation and/or presentation of the published work, specifically visualization/data presentation.</td>
<td align="right">100</td>
<td align="right">50</td>
<td align="right">0</td>
<td align="right">100</td>
</tr>
<tr class="odd">
<td align="left">Writing – Original Draft Preparation</td>
<td align="left">Creation and/or presentation of the published work, specifically writing the initial draft (including substantive translation).</td>
<td align="right">100</td>
<td align="right">60</td>
<td align="right">20</td>
<td align="right">100</td>
</tr>
<tr class="even">
<td align="left">Writing – Review &amp; Editing</td>
<td align="left">Preparation, creation and/or presentation of the published work by those from the original research group, specifically critical review, commentary or revision – including pre- or post-publication stages.</td>
<td align="right">50</td>
<td align="right">80</td>
<td align="right">20</td>
<td align="right">50</td>
</tr>
</tbody>
</table>

<div id="refs" class="references">
<div>
<p>Acemel RD, Maeso I, Gómez‐Skarmeta JL. 2017. Topologically associated domains: a successful scaffold for the evolution of gene regulation in animals. Wiley Interdisciplinary Reviews: Developmental Biology: 1–19.</p>
</div>
<div>
<p>Altshuler DM, Gibbs RA, Peltonen L, Altshuler DM, Gibbs RA, Peltonen L, Dermitzakis E, Schaffner SF, Yu F, Peltonen L, Dermitzakis E, Bonnen PE, Altshuler DM, Gibbs RA, Bakker PIW de, Deloukas P, Gabriel SB, Gwilliam R, Hunt S, Inouye M, Jia X, Palotie A, Parkin M, Whittaker P, Yu F, Chang K, Hawes A, Lewis LR, Ren Y, Wheeler D, Gibbs RA, Marie Muzny D, Barnes C, Darvishi K, Hurles M, Korn JM, Kristiansson K, Lee C, McCarroll SA, Nemesh J, Dermitzakis E, Keinan A, Montgomery SB, Pollack S, Price AL, Soranzo N, Bonnen PE, Gibbs RA, Gonzaga-Jauregui C, Keinan A, Price AL, Yu F, Anttila V, Brodeur W, Daly MJ, Leslie S, McVean G, Moutsianas L, Nguyen H, Schaffner SF, Zhang Q, Ghori MJR, McGinnis R, McLaren W, Pollack S, Price AL, Schaffner SF, Takeuchi F, Grossman SR, Shlyakhter I, Hostetter EB, Sabeti PC, Adebamowo CA, Foster MW, Gordon DR, Licinio J, Cristina Manca M, Marshall PA, Matsuda I, Ngare D, Ota Wang V, Reddy D, Rotimi CN, Royal CD, Sharp RR, Zeng C, Brooks LD, McEwen JE. 2010. Integrating common and rare genetic variation in diverse human populations. Nature 467: 52–58.</p>
</div>
<div>
<p>Andersson R, Gebhard C, Miguel-Escalada I, Hoof I, Bornholdt J, Boyd M, Chen Y, Zhao X, Schmidl C, Suzuki T, Ntini E, Arner E, Valen E, Li K, Schwarzfischer L, Glatz D, Raithel J, Lilje B, Rapin N, Bagger FO, Jørgensen M, Andersen PR, Bertin N, Rackham O, Burroughs a M, Baillie JK, Ishizu Y, Shimizu Y, Furuhata E, Maeda S, Negishi Y, Mungall CJ, Meehan TF, Lassmann T, Itoh M, Kawaji H, Kondo N, Kawai J, Lennartsson A, Daub CO, Heutink P, Hume D a, Jensen TH, Suzuki H, Hayashizaki Y, Müller F, Forrest ARR, Carninci P, Rehli M, Sandelin A. 2014. An atlas of active enhancers across human cell types and tissues. Nature 507: 455–61.</p>
</div>
<div>
<p>Andrey G, Mundlos S. 2017. The three-dimensional genome: regulating gene expression during pluripotency and development. Development 144: 3646–3658.</p>
</div>
<div>
<p>Anger GJ, Crocker S, McKenzie K, Brown KK, Morton CC, Harrison K, MacKenzie JJ. 2014. X-Linked Deafness-2 (DFNX2) Phenotype Associated With a Paracentric Inversion Upstream of &lt;i&gt;POU3F4&lt;/i&gt;. American Journal of Audiology 23: 1.</p>
</div>
<div>
<p>Ardlie KG, DeLuca DS, Segrè AV, Sullivan TJ, Young TR, Gelfand ET, Trowbridge CA, Maller JB, Tukiainen T, Lek M, Ward LD, Kheradpour P, Iriarte B, Meng Y, Palmer CD, Esko T, Winckler W, Hirschhorn JN, Kellis M, MacArthur DG, Getz G, Shabalin AA, Li G, Zhou YH, Nobel AB, Rusyn I, Wright FA, Lappalainen T, Ferreira PG, Ongen H, Rivas MA, Battle A, Mostafavi S, Monlong J, Sammeth M, Melé M, Reverter F, Goldmann JM, Koller D, Guigó R, McCarthy MI, Dermitzakis ET, Gamazon ER, Im HK, Konkashbaev A, Nicolae DL, Cox NJ, Flutre T, Wen X, Stephens M, Pritchard JK, Tu Z, Zhang B, Huang T, Long Q, Lin L, Yang J, Zhu J, Liu J, Brown A, Mestichelli B, Tidwell D, Lo E, Salvatore M, Shad S, Thomas JA, Lonsdale JT, Moser MT, Gillard BM, Karasik E, Ramsey K, Choi C, Foster BA, Syron J, Fleming J, Magazine H, Hasz R, Walters GD, Bridge JP, Miklos M, Sullivan S, Barker LK, Traino HM, Mosavel M, Siminoff LA, Valley DR, Rohrer DC, Jewell SD, Branton PA, Sobin LH, Barcus M, Qi L, McLean J, Hariharan P, Um KS, Wu S, Tabor D, Shive C, et al. 2015. The Genotype-Tissue Expression (GTEx) pilot analysis: Multitissue gene regulation in humans. Science 348: 648–660.</p>
</div>
<div>
<p>Ay F, Noble WS. 2015. Analysis methods for studying the 3D architecture of the genome. Genome Biology 16: 183.</p>
</div>
<div>
<p>Bailey SD, Zhang X, Desai K, Aid M, Corradin O, Cowper-Sal Lari R, Akhtar-Zaidi B, Scacheri PC, Haibe-Kains B, Lupien M, Cowper-Sallari R, Akhtar-Zaidi B, Scacheri PC, Haibe-Kains B, Lupien M. 2015. ZNF143 provides sequence specificity to secure chromatin interactions at gene promoters. Nature Communications 2: 6186.</p>
</div>
<div>
<p>Banerji J, Rusconi S, Schaffner W. 1981. Expression of a <span class="math inline">\(\beta\)</span>-globin gene is enhanced by remote SV40 DNA sequences. Cell 27: 299–308.</p>
</div>
<div>
<p>Bantignies F, Roure V, Comet I, Leblanc B, Schuettengruber B, Bonnet J, Tixier V, Mas A, Cavalli G. 2011. Polycomb-dependent regulatory contacts between distant Hox loci in Drosophila. Cell 144: 214–26.</p>
</div>
<div>
<p>Becker M, Mah N, Zdzieblo D, Li X, Mer A, Andrade-navarro MA, Mu AM. 2015. Epigenetic Mechanisms in Cellular Reprogramming. In: Meissner A, Walter J, editors. Epigenetics and human health Epigenetics and human health. Berlin, Heidelberg: Springer Berlin Heidelberg, p pp 141–166.</p>
</div>
<div>
<p>Berthelot C, Villar D, Horvath JE, Odom DT, Flicek P. 2017. Complexity and conservation of regulatory landscapes underlie evolutionary resilience of mammalian gene expression. Nature Ecology &amp; Evolution: 1–31.</p>
</div>
<div>
<p>Bonev B, Cavalli G. 2016. Organization and function of the 3D genome. Nature Reviews Genetics 17: 661–678.</p>
</div>
<div>
<p>Bonev B, Cohen NM, Szabo Q, Hugnot J-p, Tanay A, Cavalli G, Bonev B, Cohen NM, Szabo Q, Fritsch L, Papadopoulos GL, Lubling Y. 2017. Multiscale 3D Genome Rewiring during Mouse Article Multiscale 3D Genome Rewiring during Mouse Neural Development. Cell 171: 557.e1–557.e24.</p>
</div>
<div>
<p>Boutanaev a M, Kalmykova a I, Shevelyov YY, Nurminsky DI, Smith M, Evolution T, Masterpiece T, Helm C, Selection N. 2002. Large clusters of co-expressed genes in the Drosophila genome. Nature 420: 666–669.</p>
</div>
<div>
<p>Bouwman BA, Laat W de. 2015. Getting the genome in shape: the formation of loops, domains and compartments. Genome Biology 16: 154.</p>
</div>
<div>
<p>Brackley CA, Brown JM, Waithe D, Babbs C, Davies J, Hughes JR, Buckle VJ, Marenduzzo D. 2016. Predicting the three-dimensional folding of cis -regulatory regions in mammalian genomes using bioinformatic data and polymer models. Genome Biology: 31–36.</p>
</div>
<div>
<p>Branco MR, Pombo A. 2006. Intermingling of chromosome territories in interphase suggests role in translocations and transcription-dependent associations. PLoS Biology 4: 780–788.</p>
</div>
<div>
<p>Buenrostro JD, Giresi PG, Zaba LC, Chang HY, Greenleaf WJ. 2013. Transposition of native chromatin for fast and sensitive epigenomic profiling of open chromatin, DNA-binding proteins and nucleosome position. Nature Methods 10: 1213–1218.</p>
</div>
<div>
<p>Cabili MN, Trapnell C, Goff L, Koziol M, Tazon-Vega B, Regev A, Rinn JL. 2011. Integrative annotation of human large intergenic noncoding RNAs reveals global properties and specific subclasses. Genes &amp; development 25: 1915–27.</p>
</div>
<div>
<p>Cai J, Goodman BK, Patel AS, Mulliken JB, Van Maldergem L, Hoganson GE, Paznekas WA, Ben-Neriah Z, Sheffer R, Cunningham ML, Daentl DL, Jabs EW. 2003. Increased risk for developmental delay in Saethre-Chotzen syndrome is associated with TWIST deletions: an improved strategy for TWIST mutation screening. Human Genetics 114: 68–76.</p>
</div>
<div>
<p>Canela A, Maman Y, Jung S, Wong N, Callen E, Day A, Kieffer-Kwon K-R, Pekowska A, Zhang H, Rao SS, Huang S-c, Mckinnon PJ, Aplan PD, Pommier Y, Aiden EL, Casellas R, Nussenzweig A. 2017. Genome Organization Drives Chromosome Fragility. Cell: 1–15.</p>
</div>
<div>
<p>Cardoso AR, Oliveira M, Amorim A, Azevedo L. 2016. Major influence of repetitive elements on disease-associated copy number variants (CNVs). Human Genomics 10: 30.</p>
</div>
<div>
<p>Carroll SB. 2008. Evo-Devo and an Expanding Evolutionary Synthesis: A Genetic Theory of Morphological Evolution. Cell 134: 25–36.</p>
</div>
<div>
<p>Carter D, Chakalova L, Osborne CS, Dai YF, Fraser P. 2002. Long-range chromatin regulatory interactions in vivo. Nat Genet 32: 623–626.</p>
</div>
<div>
<p>Carvalho CMB, Lupski JR. 2016. Mechanisms underlying structural variant formation in genomic disorders. Nature Reviews Genetics 17: 224–238.</p>
</div>
<div>
<p>Chandra T, Ewels PA, Schoenfelder S, Furlan-Magaril M, Wingett SW, Kirschner K, Thuret JY, Andrews S, Fraser P, Reik W. 2015. Global reorganization of the nuclear landscape in senescent cells. Cell Reports 10: 471–484.</p>
</div>
<div>
<p>Chen Y, Wang Y, Xuan Z, Chen M, Zhang MQ. 2016. De novo deciphering three-dimensional chromatin interaction and topological domains by wavelet transformation of epigenetic profiles. Nucleic Acids Research 44: e106–e106.</p>
</div>
<div>
<p>Chubb JR, Boyle S, Perry P, Bickmore WA. 2002. Chromatin Motion Is Constrained by Association with Nuclear Compartments in Human Cells. Current Biology 12: 439–445.</p>
</div>
<div>
<p>Claussnitzer M, Dankel SN, Kim K-H, Quon G, Meuleman W, Haugen C, Glunk V, Sousa IS, Beaudry JL, Puviindran V, Abdennur NA, Liu J, Svensson P-A, Hsu Y-H, Drucker DJ, Mellgren G, Hui C-C, Hauner H, Kellis M. 2015. &lt;i&gt;FTO&lt;/i&gt; Obesity Variant Circuitry and Adipocyte Browning in Humans. New England Journal of Medicine 373: 895–907.</p>
</div>
<div>
<p>Conrad DF, Andrews TD, Carter NP, Hurles ME, Pritchard JK. 2006. A high-resolution survey of deletion polymorphism in the human genome. Nature Genetics 38: 75–81.</p>
</div>
<div>
<p>Conrad DF, Pinto D, Redon R, Feuk L, Gokcumen O, Zhang Y, Aerts J, Andrews TD, Barnes C, Campbell P, Fitzgerald T, Hu M, Ihm CH, Kristiansson K, Macarthur DG, Macdonald JR, Onyiah I, Pang AWC, Robson S, Stirrups K, Valsesia A, Walter K, Wei J, Tyler-Smith C, Carter NP, Lee C, Scherer SW, Hurles ME. 2010. Origins and functional impact of copy number variation in the human genome. Nature 464: 704–12.</p>
</div>
<div>
<p>Crane E, Bian Q, McCord RP, Lajoie BR, Wheeler BS, Ralston EJ, Uzawa S, Dekker J, Meyer BJ. 2015. Condensin-driven remodelling of X chromosome topology during dosage compensation. Nature.</p>
</div>
<div>
<p>Cremer T, Cremer C. 2001. Chromosome territories, nuclear architecture and gene regulation in mammalian cells. Nature reviews genetics 2: 292–301.</p>
</div>
<div>
<p>Cremer T, Cremer M. 2010. Chromosome territories. Cold Spring Harbor perspectives in biology 2: a003889.</p>
</div>
<div>
<p>Cremer T, Cremer M, Hübner B, Strickfaden H, Smeets D, Popken J, Sterr M, Markaki Y, Rippe K, Cremer C. 2015. The 4D nucleome: Evidence for a dynamic nuclear landscape based on co-aligned active and inactive nuclear compartments. FEBS Letters.</p>
</div>
<div>
<p>Creyghton MP, Cheng AW, Welstead GG, Kooistra T, Carey BW, Steine EJ, Hanna J, Lodato MA, Frampton GM, Sharp PA, Boyer LA, Young RA, Jaenisch R. 2010. Histone H3K27ac separates active from poised enhancers and predicts developmental state. Proceedings of the National Academy of Sciences of the United States of America 107: 21931–6.</p>
</div>
<div>
<p>Criscione SW, De Cecco M, Siranosian B, Zhang Y, Kreiling JA, Sedivy JM, Neretti N. 2016. Reorganization of chromosome architecture in replicative cellular senescence. Science advances 2: e1500882.</p>
</div>
<div>
<p>Davis AP, Grondin CJ, Johnson RJ, Sciaky D, King BL, McMorran R, Wiegers J, Wiegers TC, Mattingly CJ. 2017. The Comparative Toxicogenomics Database: Update 2017. Nucleic Acids Research 45: D972–D978.</p>
</div>
<div>
<p>Dekker J. 2014. Two ways to fold the genome during the cell cycle: insights obtained with chromosome conformation capture. Epigenetics &amp; Chromatin 7: 25.</p>
</div>
<div>
<p>Dekker J, Marti-Renom MA, Mirny LA. 2013. Exploring the three-dimensional organization of genomes: interpreting chromatin interaction data. Nature reviews. Genetics 14: 390–403.</p>
</div>
<div>
<p>Dekker J, Mirny L. 2016. The 3D Genome as Moderator of Chromosomal Communication. Cell 164: 1110–1121.</p>
</div>
<div>
<p>Dekker J, Rippe K, Dekker M, Kleckner N. 2002. Capturing chromosome conformation. Science (New York, N.Y.) 295: 1306–11.</p>
</div>
<div>
<p>Deng W, Lee J, Wang H, Miller J, Reik A, Gregory PD, Dean A, Blobel G a. 2012. Controlling long-range genomic interactions at a native locus by targeted tethering of a looping factor. Cell 149: 1233–44.</p>
</div>
<div>
<p>Deng Y, Gao L, Wang B, Guo X. 2015. HPOSim: An r package for phenotypic similarity measure and enrichment analysis based on the human phenotype ontology. PLoS ONE 10: 1–12.</p>
</div>
<div>
<p>Dickel DE, Ypsilanti AR, Pla R, Zhu Y, Barozzi I, Mannion BJ, Khin YS, Fukuda-Yuzawa Y, Plajzer-Frick I, Pickle CS, Lee EA, Harrington AN, Pham QT, Garvin TH, Kato M, Osterwalder M, Akiyama JA, Afzal V, Rubenstein JLR, Pennacchio LA, Visel A. 2018. Ultraconserved Enhancers Are Required for Normal Development. Cell 172: 491–499.e15.</p>
</div>
<div>
<p>Dileep V, Ay F, Sima J, Vera DL, Noble WS, Gilbert DM. 2015. Topologically associating domains and their long-range contacts are established during early G1 coincident with the establishment of the replication-timing program. Genome research 25: 1104–13.</p>
</div>
<div>
<p>Dimitrieva S, Bucher P. 2013. UCNEbase—a database of ultraconserved non-coding elements and genomic regulatory blocks. Nucleic Acids Research 41: D101–D109.</p>
</div>
<div>
<p>Dixon JR, Gorkin DU, Ren B. 2016. Chromatin Domains: The Unit of Chromosome Organization. Molecular Cell 62: 668–680.</p>
</div>
<div>
<p>Dixon JR, Jung I, Selvaraj S, Shen Y, Antosiewicz-Bourget JE, Lee AY, Ye Z, Kim A, Rajagopal N, Xie W, Diao Y, Liang J, Zhao H, Lobanenkov VV, Ecker JR, Thomson Ja, Ren B. 2015. Chromatin architecture reorganization during stem cell differentiation. Nature 518: 331–336.</p>
</div>
<div>
<p>Dixon JR, Selvaraj S, Yue F, Kim A, Li Y, Shen Y, Hu M, Liu JS, Ren B. 2012. Topological domains in mammalian genomes identified by analysis of chromatin interactions. Nature 485: 376–380.</p>
</div>
<div>
<p>Djebali S, Davis Ca, Merkel A, Dobin A, Lassmann T, Mortazavi A, Tanzer A, Lagarde J, Lin W, Schlesinger F, Xue C, Marinov GK, Khatun J, Williams Ba, Zaleski C, Rozowsky J, Röder M, Kokocinski F, Abdelhamid RF, Alioto T, Antoshechkin I, Baer MT, Bar NS, Batut P, Bell K, Bell I, Chakrabortty S, Chen X, Chrast J, Curado J, Derrien T, Drenkow J, Dumais E, Dumais J, Duttagupta R, Falconnet E, Fastuca M, Fejes-Toth K, Ferreira P, Foissac S, Fullwood MJ, Gao H, Gonzalez D, Gordon A, Gunawardena H, Howald C, Jha S, Johnson R, Kapranov P, King B, Kingswood C, Luo OJ, Park E, Persaud K, Preall JB, Ribeca P, Risk B, Robyr D, Sammeth M, Schaffer L, See L-H, Shahab A, Skancke J, Suzuki AM, Takahashi H, Tilgner H, Trout D, Walters N, Wang H, Wrobel J, Yu Y, Ruan X, Hayashizaki Y, Harrow J, Gerstein M, Hubbard T, Reymond A, Antonarakis SE, Hannon G, Giddings MC, Ruan Y, Wold B, Carninci P, Guigó R, Gingeras TR. 2012. Landscape of transcription in human cells. Nature 489: 101–108.</p>
</div>
<div>
<p>Dostie J, Richmond TA, Arnaout RA, Selzer RR, Lee WL, Honan TA, Rubio ED, Krumm A, Lamb J, Nusbaum C, Green RD, Dekker J. 2006. Chromosome Conformation Capture Carbon Copy (5C): A massively parallel solution for mapping interactions between genomic elements. Genome Research 16: 1299–1309.</p>
</div>
<div>
<p>Dowen JM, Fan ZP, Hnisz D, Ren G, Abraham BJ, Zhang LN, Weintraub AS, Schuijers J, Lee TI, Zhao K, Young RA. 2014. Control of Cell Identity Genes Occurs in Insulated Neighborhoods in Mammalian Chromosomes. Cell 159: 374–387.</p>
</div>
<div>
<p>Dryden NH, Broome LR, Dudbridge F, Johnson N, Orr N, Schoenfelder S, Nagano T, Andrews S, Wingett S, Kozarewa I, Assiotis I, Fenwick K, Maguire SL, Campbell J, Natrajan R, Lambros M, Perrakis E, Ashworth A, Fraser P, Fletcher O. 2014. Unbiased analysis of potential targets of breast cancer susceptibility loci by Capture Hi-C. Genome research: 1854–1868.</p>
</div>
<div>
<p>Dunham I, Kundaje A, Aldred SF, Collins PJ, Davis C a, Doyle F, Epstein CB, Frietze S, Harrow J, Kaul R, Khatun J, Lajoie BR, Landt SG, Lee B-KB-K, Pauli F, Rosenbloom KR, Sabo P, Safi A, Sanyal A, Shoresh N, Simon JM, Song L, Trinklein ND, Altshuler RC, Birney E, Brown JB, Cheng C, Djebali S, Dong X, Ernst J, Furey TS, Gerstein M, Giardine B, Greven M, Hardison RC, Harris RS, Herrero J, Hoffman MM, Iyer S, Kelllis M, Kheradpour P, Lassmann T, Li Q, Lin X, Marinov GK, Merkel A, Mortazavi A, Parker SLSCJ, Reddy TE, Rozowsky J, Schlesinger F, Thurman RE, Wang J, Ward LD, Whitfield TW, Wilder SP, Wu W, Xi HS, Yip KY, Zhuang J, Bernstein BE, Green ED, Gunter C, Snyder M, Pazin MJ, Lowdon RF, Dillon L a L, Adams LB, Kelly CJ, Zhang J, Wexler JR, Good PJ, Feingold E a, Crawford GE, Dekker J, Elinitski L, Farnham PJ, Giddings MC, Gingeras TR, Guigó R, Hubbard TJTJ, Kellis M, Kent WJ, Lieb JD, Margulies EH, Myers RM, Starnatoyannopoulos J a, Tennebaum S a, Weng Z, White KP, Wold B, Yu Y, Wrobel J, Risk B a, Gunawardena HP, Kuiper HC, Maier CW, Xie L, et al. 2012. An integrated encyclopedia of DNA elements in the human genome. Nature 489: 57–74.</p>
</div>
<div>
<p>Durand NC, Shamim MS, Machol I, Rao SS, Huntley MH, Lander ES, Aiden EL. 2016. Juicer Provides a One-Click System for Analyzing Loop-Resolution Hi-C Experiments. Cell Systems 3: 95–98.</p>
</div>
<div>
<p>Durinck S, Moreau Y, Kasprzyk A, Davis S, De Moor B, Brazma A, Huber W. 2005. BioMart and Bioconductor: A powerful link between biological databases and microarray data analysis. Bioinformatics 21: 3439–3440.</p>
</div>
<div>
<p>Durinck S, Spellman PT, Birney E, Huber W. 2009a. Mapping identifiers for the integration of genomic datasets with the R/Bioconductor package biomaRt. Nature Protocols 4: 1184–1191.</p>
</div>
<div>
<p>Durinck S, Spellman PT, Birney E, Huber W. 2009b. Mapping identifiers for the integration of genomic datasets with the R/Bioconductor package biomaRt. Nature protocols 4: 1184–1191.</p>
</div>
<div>
<p>Dzida T, Iqbal M, Charapitsa I, Reid G, Stunnenberg H, Matarese F, Grote K, Honkela A, Rattray M. 2017. Predicting stimulation-dependent enhancer-promoter interactions from ChIP-Seq time course data. PeerJ 5: e3742.</p>
</div>
<div>
<p>Eagen KP, Hartl TA, Kornberg RD. 2015. Stable Chromosome Condensation Revealed by Chromosome Conformation Capture. Cell 163: 934–946.</p>
</div>
<div>
<p>Eisenberg E, Levanon EY. 2013. Human housekeeping genes, revisited. Trends in Genetics 29: 569–574.</p>
</div>
<div>
<p>Engström PG, Sui SJH, Drivenes Ø, Becker TS, Lenhard B. 2007. Genomic regulatory blocks underlie extensive microsynteny conservation in insects. Genome Research 17: 1898–1908.</p>
</div>
<div>
<p>Fantes J, Redeker B, Breen M, Boyle S, Brown J, Fletcher J, Jones S, Bickmore W, Fukushima Y, Mannens M, Danes S, Heyningen V van, Hanson I. 1995. Aniridia-associated cytogenetic rearrangements suggest that a position effect may cause the mutant phenotype. Human Molecular Genetics 4: 415–422.</p>
</div>
<div>
<p>Farré M, Robinson TJ, Ruiz-Herrera A. 2015. An Integrative Breakage Model of genome architecture, reshuffling and evolution. BioEssays: n/a–n/a.</p>
</div>
<div>
<p>Filippova D, Patro R, Duggal G, Kingsford C. 2014. Identification of alternative topological domains in chromatin. Algorithms for molecular biology : AMB 9: 14.</p>
</div>
<div>
<p>Firth HV, Richards SM, Bevan a P, Clayton S, Corpas M, Rajan D, Van Vooren S, Moreau Y, Pettett RM, Carter NP. 2009. DECIPHER: Database of Chromosomal Imbalance and Phenotype in Humans Using Ensembl Resources. American journal of human genetics 84: 524–33.</p>
</div>
<div>
<p>Fishilevich S, Nudel R, Rappaport N, Hadar R, Plaschkes I, Iny Stein T, Rosen N, Kohn A, Twik M, Safran M, Lancet D, Cohen D. 2017. GeneHancer: genome-wide integration of enhancers and target genes in GeneCards. Database 2017: 1665–1680.</p>
</div>
<div>
<p>Flavahan W a, Drier Y, Liau BB, Gillespie SM, Venteicher a S, Stemmer-Rachamimov a O, Suva ML, Bernstein BE. 2016. Insulator dysfunction and oncogene activation in IDH mutant gliomas. Nature 529: 110–114.</p>
</div>
<div>
<p>Flicek P, Amode MR, Barrell D, Beal K, Billis K, Brent S, Carvalho-Silva D, Clapham P, Coates G, Fitzgerald S, Gil L, Girón CG, Gordon L, Hourlier T, Hunt S, Johnson N, Juettemann T, Kähäri AK, Keenan S, Kulesha E, Martin FJ, Maurel T, McLaren WM, Murphy DN, Nag R, Overduin B, Pignatelli M, Pritchard B, Pritchard E, Riat HS, Ruffier M, Sheppard D, Taylor K, Thormann A, Trevanion SJ, Vullo A, Wilder SP, Wilson M, Zadissa A, Aken BL, Birney E, Cunningham F, Harrow J, Herrero J, Hubbard TJ, Kinsella R, Muffato M, Parker A, Spudich G, Yates A, Zerbino DR, Searle SM. 2014. Ensembl 2014. Nucleic Acids Research 42: D749–D755.</p>
</div>
<div>
<p>Flomen RH, Vatcheva R, Gorman PA, Baptista PR, Groet J, Barišić I, Ligutic I, Nižetić D. 1998. Construction and Analysis of a Sequence-Ready Map in 4q25: Rieger Syndrome Can Be Caused by Haploinsufficiency ofRIEG, but Also by Chromosome Breaks ≈90 kb Upstream of This Gene. Genomics 47: 409–413.</p>
</div>
<div>
<p>Flyamer IM, Gassler J, Imakaev M, Ulyanov SV, Abdennur N, Razin SV, Mirny L, Tachibana-Konwalski K. 2017. Single-cell Hi-C reveals unique chromatin reorganization at oocyte-tozygote transition. Nature Publishing Group 544: 1–17.</p>
</div>
<div>
<p>Forcato M, Nicoletti C, Pal K, Livi CM, Ferrari F, Bicciato S. 2017. Comparison of computational methods for Hi-C data analysis. Nature Methods: 14–19.</p>
</div>
<div>
<p>Ford C, Jones K, Polani P, De Almeida J, Briggs J. 1959. A SEX-CHROMOSOME ANOMALY IN A CASE OF GONADAL DYSGENESIS (TURNER’S SYNDROME). The Lancet 273: 711–713.</p>
</div>
<div>
<p>Forrest ARR, Kawaji H, Rehli M, Baillie JK, Hoon MJL de, Lassmann T, Itoh M, Summers KM, Suzuki H, Daub CO, Kawai J, Heutink P, Hide W, Freeman TC, Lenhard B, Bajic VB, Taylor MS, Makeev VJ, Sandelin A, Hume D a, Carninci P, Hayashizaki Y. 2014. A promoter-level mammalian expression atlas. Nature 507: 462–70.</p>
</div>
<div>
<p>Franke M, Ibrahim DM, Andrey G, Schwarzer W, Heinrich V, Schöpflin R, Kraft K, Kempfer R, Jerković I, Chan W-L, Spielmann M, Timmermann B, Wittler L, Kurth I, Cambiaso P, Zuffardi O, Houge G, Lambie L, Brancati F, Pombo A, Vingron M, Spitz F, Mundlos S. 2016. Formation of new chromatin domains determines pathogenicity of genomic duplications. Nature: 1–15.</p>
</div>
<div>
<p>Fraser J, Ferrai C, Chiariello AM, Schueler M, Rito T, Laudanno G, Barbieri M, Moore BL, Kraemer DC, Aitken S, Xie SQ, Morris KJ, Itoh M, Kawaji H, Jaeger I, Hayashizaki Y, Carninci P, Forrest AR, Dostie J, Pombo A, Nicodemi M. 2015. Hierarchical folding and reorganization of chromosomes are linked to transcriptional changes in cellular differentiation. Mol Syst Biol 11: 1–14.</p>
</div>
<div>
<p>Frizell ER, Sutphen1 R, Diamond Jr. FB, Sherwood M, Overhauser J. 1998. T(1;18)(q32.1;q22.1) associated with genitourinary malformations. Clinical Genetics 54: 330–333.</p>
</div>
<div>
<p>Fudenberg G, Imakaev M, Lu C, Goloborodko A, Abdennur N, Mirny LA. 2016. Formation of Chromosomal Domains by Loop Extrusion. Cell Reports 15: 2038–2049.</p>
</div>
<div>
<p>Fullwood MJ, Liu MH, Pan YF, Liu J, Xu H, Mohamed YB, Orlov YL, Velkov S, Ho A, Mei PH, Chew EGY, Huang PYH, Welboren W-J, Han Y, Ooi HS, Ariyaratne PN, Vega VB, Luo Y, Tan PY, Choy PY, Wansa KDSA, Zhao B, Lim KS, Leow SC, Yow JS, Joseph R, Li H, Desai KV, Thomsen JS, Lee YK, Karuturi RKM, Herve T, Bourque G, Stunnenberg HG, Ruan X, Cacheux-Rataboul V, Sung W-K, Liu ET, Wei C-L, Cheung E, Ruan Y. 2009. An oestrogen-receptor-<span class="math inline">\(\alpha\)</span>-bound human chromatin interactome. Nature 462: 58–64.</p>
</div>
<div>
<p>Galil Z. 1986. Efficient algorithms for finding maximum matching in graphs. ACM Computing Surveys 18: 23–38.</p>
</div>
<div>
<p>Gibcus JH, Dekker J. 2013. The hierarchy of the 3D genome. Molecular cell 49: 773–82.</p>
</div>
<div>
<p>Giorgio E, Robyr D, Spielmann M, Ferrero E, Di Gregorio E, Imperiale D, Vaula G, Stamoulis G, Santoni F, Atzori C, Gasparini L, Ferrera D, Canale C, Guipponi M, Pennacchio LA, Antonarakis SE, Brussino A, Brusco A. 2015. A large genomic deletion leads to enhancer adoption by the lamin B1 gene: a second path to autosomal dominant adult-onset demyelinating leukodystrophy (ADLD). Human Molecular Genetics 24: 3143–3154.</p>
</div>
<div>
<p>Gómez-Marín C, Tena JJ, Acemel RD, López-Mayorga M, Naranjo S, Calle-Mustienes E de la, Maeso I, Beccari L, Aneas I, Vielmas E, Bovolenta P, Nobrega Ma, Carvajal J, Gómez-Skarmeta JL. 2015. Evolutionary comparison reveals that diverging CTCF sites are signatures of ancestral topological associating domains borders. Proceedings of the National Academy of Sciences 112: 201505463.</p>
</div>
<div>
<p>Gröschel S, Sanders MA, Hoogenboezem R, Wit E de, Bouwman BAM, Erpelinck C, Velden VHJ van der, Havermans M, Avellino R, Lom K van, Rombouts EJ, Duin M van, Döhner K, Beverloo HB, Bradner JE, Döhner H, Löwenberg B, Valk PJM, Bindels EMJ, Laat W de, Delwel R. 2014. A single oncogenic enhancer rearrangement causes concomitant EVI1 and GATA2 deregulation in leukemia. Cell 157: 369–381.</p>
</div>
<div>
<p>Gu W, Zhang F, Lupski JR. 2008. Mechanisms for human genomic rearrangements. PathoGenetics 1: 4.</p>
</div>
<div>
<p>Guo Y, Xu Q, Canzio D, Shou J, Li J, Gorkin DU, Jung I, Wu H, Zhai Y, Tang Y, Lu Y, Wu Y, Jia Z, Li W, Zhang MQ, Ren B, Krainer AR, Maniatis T, Wu Q. 2015. CRISPR Inversion of CTCF Sites Alters Genome Topology and Enhancer/Promoter Function. Cell 162: 900–910.</p>
</div>
<div>
<p>Hansen P, Hecht J, Ibn-Salem J, Menkuec BS, Roskosch S, Truss M, Robinson PN. 2016. Q-nexus: a comprehensive and efficient analysis pipeline designed for ChIP-nexus. BMC Genomics 17: 873.</p>
</div>
<div>
<p>Hansen P, Hecht J, Ibrahim DM, Krannich A, Truss M, Robinson PN. 2015. Saturation analysis of ChIP-seq data for reproducible identification of binding peaks. Genome research 25: 1391–400.</p>
</div>
<div>
<p>Harmston N, Ing-Simmons E, Perry M, Barešić A, Lenhard B. 2015. GenomicInteractions: An R/Bioconductor package for manipulating and investigating chromatin interaction data. BMC Genomics 16: 963.</p>
</div>
<div>
<p>Harmston N, Ing-Simmons E, Tan G, Perry M, Merkenschlager M, Lenhard B. 2017. Topologically associating domains are ancient features that coincide with Metazoan clusters of extreme noncoding conservation. Nature Communications 8: 441.</p>
</div>
<div>
<p>Hay D, Hughes JR, Babbs C, Davies JOJ, Graham BJ, Hanssen LLP, Kassouf MT, Oudelaar AM, Sharpe JA, Suciu MC, Telenius J, Williams R, Rode C, Li P-S, Pennacchio LA, Sloane-Stanley JA, Ayyub H, Butler S, Sauka-Spengler T, Gibbons RJ, Smith AJH, Wood WG, Higgs DR. 2016. Genetic dissection of the <span class="math inline">\(\alpha\)</span>-globin super-enhancer in vivo. Nature Genetics 48: 895–903.</p>
</div>
<div>
<p>He Q, Johnston J, Zeitlinger J. 2015. ChIP-nexus enables improved detection of in vivo transcription factor binding footprints. Nature Biotechnology 33: 395–401.</p>
</div>
<div>
<p>He X, Zhang J. 2005. Gene complexity and gene duplicability. Current Biology 15: 1016–1021.</p>
</div>
<div>
<p>Heger A, Webber C, Goodson M, Ponting CP, Lunter G. 2013. GAT: a simulation framework for testing the association of genomic intervals. Bioinformatics (Oxford, England) 29: 2046–8.</p>
</div>
<div>
<p>Heger P, Marin B, Bartkuhn M, Schierenberg E, Wiehe T. 2012. The chromatin insulator CTCF and the emergence of metazoan diversity. Proceedings of the National Academy of Sciences 109: 17507–17512.</p>
</div>
<div>
<p>Heidari N, Phanstiel DHD, He C, Grubert F, Jahanbani F, Kasowski M, Zhang MQ, Snyder MP. 2014. Genome-wide map of regulatory interactions in the human genome. Genome research 24: 1905–17.</p>
</div>
<div>
<p>Herrero J, Muffato M, Beal K, Fitzgerald S, Gordon L, Pignatelli M, Vilella AJ, Searle SMJ, Amode R, Brent S, Spooner W, Kulesha E, Yates A, Flicek P. 2016. Ensembl comparative genomics resources. Database 2016: bav096.</p>
</div>
<div>
<p>Higgins AW, Alkuraya FS, Bosco AF, Brown KK, Bruns GA, Donovan DJ, Eisenman R, Fan Y, Farra CG, Ferguson HL, Gusella JF, Harris DJ, Herrick SR, Kelly C, Kim HG, Kishikawa S, Korf BR, Kulkarni S, Lally E, Leach NT, Lemyre E, Lewis J, Ligon AH, Lu W, Maas RL, MacDonald ME, Moore SD, Peters RE, Quade BJ, Quintero-Rivera F, Saadi I, Shen Y, Shendure J, Williamson RE, Morton CC. 2008. Characterization of Apparently Balanced Chromosomal Rearrangements from the Developmental Genome Anatomy Project. American Journal of Human Genetics 82: 712–722.</p>
</div>
<div>
<p>Hindorff LA, Sethupathy P, Junkins HA, Ramos EM, Mehta JP, Collins FS, Manolio TA. 2009. Potential etiologic and functional implications of genome-wide association loci for human diseases and traits. Proceedings of the National Academy of Sciences of the United States of America 106: 9362–7.</p>
</div>
<div>
<p>Hinds DA, Kloek AP, Jen M, Chen X, Frazer KA. 2006. Common deletions and SNPs are in linkage disequilibrium in the human genome. Nature Genetics 38: 82–85.</p>
</div>
<div>
<p>Hinrichs AS, Karolchik D, Baertsch R, Barber GP, Bejerano G, Clawson H, Diekhans M, Furey TS, Harte RA, Hsu F, Hillman-Jackson J, Kuhn RM, Pedersen JS, Pohl A, Raney BJ, Rosenbloom KR, Siepel A, Smith KE, Sugnet CW, Sultan-Qurraie A, Thomas DJ, Trumbower H, Weber RJ, Weirauch M, Zweig AS, Haussler D, Kent WJ. 2006. The UCSC Genome Browser Database: update 2006. Nucleic Acids Res 34: D590—–D598.</p>
</div>
<div>
<p>Hinsch H, Hannenhalli S. 2006. Recurring genomic breaks in independent lineages support genomic fragility. BMC evolutionary biology 6: 90.</p>
</div>
<div>
<p>Hnisz D, Weintraub AS, Day DS, Valton A-l, Bak RO, Li CH, Goldmann J, Lajoie BR, Fan ZP, Sigova AA, Reddy J, Borges-Rivera D, Lee TI, Jaenisch R, Porteus MH, Dekker J, Young RA. 2016. Activation of proto-oncogenes by disruption of chromosome neighborhoods. Science (New York, N.Y.) 351: 1454–8.</p>
</div>
<div>
<p>Hoffman EA, Frey BL, Smith LM, Auble DT. 2015. Formaldehyde crosslinking: a tool for the study of chromatin complexes. The Journal of biological chemistry 290: 26404–11.</p>
</div>
<div>
<p>Hou C, Li L, Qin ZS, Corces VG. 2012. Gene Density, Transcription, and Insulators Contribute to the Partition of the Drosophila Genome into Physical Domains. Molecular Cell 48: 471–484.</p>
</div>
<div>
<p>Hsieh T-HS, Weiner A, Lajoie B, Dekker J, Friedman N, Rando OJ. 2015. Mapping Nucleosome Resolution Chromosome Folding in Yeast by Micro-C. Cell: 1–12.</p>
</div>
<div>
<p>Huang N, Lee I, Marcotte EM, Hurles ME. 2010. Characterising and Predicting Haploinsufficiency in the Human Genome. PLoS Genetics 6: e1001154.</p>
</div>
<div>
<p>Huber W, Carey VJ, Gentleman R, Anders S, Carlson M, Carvalho BS, Bravo HC, Davis S, Gatto L, Girke T, Gottardo R, Hahne F, Hansen KD, Irizarry RA, Lawrence M, Love MI, MacDonald J, Obenchain V, Oleś AK, Pagès H, Reyes A, Shannon P, Smyth GK, Tenenbaum D, Waldron L, Morgan M. 2015. Orchestrating high-throughput genomic analysis with Bioconductor. Nature Methods 12: 115–121.</p>
</div>
<div>
<p>Huminiecki L. 2004. Divergence of Spatial Gene Expression Profiles Following Species-Specific Gene Duplications in Human and Mouse. Genome Research 14: 1870–1879.</p>
</div>
<div>
<p>Iafrate AJ, Feuk L, Rivera MN, Listewnik ML, Donahoe PK, Qi Y, Scherer SW, Lee C. 2004. Detection of large-scale variation in the human genome. Nature Genetics 36: 949–951.</p>
</div>
<div>
<p>Ibn-Salem J, Köhler S, Love MI, Chung H-R, Huang N, Hurles ME, Haendel M, Washington NL, Smedley D, Mungall CJ, Lewis SE, Ott C-E, Bauer S, Schofield PN, Mundlos S, Spielmann M, Robinson PN. 2014. Deletions of chromosomal regulatory boundaries are associated with congenital disease. Genome Biology 15: 423.</p>
</div>
<div>
<p>Ibn-Salem J, Muro EM, Andrade-Navarro MA. 2017. Co-regulation of paralog genes in the three-dimensional chromatin architecture. Nucleic Acids Research 45: 81–91.</p>
</div>
<div>
<p>Jacobs PA, Strong JA. 1959. A Case of Human Intersexuality Having a Possible XXY Sex-Determining Mechanism. Nature 183: 302–303.</p>
</div>
<div>
<p>Javierre BM, Burren OS, Wilder SP, Kreuzhuber R, Hill SM, Sewitz S, Cairns J, Wingett SW, Várnai C, Thiecke MJ, Burden F, Farrow S, Cutler AJ, Rehnström K, Downes K, Grassi L, Kostadima M, Freire-Pritchett P, Wang F, Stunnenberg HG, Todd JA, Zerbino DR, Stegle O, Ouwehand WH, Frontini M, Wallace C, Spivakov M, Fraser P. 2016. Lineage-Specific Genome Architecture Links Enhancers and Non-coding Disease Variants to Target Gene Promoters. Cell 167: 1369–1384.e19.</p>
</div>
<div>
<p>Jenkins EC, Curcuru-Giordano FM, Krishna SG, Cantarella J. 1975. De novo occurrence of 46,XX,t(4;13) (q31;q14) in a mentally retarded girl. Annales de genetique 18: 117–20.</p>
</div>
<div>
<p>Joshi O, Wang S-Y, Kuznetsova T, Atlasi Y, Peng T, Fabre PJ, Habibi E, Shaik J, Saeed S, Handoko L, Richmond T, Spivakov M, Burgess D, Stunnenberg HG. 2015. Dynamic Reorganization of Extremely Long-Range Promoter-Promoter Interactions between Two States of Pluripotency. Cell Stem Cell 17: 748–757.</p>
</div>
<div>
<p>Kent WJ, Baertsch R, Hinrichs A, Miller W, Haussler D. 2003. Evolution’s cauldron: duplication, deletion, and rearrangement in the mouse and human genomes. Proceedings of the National Academy of Sciences of the United States of America 100: 11484–11489.</p>
</div>
<div>
<p>Kent WJ, Sugnet CW, Furey TS, Roskin KM, Pringle TH, Zahler AM, Haussler D. 2002. The human genome browser at UCSC. Genome research 12: 996–1006.</p>
</div>
<div>
<p>Kent WJ, Zweig AS, Barber G, Hinrichs AS, Karolchik D. 2010. BigWig and BigBed: enabling browsing of large distributed datasets. Bioinformatics 26: 2204–2207.</p>
</div>
<div>
<p>Khan A, Fornes O, Stigliani A. 2018. JASPAR 2018: update of the open-access database of transcription factor binding profiles and its web framework. Nucleic Acids Research (In print): 1–7.</p>
</div>
<div>
<p>Kikuta H, Laplante M, Navratilova P, Komisarczuk AZ, Engström PG, Fredman D, Akalin A, Caccamo M, Sealy I, Howe K, Ghislain J, Pezeron G, Mourrain P, Ellingsen S, Oates AC, Thisse C, Thisse B, Foucher I, Adolf B, Geling A, Lenhard B, Becker TS. 2007. Genomic regulatory blocks encompass multiple neighboring genes and maintain conserved synteny in vertebrates. Genome Research 17: 545–555.</p>
</div>
<div>
<p>Kim TH, Abdullaev ZK, Smith AD, Ching KA, Loukinov DI, Green RD, Zhang MQ, Lobanenkov VV, Ren B. 2007. Analysis of the Vertebrate Insulator Protein CTCF-Binding Sites in the Human Genome. Cell 128: 1231–1245.</p>
</div>
<div>
<p>Kim WK, Marcotte EM. 2008. Age-dependent evolution of the yeast protein interaction network suggests a limited role of gene duplication and divergence. PLoS computational biology 4: e1000232.</p>
</div>
<div>
<p>Kleinjan DA, Seawright A, Schedl A, Quinlan RA, Danes S, Heyningen V van. 2001. Aniridia-associated translocations, DNase hypersensitivity, sequence comparison and transgenic analysis redefine the functional domain of PAX6. Human Molecular Genetics 10: 2049–2059.</p>
</div>
<div>
<p>Kleinjan DA, Van Heyningen V. 2005. Long-Range Control of Gene Expression: Emerging Mechanisms and Disruption in Disease. Am. J. Hum. Genet 76: 8–32.</p>
</div>
<div>
<p>Kleinjan D, Heyningen V van. 1998. Position effect in human genetic disease. Human Molecular Genetics 7: 1611–1618.</p>
</div>
<div>
<p>Knight Pa, Ruiz D. 2013. A fast algorithm for matrix balancing. IMA Journal of Numerical Analysis 33: 1029–1047.</p>
</div>
<div>
<p>Koonin EV. 2005. Orthologs, paralogs, and evolutionary genomics. Annual review of genetics 39: 309–338.</p>
</div>
<div>
<p>Korbel JO, Urban AE, Affourtit JP, Godwin B, Grubert F, Simons JF, Kim PM, Palejev D, Carriero NJ, Du L, Taillon BE, Chen Z, Tanzer A, Saunders ACE, Chi J, Yang F, Carter NP, Hurles ME, Weissman SM, Harkins TT, Gerstein MB, Egholm M, Snyder M. 2007. Paired-end mapping reveals extensive structural variation in the human genome. Science (New York, N.Y.) 318: 420–6.</p>
</div>
<div>
<p>Köhler S, Doelken SC, Mungall CJ, Bauer S, Firth HV, Bailleul-Forestier I, Black GCM, Brown DL, Brudno M, Campbell J, Fitzpatrick DR, Eppig JT, Jackson AP, Freson K, Girdea M, Helbig I, Hurst JA, Jähn J, Jackson LG, Kelly AM, Ledbetter DH, Mansour S, Martin CL, Moss C, Mumford A, Ouwehand WH, Park S-MM, Riggs ER, Scott RH, Sisodiya S, Van Vooren S, Wapner RJ, Wilkie AOM, Wright CF, Vulto-van Silfhout AT, Leeuw N de, De Vries BBA, Washingthon NL, Smith CL, Westerfield M, Schofield P, Ruef BJ, Gkoutos GV, Haendel M, Smedley D, Lewis SE, Robinson PN, Vooren SV, Wapner RJ, Wilkie AOM, Wright CF, Vulto-van Silfhout AT, Leeuw ND, De Vries BBA, Washingthon NL, Smith CL, Westerfield M, Schofield P, Ruef BJ, Gkoutos GV, Haendel M, Smedley D, Lewis SE, Robinson PN, J??hn J, Jackson LG, Kelly AM, Ledbetter DH, Mansour S, Martin CL, Moss C, Mumford A, Ouwehand WH, Park S-MM, Riggs ER, Scott RH, Sisodiya S, Vooren SV, Wapner RJ, Wilkie AOM, Wright CF, Vulto-van Silfhout AT, Leeuw ND, De Vries BBA, Washingthon NL, Smith CL, Westerfield M, Schofield P, Ruef BJ, Gkoutos GV, Haendel M, Smedley D, Lewis SE, Robinson PN, Jähn J, Jackson LG, Kelly AM, Ledbetter DH, et al. 2014. The Human Phenotype Ontology project: linking molecular biology and disease through phenotype data. Nucleic acids research 42: D966–74.</p>
</div>
<div>
<p>Krijger PHL, Laat W de. 2016. Regulation of disease-associated gene expression in the 3D genome. Nature Reviews Molecular Cell Biology.</p>
</div>
<div>
<p>Laat W de, Grosveld F. 2007. Inter-chromosomal gene regulation in the mammalian cell nucleus. Current opinion in genetics &amp; development 17: 456–464.</p>
</div>
<div>
<p>Lachke SA, Higgins AW, Inagaki M, Saadi I, Xi Q, Long M, Quade BJ, Talkowski ME, Gusella JF, Fujimoto A, Robinson ML, Yang Y, Duong QT, Shapira I, Motro B, Miyoshi J, Takai Y, Morton CC, Maas RL. 2012. The cell adhesion gene PVRL3 is associated with congenital ocular defects. Human Genetics 131: 235–250.</p>
</div>
<div>
<p>Lan X, Pritchard JK. 2016. Coregulation of tandem duplicate genes slows evolution of subfunctionalization in mammals. Science (New York, N.Y.) 352: 1009–13.</p>
</div>
<div>
<p>Lappalainen I, Lopez J, Skipper L, Hefferon T, Spalding JD, Garner J, Chen C, Maguire M, Corbett M, Zhou G, Paschall J, Ananiev V, Flicek P, Church DM. 2012. dbVar and DGVa: public archives for genomic structural variation. Nucleic Acids Research 41: D936–D941.</p>
</div>
<div>
<p>Lawrence M, Gentleman R, Carey V. 2009. rtracklayer: an R package for interfacing with genome browsers. Bioinformatics 25: 1841–1842.</p>
</div>
<div>
<p>Lawrence M, Huber W, Pagès H, Aboyoun P, Carlson M, Gentleman R, Morgan MT, Carey VJ. 2013. Software for computing and annotating genomic ranges. PLoS computational biology 9: e1003118.</p>
</div>
<div>
<p>Le Dily F, Bau D, Pohl a, Vicent GP, Serra F, Soronellas D, Castellano G, Wright RHG, Ballare C, Filion G, Marti-Renom Ma, Beato M. 2014. Distinct structural transitions of chromatin topological domains correlate with coordinated hormone-induced gene regulation. Genes &amp; Development 28: 2151–2162.</p>
</div>
<div>
<p>Le TBK, Imakaev MV, Mirny L a, Laub MT. 2013. High-resolution mapping of the spatial organization of a bacterial chromosome. Science (New York, N.Y.) 342: 731–4.</p>
</div>
<div>
<p>LeJeune J, Gautier M, Turpin R. 1959. Study of somatic chromosomes from 9 mongoloid children. Comptes rendus hebdomadaires des seances de l’Academie des sciences 248: 1721–2.</p>
</div>
<div>
<p>Lelli KM, Slattery M, Mann RS. 2012. Disentangling the many layers of eukaryotic transcriptional regulation. Annual review of genetics 46: 43–68.</p>
</div>
<div>
<p>Liang J, Lacroix L, Gamot A, Cuddapah S, Queille S, Lhoumaud P, Lepetit P, Martin PGP, Vogelmann J, Court F, Hennion M, Micas G, Urbach S, Bouchez O, Nöllmann M, Zhao K, Emberly E, Cuvier O. 2014. Chromatin Immunoprecipitation Indirect Peaks Highlight Long-Range Interactions of Insulator Proteins and Pol II Pausing. Molecular Cell 53: 672–681.</p>
</div>
<div>
<p>Lieberman-Aiden E, Berkum NL van, Williams L, Imakaev M, Ragoczy T, Telling A, Amit I, Lajoie BR, Sabo PJ, Dorschner MO, Sandstrom R, Bernstein B, Bender MA, Groudine M, Gnirke A, Stamatoyannopoulos J, Mirny L a, Lander ES, Dekker J. 2009. Comprehensive mapping of long-range interactions reveals folding principles of the human genome. Science (New York, N.Y.) 326: 289–93.</p>
</div>
<div>
<p>Ligon AH, Moore SD, Parisi MA, Mealiffe ME, Harris DJ, Ferguson HL, Quade BJ, Morton CC. 2005. Constitutional Rearrangement of the Architectural Factor HMGA2: A Novel Human Phenotype Including Overgrowth and Lipomas. The American Journal of Human Genetics 76: 340–348.</p>
</div>
<div>
<p>Lin YC, Benner C, Mansson R, Heinz S, Miyazaki K, Miyazaki M, Chandra V, Bossen C, Glass CK, Murre C. 2012. Global changes in the nuclear positioning of genes and intra- and interdomain genomic interactions that orchestrate B cell fate. Nature Immunology 13: 1196–1204.</p>
</div>
<div>
<p>Long HK, Prescott SL, Wysocka J. 2016. Ever-Changing Landscapes: Transcriptional Enhancers in Development and Evolution. Cell 167: 1170–1187.</p>
</div>
<div>
<p>Lonsdale J, Thomas J, Salvatore M, Phillips R, Lo E, Shad S, Hasz R, Walters G, Garcia F, Young N, Foster B, Moser M, Karasik E, Gillard B, Ramsey K, Sullivan S, Bridge J, Magazine H, Syron J, Fleming J, Siminoff L, Traino H, Mosavel M, Barker L, Jewell S, Rohrer D, Maxim D, Filkins D, Harbach P, Cortadillo E, Berghuis B, Turner L, Hudson E, Feenstra K, Sobin L, Robb J, Branton P, Korzeniewski G, Shive C, Tabor D, Qi L, Groch K, Nampally S, Buia S, Zimmerman A, Smith A, Burges R, Robinson K, Valentino K, Bradbury D, Cosentino M, Diaz-Mayoral N, Kennedy M, Engel T, Williams P, Erickson K, Ardlie K, Winckler W, Getz G, DeLuca D, MacArthur D, Kellis M, Thomson A, Young T, Gelfand E, Donovan M, Meng Y, Grant G, Mash D, Marcus Y, Basile M, Liu J, Zhu J, Tu Z, Cox NJ, Nicolae DL, Gamazon ER, Im HK, Konkashbaev A, Pritchard J, Stevens M, Flutre T, Wen X, Dermitzakis ET, Lappalainen T, Guigo R, Monlong J, Sammeth M, Koller D, Battle A, Mostafavi S, McCarthy M, Rivas M, Maller J, Rusyn I, Nobel A, Wright F, Shabalin A, et al. 2013. The Genotype-Tissue Expression (GTEx) project. Nature Genetics 45: 580–585.</p>
</div>
<div>
<p>Lu W, Quintero-Rivera F, Fan Y, Alkuraya FS, Donovan DJ, Xi Q, Turbe-Doan A, Li Q-G, Campbell CG, Shanske AL, Sherr EH, Ahmad A, Peters R, Rilliet B, Parvex P, Bassuk AG, Harris DJ, Ferguson H, Kelly C, Walsh CA, Gronostajski RM, Devriendt K, Higgins A, Ligon AH, Quade BJ, Morton CC, Gusella JF, Maas RL. 2007. NFIA Haploinsufficiency Is Associated with a CNS Malformation Syndrome and Urinary Tract Defects. PLoS Genetics 3: e80.</p>
</div>
<div>
<p>Lun ATL, Perry M, Ing-Simmons E. 2016. Infrastructure for genomic interactions : Bioconductor classes for Hi-C , ChIA-PET and related experiments [version 1; referees: 2 approved]. F1000Research 5: 1–6.</p>
</div>
<div>
<p>Lupiáñez DG, Kraft K, Heinrich V, Krawitz P, Brancati F, Klopocki E, Horn D, Kayserili H, Opitz JM, Laxova R, Santos-Simarro F, Gilbert-Dussardier B, Wittler L, Borschiwer M, Haas SA, Osterwalder M, Franke M, Timmermann B, Hecht J, Spielmann M, Visel A, Mundlos S. 2015. Disruptions of Topological Chromatin Domains Cause Pathogenic Rewiring of Gene-Enhancer Interactions. Cell 161: 1012–1025.</p>
</div>
<div>
<p>Lupski JR, Stankiewicz P. 2005. Genomic Disorders: Molecular Mechanisms for Rearrangements and Conveyed Phenotypes. PLoS Genetics 1: e49.</p>
</div>
<div>
<p>Makova KDK, Li W-HW. 2003. Divergence in the spatial pattern of gene expression between human duplicate genes. Genome research 13: 1638–1645.</p>
</div>
<div>
<p>Mathelier A, Fornes O, Arenillas DJ, Chen C-y, Denay G, Lee J, Shi W, Shyr C, Tan G, Worsley-Hunt R, Zhang AW, Parcy F, Lenhard B, Sandelin A, Wasserman WW. 2015. JASPAR 2016: a major expansion and update of the open-access database of transcription factor binding profiles. Nucleic acids research 44: gkv1176.</p>
</div>
<div>
<p>Merkenschlager M, Nora EP. 2016. CTCF and Cohesin in Genome Folding and Transcriptional Gene Regulation. Annual review of genomics and human genetics 17: 17–43.</p>
</div>
<div>
<p>Mifsud B, Tavares-Cadete F, Young AN, Sugar R, Schoenfelder S, Ferreira L, Wingett SW, Andrews S, Grey W, Ewels P a, Herman B, Happe S, Higgs A, LeProust E, Follows G a, Fraser P, Luscombe NM, Osborne CS. 2015. Mapping long-range promoter contacts in human cells with high-resolution capture Hi-C. Nature Genetics 47.</p>
</div>
<div>
<p>Miller DT, Adam MP, Aradhya S, Biesecker LG, Brothman AR, Carter NP, Church DM, Crolla JA, Eichler EE, Epstein CJ, Faucett WA, Feuk L, Friedman JM, Hamosh A, Jackson L, Kaminsky EB, Kok K, Krantz ID, Kuhn RM, Lee C, Ostell JM, Rosenberg C, Scherer SW, Spinner NB, Stavropoulos DJ, Tepperberg JH, Thorland EC, Vermeesch JR, Waggoner DJ, Watson MS, Martin CL, Ledbetter DH. 2010. Consensus Statement: Chromosomal Microarray Is a First-Tier Clinical Diagnostic Test for Individuals with Developmental Disabilities or Congenital Anomalies. The American Journal of Human Genetics 86: 749–764.</p>
</div>
<div>
<p>Mills RE, Bennett EA, Iskow RC, Luttig CT, Tsui C, Pittard WS, Devine SE. 2006. Recently Mobilized Transposons in the Human and Chimpanzee Genomes. The American Journal of Human Genetics 78: 671–679.</p>
</div>
<div>
<p>Mizuguchi T, Fudenberg G, Mehta S, Belton J-M, Taneja N, Folco HD, FitzGerald P, Dekker J, Mirny L, Barrowman J, Grewal SIS. 2014. Cohesin-dependent globules and heterochromatin shape 3D genome architecture in S. pombe. Nature.</p>
</div>
<div>
<p>Monahan K, Lomvardas S. 2015. Monoallelic Expression of Olfactory Receptors. Annual Review of Cell and Developmental Biology 31: annurev–cellbio–100814–125308.</p>
</div>
<div>
<p>Montavon T, Thevenet L, Duboule D. 2012. Impact of copy number variations (CNVs) on long-range gene regulation at the HoxD locus. Proceedings of the National Academy of Sciences of the United States of America 109: 20204–11.</p>
</div>
<div>
<p>Mora A, Sandve GK, Gabrielsen OS, Eskeland R. 2015. In the loop: promoter-enhancer interactions and bioinformatics. Briefings in bioinformatics: 1–16.</p>
</div>
<div>
<p>Mourad R, Li L, Cuvier O. 2017. Uncovering direct and indirect molecular determinants of chromatin loops using a computational integrative approach. PLOS Computational Biology 13: e1005538.</p>
</div>
<div>
<p>Muro EM, Andrade-Navarro MA. 2010. Pseudogenes as an alternative source of natural antisense transcripts. BMC Evolutionary Biology 10: 338.</p>
</div>
<div>
<p>Murphy WJ, Larkin DM, Everts-van der Wind A, Bourque G, Tesler G, Auvil L, Beever JE, Chowdhary BP, Galibert F, Gatzke L, Hitte C, Meyers SN, Milan D, Ostrander EA, Pape G, Parker HG, Raudsepp T, Rogatcheva MB, Schook LB, Skow LC, Welge M, Womack JE, O’brien SJ, Pevzner PA, Lewin HA. 2005. Dynamics of mammalian chromosome evolution inferred from multispecies comparative maps. Science (New York, N.Y.) 309: 613–7.</p>
</div>
<div>
<p>Nagano T, Lubling Y, Várnai C, Dudley C, Leung W, Baran Y, Mendelson-cohen N. 2017. Cell-cycle dynamics of chromosomal organization at single-cell resolution. Nature Publishing Group 547: 61–67.</p>
</div>
<div>
<p>Nagy G, Czipa E, Steiner L, Nagy T, Pongor S, Nagy L, Barta E. 2016. Motif oriented high-resolution analysis of ChIP-seq data reveals the topological order of CTCF and cohesin proteins on DNA. BMC Genomics 17: 637.</p>
</div>
<div>
<p>Nambiar M, Raghavan SC. 2011. How does DNA break during chromosomal translocations? Nucleic Acids Research 39: 5813–5825.</p>
</div>
<div>
<p>Narendra V, Rocha PP, An D, Raviram R, Skok JA, Mazzoni EO, Reinberg D. 2015. Transcription. CTCF establishes discrete functional chromatin domains at the Hox clusters during differentiation. Science (New York, N.Y.) 347: 1017–21.</p>
</div>
<div>
<p>Naumova N, Imakaev M, Fudenberg G, Zhan Y, Lajoie BR, Mirny L a, Dekker J. 2013. Organization of the mitotic chromosome. Science (New York, N.Y.) 342: 948–53.</p>
</div>
<div>
<p>Naville M, Ishibashi M, Ferg M, Bengani H, Rinkwitz S, Krecsmarik M, Hawkins Ta, Wilson SW, Manning E, Chilamakuri CSR, Wilson DI, Louis A, Lucy Raymond F, Rastegar S, Strähle U, Lenhard B, Bally-Cuif L, Heyningen V van, FitzPatrick DR, Becker TS, Roest Crollius H. 2015. Long-range evolutionary constraints reveal cis-regulatory interactions on the human X chromosome. Nature Communications 6: 6904.</p>
</div>
<div>
<p>Newman S, Hermetz KE, Weckselblatt B, Rudd MK. 2015. Next-Generation Sequencing of Duplication CNVs Reveals that Most Are Tandem and Some Create Fusion Genes at Breakpoints. The American Journal of Human Genetics 96: 1–13.</p>
</div>
<div>
<p>Nikumbh S, Pfeifer N. 2017. Genetic sequence-based prediction of long-range chromatin interactions suggests a potential role of short tandem repeat sequences in genome organization. BMC Bioinformatics 18: 218.</p>
</div>
<div>
<p>Noordermeer D, Wit E de, Klous P, Werken H van de, Simonis M, Lopez-Jones M, Eussen B, Klein A de, Singer RH, Laat W de. 2011. Variegated gene expression caused by cell-specific long-range DNA interactions. Nature Cell Biology 13: 944–951.</p>
</div>
<div>
<p>Nora EP, Dekker J, Heard E. 2013. Segmental folding of chromosomes: A basis for structural and regulatory chromosomal neighborhoods? BioEssays 35: 818–828.</p>
</div>
<div>
<p>Nora EP, Lajoie BR, Schulz EG, Giorgetti L, Okamoto I, Servant N, Piolot T, Berkum NL van, Meisig J, Sedat J, Gribnau J, Barillot E, Blüthgen N, Dekker J, Heard E. 2012. Spatial partitioning of the regulatory landscape of the X-inactivation centre. Nature 485: 381–5.</p>
</div>
<div>
<p>Northcott Pa, Lee C, Zichner T, Stütz AM, Erkek S, Kawauchi D, Shih DJH, Hovestadt V, Zapatka M, Sturm D, Jones DTW, Kool M, Remke M, Cavalli FMG, Zuyderduyn S, Bader GD, VandenBerg S, Esparza LA, Ryzhova M, Wang W, Wittmann A, Stark S, Sieber L, Seker-Cin H, Linke L, Kratochwil F, Jäger N, Buchhalter I, Imbusch CD, Zipprich G, Raeder B, Schmidt S, Diessl N, Wolf S, Wiemann S, Brors B, Lawerenz C, Eils J, Warnatz H-J, Risch T, Yaspo M-L, Weber UD, Bartholomae CC, Kalle C von, Turányi E, Hauser P, Sanden E, Darabi A, Siesjö P, Sterba J, Zitterbart K, Sumerauer D, Sluis P van, Versteeg R, Volckmann R, Koster J, Schuhmann MU, Ebinger M, Grimes HL, Robinson GW, Gajjar A, Mynarek M, Hoff K von, Rutkowski S, Pietsch T, Scheurlen W, Felsberg J, Reifenberger G, Kulozik AE, Deimling A von, Witt O, Eils R, Gilbertson RJ, Korshunov A, Taylor MD, Lichter P, Korbel JO, Wechsler-Reya RJ, Pfister SM. 2014. Enhancer hijacking activates GFI1 family oncogenes in medulloblastoma. Nature.</p>
</div>
<div>
<p>Oldridge DA, Wood AC, Weichert-Leahey N, Crimmins I, Sussman R, Winter C, McDaniel LD, Diamond M, Hart LS, Zhu S, Durbin AD, Abraham BJ, Anders L, Tian L, Zhang S, Wei JS, Khan J, Bramlett K, Rahman N, Capasso M, Iolascon A, Gerhard DS, Guidry Auvil JM, Young RA, Hakonarson H, Diskin SJ, Thomas Look A, Maris JM. 2015. Genetic predisposition to neuroblastoma mediated by a LMO1 super-enhancer polymorphism. Nature 528: 418–421.</p>
</div>
<div>
<p>Ordulu Z, Kammin T, Brand H, Pillalamarri V, Redin CE, Collins RL, Blumenthal I, Hanscom C, Pereira S, Crandall BF, Gerrol P, Hayden MA, Hussain N, Kanengisser-pines B, Kantarci S, Levy B, Macera MJ, Quintero-rivera F, Spiegel E, Stevens B, Ulm JE, Warburton D, Wilkins-haug LE, Yachelevich N, Gusella JF, Talkowski ME. 2016. Structural Chromosomal Rearrangements Require Nucleotide-Level Resolution : Lessons from Next-Generation Sequencing in Prenatal Diagnosis. The American Journal of Human Genetics: 1–19.</p>
</div>
<div>
<p>Ordulu Z, Wong KE, Currall BB, Ivanov AR, Pereira S, Althari S, Gusella JF, Talkowski ME, Morton CC. 2014. Describing Sequencing Results of Structural Chromosome Rearrangements with a Suggested Next-Generation Cytogenetic Nomenclature. The American Journal of Human Genetics 94: 695–709.</p>
</div>
<div>
<p>Orlando V, Strutt H, Paro R. 1997. Analysis of Chromatin Structure byin VivoFormaldehyde Cross-Linking. Methods 11: 205–214.</p>
</div>
<div>
<p>Oti M, Falck J, Huynen MA, Zhou H. 2016. CTCF-mediated chromatin loops enclose inducible gene regulatory domains. BMC Genomics 17: 252.</p>
</div>
<div>
<p>O’Connor TR, Bailey TL. 2014. Creating and validating cis-regulatory maps of tissue-specific gene expression regulation. Nucleic acids research: 1–11.</p>
</div>
<div>
<p>Pachlopnik Schmid J, Lemoine R, Nehme N, Cormier-Daire V, Revy P, Debeurme F, Debré M, Nitschke P, Bole-Feysot C, Legeai-Mallet L, Lim A, Villartay J-P de, Picard C, Durandy A, Fischer A, de Saint Basile G. 2012. Polymerase <span class="math inline">\(\epsilon\)</span>1 mutation in a human syndrome with facial dysmorphism, immunodeficiency, livedo, and short stature (&quot;FILS syndrome&quot;). The Journal of experimental medicine 209: 2323–30.</p>
</div>
<div>
<p>Parker SCJ, Stitzel ML, Taylor DL, Orozco JM, Erdos MR, Akiyama JA, Bueren KL van, Chines PS, Narisu N, NISC Comparative Sequencing Program NCS, Black BL, Visel A, Pennacchio LA, Collins FS, National Institutes of Health Intramural Sequencing Center Comparative Sequencing Program Authors, NISC Comparative Sequencing Program Authors. 2013. Chromatin stretch enhancer states drive cell-specific gene regulation and harbor human disease risk variants. Proceedings of the National Academy of Sciences of the United States of America 110: 17921–6.</p>
</div>
<div>
<p>Petryszak R, Keays M, Tang YA, Fonseca NA, Barrera E, Burdett T, Füllgrabe A, Fuentes AM-P, Jupp S, Koskinen S, Mannion O, Huerta L, Megy K, Snow C, Williams E, Barzine M, Hastings E, Weisser H, Wright J, Jaiswal P, Huber W, Choudhary J, Parkinson HE, Brazma A. 2015. Expression Atlas update—an integrated database of gene and protein expression in humans, animals and plants. Nucleic Acids Research 44: gkv1045.</p>
</div>
<div>
<p>Pevzner P, Tesler G. 2003. Human and mouse genomic sequences reveal extensive breakpoint reuse in mammalian evolution. Proceedings of the National Academy of Sciences of the United States of America 100: 7672–7.</p>
</div>
<div>
<p>Phillips-Cremins JE, Sauria MEG, Sanyal A, Gerasimova TI, Lajoie BR, Bell JSK, Ong C-T, Hookway T a, Guo C, Sun Y, Bland MJ, Wagstaff W, Dalton S, McDevitt TC, Sen R, Dekker J, Taylor J, Corces VG. 2013. Architectural protein subclasses shape 3D organization of genomes during lineage commitment. Cell 153: 1281–95.</p>
</div>
<div>
<p>Pink RC, Wicks K, Caley DP, Punch EK, Jacobs L, Carter DRF. 2011. Pseudogenes: pseudo-functional or key regulators in health and disease? RNA (New York, N.Y.) 17: 792–8.</p>
</div>
<div>
<p>Pique-Regi R, Degner JF, Pai A a, Gaffney DJ, Gilad Y, Pritchard JK. 2011. Accurate inference of transcription factor binding from DNA sequence and chromatin accessibility data. Genome research 21: 447–55.</p>
</div>
<div>
<p>Polychronopoulos D, King JWD, Nash AJ, Tan G, Lenhard B. 2017. Conserved non-coding elements: developmental gene regulation meets genome organization. Nucleic Acids Research: 1–14.</p>
</div>
<div>
<p>Pombo A, Dillon N. 2015. Three-dimensional genome architecture: players and mechanisms. Nature Reviews Molecular Cell Biology 12.</p>
</div>
<div>
<p>Pope BD, Ryba T, Dileep V, Yue F, Wu W, Denas O, Vera DL, Wang Y, Hansen RS, Canfield TK, Thurman RE, Cheng Y, Gülsoy G, Dennis JH, Snyder MP, Stamatoyannopoulos Ja, Taylor J, Hardison RC, Kahveci T, Ren B, Gilbert DM. 2014. Topologically associating domains are stable units of replication-timing regulation. Nature 515: 402–405.</p>
</div>
<div>
<p>Ptashne M. 1986. Gene regulation by proteins acting nearby and at a distance. Nature 322: 697–701.</p>
</div>
<div>
<p>Purmann A, Toedling J, Schueler M, Carninci P, Lehrach H, Hayashizaki Y, Huber W, Sperling S. 2007. Genomic organization of transcriptomes in mammals: Coregulation and cofunctionality. Genomics 89: 580–587.</p>
</div>
<div>
<p>Quinlan AR, Hall IM. 2010. BEDTools: a flexible suite of utilities for comparing genomic features. Bioinformatics 26: 841–842.</p>
</div>
<div>
<p>Quinn JJ, Chang HY. 2016. Unique features of long non-coding RNA biogenesis and function. Nature Reviews Genetics 17: 47–62.</p>
</div>
<div>
<p>Rao SSP, Huntley MH, Durand NC, Stamenova EK, Bochkov ID, Robinson JT, Sanborn AL, Machol I, Omer AD, Lander ES, Aiden EL. 2014. A 3D map of the human genome at kilobase resolution reveals principles of chromatin looping. Cell 159: 1665–80.</p>
</div>
<div>
<p>Redin C, Brand H, Collins RL, Kammin T, Mitchell E, Hodge JC, Hanscom C, Pillalamarri V, Seabra CM, Abbott M-A, Abdul-Rahman OA, Aberg E, Adley R, Alcaraz-Estrada SL, Alkuraya FS, An Y, Anderson M-A, Antolik C, Anyane-Yeboa K, Atkin JF, Bartell T, Bernstein JA, Beyer E, Blumenthal I, Bongers EMHF, Brilstra EH, Brown CW, Brüggenwirth HT, Callewaert B, Chiang C, Corning K, Cox H, Cuppen E, Currall BB, Cushing T, David D, Deardorff MA, Dheedene A, D’Hooghe M, Vries BBA de, Earl DL, Ferguson HL, Fisher H, FitzPatrick DR, Gerrol P, Giachino D, Glessner JT, Gliem T, Grady M, Graham BH, Griffis C, Gripp KW, Gropman AL, Hanson-Kahn A, Harris DJ, Hayden MA, Hill R, Hochstenbach R, Hoffman JD, Hopkin RJ, Hubshman MW, Innes AM, Irons M, Irving M, Jacobsen JC, Janssens S, Jewett T, Johnson JP, Jongmans MC, Kahler SG, Koolen DA, Korzelius J, Kroisel PM, Lacassie Y, Lawless W, Lemyre E, Leppig K, Levin AV, Li H, Li H, Liao EC, Lim C, Lose EJ, Lucente D, Macera MJ, Manavalan P, Mandrile G, Marcelis CL, Margolin L, Mason T, Masser-Frye D, McClellan MW, Mendoza CJZ, Menten B, Middelkamp S, Mikami LR, Moe E, Mohammed S, et al. 2017. The genomic landscape of balanced cytogenetic abnormalities associated with human congenital anomalies. Nature Genetics 49: 36–45.</p>
</div>
<div>
<p>Rehm HL, Berg JS, Brooks LD, Bustamante CD, Evans JP, Landrum MJ, Ledbetter DH, Maglott DR, Martin CL, Nussbaum RL, Plon SE, Ramos EM, Sherry ST, Watson MS. 2015. ClinGen — The Clinical Genome Resource. New England Journal of Medicine 372: 2235–2242.</p>
</div>
<div>
<p>Rennie S, Dalby M, Duin L van, Andersson R. 2018. Transcriptional decomposition reveals active chromatin architectures and cell specific regulatory interactions. Nature Communications 9: 487.</p>
</div>
<div>
<p>Rhee HS, Pugh BF. 2011. Comprehensive genome-wide protein-DNA interactions detected at single-nucleotide resolution. Cell 147: 1408–19.</p>
</div>
<div>
<p>Roadmap Epigenomics Consortium, Kundaje A, Meuleman W, Ernst J, Bilenky M, Yen A, Heravi-Moussavi A, Kheradpour P, Zhang Z, Wang J, Ziller MJ, Amin V, Whitaker JW, Schultz MD, Ward LD, Sarkar A, Quon G, Sandstrom RS, Eaton ML, Wu Y-C, Pfenning AR, Wang X, Claussnitzer M, Liu Y, Coarfa C, Harris RA, Shoresh N, Epstein CB, Gjoneska E, Leung D, Xie W, Hawkins RD, Lister R, Hong C, Gascard P, Mungall AJ, Moore R, Chuah E, Tam A, Canfield TK, Hansen RS, Kaul R, Sabo PJ, Bansal MS, Carles A, Dixon JR, Farh K-H, Feizi S, Karlic R, Kim A-R, Kulkarni A, Li D, Lowdon R, Elliott G, Mercer TR, Neph SJ, Onuchic V, Polak P, Rajagopal N, Ray P, Sallari RC, Siebenthall KT, Sinnott-Armstrong Na, Stevens M, Thurman RE, Wu J, Zhang B, Zhou X, Beaudet AE, Boyer La, De Jager PL, Farnham PJ, Fisher SJ, Haussler D, Jones SJM, Li W, Marra Ma, McManus MT, Sunyaev S, Thomson Ja, Tlsty TD, Tsai L-H, Wang W, Waterland Ra, Zhang MQ, Chadwick LH, Bernstein BE, Costello JF, Ecker JR, Hirst M, Meissner A, Milosavljevic A, Ren B, Stamatoyannopoulos Ja, Wang T, Kellis M. 2015. Integrative analysis of 111 reference human epigenomes. Nature 518: 317–30.</p>
</div>
<div>
<p>Rogozin IB, Managadze D, Shabalina SA, Koonin EV. 2014. Gene family level comparative analysis of gene expression n mammals validates the ortholog conjecture. Genome Biology and Evolution 6: 754–762.</p>
</div>
<div>
<p>Roukos V, Misteli T. 2014. The biogenesis of chromosome translocations. Nature cell biology 16: 293–300.</p>
</div>
<div>
<p>Roukos V, Voss TC, Schmidt CK, Lee S, Wangsa D, Misteli T. 2013. Spatial dynamics of chromosome translocations in living cells. Science 341: 660–664.</p>
</div>
<div>
<p>Roussos P, Mitchell AC, Voloudakis G, Fullard JF, Pothula VM, Tsang J, Stahl EA, Georgakopoulos A, Ruderfer DM, Charney A, Okada Y, Siminovitch KA, Worthington J, Padyukov L, Klareskog L, Gregersen PK, Plenge RM, Raychaudhuri S, Fromer M, Purcell SM, Brennand KJ, Robakis NK, Schadt EE, Akbarian S, Sklar P. 2014. A role for noncoding variation in schizophrenia. Cell reports 9: 1417–29.</p>
</div>
<div>
<p>Rowley MJ, Nichols MH, Lyu X, Ando-Kuri M, Rivera ISM, Hermetz K, Wang P, Ruan Y, Corces VG. 2017. Evolutionarily Conserved Principles Predict 3D Chromatin Organization. Molecular Cell: 1–16.</p>
</div>
<div>
<p>Roy S, Siahpirani AF, Chasman D, Knaack S, Ay F, Stewart R, Wilson M, Sridharan R. 2015. A predictive modeling approach for cell line-specific long-range regulatory interactions. Nucleic Acids Research 43: 8694–8712.</p>
</div>
<div>
<p>Ruiz-Velasco M, Zaugg JB. 2017. Structure meets function: How chromatin organisation conveys functionality. Current Opinion in Systems Biology 1: 129–136.</p>
</div>
<div>
<p>Rutledge MT, Russo M, Belton J-M, Dekker J, Broach JR. 2015. The yeast genome undergoes significant topological reorganization in quiescence. Nucleic Acids Research: gkv723.</p>
</div>
<div>
<p>Saito T, Rehmsmeier M. 2016. Precrec: Fast and accurate precision-recall and ROC curve calculations in R. Bioinformatics 33: btw570.</p>
</div>
<div>
<p>Saito T, Rehmsmeier M. 2015. The precision-recall plot is more informative than the ROC plot when evaluating binary classifiers on imbalanced datasets. PLoS ONE 10: 1–21.</p>
</div>
<div>
<p>Sanborn AL, Rao SSP, Huang S-C, Durand NC, Huntley MH, Jewett AI, Bochkov ID, Chinnappan D, Cutkosky A, Li J, Geeting KP, Gnirke A, Melnikov A, McKenna D, Stamenova EK, Lander ES, Aiden EL. 2015. Chromatin extrusion explains key features of loop and domain formation in wild-type and engineered genomes. Proceedings of the National Academy of Sciences 112: 201518552.</p>
</div>
<div>
<p>Sanyal A, Lajoie BR, Jain G, Dekker J. 2012. The long-range interaction landscape of gene promoters. Nature 489: 109–113.</p>
</div>
<div>
<p>Sati S, Cavalli G. 2017. Chromosome conformation capture technologies and their impact in understanding genome function. Chromosoma 126: 33–44.</p>
</div>
<div>
<p>Sati S, Cavalli G. 2016. Chromosome conformation capture technologies and their impact in understanding genome function. Chromosoma.</p>
</div>
<div>
<p>Schaefer MH, Fontaine J-F, Vinayagam A, Porras P, Wanker EE, Andrade-Navarro M a. 2012. HIPPIE: Integrating protein interaction networks with experiment based quality scores. PloS one 7: e31826.</p>
</div>
<div>
<p>Schmitt AD, Hu M, Ren B. 2016. Genome-wide mapping and analysis of chromosome architecture. Nature Reviews Molecular Cell Biology.</p>
</div>
<div>
<p>Schoenfelder S, Furlan-magaril M, Mifsud B, Tavares-cadete F, Sugar R, Javierre B-m, Nagano T, Katsman Y, Sakthidevi M, Wingett SW, Dimitrova E, Dimond A, Edelman LB, Elderkin S, Tabbada K, Darbo E, Andrews S, Herman B, Higgs A, Leproust E, Osborne CS, Mitchell JA, Luscombe NM, Fraser P. 2015. The pluripotent regulatory circuitry connecting promoters to their long-range interacting elements. Genome Research: 1–16.</p>
</div>
<div>
<p>Schreiber J, Libbrecht M, Bilmes J, Noble W. 2017. Nucleotide sequence and DNaseI sensitivity are predictive of 3D chromatin architecture. bioRxiv: 1–15.</p>
</div>
<div>
<p>Sebat J, Lakshmi B, Troge J, Alexander J, Young J, Lundin P, Manér S, Massa H, Walker M, Chi M, Navin N, Lucito R, Healy J, Hicks J, Ye K, Reiner A, Gilliam TC, Trask B, Patterson N, Zetterberg A, Wigler M. 2004. Large-Scale Copy Number Polymorphism in the Human Genome. Science 305: 525–528.</p>
</div>
<div>
<p>Sekelja M, Paulsen J, Collas P. 2016. 4D nucleomes in single cells: what can computational modeling reveal about spatial chromatin conformation? Genome Biology 17: 54.</p>
</div>
<div>
<p>Sexton T, Cavalli G. 2015. The Role of Chromosome Domains in Shaping the Functional Genome. Cell 160: 1049–1059.</p>
</div>
<div>
<p>Sexton T, Yaffe E, Kenigsberg E, Bantignies F, Leblanc B, Hoichman M, Parrinello H, Tanay A, Cavalli G. 2012. Three-dimensional folding and functional organization principles of the Drosophila genome. Cell 148: 458–72.</p>
</div>
<div>
<p>Sheffield NC, Thurman RE, Song L, Safi A, Stamatoyannopoulos J a, Lenhard B, Crawford GE, Furey TS. 2013. Patterns of regulatory activity across diverse human cell types predict tissue identity , transcription factor binding , and long-range interactions predict tissue identity , transcription factor binding , and long-range interactions. Genome Research 23: 777–788.</p>
</div>
<div>
<p>Shen Y, Yue F, McCleary DF, Ye Z, Edsall L, Kuan S, Wagner U, Dixon J, Lee L, Lobanenkov VV, Ren B. 2012. A map of the cis-regulatory sequences in the mouse genome. Nature 488: 116–20.</p>
</div>
<div>
<p>Shlyueva D, Stampfel G, Stark A. 2014. Transcriptional enhancers: from properties to genome-wide predictions. Nature Reviews Genetics 15: 272–86.</p>
</div>
<div>
<p>Simonis M, Klous P, Splinter E, Moshkin Y, Willemsen R, Wit E de, Steensel B van, Laat W de. 2006. Nuclear organization of active and inactive chromatin domains uncovered by chromosome conformation capture–on-chip (4C). Nature Genetics 38: 1348–1354.</p>
</div>
<div>
<p>Sing T, Sander O, Beerenwinkel N, Lengauer T. 2005. ROCR: visualizing classifier performance in R. Bioinformatics 21: 3940–3941.</p>
</div>
<div>
<p>Smemo S, Tena JJ, Kim K-H, Gamazon ER, Sakabe NJ, Gómez-Marín C, Aneas I, Credidio FL, Sobreira DR, Wasserman NF, Lee JH, Puviindran V, Tam D, Shen M, Son JE, Vakili NA, Sung H-K, Naranjo S, Acemel RD, Manzanares M, Nagy A, Cox NJ, Hui C-C, Gomez-Skarmeta JL, Nóbrega MA. 2014. Obesity-associated variants within FTO form long-range functional connections with IRX3. Nature 507: 371–375.</p>
</div>
<div>
<p>Solovei I, Wang AS, Thanisch K, Schmidt CS, Krebs S, Zwerger M, Cohen TV, Devys D, Foisner R, Peichl L, Herrmann H, Blum H, Engelkamp D, Stewart CL, Leonhardt H, Joffe B. 2013. LBR and lamin A/C sequentially tether peripheral heterochromatin and inversely regulate differentiation. Cell 152: 584–98.</p>
</div>
<div>
<p>Song L, Crawford GE. 2010. DNase-seq: a high-resolution technique for mapping active gene regulatory elements across the genome from mammalian cells. Cold Spring Harbor protocols 2010: pdb.prot5384.</p>
</div>
<div>
<p>Spielmann M, Brancati F, Krawitz PM, Robinson PN, Ibrahim DM, Franke M, Hecht J, Lohan S, Dathe K, Nardone AM, Ferrari P, Landi A, Wittler L, Timmermann B, Chan D, Mennen U, Klopocki E, Mundlos S. 2012. Homeotic arm-to-leg transformation associated with genomic rearrangements at the PITX1 locus. American journal of human genetics 91: 629–35.</p>
</div>
<div>
<p>Spielmann M, Mundlos S. 2016. Looking beyond the genes: the role of non-coding variants in human disease. Human Molecular Genetics: ddw205.</p>
</div>
<div>
<p>Spitz F, Furlong EEM. 2012. Transcription factors: from enhancer binding to developmental control. Nature reviews. Genetics 13: 613–26.</p>
</div>
<div>
<p>Sproul D, Gilbert N, Bickmore W a. 2005. The role of chromatin structure in regulating the expression of clustered genes. Nature reviews. Genetics 6: 775–781.</p>
</div>
<div>
<p>Stankiewicz P, Lupski JR. 2002. Genome architecture, rearrangements and genomic disorders. Trends in Genetics 18: 74–82.</p>
</div>
<div>
<p>Stankiewicz P, Lupski JR. 2010. Structural Variation in the Human Genome and its Role in Disease. Annual Review of Medicine 61: 437–455.</p>
</div>
<div>
<p>Starick SR, Ibn-Salem J, Jurk M, Hernandez C, Love MI, Chung H-R, Vingron M, Thomas-Chollier M, Meijsing SH, Hernandez C, Love MI, Chung H-R, Vingron M, Thomas-Chollier M, Meijsing SH. 2015. ChIP-exo signal associated with DNA-binding motifs provide insights into the genomic binding of the glucocorticoid receptor and cooperating transcription factors. Genome research 25: 825–35.</p>
</div>
<div>
<p>Stevens TJ, Lando D, Basu S, Liam P, Cao Y, Lee SF, Leeb M, Wohlfahrt KJ, Boucher W, Shaughnessy-kirwan AO, Cramard J, Faure AJ, Ralser M, Blanco E, Morey L, Sansó M, Palayret MGS, Lehner B, Croce LD, Wutz A, Hendrich B. 2017. 3D structures of individual mammalian genomes studied by single-cell Hi-C. Nature: 1–21.</p>
</div>
<div>
<p>Symmons O, Uslu VV, Tsujimura T, Ruf S, Nassari S, Schwarzer W, Ettwiller L, Spitz F. 2014. Functional and topological characteristics of mammalian regulatory domains. Genome research 24: 390–400.</p>
</div>
<div>
<p>Talkowski ME, Ernst C, Heilbut A, Chiang C, Hanscom C, Lindgren A, Kirby A, Liu S, Muddukrishna B, Ohsumi TK, Shen Y, Borowsky M, Daly MJ, Morton CC, Gusella JF. 2011. Next-Generation Sequencing Strategies Enable Routine Detection of Balanced Chromosome Rearrangements for Clinical Diagnostics and Genetic Research. The American Journal of Human Genetics 88: 469–481.</p>
</div>
<div>
<p>Tang Z, Berlin DS, Toji L, Toruner GA, Beiswanger C, Kulkarni S, Martin CL, Emanuel BS, Christman M, Gerry NP. 2013. A dynamic database of microarray-characterized cell lines with various cytogenetic and genomic backgrounds. G3 (Bethesda, Md.) 3: 1143–9.</p>
</div>
<div>
<p>Tang Z, Luo OJ, Li X, Zheng M, Zhu JJ, Szalaj P, Trzaskoma P, Magalska A, Wlodarczyk J, Ruszczycki B, Michalski P, Piecuch E, Wang P, Wang D, Tian SZ, Penrad-mobayed M, Sachs LM, Ruan X, Wei C-l, Liu ET, Wilczynski GM, Plewczynski D, Li G, Ruan Y. 2015. CTCF-Mediated Human 3D Genome Architecture Reveals Chromatin Topology for Transcription. Cell: 1–17.</p>
</div>
<div>
<p>Theisen A, Shaffer LG. 2010. Disorders caused by chromosome abnormalities. The application of clinical genetics 3: 159–74.</p>
</div>
<div>
<p>Thurman RE, Rynes E, Humbert R, Vierstra J, Maurano MT, Haugen E, Sheffield NC, Stergachis AB, Wang H, Vernot B, Garg K, John S, Sandstrom R, Bates D, Boatman L, Canfield TK, Diegel M, Dunn D, Ebersol AK, Frum T, Giste E, Johnson AK, Johnson EM, Kutyavin T, Lajoie B, Lee B-K, Lee K, London D, Lotakis D, Neph S, Neri F, Nguyen ED, Qu H, Reynolds AP, Roach V, Safi A, Sanchez ME, Sanyal A, Shafer A, Simon JM, Song L, Vong S, Weaver M, Yan Y, Zhang Z, Zhang Z, Lenhard B, Tewari M, Dorschner MO, Hansen RS, Navas P a, Stamatoyannopoulos G, Iyer VR, Lieb JD, Sunyaev SR, Akey JM, Sabo PJ, Kaul R, Furey TS, Dekker J, Crawford GE, Stamatoyannopoulos J a. 2012. The accessible chromatin landscape of the human genome. Nature 489: 75–82.</p>
</div>
<div>
<p>Tiwari VK, Cope L, McGarvey KM, Ohm JE, Baylin SB. 2008. A novel 6C assay uncovers Polycomb-mediated higher order chromatin conformations. Genome research 18: 1171–9.</p>
</div>
<div>
<p>Tolhuis B, Palstra RJ, Splinter E, Grosveld F, Laat W de. 2002. Looping and interaction between hypersensitive sites in the active beta-globin locus. Mol Cell 10: 1453–1465.</p>
</div>
<div>
<p>Trembath DG, Semina EV, Jones DH, Patil SR, Qian Q, Amendt BA, Russo AF, Murray JC. 2004. Analysis of two translocation breakpoints and identification of a negative regulatory element in patients with Rieger’s syndrome. Birth Defects Research Part A: Clinical and Molecular Teratology 70: 82–91.</p>
</div>
<div>
<p>Velagaleti GV, Bien-Willner GA, Northup JK, Lockhart LH, Hawkins JC, Jalal SM, Withers M, Lupski JR, Stankiewicz P. 2005. Position Effects Due to Chromosome Breakpoints that Map ∼900 Kb Upstream and ∼1.3 Mb Downstream of SOX9 in Two Patients with Campomelic Dysplasia. The American Journal of Human Genetics 76: 652–662.</p>
</div>
<div>
<p>Vetro A, Dehghani MR, Kraoua L, Giorda R, Beri S, Cardarelli L, Merico M, Manolakos E, Parada-Bustamante A, Castro A, Radi O, Camerino G, Brusco A, Sabaghian M, Sofocleous C, Forzano F, Palumbo P, Palumbo O, Calvano S, Zelante L, Grammatico P, Giglio S, Basly M, Chaabouni M, Carella M, Russo G, Bonaglia MC, Zuffardi O. 2015. Testis development in the absence of SRY: chromosomal rearrangements at SOX9 and SOX3. European Journal of Human Genetics 23: 1025–1032.</p>
</div>
<div>
<p>Vietri Rudan M, Barrington C, Henderson S, Ernst C, Odom DT, Tanay A, Hadjur S. 2015. Comparative Hi-C reveals that CTCF underlies evolution of chromosomal domain architecture. Cell reports 10: 1297–309.</p>
</div>
<div>
<p>Vilella AJ, Severin J, Ureta-Vidal A, Heng L, Durbin R, Birney E. 2009. EnsemblCompara GeneTrees: Complete, duplication-aware phylogenetic trees in vertebrates. Genome Research 19: 327–335.</p>
</div>
<div>
<p>Visel A, Minovitsky S, Dubchak I, Pennacchio L a. 2007. VISTA Enhancer Browser–a database of tissue-specific human enhancers. Nucleic acids research 35: D88–92.</p>
</div>
<div>
<p>Visser M, Kayser M, Palstra R-J. 2012. HERC2 rs12913832 modulates human pigmentation by attenuating chromatin-loop formation between a long-range enhancer and the OCA2 promoter. Genome research 22: 446–55.</p>
</div>
<div>
<p>Walter J, Schermelleh L, Cremer M, Tashiro S, Cremer T. 2003. Chromosome order in HeLa cells changes during mitosis and early G1, but is stably maintained during subsequent interphase stages. The Journal of cell biology 160: 685–97.</p>
</div>
<div>
<p>Weiler KS, Wakimoto BT. 1995. Heterochromatin and gene expression in Drosophila. Annual review of genetics 29: 577–605.</p>
</div>
<div>
<p>Weintraub AS, Li CH, Zamudio AV, Sigova AA, Hannet NM, Day DS, Abraham BJ, Cohen MA, Nabet B, Buckley DL, Guo YE, Hnisz D, Jaenisch R, Bradner JE, Gray NS, Young RA. 2018. YY1 Is a Structural Regulator of Enhancer-Promoter Loops. Cell: 1–16.</p>
</div>
<div>
<p>Weischenfeldt J, Dubash T, Drainas AP, Mardin BR, Chen Y, Stütz AM, Waszak SM, Bosco G, Halvorsen AR, Raeder B, Efthymiopoulos T, Erkek S, Siegl C, Brenner H, Brustugun OT, Dieter SM, Northcott PA, Petersen I, Pfister SM, Schneider M, Solberg SK, Thunissen E, Weichert W, Zichner T, Thomas R, Peifer M, Helland A, Ball CR, Jechlinger M, Sotillo R, Glimm H, Korbel JO. 2016. Pan-cancer analysis of somatic copy-number alterations implicates IRS4 and IGF2 in enhancer hijacking. Nature Genetics.</p>
</div>
<div>
<p>Whalen S, Truty RM, Pollard KS. 2015. Enhancer-promoter interactions are encoded by complex genomic signatures on looping chromatin. Nature Genetics 48: 1–10.</p>
</div>
<div>
<p>Whyte WA, Orlando DA, Hnisz D, Abraham BJ, Lin CY, Kagey MH, Rahl PB, Lee TI, Young RA. 2013. Master transcription factors and mediator establish super-enhancers at key cell identity genes. Cell 153: 307–19.</p>
</div>
<div>
<p>Wickham H, Grolemund G. R for data science : import, tidy, transform, visualize, and model data.</p>
</div>
<div>
<p>Will AJ, Cova G, Osterwalder M, Chan W-L, Wittler L, Brieske N, Heinrich V, Villartay J-P de, Vingron M, Klopocki E, Visel A, Lupiáñez DG, Mundlos S. 2017. Composition and dosage of a multipartite enhancer cluster control developmental expression of Ihh (Indian hedgehog). Nature Genetics.</p>
</div>
<div>
<p>Wit E de, Laat W de. 2012. A decade of 3C technologies: insights into nuclear organization. Genes &amp; development 26: 11–24.</p>
</div>
<div>
<p>Wit E de, Vos ESM, Holwerda SJB, Valdes-Quezada C, Verstegen MJAM, Teunissen H, Splinter E, Wijchers PJ, Krijger PHL, Laat W de. 2015. CTCF Binding Polarity Determines Chromatin Looping. Molecular Cell 60: 676–684.</p>
</div>
<div>
<p>Won H, Torre-Ubieta L de la, Stein JL, Parikshak NN, Huang J, Opland CK, Gandal MJ, Sutton GJ, Hormozdiari F, Lu D, Lee C, Eskin E, Voineagu I, Ernst J, Geschwind DH. 2016. Chromosome conformation elucidates regulatory relationships in developing human brain. Nature: 1–20.</p>
</div>
<div>
<p>Xie S, Duan J, Li B, Zhou P, Hon GC. 2017. Multiplexed Engineering and Analysis of Combinatorial Enhancer Activity in Single Cells. Molecular Cell 66: 285–299.e5.</p>
</div>
<div>
<p>Yardımcı GG, Frank CL, Crawford GE, Ohler U. 2014. Explicit DNase sequence bias modeling enables high-resolution transcription factor footprint detection. Nucleic acids research: 1–14.</p>
</div>
<div>
<p>Ye B-Y, Shen W-L, Wang D, Li P, Zhang Z, Shi M-L, Zhang Y, Zhang F-X, Zhao Z-H. 2016. ZNF143 is involved in CTCF-mediated chromatin interactions by cooperation with cohesin and other partners. Molecular Biology 50: 431–437.</p>
</div>
<div>
<p>Zepeda-Mendoza CJ, Ibn-Salem J, Kammin T, Harris DJ, Rita D, Gripp KW, MacKenzie JJ, Gropman A, Graham B, Shaheen R, Alkuraya FS, Brasington CK, Spence EJ, Masser-Frye D, Bird LM, Spiegel E, Sparkes RL, Ordulu Z, Talkowski ME, Andrade-Navarro MA, Robinson PN, Morton CC. 2017. Computational Prediction of Position Effects of Apparently Balanced Human Chromosomal Rearrangements. American journal of human genetics 101: 206–217.</p>
</div>
<div>
<p>Zhan Y, Mariani L, Barozzi I, Schulz EG, Blüthgen N, Stadler M, Tiana G, Giorgetti L, Bluthgen N, Stadler M, Tiana G, Giorgetti L. 2017. Reciprocal insulation analysis of Hi-C data shows that TADs represent a functionally but not structurally privileged scale in the hierarchical folding of chromosomes. Genome Research 27: gr.212803.116.</p>
</div>
<div>
<p>Zhang B, Park B-H, Karpinets T, Samatova NF. 2008. From pull-down data to protein interaction networks and complexes with biological relevance. Bioinformatics (Oxford, England) 24: 979–86.</p>
</div>
<div>
<p>Zhang B, Wolynes PG. 2015. Topology, structures, and energy landscapes of human chromosomes. Proceedings of the National Academy of Sciences 112: 201506257.</p>
</div>
<div>
<p>Zhang F, Gu W, Hurles ME, Lupski JR. 2009. Copy Number Variation in Human Health, Disease, and Evolution. Annual Review of Genomics and Human Genetics 10: 451–481.</p>
</div>
<div>
<p>Zhao C, Li X, Hu H. 2016. PETModule: a motif module based approach for enhancer target gene prediction. Scientific Reports 6: 30043.</p>
</div>
<div>
<p>Zhou X, Lowdon RF, Li D, Lawson H a, Madden P a F, Costello JF, Wang T. 2013. Exploring long-range genome interactions using the WashU Epigenome Browser. Nature methods 10: 375–6.</p>
</div>
<div>
<p>Zhou X, Wang T. 2012. Using the Wash U Epigenome Browser to Examine Genome-Wide Sequencing Data. In: Current protocols in bioinformatics. Hoboken, NJ, USA: John Wiley &amp; Sons, Inc., p Unit10.10.</p>
</div>
<div>
<p>Zhu B, Zhang W, Zhang T, Liu B, Jiang J. 2015. Genome-Wide Prediction and Validation of Intergenic Enhancers in Arabidopsis Using Open Chromatin Signatures. The Plant cell 27: 2415–26.</p>
</div>
<div>
<p>Zhu Y, Chen Z, Zhang K, Wang M, Medovoy D, Whitaker JW, Ding B, Li N, Zheng L, Wang W. 2016. Constructing 3D interaction maps from 1D epigenomes. Nature Communications 7: 10812.</p>
</div>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="D-2-supplementary-figures-1.html" class="navigation navigation-prev navigation-unique" aria-label="Previous page"><i class="fa fa-angle-left"></i></a>

    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"google": false,
"weibo": false,
"instapper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/ibn-salem/thesis/edit/master/08-appendix.Rmd",
"text": "Edit"
},
"download": ["thesis.pdf", "thesis.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://cdn.bootcss.com/mathjax/2.7.1/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:" && /^https?:/.test(src))
      src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
