<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>Genome folding in evolution and disease</title>
  <meta name="description" content="This is a draft PhD thesis.">
  <meta name="generator" content="bookdown 0.5.10 and GitBook 2.6.7">

  <meta property="og:title" content="Genome folding in evolution and disease" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="This is a draft PhD thesis." />
  <meta name="github-repo" content="ibn-salem/thesis" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Genome folding in evolution and disease" />
  
  <meta name="twitter:description" content="This is a draft PhD thesis." />
  

<meta name="author" content="Jonas Ibn-Salem">


<meta name="date" content="2018-01-25">

  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  
<link rel="prev" href="index.html">
<link rel="next" href="ch-paralog-regulaion.html">
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />










<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Thesis</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Preface</a></li>
<li class="chapter" data-level="1" data-path="introduction.html"><a href="introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a><ul>
<li class="chapter" data-level="1.1" data-path="introduction.html"><a href="introduction.html#structure-of-the-intorduction"><i class="fa fa-check"></i><b>1.1</b> Structure of the Intorduction</a></li>
<li class="chapter" data-level="1.2" data-path="introduction.html"><a href="introduction.html#cell-diversity-and-regulation-of-gene-expression"><i class="fa fa-check"></i><b>1.2</b> Cell diversity and regulation of gene expression</a></li>
<li class="chapter" data-level="1.3" data-path="introduction.html"><a href="introduction.html#enhancers"><i class="fa fa-check"></i><b>1.3</b> Enhancers</a></li>
<li class="chapter" data-level="1.4" data-path="introduction.html"><a href="introduction.html#methods-to-probe-the-3d-chromatin-archtiecture"><i class="fa fa-check"></i><b>1.4</b> Methods to probe the 3D chromatin archtiecture</a><ul>
<li class="chapter" data-level="1.4.1" data-path="introduction.html"><a href="introduction.html#microscopy-based-techniques-to-visuallize-the-genome-in-3d"><i class="fa fa-check"></i><b>1.4.1</b> Microscopy-based techniques to visuallize the genome in 3D</a></li>
<li class="chapter" data-level="1.4.2" data-path="introduction.html"><a href="introduction.html#proximity-ligation-based-method-to-quantify-chromatin-interactions"><i class="fa fa-check"></i><b>1.4.2</b> Proximity-ligation based method to quantify chromatin interactions</a></li>
</ul></li>
<li class="chapter" data-level="1.5" data-path="introduction.html"><a href="introduction.html#hierarchy-of-chromatin-3d-structure"><i class="fa fa-check"></i><b>1.5</b> Hierarchy of chromatin 3D structure</a><ul>
<li class="chapter" data-level="1.5.1" data-path="introduction.html"><a href="introduction.html#chromosomal-territories-and-inter-chromosomal-contacts"><i class="fa fa-check"></i><b>1.5.1</b> Chromosomal territories and inter-chromosomal contacts</a></li>
<li class="chapter" data-level="1.5.2" data-path="introduction.html"><a href="introduction.html#ab-compartments"><i class="fa fa-check"></i><b>1.5.2</b> A/B compartments</a></li>
<li class="chapter" data-level="1.5.3" data-path="introduction.html"><a href="introduction.html#topologically-associating-domains-tads"><i class="fa fa-check"></i><b>1.5.3</b> Topologically associating domains (TADs)</a></li>
<li class="chapter" data-level="1.5.4" data-path="introduction.html"><a href="introduction.html#chromatin-loops"><i class="fa fa-check"></i><b>1.5.4</b> Chromatin loops</a></li>
</ul></li>
<li class="chapter" data-level="1.6" data-path="introduction.html"><a href="introduction.html#dynamics-of-chromatin-structure"><i class="fa fa-check"></i><b>1.6</b> Dynamics of chromatin structure</a></li>
<li class="chapter" data-level="1.7" data-path="introduction.html"><a href="introduction.html#changes-of-genome-folding-in-evoltuion-and-disease-genomes"><i class="fa fa-check"></i><b>1.7</b> Changes of genome folding in evoltuion and disease genomes</a></li>
<li class="chapter" data-level="1.8" data-path="introduction.html"><a href="introduction.html#aims-of-this-thesis"><i class="fa fa-check"></i><b>1.8</b> Aims of this thesis</a><ul>
<li class="chapter" data-level="" data-path="introduction.html"><a href="introduction.html#is-the-three-dimensonal-folding-of-genomes-associated-with-co-regulation-of-functionally-related-genes"><i class="fa fa-check"></i>Is the three-dimensonal folding of genomes associated with co-regulation of functionally related genes?</a></li>
<li class="chapter" data-level="" data-path="introduction.html"><a href="introduction.html#are-tads-functional-buidling-blocks-of-genomes-and-subjected-to-selective-preasure-during-evolution"><i class="fa fa-check"></i>Are TADs functional buidling blocks of genomes and subjected to selective preasure during evolution?</a></li>
<li class="chapter" data-level="" data-path="introduction.html"><a href="introduction.html#is-the-disruption-of-tads-by-rearrangements-also-associated-to-genetic-diseases"><i class="fa fa-check"></i>Is the disruption of TADs by rearrangements also associated to genetic diseases?</a></li>
<li class="chapter" data-level="" data-path="introduction.html"><a href="introduction.html#can-chromatin-looping-interactions-be-predicted-by-chip-seq-and-sequence-features"><i class="fa fa-check"></i>Can chromatin looping interactions be predicted by ChIP-seq and sequence features?</a></li>
</ul></li>
<li class="chapter" data-level="1.9" data-path="introduction.html"><a href="introduction.html#structure-of-this-thesis"><i class="fa fa-check"></i><b>1.9</b> Structure of this thesis</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html"><i class="fa fa-check"></i><b>2</b> Paralog genes in the 3D genome architecture</a><ul>
<li class="chapter" data-level="2.1" data-path="introduction.html"><a href="introduction.html#introduction"><i class="fa fa-check"></i><b>2.1</b> Introduction</a></li>
<li class="chapter" data-level="2.2" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#sec:paralog-methods"><i class="fa fa-check"></i><b>2.2</b> MATERIALS AND METHODS</a><ul>
<li class="chapter" data-level="2.2.1" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#selection-of-pairs-of-paralog-genes"><i class="fa fa-check"></i><b>2.2.1</b> Selection of pairs of paralog genes</a></li>
<li class="chapter" data-level="2.2.2" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#enhancers-to-gene-association"><i class="fa fa-check"></i><b>2.2.2</b> Enhancers to gene association</a></li>
<li class="chapter" data-level="2.2.3" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#topological-associating-domains"><i class="fa fa-check"></i><b>2.2.3</b> Topological associating domains</a></li>
<li class="chapter" data-level="2.2.4" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#hi-c-interaction-maps"><i class="fa fa-check"></i><b>2.2.4</b> Hi-C interaction maps</a></li>
<li class="chapter" data-level="2.2.5" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#randomization"><i class="fa fa-check"></i><b>2.2.5</b> Randomization</a></li>
<li class="chapter" data-level="2.2.6" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#statistical-tests"><i class="fa fa-check"></i><b>2.2.6</b> Statistical tests</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#results"><i class="fa fa-check"></i><b>2.3</b> RESULTS</a><ul>
<li class="chapter" data-level="2.3.1" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#distribution-of-paralog-genes-in-the-human-genome"><i class="fa fa-check"></i><b>2.3.1</b> Distribution of paralog genes in the human genome</a></li>
<li class="chapter" data-level="2.3.2" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#co-expression-of-paralog-gene-pairs-across-tissues"><i class="fa fa-check"></i><b>2.3.2</b> Co-expression of paralog gene pairs across tissues</a></li>
<li class="chapter" data-level="2.3.3" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#paralog-genes-share-enhancers"><i class="fa fa-check"></i><b>2.3.3</b> Paralog genes share enhancers</a></li>
<li class="chapter" data-level="2.3.4" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#co-localization-of-paralogs-in-tads"><i class="fa fa-check"></i><b>2.3.4</b> Co-localization of paralogs in TADs</a></li>
<li class="chapter" data-level="2.3.5" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#distal-paralog-pairs-are-enriched-for-long-range-chromatin-contacts"><i class="fa fa-check"></i><b>2.3.5</b> Distal paralog pairs are enriched for long-range chromatin contacts</a></li>
<li class="chapter" data-level="2.3.6" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#close-paralogs-have-fewer-contacts-than-expected"><i class="fa fa-check"></i><b>2.3.6</b> Close paralogs have fewer contacts than expected</a></li>
<li class="chapter" data-level="2.3.7" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#paralogs-in-mouse-and-dog-genome"><i class="fa fa-check"></i><b>2.3.7</b> Paralogs in mouse and dog genome</a></li>
<li class="chapter" data-level="2.3.8" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#orthologs-of-human-paralogs-show-conserved-co-localization"><i class="fa fa-check"></i><b>2.3.8</b> Orthologs of human paralogs show conserved co-localization</a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#discussion"><i class="fa fa-check"></i><b>2.4</b> DISCUSSION</a></li>
<li class="chapter" data-level="2.5" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#conclusion"><i class="fa fa-check"></i><b>2.5</b> CONCLUSION</a></li>
<li class="chapter" data-level="2.6" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#acknowledgements"><i class="fa fa-check"></i><b>2.6</b> ACKNOWLEDGEMENTS</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html"><i class="fa fa-check"></i><b>3</b> Stability of TADs in evolution</a><ul>
<li class="chapter" data-level="" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#abstract"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#keywords"><i class="fa fa-check"></i>Keywords</a></li>
<li class="chapter" data-level="3.1" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#introduction-1"><i class="fa fa-check"></i><b>3.1</b> Introduction</a></li>
<li class="chapter" data-level="3.2" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#results-1"><i class="fa fa-check"></i><b>3.2</b> Results</a><ul>
<li class="chapter" data-level="3.2.1" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#identification-of-evolutionary-rearrangement-breakpoints-from-whole-genome-alignments"><i class="fa fa-check"></i><b>3.2.1</b> Identification of evolutionary rearrangement breakpoints from whole-genome alignments</a></li>
<li class="chapter" data-level="3.2.2" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#rearrangement-breakpoints-are-enriched-at-tad-boundaries"><i class="fa fa-check"></i><b>3.2.2</b> Rearrangement breakpoints are enriched at TAD boundaries</a></li>
<li class="chapter" data-level="3.2.3" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#clusters-of-conserved-non-coding-elements-are-depleted-for-rearrangement-breakpoints"><i class="fa fa-check"></i><b>3.2.3</b> Clusters of conserved non-coding elements are depleted for rearrangement breakpoints</a></li>
<li class="chapter" data-level="3.2.4" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#rearranged-tads-are-associated-with-divergent-gene-expression-between-species"><i class="fa fa-check"></i><b>3.2.4</b> Rearranged TADs are associated with divergent gene expression between species</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#discussion-1"><i class="fa fa-check"></i><b>3.3</b> Discussion</a></li>
<li class="chapter" data-level="3.4" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#conclusion-1"><i class="fa fa-check"></i><b>3.4</b> Conclusion</a></li>
<li class="chapter" data-level="3.5" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#methods"><i class="fa fa-check"></i><b>3.5</b> Methods</a><ul>
<li class="chapter" data-level="3.5.1" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#rearrangement-breakpoints-from-whole-genome-alignments"><i class="fa fa-check"></i><b>3.5.1</b> Rearrangement breakpoints from whole-genome alignments</a></li>
<li class="chapter" data-level="3.5.2" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#topologically-associating-domains-and-contact-domains"><i class="fa fa-check"></i><b>3.5.2</b> Topologically associating domains and contact domains</a></li>
<li class="chapter" data-level="3.5.3" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#breakpoint-distributions-at-tads"><i class="fa fa-check"></i><b>3.5.3</b> Breakpoint distributions at TADs</a></li>
<li class="chapter" data-level="3.5.4" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#quantification-of-breakpoint-enrichment"><i class="fa fa-check"></i><b>3.5.4</b> Quantification of breakpoint enrichment</a></li>
<li class="chapter" data-level="3.5.5" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#expression-data-for-mouse-and-human-orthologs"><i class="fa fa-check"></i><b>3.5.5</b> Expression data for mouse and human orthologs</a></li>
<li class="chapter" data-level="3.5.6" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#classification-of-tads-and-genes-according-to-rearrangements-and-grbs"><i class="fa fa-check"></i><b>3.5.6</b> Classification of TADs and genes according to rearrangements and GRBs</a></li>
<li class="chapter" data-level="3.5.7" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#source-code-and-implementation-details"><i class="fa fa-check"></i><b>3.5.7</b> Source code and implementation details</a></li>
</ul></li>
<li class="chapter" data-level="3.6" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#declarations"><i class="fa fa-check"></i><b>3.6</b> Declarations</a><ul>
<li class="chapter" data-level="3.6.1" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#ethics-approval-and-consent-to-participate"><i class="fa fa-check"></i><b>3.6.1</b> Ethics approval and consent to participate</a></li>
<li class="chapter" data-level="3.6.2" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#consent-for-publication"><i class="fa fa-check"></i><b>3.6.2</b> Consent for publication</a></li>
<li class="chapter" data-level="3.6.3" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#availability-of-data-and-material"><i class="fa fa-check"></i><b>3.6.3</b> Availability of data and material</a></li>
<li class="chapter" data-level="3.6.4" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#competing-interests"><i class="fa fa-check"></i><b>3.6.4</b> Competing interests</a></li>
<li class="chapter" data-level="3.6.5" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#funding"><i class="fa fa-check"></i><b>3.6.5</b> Funding</a></li>
<li class="chapter" data-level="3.6.6" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#authors-contributions"><i class="fa fa-check"></i><b>3.6.6</b> Authors‘ contributions</a></li>
<li class="chapter" data-level="3.6.7" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#acknowledgments"><i class="fa fa-check"></i><b>3.6.7</b> Acknowledgments</a></li>
</ul></li>
<li class="chapter" data-level="3.7" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#tables"><i class="fa fa-check"></i><b>3.7</b> Tables</a><ul>
<li class="chapter" data-level="3.7.1" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#table-1"><i class="fa fa-check"></i><b>3.7.1</b> Table 1</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="ch-position-effect.html"><a href="ch-position-effect.html"><i class="fa fa-check"></i><b>4</b> Position effects of rearrangements in disease genomes</a><ul>
<li class="chapter" data-level="4.1" data-path="ch-position-effect.html"><a href="ch-position-effect.html#abstract-1"><i class="fa fa-check"></i><b>4.1</b> Abstract</a></li>
<li class="chapter" data-level="4.2" data-path="ch-position-effect.html"><a href="ch-position-effect.html#introduction-2"><i class="fa fa-check"></i><b>4.2</b> Introduction</a></li>
<li class="chapter" data-level="4.3" data-path="ch-position-effect.html"><a href="ch-position-effect.html#materials-and-methods"><i class="fa fa-check"></i><b>4.3</b> Materials and Methods</a><ul>
<li class="chapter" data-level="4.3.1" data-path="ch-position-effect.html"><a href="ch-position-effect.html#selection-of-subjects-with-apparently-balanced-chromosome-abnormalities"><i class="fa fa-check"></i><b>4.3.1</b> Selection of subjects with apparently balanced chromosome abnormalities</a></li>
<li class="chapter" data-level="4.3.2" data-path="ch-position-effect.html"><a href="ch-position-effect.html#clinical-descriptions-of-dgap-cases"><i class="fa fa-check"></i><b>4.3.2</b> Clinical descriptions of DGAP cases</a></li>
<li class="chapter" data-level="4.3.3" data-path="ch-position-effect.html"><a href="ch-position-effect.html#analysis-of-genes-bordering-the-rearrangement-breakpoints"><i class="fa fa-check"></i><b>4.3.3</b> Analysis of genes bordering the rearrangement breakpoints</a></li>
<li class="chapter" data-level="4.3.4" data-path="ch-position-effect.html"><a href="ch-position-effect.html#assessment-of-disrupted-functional-elements-and-chromatin-interactions"><i class="fa fa-check"></i><b>4.3.4</b> Assessment of disrupted functional elements and chromatin interactions</a></li>
<li class="chapter" data-level="4.3.5" data-path="ch-position-effect.html"><a href="ch-position-effect.html#ontological-analysis-of-genes-neighboring-breakpoints"><i class="fa fa-check"></i><b>4.3.5</b> Ontological analysis of genes neighboring breakpoints</a></li>
<li class="chapter" data-level="4.3.6" data-path="ch-position-effect.html"><a href="ch-position-effect.html#quantitative-real-time-pcr"><i class="fa fa-check"></i><b>4.3.6</b> Quantitative real-time PCR</a></li>
<li class="chapter" data-level="4.3.7" data-path="ch-position-effect.html"><a href="ch-position-effect.html#assessment-of-dgap-breakpoints-overlapping-with-non-coding-structural-variants-in-public-databases"><i class="fa fa-check"></i><b>4.3.7</b> Assessment of DGAP breakpoints overlapping with non-coding structural variants in public databases</a></li>
</ul></li>
<li class="chapter" data-level="4.4" data-path="ch-position-effect.html"><a href="ch-position-effect.html#results-2"><i class="fa fa-check"></i><b>4.4</b> Results</a><ul>
<li class="chapter" data-level="4.4.1" data-path="ch-position-effect.html"><a href="ch-position-effect.html#genomic-characterization-of-non-coding-breakpoints"><i class="fa fa-check"></i><b>4.4.1</b> Genomic characterization of non-coding breakpoints</a></li>
<li class="chapter" data-level="4.4.2" data-path="ch-position-effect.html"><a href="ch-position-effect.html#identification-of-genes-with-potential-position-effects"><i class="fa fa-check"></i><b>4.4.2</b> Identification of genes with potential position effects</a></li>
<li class="chapter" data-level="4.4.3" data-path="ch-position-effect.html"><a href="ch-position-effect.html#identification-of-subjects-with-shared-non-coding-chromosome-alterations-and-phenotypes"><i class="fa fa-check"></i><b>4.4.3</b> Identification of subjects with shared non-coding chromosome alterations and phenotypes</a></li>
</ul></li>
<li class="chapter" data-level="4.5" data-path="ch-position-effect.html"><a href="ch-position-effect.html#discussion-2"><i class="fa fa-check"></i><b>4.5</b> Discussion</a></li>
<li class="chapter" data-level="4.6" data-path="ch-position-effect.html"><a href="ch-position-effect.html#acknowledgements-1"><i class="fa fa-check"></i><b>4.6</b> Acknowledgements</a></li>
<li class="chapter" data-level="4.7" data-path="ch-position-effect.html"><a href="ch-position-effect.html#tables-1"><i class="fa fa-check"></i><b>4.7</b> Tables</a><ul>
<li class="chapter" data-level="4.7.1" data-path="ch-position-effect.html"><a href="ch-position-effect.html#table-1."><i class="fa fa-check"></i><b>4.7.1</b> Table 1.</a></li>
</ul></li>
<li class="chapter" data-level="4.8" data-path="ch-position-effect.html"><a href="ch-position-effect.html#figures"><i class="fa fa-check"></i><b>4.8</b> Figures</a></li>
<li class="chapter" data-level="4.9" data-path="ch-position-effect.html"><a href="ch-position-effect.html#web-resources"><i class="fa fa-check"></i><b>4.9</b> Web Resources</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="ch-loop-prediction.html"><a href="ch-loop-prediction.html"><i class="fa fa-check"></i><b>5</b> Prediction of chromatin looping interactions</a></li>
<li class="chapter" data-level="6" data-path="discussion-3.html"><a href="discussion-3.html"><i class="fa fa-check"></i><b>6</b> Discussion</a><ul>
<li class="chapter" data-level="6.1" data-path="discussion-3.html"><a href="discussion-3.html#further-directions"><i class="fa fa-check"></i><b>6.1</b> Further directions</a></li>
<li class="chapter" data-level="6.2" data-path="discussion-3.html"><a href="discussion-3.html#conclusions"><i class="fa fa-check"></i><b>6.2</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="appendix"><span><b>Appendix</b></span></li>
<li class="chapter" data-level="A" data-path="supporting-information-co-regulation-of-paralog-genes-in-the-three-dimensional-chromatin-architecture.html"><a href="supporting-information-co-regulation-of-paralog-genes-in-the-three-dimensional-chromatin-architecture.html"><i class="fa fa-check"></i><b>A</b> Supporting Information: Co-regulation of paralog genes in the three-dimensional chromatin architecture</a></li>
<li class="chapter" data-level="B" data-path="supplementary-data-evolutionary-stability-of-topologically-associating-domains-is-associated-with-conserved-gene-regulation.html"><a href="supplementary-data-evolutionary-stability-of-topologically-associating-domains-is-associated-with-conserved-gene-regulation.html"><i class="fa fa-check"></i><b>B</b> Supplementary Data: Evolutionary stability of topologically associating domains is associated with conserved gene regulation</a><ul>
<li class="chapter" data-level="B.1" data-path="supplementary-data-evolutionary-stability-of-topologically-associating-domains-is-associated-with-conserved-gene-regulation.html"><a href="supplementary-data-evolutionary-stability-of-topologically-associating-domains-is-associated-with-conserved-gene-regulation.html#TadEvoSupTab"><i class="fa fa-check"></i><b>B.1</b> Supplementary Tables</a></li>
<li class="chapter" data-level="B.2" data-path="supplementary-data-evolutionary-stability-of-topologically-associating-domains-is-associated-with-conserved-gene-regulation.html"><a href="supplementary-data-evolutionary-stability-of-topologically-associating-domains-is-associated-with-conserved-gene-regulation.html#supplementary-figures"><i class="fa fa-check"></i><b>B.2</b> Supplementary Figures</a></li>
</ul></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Build with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Genome folding in evolution and disease</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="introduction" class="section level1">
<h1><span class="header-section-number">Chapter 1</span> Introduction</h1>
<div id="structure-of-the-intorduction" class="section level2">
<h2><span class="header-section-number">1.1</span> Structure of the Intorduction</h2>
<ul>
<li>Cell diversity</li>
<li>Gene Regulation</li>
<li>Enhancers
<ul>
<li>Enhancer–promoter interaction</li>
</ul></li>
<li>Techniques to probe the 3D chromatin architecture
<ul>
<li>Imaging based techniques</li>
<li>3C based methods</li>
<li>Hi-C</li>
<li>ChIA-PET</li>
</ul></li>
<li>Chromatin 3D Structure
<ul>
<li>Hierarchy of chromatin 3D structure
<ul>
<li>Chromosomal Territories</li>
<li>A/B-Compartments</li>
<li>TADs</li>
<li>Chromatin Loops
<ul>
<li>Enhancer-Promoter loops</li>
<li>Gene Loops</li>
<li>Architectural loops</li>
</ul></li>
</ul></li>
<li>Architectural Proteins</li>
<li>Loop extrusion model</li>
</ul></li>
<li>Evolution of chromatin architecture</li>
<li>Disruption of chromatin architecture</li>
</ul>
</div>
<div id="cell-diversity-and-regulation-of-gene-expression" class="section level2">
<h2><span class="header-section-number">1.2</span> Cell diversity and regulation of gene expression</h2>
<ul>
<li>Cell diversity</li>
<li>Gene Regulation</li>
</ul>
<p>An important mechanism in eukaryotic gene regulation is the binding of transcription factors (TFs) to distal regulatory regions such as enhancers which perform looping interactions to the transcription machinery at gene promoters. Chromatin interactions can be measured by chromatin conformation capture (3C) and its high-throughput variations such as 4C, 5C, 6C, ChIA-PET and Hi-C <span class="citation">(Dekker et al., <a href="references.html#ref-Dekker2013">2013</a>; Lieberman-Aiden et al., <a href="references.html#ref-Lieberman-Aiden2009">2009</a>)</span>.</p>
<p>While genome-wide Hi-C data is still only available for a limited number of cell-types and has limited resolution, it is successfully used to re-discover important features of the three-dimensional chromatin architecture in the nucleus that is organized on different levels. In interphase, chromosomes occupy distinct territories in the nucleus <span class="citation">(Cremer and Cremer, <a href="references.html#ref-Cremer2001">2001</a>)</span> and are further organized in mega-base scale A/B-compartments that show specific preferential interaction patterns and transcriptional activity <span class="citation">(Lieberman-Aiden et al., <a href="references.html#ref-Lieberman-Aiden2009">2009</a>; Rao et al., <a href="references.html#ref-Rao2014">2014</a>)</span>. On smaller scales topologically associating domains (TADs) were identified using Hi-C <span class="citation">(Dixon et al., <a href="references.html#ref-Dixon2012">2012</a>; Nora et al., <a href="references.html#ref-Nora2012">2012</a>; Sexton et al., <a href="references.html#ref-Sexton2012">2012</a>)</span>. These are regions of several hundred kb, that have more interactions within themselves than with other regions and are separated by boundaries that insulate interactions between loci in different TADs. Interestingly, TADs are largely stable across cell-types and conserved between mammals <span class="citation">(Dixon et al., <a href="references.html#ref-Dixon2012">2012</a>, <a href="references.html#ref-Dixon2015">2015</a>; Rao et al., <a href="references.html#ref-Rao2014">2014</a>; Vietri Rudan et al., <a href="references.html#ref-VietriRudan2015">2015</a>)</span>. However, many cell-type specific chromatin interaction loops occur within TADs <span class="citation">(Dixon et al., <a href="references.html#ref-Dixon2015">2015</a>; Rao et al., <a href="references.html#ref-Rao2014">2014</a>)</span>. This indicates a stable and cell-type invariant chromatin architecture on larger scales, such as TADs, and a more dynamic and cell-type specific organization of interactions within TADs that connect enhancers and TF binding sites to regulated genes.</p>
</div>
<div id="enhancers" class="section level2">
<h2><span class="header-section-number">1.3</span> Enhancers</h2>
<p>Enhancers where originally defined as genomic regions that enhance the expression of a reporter gene, when placed experimentally in front of a minimal promoter. <span class="citation">(Banerji et al., <a href="references.html#ref-Banerji1981">1981</a>; Shlyueva et al., <a href="references.html#ref-Shlyueva2014">2014</a>)</span>. Enhancer activity can also be detected genome-wide by specific patterns of open chromatin using DNase-seq <span class="citation">(Song and Crawford, <a href="references.html#ref-Song2010">2010</a>)</span> or ATAC-seq <span class="citation">(Buenrostro et al., <a href="references.html#ref-Buenrostro2013">2013</a>)</span> or posttranslational modification of histones, such as H3K27ac by ChIP-seq <span class="citation">(Creyghton et al., <a href="references.html#ref-Creyghton2010">2010</a>)</span>. Complex regulation of developmental genes is often archived by additive affects of multiple enhancers. For example the <span class="math inline">\(\alpha\)</span>-globin gene locus is controlled by multiple enhancers, whereby each enhancer elements act independently and in an additive fashion without evidence of synergistic or higher-order effects <span class="citation">(Hay et al., <a href="references.html#ref-Hay2016">2016</a>)</span>. Also the Indian hedgehog (Ihh) locus is regulated by multiple enhancers with individual combinations of tissue specificities that function in an additive manner <span class="citation">(Will et al., <a href="references.html#ref-Will2017">2017</a>)</span>. Experimental variation for of the copy number of enhancers is associated with expression strength. Significantly reduction of the expression of the oncogene <em>PIMI</em> could not be archived by perturbing a single enhancers, but only by combinatorial repression of several weak enhancers <span class="citation">(Xie et al., <a href="references.html#ref-Xie2017">2017</a>)</span>. (Enhancers are reviewed in <span class="citation">(Spitz and Furlong, <a href="references.html#ref-Spitz2012">2012</a>)</span> and <span class="citation">(Andrey and Mundlos, <a href="references.html#ref-Andrey2017">2017</a>)</span>)</p>
</div>
<div id="methods-to-probe-the-3d-chromatin-archtiecture" class="section level2">
<h2><span class="header-section-number">1.4</span> Methods to probe the 3D chromatin archtiecture</h2>
<ul>
<li>Imaging based techniques</li>
<li>3C based methods</li>
<li>Hi-C</li>
<li>ChIA-PET</li>
</ul>
<div id="microscopy-based-techniques-to-visuallize-the-genome-in-3d" class="section level3">
<h3><span class="header-section-number">1.4.1</span> Microscopy-based techniques to visuallize the genome in 3D</h3>
<p>Historically, the organization of chromosomes and specific loci within the nucleus have mostly been studied using fluorescent <em>in situ</em> hybridization (FISH) experiments. FISH is limited to examine a few pre-defined loci a few hundred cells at once and is limited in spacial resolution. Novel super-resolution microscopy approaches such as STORM and PALM have enabled direct visualization of the fin-scale structures of the genome at unprecedented resolution <span class="citation">(Bonev and Cavalli, <a href="references.html#ref-Bonev2016">2016</a>)</span>. Labeling of specific chromatin proteins, histone marks, or genomic loci allow to analyze the dynamics of chromosomes at high resolution in living cells. However, despite spectacular technical progress, microscopy-based approaches are limited to a small number of genetic loci and do not allow a comprehensive analysis of nuclear architecture of the complete genome. Furthermore, the specific folding patterns observed in microscopy cannot be mapped to genomic coordinates which hinders the integration with other genomic data. However, future combination of imaging-based techniques with proximity-ligation experiments together with integrative computational models, might enable to study the real-time dynamics of chromatin organization with high resolution on the single cell level <span class="citation">(Stevens et al., <a href="references.html#ref-Stevens2017">2017</a>, <span class="citation">Flyamer et al. (<a href="references.html#ref-Flyamer2017">2017</a>)</span>)</span>.</p>
</div>
<div id="proximity-ligation-based-method-to-quantify-chromatin-interactions" class="section level3">
<h3><span class="header-section-number">1.4.2</span> Proximity-ligation based method to quantify chromatin interactions</h3>
<p>The frequency of interactions between different loci in the genome can be experimentally measured by proximity ligation techniques <span class="citation">(Sati and Cavalli, <a href="references.html#ref-Sati2017">2017</a>; Schmitt et al., <a href="references.html#ref-Schmitt2016">2016</a>)</span>. These protocols are variations of the chromosome conformation capture (3C) experiment <span class="citation">(Dekker et al., <a href="references.html#ref-Dekker2002">2002</a>)</span>. 3C works by the ability to crosslink two genomic loci that are in close physical proximity in the nuclear space by treating cells with formaldehyde. The crosslinked chromatin is than digested by enzymes to fragment the genomic DNA. Than the fragmented DNA is re-ligated which results in hybrid DNA molecules of restriction fragments that where in close physical proximity during crosslinking but normally originate from different regions in the linear genome sequence <span class="citation">(Andrey and Mundlos, <a href="references.html#ref-Andrey2017">2017</a>; Dekker et al., <a href="references.html#ref-Dekker2013">2013</a>)</span> (Fig. <a href="introduction.html#fig:ProximityLigation">1.1</a>A).</p>
<!-- ---------------------------------------------------------------- -->
<div class="figure" style="text-align: center"><span id="fig:ProximityLigation"></span>
<img src="figures/Dekker2013_3C.png" alt="Proximity ligation technologies to measure chromatin interactions (A) By treating cells with formaldehyde chromatin is crosslinked. After fragmentation with restriction enzymes, DNA from two loci in close physical proximity in the nucleus is ligated to a hybrid DNA molecules that is than made from DNA that originated from two regions distal in the linear genome (indicated in red and blue). (B) Different variants of the 3C experiments differ in their approaches to measure the ligation products or subsets of it in order to quantify chromatin interactions. Figure modified from (Dekker et al., 2013). " width="80%" />
<p class="caption">
Figure 1.1: <strong>Proximity ligation technologies to measure chromatin interactions</strong> <strong>(A)</strong> By treating cells with formaldehyde chromatin is crosslinked. After fragmentation with restriction enzymes, DNA from two loci in close physical proximity in the nucleus is ligated to a hybrid DNA molecules that is than made from DNA that originated from two regions distal in the linear genome (indicated in red and blue). <strong>(B)</strong> Different variants of the 3C experiments differ in their approaches to measure the ligation products or subsets of it in order to quantify chromatin interactions. Figure modified from <span class="citation">(Dekker et al., <a href="references.html#ref-Dekker2013">2013</a>)</span>. <!-- ---------------------------------------------------------------- -->
</p>
</div>

<p>There exist several 3C-based methods which differ by the way the ligation product, which represents and chromatin interaction, is measured and quantified (Fig. <a href="introduction.html#fig:ProximityLigation">1.1</a>B). The classic 3C protocol allows to quantify hybrid DNA-product by quantitative PCR using specific primers to amplify the product junction <span class="citation">(Dekker et al., <a href="references.html#ref-Dekker2002">2002</a>)</span>. In Circular chromosome conformation capture (4C) experiments, a circular PCR is used to amplify all hybrid DNA products ligated with a desired restriction fragment, e.g. a specific viewpoint of interest. These products are than sequences to generate an interaction profile measuring all interacting regions with this viewpoint <span class="citation">(Noordermeer et al., <a href="references.html#ref-Noordermeer2011">2011</a>; Simonis et al., <a href="references.html#ref-Simonis2006">2006</a>)</span>. Another variant of 3C, Carbon copy chromosome conformation capture (5C), combines 3C with hybrid capture approaches to identify up to millions of interactions in parallel between two large sets of loci, for example between a set of promoters and a set of distal regulatory elements <span class="citation">(Dostie et al., <a href="references.html#ref-Dostie2006">2006</a>; Sanyal et al., <a href="references.html#ref-Sanyal2012">2012</a>)</span>. Some methods combine chromatin immunoprecipitation to enrich for chromatin interactions between loci bound by specific proteins of interest or marked by post-translational histone modifications. One of these methods is chromatin interaction analysis by paired-end tag sequencing (ChIA-PET), which allows for genome-wide analysis of long-range interactions between sites bound by a protein of interest <span class="citation">(Fullwood et al., <a href="references.html#ref-Fullwood2009">2009</a>)</span>. Therefore, ChIA-PET data represent a selected subset of all interactions, but is an efficient alternative to measure interactions at very high resolution <span class="citation">(Tang et al., <a href="references.html#ref-Tang2015">2015</a>)</span>. The most unbiased method to quantify all pair-wise interactions genome-wide is Hi-C <span class="citation">(Lieberman-Aiden et al., <a href="references.html#ref-Lieberman-Aiden2009">2009</a>)</span>. After the initial restriction enzyme step of 3C, in Hi-C, the ends are filled with a biotin-marked nucleotide and subsequently re-ligated. A streptavidin pull-down step is used to enrich for the chimeric products, which are than sequenced using paired-end sequencing technology. Each read from the resulting read-pairs is than aligned independently to the reference genome to identify the originating position of the sequenced restriction fragment. Thereby each read pair represent a pairwise physical interaction of the corresponding regions. Interaction frequencies are usually analyzed by binning the genome into equal sized regions several kb depending on sequencing depth. While the first Hi-C study produced genome-wide interactions at 1Mb resolution <span class="citation">(Lieberman-Aiden et al., <a href="references.html#ref-Lieberman-Aiden2009">2009</a>)</span>, more recent studies could analyse folding patterns at 40kb <span class="citation">(Dixon et al., <a href="references.html#ref-Dixon2012">2012</a>)</span>, and later up to 1kb resolution <span class="citation">(Rao et al., <a href="references.html#ref-Rao2014">2014</a>)</span>.</p>
<!-- libraries fo hypbrid DNA products are further digested with a second restiction enzyme and further re-ligated to g -->
</div>
</div>
<div id="hierarchy-of-chromatin-3d-structure" class="section level2">
<h2><span class="header-section-number">1.5</span> Hierarchy of chromatin 3D structure</h2>
<ul>
<li>Chromosomal Territories</li>
<li>A/B-Compartments</li>
<li>TADs</li>
<li>Chromatin Loops
<ul>
<li>Enhancer-Promoter loops</li>
<li>Gene Loops</li>
<li>Architectural loops</li>
</ul></li>
<li>Architectural Proteins</li>
<li>(Loop extrusion model)</li>
<li>Nucleosomes / 10nm / 30nm fibers</li>
</ul>
<!-- ---------------------------------------------------------------- -->
<div class="figure" style="text-align: center"><span id="fig:GenomeHierarchy"></span>
<img src="figures/Bonev2016Fig2.png" alt="(ref:GenomeHierarchy)" width="80%" />
<p class="caption">
Figure 1.2: (ref:GenomeHierarchy)
</p>
</div>
<p>(ref:GenomeHierarchy) <strong>Hierarchical organization of chromatin three-dimensional chromatin architecture</strong> <strong>(A)</strong><br />
Figure source <span class="citation">(Bonev and Cavalli, <a href="references.html#ref-Bonev2016">2016</a>)</span> <a href="https://www.nature.com/articles/nrg.2016.112/figures/2" class="uri">https://www.nature.com/articles/nrg.2016.112/figures/2</a>. <!-- ---------------------------------------------------------------- --></p>
<div id="chromosomal-territories-and-inter-chromosomal-contacts" class="section level3">
<h3><span class="header-section-number">1.5.1</span> Chromosomal territories and inter-chromosomal contacts</h3>
<p>The eukaryotic genome is highly organized in the interpahse nucelus. Chromosomes occupy distinct spatial regions, called chromosome territories, and intermingle less than one would expect by chance <span class="citation">(Cremer and Cremer, <a href="references.html#ref-Cremer2001">2001</a>)</span>. This was first observed using imaging based approaches, and is reflected in Hi-C interaction maps, where inter-chromosomal contacts occur order of magnitudes less frequent than intra-chromosomal contacts <span class="citation">(Lieberman-Aiden et al., <a href="references.html#ref-Lieberman-Aiden2009">2009</a>)</span>. However, despite this spatial segregation of chromosome, intermingling of chromosome occurs and is associated with chromosomal translocations <span class="citation">(Branco and Pombo, <a href="references.html#ref-Branco2006">2006</a>, <span class="citation">Roukos et al. (<a href="references.html#ref-Roukos2013">2013</a>)</span>, <span class="citation">Roukos and Misteli (<a href="references.html#ref-Roukos2014">2014</a>)</span>)</span>. There are also specific gene regulatory interactions between different chromosomes <span class="citation">(Laat and Grosveld, <a href="references.html#ref-DeLaat2007">2007</a>)</span>, for example olfactory receptor genes cluster densely in the nucleus of olfactory neurons to facilitate monoallelic expression of a single receptor gene per cell <span class="citation">(Monahan and Lomvardas, <a href="references.html#ref-Monahan2015">2015</a>)</span>. While there are specific inter-chromosomal contacts, the genome is non-randomly organized in chromosomes which occupy distinct territories in the spacial nucleus.</p>
</div>
<div id="ab-compartments" class="section level3">
<h3><span class="header-section-number">1.5.2</span> A/B compartments</h3>
<p>The ability to measure genome-wide chromatin contacts using Hi-C, revealed that individual regions on chromosomes segregate by preferential interactions into two compartments mayor clusters, referred to as A/B-compartments <span class="citation">(Lieberman-Aiden et al., <a href="references.html#ref-Lieberman-Aiden2009">2009</a>)</span>. Interestingly, regions A-compartments are associated with active histone-modifications and active transcription, whereas B-compartment is associated with heterochromatin, lamina association, and repressed genes.</p>
<!--  further organized in mega-base scale -->
<!-- A/B-compartments that show specific preferential interaction patterns -->
<!-- and transcriptional activity [@Lieberman-Aiden2009; @Rao2014]. -->
</div>
<div id="topologically-associating-domains-tads" class="section level3">
<h3><span class="header-section-number">1.5.3</span> Topologically associating domains (TADs)</h3>
<p>Compartments could be identified by clustering of long-range interactions in Hi-C maps with bin resolution of 1 Mb. In 2012 higher resolution Hi-C maps of up to 40 kb lead to the identification of genomic regions with preferential interactions wiht them. These genomic regions were termed topologically associating domains (TADs) <span class="citation">(Dixon et al., <a href="references.html#ref-Dixon2012">2012</a>; Nora et al., <a href="references.html#ref-Nora2012">2012</a>; Sexton et al., <a href="references.html#ref-Sexton2012">2012</a>)</span>. They are opperationlly defined as genomic regions with frequent interactions of loci within the domain and decreased interactions across domain boundares <span class="citation">(<span class="citeproc-not-found" data-reference-id="Nora2016"><strong>???</strong></span>)</span>.</p>
<p>TADs can be identified from Hi-C interaction maps computationally by different algorithms <span class="citation">(Ay and Noble, <a href="references.html#ref-Ay2015">2015</a>)</span>. The directionality index is a score for each bin in the Hi-C matrix that quantifies the number of upstream versus downstream interactions of this bin. Using hidden Markov models TAD boundaries where than identified in regions where DI is changing drastically <span class="citation">(Dixon et al., <a href="references.html#ref-Dixon2012">2012</a>)</span>. Other algorithms compute an insulation score as the extent to which interactions to cross potential TAD boundaries <span class="citation">(Crane et al., <a href="references.html#ref-Crane2015">2015</a>)</span>. Later the Arrowhead algorithm was introduced to find “contact domains” as smaller nested structures along the diagonal of high resolution Hi-C matrices <span class="citation">(Rao et al., <a href="references.html#ref-Rao2014">2014</a>)</span>. Furthermore, when analyzing Hi-C interactions at differnt length scales, hierarchies of TADs and sub-TADs could be identifed that overlap each other <span class="citation">(Filippova et al., <a href="references.html#ref-Filippova2014">2014</a>, <span class="citation">Fraser et al. (<a href="references.html#ref-Fraser2015">2015</a>)</span>)</span>.<br />
The different algorithms and parameters used in each is only one source of variation in reliably identifying TADs. Also the resolution of Hi-C maps, which is mainly defined by sequencing depths but also the Hi-C protocol itself <span class="citation">(<span class="citeproc-not-found" data-reference-id="Rao2015"><strong>???</strong></span>)</span>, as well as different normalization strategies for Hi-C contacts introduce variability <span class="citation">(Forcato et al., <a href="references.html#ref-Forcato2017">2017</a>)</span>. Therefore the number and size of TADs varies between different studies and cannot be directly compared.</p>
<p>The first studies on TADs identifed around 3000 TADs in human and mouse genomes with a median size of ~800 kb <span class="citation">(Dixon et al., <a href="references.html#ref-Dixon2012">2012</a>)</span> and ~100 kb in <em>Drosophila</em> gneomes <span class="citation">(Sexton et al., <a href="references.html#ref-Sexton2012">2012</a>)</span>. Analysis of 1kb or 5kb resolution Hi-C matrixes resulted in nested “contact domains” with median size of 185 kb (range 40 kb - 3 Mb) in human and mouse cells <span class="citation">(Rao et al., <a href="references.html#ref-Rao2014">2014</a>)</span>.</p>
<p>Importantly, TADs might be equivalent to “chromatin domains” of 10 kb - 1 Mb in size detected by microscopy approaches <span class="citation">(Cremer and Cremer, <a href="references.html#ref-Cremer2010">2010</a>; Gibcus and Dekker, <a href="references.html#ref-Gibcus2013">2013</a>)</span>. Annother connection of Hi-C derived interaction maps with previous microscopy observations, is that TADs in <em>Drosophila</em> correspond to bands of polytene chromosomes <span class="citation">(Eagen et al., <a href="references.html#ref-Eagen2015">2015</a>)</span>.</p>
<p>The spatial positioning of TADs correlate with many genomic features measured along the linear genome <span class="citation">(Merkenschlager and Nora, <a href="references.html#ref-Merkenschlager2016">2016</a>)</span>. TAD boudnaries are enriched for binding of “insulator protins”, such as CTCF in mammals and CP190 in Drosophila <span class="citation">(Dixon et al., <a href="references.html#ref-Dixon2012">2012</a>; Sexton et al., <a href="references.html#ref-Sexton2012">2012</a>)</span>. Furthermore, TAD boundares are associated with active chromatin, such as H3K4me3 and H3K27me3, DNase I hypersensivity, actie transcription, and house-keeping genes <span class="citation">(Dixon et al., <a href="references.html#ref-Dixon2012">2012</a>)</span>. Furthermore, TADs correspond to regions of early and late replication timing <span class="citation">(<span class="citeproc-not-found" data-reference-id="Dileep2015"><strong>???</strong></span>; Pope et al., <a href="references.html#ref-Pope2014">2014</a>)</span> and lamina associated domains (LADs) <span class="citation">(Dixon et al., <a href="references.html#ref-Dixon2012">2012</a>)</span>. Importantly, enhancer-promoter interactions seems to be mostly constrained within TADs <span class="citation">(Shen et al., <a href="references.html#ref-Shen2012">2012</a>)</span></p>
<p>There is acumulating evidence, that TADs are fundamental units of chromosome organization <span class="citation">(Dixon et al., <a href="references.html#ref-Dixon2016">2016</a>)</span></p>
<ul>
<li>[X] Definition, number, and size</li>
<li>[X] Algorithms to identify TADs</li>
<li>Functional features</li>
<li>[X] CTCF binding at boundaries</li>
<li>[X] housekeeping genes at boundaries</li>
<li>enhancer-gene associations within TADs</li>
</ul>
</div>
<div id="chromatin-loops" class="section level3">
<h3><span class="header-section-number">1.5.4</span> Chromatin loops</h3>
<p>In summary, these findings suggest a hierarchical organization of chromosome architecture. First, dynamic nucleosome contacts from clutches and fibers. These engage in dynamic long-range chromatin loops, some of which are stabilized by architectural proteins, such as CTCF and cohesin, and lead to the formation of TADs. TADs in tern cluster by their epigenomic type into A/B compartments and coalescence of compartments in the same chromosome forms chromosome territories <span class="citation">(<span class="citeproc-not-found" data-reference-id="Bonnev2016"><strong>???</strong></span>)</span>.</p>
</div>
</div>
<div id="dynamics-of-chromatin-structure" class="section level2">
<h2><span class="header-section-number">1.6</span> Dynamics of chromatin structure</h2>
<ul>
<li>Cell-type invariant TADs<br />
</li>
<li>Conservation of TADs</li>
</ul>
</div>
<div id="changes-of-genome-folding-in-evoltuion-and-disease-genomes" class="section level2">
<h2><span class="header-section-number">1.7</span> Changes of genome folding in evoltuion and disease genomes</h2>
<ul>
<li>Evolution of chromatin architecture</li>
<li>Disruption of chromatin architecture</li>
</ul>
</div>
<div id="aims-of-this-thesis" class="section level2">
<h2><span class="header-section-number">1.8</span> Aims of this thesis</h2>
<p>In this PhD thesis, I analyze TADs and chromatin interactions with respect to gene expression regulation. More specifically, we find out weather TADs represent only structural units of the genome or also functional building blocks, in which genes regulation is coordinated.</p>
<div id="is-the-three-dimensonal-folding-of-genomes-associated-with-co-regulation-of-functionally-related-genes" class="section level3 unnumbered">
<h3>Is the three-dimensonal folding of genomes associated with co-regulation of functionally related genes?</h3>
<ul>
<li>How are duplicated genes during evolution distributed in the three-dimensional genome architecture?</li>
<li>Can TADs provide regulatory environment for co-regulation of duplicated genes?</li>
<li>(Are TADs functional genomic units in which genes are co-regulated?)</li>
<li>(Are paralog genes co-regulated in the 3D chromatin architecture of genomes?)</li>
</ul>
</div>
<div id="are-tads-functional-buidling-blocks-of-genomes-and-subjected-to-selective-preasure-during-evolution" class="section level3 unnumbered">
<h3>Are TADs functional buidling blocks of genomes and subjected to selective preasure during evolution?</h3>
<ul>
<li>Are TADs conserved during evolution or disrupted by rearrangements?</li>
<li>Are changes of TADs during evolution associated with changes in gene expression profiles?</li>
<li>(Are TADs stable units that are often transmitted as a whole than disrupted by rearrangements?)</li>
</ul>
</div>
<div id="is-the-disruption-of-tads-by-rearrangements-also-associated-to-genetic-diseases" class="section level3 unnumbered">
<h3>Is the disruption of TADs by rearrangements also associated to genetic diseases?</h3>
<ul>
<li>Can TADs be used to interpret position effects of rearrangements in genetic diseases?</li>
<li>Can chromatin interaction data and TADs be integrated with phenotype data to predict pathomechanism of balanced chromosomal rearrangements?</li>
</ul>
</div>
<div id="can-chromatin-looping-interactions-be-predicted-by-chip-seq-and-sequence-features" class="section level3 unnumbered">
<h3>Can chromatin looping interactions be predicted by ChIP-seq and sequence features?</h3>
<ul>
<li>Are there signals form TF ChIP-seq data at chromatin looping anchors that predict long-range contacts?</li>
<li>Does the gnomic sequence encode features that are predictive for chromatin looping interactions?</li>
<li>Can we provide a computational method to predict chromatin looping interactions in specific cell-types and conditions of interest?</li>
</ul>
</div>
</div>
<div id="structure-of-this-thesis" class="section level2">
<h2><span class="header-section-number">1.9</span> Structure of this thesis</h2>
<p>These questions are addressed and discussed in the following for chapters. First, we focus on duplicated genes in the human genome. Because of their related sequence and function, shared evolutionary history, and close co-localization in the genome they represent an interesting model to study how genome folding is related to regulation of gene expression during evolution (Chapter <a href="ch-paralog-regulaion.html#ch:paralog_regulaion">Paralog genes</a>). Furthermore, we make use of deeply sequenced genomes of other vertebrates to systematically investigate whether TADs represent conserved building blocks of genomes and whether rearrangements are associated with altered gene expression programs (Chapter <a href="ch-TAD-evolution.html#ch:TAD_evolution">TAD evolution</a>). Next, we will address disruption of chromatin organization by analyzing disease associated rearrangement breakpoints from whole-genome sequenced patients of various genetic diseases to explain miss-regulation by disruption of TADs and chromatin contacts in diseases (Chapter <a href="ch-position-effect.html#ch:position_effect">Position effect</a>). Finally, we will make use of recent insights in chromatin loop formation to provide a computational tool to predict chromatin loops from largely available genomic data, with the aim to facilitate association of TF binding sites in enhancers to regulated genes in many cell-types and condition, for which Hi-C like data is not available (Chapter <a href="ch-loop-prediction.html#ch:loop_prediction">Loop prediction</a>).</p>

<!-- # Co-regulation of paralog genes in the three-dimensional chromatin architecture {#ch:paralog} -->
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="index.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="ch-paralog-regulaion.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"google": false,
"weibo": false,
"instapper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/ibn-salem/thesis/edit/master/01-intro.Rmd",
"text": "Edit"
},
"download": ["thesis.pdf", "thesis.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "";
    if (src === "" || src === "true") src = "https://cdn.bootcss.com/mathjax/2.7.1/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:" && /^https?:/.test(src))
      src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
