<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>Genome folding in evolution and disease</title>
  <meta name="description" content="A draft PhD thesis on genome folding in evoultion and disease.">
  <meta name="generator" content="bookdown 0.6.1 and GitBook 2.6.7">

  <meta property="og:title" content="Genome folding in evolution and disease" />
  <meta property="og:type" content="book" />
  <meta property="og:url" content="https://ibn-salem.github.io/thesis/" />
  
  <meta property="og:description" content="A draft PhD thesis on genome folding in evoultion and disease." />
  <meta name="github-repo" content="ibn-salem/thesis" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Genome folding in evolution and disease" />
  
  <meta name="twitter:description" content="A draft PhD thesis on genome folding in evoultion and disease." />
  

<meta name="author" content="Jonas Ibn-Salem">


<meta name="date" content="2018-03-02">

  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  
<link rel="prev" href="6-8-conclusions.html">
<link rel="next" href="A-sup-paralog-regulation.html">
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />










<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Thesis</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="1" data-path="1-intro.html"><a href="1-intro.html"><i class="fa fa-check"></i><b>1</b> Introduction</a><ul>
<li class="chapter" data-level="1.1" data-path="1-1-regulation-of-expression.html"><a href="1-1-regulation-of-expression.html"><i class="fa fa-check"></i><b>1.1</b> Regulation of gene expression</a></li>
<li class="chapter" data-level="1.2" data-path="1-2-distal-regulation-by-enhancers.html"><a href="1-2-distal-regulation-by-enhancers.html"><i class="fa fa-check"></i><b>1.2</b> Distal regulation by enhancers</a></li>
<li class="chapter" data-level="1.3" data-path="1-3-methods-to-probe-the-3d-chromatin-architecture.html"><a href="1-3-methods-to-probe-the-3d-chromatin-architecture.html"><i class="fa fa-check"></i><b>1.3</b> Methods to probe the 3D chromatin architecture</a><ul>
<li class="chapter" data-level="1.3.1" data-path="1-3-methods-to-probe-the-3d-chromatin-architecture.html"><a href="1-3-methods-to-probe-the-3d-chromatin-architecture.html#microscopy-based-techniques-to-visualize-the-genome-in-3d"><i class="fa fa-check"></i><b>1.3.1</b> Microscopy-based techniques to visualize the genome in 3D</a></li>
<li class="chapter" data-level="1.3.2" data-path="1-3-methods-to-probe-the-3d-chromatin-architecture.html"><a href="1-3-methods-to-probe-the-3d-chromatin-architecture.html#proximity-ligation"><i class="fa fa-check"></i><b>1.3.2</b> Proximity-ligation based method to quantify chromatin interactions</a></li>
</ul></li>
<li class="chapter" data-level="1.4" data-path="1-4-hierarchy-of-chromatin-3d-structure.html"><a href="1-4-hierarchy-of-chromatin-3d-structure.html"><i class="fa fa-check"></i><b>1.4</b> Hierarchy of chromatin 3D structure</a><ul>
<li class="chapter" data-level="1.4.1" data-path="1-4-hierarchy-of-chromatin-3d-structure.html"><a href="1-4-hierarchy-of-chromatin-3d-structure.html#chromosomal-territories-and-inter-chromosomal-contacts"><i class="fa fa-check"></i><b>1.4.1</b> Chromosomal territories and inter-chromosomal contacts</a></li>
<li class="chapter" data-level="1.4.2" data-path="1-4-hierarchy-of-chromatin-3d-structure.html"><a href="1-4-hierarchy-of-chromatin-3d-structure.html#ab-compartments"><i class="fa fa-check"></i><b>1.4.2</b> A/B compartments</a></li>
<li class="chapter" data-level="1.4.3" data-path="1-4-hierarchy-of-chromatin-3d-structure.html"><a href="1-4-hierarchy-of-chromatin-3d-structure.html#topologically-associating-domains-tads"><i class="fa fa-check"></i><b>1.4.3</b> Topologically associating domains (TADs)</a></li>
<li class="chapter" data-level="1.4.4" data-path="1-4-hierarchy-of-chromatin-3d-structure.html"><a href="1-4-hierarchy-of-chromatin-3d-structure.html#hierarchy-of-domain-structures-across-genomic-length-scales"><i class="fa fa-check"></i><b>1.4.4</b> Hierarchy of domain structures across genomic length scales</a></li>
<li class="chapter" data-level="1.4.5" data-path="1-4-hierarchy-of-chromatin-3d-structure.html"><a href="1-4-hierarchy-of-chromatin-3d-structure.html#chromatin-looping-interactions"><i class="fa fa-check"></i><b>1.4.5</b> Chromatin looping interactions</a></li>
<li class="chapter" data-level="1.4.6" data-path="1-4-hierarchy-of-chromatin-3d-structure.html"><a href="1-4-hierarchy-of-chromatin-3d-structure.html#loop-mechanism"><i class="fa fa-check"></i><b>1.4.6</b> TAD and loop formation by architectural proteins</a></li>
</ul></li>
<li class="chapter" data-level="1.5" data-path="1-5-dynamics-of-chromatin-structure.html"><a href="1-5-dynamics-of-chromatin-structure.html"><i class="fa fa-check"></i><b>1.5</b> Dynamics of chromatin structure</a><ul>
<li class="chapter" data-level="1.5.1" data-path="1-5-dynamics-of-chromatin-structure.html"><a href="1-5-dynamics-of-chromatin-structure.html#dynamics-across-the-cell-cycle"><i class="fa fa-check"></i><b>1.5.1</b> Dynamics across the cell cycle</a></li>
<li class="chapter" data-level="1.5.2" data-path="1-5-dynamics-of-chromatin-structure.html"><a href="1-5-dynamics-of-chromatin-structure.html#dynamics-across-cell-types-and-differntiation"><i class="fa fa-check"></i><b>1.5.2</b> Dynamics across cell types and differntiation</a></li>
</ul></li>
<li class="chapter" data-level="1.6" data-path="1-6-evolution-of-chromatin-organization.html"><a href="1-6-evolution-of-chromatin-organization.html"><i class="fa fa-check"></i><b>1.6</b> Evolution of chromatin organization</a></li>
<li class="chapter" data-level="1.7" data-path="1-7-disruption-of-chromatin-architecture-in-disease.html"><a href="1-7-disruption-of-chromatin-architecture-in-disease.html"><i class="fa fa-check"></i><b>1.7</b> Disruption of chromatin architecture in disease</a></li>
<li class="chapter" data-level="1.8" data-path="1-8-aims-of-this-thesis.html"><a href="1-8-aims-of-this-thesis.html"><i class="fa fa-check"></i><b>1.8</b> Aims of this thesis</a></li>
<li class="chapter" data-level="1.9" data-path="1-9-structure-of-this-thesis.html"><a href="1-9-structure-of-this-thesis.html"><i class="fa fa-check"></i><b>1.9</b> Structure of this thesis</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="2-paralog-regulation.html"><a href="2-paralog-regulation.html"><i class="fa fa-check"></i><b>2</b> Paralog genes in the 3D genome architecture</a><ul>
<li class="chapter" data-level="" data-path="2-paralog-regulation.html"><a href="2-paralog-regulation.html#preamble"><i class="fa fa-check"></i>Preamble</a></li>
<li class="chapter" data-level="" data-path="2-paralog-regulation.html"><a href="2-paralog-regulation.html#abstract-1"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="2.1" data-path="2-1-introduction.html"><a href="2-1-introduction.html"><i class="fa fa-check"></i><b>2.1</b> Introduction</a></li>
<li class="chapter" data-level="2.2" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html"><i class="fa fa-check"></i><b>2.2</b> Materials and methods</a><ul>
<li class="chapter" data-level="2.2.1" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html#selection-of-pairs-of-paralog-genes"><i class="fa fa-check"></i><b>2.2.1</b> Selection of pairs of paralog genes</a></li>
<li class="chapter" data-level="2.2.2" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html#enhancers-to-gene-association"><i class="fa fa-check"></i><b>2.2.2</b> Enhancers to gene association</a></li>
<li class="chapter" data-level="2.2.3" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html#topological-associating-domains"><i class="fa fa-check"></i><b>2.2.3</b> Topological associating domains</a></li>
<li class="chapter" data-level="2.2.4" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html#hi-c-interaction-maps"><i class="fa fa-check"></i><b>2.2.4</b> Hi-C interaction maps</a></li>
<li class="chapter" data-level="2.2.5" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html#randomization"><i class="fa fa-check"></i><b>2.2.5</b> Randomization</a></li>
<li class="chapter" data-level="2.2.6" data-path="2-2-sec-paralog-methods.html"><a href="2-2-sec-paralog-methods.html#statistical-tests"><i class="fa fa-check"></i><b>2.2.6</b> Statistical tests</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="2-3-results.html"><a href="2-3-results.html"><i class="fa fa-check"></i><b>2.3</b> Results</a><ul>
<li class="chapter" data-level="2.3.1" data-path="2-3-results.html"><a href="2-3-results.html#distribution-of-paralog-genes-in-the-human-genome"><i class="fa fa-check"></i><b>2.3.1</b> Distribution of paralog genes in the human genome</a></li>
<li class="chapter" data-level="2.3.2" data-path="2-3-results.html"><a href="2-3-results.html#co-expression-of-paralog-gene-pairs-across-tissues"><i class="fa fa-check"></i><b>2.3.2</b> Co-expression of paralog gene pairs across tissues</a></li>
<li class="chapter" data-level="2.3.3" data-path="2-3-results.html"><a href="2-3-results.html#paralog-genes-share-enhancers"><i class="fa fa-check"></i><b>2.3.3</b> Paralog genes share enhancers</a></li>
<li class="chapter" data-level="2.3.4" data-path="2-3-results.html"><a href="2-3-results.html#co-localization-of-paralogs-in-tads"><i class="fa fa-check"></i><b>2.3.4</b> Co-localization of paralogs in TADs</a></li>
<li class="chapter" data-level="2.3.5" data-path="2-3-results.html"><a href="2-3-results.html#distal-paralog-pairs-are-enriched-for-long-range-chromatin-contacts"><i class="fa fa-check"></i><b>2.3.5</b> Distal paralog pairs are enriched for long-range chromatin contacts</a></li>
<li class="chapter" data-level="2.3.6" data-path="2-3-results.html"><a href="2-3-results.html#close-paralogs-have-fewer-contacts-than-expected"><i class="fa fa-check"></i><b>2.3.6</b> Close paralogs have fewer contacts than expected</a></li>
<li class="chapter" data-level="2.3.7" data-path="2-3-results.html"><a href="2-3-results.html#paralogs-in-mouse-and-dog-genome"><i class="fa fa-check"></i><b>2.3.7</b> Paralogs in mouse and dog genome</a></li>
<li class="chapter" data-level="2.3.8" data-path="2-3-results.html"><a href="2-3-results.html#orthologs-of-human-paralogs-show-conserved-co-localization"><i class="fa fa-check"></i><b>2.3.8</b> Orthologs of human paralogs show conserved co-localization</a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="2-4-discussions.html"><a href="2-4-discussions.html"><i class="fa fa-check"></i><b>2.4</b> Discussions</a></li>
<li class="chapter" data-level="2.5" data-path="2-5-conclusion.html"><a href="2-5-conclusion.html"><i class="fa fa-check"></i><b>2.5</b> Conclusion</a></li>
<li class="chapter" data-level="" data-path="acknowledgements.html"><a href="acknowledgements.html"><i class="fa fa-check"></i>Acknowledgements</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="3-TAD-evolution.html"><a href="3-TAD-evolution.html"><i class="fa fa-check"></i><b>3</b> Stability of TADs in evolution</a><ul>
<li class="chapter" data-level="" data-path="3-TAD-evolution.html"><a href="3-TAD-evolution.html#preamble-1"><i class="fa fa-check"></i>Preamble</a></li>
<li class="chapter" data-level="" data-path="3-TAD-evolution.html"><a href="3-TAD-evolution.html#abstract-2"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="3.1" data-path="3-1-introduction-1.html"><a href="3-1-introduction-1.html"><i class="fa fa-check"></i><b>3.1</b> Introduction</a></li>
<li class="chapter" data-level="3.2" data-path="3-2-results-1.html"><a href="3-2-results-1.html"><i class="fa fa-check"></i><b>3.2</b> Results</a><ul>
<li class="chapter" data-level="3.2.1" data-path="3-2-results-1.html"><a href="3-2-results-1.html#identification-of-evolutionary-rearrangement-breakpoints-from-whole-genome-alignments"><i class="fa fa-check"></i><b>3.2.1</b> Identification of evolutionary rearrangement breakpoints from whole-genome alignments</a></li>
<li class="chapter" data-level="3.2.2" data-path="3-2-results-1.html"><a href="3-2-results-1.html#rearrangement-breakpoints-are-enriched-at-tad-boundaries"><i class="fa fa-check"></i><b>3.2.2</b> Rearrangement breakpoints are enriched at TAD boundaries</a></li>
<li class="chapter" data-level="3.2.3" data-path="3-2-results-1.html"><a href="3-2-results-1.html#clusters-of-conserved-non-coding-elements-are-depleted-for-rearrangement-breakpoints"><i class="fa fa-check"></i><b>3.2.3</b> Clusters of conserved non-coding elements are depleted for rearrangement breakpoints</a></li>
<li class="chapter" data-level="3.2.4" data-path="3-2-results-1.html"><a href="3-2-results-1.html#rearranged-tads-are-associated-with-divergent-gene-expression-between-species"><i class="fa fa-check"></i><b>3.2.4</b> Rearranged TADs are associated with divergent gene expression between species</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="3-3-discussion.html"><a href="3-3-discussion.html"><i class="fa fa-check"></i><b>3.3</b> Discussion</a></li>
<li class="chapter" data-level="3.4" data-path="3-4-conclusion-1.html"><a href="3-4-conclusion-1.html"><i class="fa fa-check"></i><b>3.4</b> Conclusion</a></li>
<li class="chapter" data-level="3.5" data-path="3-5-methods.html"><a href="3-5-methods.html"><i class="fa fa-check"></i><b>3.5</b> Methods</a><ul>
<li class="chapter" data-level="3.5.1" data-path="3-5-methods.html"><a href="3-5-methods.html#rearrangement-breakpoints-from-whole-genome-alignments"><i class="fa fa-check"></i><b>3.5.1</b> Rearrangement breakpoints from whole-genome alignments</a></li>
<li class="chapter" data-level="3.5.2" data-path="3-5-methods.html"><a href="3-5-methods.html#topologically-associating-domains-and-contact-domains"><i class="fa fa-check"></i><b>3.5.2</b> Topologically associating domains and contact domains</a></li>
<li class="chapter" data-level="3.5.3" data-path="3-5-methods.html"><a href="3-5-methods.html#breakpoint-distributions-at-tads"><i class="fa fa-check"></i><b>3.5.3</b> Breakpoint distributions at TADs</a></li>
<li class="chapter" data-level="3.5.4" data-path="3-5-methods.html"><a href="3-5-methods.html#quantification-of-breakpoint-enrichment"><i class="fa fa-check"></i><b>3.5.4</b> Quantification of breakpoint enrichment</a></li>
<li class="chapter" data-level="3.5.5" data-path="3-5-methods.html"><a href="3-5-methods.html#expression-data-for-mouse-and-human-orthologs"><i class="fa fa-check"></i><b>3.5.5</b> Expression data for mouse and human orthologs</a></li>
<li class="chapter" data-level="3.5.6" data-path="3-5-methods.html"><a href="3-5-methods.html#classification-of-tads-and-genes-according-to-rearrangements-and-grbs"><i class="fa fa-check"></i><b>3.5.6</b> Classification of TADs and genes according to rearrangements and GRBs</a></li>
<li class="chapter" data-level="3.5.7" data-path="3-5-methods.html"><a href="3-5-methods.html#source-code-and-implementation-details"><i class="fa fa-check"></i><b>3.5.7</b> Source code and implementation details</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="declarations.html"><a href="declarations.html"><i class="fa fa-check"></i>Declarations</a><ul>
<li class="chapter" data-level="" data-path="declarations.html"><a href="declarations.html#availability-of-data-and-material"><i class="fa fa-check"></i>Availability of data and material</a></li>
<li class="chapter" data-level="" data-path="declarations.html"><a href="declarations.html#competing-interests"><i class="fa fa-check"></i>Competing interests</a></li>
<li class="chapter" data-level="" data-path="declarations.html"><a href="declarations.html#authors-contributions"><i class="fa fa-check"></i>Authors‘ contributions</a></li>
<li class="chapter" data-level="" data-path="declarations.html"><a href="declarations.html#acknowledgments"><i class="fa fa-check"></i>Acknowledgments</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="4-position-effect.html"><a href="4-position-effect.html"><i class="fa fa-check"></i><b>4</b> Position effects of rearrangements in disease genomes</a><ul>
<li class="chapter" data-level="" data-path="4-position-effect.html"><a href="4-position-effect.html#preamble-2"><i class="fa fa-check"></i>Preamble</a></li>
<li class="chapter" data-level="" data-path="4-position-effect.html"><a href="4-position-effect.html#abstract-3"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="4.1" data-path="4-1-introduction-2.html"><a href="4-1-introduction-2.html"><i class="fa fa-check"></i><b>4.1</b> Introduction</a></li>
<li class="chapter" data-level="4.2" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html"><i class="fa fa-check"></i><b>4.2</b> Materials and Methods</a><ul>
<li class="chapter" data-level="4.2.1" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html#selection-of-subjects-with-apparently-balanced-chromosome-abnormalities"><i class="fa fa-check"></i><b>4.2.1</b> Selection of subjects with apparently balanced chromosome abnormalities</a></li>
<li class="chapter" data-level="4.2.2" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html#clinical-descriptions-of-dgap-cases"><i class="fa fa-check"></i><b>4.2.2</b> Clinical descriptions of DGAP cases</a></li>
<li class="chapter" data-level="4.2.3" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html#analysis-of-genes-bordering-the-rearrangement-breakpoints"><i class="fa fa-check"></i><b>4.2.3</b> Analysis of genes bordering the rearrangement breakpoints</a></li>
<li class="chapter" data-level="4.2.4" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html#assessment-of-disrupted-functional-elements-and-chromatin-interactions-bordering-rearrangement-breakpoints"><i class="fa fa-check"></i><b>4.2.4</b> Assessment of disrupted functional elements and chromatin interactions bordering rearrangement breakpoints</a></li>
<li class="chapter" data-level="4.2.5" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html#ontological-analysis-of-genes-neighboring-breakpoints"><i class="fa fa-check"></i><b>4.2.5</b> Ontological analysis of genes neighboring breakpoints</a></li>
<li class="chapter" data-level="4.2.6" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html#quantitative-real-time-pcr"><i class="fa fa-check"></i><b>4.2.6</b> Quantitative real-time PCR</a></li>
<li class="chapter" data-level="4.2.7" data-path="4-2-materials-and-methods.html"><a href="4-2-materials-and-methods.html#assessment-of-dgap-breakpoints-overlapping-with-non-coding-structural-variants-in-public-databases"><i class="fa fa-check"></i><b>4.2.7</b> Assessment of DGAP breakpoints overlapping with non-coding structural variants in public databases</a></li>
</ul></li>
<li class="chapter" data-level="4.3" data-path="4-3-results-2.html"><a href="4-3-results-2.html"><i class="fa fa-check"></i><b>4.3</b> Results</a><ul>
<li class="chapter" data-level="4.3.1" data-path="4-3-results-2.html"><a href="4-3-results-2.html#genomic-characterization-of-non-coding-breakpoints"><i class="fa fa-check"></i><b>4.3.1</b> Genomic characterization of non-coding breakpoints</a></li>
<li class="chapter" data-level="4.3.2" data-path="4-3-results-2.html"><a href="4-3-results-2.html#identification-of-genes-with-potential-position-effects"><i class="fa fa-check"></i><b>4.3.2</b> Identification of genes with potential position effects</a></li>
<li class="chapter" data-level="4.3.3" data-path="4-3-results-2.html"><a href="4-3-results-2.html#identification-of-subjects-with-shared-non-coding-chromosome-alterations-and-phenotypes"><i class="fa fa-check"></i><b>4.3.3</b> Identification of subjects with shared non-coding chromosome alterations and phenotypes</a></li>
</ul></li>
<li class="chapter" data-level="4.4" data-path="4-4-discussion-1.html"><a href="4-4-discussion-1.html"><i class="fa fa-check"></i><b>4.4</b> Discussion</a></li>
<li class="chapter" data-level="" data-path="declarations-1.html"><a href="declarations-1.html"><i class="fa fa-check"></i>Declarations</a><ul>
<li class="chapter" data-level="" data-path="declarations-1.html"><a href="declarations-1.html#acknowledgements-1"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="" data-path="declarations-1.html"><a href="declarations-1.html#web-resources"><i class="fa fa-check"></i>Web Resources</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="5-loop.html"><a href="5-loop.html"><i class="fa fa-check"></i><b>5</b> Prediction of chromatin looping interactions</a><ul>
<li class="chapter" data-level="" data-path="5-loop.html"><a href="5-loop.html#preamble-3"><i class="fa fa-check"></i>Preamble</a></li>
<li class="chapter" data-level="" data-path="5-loop.html"><a href="5-loop.html#abstract-4"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="5.1" data-path="5-1-introduction-3.html"><a href="5-1-introduction-3.html"><i class="fa fa-check"></i><b>5.1</b> Introduction</a></li>
<li class="chapter" data-level="5.2" data-path="5-2-results-3.html"><a href="5-2-results-3.html"><i class="fa fa-check"></i><b>5.2</b> Results</a><ul>
<li class="chapter" data-level="5.2.1" data-path="5-2-results-3.html"><a href="5-2-results-3.html#ctcf-motif-pairs-as-candidate-chromatin-loop-anchors"><i class="fa fa-check"></i><b>5.2.1</b> CTCF motif pairs as candidate chromatin loop anchors</a></li>
<li class="chapter" data-level="5.2.2" data-path="5-2-results-3.html"><a href="5-2-results-3.html#similarity-of-chip-seq-signals-at-looping-ctcf-motifs"><i class="fa fa-check"></i><b>5.2.2</b> Similarity of ChIP-seq signals at looping CTCF motifs</a></li>
<li class="chapter" data-level="5.2.3" data-path="5-2-results-3.html"><a href="5-2-results-3.html#genomic-sequence-features-of-ctcf-motif-pairs-are-associated-with-looping"><i class="fa fa-check"></i><b>5.2.3</b> Genomic sequence features of CTCF motif pairs are associated with looping</a></li>
<li class="chapter" data-level="5.2.4" data-path="5-2-results-3.html"><a href="5-2-results-3.html#chromatin-loop-prediction-using-7c"><i class="fa fa-check"></i><b>5.2.4</b> Chromatin loop prediction using 7C</a></li>
<li class="chapter" data-level="5.2.5" data-path="5-2-results-3.html"><a href="5-2-results-3.html#prediction-performance-evaluation"><i class="fa fa-check"></i><b>5.2.5</b> Prediction performance evaluation</a></li>
<li class="chapter" data-level="5.2.6" data-path="5-2-results-3.html"><a href="5-2-results-3.html#prediction-performance-of-7c-with-sequence-features-and-single-tf-chip-seq-data-sets"><i class="fa fa-check"></i><b>5.2.6</b> Prediction performance of 7C with sequence features and single TF ChIP-seq data sets</a></li>
<li class="chapter" data-level="5.2.7" data-path="5-2-results-3.html"><a href="5-2-results-3.html#comparison-of-transcription-factors-by-prediction-performance"><i class="fa fa-check"></i><b>5.2.7</b> Comparison of transcription factors by prediction performance</a></li>
<li class="chapter" data-level="5.2.8" data-path="5-2-results-3.html"><a href="5-2-results-3.html#prediction-performance-in-other-cell-types-and-for-different-tfs"><i class="fa fa-check"></i><b>5.2.8</b> Prediction performance in other cell types and for different TFs</a></li>
<li class="chapter" data-level="5.2.9" data-path="5-2-results-3.html"><a href="5-2-results-3.html#the-high-resolution-of-chip-nexus-improves-prediction-performance"><i class="fa fa-check"></i><b>5.2.9</b> The high resolution of ChIP-nexus improves prediction performance</a></li>
</ul></li>
<li class="chapter" data-level="5.3" data-path="5-3-discussion-2.html"><a href="5-3-discussion-2.html"><i class="fa fa-check"></i><b>5.3</b> Discussion</a></li>
<li class="chapter" data-level="5.4" data-path="5-4-conclusion-2.html"><a href="5-4-conclusion-2.html"><i class="fa fa-check"></i><b>5.4</b> Conclusion</a></li>
<li class="chapter" data-level="5.5" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html"><i class="fa fa-check"></i><b>5.5</b> Methods</a><ul>
<li class="chapter" data-level="5.5.1" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#ctcf-motifs-in-the-human-genome"><i class="fa fa-check"></i><b>5.5.1</b> CTCF motifs in the human genome</a></li>
<li class="chapter" data-level="5.5.2" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#loop-interaction-data-for-training-and-validation"><i class="fa fa-check"></i><b>5.5.2</b> Loop interaction data for training and validation</a></li>
<li class="chapter" data-level="5.5.3" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#chip-seq-datasets-in-gm12878-cells"><i class="fa fa-check"></i><b>5.5.3</b> ChIP-seq datasets in GM12878 cells</a></li>
<li class="chapter" data-level="5.5.4" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#chip-seq-data-types"><i class="fa fa-check"></i><b>5.5.4</b> ChIP-seq data types</a></li>
<li class="chapter" data-level="5.5.5" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#chip-nexus-data-processing-for-rad21-and-scm3"><i class="fa fa-check"></i><b>5.5.5</b> ChIP-nexus data processing for RAD21 and SCM3</a></li>
<li class="chapter" data-level="5.5.6" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#similarity-of-chip-seq-profiles-as-correlation-of-coverage-around-motifs"><i class="fa fa-check"></i><b>5.5.6</b> Similarity of ChIP-seq profiles as correlation of coverage around motifs</a></li>
<li class="chapter" data-level="5.5.7" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#genomic-sequence-features-of-chromatin-loops"><i class="fa fa-check"></i><b>5.5.7</b> Genomic sequence features of chromatin loops</a></li>
<li class="chapter" data-level="5.5.8" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#chromatin-loop-prediction-model"><i class="fa fa-check"></i><b>5.5.8</b> Chromatin Loop prediction model</a></li>
<li class="chapter" data-level="5.5.9" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#training-and-validation-of-prediction-model"><i class="fa fa-check"></i><b>5.5.9</b> Training and validation of prediction model</a></li>
<li class="chapter" data-level="5.5.10" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#analysis-of-prediction-performance"><i class="fa fa-check"></i><b>5.5.10</b> Analysis of prediction performance</a></li>
<li class="chapter" data-level="5.5.11" data-path="5-5-methods-1.html"><a href="5-5-methods-1.html#implementation-of-7c-and-compatibility-to-other-tools"><i class="fa fa-check"></i><b>5.5.11</b> Implementation of 7C and compatibility to other tools</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="declarations-2.html"><a href="declarations-2.html"><i class="fa fa-check"></i>Declarations</a><ul>
<li class="chapter" data-level="" data-path="declarations-2.html"><a href="declarations-2.html#availability-of-data-and-material-1"><i class="fa fa-check"></i>Availability of data and material</a></li>
<li class="chapter" data-level="" data-path="declarations-2.html"><a href="declarations-2.html#competing-interests-1"><i class="fa fa-check"></i>Competing interests</a></li>
<li class="chapter" data-level="" data-path="declarations-2.html"><a href="declarations-2.html#authors-contributions-1"><i class="fa fa-check"></i>Authors‘ contributions</a></li>
<li class="chapter" data-level="" data-path="declarations-2.html"><a href="declarations-2.html#acknowledgments-1"><i class="fa fa-check"></i>Acknowledgments</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="6-thesis-discussion.html"><a href="6-thesis-discussion.html"><i class="fa fa-check"></i><b>6</b> Discussion</a><ul>
<li class="chapter" data-level="6.1" data-path="6-1-co-regulation-of-functionally-related-genes-in-tads.html"><a href="6-1-co-regulation-of-functionally-related-genes-in-tads.html"><i class="fa fa-check"></i><b>6.1</b> Co-regulation of functionally related genes in TADs</a></li>
<li class="chapter" data-level="6.2" data-path="6-2-evolution-by-gene-duplications-and-altered-regulatory-environments.html"><a href="6-2-evolution-by-gene-duplications-and-altered-regulatory-environments.html"><i class="fa fa-check"></i><b>6.2</b> Evolution by gene duplications and altered regulatory environments</a></li>
<li class="chapter" data-level="6.3" data-path="6-3-tads-are-stable-across-large-evolutionary-time-scales.html"><a href="6-3-tads-are-stable-across-large-evolutionary-time-scales.html"><i class="fa fa-check"></i><b>6.3</b> TADs are stable across large evolutionary time-scales</a></li>
<li class="chapter" data-level="6.4" data-path="6-4-TAD-disease.html"><a href="6-4-TAD-disease.html"><i class="fa fa-check"></i><b>6.4</b> Gene expression changes by altered TADs in disease</a></li>
<li class="chapter" data-level="6.5" data-path="6-5-towards-predicting-regulatory-pathomechanisms-of-structural-variants.html"><a href="6-5-towards-predicting-regulatory-pathomechanisms-of-structural-variants.html"><i class="fa fa-check"></i><b>6.5</b> Towards predicting regulatory pathomechanisms of structural variants</a></li>
<li class="chapter" data-level="6.6" data-path="6-6-towards-studying-genome-folding-in-specific-conditions.html"><a href="6-6-towards-studying-genome-folding-in-specific-conditions.html"><i class="fa fa-check"></i><b>6.6</b> Towards studying genome folding in specific conditions</a><ul>
<li class="chapter" data-level="6.6.1" data-path="6-6-towards-studying-genome-folding-in-specific-conditions.html"><a href="6-6-towards-studying-genome-folding-in-specific-conditions.html#interpretation-of-non-coding-variants-by-interacting-genes"><i class="fa fa-check"></i><b>6.6.1</b> Interpretation of non-coding variants by interacting genes</a></li>
<li class="chapter" data-level="6.6.2" data-path="6-6-towards-studying-genome-folding-in-specific-conditions.html"><a href="6-6-towards-studying-genome-folding-in-specific-conditions.html#cell-type-specific-regulatory-interactions-between-enhancers-and-genes"><i class="fa fa-check"></i><b>6.6.2</b> Cell type-specific regulatory interactions between enhancers and genes</a></li>
<li class="chapter" data-level="6.6.3" data-path="6-6-towards-studying-genome-folding-in-specific-conditions.html"><a href="6-6-towards-studying-genome-folding-in-specific-conditions.html#variability-of-long-range-interactions-across-individuals"><i class="fa fa-check"></i><b>6.6.3</b> Variability of long-range interactions across individuals</a></li>
<li class="chapter" data-level="6.6.4" data-path="6-6-towards-studying-genome-folding-in-specific-conditions.html"><a href="6-6-towards-studying-genome-folding-in-specific-conditions.html#single-cell"><i class="fa fa-check"></i><b>6.6.4</b> Genome folding in single cells</a></li>
<li class="chapter" data-level="6.6.5" data-path="6-6-towards-studying-genome-folding-in-specific-conditions.html"><a href="6-6-towards-studying-genome-folding-in-specific-conditions.html#constantly-improving-targeted-experimental-methods."><i class="fa fa-check"></i><b>6.6.5</b> Constantly improving targeted experimental methods.</a></li>
</ul></li>
<li class="chapter" data-level="6.7" data-path="6-7-molecular-mechanisms-driving-genome-folding.html"><a href="6-7-molecular-mechanisms-driving-genome-folding.html"><i class="fa fa-check"></i><b>6.7</b> Molecular mechanisms driving genome folding</a></li>
<li class="chapter" data-level="6.8" data-path="6-8-conclusions.html"><a href="6-8-conclusions.html"><i class="fa fa-check"></i><b>6.8</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="appendix"><span><b>Appendix</b></span></li>
<li class="chapter" data-level="A" data-path="A-sup-paralog-regulation.html"><a href="A-sup-paralog-regulation.html"><i class="fa fa-check"></i><b>A</b> Supporting Information: Co-regulation of paralog genes</a></li>
<li class="chapter" data-level="B" data-path="B-sup-tad-evolution.html"><a href="B-sup-tad-evolution.html"><i class="fa fa-check"></i><b>B</b> Supplementary Data: Stability of TADs in evolution</a><ul>
<li class="chapter" data-level="B.1" data-path="B-1-TadEvoSupTab.html"><a href="B-1-TadEvoSupTab.html"><i class="fa fa-check"></i><b>B.1</b> Supplementary Tables</a></li>
<li class="chapter" data-level="B.2" data-path="B-2-supplementary-figures.html"><a href="B-2-supplementary-figures.html"><i class="fa fa-check"></i><b>B.2</b> Supplementary Figures</a></li>
</ul></li>
<li class="chapter" data-level="C" data-path="C-sup-position-effect.html"><a href="C-sup-position-effect.html"><i class="fa fa-check"></i><b>C</b> Supplemental Data: Position effects of rearrangements in disease genomes</a><ul>
<li class="chapter" data-level="C.1" data-path="C-1-supplemental-note.html"><a href="C-1-supplemental-note.html"><i class="fa fa-check"></i><b>C.1</b> Supplemental Note</a><ul>
<li class="chapter" data-level="C.1.1" data-path="C-1-supplemental-note.html"><a href="C-1-supplemental-note.html#case-reports"><i class="fa fa-check"></i><b>C.1.1</b> Case Reports</a></li>
<li class="chapter" data-level="C.1.2" data-path="C-1-supplemental-note.html"><a href="C-1-supplemental-note.html#nucleotide-level-nomenclature-for-dgap-karyotypes"><i class="fa fa-check"></i><b>C.1.2</b> Nucleotide Level Nomenclature for DGAP karyotypes</a></li>
</ul></li>
<li class="chapter" data-level="C.2" data-path="C-2-supplemental-figure.html"><a href="C-2-supplemental-figure.html"><i class="fa fa-check"></i><b>C.2</b> Supplemental Figure</a></li>
<li class="chapter" data-level="C.3" data-path="C-3-position-effect-sub-tab.html"><a href="C-3-position-effect-sub-tab.html"><i class="fa fa-check"></i><b>C.3</b> Supplemental Table Legends</a></li>
</ul></li>
<li class="chapter" data-level="D" data-path="D-loop-support.html"><a href="D-loop-support.html"><i class="fa fa-check"></i><b>D</b> Supplemental Information: Prediction of chromatin looping interactions</a><ul>
<li class="chapter" data-level="D.1" data-path="D-1-LoopPredSupTab.html"><a href="D-1-LoopPredSupTab.html"><i class="fa fa-check"></i><b>D.1</b> Supplementary Tables</a></li>
<li class="chapter" data-level="D.2" data-path="D-2-supplementary-figures-1.html"><a href="D-2-supplementary-figures-1.html"><i class="fa fa-check"></i><b>D.2</b> Supplementary Figures</a></li>
</ul></li>
<li class="chapter" data-level="E" data-path="E-contribution-to-individual-publications.html"><a href="E-contribution-to-individual-publications.html"><i class="fa fa-check"></i><b>E</b> Contribution to individual publications</a></li>
<li class="chapter" data-level="" data-path="zusammenfassung.html"><a href="zusammenfassung.html"><i class="fa fa-check"></i>Zusammenfassung</a></li>
<li class="chapter" data-level="" data-path="curriculum-vitae.html"><a href="curriculum-vitae.html"><i class="fa fa-check"></i>Curriculum vitae</a></li>
<li class="chapter" data-level="" data-path="acknowledgements-2.html"><a href="acknowledgements-2.html"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Build with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Genome folding in evolution and disease</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="references" class="section level1 unnumbered">
<h1>References</h1>
<div id="refs">
<div id="ref-Acemel2017">
<p>Acemel RD, Maeso I, Gómez‐Skarmeta JL. 2017. Topologically associated domains: A successful scaffold for the evolution of gene regulation in animals. <em>Wiley Interdisciplinary Reviews: Developmental Biology</em> 1–19.</p>
</div>
<div id="ref-Achinger-Kawecka2017">
<p>Achinger-Kawecka J, Taberlay PC, Clark SJ. 2016. Alterations in three-dimensional organization of the cancer genome and epigenome. <em>Cold Spring Harbor Symposia on Quantitative Biology</em> <strong>LXXXI</strong>: 031013.</p>
</div>
<div id="ref-Allen2015">
<p>Allen BL, Taatjes DJ. 2015. The mediator complex: A central integrator of transcription. <em>Nature Reviews Molecular Cell Biology</em> <strong>16</strong>: 155–166.</p>
</div>
<div id="ref-Andersson2014">
<p>Andersson R, Gebhard C, Miguel-Escalada I, Hoof I, Bornholdt J, Boyd M, Chen Y, Zhao X, Schmidl C, Suzuki T, et al. 2014. An atlas of active enhancers across human cell types and tissues. <em>Nature</em> <strong>507</strong>: 455–61.</p>
</div>
<div id="ref-Andrey2017">
<p>Andrey G, Mundlos S. 2017. The three-dimensional genome: Regulating gene expression during pluripotency and development. <em>Development</em> <strong>144</strong>: 3646–3658.</p>
</div>
<div id="ref-Anger2014">
<p>Anger GJ, Crocker S, McKenzie K, Brown KK, Morton CC, Harrison K, MacKenzie JJ. 2014. X-linked deafness-2 (dfnx2) phenotype associated with a paracentric inversion upstream of &lt;i&gt;POU3F4&lt;/i&gt;. <em>American Journal of Audiology</em> <strong>23</strong>: 1.</p>
</div>
<div id="ref-GTExConsortium2015">
<p>Ardlie KG, DeLuca DS, Segrè AV, Sullivan TJ, Young TR, Gelfand ET, Trowbridge CA, Maller JB, Tukiainen T, Lek M, et al. 2015. The genotype-tissue expression (gtex) pilot analysis: Multitissue gene regulation in humans. <em>Science</em> <strong>348</strong>: 648–660.</p>
</div>
<div id="ref-Auton2015">
<p>Auton A, Abecasis GR, Altshuler DM, Durbin RM, Abecasis GR, Bentley DR, Chakravarti A, Clark AG, Donnelly P, Eichler EE, et al. 2015. A global reference for human genetic variation. <em>Nature</em> <strong>526</strong>: 68–74.</p>
</div>
<div id="ref-Ay2015">
<p>Ay F, Noble WS. 2015. Analysis methods for studying the 3D architecture of the genome. <em>Genome Biology</em> <strong>16</strong>: 183.</p>
</div>
<div id="ref-Bailey2015">
<p>Bailey SD, Zhang X, Desai K, Aid M, Corradin O, Cowper-Sal Lari R, Akhtar-Zaidi B, Scacheri PC, Haibe-Kains B, Lupien M, et al. 2015. ZNF143 provides sequence specificity to secure chromatin interactions at gene promoters. <em>Nature Communications</em> <strong>2</strong>: 6186.</p>
</div>
<div id="ref-Banerji1981">
<p>Banerji J, Rusconi S, Schaffner W. 1981. Expression of a <span class="math inline">\(\beta\)</span>-globin gene is enhanced by remote sv40 dna sequences. <em>Cell</em> <strong>27</strong>: 299–308.</p>
</div>
<div id="ref-Bantignies2011">
<p>Bantignies F, Roure V, Comet I, Leblanc B, Schuettengruber B, Bonnet J, Tixier V, Mas A, Cavalli G. 2011. Polycomb-dependent regulatory contacts between distant hox loci in drosophila. <em>Cell</em> <strong>144</strong>: 214–26.</p>
</div>
<div id="ref-Barrington2017">
<p>Barrington C, Finn R, Hadjur S. 2017. Cohesin biology meets the loop extrusion model. <em>Chromosome Research</em> 1–10.</p>
</div>
<div id="ref-Beagrie2017">
<p>Beagrie RA, Scialdone A, Schueler M, Kraemer DCA, Chotalia M, Xie SQ, Barbieri M, Santiago I de, Lavitas L-M, Branco MR, et al. 2017. Complex multi-enhancer contacts captured by genome architecture mapping. <em>Nature</em>.</p>
</div>
<div id="ref-Becker2015">
<p>Becker M, Mah N, Zdzieblo D, Li X, Mer A, Andrade-navarro MA, Mu AM. 2015. Epigenetic mechanisms in cellular reprogramming. In <em>Epigenetics and human health</em> (eds. A. Meissner and J. Walter), <em>Epigenetics and human health</em>, pp. pp 141–166, Springer Berlin Heidelberg, Berlin, Heidelberg.</p>
</div>
<div id="ref-Berlivet2013">
<p>Berlivet S, Paquette D, Dumouchel A, Langlais D, Dostie J, Kmita M. 2013. Clustering of tissue-specific sub-tads accompanies the regulation of hoxa genes in developing limbs. <em>PLoS Genetics</em> <strong>9</strong>.</p>
</div>
<div id="ref-Berthelot2017">
<p>Berthelot C, Villar D, Horvath JE, Odom DT, Flicek P. 2017. Complexity and conservation of regulatory landscapes underlie evolutionary resilience of mammalian gene expression. <em>Nature Ecology &amp; Evolution</em> 1–31.</p>
</div>
<div id="ref-Bonev2016">
<p>Bonev B, Cavalli G. 2016. Organization and function of the 3D genome. <em>Nature Reviews Genetics</em> <strong>17</strong>: 661–678.</p>
</div>
<div id="ref-Bonev2017">
<p>Bonev B, Cohen NM, Szabo Q, Hugnot J-p, Tanay A, Cavalli G, Bonev B, Cohen NM, Szabo Q, Fritsch L, et al. 2017. Multiscale 3D genome rewiring during mouse article multiscale 3D genome rewiring during mouse neural development. <em>Cell</em> <strong>171</strong>: 557.e1–557.e24.</p>
</div>
<div id="ref-Boutanaev2002">
<p>Boutanaev a M, Kalmykova a I, Shevelyov YY, Nurminsky DI, Smith M, Evolution T, Masterpiece T, Helm C, Selection N. 2002. Large clusters of co-expressed genes in the drosophila genome. <em>Nature</em> <strong>420</strong>: 666–669.</p>
</div>
<div id="ref-Bouwman2015">
<p>Bouwman BA, Laat W de. 2015. Getting the genome in shape: The formation of loops, domains and compartments. <em>Genome Biology</em> <strong>16</strong>: 154.</p>
</div>
<div id="ref-Brackley2016">
<p>Brackley CA, Brown JM, Waithe D, Babbs C, Davies J, Hughes JR, Buckle VJ, Marenduzzo D. 2016. Predicting the three-dimensional folding of cis -regulatory regions in mammalian genomes using bioinformatic data and polymer models. <em>Genome Biology</em> 31–36.</p>
</div>
<div id="ref-Branco2006">
<p>Branco MR, Pombo A. 2006. Intermingling of chromosome territories in interphase suggests role in translocations and transcription-dependent associations. <em>PLoS Biology</em> <strong>4</strong>: 780–788.</p>
</div>
<div id="ref-Brookes2015">
<p>Brookes AJ, Robinson PN. 2015. Human genotype–phenotype databases: Aims, challenges and opportunities. <em>Nature Reviews Genetics</em>.</p>
</div>
<div id="ref-Buenrostro2013">
<p>Buenrostro JD, Giresi PG, Zaba LC, Chang HY, Greenleaf WJ. 2013. Transposition of native chromatin for fast and sensitive epigenomic profiling of open chromatin, dna-binding proteins and nucleosome position. <em>Nature Methods</em> <strong>10</strong>: 1213–1218.</p>
</div>
<div id="ref-Buenrostro2015">
<p>Buenrostro JD, Wu B, Litzenburger UM, Ruff D, Gonzales ML, Snyder MP, Chang HY, Greenleaf WJ. 2015. Single-cell chromatin accessibility reveals principles of regulatory variation. <em>Nature</em> <strong>523</strong>: 486–90.</p>
</div>
<div id="ref-Cabili2011">
<p>Cabili MN, Trapnell C, Goff L, Koziol M, Tazon-Vega B, Regev A, Rinn JL. 2011. Integrative annotation of human large intergenic noncoding rnas reveals global properties and specific subclasses. <em>Genes &amp; development</em> <strong>25</strong>: 1915–27.</p>
</div>
<div id="ref-Cai2003">
<p>Cai J, Goodman BK, Patel AS, Mulliken JB, Van Maldergem L, Hoganson GE, Paznekas WA, Ben-Neriah Z, Sheffer R, Cunningham ML, et al. 2003. Increased risk for developmental delay in saethre-chotzen syndrome is associated with twist deletions: An improved strategy for twist mutation screening. <em>Human Genetics</em> <strong>114</strong>: 68–76.</p>
</div>
<div id="ref-Canela2017">
<p>Canela A, Maman Y, Jung S, Wong N, Callen E, Day A, Kieffer-Kwon K-R, Pekowska A, Zhang H, Rao SS, et al. 2017. Genome organization drives chromosome fragility. <em>Cell</em> 1–15.</p>
</div>
<div id="ref-Cardoso2016">
<p>Cardoso AR, Oliveira M, Amorim A, Azevedo L. 2016. Major influence of repetitive elements on disease-associated copy number variants (cnvs). <em>Human Genomics</em> <strong>10</strong>: 30.</p>
</div>
<div id="ref-Carroll2008">
<p>Carroll SB. 2008. Evo-devo and an expanding evolutionary synthesis: A genetic theory of morphological evolution. <em>Cell</em> <strong>134</strong>: 25–36.</p>
</div>
<div id="ref-Carter2002">
<p>Carter D, Chakalova L, Osborne CS, Dai YF, Fraser P. 2002. Long-range chromatin regulatory interactions in vivo. <em>Nat Genet</em> <strong>32</strong>: 623–626.</p>
</div>
<div id="ref-Carvalho2016">
<p>Carvalho CMB, Lupski JR. 2016. Mechanisms underlying structural variant formation in genomic disorders. <em>Nature Reviews Genetics</em> <strong>17</strong>: 224–238.</p>
</div>
<div id="ref-Chandra2015">
<p>Chandra T, Ewels PA, Schoenfelder S, Furlan-Magaril M, Wingett SW, Kirschner K, Thuret JY, Andrews S, Fraser P, Reik W. 2015. Global reorganization of the nuclear landscape in senescent cells. <em>Cell Reports</em> <strong>10</strong>: 471–484.</p>
</div>
<div id="ref-Chen2016">
<p>Chen Y, Wang Y, Xuan Z, Chen M, Zhang MQ. 2016. De novo deciphering three-dimensional chromatin interaction and topological domains by wavelet transformation of epigenetic profiles. <em>Nucleic Acids Research</em> <strong>44</strong>: e106–e106.</p>
</div>
<div id="ref-Chubb2002">
<p>Chubb JR, Boyle S, Perry P, Bickmore WA. 2002. Chromatin motion is constrained by association with nuclear compartments in human cells. <em>Current Biology</em> <strong>12</strong>: 439–445.</p>
</div>
<div id="ref-Claussnitzer2015">
<p>Claussnitzer M, Dankel SN, Kim K-H, Quon G, Meuleman W, Haugen C, Glunk V, Sousa IS, Beaudry JL, Puviindran V, et al. 2015. &lt;i&gt;FTO&lt;/i&gt; obesity variant circuitry and adipocyte browning in humans. <em>New England Journal of Medicine</em> <strong>373</strong>: 895–907.</p>
</div>
<div id="ref-Conrad2006">
<p>Conrad DF, Andrews TD, Carter NP, Hurles ME, Pritchard JK. 2006. A high-resolution survey of deletion polymorphism in the human genome. <em>Nature Genetics</em> <strong>38</strong>: 75–81.</p>
</div>
<div id="ref-Conrad2010">
<p>Conrad DF, Pinto D, Redon R, Feuk L, Gokcumen O, Zhang Y, Aerts J, Andrews TD, Barnes C, Campbell P, et al. 2010. Origins and functional impact of copy number variation in the human genome. <em>Nature</em> <strong>464</strong>: 704–12.</p>
</div>
<div id="ref-Crane2015">
<p>Crane E, Bian Q, McCord RP, Lajoie BR, Wheeler BS, Ralston EJ, Uzawa S, Dekker J, Meyer BJ. 2015. Condensin-driven remodelling of x chromosome topology during dosage compensation. <em>Nature</em>.</p>
</div>
<div id="ref-Cremer2001">
<p>Cremer T, Cremer C. 2001. Chromosome territories, nuclear architecture and gene regulation in mammalian cells. <em>Nature reviews genetics</em> <strong>2</strong>: 292–301.</p>
</div>
<div id="ref-Cremer2010">
<p>Cremer T, Cremer M. 2010. Chromosome territories. <em>Cold Spring Harbor perspectives in biology</em> <strong>2</strong>: a003889.</p>
</div>
<div id="ref-Cremer2015">
<p>Cremer T, Cremer M, Hübner B, Strickfaden H, Smeets D, Popken J, Sterr M, Markaki Y, Rippe K, Cremer C. 2015. The 4D nucleome: Evidence for a dynamic nuclear landscape based on co-aligned active and inactive nuclear compartments. <em>FEBS Letters</em>.</p>
</div>
<div id="ref-Creyghton2010">
<p>Creyghton MP, Cheng AW, Welstead GG, Kooistra T, Carey BW, Steine EJ, Hanna J, Lodato MA, Frampton GM, Sharp PA, et al. 2010. Histone h3k27ac separates active from poised enhancers and predicts developmental state. <em>Proceedings of the National Academy of Sciences of the United States of America</em> <strong>107</strong>: 21931–6.</p>
</div>
<div id="ref-Criscione2016">
<p>Criscione SW, De Cecco M, Siranosian B, Zhang Y, Kreiling JA, Sedivy JM, Neretti N. 2016. Reorganization of chromosome architecture in replicative cellular senescence. <em>Science advances</em> <strong>2</strong>: e1500882.</p>
</div>
<div id="ref-Dali2017">
<p>Dali R, Blanchette M. 2016. A critical assessment of topologically associating domain prediction tools. <em>Nucleic Acids Research</em> <strong>45</strong>: 2994–3005.</p>
</div>
<div id="ref-Davies2017">
<p>Davies JOJ, Oudelaar AM, Higgs DR, Hughes JR. 2017. How best to identify chromosomal interactions : A comparison of approaches. <em>Nature Publishing Group</em> <strong>14</strong>.</p>
</div>
<div id="ref-Davies2016">
<p>Davies JOJ, Telenius JM, McGowan SJ, Roberts NA, Taylor S, Higgs DR, Hughes JR. 2016. Multiplexed analysis of chromosome conformation at vastly improved sensitivity. <em>Nature Methods</em> <strong>13</strong>: 74–80.</p>
</div>
<div id="ref-Davis2017">
<p>Davis AP, Grondin CJ, Johnson RJ, Sciaky D, King BL, McMorran R, Wiegers J, Wiegers TC, Mattingly CJ. 2017. The comparative toxicogenomics database: Update 2017. <em>Nucleic Acids Research</em> <strong>45</strong>: D972–D978.</p>
</div>
<div id="ref-Dekker2014">
<p>Dekker J. 2014. Two ways to fold the genome during the cell cycle: Insights obtained with chromosome conformation capture. <em>Epigenetics &amp; Chromatin</em> <strong>7</strong>: 25.</p>
</div>
<div id="ref-Dekker2017a">
<p>Dekker J, Belmont AS, Guttman M, Leshyk VO, Lis JT, Lomvardas S, Mirny LA, O’Shea CC, Park PJ, Ren B, et al. 2017. The 4D nucleome project. <em>bioRxiv</em> <strong>549</strong>: 103499.</p>
</div>
<div id="ref-Dekker2015">
<p>Dekker J, Heard E. 2015. Structural and functional diversity of topologically associating domains. <em>FEBS Letters</em>.</p>
</div>
<div id="ref-Dekker2013">
<p>Dekker J, Marti-Renom MA, Mirny LA. 2013. Exploring the three-dimensional organization of genomes: Interpreting chromatin interaction data. <em>Nature reviews Genetics</em> <strong>14</strong>: 390–403.</p>
</div>
<div id="ref-Dekker2016">
<p>Dekker J, Mirny L. 2016. The 3D genome as moderator of chromosomal communication. <em>Cell</em> <strong>164</strong>: 1110–1121.</p>
</div>
<div id="ref-Dekker2002">
<p>Dekker J, Rippe K, Dekker M, Kleckner N. 2002. Capturing chromosome conformation. <em>Science (New York, NY)</em> <strong>295</strong>: 1306–11.</p>
</div>
<div id="ref-Deng2012">
<p>Deng W, Lee J, Wang H, Miller J, Reik A, Gregory PD, Dean A, Blobel G a. 2012. Controlling long-range genomic interactions at a native locus by targeted tethering of a looping factor. <em>Cell</em> <strong>149</strong>: 1233–44.</p>
</div>
<div id="ref-Deng2015">
<p>Deng Y, Gao L, Wang B, Guo X. 2015. HPOSim: An r package for phenotypic similarity measure and enrichment analysis based on the human phenotype ontology. <em>PLoS ONE</em> <strong>10</strong>: 1–12.</p>
</div>
<div id="ref-Denker2016">
<p>Denker A, Laat W de. 2016. The second decade of 3C technologies: Detailed insights into nuclear organization. <em>Genes &amp; development</em> <strong>30</strong>: 1357–82.</p>
</div>
<div id="ref-DiPierro2017">
<p>Di Pierro M, Cheng RR, Lieberman Aiden E, Wolynes PG, Onuchic JN. 2017. De novo prediction of human chromosome structures: Epigenetic marking patterns encode genome architecture. <em>Proceedings of the National Academy of Sciences of the United States of America</em> <strong>114</strong>: 12126–12131.</p>
</div>
<div id="ref-Dickel2018">
<p>Dickel DE, Ypsilanti AR, Pla R, Zhu Y, Barozzi I, Mannion BJ, Khin YS, Fukuda-Yuzawa Y, Plajzer-Frick I, Pickle CS, et al. 2018. Ultraconserved enhancers are required for normal development. <em>Cell</em> <strong>172</strong>: 491–499.e15.</p>
</div>
<div id="ref-Dileep2015">
<p>Dileep V, Ay F, Sima J, Vera DL, Noble WS, Gilbert DM. 2015. Topologically associating domains and their long-range contacts are established during early g1 coincident with the establishment of the replication-timing program. <em>Genome research</em> <strong>25</strong>: 1104–13.</p>
</div>
<div id="ref-Dimitrieva2013">
<p>Dimitrieva S, Bucher P. 2013. UCNEbase—a database of ultraconserved non-coding elements and genomic regulatory blocks. <em>Nucleic Acids Research</em> <strong>41</strong>: D101–D109.</p>
</div>
<div id="ref-Dixon2016">
<p>Dixon JR, Gorkin DU, Ren B. 2016. Chromatin domains: The unit of chromosome organization. <em>Molecular Cell</em> <strong>62</strong>: 668–680.</p>
</div>
<div id="ref-Dixon2015">
<p>Dixon JR, Jung I, Selvaraj S, Shen Y, Antosiewicz-Bourget JE, Lee AY, Ye Z, Kim A, Rajagopal N, Xie W, et al. 2015. Chromatin architecture reorganization during stem cell differentiation. <em>Nature</em> <strong>518</strong>: 331–336.</p>
</div>
<div id="ref-Dixon2012">
<p>Dixon JR, Selvaraj S, Yue F, Kim A, Li Y, Shen Y, Hu M, Liu JS, Ren B. 2012. Topological domains in mammalian genomes identified by analysis of chromatin interactions. <em>Nature</em> <strong>485</strong>: 376–380.</p>
</div>
<div id="ref-Djebali2012">
<p>Djebali S, Davis Ca, Merkel A, Dobin A, Lassmann T, Mortazavi A, Tanzer A, Lagarde J, Lin W, Schlesinger F, et al. 2012. Landscape of transcription in human cells. <em>Nature</em> <strong>489</strong>: 101–108.</p>
</div>
<div id="ref-Dostie2006">
<p>Dostie J, Richmond TA, Arnaout RA, Selzer RR, Lee WL, Honan TA, Rubio ED, Krumm A, Lamb J, Nusbaum C, et al. 2006. Chromosome conformation capture carbon copy (5C): A massively parallel solution for mapping interactions between genomic elements. <em>Genome Research</em> <strong>16</strong>: 1299–1309.</p>
</div>
<div id="ref-Dowen2014">
<p>Dowen JM, Fan ZP, Hnisz D, Ren G, Abraham BJ, Zhang LN, Weintraub AS, Schuijers J, Lee TI, Zhao K, et al. 2014. Control of cell identity genes occurs in insulated neighborhoods in mammalian chromosomes. <em>Cell</em> <strong>159</strong>: 374–387.</p>
</div>
<div id="ref-Dryden2014">
<p>Dryden NH, Broome LR, Dudbridge F, Johnson N, Orr N, Schoenfelder S, Nagano T, Andrews S, Wingett S, Kozarewa I, et al. 2014. Unbiased analysis of potential targets of breast cancer susceptibility loci by capture hi-c. <em>Genome research</em> 1854–1868.</p>
</div>
<div id="ref-Dunham2012">
<p>Dunham I, Kundaje A, Aldred SF, Collins PJ, Davis C a, Doyle F, Epstein CB, Frietze S, Harrow J, Kaul R, et al. 2012. An integrated encyclopedia of dna elements in the human genome. <em>Nature</em> <strong>489</strong>: 57–74.</p>
</div>
<div id="ref-Durand2016">
<p>Durand NC, Shamim MS, Machol I, Rao SS, Huntley MH, Lander ES, Aiden EL. 2016. Juicer provides a one-click system for analyzing loop-resolution hi-c experiments. <em>Cell Systems</em> <strong>3</strong>: 95–98.</p>
</div>
<div id="ref-Durinck2005">
<p>Durinck S, Moreau Y, Kasprzyk A, Davis S, De Moor B, Brazma A, Huber W. 2005. BioMart and bioconductor: A powerful link between biological databases and microarray data analysis. <em>Bioinformatics</em> <strong>21</strong>: 3439–3440.</p>
</div>
<div id="ref-Durinck2009">
<p>Durinck S, Spellman PT, Birney E, Huber W. 2009a. Mapping identifiers for the integration of genomic datasets with the r/bioconductor package biomaRt. <em>Nature Protocols</em> <strong>4</strong>: 1184–1191.</p>
</div>
<div id="ref-Durinck2009b">
<p>Durinck S, Spellman PT, Birney E, Huber W. 2009b. Mapping identifiers for the integration of genomic datasets with the r/bioconductor package biomaRt. <em>Nature protocols</em> <strong>4</strong>: 1184–1191.</p>
</div>
<div id="ref-Dzida2017">
<p>Dzida T, Iqbal M, Charapitsa I, Reid G, Stunnenberg H, Matarese F, Grote K, Honkela A, Rattray M. 2017. Predicting stimulation-dependent enhancer-promoter interactions from chip-seq time course data. <em>PeerJ</em> <strong>5</strong>: e3742.</p>
</div>
<div id="ref-Eagen2015">
<p>Eagen KP, Hartl TA, Kornberg RD. 2015. Stable chromosome condensation revealed by chromosome conformation capture. <em>Cell</em> <strong>163</strong>: 934–946.</p>
</div>
<div id="ref-Eisenberg2013">
<p>Eisenberg E, Levanon EY. 2013. Human housekeeping genes, revisited. <em>Trends in Genetics</em> <strong>29</strong>: 569–574.</p>
</div>
<div id="ref-Engstrom2007">
<p>Engström PG, Sui SJH, Drivenes Ø, Becker TS, Lenhard B. 2007. Genomic regulatory blocks underlie extensive microsynteny conservation in insects. <em>Genome Research</em> <strong>17</strong>: 1898–1908.</p>
</div>
<div id="ref-Falk2018">
<p>Falk M, Feodorova Y, Naumova N, Imakaev M, Lajoie BR, Leonhardt H, Joffe B, Dekker J, Fudenberg G, Solovei I, et al. 2018. Heterochromatin drives organization of conventional and inverted nuclei. <em>bioRxiv</em> 244038.</p>
</div>
<div id="ref-Fantes1995">
<p>Fantes J, Redeker B, Breen M, Boyle S, Brown J, Fletcher J, Jones S, Bickmore W, Fukushima Y, Mannens M, et al. 1995. Aniridia-associated cytogenetic rearrangements suggest that a position effect may cause the mutant phenotype. <em>Human Molecular Genetics</em> <strong>4</strong>: 415–422.</p>
</div>
<div id="ref-Farre2015">
<p>Farré M, Robinson TJ, Ruiz-Herrera A. 2015. An integrative breakage model of genome architecture, reshuffling and evolution. <em>BioEssays</em> n/a–n/a.</p>
</div>
<div id="ref-Filippova2014">
<p>Filippova D, Patro R, Duggal G, Kingsford C. 2014. Identification of alternative topological domains in chromatin. <em>Algorithms for molecular biology : AMB</em> <strong>9</strong>: 14.</p>
</div>
<div id="ref-Firth2009">
<p>Firth HV, Richards SM, Bevan a P, Clayton S, Corpas M, Rajan D, Van Vooren S, Moreau Y, Pettett RM, Carter NP. 2009. DECIPHER: Database of chromosomal imbalance and phenotype in humans using ensembl resources. <em>American journal of human genetics</em> <strong>84</strong>: 524–33.</p>
</div>
<div id="ref-Fishilevich2017">
<p>Fishilevich S, Nudel R, Rappaport N, Hadar R, Plaschkes I, Iny Stein T, Rosen N, Kohn A, Twik M, Safran M, et al. 2017. GeneHancer: Genome-wide integration of enhancers and target genes in genecards. <em>Database</em> <strong>2017</strong>: 1665–1680.</p>
</div>
<div id="ref-Flavahan2016">
<p>Flavahan W a, Drier Y, Liau BB, Gillespie SM, Venteicher a S, Stemmer-Rachamimov a O, Suva ML, Bernstein BE. 2016. Insulator dysfunction and oncogene activation in idh mutant gliomas. <em>Nature</em> <strong>529</strong>: 110–114.</p>
</div>
<div id="ref-Flicek2014">
<p>Flicek P, Amode MR, Barrell D, Beal K, Billis K, Brent S, Carvalho-Silva D, Clapham P, Coates G, Fitzgerald S, et al. 2014. Ensembl 2014. <em>Nucleic Acids Research</em> <strong>42</strong>: D749–D755.</p>
</div>
<div id="ref-Flomen1998">
<p>Flomen RH, Vatcheva R, Gorman PA, Baptista PR, Groet J, Barišić I, Ligutic I, Nižetić D. 1998. Construction and analysis of a sequence-ready map in 4q25: Rieger syndrome can be caused by haploinsufficiency ofRIEG, but also by chromosome breaks ≈90 kb upstream of this gene. <em>Genomics</em> <strong>47</strong>: 409–413.</p>
</div>
<div id="ref-Flottmann2017">
<p>Flöttmann R, Kragesteen BK, Geuer S, Socha M, Allou L, Sowińska-Seidler A, Bosquillon de Jarcy L, Wagner J, Jamsheer A, Oehl-Jaschkowitz B, et al. 2017. Noncoding copy-number variations are associated with congenital limb malformation. <em>Genet Med</em> <strong>00</strong>: 1–9.</p>
</div>
<div id="ref-Flyamer2017">
<p>Flyamer IM, Gassler J, Imakaev M, Ulyanov SV, Abdennur N, Razin SV, Mirny L, Tachibana-Konwalski K. 2017. Single-cell hi-c reveals unique chromatin reorganization at oocyte-tozygote transition. <em>Nature Publishing Group</em> <strong>544</strong>: 1–17.</p>
</div>
<div id="ref-Forcato2017">
<p>Forcato M, Nicoletti C, Pal K, Livi CM, Ferrari F, Bicciato S. 2017. Comparison of computational methods for hi-c data analysis. <em>Nature Methods</em> 14–19.</p>
</div>
<div id="ref-Ford1959">
<p>Ford C, Jones K, Polani P, De Almeida J, Briggs J. 1959. A sex-chromosome anomaly in a case of gonadal dysgenesis (turner’S syndrome). <em>The Lancet</em> <strong>273</strong>: 711–713.</p>
</div>
<div id="ref-Forrest2014">
<p>Forrest ARR, Kawaji H, Rehli M, Baillie JK, Hoon MJL de, Lassmann T, Itoh M, Summers KM, Suzuki H, Daub CO, et al. 2014. A promoter-level mammalian expression atlas. <em>Nature</em> <strong>507</strong>: 462–70.</p>
</div>
<div id="ref-Fortin2015">
<p>Fortin J-P, Hansen KD. 2015. Reconstructing a/b compartments as revealed by hi-c using long-range correlations in epigenetic data. <em>Genome Biology</em> <strong>16</strong>: 180.</p>
</div>
<div id="ref-Franke2017">
<p>Franke M. 2017. The role of higher-order chromatin organization at the sox9 locus in gene regulation and disease dissertation. PhD thesis, Freien Universität Berlin.</p>
</div>
<div id="ref-Franke2016">
<p>Franke M, Ibrahim DM, Andrey G, Schwarzer W, Heinrich V, Schöpflin R, Kraft K, Kempfer R, Jerković I, Chan W-L, et al. 2016. Formation of new chromatin domains determines pathogenicity of genomic duplications. <em>Nature</em> 1–15.</p>
</div>
<div id="ref-Fraser2015">
<p>Fraser J, Ferrai C, Chiariello AM, Schueler M, Rito T, Laudanno G, Barbieri M, Moore BL, Kraemer DC, Aitken S, et al. 2015. Hierarchical folding and reorganization of chromosomes are linked to transcriptional changes in cellular differentiation. <em>Mol Syst Biol</em> <strong>11</strong>: 1–14.</p>
</div>
<div id="ref-Frizell1998">
<p>Frizell ER, Sutphen1 R, Diamond Jr. FB, Sherwood M, Overhauser J. 1998. T(1;18)(q32.1;q22.1) associated with genitourinary malformations. <em>Clinical Genetics</em> <strong>54</strong>: 330–333.</p>
</div>
<div id="ref-Fudenberg2018">
<p>Fudenberg G, Abdennur N, Imakaev M, Goloborodko A, Mirny L. 2018. Emerging evidence of chromosome folding by loop extrusion. <em>bioRxiv</em> 264648.</p>
</div>
<div id="ref-Fudenberg2016">
<p>Fudenberg G, Imakaev M, Lu C, Goloborodko A, Abdennur N, Mirny LA. 2016. Formation of chromosomal domains by loop extrusion. <em>Cell Reports</em> <strong>15</strong>: 2038–2049.</p>
</div>
<div id="ref-Fullwood2009">
<p>Fullwood MJ, Liu MH, Pan YF, Liu J, Xu H, Mohamed YB, Orlov YL, Velkov S, Ho A, Mei PH, et al. 2009. An oestrogen-receptor-<span class="math inline">\(\alpha\)</span>-bound human chromatin interactome. <em>Nature</em> <strong>462</strong>: 58–64.</p>
</div>
<div id="ref-Galil1986">
<p>Galil Z. 1986. Efficient algorithms for finding maximum matching in graphs. <em>ACM Computing Surveys</em> <strong>18</strong>: 23–38.</p>
</div>
<div id="ref-Gerber2018">
<p>Gerber S, Fournier D, Hewel C, Horenko I. 2018. Imputation of posterior linkage probability relations reveals a significant influence of structural 3D constraints on linkage disequilibrium. <em>bioRxiv</em> 255315.</p>
</div>
<div id="ref-Ghavi-helm2014">
<p>Ghavi-Helm Y, Klein F a, Pakozdi T, Ciglar L, Noordermeer D, Huber W, Furlong EEM. 2014. Enhancer loops appear stable during development and are associated with paused polymerase. <em>Nature</em> <strong>512</strong>: 96–100.</p>
</div>
<div id="ref-Gibcus2013">
<p>Gibcus JH, Dekker J. 2013. The hierarchy of the 3D genome. <em>Molecular cell</em> <strong>49</strong>: 773–82.</p>
</div>
<div id="ref-Giorgio2015">
<p>Giorgio E, Robyr D, Spielmann M, Ferrero E, Di Gregorio E, Imperiale D, Vaula G, Stamoulis G, Santoni F, Atzori C, et al. 2015. A large genomic deletion leads to enhancer adoption by the lamin b1 gene: A second path to autosomal dominant adult-onset demyelinating leukodystrophy (adld). <em>Human Molecular Genetics</em> <strong>24</strong>: 3143–3154.</p>
</div>
<div id="ref-Gomez-Diaz2014">
<p>Gómez-Díaz E, Corces VG. 2014. Architectural proteins: Regulators of 3D genome organization in cell fate. <em>Trends in cell biology</em> <strong>32</strong>: 1–9.</p>
</div>
<div id="ref-Gomez-Marin2015">
<p>Gómez-Marín C, Tena JJ, Acemel RD, López-Mayorga M, Naranjo S, Calle-Mustienes E de la, Maeso I, Beccari L, Aneas I, Vielmas E, et al. 2015. Evolutionary comparison reveals that diverging ctcf sites are signatures of ancestral topological associating domains borders. <em>Proceedings of the National Academy of Sciences</em> <strong>112</strong>: 201505463.</p>
</div>
<div id="ref-Groschel2014">
<p>Gröschel S, Sanders MA, Hoogenboezem R, Wit E de, Bouwman BAM, Erpelinck C, Velden VHJ van der, Havermans M, Avellino R, Lom K van, et al. 2014. A single oncogenic enhancer rearrangement causes concomitant evi1 and gata2 deregulation in leukemia. <em>Cell</em> <strong>157</strong>: 369–381.</p>
</div>
<div id="ref-Grubert2015">
<p>Grubert F, Zaugg JB, Kasowski M, Ursu O, Spacek DV, Martin AR, Greenside P, Srivas R, Phanstiel DH, Pekowska A, et al. 2015. Genetic control of chromatin states in humans involves local and distal chromosomal interactions. <em>Cell</em> 1–15.</p>
</div>
<div id="ref-Gu2008">
<p>Gu W, Zhang F, Lupski JR. 2008. Mechanisms for human genomic rearrangements. <em>PathoGenetics</em> <strong>1</strong>: 4.</p>
</div>
<div id="ref-Guo2015">
<p>Guo Y, Xu Q, Canzio D, Shou J, Li J, Gorkin DU, Jung I, Wu H, Zhai Y, Tang Y, et al. 2015. CRISPR inversion of ctcf sites alters genome topology and enhancer/promoter function. <em>Cell</em> <strong>162</strong>: 900–910.</p>
</div>
<div id="ref-Hansen2016">
<p>Hansen P, Hecht J, Ibn-Salem J, Menkuec BS, Roskosch S, Truss M, Robinson PN. 2016. Q-nexus: A comprehensive and efficient analysis pipeline designed for chip-nexus. <em>BMC Genomics</em> <strong>17</strong>: 873.</p>
</div>
<div id="ref-Hansen2015">
<p>Hansen P, Hecht J, Ibrahim DM, Krannich A, Truss M, Robinson PN. 2015. Saturation analysis of chip-seq data for reproducible identification of binding peaks. <em>Genome research</em> <strong>25</strong>: 1391–400.</p>
</div>
<div id="ref-Harmston2015">
<p>Harmston N, Ing-Simmons E, Perry M, Barešić A, Lenhard B. 2015. GenomicInteractions: An r/bioconductor package for manipulating and investigating chromatin interaction data. <em>BMC Genomics</em> <strong>16</strong>: 963.</p>
</div>
<div id="ref-Harmston2017">
<p>Harmston N, Ing-Simmons E, Tan G, Perry M, Merkenschlager M, Lenhard B. 2017. Topologically associating domains are ancient features that coincide with metazoan clusters of extreme noncoding conservation. <em>Nature Communications</em> <strong>8</strong>: 441.</p>
</div>
<div id="ref-Hay2016">
<p>Hay D, Hughes JR, Babbs C, Davies JOJ, Graham BJ, Hanssen LLP, Kassouf MT, Oudelaar AM, Sharpe JA, Suciu MC, et al. 2016. Genetic dissection of the <span class="math inline">\(\alpha\)</span>-globin super-enhancer in vivo. <em>Nature Genetics</em> <strong>48</strong>: 895–903.</p>
</div>
<div id="ref-He2015a">
<p>He Q, Johnston J, Zeitlinger J. 2015. ChIP-nexus enables improved detection of in vivo transcription factor binding footprints. <em>Nature Biotechnology</em> <strong>33</strong>: 395–401.</p>
</div>
<div id="ref-He2005">
<p>He X, Zhang J. 2005. Gene complexity and gene duplicability. <em>Current Biology</em> <strong>15</strong>: 1016–1021.</p>
</div>
<div id="ref-Heger2013">
<p>Heger A, Webber C, Goodson M, Ponting CP, Lunter G. 2013. GAT: A simulation framework for testing the association of genomic intervals. <em>Bioinformatics (Oxford, England)</em> <strong>29</strong>: 2046–8.</p>
</div>
<div id="ref-Heger2012">
<p>Heger P, Marin B, Bartkuhn M, Schierenberg E, Wiehe T. 2012. The chromatin insulator ctcf and the emergence of metazoan diversity. <em>Proceedings of the National Academy of Sciences</em> <strong>109</strong>: 17507–17512.</p>
</div>
<div id="ref-Heidari2014">
<p>Heidari N, Phanstiel DHD, He C, Grubert F, Jahanbani F, Kasowski M, Zhang MQ, Snyder MP. 2014. Genome-wide map of regulatory interactions in the human genome. <em>Genome research</em> <strong>24</strong>: 1905–17.</p>
</div>
<div id="ref-Herrero2016">
<p>Herrero J, Muffato M, Beal K, Fitzgerald S, Gordon L, Pignatelli M, Vilella AJ, Searle SMJ, Amode R, Brent S, et al. 2016. Ensembl comparative genomics resources. <em>Database</em> <strong>2016</strong>: bav096.</p>
</div>
<div id="ref-Higgins2008">
<p>Higgins AW, Alkuraya FS, Bosco AF, Brown KK, Bruns GA, Donovan DJ, Eisenman R, Fan Y, Farra CG, Ferguson HL, et al. 2008. Characterization of apparently balanced chromosomal rearrangements from the developmental genome anatomy project. <em>American Journal of Human Genetics</em> <strong>82</strong>: 712–722.</p>
</div>
<div id="ref-Hindorff2009">
<p>Hindorff LA, Sethupathy P, Junkins HA, Ramos EM, Mehta JP, Collins FS, Manolio TA. 2009. Potential etiologic and functional implications of genome-wide association loci for human diseases and traits. <em>Proceedings of the National Academy of Sciences of the United States of America</em> <strong>106</strong>: 9362–7.</p>
</div>
<div id="ref-Hinds2006">
<p>Hinds DA, Kloek AP, Jen M, Chen X, Frazer KA. 2006. Common deletions and snps are in linkage disequilibrium in the human genome. <em>Nature Genetics</em> <strong>38</strong>: 82–85.</p>
</div>
<div id="ref-Hinrichs2006">
<p>Hinrichs AS, Karolchik D, Baertsch R, Barber GP, Bejerano G, Clawson H, Diekhans M, Furey TS, Harte RA, Hsu F, et al. 2006. The ucsc genome browser database: Update 2006. <em>Nucleic Acids Res</em> <strong>34</strong>: D590—–D598.</p>
</div>
<div id="ref-Hinsch2006">
<p>Hinsch H, Hannenhalli S. 2006. Recurring genomic breaks in independent lineages support genomic fragility. <em>BMC evolutionary biology</em> <strong>6</strong>: 90.</p>
</div>
<div id="ref-Hnisz2016a">
<p>Hnisz D, Day DS, Young RA. 2016a. Insulated neighborhoods: Structural and functional units of mammalian gene control. <em>Cell</em> <strong>167</strong>: 1188–1200.</p>
</div>
<div id="ref-Hnisz2016">
<p>Hnisz D, Weintraub AS, Day DS, Valton A-l, Bak RO, Li CH, Goldmann J, Lajoie BR, Fan ZP, Sigova AA, et al. 2016b. Activation of proto-oncogenes by disruption of chromosome neighborhoods. <em>Science (New York, NY)</em> <strong>351</strong>: 1454–8.</p>
</div>
<div id="ref-Hoffman2015">
<p>Hoffman EA, Frey BL, Smith LM, Auble DT. 2015. Formaldehyde crosslinking: A tool for the study of chromatin complexes. <em>The Journal of biological chemistry</em> <strong>290</strong>: 26404–11.</p>
</div>
<div id="ref-Hou2012">
<p>Hou C, Li L, Qin ZS, Corces VG. 2012. Gene density, transcription, and insulators contribute to the partition of the drosophila genome into physical domains. <em>Molecular Cell</em> <strong>48</strong>: 471–484.</p>
</div>
<div id="ref-Hsieh2015">
<p>Hsieh T-HS, Weiner A, Lajoie B, Dekker J, Friedman N, Rando OJ. 2015. Mapping nucleosome resolution chromosome folding in yeast by micro-c. <em>Cell</em> 1–12.</p>
</div>
<div id="ref-Huang2010">
<p>Huang N, Lee I, Marcotte EM, Hurles ME. 2010. Characterising and predicting haploinsufficiency in the human genome ed. M.H. Schierup. <em>PLoS Genetics</em> <strong>6</strong>: e1001154.</p>
</div>
<div id="ref-Huber2015">
<p>Huber W, Carey VJ, Gentleman R, Anders S, Carlson M, Carvalho BS, Bravo HC, Davis S, Gatto L, Girke T, et al. 2015. Orchestrating high-throughput genomic analysis with bioconductor. <em>Nature Methods</em> <strong>12</strong>: 115–121.</p>
</div>
<div id="ref-Hughes2014">
<p>Hughes JR, Roberts N, McGowan S, Hay D, Giannoulatou E, Lynch M, De Gobbi M, Taylor S, Gibbons R, Higgs DR. 2014. Analysis of hundreds of cis-regulatory landscapes at high resolution in a single, high-throughput experiment. <em>Nature genetics</em> <strong>46</strong>: 205–212.</p>
</div>
<div id="ref-Huminiecki2004">
<p>Huminiecki L. 2004. Divergence of spatial gene expression profiles following species-specific gene duplications in human and mouse. <em>Genome Research</em> <strong>14</strong>: 1870–1879.</p>
</div>
<div id="ref-Iafrate2004">
<p>Iafrate AJ, Feuk L, Rivera MN, Listewnik ML, Donahoe PK, Qi Y, Scherer SW, Lee C. 2004. Detection of large-scale variation in the human genome. <em>Nature Genetics</em> <strong>36</strong>: 949–951.</p>
</div>
<div id="ref-Ibn-Salem2014">
<p>Ibn-Salem J, Köhler S, Love MI, Chung H-R, Huang N, Hurles ME, Haendel M, Washington NL, Smedley D, Mungall CJ, et al. 2014. Deletions of chromosomal regulatory boundaries are associated with congenital disease. <em>Genome Biology</em> <strong>15</strong>: 423.</p>
</div>
<div id="ref-Ibn-Salem2017">
<p>Ibn-Salem J, Muro EM, Andrade-Navarro MA. 2017. Co-regulation of paralog genes in the three-dimensional chromatin architecture. <em>Nucleic Acids Research</em> <strong>45</strong>: 81–91.</p>
</div>
<div id="ref-Ing-simmons2015">
<p>Ing-Simmons E, Seitan VC, Faure AJ, Flicek P, Carroll T, Dekker J, Fisher AG, Lenhard B, Merkenschlager M. 2015. Spatial enhancer clustering and regulation of enhancer-proximal genes by cohesin. <em>Genome research</em> <strong>25</strong>: 504–513.</p>
</div>
<div id="ref-Innan2010">
<p>Innan H, Kondrashov F. 2010. The evolution of gene duplications: Classifying and distinguishing between models. <em>Nature reviews Genetics</em> <strong>11</strong>: 97–108.</p>
</div>
<div id="ref-Altshuler2010">
<p>International HapMap 3 Consortium, Altshuler DM, Gibbs RA, Peltonen L, Altshuler DM, Gibbs RA, Peltonen L, Dermitzakis E, Schaffner SF, Yu F, et al. 2010. Integrating common and rare genetic variation in diverse human populations. <em>Nature</em> <strong>467</strong>: 52–8.</p>
</div>
<div id="ref-Jacobs1959">
<p>Jacobs PA, Strong JA. 1959. A case of human intersexuality having a possible xxy sex-determining mechanism. <em>Nature</em> <strong>183</strong>: 302–303.</p>
</div>
<div id="ref-Javierre2016">
<p>Javierre BM, Burren OS, Wilder SP, Kreuzhuber R, Hill SM, Sewitz S, Cairns J, Wingett SW, Várnai C, Thiecke MJ, et al. 2016. Lineage-specific genome architecture links enhancers and non-coding disease variants to target gene promoters. <em>Cell</em> <strong>167</strong>: 1369–1384.e19.</p>
</div>
<div id="ref-Jenkins1975">
<p>Jenkins EC, Curcuru-Giordano FM, Krishna SG, Cantarella J. 1975. De novo occurrence of 46,XX,t(4;13) (q31;q14) in a mentally retarded girl. <em>Annales de genetique</em> <strong>18</strong>: 117–20.</p>
</div>
<div id="ref-Ji2016">
<p>Ji X, Dadon DB, Powell BE, Fan ZP, Borges-Rivera D, Shachar S, Weintraub AS, Hnisz D, Pegoraro G, Lee TI, et al. 2016. 3D chromosome regulatory landscape of human pluripotent cells. <em>Cell Stem Cell</em> <strong>18</strong>: 262–275.</p>
</div>
<div id="ref-Joshi2015">
<p>Joshi O, Wang S-Y, Kuznetsova T, Atlasi Y, Peng T, Fabre PJ, Habibi E, Shaik J, Saeed S, Handoko L, et al. 2015. Dynamic reorganization of extremely long-range promoter-promoter interactions between two states of pluripotency. <em>Cell Stem Cell</em> <strong>17</strong>: 748–757.</p>
</div>
<div id="ref-Kalhor2011">
<p>Kalhor R, Tjong H, Jayathilaka N, Alber F, Chen L. 2011. Genome architectures revealed by tethered chromosome conformation capture and population-based modeling. <em>Nature Biotechnology</em> <strong>30</strong>: 90–98.</p>
</div>
<div id="ref-Katainen2015">
<p>Katainen R, Dave K, Pitkänen E, Palin K, Kivioja T, Välimäki N, Gylfe AE, Ristolainen H, Hänninen UA, Cajuso T, et al. 2015. CTCF/cohesin-binding sites are frequently mutated in cancer. <em>Nature Genetics</em> <strong>advance on</strong>: 818–21.</p>
</div>
<div id="ref-Kent2003">
<p>Kent WJ, Baertsch R, Hinrichs A, Miller W, Haussler D. 2003. Evolution’s cauldron: Duplication, deletion, and rearrangement in the mouse and human genomes. <em>Proceedings of the National Academy of Sciences of the United States of America</em> <strong>100</strong>: 11484–11489.</p>
</div>
<div id="ref-Kent2002">
<p>Kent WJ, Sugnet CW, Furey TS, Roskin KM, Pringle TH, Zahler AM, Haussler D. 2002. The human genome browser at ucsc. <em>Genome research</em> <strong>12</strong>: 996–1006.</p>
</div>
<div id="ref-Kent2010">
<p>Kent WJ, Zweig AS, Barber G, Hinrichs AS, Karolchik D. 2010. BigWig and bigbed: Enabling browsing of large distributed datasets. <em>Bioinformatics</em> <strong>26</strong>: 2204–2207.</p>
</div>
<div id="ref-Khan2018">
<p>Khan A, Fornes O, Stigliani A. 2018. JASPAR 2018: Update of the open-access database of transcription factor binding profiles and its web framework. <em>Nucleic Acids Research</em> <strong>(In print)</strong>: 1–7.</p>
</div>
<div id="ref-Kikuta2007">
<p>Kikuta H, Laplante M, Navratilova P, Komisarczuk AZ, Engström PG, Fredman D, Akalin A, Caccamo M, Sealy I, Howe K, et al. 2007. Genomic regulatory blocks encompass multiple neighboring genes and maintain conserved synteny in vertebrates. <em>Genome Research</em> <strong>17</strong>: 545–555.</p>
</div>
<div id="ref-Kim2007">
<p>Kim TH, Abdullaev ZK, Smith AD, Ching KA, Loukinov DI, Green RD, Zhang MQ, Lobanenkov VV, Ren B. 2007. Analysis of the vertebrate insulator protein ctcf-binding sites in the human genome. <em>Cell</em> <strong>128</strong>: 1231–1245.</p>
</div>
<div id="ref-Kim2008">
<p>Kim WK, Marcotte EM. 2008. Age-dependent evolution of the yeast protein interaction network suggests a limited role of gene duplication and divergence. <em>PLoS computational biology</em> <strong>4</strong>: e1000232.</p>
</div>
<div id="ref-Kleinjan2001">
<p>Kleinjan DA, Seawright A, Schedl A, Quinlan RA, Danes S, Heyningen V van. 2001. Aniridia-associated translocations, dnase hypersensitivity, sequence comparison and transgenic analysis redefine the functional domain of pax6. <em>Human Molecular Genetics</em> <strong>10</strong>: 2049–2059.</p>
</div>
<div id="ref-Kleinjan2005">
<p>Kleinjan DA, Van Heyningen V. 2005. Long-range control of gene expression: Emerging mechanisms and disruption in disease. <em>Am J Hum Genet</em> <strong>76</strong>: 8–32.</p>
</div>
<div id="ref-Kleinjan1998">
<p>Kleinjan D, Heyningen V van. 1998. Position effect in human genetic disease. <em>Human Molecular Genetics</em> <strong>7</strong>: 1611–1618.</p>
</div>
<div id="ref-Knight2013">
<p>Knight Pa, Ruiz D. 2013. A fast algorithm for matrix balancing. <em>IMA Journal of Numerical Analysis</em> <strong>33</strong>: 1029–1047.</p>
</div>
<div id="ref-Kolovos2018">
<p>Kolovos P, Brouwer RWW, Kockx CEM, Lesnussa M, Kepper N, Zuin J, Imam AMA, Werken HJG van de, Wendt KS, Knoch TA, et al. 2018. Investigation of the spatial structure and interactions of the genome at sub-kilobase-pair resolution using t2c. <em>Nature Protocols</em> <strong>13</strong>: 459–477.</p>
</div>
<div id="ref-Koonin2005">
<p>Koonin EV. 2005. Orthologs, paralogs, and evolutionary genomics. <em>Annual review of genetics</em> <strong>39</strong>: 309–338.</p>
</div>
<div id="ref-Korbel2007">
<p>Korbel JO, Urban AE, Affourtit JP, Godwin B, Grubert F, Simons JF, Kim PM, Palejev D, Carriero NJ, Du L, et al. 2007. Paired-end mapping reveals extensive structural variation in the human genome. <em>Science (New York, NY)</em> <strong>318</strong>: 420–6.</p>
</div>
<div id="ref-Kohler2014">
<p>Köhler S, Doelken SC, Mungall CJ, Bauer S, Firth HV, Bailleul-Forestier I, Black GCM, Brown DL, Brudno M, Campbell J, et al. 2014. The human phenotype ontology project: Linking molecular biology and disease through phenotype data. <em>Nucleic acids research</em> <strong>42</strong>: D966–74.</p>
</div>
<div id="ref-Kraft2015">
<p>Kraft K, Geuer S, Will AJ, Chan WL, Paliou C, Borschiwer M, Harabula I, Wittler L, Franke M, Ibrahim DM, et al. 2015. Deletions, inversions, duplications: Engineering of structural variants using crispr/cas in mice. <em>Cell Reports</em> <strong>10</strong>: 833–839.</p>
</div>
<div id="ref-Krijger2016">
<p>Krijger PHL, Laat W de. 2016. Regulation of disease-associated gene expression in the 3D genome. <em>Nature Reviews Molecular Cell Biology</em>.</p>
</div>
<div id="ref-DeLaat2007">
<p>Laat W de, Grosveld F. 2007. Inter-chromosomal gene regulation in the mammalian cell nucleus. <em>Current opinion in genetics &amp; development</em> <strong>17</strong>: 456–464.</p>
</div>
<div id="ref-Lachke2012">
<p>Lachke SA, Higgins AW, Inagaki M, Saadi I, Xi Q, Long M, Quade BJ, Talkowski ME, Gusella JF, Fujimoto A, et al. 2012. The cell adhesion gene pvrl3 is associated with congenital ocular defects. <em>Human Genetics</em> <strong>131</strong>: 235–250.</p>
</div>
<div id="ref-Lan2016">
<p>Lan X, Pritchard JK. 2016. Coregulation of tandem duplicate genes slows evolution of subfunctionalization in mammals. <em>Science (New York, NY)</em> <strong>352</strong>: 1009–13.</p>
</div>
<div id="ref-Lander2011">
<p>Lander ES. 2011. Initial impact of the sequencing of the human genome. <em>Nature</em> <strong>470</strong>: 187–197.</p>
</div>
<div id="ref-Lander2001">
<p>Lander ES, Linton LM, Birren B, Nusbaum C, Zody MC, Baldwin J, Devon K, Dewar K, Doyle M, FitzHugh W, et al. 2001. Initial sequencing and analysis of the human genome. <em>Nature</em> <strong>409</strong>: 860–921.</p>
</div>
<div id="ref-Lappalainen2013">
<p>Lappalainen I, Lopez J, Skipper L, Hefferon T, Spalding JD, Garner J, Chen C, Maguire M, Corbett M, Zhou G, et al. 2012. DbVar and dgva: Public archives for genomic structural variation. <em>Nucleic Acids Research</em> <strong>41</strong>: D936–D941.</p>
</div>
<div id="ref-Lawrence2009">
<p>Lawrence M, Gentleman R, Carey V. 2009. Rtracklayer: An r package for interfacing with genome browsers. <em>Bioinformatics</em> <strong>25</strong>: 1841–1842.</p>
</div>
<div id="ref-Lawrence2013">
<p>Lawrence M, Huber W, Pagès H, Aboyoun P, Carlson M, Gentleman R, Morgan MT, Carey VJ. 2013. Software for computing and annotating genomic ranges. <em>PLoS computational biology</em> <strong>9</strong>: e1003118.</p>
</div>
<div id="ref-LeDily2014">
<p>Le Dily F, Bau D, Pohl a, Vicent GP, Serra F, Soronellas D, Castellano G, Wright RHG, Ballare C, Filion G, et al. 2014. Distinct structural transitions of chromatin topological domains correlate with coordinated hormone-induced gene regulation. <em>Genes &amp; Development</em> <strong>28</strong>: 2151–2162.</p>
</div>
<div id="ref-Le2013">
<p>Le TBK, Imakaev MV, Mirny L a, Laub MT. 2013. High-resolution mapping of the spatial organization of a bacterial chromosome. <em>Science (New York, NY)</em> <strong>342</strong>: 731–4.</p>
</div>
<div id="ref-Lejeune1959">
<p>LeJeune J, Gautier M, Turpin R. 1959. Study of somatic chromosomes from 9 mongoloid children. <em>Comptes rendus hebdomadaires des seances de l’Academie des sciences</em> <strong>248</strong>: 1721–2.</p>
</div>
<div id="ref-Lelli2012">
<p>Lelli KM, Slattery M, Mann RS. 2012. Disentangling the many layers of eukaryotic transcriptional regulation. <em>Annual review of genetics</em> <strong>46</strong>: 43–68.</p>
</div>
<div id="ref-Liang2014">
<p>Liang J, Lacroix L, Gamot A, Cuddapah S, Queille S, Lhoumaud P, Lepetit P, Martin PGP, Vogelmann J, Court F, et al. 2014. Chromatin immunoprecipitation indirect peaks highlight long-range interactions of insulator proteins and pol ii pausing. <em>Molecular Cell</em> <strong>53</strong>: 672–681.</p>
</div>
<div id="ref-Lieberman-Aiden2009">
<p>Lieberman-Aiden E, Berkum NL van, Williams L, Imakaev M, Ragoczy T, Telling A, Amit I, Lajoie BR, Sabo PJ, Dorschner MO, et al. 2009. Comprehensive mapping of long-range interactions reveals folding principles of the human genome. <em>Science (New York, NY)</em> <strong>326</strong>: 289–93.</p>
</div>
<div id="ref-Ligon2005">
<p>Ligon AH, Moore SD, Parisi MA, Mealiffe ME, Harris DJ, Ferguson HL, Quade BJ, Morton CC. 2005. Constitutional rearrangement of the architectural factor hmga2: A novel human phenotype including overgrowth and lipomas. <em>The American Journal of Human Genetics</em> <strong>76</strong>: 340–348.</p>
</div>
<div id="ref-Lin2012">
<p>Lin YC, Benner C, Mansson R, Heinz S, Miyazaki K, Miyazaki M, Chandra V, Bossen C, Glass CK, Murre C. 2012. Global changes in the nuclear positioning of genes and intra- and interdomain genomic interactions that orchestrate b cell fate. <em>Nature Immunology</em> <strong>13</strong>: 1196–1204.</p>
</div>
<div id="ref-Liu2016b">
<p>Liu Y, Sarkar A, Kheradpour P, Ernst J, Kellis M. 2017. Evidence of reduced recombination rate in human regulatory domains. <em>Genome Biology</em> <strong>18</strong>: 193.</p>
</div>
<div id="ref-Long2016">
<p>Long HK, Prescott SL, Wysocka J. 2016. Ever-changing landscapes: Transcriptional enhancers in development and evolution. <em>Cell</em> <strong>167</strong>: 1170–1187.</p>
</div>
<div id="ref-Lonsdale2013">
<p>Lonsdale J, Thomas J, Salvatore M, Phillips R, Lo E, Shad S, Hasz R, Walters G, Garcia F, Young N, et al. 2013. The genotype-tissue expression (gtex) project. <em>Nature Genetics</em> <strong>45</strong>: 580–585.</p>
</div>
<div id="ref-Lu2007">
<p>Lu W, Quintero-Rivera F, Fan Y, Alkuraya FS, Donovan DJ, Xi Q, Turbe-Doan A, Li Q-G, Campbell CG, Shanske AL, et al. 2007. NFIA haploinsufficiency is associated with a cns malformation syndrome and urinary tract defects. <em>PLoS Genetics</em> <strong>3</strong>: e80.</p>
</div>
<div id="ref-Lun2016">
<p>Lun ATL, Perry M, Ing-Simmons E. 2016. Infrastructure for genomic interactions : Bioconductor classes for hi-c , chia-pet and related experiments [version 1; referees: 2 approved]. <em>F1000Research</em> <strong>5</strong>: 1–6.</p>
</div>
<div id="ref-Lupianez2015">
<p>Lupiáñez DG, Kraft K, Heinrich V, Krawitz P, Brancati F, Klopocki E, Horn D, Kayserili H, Opitz JM, Laxova R, et al. 2015. Disruptions of topological chromatin domains cause pathogenic rewiring of gene-enhancer interactions. <em>Cell</em> <strong>161</strong>: 1012–1025.</p>
</div>
<div id="ref-Lupski2005">
<p>Lupski JR, Stankiewicz P. 2005. Genomic disorders: Molecular mechanisms for rearrangements and conveyed phenotypes. <em>PLoS Genetics</em> <strong>1</strong>: e49.</p>
</div>
<div id="ref-Lynch2003">
<p>Lynch M, Conery JS. 2003. The evolutionary demography of duplicate genes. In <em>Genome evolution</em>, pp. 35–44, Springer Netherlands, Dordrecht.</p>
</div>
<div id="ref-Makova2003">
<p>Makova KDK, Li W-HW. 2003. Divergence in the spatial pattern of gene expression between human duplicate genes. <em>Genome research</em> <strong>13</strong>: 1638–1645.</p>
</div>
<div id="ref-Mathelier2015">
<p>Mathelier A, Fornes O, Arenillas DJ, Chen C-y, Denay G, Lee J, Shi W, Shyr C, Tan G, Worsley-Hunt R, et al. 2015. JASPAR 2016: A major expansion and update of the open-access database of transcription factor binding profiles. <em>Nucleic acids research</em> <strong>44</strong>: gkv1176.</p>
</div>
<div id="ref-Merkenschlager2016">
<p>Merkenschlager M, Nora EP. 2016. CTCF and cohesin in genome folding and transcriptional gene regulation. <em>Annual review of genomics and human genetics</em> <strong>17</strong>: 17–43.</p>
</div>
<div id="ref-Mifsud2015">
<p>Mifsud B, Tavares-Cadete F, Young AN, Sugar R, Schoenfelder S, Ferreira L, Wingett SW, Andrews S, Grey W, Ewels P a, et al. 2015. Mapping long-range promoter contacts in human cells with high-resolution capture hi-c. <em>Nature Genetics</em> <strong>47</strong>.</p>
</div>
<div id="ref-Miller2010">
<p>Miller DT, Adam MP, Aradhya S, Biesecker LG, Brothman AR, Carter NP, Church DM, Crolla JA, Eichler EE, Epstein CJ, et al. 2010. Consensus statement: Chromosomal microarray is a first-tier clinical diagnostic test for individuals with developmental disabilities or congenital anomalies. <em>The American Journal of Human Genetics</em> <strong>86</strong>: 749–764.</p>
</div>
<div id="ref-Mills2006">
<p>Mills RE, Bennett EA, Iskow RC, Luttig CT, Tsui C, Pittard WS, Devine SE. 2006. Recently mobilized transposons in the human and chimpanzee genomes. <em>The American Journal of Human Genetics</em> <strong>78</strong>: 671–679.</p>
</div>
<div id="ref-Mizuguchi2014">
<p>Mizuguchi T, Fudenberg G, Mehta S, Belton J-M, Taneja N, Folco HD, FitzGerald P, Dekker J, Mirny L, Barrowman J, et al. 2014. Cohesin-dependent globules and heterochromatin shape 3D genome architecture in s. Pombe. <em>Nature</em>.</p>
</div>
<div id="ref-Monahan2015">
<p>Monahan K, Lomvardas S. 2015. Monoallelic expression of olfactory receptors. <em>Annual Review of Cell and Developmental Biology</em> <strong>31</strong>: annurev–cellbio–100814–125308.</p>
</div>
<div id="ref-Monahan2017">
<p>Monahan K, Schieren I, Cheung J, Mumbey-Wafula A, Monuki ES, Lomvardas S. 2017. Cooperative interactions enable singular olfactory receptor expression in mouse olfactory neurons. <em>eLife</em> <strong>6</strong>: 1–32.</p>
</div>
<div id="ref-Montavon2012">
<p>Montavon T, Thevenet L, Duboule D. 2012. Impact of copy number variations (cnvs) on long-range gene regulation at the hoxd locus. <em>Proceedings of the National Academy of Sciences of the United States of America</em> <strong>109</strong>: 20204–11.</p>
</div>
<div id="ref-Mora2015">
<p>Mora A, Sandve GK, Gabrielsen OS, Eskeland R. 2015. In the loop: Promoter-enhancer interactions and bioinformatics. <em>Briefings in bioinformatics</em> 1–16.</p>
</div>
<div id="ref-Mourad2017">
<p>Mourad R, Li L, Cuvier O. 2017. Uncovering direct and indirect molecular determinants of chromatin loops using a computational integrative approach. <em>PLOS Computational Biology</em> <strong>13</strong>: e1005538.</p>
</div>
<div id="ref-Mumbach2016">
<p>Mumbach MR, Rubin AJ, Flynn RA, Dai C, Khavari PA, Greenleaf WJ, Chang HY. 2016. HiChIP: Efficient and sensitive analysis of protein-directed genome architecture. <em>Nature methods</em> <strong>13</strong>: 919–922.</p>
</div>
<div id="ref-Muro2010">
<p>Muro EM, Andrade-Navarro MA. 2010. Pseudogenes as an alternative source of natural antisense transcripts. <em>BMC Evolutionary Biology</em> <strong>10</strong>: 338.</p>
</div>
<div id="ref-Murphy2005">
<p>Murphy WJ, Larkin DM, Everts-van der Wind A, Bourque G, Tesler G, Auvil L, Beever JE, Chowdhary BP, Galibert F, Gatzke L, et al. 2005. Dynamics of mammalian chromosome evolution inferred from multispecies comparative maps. <em>Science (New York, NY)</em> <strong>309</strong>: 613–7.</p>
</div>
<div id="ref-Nagano2013">
<p>Nagano T, Lubling Y, Stevens TJ, Schoenfelder S, Yaffe E, Dean W, Laue ED, Tanay A, Fraser P. 2013. Single-cell hi-c reveals cell-to-cell variability in chromosome structure. <em>Nature</em> <strong>502</strong>: 59–64.</p>
</div>
<div id="ref-Nagy2016">
<p>Nagy G, Czipa E, Steiner L, Nagy T, Pongor S, Nagy L, Barta E. 2016. Motif oriented high-resolution analysis of chip-seq data reveals the topological order of ctcf and cohesin proteins on dna. <em>BMC Genomics</em> <strong>17</strong>: 637.</p>
</div>
<div id="ref-Nambiar2011">
<p>Nambiar M, Raghavan SC. 2011. How does dna break during chromosomal translocations? <em>Nucleic Acids Research</em> <strong>39</strong>: 5813–5825.</p>
</div>
<div id="ref-Narendra2015">
<p>Narendra V, Rocha PP, An D, Raviram R, Skok JA, Mazzoni EO, Reinberg D. 2015. Transcription. CTCF establishes discrete functional chromatin domains at the hox clusters during differentiation. <em>Science (New York, NY)</em> <strong>347</strong>: 1017–21.</p>
</div>
<div id="ref-Naumova2013">
<p>Naumova N, Imakaev M, Fudenberg G, Zhan Y, Lajoie BR, Mirny L a, Dekker J. 2013. Organization of the mitotic chromosome. <em>Science (New York, NY)</em> <strong>342</strong>: 948–53.</p>
</div>
<div id="ref-Naville2015">
<p>Naville M, Ishibashi M, Ferg M, Bengani H, Rinkwitz S, Krecsmarik M, Hawkins Ta, Wilson SW, Manning E, Chilamakuri CSR, et al. 2015. Long-range evolutionary constraints reveal cis-regulatory interactions on the human x chromosome. <em>Nature Communications</em> <strong>6</strong>: 6904.</p>
</div>
<div id="ref-Newman2015">
<p>Newman S, Hermetz KE, Weckselblatt B, Rudd MK. 2015. Next-generation sequencing of duplication cnvs reveals that most are tandem and some create fusion genes at breakpoints. <em>The American Journal of Human Genetics</em> <strong>96</strong>: 1–13.</p>
</div>
<div id="ref-Nikumbh2017">
<p>Nikumbh S, Pfeifer N. 2017. Genetic sequence-based prediction of long-range chromatin interactions suggests a potential role of short tandem repeat sequences in genome organization. <em>BMC Bioinformatics</em> <strong>18</strong>: 218.</p>
</div>
<div id="ref-Noordermeer2011">
<p>Noordermeer D, Wit E de, Klous P, Werken H van de, Simonis M, Lopez-Jones M, Eussen B, Klein A de, Singer RH, Laat W de. 2011. Variegated gene expression caused by cell-specific long-range dna interactions. <em>Nature Cell Biology</em> <strong>13</strong>: 944–951.</p>
</div>
<div id="ref-Nora2013">
<p>Nora EP, Dekker J, Heard E. 2013. Segmental folding of chromosomes: A basis for structural and regulatory chromosomal neighborhoods? <em>BioEssays</em> <strong>35</strong>: 818–828.</p>
</div>
<div id="ref-Nora2017">
<p>Nora EP, Goloborodko A, Valton A-L, Gibcus JH, Uebersohn A, Abdennur N, Dekker J, Mirny L, Bruneau B. 2017. Targeted degradation of ctcf decouples local insulation of chromosome domains from higher-order genomic compartmentalization. <em>bioRxiv</em> 095802.</p>
</div>
<div id="ref-Nora2012">
<p>Nora EP, Lajoie BR, Schulz EG, Giorgetti L, Okamoto I, Servant N, Piolot T, Berkum NL van, Meisig J, Sedat J, et al. 2012. Spatial partitioning of the regulatory landscape of the x-inactivation centre. <em>Nature</em> <strong>485</strong>: 381–5.</p>
</div>
<div id="ref-Northcott2014">
<p>Northcott Pa, Lee C, Zichner T, Stütz AM, Erkek S, Kawauchi D, Shih DJH, Hovestadt V, Zapatka M, Sturm D, et al. 2014. Enhancer hijacking activates gfi1 family oncogenes in medulloblastoma. <em>Nature</em>.</p>
</div>
<div id="ref-Oldridge2015">
<p>Oldridge DA, Wood AC, Weichert-Leahey N, Crimmins I, Sussman R, Winter C, McDaniel LD, Diamond M, Hart LS, Zhu S, et al. 2015. Genetic predisposition to neuroblastoma mediated by a lmo1 super-enhancer polymorphism. <em>Nature</em> <strong>528</strong>: 418–421.</p>
</div>
<div id="ref-Ong2014">
<p>Ong C-T, Corces VG. 2014. CTCF: An architectural protein bridging genome topology and function. <em>Nature Reviews Genetics</em>.</p>
</div>
<div id="ref-Ordulu2016">
<p>Ordulu Z, Kammin T, Brand H, Pillalamarri V, Redin CE, Collins RL, Blumenthal I, Hanscom C, Pereira S, Crandall BF, et al. 2016. Structural chromosomal rearrangements require nucleotide-level resolution : Lessons from next-generation sequencing in prenatal diagnosis. <em>The American Journal of Human Genetics</em> 1–19.</p>
</div>
<div id="ref-Ordulu2014">
<p>Ordulu Z, Wong KE, Currall BB, Ivanov AR, Pereira S, Althari S, Gusella JF, Talkowski ME, Morton CC. 2014. Describing sequencing results of structural chromosome rearrangements with a suggested next-generation cytogenetic nomenclature. <em>The American Journal of Human Genetics</em> <strong>94</strong>: 695–709.</p>
</div>
<div id="ref-Orlando1997">
<p>Orlando V, Strutt H, Paro R. 1997. Analysis of chromatin structure byin vivoformaldehyde cross-linking. <em>Methods</em> <strong>11</strong>: 205–214.</p>
</div>
<div id="ref-Oti2016">
<p>Oti M, Falck J, Huynen MA, Zhou H. 2016. CTCF-mediated chromatin loops enclose inducible gene regulatory domains. <em>BMC Genomics</em> <strong>17</strong>: 252.</p>
</div>
<div id="ref-OConnor2014">
<p>O’Connor TR, Bailey TL. 2014. Creating and validating cis-regulatory maps of tissue-specific gene expression regulation. <em>Nucleic acids research</em> 1–11.</p>
</div>
<div id="ref-PachlopnikSchmid2012">
<p>Pachlopnik Schmid J, Lemoine R, Nehme N, Cormier-Daire V, Revy P, Debeurme F, Debré M, Nitschke P, Bole-Feysot C, Legeai-Mallet L, et al. 2012. Polymerase <span class="math inline">\(\epsilon\)</span>1 mutation in a human syndrome with facial dysmorphism, immunodeficiency, livedo, and short stature (&quot;fils syndrome&quot;). <em>The Journal of experimental medicine</em> <strong>209</strong>: 2323–30.</p>
</div>
<div id="ref-Palstra2003">
<p>Palstra R-J, Tolhuis B, Splinter E, Nijmeijer R, Grosveld F, Laat W de. 2003. The <span class="math inline">\(\beta\)</span>-globin nuclear compartment in development and erythroid differentiation. <em>Nature Genetics</em> <strong>35</strong>: 190–194.</p>
</div>
<div id="ref-Parker2013">
<p>Parker SCJ, Stitzel ML, Taylor DL, Orozco JM, Erdos MR, Akiyama JA, Bueren KL van, Chines PS, Narisu N, NISC Comparative Sequencing Program NCS, et al. 2013. Chromatin stretch enhancer states drive cell-specific gene regulation and harbor human disease risk variants. <em>Proceedings of the National Academy of Sciences of the United States of America</em> <strong>110</strong>: 17921–6.</p>
</div>
<div id="ref-Petryszak2015">
<p>Petryszak R, Keays M, Tang YA, Fonseca NA, Barrera E, Burdett T, Füllgrabe A, Fuentes AM-P, Jupp S, Koskinen S, et al. 2015. Expression atlas update—an integrated database of gene and protein expression in humans, animals and plants. <em>Nucleic Acids Research</em> <strong>44</strong>: gkv1045.</p>
</div>
<div id="ref-Pevzner2003">
<p>Pevzner P, Tesler G. 2003. Human and mouse genomic sequences reveal extensive breakpoint reuse in mammalian evolution. <em>Proceedings of the National Academy of Sciences of the United States of America</em> <strong>100</strong>: 7672–7.</p>
</div>
<div id="ref-Phillips-Cremins2013">
<p>Phillips-Cremins JE, Sauria MEG, Sanyal A, Gerasimova TI, Lajoie BR, Bell JSK, Ong C-T, Hookway T a, Guo C, Sun Y, et al. 2013. Architectural protein subclasses shape 3D organization of genomes during lineage commitment. <em>Cell</em> <strong>153</strong>: 1281–95.</p>
</div>
<div id="ref-Pink2011">
<p>Pink RC, Wicks K, Caley DP, Punch EK, Jacobs L, Carter DRF. 2011. Pseudogenes: Pseudo-functional or key regulators in health and disease? <em>RNA (New York, NY)</em> <strong>17</strong>: 792–8.</p>
</div>
<div id="ref-Pique-Regi2011">
<p>Pique-Regi R, Degner JF, Pai A a, Gaffney DJ, Gilad Y, Pritchard JK. 2011. Accurate inference of transcription factor binding from dna sequence and chromatin accessibility data. <em>Genome research</em> <strong>21</strong>: 447–55.</p>
</div>
<div id="ref-Polychronopoulos2017">
<p>Polychronopoulos D, King JWD, Nash AJ, Tan G, Lenhard B. 2017. Conserved non-coding elements: Developmental gene regulation meets genome organization. <em>Nucleic Acids Research</em> 1–14.</p>
</div>
<div id="ref-Pombo2015">
<p>Pombo A, Dillon N. 2015. Three-dimensional genome architecture: Players and mechanisms. <em>Nature Reviews Molecular Cell Biology</em> <strong>12</strong>.</p>
</div>
<div id="ref-Pope2014">
<p>Pope BD, Ryba T, Dileep V, Yue F, Wu W, Denas O, Vera DL, Wang Y, Hansen RS, Canfield TK, et al. 2014. Topologically associating domains are stable units of replication-timing regulation. <em>Nature</em> <strong>515</strong>: 402–405.</p>
</div>
<div id="ref-Prakash2017">
<p>Prakash K, Fournier D. 2018. Evidence for the implication of the histone code in building the genome structure. <em>Biosystems</em> <strong>164</strong>: 49–59.</p>
</div>
<div id="ref-Ptashne1986">
<p>Ptashne M. 1986. Gene regulation by proteins acting nearby and at a distance. <em>Nature</em> <strong>322</strong>: 697–701.</p>
</div>
<div id="ref-Purmann2007">
<p>Purmann A, Toedling J, Schueler M, Carninci P, Lehrach H, Hayashizaki Y, Huber W, Sperling S. 2007. Genomic organization of transcriptomes in mammals: Coregulation and cofunctionality. <em>Genomics</em> <strong>89</strong>: 580–587.</p>
</div>
<div id="ref-Quinlan2010">
<p>Quinlan AR, Hall IM. 2010. BEDTools: A flexible suite of utilities for comparing genomic features. <em>Bioinformatics</em> <strong>26</strong>: 841–842.</p>
</div>
<div id="ref-Quinn2016">
<p>Quinn JJ, Chang HY. 2016. Unique features of long non-coding rna biogenesis and function. <em>Nature Reviews Genetics</em> <strong>17</strong>: 47–62.</p>
</div>
<div id="ref-Ramani2016">
<p>Ramani V, Deng X, Gunderson KL, Steemers FJ, Disteche CM, Noble WS, Duan Z, Shendure J. 2016. Massively multiplex single-cell hi-c. <em>bioRxiv</em> <strong>14</strong>: 065052.</p>
</div>
<div id="ref-Rao2017">
<p>Rao SS, Huang S-C, Hilaire BGS, Engreitz JM, Perez EM, Kieffer-Kwon K-R, Sanborn AL, Johnstone SE, Bascom GD, Bochkov ID, et al. 2017. Cohesin loss eliminates all loop domains. <em>Cell</em> <strong>171</strong>: 305–320.e24.</p>
</div>
<div id="ref-Rao2014">
<p>Rao SSP, Huntley MH, Durand NC, Stamenova EK, Bochkov ID, Robinson JT, Sanborn AL, Machol I, Omer AD, Lander ES, et al. 2014. A 3D map of the human genome at kilobase resolution reveals principles of chromatin looping. <em>Cell</em> <strong>159</strong>: 1665–80.</p>
</div>
<div id="ref-Redin2017">
<p>Redin C, Brand H, Collins RL, Kammin T, Mitchell E, Hodge JC, Hanscom C, Pillalamarri V, Seabra CM, Abbott M-A, et al. 2017. The genomic landscape of balanced cytogenetic abnormalities associated with human congenital anomalies. <em>Nature Genetics</em> <strong>49</strong>: 36–45.</p>
</div>
<div id="ref-Rehm2015">
<p>Rehm HL, Berg JS, Brooks LD, Bustamante CD, Evans JP, Landrum MJ, Ledbetter DH, Maglott DR, Martin CL, Nussbaum RL, et al. 2015. ClinGen — the clinical genome resource. <em>New England Journal of Medicine</em> <strong>372</strong>: 2235–2242.</p>
</div>
<div id="ref-Rennie2018">
<p>Rennie S, Dalby M, Duin L van, Andersson R. 2018. Transcriptional decomposition reveals active chromatin architectures and cell specific regulatory interactions. <em>Nature Communications</em> <strong>9</strong>: 487.</p>
</div>
<div id="ref-Rhee2011">
<p>Rhee HS, Pugh BF. 2011. Comprehensive genome-wide protein-dna interactions detected at single-nucleotide resolution. <em>Cell</em> <strong>147</strong>: 1408–19.</p>
</div>
<div id="ref-Kundaje2015">
<p>Roadmap Epigenomics Consortium, Kundaje A, Meuleman W, Ernst J, Bilenky M, Yen A, Heravi-Moussavi A, Kheradpour P, Zhang Z, Wang J, et al. 2015. Integrative analysis of 111 reference human epigenomes. <em>Nature</em> <strong>518</strong>: 317–30.</p>
</div>
<div id="ref-Rogozin2014">
<p>Rogozin IB, Managadze D, Shabalina SA, Koonin EV. 2014. Gene family level comparative analysis of gene expression n mammals validates the ortholog conjecture. <em>Genome Biology and Evolution</em> <strong>6</strong>: 754–762.</p>
</div>
<div id="ref-Rotem2015">
<p>Rotem A, Ram O, Shoresh N, Sperling RA, Goren A, Weitz DA, Bernstein BE. 2015. Single-cell chip-seq reveals cell subpopulations defined by chromatin state. <em>Nature Biotechnology</em>.</p>
</div>
<div id="ref-Roukos2014">
<p>Roukos V, Misteli T. 2014. The biogenesis of chromosome translocations. <em>Nature cell biology</em> <strong>16</strong>: 293–300.</p>
</div>
<div id="ref-Roukos2013">
<p>Roukos V, Voss TC, Schmidt CK, Lee S, Wangsa D, Misteli T. 2013. Spatial dynamics of chromosome translocations in living cells. <em>Science</em> <strong>341</strong>: 660–664.</p>
</div>
<div id="ref-Roussos2014">
<p>Roussos P, Mitchell AC, Voloudakis G, Fullard JF, Pothula VM, Tsang J, Stahl EA, Georgakopoulos A, Ruderfer DM, Charney A, et al. 2014. A role for noncoding variation in schizophrenia. <em>Cell reports</em> <strong>9</strong>: 1417–29.</p>
</div>
<div id="ref-Rowley2016">
<p>Rowley MJ, Corces VG. 2016. The three-dimensional genome: Principles and roles of long-distance interactions. <em>Current Opinion in Cell Biology</em> <strong>40</strong>: 8–14.</p>
</div>
<div id="ref-Rowley2017">
<p>Rowley MJ, Nichols MH, Lyu X, Ando-Kuri M, Rivera ISM, Hermetz K, Wang P, Ruan Y, Corces VG. 2017. Evolutionarily conserved principles predict 3D chromatin organization. <em>Molecular Cell</em> 1–16.</p>
</div>
<div id="ref-Roy2015">
<p>Roy S, Siahpirani AF, Chasman D, Knaack S, Ay F, Stewart R, Wilson M, Sridharan R. 2015. A predictive modeling approach for cell line-specific long-range regulatory interactions. <em>Nucleic Acids Research</em> <strong>43</strong>: 8694–8712.</p>
</div>
<div id="ref-Ruiz-Velasco2017">
<p>Ruiz-Velasco M, Zaugg JB. 2017. Structure meets function: How chromatin organisation conveys functionality. <em>Current Opinion in Systems Biology</em> <strong>1</strong>: 129–136.</p>
</div>
<div id="ref-Rutledge2015">
<p>Rutledge MT, Russo M, Belton J-M, Dekker J, Broach JR. 2015. The yeast genome undergoes significant topological reorganization in quiescence. <em>Nucleic Acids Research</em> gkv723.</p>
</div>
<div id="ref-Sahlen2015">
<p>Sahlén P, Abdullayev I, Ramsköld D, Matskova L, Rilakovic N, Lötstedt B, Albert TJ, Lundeberg J, Sandberg R. 2015. Genome-wide mapping of promoter-anchored interactions with close to single-enhancer resolution. <em>Genome Biology</em> <strong>16</strong>: 156.</p>
</div>
<div id="ref-Saito2017">
<p>Saito T, Rehmsmeier M. 2016. Precrec: Fast and accurate precision-recall and roc curve calculations in r. <em>Bioinformatics</em> <strong>33</strong>: btw570.</p>
</div>
<div id="ref-Saito2015">
<p>Saito T, Rehmsmeier M. 2015. The precision-recall plot is more informative than the roc plot when evaluating binary classifiers on imbalanced datasets. <em>PLoS ONE</em> <strong>10</strong>: 1–21.</p>
</div>
<div id="ref-Sanborn2015">
<p>Sanborn AL, Rao SSP, Huang S-C, Durand NC, Huntley MH, Jewett AI, Bochkov ID, Chinnappan D, Cutkosky A, Li J, et al. 2015. Chromatin extrusion explains key features of loop and domain formation in wild-type and engineered genomes. <em>Proceedings of the National Academy of Sciences</em> <strong>112</strong>: 201518552.</p>
</div>
<div id="ref-Sanyal2012">
<p>Sanyal A, Lajoie BR, Jain G, Dekker J. 2012. The long-range interaction landscape of gene promoters. <em>Nature</em> <strong>489</strong>: 109–113.</p>
</div>
<div id="ref-Sati2017">
<p>Sati S, Cavalli G. 2017. Chromosome conformation capture technologies and their impact in understanding genome function. <em>Chromosoma</em> <strong>126</strong>: 33–44.</p>
</div>
<div id="ref-Sati2016">
<p>Sati S, Cavalli G. 2016. Chromosome conformation capture technologies and their impact in understanding genome function. <em>Chromosoma</em>.</p>
</div>
<div id="ref-Schaefer2012">
<p>Schaefer MH, Fontaine J-F, Vinayagam A, Porras P, Wanker EE, Andrade-Navarro M a. 2012. HIPPIE: Integrating protein interaction networks with experiment based quality scores. <em>PloS one</em> <strong>7</strong>: e31826.</p>
</div>
<div id="ref-Schmitt2016">
<p>Schmitt AD, Hu M, Ren B. 2016. Genome-wide mapping and analysis of chromosome architecture. <em>Nature Reviews Molecular Cell Biology</em>.</p>
</div>
<div id="ref-Schoenfelder2015">
<p>Schoenfelder S, Furlan-magaril M, Mifsud B, Tavares-cadete F, Sugar R, Javierre B-m, Nagano T, Katsman Y, Sakthidevi M, Wingett SW, et al. 2015. The pluripotent regulatory circuitry connecting promoters to their long-range interacting elements. <em>Genome Research</em> 1–16.</p>
</div>
<div id="ref-Schreiber2017">
<p>Schreiber J, Libbrecht M, Bilmes J, Noble W. 2017. Nucleotide sequence and dnasei sensitivity are predictive of 3D chromatin architecture. <em>bioRxiv</em> 1–15.</p>
</div>
<div id="ref-Schwarzer2017">
<p>Schwarzer W, Abdennur N, Goloborodko A, Pekowska A, Fudenberg G, Loe-Mie Y, Fonseca NA, Huber W, H Haering C, Mirny L, et al. 2017. Two independent modes of chromatin organization revealed by cohesin removal. <em>Nature</em> <strong>551</strong>: 51–56.</p>
</div>
<div id="ref-Sebat2004">
<p>Sebat J, Lakshmi B, Troge J, Alexander J, Young J, Lundin P, Manér S, Massa H, Walker M, Chi M, et al. 2004. Large-scale copy number polymorphism in the human genome. <em>Science</em> <strong>305</strong>: 525–528.</p>
</div>
<div id="ref-Seitan2013">
<p>Seitan VC, Faure AJ, Zhan Y, McCord RP, Lajoie BR, Ing-Simmons E, Lenhard B, Giorgetti L, Heard E, Fisher AG, et al. 2013. Cohesin-based chromatin interactions enable regulated gene expression within preexisting architectural compartments. <em>Genome Research</em> <strong>23</strong>: 2066–2077.</p>
</div>
<div id="ref-Sekelja2016">
<p>Sekelja M, Paulsen J, Collas P. 2016. 4D nucleomes in single cells: What can computational modeling reveal about spatial chromatin conformation? <em>Genome Biology</em> <strong>17</strong>: 54.</p>
</div>
<div id="ref-Sexton2015">
<p>Sexton T, Cavalli G. 2015. The role of chromosome domains in shaping the functional genome. <em>Cell</em> <strong>160</strong>: 1049–1059.</p>
</div>
<div id="ref-Sexton2012">
<p>Sexton T, Yaffe E, Kenigsberg E, Bantignies F, Leblanc B, Hoichman M, Parrinello H, Tanay A, Cavalli G. 2012. Three-dimensional folding and functional organization principles of the drosophila genome. <em>Cell</em> <strong>148</strong>: 458–72.</p>
</div>
<div id="ref-Sheffield2013">
<p>Sheffield NC, Thurman RE, Song L, Safi A, Stamatoyannopoulos J a, Lenhard B, Crawford GE, Furey TS. 2013. Patterns of regulatory activity across diverse human cell types predict tissue identity , transcription factor binding , and long-range interactions predict tissue identity , transcription factor binding , and long-range interactions. <em>Genome Research</em> <strong>23</strong>: 777–788.</p>
</div>
<div id="ref-Shen2012">
<p>Shen Y, Yue F, McCleary DF, Ye Z, Edsall L, Kuan S, Wagner U, Dixon J, Lee L, Lobanenkov VV, et al. 2012. A map of the cis-regulatory sequences in the mouse genome. <em>Nature</em> <strong>488</strong>: 116–20.</p>
</div>
<div id="ref-Shlyueva2014">
<p>Shlyueva D, Stampfel G, Stark A. 2014. Transcriptional enhancers: From properties to genome-wide predictions. <em>Nature Reviews Genetics</em> <strong>15</strong>: 272–86.</p>
</div>
<div id="ref-Simonis2006">
<p>Simonis M, Klous P, Splinter E, Moshkin Y, Willemsen R, Wit E de, Steensel B van, Laat W de. 2006. Nuclear organization of active and inactive chromatin domains uncovered by chromosome conformation capture–on-chip (4C). <em>Nature Genetics</em> <strong>38</strong>: 1348–1354.</p>
</div>
<div id="ref-Sing2005">
<p>Sing T, Sander O, Beerenwinkel N, Lengauer T. 2005. ROCR: Visualizing classifier performance in r. <em>Bioinformatics</em> <strong>21</strong>: 3940–3941.</p>
</div>
<div id="ref-Smemo2014">
<p>Smemo S, Tena JJ, Kim K-H, Gamazon ER, Sakabe NJ, Gómez-Marín C, Aneas I, Credidio FL, Sobreira DR, Wasserman NF, et al. 2014. Obesity-associated variants within fto form long-range functional connections with irx3. <em>Nature</em> <strong>507</strong>: 371–375.</p>
</div>
<div id="ref-Sofueva2013">
<p>Sofueva S, Yaffe E, Chan W-C, Georgopoulou D, Vietri Rudan M, Mira-Bontenbal H, Pollard SM, Schroth GP, Tanay A, Hadjur S. 2013. Cohesin-mediated interactions organize chromosomal domain architecture. <em>The EMBO journal</em> <strong>32</strong>: 3119–29.</p>
</div>
<div id="ref-Solovei2013">
<p>Solovei I, Wang AS, Thanisch K, Schmidt CS, Krebs S, Zwerger M, Cohen TV, Devys D, Foisner R, Peichl L, et al. 2013. LBR and lamin a/c sequentially tether peripheral heterochromatin and inversely regulate differentiation. <em>Cell</em> <strong>152</strong>: 584–98.</p>
</div>
<div id="ref-Song2010">
<p>Song L, Crawford GE. 2010. DNase-seq: A high-resolution technique for mapping active gene regulatory elements across the genome from mammalian cells. <em>Cold Spring Harbor protocols</em> <strong>2010</strong>: pdb.prot5384.</p>
</div>
<div id="ref-Spielmann2012">
<p>Spielmann M, Brancati F, Krawitz PM, Robinson PN, Ibrahim DM, Franke M, Hecht J, Lohan S, Dathe K, Nardone AM, et al. 2012. Homeotic arm-to-leg transformation associated with genomic rearrangements at the pitx1 locus. <em>American journal of human genetics</em> <strong>91</strong>: 629–35.</p>
</div>
<div id="ref-Spielmann2016">
<p>Spielmann M, Mundlos S. 2016. Looking beyond the genes: The role of non-coding variants in human disease. <em>Human Molecular Genetics</em> ddw205.</p>
</div>
<div id="ref-Spielmann2013a">
<p>Spielmann M, Mundlos S. 2013. Structural variations, the regulatory landscape of the genome and their alteration in human disease. <em>BioEssays : news and reviews in molecular, cellular and developmental biology</em> 1–11.</p>
</div>
<div id="ref-Spitz2012">
<p>Spitz F, Furlong EEM. 2012. Transcription factors: From enhancer binding to developmental control. <em>Nature reviews Genetics</em> <strong>13</strong>: 613–26.</p>
</div>
<div id="ref-Sproul2005">
<p>Sproul D, Gilbert N, Bickmore W a. 2005. The role of chromatin structure in regulating the expression of clustered genes. <em>Nature reviews Genetics</em> <strong>6</strong>: 775–781.</p>
</div>
<div id="ref-Stankiewicz2002">
<p>Stankiewicz P, Lupski JR. 2002. Genome architecture, rearrangements and genomic disorders. <em>Trends in Genetics</em> <strong>18</strong>: 74–82.</p>
</div>
<div id="ref-Stankiewicz2010">
<p>Stankiewicz P, Lupski JR. 2010. Structural variation in the human genome and its role in disease. <em>Annual Review of Medicine</em> <strong>61</strong>: 437–455.</p>
</div>
<div id="ref-Starick2015">
<p>Starick SR, Ibn-Salem J, Jurk M, Hernandez C, Love MI, Chung H-R, Vingron M, Thomas-Chollier M, Meijsing SH, Hernandez C, et al. 2015. ChIP-exo signal associated with dna-binding motifs provide insights into the genomic binding of the glucocorticoid receptor and cooperating transcription factors. <em>Genome research</em> <strong>25</strong>: 825–35.</p>
</div>
<div id="ref-Stevens2017">
<p>Stevens TJ, Lando D, Basu S, Liam P, Cao Y, Lee SF, Leeb M, Wohlfahrt KJ, Boucher W, Shaughnessy-kirwan AO, et al. 2017. 3D structures of individual mammalian genomes studied by single-cell hi-c. <em>Nature</em> 1–21.</p>
</div>
<div id="ref-Strom2017">
<p>Strom AR, Emelyanov AV, Mir M, Fyodorov DV, Darzacq X, Karpen GH. 2017. Phase separation drives heterochromatin domain formation. <em>Nature Publishing Group</em> <strong>547</strong>: 241–245.</p>
</div>
<div id="ref-Sudmant2015">
<p>Sudmant PH, Rausch T, Gardner EJ, Handsaker RE, Abyzov A, Huddleston J, Zhang Y, Ye K, Jun G, Hsi-Yang Fritz M, et al. 2015. An integrated map of structural variation in 2,504 human genomes. <em>Nature</em> <strong>526</strong>: 75–81.</p>
</div>
<div id="ref-Symmons2016">
<p>Symmons O, Pan L, Remeseiro S, Aktas T, Klein F, Huber W, Spitz F, Amano T, Sagai T, Tanabe H, et al. 2016. The shh topological domain facilitates the action of remote enhancers by reducing the effects of genomic distances. <em>Developmental Cell</em> <strong>0</strong>: 47–57.</p>
</div>
<div id="ref-Symmons2014">
<p>Symmons O, Uslu VV, Tsujimura T, Ruf S, Nassari S, Schwarzer W, Ettwiller L, Spitz F. 2014. Functional and topological characteristics of mammalian regulatory domains. <em>Genome research</em> <strong>24</strong>: 390–400.</p>
</div>
<div id="ref-Talkowski2011">
<p>Talkowski ME, Ernst C, Heilbut A, Chiang C, Hanscom C, Lindgren A, Kirby A, Liu S, Muddukrishna B, Ohsumi TK, et al. 2011. Next-generation sequencing strategies enable routine detection of balanced chromosome rearrangements for clinical diagnostics and genetic research. <em>The American Journal of Human Genetics</em> <strong>88</strong>: 469–481.</p>
</div>
<div id="ref-Tang2013">
<p>Tang Z, Berlin DS, Toji L, Toruner GA, Beiswanger C, Kulkarni S, Martin CL, Emanuel BS, Christman M, Gerry NP. 2013. A dynamic database of microarray-characterized cell lines with various cytogenetic and genomic backgrounds. <em>G3 (Bethesda, Md)</em> <strong>3</strong>: 1143–9.</p>
</div>
<div id="ref-Tang2015">
<p>Tang Z, Luo OJ, Li X, Zheng M, Zhu JJ, Szalaj P, Trzaskoma P, Magalska A, Wlodarczyk J, Ruszczycki B, et al. 2015. CTCF-mediated human 3D genome architecture reveals chromatin topology for transcription. <em>Cell</em> 1–17.</p>
</div>
<div id="ref-Theisen2010">
<p>Theisen A, Shaffer LG. 2010. Disorders caused by chromosome abnormalities. <em>The application of clinical genetics</em> <strong>3</strong>: 159–74.</p>
</div>
<div id="ref-Thurman2012">
<p>Thurman RE, Rynes E, Humbert R, Vierstra J, Maurano MT, Haugen E, Sheffield NC, Stergachis AB, Wang H, Vernot B, et al. 2012. The accessible chromatin landscape of the human genome. <em>Nature</em> <strong>489</strong>: 75–82.</p>
</div>
<div id="ref-Tiwari2008">
<p>Tiwari VK, Cope L, McGarvey KM, Ohm JE, Baylin SB. 2008. A novel 6C assay uncovers polycomb-mediated higher order chromatin conformations. <em>Genome research</em> <strong>18</strong>: 1171–9.</p>
</div>
<div id="ref-Tolhuis2002">
<p>Tolhuis B, Palstra RJ, Splinter E, Grosveld F, Laat W de. 2002. Looping and interaction between hypersensitive sites in the active beta-globin locus. <em>Mol Cell</em> <strong>10</strong>: 1453–1465.</p>
</div>
<div id="ref-Trembath2004">
<p>Trembath DG, Semina EV, Jones DH, Patil SR, Qian Q, Amendt BA, Russo AF, Murray JC. 2004. Analysis of two translocation breakpoints and identification of a negative regulatory element in patients with rieger’s syndrome. <em>Birth Defects Research Part A: Clinical and Molecular Teratology</em> <strong>70</strong>: 82–91.</p>
</div>
<div id="ref-Ulianov2017">
<p>Ulianov SV, Galitsyna AA, Flyamer IM, Golov AK, Khrameeva EE, Imakaev MV, Abdennur NA, Gelfand MS, Gavrilov AA, Razin SV. 2017. Activation of the alpha-globin gene expression correlates with dramatic upregulation of nearby non-globin genes and changes in local and large-scale chromatin spatial structure. <em>Epigenetics &amp; Chromatin</em> <strong>10</strong>: 35.</p>
</div>
<div id="ref-Ulianov2015">
<p>Ulianov SV, Khrameeva EE, Gavrilov AA, Flyamer IM, Kos P, Mikhaleva EA, Penin AA, Logacheva MD, Imakaev MV, Chertovich A, et al. 2015. Active chromatin and transcription play a key role in chromosome partitioning into topologically associating domains. <em>Genome research</em> <strong>26</strong>: 70–84.</p>
</div>
<div id="ref-Bortle2014">
<p>Van Bortle K, Nichols MH, Li L, Ong C-T, Takenaka N, Qin ZS, Corces VG. 2014. Insulator function and topological domain border strength scale with architectural protein occupancy. <em>Genome biology</em> <strong>15</strong>: R82.</p>
</div>
<div id="ref-Velagaleti2005">
<p>Velagaleti GV, Bien-Willner GA, Northup JK, Lockhart LH, Hawkins JC, Jalal SM, Withers M, Lupski JR, Stankiewicz P. 2005. Position effects due to chromosome breakpoints that map ∼900 kb upstream and ∼1.3 mb downstream of sox9 in two patients with campomelic dysplasia. <em>The American Journal of Human Genetics</em> <strong>76</strong>: 652–662.</p>
</div>
<div id="ref-Vetro2015">
<p>Vetro A, Dehghani MR, Kraoua L, Giorda R, Beri S, Cardarelli L, Merico M, Manolakos E, Parada-Bustamante A, Castro A, et al. 2015. Testis development in the absence of sry: Chromosomal rearrangements at sox9 and sox3. <em>European Journal of Human Genetics</em> <strong>23</strong>: 1025–1032.</p>
</div>
<div id="ref-VietriRudan2015">
<p>Vietri Rudan M, Barrington C, Henderson S, Ernst C, Odom DT, Tanay A, Hadjur S. 2015. Comparative hi-c reveals that ctcf underlies evolution of chromosomal domain architecture. <em>Cell reports</em> <strong>10</strong>: 1297–309.</p>
</div>
<div id="ref-Vilella2009">
<p>Vilella AJ, Severin J, Ureta-Vidal A, Heng L, Durbin R, Birney E. 2009. EnsemblCompara genetrees: Complete, duplication-aware phylogenetic trees in vertebrates. <em>Genome Research</em> <strong>19</strong>: 327–335.</p>
</div>
<div id="ref-Visel2007">
<p>Visel A, Minovitsky S, Dubchak I, Pennacchio L a. 2007. VISTA enhancer browser–a database of tissue-specific human enhancers. <em>Nucleic acids research</em> <strong>35</strong>: D88–92.</p>
</div>
<div id="ref-Visser2012">
<p>Visser M, Kayser M, Palstra R-J. 2012. HERC2 rs12913832 modulates human pigmentation by attenuating chromatin-loop formation between a long-range enhancer and the oca2 promoter. <em>Genome research</em> <strong>22</strong>: 446–55.</p>
</div>
<div id="ref-Walter2003">
<p>Walter J, Schermelleh L, Cremer M, Tashiro S, Cremer T. 2003. Chromosome order in hela cells changes during mitosis and early g1, but is stably maintained during subsequent interphase stages. <em>The Journal of cell biology</em> <strong>160</strong>: 685–97.</p>
</div>
<div id="ref-Wang2016d">
<p>Wang S, Su J-H, Beliveau BJ, Bintu B, Moffitt JR, Wu C-t, Zhuang X. 2016. Spatial organization of chromatin domains and compartments in single chromosomes. <em>Science</em> <strong>353</strong>: 598–602.</p>
</div>
<div id="ref-Waszak2015">
<p>Waszak SM, Delaneau O, Gschwind AR, Kilpinen H, Raghav SK, Witwicki RM, Orioli A, Wiederkehr M, Panousis NI, Yurovsky A, et al. 2015. Population variation and genetic control of modular chromatin architecture in humans. <em>Cell</em> 1–12.</p>
</div>
<div id="ref-Weiler1995">
<p>Weiler KS, Wakimoto BT. 1995. Heterochromatin and gene expression in drosophila. <em>Annual review of genetics</em> <strong>29</strong>: 577–605.</p>
</div>
<div id="ref-Weintraub2018">
<p>Weintraub AS, Li CH, Zamudio AV, Sigova AA, Hannet NM, Day DS, Abraham BJ, Cohen MA, Nabet B, Buckley DL, et al. 2018. YY1 is a structural regulator of enhancer-promoter loops. <em>Cell</em> 1–16.</p>
</div>
<div id="ref-Weischenfeldt2016">
<p>Weischenfeldt J, Dubash T, Drainas AP, Mardin BR, Chen Y, Stütz AM, Waszak SM, Bosco G, Halvorsen AR, Raeder B, et al. 2016. Pan-cancer analysis of somatic copy-number alterations implicates irs4 and igf2 in enhancer hijacking. <em>Nature Genetics</em>.</p>
</div>
<div id="ref-Whalen2016">
<p>Whalen S, Truty RM, Pollard KS. 2015. Enhancer-promoter interactions are encoded by complex genomic signatures on looping chromatin. <em>Nature Genetics</em> <strong>48</strong>: 1–10.</p>
</div>
<div id="ref-Whyte2013">
<p>Whyte WA, Orlando DA, Hnisz D, Abraham BJ, Lin CY, Kagey MH, Rahl PB, Lee TI, Young RA. 2013. Master transcription factors and mediator establish super-enhancers at key cell identity genes. <em>Cell</em> <strong>153</strong>: 307–19.</p>
</div>
<div id="ref-Wickham2017">
<p>Wickham H, Grolemund G. 2017. <em>R for data science : Import, tidy, transform, visualize, and model data</em>.</p>
</div>
<div id="ref-Will2017">
<p>Will AJ, Cova G, Osterwalder M, Chan W-L, Wittler L, Brieske N, Heinrich V, Villartay J-P de, Vingron M, Klopocki E, et al. 2017. Composition and dosage of a multipartite enhancer cluster control developmental expression of ihh (indian hedgehog). <em>Nature Genetics</em>.</p>
</div>
<div id="ref-DeWit2012">
<p>Wit E de, Laat W de. 2012. A decade of 3C technologies: Insights into nuclear organization. <em>Genes &amp; development</em> <strong>26</strong>: 11–24.</p>
</div>
<div id="ref-deWit2015">
<p>Wit E de, Vos ESM, Holwerda SJB, Valdes-Quezada C, Verstegen MJAM, Teunissen H, Splinter E, Wijchers PJ, Krijger PHL, Laat W de. 2015. CTCF binding polarity determines chromatin looping. <em>Molecular Cell</em> <strong>60</strong>: 676–684.</p>
</div>
<div id="ref-Won2016">
<p>Won H, Torre-Ubieta L de la, Stein JL, Parikshak NN, Huang J, Opland CK, Gandal MJ, Sutton GJ, Hormozdiari F, Lu D, et al. 2016. Chromosome conformation elucidates regulatory relationships in developing human brain. <em>Nature</em> 1–20.</p>
</div>
<div id="ref-Xie2017">
<p>Xie S, Duan J, Li B, Zhou P, Hon GC. 2017. Multiplexed engineering and analysis of combinatorial enhancer activity in single cells. <em>Molecular Cell</em> <strong>66</strong>: 285–299.e5.</p>
</div>
<div id="ref-Yardmc2014">
<p>Yardımcı GG, Frank CL, Crawford GE, Ohler U. 2014. Explicit dnase sequence bias modeling enables high-resolution transcription factor footprint detection. <em>Nucleic acids research</em> 1–14.</p>
</div>
<div id="ref-Ye2016b">
<p>Ye B-Y, Shen W-L, Wang D, Li P, Zhang Z, Shi M-L, Zhang Y, Zhang F-X, Zhao Z-H. 2016. ZNF143 is involved in ctcf-mediated chromatin interactions by cooperation with cohesin and other partners. <em>Molecular Biology</em> <strong>50</strong>: 431–437.</p>
</div>
<div id="ref-Yu2017">
<p>Yu M, Ren B. 2017. The three-dimensional organization of mammalian genomes. <em>Annual Review of Cell and Developmental Biology</em> <strong>33</strong>: annurev–cellbio–100616–060531.</p>
</div>
<div id="ref-Zepeda-Mendoza2017">
<p>Zepeda-Mendoza CJ, Ibn-Salem J, Kammin T, Harris DJ, Rita D, Gripp KW, MacKenzie JJ, Gropman A, Graham B, Shaheen R, et al. 2017. Computational prediction of position effects of apparently balanced human chromosomal rearrangements. <em>American journal of human genetics</em> <strong>101</strong>: 206–217.</p>
</div>
<div id="ref-Zhan2017">
<p>Zhan Y, Mariani L, Barozzi I, Schulz EG, Blüthgen N, Stadler M, Tiana G, Giorgetti L, Bluthgen N, Stadler M, et al. 2017. Reciprocal insulation analysis of hi-c data shows that tads represent a functionally but not structurally privileged scale in the hierarchical folding of chromosomes. <em>Genome Research</em> <strong>27</strong>: gr.212803.116.</p>
</div>
<div id="ref-Zhang2008">
<p>Zhang B, Park B-H, Karpinets T, Samatova NF. 2008. From pull-down data to protein interaction networks and complexes with biological relevance. <em>Bioinformatics (Oxford, England)</em> <strong>24</strong>: 979–86.</p>
</div>
<div id="ref-Zhang2015">
<p>Zhang B, Wolynes PG. 2015. Topology, structures, and energy landscapes of human chromosomes. <em>Proceedings of the National Academy of Sciences</em> <strong>112</strong>: 201506257.</p>
</div>
<div id="ref-Zhang2009">
<p>Zhang F, Gu W, Hurles ME, Lupski JR. 2009. Copy number variation in human health, disease, and evolution. <em>Annual Review of Genomics and Human Genetics</em> <strong>10</strong>: 451–481.</p>
</div>
<div id="ref-Zhao2016">
<p>Zhao C, Li X, Hu H. 2016. PETModule: A motif module based approach for enhancer target gene prediction. <em>Scientific Reports</em> <strong>6</strong>: 30043.</p>
</div>
<div id="ref-Zhou2013">
<p>Zhou X, Lowdon RF, Li D, Lawson H a, Madden P a F, Costello JF, Wang T. 2013. Exploring long-range genome interactions using the washu epigenome browser. <em>Nature methods</em> <strong>10</strong>: 375–6.</p>
</div>
<div id="ref-Zhou2012">
<p>Zhou X, Wang T. 2012. Using the wash u epigenome browser to examine genome-wide sequencing data. In <em>Current protocols in bioinformatics</em>, Vol. Chapter 10 of, p. Unit10.10, John Wiley &amp; Sons, Inc., Hoboken, NJ, USA.</p>
</div>
<div id="ref-Zhu2015">
<p>Zhu B, Zhang W, Zhang T, Liu B, Jiang J. 2015. Genome-wide prediction and validation of intergenic enhancers in arabidopsis using open chromatin signatures. <em>The Plant cell</em> <strong>27</strong>: 2415–26.</p>
</div>
<div id="ref-Zhu2016">
<p>Zhu Y, Chen Z, Zhang K, Wang M, Medovoy D, Whitaker JW, Ding B, Li N, Zheng L, Wang W. 2016. Constructing 3D interaction maps from 1D epigenomes. <em>Nature Communications</em> <strong>7</strong>: 10812.</p>
</div>
<div id="ref-Zuin2014">
<p>Zuin J, Dixon JR, Reijden MIJa van der, Ye Z, Kolovos P, Brouwer RWW, Corput MPC van de, Werken HJG van de, Knoch Ta, IJcken WFJ van, et al. 2014. Cohesin and ctcf differentially affect chromatin architecture and gene expression in human cells. <em>Proceedings of the National Academy of Sciences of the United States of America</em> <strong>111</strong>: 996–1001.</p>
</div>
</div>

</div>



<!-- ---------------------------------------------------------------- -->
<!-- # Co-regulation of paralog genes: Supporting Information {#sup-paralog-regulation} -->
</div>
            </section>

          </div>
        </div>
      </div>
<a href="6-8-conclusions.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="A-sup-paralog-regulation.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": true,
"facebook": false,
"twitter": true,
"google": false,
"weibo": false,
"instapper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/ibn-salem/thesis/edit/master/07-references.Rmd",
"text": "Edit"
},
"download": ["thesis.pdf", "thesis.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://cdn.bootcss.com/mathjax/2.7.1/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:" && /^https?:/.test(src))
      src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
