<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>Genome folding in evolution and disease</title>
  <meta name="description" content="This is a draft PhD thesis.">
  <meta name="generator" content="bookdown 0.5.10 and GitBook 2.6.7">

  <meta property="og:title" content="Genome folding in evolution and disease" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="This is a draft PhD thesis." />
  <meta name="github-repo" content="ibn-salem/thesis" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Genome folding in evolution and disease" />
  
  <meta name="twitter:description" content="This is a draft PhD thesis." />
  

<meta name="author" content="Jonas Ibn-Salem">


<meta name="date" content="2018-01-25">

  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  
<link rel="prev" href="discussion-3.html">
<link rel="next" href="supporting-information-co-regulation-of-paralog-genes-in-the-three-dimensional-chromatin-architecture.html">
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />










<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Thesis</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Preface</a></li>
<li class="chapter" data-level="1" data-path="introduction.html"><a href="introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a><ul>
<li class="chapter" data-level="1.1" data-path="introduction.html"><a href="introduction.html#structure-of-the-intorduction"><i class="fa fa-check"></i><b>1.1</b> Structure of the Intorduction</a></li>
<li class="chapter" data-level="1.2" data-path="introduction.html"><a href="introduction.html#cell-diversity-and-regulation-of-gene-expression"><i class="fa fa-check"></i><b>1.2</b> Cell diversity and regulation of gene expression</a></li>
<li class="chapter" data-level="1.3" data-path="introduction.html"><a href="introduction.html#enhancers"><i class="fa fa-check"></i><b>1.3</b> Enhancers</a></li>
<li class="chapter" data-level="1.4" data-path="introduction.html"><a href="introduction.html#methods-to-probe-the-3d-chromatin-archtiecture"><i class="fa fa-check"></i><b>1.4</b> Methods to probe the 3D chromatin archtiecture</a><ul>
<li class="chapter" data-level="1.4.1" data-path="introduction.html"><a href="introduction.html#microscopy-based-techniques-to-visuallize-the-genome-in-3d"><i class="fa fa-check"></i><b>1.4.1</b> Microscopy-based techniques to visuallize the genome in 3D</a></li>
<li class="chapter" data-level="1.4.2" data-path="introduction.html"><a href="introduction.html#proximity-ligation-based-method-to-quantify-chromatin-interactions"><i class="fa fa-check"></i><b>1.4.2</b> Proximity-ligation based method to quantify chromatin interactions</a></li>
</ul></li>
<li class="chapter" data-level="1.5" data-path="introduction.html"><a href="introduction.html#hierarchy-of-chromatin-3d-structure"><i class="fa fa-check"></i><b>1.5</b> Hierarchy of chromatin 3D structure</a><ul>
<li class="chapter" data-level="1.5.1" data-path="introduction.html"><a href="introduction.html#chromosomal-territories-and-inter-chromosomal-contacts"><i class="fa fa-check"></i><b>1.5.1</b> Chromosomal territories and inter-chromosomal contacts</a></li>
<li class="chapter" data-level="1.5.2" data-path="introduction.html"><a href="introduction.html#ab-compartments"><i class="fa fa-check"></i><b>1.5.2</b> A/B compartments</a></li>
<li class="chapter" data-level="1.5.3" data-path="introduction.html"><a href="introduction.html#topologically-associating-domains-tads"><i class="fa fa-check"></i><b>1.5.3</b> Topologically associating domains (TADs)</a></li>
<li class="chapter" data-level="1.5.4" data-path="introduction.html"><a href="introduction.html#chromatin-loops"><i class="fa fa-check"></i><b>1.5.4</b> Chromatin loops</a></li>
</ul></li>
<li class="chapter" data-level="1.6" data-path="introduction.html"><a href="introduction.html#dynamics-of-chromatin-structure"><i class="fa fa-check"></i><b>1.6</b> Dynamics of chromatin structure</a></li>
<li class="chapter" data-level="1.7" data-path="introduction.html"><a href="introduction.html#changes-of-genome-folding-in-evoltuion-and-disease-genomes"><i class="fa fa-check"></i><b>1.7</b> Changes of genome folding in evoltuion and disease genomes</a></li>
<li class="chapter" data-level="1.8" data-path="introduction.html"><a href="introduction.html#aims-of-this-thesis"><i class="fa fa-check"></i><b>1.8</b> Aims of this thesis</a><ul>
<li class="chapter" data-level="" data-path="introduction.html"><a href="introduction.html#is-the-three-dimensonal-folding-of-genomes-associated-with-co-regulation-of-functionally-related-genes"><i class="fa fa-check"></i>Is the three-dimensonal folding of genomes associated with co-regulation of functionally related genes?</a></li>
<li class="chapter" data-level="" data-path="introduction.html"><a href="introduction.html#are-tads-functional-buidling-blocks-of-genomes-and-subjected-to-selective-preasure-during-evolution"><i class="fa fa-check"></i>Are TADs functional buidling blocks of genomes and subjected to selective preasure during evolution?</a></li>
<li class="chapter" data-level="" data-path="introduction.html"><a href="introduction.html#is-the-disruption-of-tads-by-rearrangements-also-associated-to-genetic-diseases"><i class="fa fa-check"></i>Is the disruption of TADs by rearrangements also associated to genetic diseases?</a></li>
<li class="chapter" data-level="" data-path="introduction.html"><a href="introduction.html#can-chromatin-looping-interactions-be-predicted-by-chip-seq-and-sequence-features"><i class="fa fa-check"></i>Can chromatin looping interactions be predicted by ChIP-seq and sequence features?</a></li>
</ul></li>
<li class="chapter" data-level="1.9" data-path="introduction.html"><a href="introduction.html#structure-of-this-thesis"><i class="fa fa-check"></i><b>1.9</b> Structure of this thesis</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html"><i class="fa fa-check"></i><b>2</b> Paralog genes in the 3D genome architecture</a><ul>
<li class="chapter" data-level="2.1" data-path="introduction.html"><a href="introduction.html#introduction"><i class="fa fa-check"></i><b>2.1</b> Introduction</a></li>
<li class="chapter" data-level="2.2" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#sec:paralog-methods"><i class="fa fa-check"></i><b>2.2</b> MATERIALS AND METHODS</a><ul>
<li class="chapter" data-level="2.2.1" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#selection-of-pairs-of-paralog-genes"><i class="fa fa-check"></i><b>2.2.1</b> Selection of pairs of paralog genes</a></li>
<li class="chapter" data-level="2.2.2" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#enhancers-to-gene-association"><i class="fa fa-check"></i><b>2.2.2</b> Enhancers to gene association</a></li>
<li class="chapter" data-level="2.2.3" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#topological-associating-domains"><i class="fa fa-check"></i><b>2.2.3</b> Topological associating domains</a></li>
<li class="chapter" data-level="2.2.4" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#hi-c-interaction-maps"><i class="fa fa-check"></i><b>2.2.4</b> Hi-C interaction maps</a></li>
<li class="chapter" data-level="2.2.5" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#randomization"><i class="fa fa-check"></i><b>2.2.5</b> Randomization</a></li>
<li class="chapter" data-level="2.2.6" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#statistical-tests"><i class="fa fa-check"></i><b>2.2.6</b> Statistical tests</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#results"><i class="fa fa-check"></i><b>2.3</b> RESULTS</a><ul>
<li class="chapter" data-level="2.3.1" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#distribution-of-paralog-genes-in-the-human-genome"><i class="fa fa-check"></i><b>2.3.1</b> Distribution of paralog genes in the human genome</a></li>
<li class="chapter" data-level="2.3.2" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#co-expression-of-paralog-gene-pairs-across-tissues"><i class="fa fa-check"></i><b>2.3.2</b> Co-expression of paralog gene pairs across tissues</a></li>
<li class="chapter" data-level="2.3.3" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#paralog-genes-share-enhancers"><i class="fa fa-check"></i><b>2.3.3</b> Paralog genes share enhancers</a></li>
<li class="chapter" data-level="2.3.4" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#co-localization-of-paralogs-in-tads"><i class="fa fa-check"></i><b>2.3.4</b> Co-localization of paralogs in TADs</a></li>
<li class="chapter" data-level="2.3.5" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#distal-paralog-pairs-are-enriched-for-long-range-chromatin-contacts"><i class="fa fa-check"></i><b>2.3.5</b> Distal paralog pairs are enriched for long-range chromatin contacts</a></li>
<li class="chapter" data-level="2.3.6" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#close-paralogs-have-fewer-contacts-than-expected"><i class="fa fa-check"></i><b>2.3.6</b> Close paralogs have fewer contacts than expected</a></li>
<li class="chapter" data-level="2.3.7" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#paralogs-in-mouse-and-dog-genome"><i class="fa fa-check"></i><b>2.3.7</b> Paralogs in mouse and dog genome</a></li>
<li class="chapter" data-level="2.3.8" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#orthologs-of-human-paralogs-show-conserved-co-localization"><i class="fa fa-check"></i><b>2.3.8</b> Orthologs of human paralogs show conserved co-localization</a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#discussion"><i class="fa fa-check"></i><b>2.4</b> DISCUSSION</a></li>
<li class="chapter" data-level="2.5" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#conclusion"><i class="fa fa-check"></i><b>2.5</b> CONCLUSION</a></li>
<li class="chapter" data-level="2.6" data-path="ch-paralog-regulaion.html"><a href="ch-paralog-regulaion.html#acknowledgements"><i class="fa fa-check"></i><b>2.6</b> ACKNOWLEDGEMENTS</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html"><i class="fa fa-check"></i><b>3</b> Stability of TADs in evolution</a><ul>
<li class="chapter" data-level="" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#abstract"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#keywords"><i class="fa fa-check"></i>Keywords</a></li>
<li class="chapter" data-level="3.1" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#introduction-1"><i class="fa fa-check"></i><b>3.1</b> Introduction</a></li>
<li class="chapter" data-level="3.2" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#results-1"><i class="fa fa-check"></i><b>3.2</b> Results</a><ul>
<li class="chapter" data-level="3.2.1" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#identification-of-evolutionary-rearrangement-breakpoints-from-whole-genome-alignments"><i class="fa fa-check"></i><b>3.2.1</b> Identification of evolutionary rearrangement breakpoints from whole-genome alignments</a></li>
<li class="chapter" data-level="3.2.2" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#rearrangement-breakpoints-are-enriched-at-tad-boundaries"><i class="fa fa-check"></i><b>3.2.2</b> Rearrangement breakpoints are enriched at TAD boundaries</a></li>
<li class="chapter" data-level="3.2.3" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#clusters-of-conserved-non-coding-elements-are-depleted-for-rearrangement-breakpoints"><i class="fa fa-check"></i><b>3.2.3</b> Clusters of conserved non-coding elements are depleted for rearrangement breakpoints</a></li>
<li class="chapter" data-level="3.2.4" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#rearranged-tads-are-associated-with-divergent-gene-expression-between-species"><i class="fa fa-check"></i><b>3.2.4</b> Rearranged TADs are associated with divergent gene expression between species</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#discussion-1"><i class="fa fa-check"></i><b>3.3</b> Discussion</a></li>
<li class="chapter" data-level="3.4" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#conclusion-1"><i class="fa fa-check"></i><b>3.4</b> Conclusion</a></li>
<li class="chapter" data-level="3.5" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#methods"><i class="fa fa-check"></i><b>3.5</b> Methods</a><ul>
<li class="chapter" data-level="3.5.1" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#rearrangement-breakpoints-from-whole-genome-alignments"><i class="fa fa-check"></i><b>3.5.1</b> Rearrangement breakpoints from whole-genome alignments</a></li>
<li class="chapter" data-level="3.5.2" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#topologically-associating-domains-and-contact-domains"><i class="fa fa-check"></i><b>3.5.2</b> Topologically associating domains and contact domains</a></li>
<li class="chapter" data-level="3.5.3" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#breakpoint-distributions-at-tads"><i class="fa fa-check"></i><b>3.5.3</b> Breakpoint distributions at TADs</a></li>
<li class="chapter" data-level="3.5.4" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#quantification-of-breakpoint-enrichment"><i class="fa fa-check"></i><b>3.5.4</b> Quantification of breakpoint enrichment</a></li>
<li class="chapter" data-level="3.5.5" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#expression-data-for-mouse-and-human-orthologs"><i class="fa fa-check"></i><b>3.5.5</b> Expression data for mouse and human orthologs</a></li>
<li class="chapter" data-level="3.5.6" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#classification-of-tads-and-genes-according-to-rearrangements-and-grbs"><i class="fa fa-check"></i><b>3.5.6</b> Classification of TADs and genes according to rearrangements and GRBs</a></li>
<li class="chapter" data-level="3.5.7" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#source-code-and-implementation-details"><i class="fa fa-check"></i><b>3.5.7</b> Source code and implementation details</a></li>
</ul></li>
<li class="chapter" data-level="3.6" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#declarations"><i class="fa fa-check"></i><b>3.6</b> Declarations</a><ul>
<li class="chapter" data-level="3.6.1" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#ethics-approval-and-consent-to-participate"><i class="fa fa-check"></i><b>3.6.1</b> Ethics approval and consent to participate</a></li>
<li class="chapter" data-level="3.6.2" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#consent-for-publication"><i class="fa fa-check"></i><b>3.6.2</b> Consent for publication</a></li>
<li class="chapter" data-level="3.6.3" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#availability-of-data-and-material"><i class="fa fa-check"></i><b>3.6.3</b> Availability of data and material</a></li>
<li class="chapter" data-level="3.6.4" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#competing-interests"><i class="fa fa-check"></i><b>3.6.4</b> Competing interests</a></li>
<li class="chapter" data-level="3.6.5" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#funding"><i class="fa fa-check"></i><b>3.6.5</b> Funding</a></li>
<li class="chapter" data-level="3.6.6" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#authors-contributions"><i class="fa fa-check"></i><b>3.6.6</b> Authors‘ contributions</a></li>
<li class="chapter" data-level="3.6.7" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#acknowledgments"><i class="fa fa-check"></i><b>3.6.7</b> Acknowledgments</a></li>
</ul></li>
<li class="chapter" data-level="3.7" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#tables"><i class="fa fa-check"></i><b>3.7</b> Tables</a><ul>
<li class="chapter" data-level="3.7.1" data-path="ch-TAD-evolution.html"><a href="ch-TAD-evolution.html#table-1"><i class="fa fa-check"></i><b>3.7.1</b> Table 1</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="ch-position-effect.html"><a href="ch-position-effect.html"><i class="fa fa-check"></i><b>4</b> Position effects of rearrangements in disease genomes</a><ul>
<li class="chapter" data-level="4.1" data-path="ch-position-effect.html"><a href="ch-position-effect.html#abstract-1"><i class="fa fa-check"></i><b>4.1</b> Abstract</a></li>
<li class="chapter" data-level="4.2" data-path="ch-position-effect.html"><a href="ch-position-effect.html#introduction-2"><i class="fa fa-check"></i><b>4.2</b> Introduction</a></li>
<li class="chapter" data-level="4.3" data-path="ch-position-effect.html"><a href="ch-position-effect.html#materials-and-methods"><i class="fa fa-check"></i><b>4.3</b> Materials and Methods</a><ul>
<li class="chapter" data-level="4.3.1" data-path="ch-position-effect.html"><a href="ch-position-effect.html#selection-of-subjects-with-apparently-balanced-chromosome-abnormalities"><i class="fa fa-check"></i><b>4.3.1</b> Selection of subjects with apparently balanced chromosome abnormalities</a></li>
<li class="chapter" data-level="4.3.2" data-path="ch-position-effect.html"><a href="ch-position-effect.html#clinical-descriptions-of-dgap-cases"><i class="fa fa-check"></i><b>4.3.2</b> Clinical descriptions of DGAP cases</a></li>
<li class="chapter" data-level="4.3.3" data-path="ch-position-effect.html"><a href="ch-position-effect.html#analysis-of-genes-bordering-the-rearrangement-breakpoints"><i class="fa fa-check"></i><b>4.3.3</b> Analysis of genes bordering the rearrangement breakpoints</a></li>
<li class="chapter" data-level="4.3.4" data-path="ch-position-effect.html"><a href="ch-position-effect.html#assessment-of-disrupted-functional-elements-and-chromatin-interactions"><i class="fa fa-check"></i><b>4.3.4</b> Assessment of disrupted functional elements and chromatin interactions</a></li>
<li class="chapter" data-level="4.3.5" data-path="ch-position-effect.html"><a href="ch-position-effect.html#ontological-analysis-of-genes-neighboring-breakpoints"><i class="fa fa-check"></i><b>4.3.5</b> Ontological analysis of genes neighboring breakpoints</a></li>
<li class="chapter" data-level="4.3.6" data-path="ch-position-effect.html"><a href="ch-position-effect.html#quantitative-real-time-pcr"><i class="fa fa-check"></i><b>4.3.6</b> Quantitative real-time PCR</a></li>
<li class="chapter" data-level="4.3.7" data-path="ch-position-effect.html"><a href="ch-position-effect.html#assessment-of-dgap-breakpoints-overlapping-with-non-coding-structural-variants-in-public-databases"><i class="fa fa-check"></i><b>4.3.7</b> Assessment of DGAP breakpoints overlapping with non-coding structural variants in public databases</a></li>
</ul></li>
<li class="chapter" data-level="4.4" data-path="ch-position-effect.html"><a href="ch-position-effect.html#results-2"><i class="fa fa-check"></i><b>4.4</b> Results</a><ul>
<li class="chapter" data-level="4.4.1" data-path="ch-position-effect.html"><a href="ch-position-effect.html#genomic-characterization-of-non-coding-breakpoints"><i class="fa fa-check"></i><b>4.4.1</b> Genomic characterization of non-coding breakpoints</a></li>
<li class="chapter" data-level="4.4.2" data-path="ch-position-effect.html"><a href="ch-position-effect.html#identification-of-genes-with-potential-position-effects"><i class="fa fa-check"></i><b>4.4.2</b> Identification of genes with potential position effects</a></li>
<li class="chapter" data-level="4.4.3" data-path="ch-position-effect.html"><a href="ch-position-effect.html#identification-of-subjects-with-shared-non-coding-chromosome-alterations-and-phenotypes"><i class="fa fa-check"></i><b>4.4.3</b> Identification of subjects with shared non-coding chromosome alterations and phenotypes</a></li>
</ul></li>
<li class="chapter" data-level="4.5" data-path="ch-position-effect.html"><a href="ch-position-effect.html#discussion-2"><i class="fa fa-check"></i><b>4.5</b> Discussion</a></li>
<li class="chapter" data-level="4.6" data-path="ch-position-effect.html"><a href="ch-position-effect.html#acknowledgements-1"><i class="fa fa-check"></i><b>4.6</b> Acknowledgements</a></li>
<li class="chapter" data-level="4.7" data-path="ch-position-effect.html"><a href="ch-position-effect.html#tables-1"><i class="fa fa-check"></i><b>4.7</b> Tables</a><ul>
<li class="chapter" data-level="4.7.1" data-path="ch-position-effect.html"><a href="ch-position-effect.html#table-1."><i class="fa fa-check"></i><b>4.7.1</b> Table 1.</a></li>
</ul></li>
<li class="chapter" data-level="4.8" data-path="ch-position-effect.html"><a href="ch-position-effect.html#figures"><i class="fa fa-check"></i><b>4.8</b> Figures</a></li>
<li class="chapter" data-level="4.9" data-path="ch-position-effect.html"><a href="ch-position-effect.html#web-resources"><i class="fa fa-check"></i><b>4.9</b> Web Resources</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="ch-loop-prediction.html"><a href="ch-loop-prediction.html"><i class="fa fa-check"></i><b>5</b> Prediction of chromatin looping interactions</a></li>
<li class="chapter" data-level="6" data-path="discussion-3.html"><a href="discussion-3.html"><i class="fa fa-check"></i><b>6</b> Discussion</a><ul>
<li class="chapter" data-level="6.1" data-path="discussion-3.html"><a href="discussion-3.html#further-directions"><i class="fa fa-check"></i><b>6.1</b> Further directions</a></li>
<li class="chapter" data-level="6.2" data-path="discussion-3.html"><a href="discussion-3.html#conclusions"><i class="fa fa-check"></i><b>6.2</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="appendix"><span><b>Appendix</b></span></li>
<li class="chapter" data-level="A" data-path="supporting-information-co-regulation-of-paralog-genes-in-the-three-dimensional-chromatin-architecture.html"><a href="supporting-information-co-regulation-of-paralog-genes-in-the-three-dimensional-chromatin-architecture.html"><i class="fa fa-check"></i><b>A</b> Supporting Information: Co-regulation of paralog genes in the three-dimensional chromatin architecture</a></li>
<li class="chapter" data-level="B" data-path="supplementary-data-evolutionary-stability-of-topologically-associating-domains-is-associated-with-conserved-gene-regulation.html"><a href="supplementary-data-evolutionary-stability-of-topologically-associating-domains-is-associated-with-conserved-gene-regulation.html"><i class="fa fa-check"></i><b>B</b> Supplementary Data: Evolutionary stability of topologically associating domains is associated with conserved gene regulation</a><ul>
<li class="chapter" data-level="B.1" data-path="supplementary-data-evolutionary-stability-of-topologically-associating-domains-is-associated-with-conserved-gene-regulation.html"><a href="supplementary-data-evolutionary-stability-of-topologically-associating-domains-is-associated-with-conserved-gene-regulation.html#TadEvoSupTab"><i class="fa fa-check"></i><b>B.1</b> Supplementary Tables</a></li>
<li class="chapter" data-level="B.2" data-path="supplementary-data-evolutionary-stability-of-topologically-associating-domains-is-associated-with-conserved-gene-regulation.html"><a href="supplementary-data-evolutionary-stability-of-topologically-associating-domains-is-associated-with-conserved-gene-regulation.html#supplementary-figures"><i class="fa fa-check"></i><b>B.2</b> Supplementary Figures</a></li>
</ul></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Build with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Genome folding in evolution and disease</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="references" class="section level1 unnumbered">
<h1>References</h1>
<div id="refs">
<div id="ref-Acemel2017">
<p>Acemel, R.D., Maeso, I., and Gómez‐Skarmeta, J.L. (2017). Topologically associated domains: a successful scaffold for the evolution of gene regulation in animals. Wiley Interdisciplinary Reviews: Developmental Biology 1–19.</p>
</div>
<div id="ref-Andersson2014">
<p>Andersson, R., Gebhard, C., Miguel-Escalada, I., Hoof, I., Bornholdt, J., Boyd, M., Chen, Y., Zhao, X., Schmidl, C., Suzuki, T., et al. (2014). An atlas of active enhancers across human cell types and tissues. Nature <em>507</em>, 455–461.</p>
</div>
<div id="ref-Andrey2017">
<p>Andrey, G., and Mundlos, S. (2017). The three-dimensional genome: regulating gene expression during pluripotency and development. 3646–3658.</p>
</div>
<div id="ref-GTExConsortium2015">
<p>Ardlie, K.G., DeLuca, D.S., Segrè, A.V., Sullivan, T.J., Young, T.R., Gelfand, E.T., Trowbridge, C.A., Maller, J.B., Tukiainen, T., Lek, M., et al. (2015). The Genotype-Tissue Expression (GTEx) pilot analysis: Multitissue gene regulation in humans. Science <em>348</em>, 648–660.</p>
</div>
<div id="ref-Ay2015">
<p>Ay, F., and Noble, W.S. (2015). Analysis methods for studying the 3D architecture of the genome. Genome Biology <em>16</em>, 183.</p>
</div>
<div id="ref-Banerji1981">
<p>Banerji, J., Rusconi, S., and Schaffner, W. (1981). Expression of a <span class="math inline">\(\beta\)</span>-globin gene is enhanced by remote SV40 DNA sequences. Cell <em>27</em>, 299–308.</p>
</div>
<div id="ref-Becker2015">
<p>Becker, M., Mah, N., Zdzieblo, D., Li, X., Mer, A., Andrade-navarro, M.A., and Mu, A.M. (2015). Epigenetic Mechanisms in Cellular Reprogramming. In Epigenetics and Human Health, A. Meissner, and J. Walter, eds. (Berlin, Heidelberg: Springer Berlin Heidelberg), pp. pp 141–166.</p>
</div>
<div id="ref-Bonev2016">
<p>Bonev, B., and Cavalli, G. (2016). Organization and function of the 3D genome. Nature Reviews Genetics <em>17</em>, 661–678.</p>
</div>
<div id="ref-Boutanaev2002">
<p>Boutanaev, a M., Kalmykova, a I., Shevelyov, Y.Y., Nurminsky, D.I., Smith, M., Evolution, T., Masterpiece, T., Helm, C., and Selection, N. (2002). Large clusters of co-expressed genes in the Drosophila genome. Nature <em>420</em>, 666–669.</p>
</div>
<div id="ref-Branco2006">
<p>Branco, M.R., and Pombo, A. (2006). Intermingling of chromosome territories in interphase suggests role in translocations and transcription-dependent associations. PLoS Biology <em>4</em>, 780–788.</p>
</div>
<div id="ref-Buenrostro2013">
<p>Buenrostro, J.D., Giresi, P.G., Zaba, L.C., Chang, H.Y., and Greenleaf, W.J. (2013). Transposition of native chromatin for fast and sensitive epigenomic profiling of open chromatin, DNA-binding proteins and nucleosome position. Nature Methods <em>10</em>, 1213–1218.</p>
</div>
<div id="ref-Cai2003">
<p>Cai, J., Goodman, B.K., Patel, A.S., Mulliken, J.B., Van Maldergem, L., Hoganson, G.E., Paznekas, W.A., Ben-Neriah, Z., Sheffer, R., Cunningham, M.L., et al. (2003). Increased risk for developmental delay in Saethre-Chotzen syndrome is associated with TWIST deletions: an improved strategy for TWIST mutation screening. Human Genetics <em>114</em>, 68–76.</p>
</div>
<div id="ref-Canela2017">
<p>Canela, A., Maman, Y., Jung, S., Wong, N., Callen, E., Day, A., Kieffer-Kwon, K.-R., Pekowska, A., Zhang, H., Rao, S.S., et al. (2017). Genome Organization Drives Chromosome Fragility. Cell 1–15.</p>
</div>
<div id="ref-Carroll2008">
<p>Carroll, S.B. (2008). Evo-Devo and an Expanding Evolutionary Synthesis: A Genetic Theory of Morphological Evolution. Cell <em>134</em>, 25–36.</p>
</div>
<div id="ref-Carter2002">
<p>Carter, D., Chakalova, L., Osborne, C.S., Dai, Y.F., and Fraser, P. (2002). Long-range chromatin regulatory interactions in vivo. Nat Genet <em>32</em>, 623–626.</p>
</div>
<div id="ref-Claussnitzer2015">
<p>Claussnitzer, M., Dankel, S.N., Kim, K.-H., Quon, G., Meuleman, W., Haugen, C., Glunk, V., Sousa, I.S., Beaudry, J.L., Puviindran, V., et al. (2015). &lt;i&gt;FTO&lt;/i&gt; Obesity Variant Circuitry and Adipocyte Browning in Humans. New England Journal of Medicine <em>373</em>, 895–907.</p>
</div>
<div id="ref-Crane2015">
<p>Crane, E., Bian, Q., McCord, R.P., Lajoie, B.R., Wheeler, B.S., Ralston, E.J., Uzawa, S., Dekker, J., and Meyer, B.J. (2015). Condensin-driven remodelling of X chromosome topology during dosage compensation. Nature.</p>
</div>
<div id="ref-Cremer2001">
<p>Cremer, T., and Cremer, C. (2001). Chromosome territories, nuclear architecture and gene regulation in mammalian cells. Nature Reviews Genetics <em>2</em>, 292–301.</p>
</div>
<div id="ref-Cremer2010">
<p>Cremer, T., and Cremer, M. (2010). Chromosome territories. Cold Spring Harbor Perspectives in Biology <em>2</em>, a003889.</p>
</div>
<div id="ref-Cremer2015">
<p>Cremer, T., Cremer, M., Hübner, B., Strickfaden, H., Smeets, D., Popken, J., Sterr, M., Markaki, Y., Rippe, K., and Cremer, C. (2015). The 4D nucleome: Evidence for a dynamic nuclear landscape based on co-aligned active and inactive nuclear compartments. FEBS Letters.</p>
</div>
<div id="ref-Creyghton2010">
<p>Creyghton, M.P., Cheng, A.W., Welstead, G.G., Kooistra, T., Carey, B.W., Steine, E.J., Hanna, J., Lodato, M.A., Frampton, G.M., Sharp, P.A., et al. (2010). Histone H3K27ac separates active from poised enhancers and predicts developmental state. Proceedings of the National Academy of Sciences of the United States of America <em>107</em>, 21931–21936.</p>
</div>
<div id="ref-Dekker2002">
<p>Dekker, J., Rippe, K., Dekker, M., and Kleckner, N. (2002). Capturing chromosome conformation. Science (New York, N.Y.) <em>295</em>, 1306–1311.</p>
</div>
<div id="ref-Dekker2013">
<p>Dekker, J., Marti-Renom, M.A., and Mirny, L.A. (2013). Exploring the three-dimensional organization of genomes: interpreting chromatin interaction data. Nature Reviews. Genetics <em>14</em>, 390–403.</p>
</div>
<div id="ref-Deng2012">
<p>Deng, W., Lee, J., Wang, H., Miller, J., Reik, A., Gregory, P.D., Dean, A., and Blobel, G. a (2012). Controlling long-range genomic interactions at a native locus by targeted tethering of a looping factor. Cell <em>149</em>, 1233–1244.</p>
</div>
<div id="ref-Dimitrieva2013">
<p>Dimitrieva, S., and Bucher, P. (2013). UCNEbase—a database of ultraconserved non-coding elements and genomic regulatory blocks. Nucleic Acids Research <em>41</em>, D101–D109.</p>
</div>
<div id="ref-Dixon2012">
<p>Dixon, J.R., Selvaraj, S., Yue, F., Kim, A., Li, Y., Shen, Y., Hu, M., Liu, J.S., and Ren, B. (2012). Topological domains in mammalian genomes identified by analysis of chromatin interactions. Nature <em>485</em>, 376–380.</p>
</div>
<div id="ref-Dixon2015">
<p>Dixon, J.R., Jung, I., Selvaraj, S., Shen, Y., Antosiewicz-Bourget, J.E., Lee, A.Y., Ye, Z., Kim, A., Rajagopal, N., Xie, W., et al. (2015). Chromatin architecture reorganization during stem cell differentiation. Nature <em>518</em>, 331–336.</p>
</div>
<div id="ref-Dixon2016">
<p>Dixon, J.R., Gorkin, D.U., and Ren, B. (2016). Chromatin Domains: The Unit of Chromosome Organization. Molecular Cell <em>62</em>, 668–680.</p>
</div>
<div id="ref-Djebali2012">
<p>Djebali, S., Davis, C.a., Merkel, A., Dobin, A., Lassmann, T., Mortazavi, A., Tanzer, A., Lagarde, J., Lin, W., Schlesinger, F., et al. (2012). Landscape of transcription in human cells. Nature <em>489</em>, 101–108.</p>
</div>
<div id="ref-Dostie2006">
<p>Dostie, J., Richmond, T.A., Arnaout, R.A., Selzer, R.R., Lee, W.L., Honan, T.A., Rubio, E.D., Krumm, A., Lamb, J., Nusbaum, C., et al. (2006). Chromosome Conformation Capture Carbon Copy (5C): A massively parallel solution for mapping interactions between genomic elements. Genome Research <em>16</em>, 1299–1309.</p>
</div>
<div id="ref-Durinck2005">
<p>Durinck, S., Moreau, Y., Kasprzyk, A., Davis, S., De Moor, B., Brazma, A., and Huber, W. (2005). BioMart and Bioconductor: A powerful link between biological databases and microarray data analysis. Bioinformatics <em>21</em>, 3439–3440.</p>
</div>
<div id="ref-Durinck2009b">
<p>Durinck, S., Spellman, P.T., Birney, E., and Huber, W. (2009a). Mapping identifiers for the integration of genomic datasets with the R/Bioconductor package biomaRt. Nature Protocols <em>4</em>, 1184–1191.</p>
</div>
<div id="ref-Durinck2009">
<p>Durinck, S., Spellman, P.T., Birney, E., and Huber, W. (2009b). Mapping identifiers for the integration of genomic datasets with the R/Bioconductor package biomaRt. Nature Protocols <em>4</em>, 1184–1191.</p>
</div>
<div id="ref-Eagen2015">
<p>Eagen, K.P., Hartl, T.A., and Kornberg, R.D. (2015). Stable Chromosome Condensation Revealed by Chromosome Conformation Capture. Cell <em>163</em>, 934–946.</p>
</div>
<div id="ref-Eisenberg2013">
<p>Eisenberg, E., and Levanon, E.Y. (2013). Human housekeeping genes, revisited. Trends in Genetics <em>29</em>, 569–574.</p>
</div>
<div id="ref-Engstrom2007">
<p>Engström, P.G., Sui, S.J.H., Drivenes, Ø., Becker, T.S., and Lenhard, B. (2007). Genomic regulatory blocks underlie extensive microsynteny conservation in insects. Genome Research <em>17</em>, 1898–1908.</p>
</div>
<div id="ref-Fantes1995">
<p>Fantes, J., Redeker, B., Breen, M., Boyle, S., Brown, J., Fletcher, J., Jones, S., Bickmore, W., Fukushima, Y., Mannens, M., et al. (1995). Aniridia-associated cytogenetic rearrangements suggest that a position effect may cause the mutant phenotype. Human Molecular Genetics <em>4</em>, 415–422.</p>
</div>
<div id="ref-Farre2015">
<p>Farré, M., Robinson, T.J., and Ruiz-Herrera, A. (2015). An Integrative Breakage Model of genome architecture, reshuffling and evolution. BioEssays n/a–n/a.</p>
</div>
<div id="ref-Filippova2014">
<p>Filippova, D., Patro, R., Duggal, G., and Kingsford, C. (2014). Identification of alternative topological domains in chromatin. Algorithms for Molecular Biology : AMB <em>9</em>, 14.</p>
</div>
<div id="ref-Firth2009">
<p>Firth, H.V., Richards, S.M., Bevan, a P., Clayton, S., Corpas, M., Rajan, D., Van Vooren, S., Moreau, Y., Pettett, R.M., and Carter, N.P. (2009). DECIPHER: Database of Chromosomal Imbalance and Phenotype in Humans Using Ensembl Resources. American Journal of Human Genetics <em>84</em>, 524–533.</p>
</div>
<div id="ref-Flavahan2016">
<p>Flavahan, W. a, Drier, Y., Liau, B.B., Gillespie, S.M., Venteicher, a S., Stemmer-Rachamimov, a O., Suva, M.L., and Bernstein, B.E. (2016). Insulator dysfunction and oncogene activation in IDH mutant gliomas. Nature <em>529</em>, 110–114.</p>
</div>
<div id="ref-Flicek2014">
<p>Flicek, P., Amode, M.R., Barrell, D., Beal, K., Billis, K., Brent, S., Carvalho-Silva, D., Clapham, P., Coates, G., Fitzgerald, S., et al. (2014). Ensembl 2014. Nucleic Acids Research <em>42</em>, D749–D755.</p>
</div>
<div id="ref-Flomen1998">
<p>Flomen, R.H., Vatcheva, R., Gorman, P.A., Baptista, P.R., Groet, J., Barišić, I., Ligutic, I., and Nižetić, D. (1998). Construction and Analysis of a Sequence-Ready Map in 4q25: Rieger Syndrome Can Be Caused by Haploinsufficiency ofRIEG, but Also by Chromosome Breaks ≈90 kb Upstream of This Gene. Genomics <em>47</em>, 409–413.</p>
</div>
<div id="ref-Flyamer2017">
<p>Flyamer, I.M., Gassler, J., Imakaev, M., Ulyanov, S.V., Abdennur, N., Razin, S.V., Mirny, L., and Tachibana-Konwalski, K. (2017). Single-cell Hi-C reveals unique chromatin reorganization at oocyte-tozygote transition. Nature Publishing Group <em>544</em>, 1–17.</p>
</div>
<div id="ref-Forcato2017">
<p>Forcato, M., Nicoletti, C., Pal, K., Livi, C.M., Ferrari, F., and Bicciato, S. (2017). Comparison of computational methods for Hi-C data analysis. Nature Methods 14–19.</p>
</div>
<div id="ref-Forrest2014">
<p>Forrest, A.R.R., Kawaji, H., Rehli, M., Baillie, J.K., Hoon, M.J.L. de, Lassmann, T., Itoh, M., Summers, K.M., Suzuki, H., Daub, C.O., et al. (2014). A promoter-level mammalian expression atlas. Nature <em>507</em>, 462–470.</p>
</div>
<div id="ref-Franke2016">
<p>Franke, M., Ibrahim, D.M., Andrey, G., Schwarzer, W., Heinrich, V., Schöpflin, R., Kraft, K., Kempfer, R., Jerković, I., Chan, W.-L., et al. (2016). Formation of new chromatin domains determines pathogenicity of genomic duplications. Nature 1–15.</p>
</div>
<div id="ref-Fraser2015">
<p>Fraser, J., Ferrai, C., Chiariello, A.M., Schueler, M., Rito, T., Laudanno, G., Barbieri, M., Moore, B.L., Kraemer, D.C., Aitken, S., et al. (2015). Hierarchical folding and reorganization of chromosomes are linked to transcriptional changes in cellular differentiation. Mol Syst Biol <em>11</em>, 1–14.</p>
</div>
<div id="ref-Fullwood2009">
<p>Fullwood, M.J., Liu, M.H., Pan, Y.F., Liu, J., Xu, H., Mohamed, Y.B., Orlov, Y.L., Velkov, S., Ho, A., Mei, P.H., et al. (2009). An oestrogen-receptor-<span class="math inline">\(\alpha\)</span>-bound human chromatin interactome. Nature <em>462</em>, 58–64.</p>
</div>
<div id="ref-Galil1986">
<p>Galil, Z. (1986). Efficient algorithms for finding maximum matching in graphs. ACM Computing Surveys <em>18</em>, 23–38.</p>
</div>
<div id="ref-Gibcus2013">
<p>Gibcus, J.H., and Dekker, J. (2013). The hierarchy of the 3D genome. Molecular Cell <em>49</em>, 773–782.</p>
</div>
<div id="ref-Gomez-Marin2015">
<p>Gómez-Marín, C., Tena, J.J., Acemel, R.D., López-Mayorga, M., Naranjo, S., Calle-Mustienes, E. de la, Maeso, I., Beccari, L., Aneas, I., Vielmas, E., et al. (2015). Evolutionary comparison reveals that diverging CTCF sites are signatures of ancestral topological associating domains borders. Proceedings of the National Academy of Sciences <em>112</em>, 201505463.</p>
</div>
<div id="ref-Harmston2017">
<p>Harmston, N., Ing-Simmons, E., Tan, G., Perry, M., Merkenschlager, M., and Lenhard, B. (2017). Topologically associating domains are ancient features that coincide with Metazoan clusters of extreme noncoding conservation. Nature Communications <em>8</em>, 441.</p>
</div>
<div id="ref-Hay2016">
<p>Hay, D., Hughes, J.R., Babbs, C., Davies, J.O.J., Graham, B.J., Hanssen, L.L.P., Kassouf, M.T., Oudelaar, A.M., Sharpe, J.A., Suciu, M.C., et al. (2016). Genetic dissection of the <span class="math inline">\(\alpha\)</span>-globin super-enhancer in vivo. Nature Genetics <em>48</em>, 895–903.</p>
</div>
<div id="ref-He2005">
<p>He, X., and Zhang, J. (2005). Gene complexity and gene duplicability. Current Biology <em>15</em>, 1016–1021.</p>
</div>
<div id="ref-Heger2013">
<p>Heger, A., Webber, C., Goodson, M., Ponting, C.P., and Lunter, G. (2013). GAT: a simulation framework for testing the association of genomic intervals. Bioinformatics (Oxford, England) <em>29</em>, 2046–2048.</p>
</div>
<div id="ref-Herrero2016">
<p>Herrero, J., Muffato, M., Beal, K., Fitzgerald, S., Gordon, L., Pignatelli, M., Vilella, A.J., Searle, S.M.J., Amode, R., Brent, S., et al. (2016). Ensembl comparative genomics resources. Database <em>2016</em>, bav096.</p>
</div>
<div id="ref-Higgins2008">
<p>Higgins, A.W., Alkuraya, F.S., Bosco, A.F., Brown, K.K., Bruns, G.A., Donovan, D.J., Eisenman, R., Fan, Y., Farra, C.G., Ferguson, H.L., et al. (2008). Characterization of Apparently Balanced Chromosomal Rearrangements from the Developmental Genome Anatomy Project. American Journal of Human Genetics <em>82</em>, 712–722.</p>
</div>
<div id="ref-Hinrichs2006">
<p>Hinrichs, A.S., Karolchik, D., Baertsch, R., Barber, G.P., Bejerano, G., Clawson, H., Diekhans, M., Furey, T.S., Harte, R.A., Hsu, F., et al. (2006). The UCSC Genome Browser Database: update 2006. Nucleic Acids Res <em>34</em>, D590—–D598.</p>
</div>
<div id="ref-Hinsch2006">
<p>Hinsch, H., and Hannenhalli, S. (2006). Recurring genomic breaks in independent lineages support genomic fragility. BMC Evolutionary Biology <em>6</em>, 90.</p>
</div>
<div id="ref-Hnisz2016">
<p>Hnisz, D., Weintraub, A.S., Day, D.S., Valton, A.-l., Bak, R.O., Li, C.H., Goldmann, J., Lajoie, B.R., Fan, Z.P., Sigova, A.A., et al. (2016). Activation of proto-oncogenes by disruption of chromosome neighborhoods. Science (New York, N.Y.) <em>351</em>, 1454–1458.</p>
</div>
<div id="ref-Hou2012">
<p>Hou, C., Li, L., Qin, Z.S., and Corces, V.G. (2012). Gene Density, Transcription, and Insulators Contribute to the Partition of the Drosophila Genome into Physical Domains. Molecular Cell <em>48</em>, 471–484.</p>
</div>
<div id="ref-Hsieh2015">
<p>Hsieh, T.-H.S., Weiner, A., Lajoie, B., Dekker, J., Friedman, N., and Rando, O.J. (2015). Mapping Nucleosome Resolution Chromosome Folding in Yeast by Micro-C. Cell 1–12.</p>
</div>
<div id="ref-Huang2010">
<p>Huang, N., Lee, I., Marcotte, E.M., and Hurles, M.E. (2010). Characterising and Predicting Haploinsufficiency in the Human Genome. PLoS Genetics <em>6</em>, e1001154.</p>
</div>
<div id="ref-Huber2015">
<p>Huber, W., Carey, V.J., Gentleman, R., Anders, S., Carlson, M., Carvalho, B.S., Bravo, H.C., Davis, S., Gatto, L., Girke, T., et al. (2015). Orchestrating high-throughput genomic analysis with Bioconductor. Nature Methods <em>12</em>, 115–121.</p>
</div>
<div id="ref-Huminiecki2004">
<p>Huminiecki, L. (2004). Divergence of Spatial Gene Expression Profiles Following Species-Specific Gene Duplications in Human and Mouse. Genome Research <em>14</em>, 1870–1879.</p>
</div>
<div id="ref-Iafrate2004">
<p>Iafrate, A.J., Feuk, L., Rivera, M.N., Listewnik, M.L., Donahoe, P.K., Qi, Y., Scherer, S.W., and Lee, C. (2004). Detection of large-scale variation in the human genome. Nature Genetics <em>36</em>, 949–951.</p>
</div>
<div id="ref-Ibn-Salem2014">
<p>Ibn-Salem, J., Köhler, S., Love, M.I., Chung, H.-R., Huang, N., Hurles, M.E., Haendel, M., Washington, N.L., Smedley, D., Mungall, C.J., et al. (2014). Deletions of chromosomal regulatory boundaries are associated with congenital disease. Genome Biology <em>15</em>, 423.</p>
</div>
<div id="ref-Ibn-Salem2017">
<p>Ibn-Salem, J., Muro, E.M., and Andrade-Navarro, M.A. (2017). Co-regulation of paralog genes in the three-dimensional chromatin architecture. Nucleic Acids Research <em>45</em>, 81–91.</p>
</div>
<div id="ref-Kent2002">
<p>Kent, W.J., Sugnet, C.W., Furey, T.S., Roskin, K.M., Pringle, T.H., Zahler, A.M., and Haussler, D. (2002). The human genome browser at UCSC. Genome Research <em>12</em>, 996–1006.</p>
</div>
<div id="ref-Kent2003">
<p>Kent, W.J., Baertsch, R., Hinrichs, A., Miller, W., and Haussler, D. (2003). Evolution’s cauldron: duplication, deletion, and rearrangement in the mouse and human genomes. Proceedings of the National Academy of Sciences of the United States of America <em>100</em>, 11484–11489.</p>
</div>
<div id="ref-Kikuta2007">
<p>Kikuta, H., Laplante, M., Navratilova, P., Komisarczuk, A.Z., Engström, P.G., Fredman, D., Akalin, A., Caccamo, M., Sealy, I., Howe, K., et al. (2007). Genomic regulatory blocks encompass multiple neighboring genes and maintain conserved synteny in vertebrates. Genome Research <em>17</em>, 545–555.</p>
</div>
<div id="ref-Kim2008">
<p>Kim, W.K., and Marcotte, E.M. (2008). Age-dependent evolution of the yeast protein interaction network suggests a limited role of gene duplication and divergence. PLoS Computational Biology <em>4</em>, e1000232.</p>
</div>
<div id="ref-Kleinjan1998">
<p>Kleinjan, D., and Heyningen, V. van (1998). Position effect in human genetic disease. Human Molecular Genetics <em>7</em>, 1611–1618.</p>
</div>
<div id="ref-Kleinjan2005">
<p>Kleinjan, D.A., and Van Heyningen, V. (2005). Long-Range Control of Gene Expression: Emerging Mechanisms and Disruption in Disease. Am. J. Hum. Genet <em>76</em>, 8–32.</p>
</div>
<div id="ref-Knight2013">
<p>Knight, P.a., and Ruiz, D. (2013). A fast algorithm for matrix balancing. IMA Journal of Numerical Analysis <em>33</em>, 1029–1047.</p>
</div>
<div id="ref-Koonin2005">
<p>Koonin, E.V. (2005). Orthologs, paralogs, and evolutionary genomics. Annual Review of Genetics <em>39</em>, 309–338.</p>
</div>
<div id="ref-Kohler2014">
<p>Köhler, S., Doelken, S.C., Mungall, C.J., Bauer, S., Firth, H.V., Bailleul-Forestier, I., Black, G.C.M., Brown, D.L., Brudno, M., Campbell, J., et al. (2014). The Human Phenotype Ontology project: linking molecular biology and disease through phenotype data. Nucleic Acids Research <em>42</em>, D966–74.</p>
</div>
<div id="ref-DeLaat2007">
<p>Laat, W. de, and Grosveld, F. (2007). Inter-chromosomal gene regulation in the mammalian cell nucleus. Current Opinion in Genetics &amp; Development <em>17</em>, 456–464.</p>
</div>
<div id="ref-Lan2016">
<p>Lan, X., and Pritchard, J.K. (2016). Coregulation of tandem duplicate genes slows evolution of subfunctionalization in mammals. Science (New York, N.Y.) <em>352</em>, 1009–1013.</p>
</div>
<div id="ref-Lawrence2009">
<p>Lawrence, M., Gentleman, R., and Carey, V. (2009). rtracklayer: an R package for interfacing with genome browsers. Bioinformatics <em>25</em>, 1841–1842.</p>
</div>
<div id="ref-Lawrence2013">
<p>Lawrence, M., Huber, W., Pagès, H., Aboyoun, P., Carlson, M., Gentleman, R., Morgan, M.T., and Carey, V.J. (2013). Software for computing and annotating genomic ranges. PLoS Computational Biology <em>9</em>, e1003118.</p>
</div>
<div id="ref-LeDily2014">
<p>Le Dily, F., Bau, D., Pohl, a., Vicent, G.P., Serra, F., Soronellas, D., Castellano, G., Wright, R.H.G., Ballare, C., Filion, G., et al. (2014). Distinct structural transitions of chromatin topological domains correlate with coordinated hormone-induced gene regulation. Genes &amp; Development <em>28</em>, 2151–2162.</p>
</div>
<div id="ref-Lieberman-Aiden2009">
<p>Lieberman-Aiden, E., Berkum, N.L. van, Williams, L., Imakaev, M., Ragoczy, T., Telling, A., Amit, I., Lajoie, B.R., Sabo, P.J., Dorschner, M.O., et al. (2009). Comprehensive mapping of long-range interactions reveals folding principles of the human genome. Science (New York, N.Y.) <em>326</em>, 289–293.</p>
</div>
<div id="ref-Ligon2005">
<p>Ligon, A.H., Moore, S.D., Parisi, M.A., Mealiffe, M.E., Harris, D.J., Ferguson, H.L., Quade, B.J., and Morton, C.C. (2005). Constitutional Rearrangement of the Architectural Factor HMGA2: A Novel Human Phenotype Including Overgrowth and Lipomas. The American Journal of Human Genetics <em>76</em>, 340–348.</p>
</div>
<div id="ref-Long2016">
<p>Long, H.K., Prescott, S.L., and Wysocka, J. (2016). Ever-Changing Landscapes: Transcriptional Enhancers in Development and Evolution. Cell <em>167</em>, 1170–1187.</p>
</div>
<div id="ref-Lupianez2015">
<p>Lupiáñez, D.G., Kraft, K., Heinrich, V., Krawitz, P., Brancati, F., Klopocki, E., Horn, D., Kayserili, H., Opitz, J.M., Laxova, R., et al. (2015). Disruptions of Topological Chromatin Domains Cause Pathogenic Rewiring of Gene-Enhancer Interactions. Cell <em>161</em>, 1012–1025.</p>
</div>
<div id="ref-Makova2003">
<p>Makova, K.D.K., and Li, W.-H.W. (2003). Divergence in the spatial pattern of gene expression between human duplicate genes. Genome Research <em>13</em>, 1638–1645.</p>
</div>
<div id="ref-Merkenschlager2016">
<p>Merkenschlager, M., and Nora, E.P. (2016). CTCF and Cohesin in Genome Folding and Transcriptional Gene Regulation. Annual Review of Genomics and Human Genetics <em>17</em>, 17–43.</p>
</div>
<div id="ref-Mifsud2015">
<p>Mifsud, B., Tavares-Cadete, F., Young, A.N., Sugar, R., Schoenfelder, S., Ferreira, L., Wingett, S.W., Andrews, S., Grey, W., Ewels, P. a, et al. (2015). Mapping long-range promoter contacts in human cells with high-resolution capture Hi-C. Nature Genetics <em>47</em>.</p>
</div>
<div id="ref-Mills2006">
<p>Mills, R.E., Bennett, E.A., Iskow, R.C., Luttig, C.T., Tsui, C., Pittard, W.S., and Devine, S.E. (2006). Recently Mobilized Transposons in the Human and Chimpanzee Genomes. The American Journal of Human Genetics <em>78</em>, 671–679.</p>
</div>
<div id="ref-Mizuguchi2014">
<p>Mizuguchi, T., Fudenberg, G., Mehta, S., Belton, J.-M., Taneja, N., Folco, H.D., FitzGerald, P., Dekker, J., Mirny, L., Barrowman, J., et al. (2014). Cohesin-dependent globules and heterochromatin shape 3D genome architecture in S. pombe. Nature.</p>
</div>
<div id="ref-Monahan2015">
<p>Monahan, K., and Lomvardas, S. (2015). Monoallelic Expression of Olfactory Receptors. Annual Review of Cell and Developmental Biology <em>31</em>, annurev–cellbio–100814–125308.</p>
</div>
<div id="ref-Montavon2012">
<p>Montavon, T., Thevenet, L., and Duboule, D. (2012). Impact of copy number variations (CNVs) on long-range gene regulation at the HoxD locus. Proceedings of the National Academy of Sciences of the United States of America <em>109</em>, 20204–20211.</p>
</div>
<div id="ref-Murphy2005">
<p>Murphy, W.J., Larkin, D.M., Everts-van der Wind, A., Bourque, G., Tesler, G., Auvil, L., Beever, J.E., Chowdhary, B.P., Galibert, F., Gatzke, L., et al. (2005). Dynamics of mammalian chromosome evolution inferred from multispecies comparative maps. Science (New York, N.Y.) <em>309</em>, 613–617.</p>
</div>
<div id="ref-Nagano2017">
<p>Nagano, T., Lubling, Y., Várnai, C., Dudley, C., Leung, W., Baran, Y., and Mendelson-cohen, N. (2017). Cell-cycle dynamics of chromosomal organization at single-cell resolution. Nature Publishing Group <em>547</em>, 61–67.</p>
</div>
<div id="ref-Narendra2015">
<p>Narendra, V., Rocha, P.P., An, D., Raviram, R., Skok, J.A., Mazzoni, E.O., and Reinberg, D. (2015). Transcription. CTCF establishes discrete functional chromatin domains at the Hox clusters during differentiation. Science (New York, N.Y.) <em>347</em>, 1017–1021.</p>
</div>
<div id="ref-Newman2015">
<p>Newman, S., Hermetz, K.E., Weckselblatt, B., and Rudd, M.K. (2015). Next-Generation Sequencing of Duplication CNVs Reveals that Most Are Tandem and Some Create Fusion Genes at Breakpoints. The American Journal of Human Genetics <em>96</em>, 1–13.</p>
</div>
<div id="ref-Noordermeer2011">
<p>Noordermeer, D., Wit, E. de, Klous, P., Werken, H. van de, Simonis, M., Lopez-Jones, M., Eussen, B., Klein, A. de, Singer, R.H., and Laat, W. de (2011). Variegated gene expression caused by cell-specific long-range DNA interactions. Nature Cell Biology <em>13</em>, 944–951.</p>
</div>
<div id="ref-Nora2012">
<p>Nora, E.P., Lajoie, B.R., Schulz, E.G., Giorgetti, L., Okamoto, I., Servant, N., Piolot, T., Berkum, N.L. van, Meisig, J., Sedat, J., et al. (2012). Spatial partitioning of the regulatory landscape of the X-inactivation centre. Nature <em>485</em>, 381–385.</p>
</div>
<div id="ref-Nora2013">
<p>Nora, E.P., Dekker, J., and Heard, E. (2013). Segmental folding of chromosomes: A basis for structural and regulatory chromosomal neighborhoods? BioEssays <em>35</em>, 818–828.</p>
</div>
<div id="ref-Northcott2014">
<p>Northcott, P.a., Lee, C., Zichner, T., Stütz, A.M., Erkek, S., Kawauchi, D., Shih, D.J.H., Hovestadt, V., Zapatka, M., Sturm, D., et al. (2014). Enhancer hijacking activates GFI1 family oncogenes in medulloblastoma. Nature.</p>
</div>
<div id="ref-Oldridge2015">
<p>Oldridge, D.A., Wood, A.C., Weichert-Leahey, N., Crimmins, I., Sussman, R., Winter, C., McDaniel, L.D., Diamond, M., Hart, L.S., Zhu, S., et al. (2015). Genetic predisposition to neuroblastoma mediated by a LMO1 super-enhancer polymorphism. Nature <em>528</em>, 418–421.</p>
</div>
<div id="ref-Ordulu2016">
<p>Ordulu, Z., Kammin, T., Brand, H., Pillalamarri, V., Redin, C.E., Collins, R.L., Blumenthal, I., Hanscom, C., Pereira, S., Crandall, B.F., et al. (2016). Structural Chromosomal Rearrangements Require Nucleotide-Level Resolution : Lessons from Next-Generation Sequencing in Prenatal Diagnosis. The American Journal of Human Genetics 1–19.</p>
</div>
<div id="ref-Petryszak2015">
<p>Petryszak, R., Keays, M., Tang, Y.A., Fonseca, N.A., Barrera, E., Burdett, T., Füllgrabe, A., Fuentes, A.M.-P., Jupp, S., Koskinen, S., et al. (2015). Expression Atlas update—an integrated database of gene and protein expression in humans, animals and plants. Nucleic Acids Research <em>44</em>, gkv1045.</p>
</div>
<div id="ref-Pevzner2003">
<p>Pevzner, P., and Tesler, G. (2003). Human and mouse genomic sequences reveal extensive breakpoint reuse in mammalian evolution. Proceedings of the National Academy of Sciences of the United States of America <em>100</em>, 7672–7677.</p>
</div>
<div id="ref-Phillips-Cremins2013">
<p>Phillips-Cremins, J.E., Sauria, M.E.G., Sanyal, A., Gerasimova, T.I., Lajoie, B.R., Bell, J.S.K., Ong, C.-T., Hookway, T. a, Guo, C., Sun, Y., et al. (2013). Architectural protein subclasses shape 3D organization of genomes during lineage commitment. Cell <em>153</em>, 1281–1295.</p>
</div>
<div id="ref-Polychronopoulos2017">
<p>Polychronopoulos, D., King, J.W.D., Nash, A.J., Tan, G., and Lenhard, B. (2017). Conserved non-coding elements: developmental gene regulation meets genome organization. Nucleic Acids Research 1–14.</p>
</div>
<div id="ref-Pope2014">
<p>Pope, B.D., Ryba, T., Dileep, V., Yue, F., Wu, W., Denas, O., Vera, D.L., Wang, Y., Hansen, R.S., Canfield, T.K., et al. (2014). Topologically associating domains are stable units of replication-timing regulation. Nature <em>515</em>, 402–405.</p>
</div>
<div id="ref-Ptashne1986">
<p>Ptashne, M. (1986). Gene regulation by proteins acting nearby and at a distance. Nature <em>322</em>, 697–701.</p>
</div>
<div id="ref-Purmann2007">
<p>Purmann, A., Toedling, J., Schueler, M., Carninci, P., Lehrach, H., Hayashizaki, Y., Huber, W., and Sperling, S. (2007). Genomic organization of transcriptomes in mammals: Coregulation and cofunctionality. Genomics <em>89</em>, 580–587.</p>
</div>
<div id="ref-Quinn2016">
<p>Quinn, J.J., and Chang, H.Y. (2016). Unique features of long non-coding RNA biogenesis and function. Nature Reviews Genetics <em>17</em>, 47–62.</p>
</div>
<div id="ref-Rao2014">
<p>Rao, S.S.P., Huntley, M.H., Durand, N.C., Stamenova, E.K., Bochkov, I.D., Robinson, J.T., Sanborn, A.L., Machol, I., Omer, A.D., Lander, E.S., et al. (2014). A 3D map of the human genome at kilobase resolution reveals principles of chromatin looping. Cell <em>159</em>, 1665–1680.</p>
</div>
<div id="ref-Redin2016">
<p>Redin, C., Brand, H., Collins, R.L., Kammin, T., Mitchell, E., Hodge, J.C., Hanscom, C., Pillalamarri, V., Seabra, C.M., Abbott, M.-A., et al. (2016). The genomic landscape of balanced cytogenetic abnormalities associated with human congenital anomalies. Nature Genetics.</p>
</div>
<div id="ref-Rehm2015">
<p>Rehm, H.L., Berg, J.S., Brooks, L.D., Bustamante, C.D., Evans, J.P., Landrum, M.J., Ledbetter, D.H., Maglott, D.R., Martin, C.L., Nussbaum, R.L., et al. (2015). ClinGen — The Clinical Genome Resource. New England Journal of Medicine <em>372</em>, 2235–2242.</p>
</div>
<div id="ref-Rogozin2014">
<p>Rogozin, I.B., Managadze, D., Shabalina, S.A., and Koonin, E.V. (2014). Gene family level comparative analysis of gene expression n mammals validates the ortholog conjecture. Genome Biology and Evolution <em>6</em>, 754–762.</p>
</div>
<div id="ref-Roukos2014">
<p>Roukos, V., and Misteli, T. (2014). The biogenesis of chromosome translocations. Nature Cell Biology <em>16</em>, 293–300.</p>
</div>
<div id="ref-Roukos2013">
<p>Roukos, V., Voss, T.C., Schmidt, C.K., Lee, S., Wangsa, D., and Misteli, T. (2013). Spatial dynamics of chromosome translocations in living cells. Science <em>341</em>, 660–664.</p>
</div>
<div id="ref-Sanyal2012">
<p>Sanyal, A., Lajoie, B.R., Jain, G., and Dekker, J. (2012). The long-range interaction landscape of gene promoters. Nature <em>489</em>, 109–113.</p>
</div>
<div id="ref-Sati2017">
<p>Sati, S., and Cavalli, G. (2017). Chromosome conformation capture technologies and their impact in understanding genome function. Chromosoma <em>126</em>, 33–44.</p>
</div>
<div id="ref-Schaefer2012">
<p>Schaefer, M.H., Fontaine, J.-F., Vinayagam, A., Porras, P., Wanker, E.E., and Andrade-Navarro, M. a (2012). HIPPIE: Integrating protein interaction networks with experiment based quality scores. PloS One <em>7</em>, e31826.</p>
</div>
<div id="ref-Schmitt2016">
<p>Schmitt, A.D., Hu, M., and Ren, B. (2016). Genome-wide mapping and analysis of chromosome architecture. Nature Reviews Molecular Cell Biology.</p>
</div>
<div id="ref-Schoenfelder2015">
<p>Schoenfelder, S., Furlan-magaril, M., Mifsud, B., Tavares-cadete, F., Sugar, R., Javierre, B.-m., Nagano, T., Katsman, Y., Sakthidevi, M., Wingett, S.W., et al. (2015). The pluripotent regulatory circuitry connecting promoters to their long-range interacting elements. Genome Research 1–16.</p>
</div>
<div id="ref-Sekelja2016">
<p>Sekelja, M., Paulsen, J., and Collas, P. (2016). 4D nucleomes in single cells: what can computational modeling reveal about spatial chromatin conformation? Genome Biology <em>17</em>, 54.</p>
</div>
<div id="ref-Sexton2012">
<p>Sexton, T., Yaffe, E., Kenigsberg, E., Bantignies, F., Leblanc, B., Hoichman, M., Parrinello, H., Tanay, A., and Cavalli, G. (2012). Three-dimensional folding and functional organization principles of the Drosophila genome. Cell <em>148</em>, 458–472.</p>
</div>
<div id="ref-Shen2012">
<p>Shen, Y., Yue, F., McCleary, D.F., Ye, Z., Edsall, L., Kuan, S., Wagner, U., Dixon, J., Lee, L., Lobanenkov, V.V., et al. (2012). A map of the cis-regulatory sequences in the mouse genome. Nature <em>488</em>, 116–120.</p>
</div>
<div id="ref-Shlyueva2014">
<p>Shlyueva, D., Stampfel, G., and Stark, A. (2014). Transcriptional enhancers: from properties to genome-wide predictions. Nature Reviews Genetics <em>15</em>, 272–286.</p>
</div>
<div id="ref-Simonis2006">
<p>Simonis, M., Klous, P., Splinter, E., Moshkin, Y., Willemsen, R., Wit, E. de, Steensel, B. van, and Laat, W. de (2006). Nuclear organization of active and inactive chromatin domains uncovered by chromosome conformation capture–on-chip (4C). Nature Genetics <em>38</em>, 1348–1354.</p>
</div>
<div id="ref-Song2010">
<p>Song, L., and Crawford, G.E. (2010). DNase-seq: a high-resolution technique for mapping active gene regulatory elements across the genome from mammalian cells. Cold Spring Harbor Protocols <em>2010</em>, pdb.prot5384.</p>
</div>
<div id="ref-Spielmann2016">
<p>Spielmann, M., and Mundlos, S. (2016). Looking beyond the genes: the role of non-coding variants in human disease. Human Molecular Genetics ddw205.</p>
</div>
<div id="ref-Spielmann2012">
<p>Spielmann, M., Brancati, F., Krawitz, P.M., Robinson, P.N., Ibrahim, D.M., Franke, M., Hecht, J., Lohan, S., Dathe, K., Nardone, A.M., et al. (2012). Homeotic arm-to-leg transformation associated with genomic rearrangements at the PITX1 locus. American Journal of Human Genetics <em>91</em>, 629–635.</p>
</div>
<div id="ref-Spitz2012">
<p>Spitz, F., and Furlong, E.E.M. (2012). Transcription factors: from enhancer binding to developmental control. Nature Reviews. Genetics <em>13</em>, 613–626.</p>
</div>
<div id="ref-Sproul2005">
<p>Sproul, D., Gilbert, N., and Bickmore, W. a (2005). The role of chromatin structure in regulating the expression of clustered genes. Nature Reviews. Genetics <em>6</em>, 775–781.</p>
</div>
<div id="ref-Stevens2017">
<p>Stevens, T.J., Lando, D., Basu, S., Liam, P., Cao, Y., Lee, S.F., Leeb, M., Wohlfahrt, K.J., Boucher, W., Shaughnessy-kirwan, A.O., et al. (2017). 3D structures of individual mammalian genomes studied by single-cell Hi-C. Nature 1–21.</p>
</div>
<div id="ref-Symmons2014">
<p>Symmons, O., Uslu, V.V., Tsujimura, T., Ruf, S., Nassari, S., Schwarzer, W., Ettwiller, L., and Spitz, F. (2014). Functional and topological characteristics of mammalian regulatory domains. Genome Research <em>24</em>, 390–400.</p>
</div>
<div id="ref-Tang2015">
<p>Tang, Z., Luo, O.J., Li, X., Zheng, M., Zhu, J.J., Szalaj, P., Trzaskoma, P., Magalska, A., Wlodarczyk, J., Ruszczycki, B., et al. (2015). CTCF-Mediated Human 3D Genome Architecture Reveals Chromatin Topology for Transcription. Cell 1–17.</p>
</div>
<div id="ref-Theisen2010">
<p>Theisen, A., and Shaffer, L.G. (2010). Disorders caused by chromosome abnormalities. The Application of Clinical Genetics <em>3</em>, 159–174.</p>
</div>
<div id="ref-Thurman2012">
<p>Thurman, R.E., Rynes, E., Humbert, R., Vierstra, J., Maurano, M.T., Haugen, E., Sheffield, N.C., Stergachis, A.B., Wang, H., Vernot, B., et al. (2012). The accessible chromatin landscape of the human genome. Nature <em>489</em>, 75–82.</p>
</div>
<div id="ref-Tolhuis2002">
<p>Tolhuis, B., Palstra, R.J., Splinter, E., Grosveld, F., and Laat, W. de (2002). Looping and interaction between hypersensitive sites in the active beta-globin locus. Mol Cell <em>10</em>, 1453–1465.</p>
</div>
<div id="ref-Velagaleti2005">
<p>Velagaleti, G.V., Bien-Willner, G.A., Northup, J.K., Lockhart, L.H., Hawkins, J.C., Jalal, S.M., Withers, M., Lupski, J.R., and Stankiewicz, P. (2005). Position Effects Due to Chromosome Breakpoints that Map ∼900 Kb Upstream and ∼1.3 Mb Downstream of SOX9 in Two Patients with Campomelic Dysplasia. The American Journal of Human Genetics <em>76</em>, 652–662.</p>
</div>
<div id="ref-VietriRudan2015">
<p>Vietri Rudan, M., Barrington, C., Henderson, S., Ernst, C., Odom, D.T., Tanay, A., and Hadjur, S. (2015). Comparative Hi-C reveals that CTCF underlies evolution of chromosomal domain architecture. Cell Reports <em>10</em>, 1297–1309.</p>
</div>
<div id="ref-Vilella2009">
<p>Vilella, A.J., Severin, J., Ureta-Vidal, A., Heng, L., Durbin, R., and Birney, E. (2009). EnsemblCompara GeneTrees: Complete, duplication-aware phylogenetic trees in vertebrates. Genome Research <em>19</em>, 327–335.</p>
</div>
<div id="ref-Visser2012">
<p>Visser, M., Kayser, M., and Palstra, R.-J. (2012). HERC2 rs12913832 modulates human pigmentation by attenuating chromatin-loop formation between a long-range enhancer and the OCA2 promoter. Genome Research <em>22</em>, 446–455.</p>
</div>
<div id="ref-Weiler1995">
<p>Weiler, K.S., and Wakimoto, B.T. (1995). Heterochromatin and gene expression in Drosophila. Annual Review of Genetics <em>29</em>, 577–605.</p>
</div>
<div id="ref-Weischenfeldt2016">
<p>Weischenfeldt, J., Dubash, T., Drainas, A.P., Mardin, B.R., Chen, Y., Stütz, A.M., Waszak, S.M., Bosco, G., Halvorsen, A.R., Raeder, B., et al. (2016). Pan-cancer analysis of somatic copy-number alterations implicates IRS4 and IGF2 in enhancer hijacking. Nature Genetics.</p>
</div>
<div id="ref-Wickham2017">
<p>Wickham, H., and Grolemund, G. R for data science : import, tidy, transform, visualize, and model data.</p>
</div>
<div id="ref-Will2017">
<p>Will, A.J., Cova, G., Osterwalder, M., Chan, W.-L., Wittler, L., Brieske, N., Heinrich, V., Villartay, J.-P. de, Vingron, M., Klopocki, E., et al. (2017). Composition and dosage of a multipartite enhancer cluster control developmental expression of Ihh (Indian hedgehog). Nature Genetics.</p>
</div>
<div id="ref-Xie2017">
<p>Xie, S., Duan, J., Li, B., Zhou, P., and Hon, G.C. (2017). Multiplexed Engineering and Analysis of Combinatorial Enhancer Activity in Single Cells. Molecular Cell <em>66</em>, 285–299.e5.</p>
</div>
<div id="ref-Zepeda-Mendoza2017">
<p>Zepeda-Mendoza, C.J., Ibn-Salem, J., Kammin, T., Harris, D.J., Rita, D., Gripp, K.W., MacKenzie, J.J., Gropman, A., Graham, B., Shaheen, R., et al. (2017). Computational Prediction of Position Effects of Apparently Balanced Human Chromosomal Rearrangements. American Journal of Human Genetics <em>101</em>, 206–217.</p>
</div>
<div id="ref-Zhan2017">
<p>Zhan, Y., Mariani, L., Barozzi, I., Schulz, E.G., Blüthgen, N., Stadler, M., Tiana, G., Giorgetti, L., Bluthgen, N., Stadler, M., et al. (2017). Reciprocal insulation analysis of Hi-C data shows that TADs represent a functionally but not structurally privileged scale in the hierarchical folding of chromosomes. Genome Research <em>27</em>, gr.212803.116.</p>
</div>
<div id="ref-Zhang2015">
<p>Zhang, B., and Wolynes, P.G. (2015). Topology, structures, and energy landscapes of human chromosomes. Proceedings of the National Academy of Sciences <em>112</em>, 201506257.</p>
</div>
<div id="ref-Zhang2009">
<p>Zhang, F., Gu, W., Hurles, M.E., and Lupski, J.R. (2009). Copy Number Variation in Human Health, Disease, and Evolution. Annual Review of Genomics and Human Genetics <em>10</em>, 451–481.</p>
</div>
</div>

</div>



            </section>

          </div>
        </div>
      </div>
<a href="discussion-3.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="supporting-information-co-regulation-of-paralog-genes-in-the-three-dimensional-chromatin-architecture.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"google": false,
"weibo": false,
"instapper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/ibn-salem/thesis/edit/master/07-references.Rmd",
"text": "Edit"
},
"download": ["thesis.pdf", "thesis.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "";
    if (src === "" || src === "true") src = "https://cdn.bootcss.com/mathjax/2.7.1/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:" && /^https?:/.test(src))
      src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
